<html>
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
<title>b96b42b76549d3bfd908e1d92d051f89</title>
<script type="text/javascript">
function showhide(id)
{
    var e = document.getElementById(id);
    e.style.display = (e.style.display == 'block') ? 'none' : 'block';
}
</script>
<style>
rect {
transition: .6s fill;
fill: #D3D3D3;
opacity: 0;
}
rect:hover {
    fill: #D3D3D3;
    opacity: 0.2;
}

.outline {
    clear: both;
}

.svg-container {
    width: 100%;
    max-width: 1394px;
    margin-left: auto;
    margin-right: auto;
    display: block;
}

.svg-content {
    width: 100%;
}

.container {
    width: 100%;
}
</style>

</head>

<body>

<div class="container"><div class="spacer">&nbsp;</div><div class="svg-container"><svg version="1.1"  preserveAspectRatio="xMinYMin meet" class="svg-content" viewBox="0 0 1394 8192" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
<image width="1394" height="8192" xlink:href="00092-b96b42b76549d3bfd908e1d92d051f89-assets/00092-b96b42b76549d3bfd908e1d92d051f89.png"></image>
<a xlink:href="http://doi.org/10.7554/eLife.19466" xlink:title="http://doi.org/10.7554/eLife.19466"><rect x="913" y="36" width="88" height="8"/></a>
<a xlink:href="http://doi.org/10.1073/pnas.1611028113" xlink:title="http://doi.org/10.1073/pnas.1611028113"><rect x="913" y="86" width="109" height="8"/></a>
<a xlink:href="http://doi.org/10.1017/pab.2017.21" xlink:title="http://doi.org/10.1017/pab.2017.21"><rect x="913" y="128" width="91" height="8"/></a>
<a xlink:href="http://doi.org/10.1038/s41586-019-1749-3" xlink:title="http://doi.org/10.1038/s41586-019-1749-3"><rect x="913" y="179" width="115" height="8"/></a>
<a xlink:href="http://darwin-online.org.uk/" xlink:title="http://darwin-online.org.uk/"><rect x="782" y="219" width="61" height="8"/></a>
<a xlink:href="http://wallace-online.org/" xlink:title="http://wallace-online.org/"><rect x="782" y="243" width="54" height="8"/></a>
<a xlink:href="http://www.gutenberg.org/files/2009/2009-h/2009-h.htm" xlink:title="http://www.gutenberg.org/files/2009/2009-h/2009-h.htm"><rect x="942" y="258" width="42" height="8"/></a>
<a xlink:href="http://darwin-online.org.uk/contents.html" xlink:title="http://darwin-online.org.uk/contents.html"><rect x="942" y="282" width="104" height="8"/></a>
<a xlink:href="https://archive.org/search.php?query=%28origin%20species%29%20AND%20creator%3A%28darwin%29" xlink:title="https://archive.org/search.php?query=%28origin%20species%29%20AND%20creator%3A%28darwin%29"><rect x="942" y="297" width="33" height="8"/></a>
<a xlink:href="http://doi.org/10.1242/jeb.106310" xlink:title="http://doi.org/10.1242/jeb.106310"><rect x="782" y="339" width="87" height="8"/></a>
<a xlink:href="http://doi.org/10.1098/rsfs.2016.0159" xlink:title="http://doi.org/10.1098/rsfs.2016.0159"><rect x="782" y="381" width="99" height="8"/></a>
<a xlink:href="http://doi.org/10.1007/s10539-018-9633-3" xlink:title="http://doi.org/10.1007/s10539-018-9633-3"><rect x="782" y="527" width="115" height="8"/></a>
<a xlink:href="http://doi.org/10.1111/evo.13686" xlink:title="http://doi.org/10.1111/evo.13686"><rect x="782" y="661" width="85" height="8"/></a>
<a xlink:href="http://doi.org/10.1038/s41561-019-0392-9" xlink:title="http://doi.org/10.1038/s41561-019-0392-9"><rect x="782" y="703" width="115" height="8"/></a>
<a xlink:href="http://doi.org/10.1038/s41559-019-0835-0" xlink:title="http://doi.org/10.1038/s41559-019-0835-0"><rect x="782" y="754" width="115" height="8"/></a>
<a xlink:href="https://www.ribbonfarm.com/2018/04/24/survival-of-the-mediocre-mediocre/" xlink:title="https://www.ribbonfarm.com/2018/04/24/survival-of-the-mediocre-mediocre/"><rect x="782" y="924" width="140" height="8"/></a>
<a xlink:href="https://www.ribbonfarm.com/2018/04/24/survival-of-the-mediocre-mediocre/" xlink:title="https://www.ribbonfarm.com/2018/04/24/survival-of-the-mediocre-mediocre/"><rect x="782" y="933" width="73" height="8"/></a>
<a xlink:href="http://doi.org/10.1086/697201" xlink:title="http://doi.org/10.1086/697201"><rect x="782" y="1030" width="76" height="8"/></a>
<a xlink:href="http://doi.org/10.1016/j.cell.2017.03.025" xlink:title="http://doi.org/10.1016/j.cell.2017.03.025"><rect x="782" y="1080" width="107" height="8"/></a>
<a xlink:href="http://doi.org/10.1038/srep46337" xlink:title="http://doi.org/10.1038/srep46337"><rect x="782" y="1139" width="85" height="8"/></a>
<a xlink:href="http://doi.org/10.1016/j.neuron.2017.01.033" xlink:title="http://doi.org/10.1016/j.neuron.2017.01.033"><rect x="858" y="1205" width="118" height="8"/></a>
<a xlink:href="https://www.sciencedaily.com/releases/2017/02/170223124227.htm" xlink:title="https://www.sciencedaily.com/releases/2017/02/170223124227.htm"><rect x="858" y="1256" width="149" height="8"/></a>
<a xlink:href="https://www.sciencedaily.com/releases/2017/02/170223124227.htm" xlink:title="https://www.sciencedaily.com/releases/2017/02/170223124227.htm"><rect x="858" y="1265" width="90" height="8"/></a>
<a xlink:href="http://doi.org/10.1016/j.cell.2017.01.038" xlink:title="http://doi.org/10.1016/j.cell.2017.01.038"><rect x="858" y="1324" width="107" height="8"/></a>
<a xlink:href="http://doi.org/10.1126/science.aaw0040" xlink:title="http://doi.org/10.1126/science.aaw0040"><rect x="858" y="1366" width="106" height="8"/></a>
<a xlink:href="http://doi.org/10.1038/s41559-016-0038" xlink:title="http://doi.org/10.1038/s41559-016-0038"><rect x="782" y="1407" width="109" height="8"/></a>
<a xlink:href="http://doi.org/10.1038/ncomms13229" xlink:title="http://doi.org/10.1038/ncomms13229"><rect x="921" y="1457" width="98" height="8"/></a>
<a xlink:href="http://doi.org/10.1038/ncomms12461" xlink:title="http://doi.org/10.1038/ncomms12461"><rect x="858" y="1491" width="98" height="8"/></a>
<a xlink:href="http://doi.org/10.1126/science.aah4204" xlink:title="http://doi.org/10.1126/science.aah4204"><rect x="782" y="1533" width="104" height="8"/></a>
<a xlink:href="https://mosaicscience.com/story/ancient-human-evolution-neanderthal-genetics" xlink:title="https://mosaicscience.com/story/ancient-human-evolution-neanderthal-genetics"><rect x="782" y="1789" width="148" height="8"/></a>
<a xlink:href="https://mosaicscience.com/story/ancient-human-evolution-neanderthal-genetics" xlink:title="https://mosaicscience.com/story/ancient-human-evolution-neanderthal-genetics"><rect x="782" y="1798" width="92" height="8"/></a>
<a xlink:href="https://mobile.nytimes.com/2015/06/16/science/dna-deciphers-roots-of-modern-europeans.html?_r=0&referer=https://duckduckgo.com/" xlink:title="https://mobile.nytimes.com/2015/06/16/science/dna-deciphers-roots-of-modern-europeans.html?_r=0&amp;referer=https://duckduckgo.com/"><rect x="956" y="1897" width="143" height="8"/></a>
<a xlink:href="https://mobile.nytimes.com/2015/06/16/science/dna-deciphers-roots-of-modern-europeans.html?_r=0&referer=https://duckduckgo.com/" xlink:title="https://mobile.nytimes.com/2015/06/16/science/dna-deciphers-roots-of-modern-europeans.html?_r=0&amp;referer=https://duckduckgo.com/"><rect x="956" y="1906" width="130" height="8"/></a>
<a xlink:href="http://www.nature.com/news/european-languages-linked-to-migration-from-the-east-1.16919" xlink:title="http://www.nature.com/news/european-languages-linked-to-migration-from-the-east-1.16919"><rect x="956" y="1922" width="148" height="8"/></a>
<a xlink:href="http://www.nature.com/news/european-languages-linked-to-migration-from-the-east-1.16919" xlink:title="http://www.nature.com/news/european-languages-linked-to-migration-from-the-east-1.16919"><rect x="956" y="1930" width="99" height="8"/></a>
<a xlink:href="http://www.nature.com/articles/nature13673.epdf?referrer_access_token=dUGFQ4p5aSftizrGNtcLtdRgN0jAjWel9jnR3ZoTv0MaJb0jOMpBA1y27HQGlCUiHaqx24ysNx_tIj9u3K68hIb67qNeR62pjGLMM_oheXxudlZ3ujrsQdcQ3y1Hv43qAtla3QkjTusatPWqibFrGLtYM3oNKg5nWyQD1eagFah-7e4zquIYKbz-elLhtQJkPiggMbPOOQtzFYi1DswNxZwPkSWHeMCYkX--l9qLsMH8I6o9psIi6HKUXdio5DylzrTjc4FaiOtO4VzrDa3EPg%3D%3D&tracking_referrer=www.nature.com" xlink:title="http://www.nature.com/articles/nature13673.epdf?referrer_access_token=dUGFQ4p5aSftizrGNtcLtdRgN0jAjWel9jnR3ZoTv0MaJb0jOMpBA1y27HQGlCUiHaqx24ysNx_tIj9u3K68hIb67qNeR62pjGLMM_oheXxudlZ3ujrsQdcQ3y1Hv43qAtla3QkjTusatPWqibFrGLtYM3oNKg5nWyQD1eagFah-7e4zquIYKbz-elLhtQJkPiggMbPOOQtzFYi1DswNxZwPkSWHeMCYkX--l9qLsMH8I6o9psIi6HKUXdio5DylzrTjc4FaiOtO4VzrDa3EPg%3D%3D&amp;tracking_referrer=www.nature.com"><rect x="956" y="1963" width="114" height="8"/></a>
<a xlink:href="http://doi.org/10.1038/nature21674" xlink:title="http://doi.org/10.1038/nature21674"><rect x="956" y="2005" width="90" height="8"/></a>
<a xlink:href="http://doi.org/10.1371/journal.pcbi.1005435" xlink:title="http://doi.org/10.1371/journal.pcbi.1005435"><rect x="782" y="2046" width="116" height="8"/></a>
<a xlink:href="https://doi.org/10.1038/ncomms15172" xlink:title="https://doi.org/10.1038/ncomms15172"><rect x="782" y="2097" width="98" height="8"/></a>
<a xlink:href="http://doi.org/10.1038/s41467-019-09579-y" xlink:title="http://doi.org/10.1038/s41467-019-09579-y"><rect x="782" y="2139" width="118" height="8"/></a>
<a xlink:href="http://doi.org/10.1093/molbev/msx163" xlink:title="http://doi.org/10.1093/molbev/msx163"><rect x="782" y="2189" width="100" height="8"/></a>
<a xlink:href="http://doi.org/10.1038/s41598-018-19853-6" xlink:title="http://doi.org/10.1038/s41598-018-19853-6"><rect x="782" y="2231" width="119" height="8"/></a>
<a xlink:href="http://doi.org/10.7208/chicago/9780226384399.001.0001" xlink:title="http://doi.org/10.7208/chicago/9780226384399.001.0001"><rect x="782" y="2297" width="121" height="8"/></a>
<a xlink:href="http://doi.org/10.7208/chicago/9780226384399.001.0001" xlink:title="http://doi.org/10.7208/chicago/9780226384399.001.0001"><rect x="782" y="2306" width="84" height="8"/></a>
<a xlink:href="http://doi.org/10.1038/s41559-017-0418-x" xlink:title="http://doi.org/10.1038/s41559-017-0418-x"><rect x="949" y="2363" width="114" height="8"/></a>
<a xlink:href="http://doi.org/10.1038/s41559-018-0701-5" xlink:title="http://doi.org/10.1038/s41559-018-0701-5"><rect x="782" y="2435" width="115" height="8"/></a>
<a xlink:href="http://doi.org/10.1126/science.aao4593" xlink:title="http://doi.org/10.1126/science.aao4593"><rect x="782" y="2484" width="104" height="8"/></a>
<a xlink:href="http://doi.org/10.1038/s41586-018-0273-1" xlink:title="http://doi.org/10.1038/s41586-018-0273-1"><rect x="782" y="2631" width="115" height="8"/></a>
<a xlink:href="http://doi.org/10.1038/s41467-018-04218-4" xlink:title="http://doi.org/10.1038/s41467-018-04218-4"><rect x="782" y="2857" width="119" height="8"/></a>
<a xlink:href="http://biology.usf.edu/ib/data/flyers/SCHWARTZ-FISH-LABELING-1-2015.pdf" xlink:title="http://biology.usf.edu/ib/data/flyers/SCHWARTZ-FISH-LABELING-1-2015.pdf"><rect x="997" y="2961" width="146" height="8"/></a>
<a xlink:href="http://biology.usf.edu/ib/data/flyers/SCHWARTZ-FISH-LABELING-1-2015.pdf" xlink:title="http://biology.usf.edu/ib/data/flyers/SCHWARTZ-FISH-LABELING-1-2015.pdf"><rect x="997" y="2970" width="68" height="8"/></a>
<a xlink:href="http://io9.com/confirmation-that-photosynthesizing-sea-slugs-steal-gen-1683702602" xlink:title="http://io9.com/confirmation-that-photosynthesizing-sea-slugs-steal-gen-1683702602"><rect x="997" y="2985" width="9" height="8"/></a>
<a xlink:href="http://doi.org/10.1073/pnas.1608765113" xlink:title="http://doi.org/10.1073/pnas.1608765113"><rect x="894" y="3036" width="109" height="8"/></a>
<a xlink:href="http://doi.org/10.1038/s41467-018-04955-6" xlink:title="http://doi.org/10.1038/s41467-018-04955-6"><rect x="782" y="3095" width="119" height="8"/></a>
<a xlink:href="http://doi.org/10.7554/elife.48212" xlink:title="http://doi.org/10.7554/elife.48212"><rect x="782" y="3146" width="85" height="8"/></a>
<a xlink:href="http://doi.org/10.1038/s41467-020-14500-z" xlink:title="http://doi.org/10.1038/s41467-020-14500-z"><rect x="782" y="3197" width="118" height="8"/></a>
<a xlink:href="http://doi.org/10.1371/journal.pgen.1008615" xlink:title="http://doi.org/10.1371/journal.pgen.1008615"><rect x="782" y="3247" width="119" height="8"/></a>
<a xlink:href="http://doi.org/10.1016/j.molcel.2019.02.024" xlink:title="http://doi.org/10.1016/j.molcel.2019.02.024"><rect x="829" y="3298" width="117" height="8"/></a>
<a xlink:href="http://doi.org/10.1088/1478-3975/8/1/015001" xlink:title="http://doi.org/10.1088/1478-3975/8/1/015001"><rect x="829" y="3463" width="126" height="8"/></a>
<a xlink:href="http://doi.org/10.1002/path.5161" xlink:title="http://doi.org/10.1002/path.5161"><rect x="829" y="3611" width="83" height="8"/></a>
<a xlink:href="http://doi.org/10.1126/science.aat1699" xlink:title="http://doi.org/10.1126/science.aat1699"><rect x="829" y="3715" width="102" height="8"/></a>
<a xlink:href="http://doi.org/10.1126/science.aau9923" xlink:title="http://doi.org/10.1126/science.aau9923"><rect x="966" y="3818" width="104" height="8"/></a>
<a xlink:href="http://doi.org/10.1016/j.gde.2015.03.005" xlink:title="http://doi.org/10.1016/j.gde.2015.03.005"><rect x="966" y="3859" width="109" height="8"/></a>
<a xlink:href="http://doi.org/10.1073/pnas.1612290113" xlink:title="http://doi.org/10.1073/pnas.1612290113"><rect x="829" y="3925" width="109" height="8"/></a>
<a xlink:href="http://doi.org/10.1038/s41467-018-05367-2" xlink:title="http://doi.org/10.1038/s41467-018-05367-2"><rect x="829" y="3976" width="119" height="8"/></a>
<a xlink:href="https://dx.doi.org/10.18632%2Foncotarget.3174" xlink:title="https://dx.doi.org/10.18632%2Foncotarget.3174"><rect x="829" y="4036" width="117" height="8"/></a>
<a xlink:href="http://doi.org/10.1371/journal.pbio.2007066" xlink:title="http://doi.org/10.1371/journal.pbio.2007066"><rect x="829" y="4069" width="117" height="8"/></a>
<a xlink:href="http://doi.org/10.1016/j.it.2016.08.002" xlink:title="http://doi.org/10.1016/j.it.2016.08.002"><rect x="856" y="4111" width="100" height="8"/></a>
<a xlink:href="http://doi.org/10.1016/j.molcel.2016.11.012" xlink:title="http://doi.org/10.1016/j.molcel.2016.11.012"><rect x="957" y="4161" width="117" height="8"/></a>
<a xlink:href="http://doi.org/10.1038/nature14432" xlink:title="http://doi.org/10.1038/nature14432"><rect x="1048" y="4203" width="90" height="8"/></a>
<a xlink:href="http://doi.org/10.1172/JCI89927" xlink:title="http://doi.org/10.1172/JCI89927"><rect x="1051" y="4253" width="82" height="8"/></a>
<a xlink:href="http://doi.org/10.1016/j.celrep.2018.08.033" xlink:title="http://doi.org/10.1016/j.celrep.2018.08.033"><rect x="856" y="4304" width="116" height="8"/></a>
<a xlink:href="http://doi.org/10.1016/j.cell.2017.10.029" xlink:title="http://doi.org/10.1016/j.cell.2017.10.029"><rect x="782" y="4346" width="107" height="8"/></a>
<a xlink:href="http://doi.org/10.1111/mec.14519" xlink:title="http://doi.org/10.1111/mec.14519"><rect x="782" y="4396" width="87" height="8"/></a>
<a xlink:href="http://doi.org/10.1128/mBio.01033-18" xlink:title="http://doi.org/10.1128/mBio.01033-18"><rect x="782" y="4463" width="100" height="8"/></a>
<a xlink:href="https://doi.org/10.1016/j.devcel.2018.10.011" xlink:title="https://doi.org/10.1016/j.devcel.2018.10.011"><rect x="782" y="4505" width="117" height="8"/></a>
<a xlink:href="http://doi.org/10.1038/s41559-017-0435-9" xlink:title="http://doi.org/10.1038/s41559-017-0435-9"><rect x="782" y="4546" width="115" height="8"/></a>
<a xlink:href="http://doi.org/10.1093/gbe/evz058" xlink:title="http://doi.org/10.1093/gbe/evz058"><rect x="782" y="4605" width="88" height="8"/></a>
<a xlink:href="http://doi.org/10.1016/j.cub.2018.06.075" xlink:title="http://doi.org/10.1016/j.cub.2018.06.075"><rect x="782" y="4711" width="109" height="8"/></a>
<a xlink:href="http://doi.org/10.1038/s41559-018-0568-5" xlink:title="http://doi.org/10.1038/s41559-018-0568-5"><rect x="782" y="4836" width="115" height="8"/></a>
<a xlink:href="http://doi.org/10.1126/science.aao6809" xlink:title="http://doi.org/10.1126/science.aao6809"><rect x="782" y="4978" width="104" height="8"/></a>
<a xlink:href="http://doi.org/10.1016/j.tree.2018.05.008" xlink:title="http://doi.org/10.1016/j.tree.2018.05.008"><rect x="782" y="5011" width="109" height="8"/></a>
<a xlink:href="https://aeon.co/essays/what-the-ctenophore-says-about-the-evolution-of-intelligence" xlink:title="https://aeon.co/essays/what-the-ctenophore-says-about-the-evolution-of-intelligence"><rect x="872" y="5118" width="147" height="8"/></a>
<a xlink:href="https://aeon.co/essays/what-the-ctenophore-says-about-the-evolution-of-intelligence" xlink:title="https://aeon.co/essays/what-the-ctenophore-says-about-the-evolution-of-intelligence"><rect x="872" y="5127" width="102" height="8"/></a>
<a xlink:href="https://doi.org/10.1007/s12304-015-9242-3" xlink:title="https://doi.org/10.1007/s12304-015-9242-3"><rect x="872" y="5508" width="115" height="8"/></a>
<a xlink:href="https://doi.org/10.1098/rsbl.2009.1024" xlink:title="https://doi.org/10.1098/rsbl.2009.1024"><rect x="1033" y="5592" width="99" height="8"/></a>
<a xlink:href="http://bristol.academia.edu/SSahney/Papers" xlink:title="http://bristol.academia.edu/SSahney/Papers"><rect x="1183" y="5579" width="115" height="8"/></a>
<a xlink:href="http://doi.org/10.1111/j.1461-0248.2009.01350.x" xlink:title="http://doi.org/10.1111/j.1461-0248.2009.01350.x"><rect x="872" y="5634" width="135" height="8"/></a>
<a xlink:href="http://www.cogsci.uci.edu/~ddhoff/interface.pdf" xlink:title="http://www.cogsci.uci.edu/~ddhoff/interface.pdf"><rect x="872" y="5868" width="110" height="8"/></a>
<a xlink:href="http://doi.org/10.1093/molbev/msw127" xlink:title="http://doi.org/10.1093/molbev/msw127"><rect x="1048" y="6187" width="102" height="8"/></a>
<a xlink:href="http://doi.org/10.1016/j.anbehav.2016.08.009" xlink:title="http://doi.org/10.1016/j.anbehav.2016.08.009"><rect x="872" y="6237" width="123" height="8"/></a>
<a xlink:href="http://doi.org/10.1038/srep34077" xlink:title="http://doi.org/10.1038/srep34077"><rect x="872" y="6288" width="85" height="8"/></a>
<a xlink:href="http://doi.org/10.1016/j.gr.2016.08.003" xlink:title="http://doi.org/10.1016/j.gr.2016.08.003"><rect x="872" y="6330" width="103" height="8"/></a>
<a xlink:href="http://doi.org/10.1257/aer.103.1.1" xlink:title="http://doi.org/10.1257/aer.103.1.1"><rect x="872" y="6450" width="87" height="8"/></a>
<a xlink:href="http://doi.org/10.1038/s41559-019-0846-x" xlink:title="http://doi.org/10.1038/s41559-019-0846-x"><rect x="782" y="6561" width="114" height="8"/></a>
<a xlink:href="http://doi.org/10.1126/science.aai9067" xlink:title="http://doi.org/10.1126/science.aai9067"><rect x="782" y="6602" width="102" height="8"/></a>
<a xlink:href="http://doi.org/10.1016/j.tree.2018.05.005" xlink:title="http://doi.org/10.1016/j.tree.2018.05.005"><rect x="782" y="6653" width="109" height="8"/></a>
<a xlink:href="http://doi.org/10.1038/nature25778" xlink:title="http://doi.org/10.1038/nature25778"><rect x="782" y="6686" width="90" height="8"/></a>
<a xlink:href="http://doi.org/10.1038/nature25738" xlink:title="http://doi.org/10.1038/nature25738"><rect x="782" y="6728" width="90" height="8"/></a>
<a xlink:href="https://doi.org/10.1016/j.cell.2018.03.054" xlink:title="https://doi.org/10.1016/j.cell.2018.03.054"><rect x="782" y="6769" width="107" height="8"/></a>
<a xlink:href="http://doi.org/10.1073/pnas.1711788115" xlink:title="http://doi.org/10.1073/pnas.1711788115"><rect x="782" y="6820" width="109" height="8"/></a>
<a xlink:href="https://doi.org/10.1016/j.ajhg.2009.09.006" xlink:title="https://doi.org/10.1016/j.ajhg.2009.09.006"><rect x="782" y="6905" width="110" height="8"/></a>
<a xlink:href="http://doi.org/10.1038/s41586-018-0299-4" xlink:title="http://doi.org/10.1038/s41586-018-0299-4"><rect x="782" y="6947" width="115" height="8"/></a>
<a xlink:href="http://doi.org/10.1093/nar/gkz1197" xlink:title="http://doi.org/10.1093/nar/gkz1197"><rect x="782" y="6988" width="90" height="8"/></a>
<a xlink:href="http://doi.org/10.1038/s41467-018-03294-w" xlink:title="http://doi.org/10.1038/s41467-018-03294-w"><rect x="782" y="7045" width="120" height="8"/></a>
<a xlink:href="http://doi.org/10.7554/eLife.33014" xlink:title="http://doi.org/10.7554/eLife.33014"><rect x="782" y="7111" width="88" height="8"/></a>
<a xlink:href="http://doi.org/10.1038/s41589-018-0187-0" xlink:title="http://doi.org/10.1038/s41589-018-0187-0"><rect x="470" y="6330" width="115" height="8"/></a>
<a xlink:href="http://doi.org/10.1038/s41477-018-0311-x" xlink:title="http://doi.org/10.1038/s41477-018-0311-x"><rect x="491" y="6363" width="114" height="8"/></a>
<a xlink:href="http://doi.org/10.1038/s41467-018-07784-9" xlink:title="http://doi.org/10.1038/s41467-018-07784-9"><rect x="473" y="5598" width="119" height="8"/></a>
<a xlink:href="http://doi.org/10.1073/pnas.1807709115" xlink:title="http://doi.org/10.1073/pnas.1807709115"><rect x="469" y="5649" width="109" height="8"/></a>
<a xlink:href="http://doi.org/10.7554/elife.36317" xlink:title="http://doi.org/10.7554/elife.36317"><rect x="487" y="5708" width="85" height="8"/></a>
<a xlink:href="http://doi.org/10.1016/j.celrep.2018.07.042" xlink:title="http://doi.org/10.1016/j.celrep.2018.07.042"><rect x="471" y="5759" width="116" height="8"/></a>
<a xlink:href="http://doi.org/10.1101/gr.206706.116" xlink:title="http://doi.org/10.1101/gr.206706.116"><rect x="476" y="5810" width="97" height="8"/></a>
<a xlink:href="http://doi.org/10.1038/nn.4388" xlink:title="http://doi.org/10.1038/nn.4388"><rect x="470" y="5852" width="77" height="8"/></a>
<a xlink:href="http://doi.org/10.1371/journal.ppat.1006726" xlink:title="http://doi.org/10.1371/journal.ppat.1006726"><rect x="472" y="5902" width="117" height="8"/></a>
<a xlink:href="http://doi.org/10.1126/scisignal.aaw9277" xlink:title="http://doi.org/10.1126/scisignal.aaw9277"><rect x="469" y="5953" width="108" height="8"/></a>
<a xlink:href="http://doi.org/10.1371/journal.pbio.2006810" xlink:title="http://doi.org/10.1371/journal.pbio.2006810"><rect x="389" y="5501" width="117" height="8"/></a>
<a xlink:href="http://doi.org/10.1371/journal.pgen.1007155" xlink:title="http://doi.org/10.1371/journal.pgen.1007155"><rect x="437" y="5393" width="119" height="8"/></a>
<a xlink:href="http://doi.org/10.1093/gbe/evz210" xlink:title="http://doi.org/10.1093/gbe/evz210"><rect x="421" y="5443" width="88" height="8"/></a>
<a xlink:href="http://doi.org/10.1038/ncomms14665" xlink:title="http://doi.org/10.1038/ncomms14665"><rect x="330" y="3383" width="98" height="8"/></a>
<a xlink:href="http://doi.org/10.1371/journal.pbio.2003706" xlink:title="http://doi.org/10.1371/journal.pbio.2003706"><rect x="319" y="3425" width="117" height="8"/></a>
<a xlink:href="http://doi.org/10.1038/nature25487" xlink:title="http://doi.org/10.1038/nature25487"><rect x="318" y="3467" width="90" height="8"/></a>
<a xlink:href="http://dx.doi.org/10.1016/j.ccell.2014.09.010" xlink:title="http://dx.doi.org/10.1016/j.ccell.2014.09.010"><rect x="323" y="3586" width="120" height="8"/></a>
<a xlink:href="http://doi.org/10.1016/j.cell.2019.12.012" xlink:title="http://doi.org/10.1016/j.cell.2019.12.012"><rect x="325" y="3652" width="107" height="8"/></a>
<a xlink:href="http://doi.org/10.1016/j.cell.2019.05.024" xlink:title="http://doi.org/10.1016/j.cell.2019.05.024"><rect x="342" y="3703" width="107" height="8"/></a>
<a xlink:href="http://doi.org/10.1038/s41467-018-06236-8" xlink:title="http://doi.org/10.1038/s41467-018-06236-8"><rect x="322" y="3754" width="119" height="8"/></a>
<a xlink:href="http://doi.org/10.1038/nature23262" xlink:title="http://doi.org/10.1038/nature23262"><rect x="321" y="3805" width="100" height="8"/></a>
<a xlink:href="http://doi.org/10.1016/j.celrep.2016.08.066" xlink:title="http://doi.org/10.1016/j.celrep.2016.08.066"><rect x="333" y="3847" width="116" height="8"/></a>
<a xlink:href="http://doi.org/10.1038/nature21683" xlink:title="http://doi.org/10.1038/nature21683"><rect x="333" y="3897" width="90" height="8"/></a>
<a xlink:href="http://doi.org/10.1126/science.aam5339" xlink:title="http://doi.org/10.1126/science.aam5339"><rect x="323" y="3948" width="106" height="8"/></a>
<a xlink:href="http://doi.org/10.1002/ijc.31976" xlink:title="http://doi.org/10.1002/ijc.31976"><rect x="325" y="4007" width="80" height="8"/></a>
<a xlink:href="http://doi.org/10.3389/fnins.2019.00173" xlink:title="http://doi.org/10.3389/fnins.2019.00173"><rect x="330" y="4110" width="106" height="8"/></a>
<a xlink:href="http://doi.org/10.1098/rsif.2020.0154" xlink:title="http://doi.org/10.1098/rsif.2020.0154"><rect x="357" y="4143" width="96" height="8"/></a>
<a xlink:href="http://doi.org/10.1002/bies.201900254" xlink:title="http://doi.org/10.1002/bies.201900254"><rect x="359" y="4194" width="102" height="8"/></a>
<a xlink:href="http://doi.org/10.1073/pnas.1808221115" xlink:title="http://doi.org/10.1073/pnas.1808221115"><rect x="361" y="4245" width="109" height="8"/></a>
<a xlink:href="http://doi.org/10.1038/s41477-018-0318-3" xlink:title="http://doi.org/10.1038/s41477-018-0318-3"><rect x="352" y="4313" width="115" height="8"/></a>
<a xlink:href="http://doi.org/10.1038/nature23482" xlink:title="http://doi.org/10.1038/nature23482"><rect x="361" y="4355" width="90" height="8"/></a>
<a xlink:href="http://doi.org/10.1017/s2040174418000880" xlink:title="http://doi.org/10.1017/s2040174418000880"><rect x="359" y="4405" width="120" height="8"/></a>
<a xlink:href="http://doi.org/10.1158/2159-8290.CD-16-1292" xlink:title="http://doi.org/10.1158/2159-8290.CD-16-1292"><rect x="317" y="4479" width="128" height="8"/></a>
<a xlink:href="http://doi.org/10.1038/s41593-017-0029-5" xlink:title="http://doi.org/10.1038/s41593-017-0029-5"><rect x="309" y="4560" width="115" height="8"/></a>
<a xlink:href="http://doi.org/10.15252/embr.201846429" xlink:title="http://doi.org/10.15252/embr.201846429"><rect x="306" y="4602" width="109" height="8"/></a>
<a xlink:href="http://doi.org/10.1126/sciadv.1501340" xlink:title="http://doi.org/10.1126/sciadv.1501340"><rect x="309" y="4659" width="101" height="8"/></a>
<a xlink:href="http://doi.org/10.1016/j.celrep.2020.107573" xlink:title="http://doi.org/10.1016/j.celrep.2020.107573"><rect x="307" y="4716" width="118" height="8"/></a>
<a xlink:href="http://doi.org/10.1016/j.cell.2016.09.017" xlink:title="http://doi.org/10.1016/j.cell.2016.09.017"><rect x="320" y="4758" width="107" height="8"/></a>
<a xlink:href="http://doi.org/10.1126/scitranslmed.aat8462" xlink:title="http://doi.org/10.1126/scitranslmed.aat8462"><rect x="328" y="4808" width="117" height="8"/></a>
<a xlink:href="http://doi.org/10.1371/journal.pgen.1006391" xlink:title="http://doi.org/10.1371/journal.pgen.1006391"><rect x="316" y="4859" width="119" height="8"/></a>
<a xlink:href="http://doi.org/10.1371/journal.pcbi.1005385" xlink:title="http://doi.org/10.1371/journal.pcbi.1005385"><rect x="316" y="4927" width="116" height="8"/></a>
<a xlink:href="http://doi.org/10.1038/mp.2010.136" xlink:title="http://doi.org/10.1038/mp.2010.136"><rect x="317" y="4977" width="94" height="8"/></a>
<a xlink:href="http://doi.org/10.1038/ncomms13171" xlink:title="http://doi.org/10.1038/ncomms13171"><rect x="333" y="5019" width="98" height="8"/></a>
<a xlink:href="http://doi.org/10.1016/j.celrep.2017.11.031" xlink:title="http://doi.org/10.1016/j.celrep.2017.11.031"><rect x="323" y="5070" width="116" height="8"/></a>
<a xlink:href="http://doi.org/10.1371/journal.pbio.2003992" xlink:title="http://doi.org/10.1371/journal.pbio.2003992"><rect x="316" y="5112" width="117" height="8"/></a>
<a xlink:href="http://doi.org/10.1126/science.aan6877" xlink:title="http://doi.org/10.1126/science.aan6877"><rect x="329" y="5219" width="104" height="8"/></a>
<a xlink:href="http://doi.org/10.1016/j.devcel.2019.06.012" xlink:title="http://doi.org/10.1016/j.devcel.2019.06.012"><rect x="383" y="5335" width="117" height="8"/></a>
<a xlink:href="http://doi.org/10.1016/j.alcohol.2016.10.010" xlink:title="http://doi.org/10.1016/j.alcohol.2016.10.010"><rect x="191" y="3058" width="119" height="8"/></a>
<a xlink:href="http://doi.org/10.1017/s0954579417001213" xlink:title="http://doi.org/10.1017/s0954579417001213"><rect x="198" y="3099" width="120" height="8"/></a>
<a xlink:href="http://doi.org/10.1111/jeb.13432" xlink:title="http://doi.org/10.1111/jeb.13432"><rect x="346" y="2466" width="83" height="8"/></a>
<a xlink:href="https://www.youtube.com/watch?v=hCqdOCO7kMg" xlink:title="https://www.youtube.com/watch?v=hCqdOCO7kMg"><rect x="584" y="2237" width="24" height="8"/></a>
</svg>
</div>
</div>

<br/>
<a href="javascript:showhide('outlineDiv')">Outline show/hide</a>
<div id="outlineDiv" style="display:none;">
<h1>00092</h1>
<p>eve</p>
<h2>05092</h2>
<ul>
<li>diversity, or why evolution happens
<ul>
<li>
<p><em>origin and evolution of transporter substrate specificity within the npf family</em><br />
morten egevang jørgensen et al. 2017<br />
<a href="http://doi.org/10.7554/eLife.19466">doi.org/10.7554/eLife.19466</a></p>
<ul>
<li>
<p>Despite vast diversity in metabolites and the matching substrate specificity of their transporters, little is known about how evolution of transporter substrate specificities is linked to emergence of substrates via evolution of biosynthetic pathways. Transporter specificity towards the recently evolved glucosinolates characteristic of <em>Brassicales</em> is shown to evolve prior to emergence of glucosinolate biosynthesis. Furthermore, we show that glucosinolate transporters belonging to the ubiquitous NRT1/PTR FAMILY (NPF) likely evolved from transporters of the ancestral cyanogenic glucosides found across more than 2500 species outside of the <em>Brassicales</em>. Biochemical characterization of orthologs along the phylogenetic lineage from cassava to <em>A. thaliana,</em> suggests that alterations in the electrogenicity of the transporters accompanied changes in substrate specificity. Linking the evolutionary path of transporter substrate specificities to that of the biosynthetic pathways, exemplify how transporter substrate specificities originate and evolve as new biosynthesis pathways emerge.</p>
<p>All living cells are surrounded by membranes that protect them from the external environment. The membrane contains proteins called transporters, which move nutrients and other molecules (known as substrates) across the membrane. A variety of transporters have evolved to move the hundreds of thousands of different substrates found in nature.<br />
Plant cells make many different compounds to protect themselves from pests and diseases. A group of transporters known as the NPF family move some of these compounds across the cells outer membrane. The types of substrates they transport vary in different plants. In cassava, for example, NPF transporters move compounds called cyanogenic glucosides, which are poisonous to humans and other animals. On the other hand, NPF transporters in another plant called <em>Arabidopsis thaliana</em> can move bitter-tasting compounds called glucosinolates. The process that makes glucosinolates in plants evolved from the process that makes cyanogenic glucosides.<br />
Can transporters evolve the ability to move a new substrate before or after that substrate first appears? To answer this question, Jørgensen et al. studied the NPF family in <em>A. thaliana</em>, cassava and another plant called papaya that makes both cyanogenic glucosides and glucosinolates. The experiments suggest that NPF transporters able to move both cyanogenic glucosides and glucosinolates evolved before plants evolved the ability to make glucosinolates. Later in evolution, these multi-specific transporters specialized to only move glucosinolates. Jørgensen et al. also show that early glucosinolate transporters could move a broad variety of glucosinolates but later evolved to only transport particular types.<br />
These findings show how transporters and the processes that make compounds in cells may evolve together. A future challenge will be to understand the molecular changes in a transporter that make it specific for a certain substrate. This may help researchers to develop new ways of controlling the amount of toxic compounds in crops we eat by manipulating how the compounds are transported.</p>
</li>
</ul>
</li>
<li>
<p><em>environmental change explains cichlid adaptive radiation at lake malawi over the past 1.2 million years</em><br />
sarah j. ivory et al. 2016<br />
<a href="http://doi.org/10.1073/pnas.1611028113">doi.org/10.1073/pnas.1611028113</a></p>
<ul>
<li>
<p>Tropical African lakes are well-known to house exceptionally biodiverse assemblages of fish and other aquatic fauna, which are thought to be at risk in the future. Although the modern assemblages are well-studied, direct evidence of the origin of this incredible wealth of species and the mechanisms that drive speciation are virtually unknown. We use a long sedimentary record from Lake Malawi to show that over the last 1.2 My both large-scale climatic and tectonic changes resulted in wet–dry transitions that led to extraordinary habitat variability and rapid diversification events. This work allows us to understand the environmental context of aquatic evolution in the most biodiverse tropical lake.</p>
<p>Long paleoecological records are critical for understanding evolutionary responses to environmental forcing and unparalleled tools for elucidating the mechanisms that lead to the development of regions of high biodiversity. We use a 1.2-My record from Lake Malawi, a textbook example of biological diversification, to document how climate and tectonics have driven ecosystem and evolutionary dynamics. Before ∼800 ka, Lake Malawi was much shallower than today, with higher frequency but much lower amplitude water-level and oxygenation changes. Since ∼800 ka, the lake has experienced much larger environmental fluctuations, best explained by a punctuated, tectonically driven rise in its outlet location and level. Following the reorganization of the basin, a change in the pacing of hydroclimate variability associated with the Mid-Pleistocene Transition resulted in hydrologic change dominated by precession rather than the high-latitude teleconnections recorded elsewhere. During this time, extended, deep lake phases have abruptly alternated with times of extreme aridity and ecosystem variability. Repeated crossings of hydroclimatic thresholds within the lake system were critical for establishing the rhythm of diversification, hybridization, and extinction that dominate the modern system. The chronology of these changes closely matches both the timing and pattern of phylogenetic history inferred independently for the lake’s extraordinary array of cichlid fish species, suggesting a direct link between environmental and evolutionary dynamics.</p>
</li>
</ul>
</li>
<li>
<p><em>signal or noise? a null model method for evaluating the significance of turnover pulses</em><br />
w. andrew barr 2017<br />
<a href="http://doi.org/10.1017/pab.2017.21">doi.org/10.1017/pab.2017.21</a></p>
<ul>
<li>Patterns of turnover in the mammalian fossil record have been interpreted as reflecting “pulses” of originations and extinctions hypothesized to be driven by climate change. However, criteria for determining what constitutes a meaningful pulse have been idiosyncratic, and investigations of turnover patterns in mammals have yielded mixed results.<br />
This study presents simple simulations of fossil records in which origination and extinction probabilities for each lineage are held constant. Nonetheless, the total number of turnover events per time bin varies stochastically, producing statistical “noise.” Various simulation and analytical assumptions are examined to determine their impact on the type I error rate (i.e., how often “pulses” are detected in a purely stochastic process).<br />
Results suggest that simple analytical parameters (length of time bins and turnover-pulse criterion) have predictable and straightforward effects on false-positive rates. Furthermore, “pulses” of turnover of a magnitude similar to that observed in the terrestrial mammalian fossil record may be quite common under realistic analytical conditions.<br />
The null turnover model offers a practical way to evaluate the significance of observed turnover events in future empirical studies of the fossil record. In evaluating the significance of a “pulse” of fossil origination or extinction events, analytical parameters can be explored using this null model to determine the approximate type I error rate for a set of parameters. Because false-positive rates are shown to be quite high, functional trait-based approaches may offer more reliable indicators of the impact of climate change on turnover dynamics.</li>
</ul>
</li>
<li>
<p><em>high-resolution lineage tracking reveals travelling wave of adaptation in laboratory yeast</em><br />
alex n. nguyen ba et al. 2019<br />
<a href="http://doi.org/10.1038/s41586-019-1749-3">doi.org/10.1038/s41586-019-1749-3</a></p>
<ul>
<li>
<p>“We have been taught that evolution ‘is slow’ and involves the ‘survival of the fittest,’ added Alex N. Nguyen Ba, a post-doctoral fellow in Desai’s lab. “It turns out that molecular evolution doesn’t work that way. It’s actually much faster than how we’ve been taught. This makes evolution way more complex than what has been anticipated.” Nguyen Ba is one of three co-lead authors of the new study, along with Ivana Cvijović and José I. Rojas Echenique<br />
Such evolution has been posited mathematically over the past two decades. However, previous lab experiments have not been able to prove or disprove the theory. Rather, they have only been able to examine the process with high resolution over a short period of time, or with low resolution over a long period of time. Collectively, Desai explained, the paper’s authors — who include Katherine R. Lawrence of MIT and Harvard’s Artur Rego-Costa, along with Xianan Liu of Stanford and Sasha F. Levy of SLAC National Accelerator Laboratory — have done both other kinds of studies.<br />
This new study does both.<br />
“We can identify every single relevant beneficial mutation,” said Nguyen Ba, citing new technology that allowed the research team to follow specific genomes (or lineages) for approximately a thousand generations.<br />
Cvijović, formerly a graduate student in Desai’s lab and now a researcher at Princeton, said the research could have gone on indefinitely: “A thousand generations is about three months of growth in our conditions. That’s enough time to see big changes happening.”<br />
Such in-depth, long-term research was possible because of a technological advance in the methodology that allowed what Nguyen Ba called the “re-barcoding” of DNA.<br />
Using an enzyme to place a marker, the “barcode,” at a specific DNA site, the researchers were able to follow the DNA of yeast through multiple generations. By re-tagging and re-barcoding subsequent generations to record their lineage, the team could then observe how this DNA was transmitted, noting what survived, and what thrived — or came to dominate — as generations passed.<br />
What they discovered included a few surprises.<br />
According to the existing theory, the “fittest” DNA would be that which showed up most frequently in subsequent generations. However, the scientists observed “fluctuations” that the theories could not account for.<br />
“Mutations and genotypes that seem to have fallen behind can leapfrog and dominate,” said Cvijović.<br />
What that means, she says, will be the subject of future research. However, it implies that evolution is, indeed, even more complex than previously thought.<br />
“Our experiment suggests there may be a wide range of a large number of strongly beneficial mutations,” she said. “And their benefits are both very strong and very different from one another.”</p>
</li>
<li>
<p><em>abstract</em> In rapidly adapting asexual populations, including many microbial pathogens and viruses, numerous mutant lineages often compete for dominance within the population. These complex evolutionary dynamics determine the outcomes of adaptation, but have been difficult to observe directly. Previous studies have used whole-genome sequencing to follow molecular adaptation; however, these methods have limited resolution in microbial populations. Here we introduce a renewable barcoding system to observe evolutionary dynamics at high resolution in laboratory budding yeast. We find nested patterns of interference and hitchhiking even at low frequencies. These events are driven by the continuous appearance of new mutations that modify the fates of existing lineages before they reach substantial frequencies. We observe how the distribution of fitness within the population changes over time, and find a travelling wave of adaptation that has been predicted by theory. We show that clonal competition creates a dynamical ‘rich-get-richer’ effect: fitness advantages that are acquired early in evolution drive clonal expansions, which increase the chances of acquiring future mutations. However, less-fit lineages also routinely leapfrog over strains of higher fitness. Our results demonstrate that this combination of factors, which is not accounted for in existing models of evolutionary dynamics, is critical in determining the rate, predictability and molecular basis of adaptation.</p>
</li>
</ul>
</li>
</ul>
</li>
</ul>
<h2>05192</h2>
<ul>
<li>
<p>nautural preservation, not selection</p>
</li>
<li>
<p>darwin online<br />
<a href="http://darwin-online.org.uk/">darwin-online.org.uk</a></p>
<ul>
<li>
<p>make chapter–by–chapter comparison of all editions of <em>on the origin of species</em> using ithoughts</p>
</li>
<li>
<p><em>the descent of man selection in relation to sex</em><br />
charles darwin</p>
</li>
<li>
<p>I found the scan of the 1907 issue was the easiest to use of the more recent ones available here<br />
Link: <a href="http://darwin-online.org.uk/content/frameset?pageseq=1&amp;itemID=F401&amp;viewtype=text">darwin-online.org.uk/content/frameset</a></p>
</li>
</ul>
</li>
<li>
<p>wallace online<br />
<a href="http://wallace-online.org/">wallace-online.org</a></p>
</li>
<li>
<p><em>the origin of species by means of natural selection, or the preservation of favoured races in the struggle for life</em><br />
charles darwin 1876 6th edition<br />
[first issue of final text]</p>
<ul>
<li>
<p><a href="http://www.gutenberg.org/files/2009/2009-h/2009-h.htm">gutenberg.org</a> text html</p>
</li>
<li>
<p>1912 hurst usa printing<br />
<a href="http://darwin-online.org.uk/contents.html">darwin-online.org.uk/contents.html</a></p>
</li>
<li>
<p><a href="https://archive.org/search.php?query=%28origin%20species%29%20AND%20creator%3A%28darwin%29">archive.org</a></p>
</li>
</ul>
</li>
<li>
<p><em>evolution beyond neo-darwinism: a new conceptual framework</em><br />
denis noble 2015<br />
<a href="http://doi.org/10.1242/jeb.106310">doi.org/10.1242/jeb.106310</a></p>
<ul>
<li><em>abstract</em> Experimental results in epigenetics and related fields of biological research show that the Modern Synthesis (neo-Darwinist) theory of evolution requires either extension or replacement. This article examines the conceptual framework of neo-Darwinism, including the concepts of ‘gene’, ‘selfish’, ‘code’, ‘program’, ‘blueprint’, ‘book of life’, ‘replicator’ and ‘vehicle’. This form of representation is a barrier to extending or replacing existing theory as it confuses conceptual and empirical matters. These need to be clearly distinguished. In the case of the central concept of ‘gene’, the definition has moved all the way from describing a necessary cause (defined in terms of the inheritable phenotype itself) to an empirically testable hypothesis (in terms of causation by DNA sequences). Neo-Darwinism also privileges ‘genes’ in causation, whereas in multi-way networks of interactions there can be no privileged cause. An alternative conceptual framework is proposed that avoids these problems, and which is more favourable to an integrated systems view of evolution.</li>
</ul>
</li>
<li>
<p><em>evolution viewed from physics, physiology and medicine</em><br />
denis noble 2017<br />
<a href="http://doi.org/10.1098/rsfs.2016.0159">doi.org/10.1098/rsfs.2016.0159</a></p>
<ul>
<li><em>abstract</em> Stochasticity is harnessed by organisms to generate functionality. Randomness does not, therefore, necessarily imply lack of function or ‘blind chance’ at higher levels. In this respect, biology must resemble physics in generating order from disorder. This fact is contrary to Schrödinger’s idea of biology generating phenotypic order from _molecular-_level order, which inspired the central dogma of molecular biology. The order originates at higher levels, which constrain the components at lower levels. We now know that this includes the genome, which is controlled by patterns of transcription factors and various epigenetic and reorganization mechanisms. These processes can occur in response to environmental stress, so that the genome becomes ‘a highly sensitive organ of the cell’ (McClintock). Organisms have evolved to be able to cope with many variations at the molecular level. Organisms also make use of physical processes in evolution and development when it is possible to arrive at functional development without the necessity to store all information in DNA sequences. This view of development and evolution differs radically from that of neo-Darwinism with its emphasis on blind chance as the origin of variation. Blind chance is necessary, but the origin of functional variation is not at the molecular level. These observations derive from and reinforce the principle of biological relativity, which holds that there is no privileged level of causation. They also have important implications for medical science.</li>
</ul>
</li>
<li>
<p><em>dance to the tune of life. biological relativity</em><br />
denis noble 2016 not yet read</p>
</li>
<li>
<p><em>the music of life, biology beyond the genome</em><br />
debis noble 2006 not yet read</p>
</li>
<li>
<p><em>what’s wrong with the modern evolutionary synthesis? a critical reply to welch (2017)</em><br />
koen b. tanghe et al. 2018<br />
<a href="http://doi.org/10.1007/s10539-018-9633-3">doi.org/10.1007/s10539-018-9633-3</a></p>
<p>don’t agree with this paper, but the mayr quote is interesting from a diversity viewpoint:</p>
<ul>
<li>“Mayr, arguably one of the main architects of the MS, in particular, always remained a staunch critic of the reduction of evolution to changes in allele frequencies. In the prologue to The Evolutionary Synthesis (1980, p. 12), he writes (contra Dobzhansky): “It is simply not true that evolution can be explained as a change in gene frequencies. (…) Changes in gene frequency are a by-product of adaptation and of the origin of evolutionary diversity (induced by natural selection) and not the other way around.” His Systematics and the Origin of Species (1942) was inspired by the one-sidedness of the evolutionary literature at the time. The emphasis was al-most exclusively on the change of gene frequencies in one single gene pool, as if speciation did not exist (Mayr 1992). Against this view, he argued that divergence was the key concept in evolution (Provine 1986, pp. 478-479).”</li>
</ul>
</li>
<li>
<p><em>evolution of shape and leverage of bird beaks reflects feeding ecology, but not as strongly as expected</em><br />
sam van wassenbergh, simon baeckens 2019<br />
<a href="http://doi.org/10.1111/evo.13686">doi.org/10.1111/evo.13686</a></p>
<ul>
<li>
<p>The observation that Galapagos finch species possessed different beak shapes to obtain different foods was central to the theory of evolution by natural selection, and it has been assumed that this form-function relationship holds true across all species of bird.</p>
<p>However, a new study published in the journal Evolution suggests the beaks of birds are not as adapted to the food types they feed on as it is generally believed.</p>
<p>An international team of scientists from the United Kingdom, Spain and the US used computational and mathematical techniques to better understand the connection between beak shapes and functions in living birds.</p>
<p>By measuring beak shape in a wide range of modern bird species from museum collections and looking at information about how the beak is used by different species to eat different foods, the team were able to assess the link between beak shape and feeding behaviour.</p>
<p>Professor Emily Rayfield, from the University of Bristol's School of Earth Sciences, and senior author of the study, said: &quot;This is, to our knowledge, the first approach to test a long-standing principle in biology: that the beak shape and function of birds is tightly linked to their feeding ecologies.&quot;</p>
<p>Guillermo Navalón, lead author of the study and a final year PhD student at Bristol's School of Earth Sciences, added: &quot;The connection between beak shapes and feeding ecology in birds was much weaker and more complex than we expected and that while there is definitely a relationship there, many species with similarly shaped beaks forage in entirely different ways and on entirely different kinds of food.</p>
<p>&quot;This is something that has been shown in other animal groups, but in birds this relationship was always assumed to be stronger.&quot;</p>
<p>Co-author, Dr Jesús Marugán-Lobón from Universidad Autónoma de Madrid, said: &quot;These results only made sense when you realise birds use the beak for literally everything!</p>
<p>&quot;Therefore, also makes sense they evolved a versatile tool not just for getting food, but also to accomplish many other tasks.&quot;</p>
<p>The study is part of a larger research effort by the team in collaboration with researchers from other universities across Europe and the US to better understand the main drivers of the evolution of the skull in birds.</p>
<p>Dr Jen Bright, co-author from the University of South Florida, said: &quot;We have seen similar results before in birds of prey, but this is the first time we studied the link between beak shape and ecology across all bird groups.</p>
<p>&quot;We looked at a huge range of beak shapes and feeding ecologies: hummingbirds, eagles, parrots, puffins, flamingos, pretty much every beak you can think of.&quot;</p>
<p>Guillermo Navalón added: &quot;These results have important implications for the study of fossil birds.</p>
<p>&quot;We have to be careful about inferring ecology in ancient birds, which we often assume based solely on the shape of the beak.</p>
<p>&quot;Really, we're just starting to scratch the surface, and a lot more research is needed to fully understand the drivers behind beak shape evolution.&quot;</p>
</li>
<li>
<p><em>abstract</em> Is feeding ecology the main driver of beak diversification in modern birds? Taking a broad‐scale interspecific comparative approach, Navalón et al. (2019) found a relationship between feeding ecology (diet and feeding behavior) and beak morphology (shape and leverage), although much of the observed variation remained unexplained. This low explanatory power may suggest that variation in the multitude of non‐feeding functions of the beak also influences its evolution.</p>
</li>
</ul>
</li>
<li>
<p><em>jurassic shift from abiotic to biotic control on marine ecological success</em><br />
kilian eichenseer et al. 2019<br />
<a href="http://doi.org/10.1038/s41561-019-0392-9">doi.org/10.1038/s41561-019-0392-9</a></p>
<ul>
<li>
<p>proliferation of calcium carbonate-secreting plankton and their subsequent deposition on the ocean floor.</p>
<p>They believe the rise of this plankton stabilised the chemical composition of the ocean and provided the conditions for one of the most prominent diversifications of marine life in Earth's history.</p>
<p>The research was led by academics from the University of Plymouth's School of Geography, Earth and Environmental Sciences and School of Computing, Electronics and Mathematics, in cooperation with scientists from the University of Bergen in Norway, and the University of Erlangen-Nuremberg in Germany.</p>
<p>PhD candidate Kilian Eichenseer, the study's lead author, explained the impact of calcifying plankton: &quot;Today, huge areas of the ocean floor are covered with the equivalent of chalk, made up of microscopic organisms that rose to dominance in the middle of the Jurassic period. The chalky mass helps to balance out the acidity of the ocean and, with that balance in place, organisms are less at the mercy of short-term perturbations of ocean chemistry than they might have been previously. It is easier to secrete a shell, regardless of its mineralogy, if the ocean chemistry is stable.&quot;</p>
<p>The aim of the research was to test the hypothesis that the evolutionary importance of the non-biological environment had declined through geological time.</p>
<p>Since its emergence more than 540 million years ago, multicellular life evolved under the influence of both the non-biological and the biological environment, but how the balance between these factors changed remained largely unknown.</p>
<p>Calcified seashells provide an ideal test to answer this question, as aragonite and calcite -- the minerals making up seashells -- also form non-biologically in the ocean.</p>
<p>In their study, the authors used the vast global fossil record of marine organisms that secreted calcium carbonate, which encompasses more than 400,000 samples dating from 10,000 years BC up to around 500 million years ago.</p>
<p>Using reconstructions of the temperature and the ocean water composition of the past, the authors estimated the proportion of aragonite and calcite that formed inorganically in the ocean in 85 geological stages across 500 million years.</p>
<p>Through a series of specially developed statistical analyses, this inorganic pattern of aragonite-calcite seas was then compared with seashell mineral composition over the same time.</p>
<p>The results show that up until the middle of the Jurassic period, around 170 million years ago, the ecological success of shell-secreting marine organisms was tightly coupled to their shell composition: organisms that secreted the mineral that was environmentally favoured had an evolutionary advantage.</p>
<p>However, the Earth-Life system was revolutionised forever by the rise of calcifying plankton, which expanded the production of calcium carbonate from continental shelves to the open ocean.</p>
<p>This ensured that the evolutionary impact of episodes of severe climate changes, and resulting ocean acidification, was less severe than comparable events earlier in Earth history.</p>
<p>Dr Uwe Balthasar, Lecturer in Palaeontology, first published research exploring the dominance of aragonite and calcite in the marine environment in 2015. He said: &quot;During the Earth's history there have been several major events that shaped the evolution of life on our planet, such as the five big mass extinctions or the radiation of complex animals during the 'Cambrian Explosion'. Our research identifies a previously overlooked event of this magnitude around 170 million years ago when the emergence of calcium carbonate-secreting plankton lifted constraints on the evolution of other marine organisms that we did not know existed. As a result, life in the ocean has diversified to levels far beyond what existed before.&quot;</p>
</li>
<li>
<p><em>abstract</em> Environmental change and biotic interactions both govern the evolution of the biosphere, but the relative importance of these drivers over geological time remains largely unknown. Previous work suggests that, unlike environmental parameters, diversity dynamics differ profoundly between the Palaeozoic and post-Palaeozoic eras. Here we use the fossil record to test the hypothesis that the influence of ocean chemistry and climate on the ecological success of marine calcifiers decreased throughout the Phanerozoic eon. Marine calcifiers build skeletons of calcite or aragonite, and the precipitation of these calcium carbonate polymorphs is governed by the magnesium-to-calcium ratio and temperature in abiotic systems. We developed an environmental forcing model based on secular changes of ocean chemistry and temperature and assessed how well the model predicts the proliferation of skeletal taxa with respect to calcium carbonate polymorphs. Abiotic forcing governs the ecological success of aragonitic calcifiers from the Ordovician to the Middle Jurassic, but not thereafter. This regime shift coincides with the proliferation of calcareous plankton in the mid-Mesozoic. The deposition of biomineralizing plankton on the ocean floor buffers CO2 excursions and stabilizes Earth’s biochemical cycle, and thus mitigates the evolutionary impact of environmental change on the marine biota.</p>
</li>
</ul>
</li>
</ul>
<h2>05292</h2>
<ul>
<li>
<p><em>morphospace expansion paces taxonomic diversification after end cretaceous mass extinction</em><br />
christopher m. lowery, andrew j. fraass 2019<br />
<a href="http://doi.org/10.1038/s41559-019-0835-0">doi.org/10.1038/s41559-019-0835-0</a></p>
<ul>
<li>
<p>It takes at least 10 million years for life to fully recover after a mass extinction, a speed limit for the recovery of species diversity that is well known among scientists. Explanations for this apparent rule have usually invoked environmental factors, but research led by The University of Texas at Austin links the lag to something different: evolution.</p>
<p>The recovery speed limit has been observed across the fossil record, from the &quot;Great Dying&quot; that wiped out nearly all ocean life 252 million years ago to the massive asteroid strike that killed all nonavian dinosaurs. The study, published April 8 in the journal Nature Ecology &amp; Evolution, focused on the later example. It looks at how life recovered after Earth's most recent mass extinction, which snuffed out most dinosaurs 66 million years ago. The asteroid impact that triggered the extinction is the only event in Earth's history that brought about global change faster than present-day climate change, so the authors said the study could offer important insight on recovery from ongoing, human-caused extinction events.</p>
<p>The idea that evolution -- specifically, how long it takes surviving species to evolve traits that help them fill open ecological niches or create new ones -- could be behind the extinction recovery speed limit is a theory proposed 20 years ago. This study is the first to find evidence for it in the fossil record, the researchers said.</p>
<p>The team tracked recovery over time using fossils from a type of plankton called foraminifera, or forams. The researchers compared foram diversity with their physical complexity. They found that total complexity recovered before the number of species -- a finding that suggests that a certain level of ecological complexity is needed before diversification can take off.</p>
<p>In other words, mass extinctions wipe out a storehouse of evolutionary innovations from eons past. The speed limit is related to the time it takes to build up a new inventory of traits that can produce new species at a rate comparable to before the extinction event.</p>
<p>Lead author Christopher Lowery, a research associate at the University of Texas Institute for Geophysics (UTIG), said that the close association of foram complexity with the recovery speed limit points to evolution as the speed control.</p>
<p>&quot;We see this in our study, but the implication should be that these same processes would be active in all other extinctions,&quot; Lowery said. &quot;I think this is the likely explanation for the speed limit of recovery for everything.&quot;</p>
<p>Lowery co-authored the paper with Andrew Fraass, a research associate at the University of Bristol who did the research while at Sam Houston State University. UTIG is a research unit of the UT Jackson School of Geosciences.</p>
<p>The researchers were inspired to look into the link between recovery and evolution because of earlier research that found recovery took millions of years despite many areas being habitable soon after Earth's most recent mass extinction. This suggested a control factor other than the environment alone.</p>
<p>They found that although foram diversity as a whole was decimated by the asteroid, the species that survived bounced back quickly to refill available niches. However, after this initial recovery, further spikes in species diversity had to wait for the evolution of new traits. As the speed limit would predict, 10 million years after extinction, the overall diversity of forams was nearly back to levels observed before the extinction event. Foram fossils are prolific in ocean sediments around the world, allowing the researchers to closely track species diversity without any large gaps in time.</p>
<p>Pincelli Hull, an assistant professor at Yale University, said the paper sheds light on factors driving recovery.</p>
<p>&quot;Before this study, people could have told you about the basic patterns in diversity and complexity, but they wouldn't have been able to answer how they relate to one another in a quantitative sense,&quot; she said.</p>
<p>The authors said that recovery from past extinctions offers a road map for what might come after the modern ongoing extinction, which is driven by climate change, habitat loss, invasive species and other factors.</p>
</li>
<li>
<p><em>abstract</em> Highly resolved palaeontological records can address a key question about our current climate crisis: how long will it be before the biosphere rebounds from our actions? There are many ways to conceptualize the recovery of the biosphere; here, we focus on the global recovery of species diversity. Mass extinction may be expected to be followed by rapid speciation, but the fossil record contains many instances where speciation is delayed—a phenomenon about which we have a poor understanding. A probable explanation for this delay is that extinctions eliminate morphospace as they curtail diversity, and the delay in diversification is a result of the time needed for new innovations to rebuild morphospace, which can then be filled out by new species. Here, we test this morphospace reconstruction hypothesis using the morphological complexity of planktic foraminifer tests after the Cretaceous–Palaeogene mass extinction. We show that increases in complexity precede changes in diversity, indicating that plankton are colonizing new morphospace, then slowly filling it in. Preliminary diversification is associated with a rapid increase in the complexity of groups refilling relict Cretaceous ecospace. Subsequent jumps in complexity are driven by evolutionary innovations (development of spines and photosymbionts), which open new niche space. The recovery of diversity is paced by the construction of new morphospace, implying a fundamental speed limit on diversification after an extinction event.</p>
</li>
</ul>
</li>
</ul>
<h2>05392</h2>
<ul>
<li>
<p><em>transcendence: how humans evolved through fire, language, beauty, and time</em><br />
gaia vince 2020</p>
</li>
<li>
<p><em>sapiens: a brief history of humankind</em><br />
yuval noah harari 2014 97807710-3852-5</p>
<ul>
<li>
<p><em>homo deus</em><br />
yuval noah harari 2017<br />
not as good as <em>sapiens</em></p>
</li>
<li>
<p><em>21 lessons for the 21st century</em><br />
yuval noah harari 2018</p>
</li>
</ul>
</li>
<li>
<p><em>survival of the mediocre mediocre</em><br />
venkatesh rao 2018<br />
<a href="https://www.ribbonfarm.com/2018/04/24/survival-of-the-mediocre-mediocre/">www.ribbonfarm.com/2018/04/24/survival-of-the-mediocre-mediocre/</a></p>
<ul>
<li>the points on evolution seems close to our idea of diversity, as “optionality” — he believes “mediocrity” provides a source of optionality</li>
<li>“the reason biological evolution from dinosaurs to humans seems to be driven by survival of the mediocre is that it is always up against an asymmetrically more powerful adversary, the unknowns of nature itself. The guerrilla way is the only way. Mediocrity is the only source of advantage.”</li>
<li>“To pursue mediocrity is to procrastinate on optimizing for the current fitness function because it might change at any time.”</li>
</ul>
</li>
</ul>
<h2>05492</h2>
<ul>
<li>
<p><em>frequency dependence and ecological drift shape coexistence of species with similar niches</em><br />
erik i. svensson et al. 2018<br />
<a href="http://doi.org/10.1086/697201">doi.org/10.1086/697201</a><br />
a tough read</p>
<ul>
<li>The coexistence of ecologically similar species might be counteracted by ecological drift and demographic stochasticity, both of which erode local diversity. With niche differentiation, species can be maintained through performance trade-offs between environments, but trade-offs are difficult to invoke for species with similar ecological niches. Such similar species might then go locally extinct due to stochastic ecological drift, but there is little empirical evidence for such processes. Previous studies have relied on biogeographical surveys and inferred process from pattern, while experimental field investigations of ecological drift are rare. Mechanisms preserving local species diversity, such as frequency dependence (e.g., rare-species advantages), can oppose local ecological drift, but the combined effects of ecological drift and such counteracting forces have seldom been investigated. Here, we investigate mechanisms between coexistence of ecologically similar but strongly sexually differentiated damselfly species (<em>Calopteryx virgo</em> and <em>Calopteryx splendens</em>). Combining field surveys, behavioral observations, experimental manipulations of species frequencies and densities, and simulation modeling, we demonstrate that species coexistence is shaped by the opposing forces of ecological drift and negative frequency dependence (rare-species advantage), generated by interference competition. Stochastic and deterministic processes therefore jointly shape coexistence. The role of negative frequency dependence in delaying the loss of ecologically similar species, such as those formed by sexual selection, should therefore be considered in community assembly, macroecology, macroevolution, and biogeography.</li>
</ul>
</li>
</ul>
<h2>05592</h2>
<ul>
<li>
<p><em>trade-off between transcriptome plasticity and genome evolution in cephalopods</em><br />
liscovitch-brauer et al. 2017<br />
<a href="http://doi.org/10.1016/j.cell.2017.03.025">doi.org/10.1016/j.cell.2017.03.025</a></p>
<ul>
<li>
<p>•Unlike other taxa, cephalopods diversify their proteomes extensively by RNA editing<br />
•Extensive recoding is specific to the behaviorally complex coleiods<br />
•Unlike mammals, cephalopod recoding is evolutionarily conserved and often adaptive<br />
•Transcriptome diversification comes at the expense of slowed-down genome evolution</p>
<p>RNA editing, a post-transcriptional process, allows the diversification of proteomes beyond the genomic blueprint; however it is infrequently used among animals for this purpose. Recent reports suggesting increased levels of RNA editing in squids thus raise the question of the nature and effects of these events. We here show that RNA editing is particularly common in behaviorally sophisticated coleoid cephalopods, with tens of thousands of evolutionarily conserved sites. Editing is enriched in the nervous system, affecting molecules pertinent for excitability and neuronal morphology. The genomic sequence flanking editing sites is highly conserved, suggesting that the process confers a selective advantage. Due to the large number of sites, the surrounding conservation greatly reduces the number of mutations and genomic polymorphisms in protein-coding regions. This trade-off between genome evolution and transcriptome plasticity highlights the importance of RNA recoding as a strategy for diversifying proteins, particularly those associated with neural function.</p>
</li>
</ul>
</li>
</ul>
<h2>05692</h2>
<ul>
<li>
<p><em>indolepropionic acid and novel lipid metabolites are associated with a lower risk of type 2 diabetes in the finnish diabetes prevention study</em><br />
vanessa d. de mello et al. 2017<br />
<a href="http://doi.org/10.1038/srep46337">doi.org/10.1038/srep46337</a></p>
<ul>
<li>Wide-scale profiling technologies including metabolomics broaden the possibility of novel discoveries related to the pathogenesis of type 2 diabetes (T2D). By applying non-targeted metabolomics approach, we investigated here whether serum metabolite profile predicts T2D in a well-characterized study population with impaired glucose tolerance by examining two groups of individuals who took part in the Finnish Diabetes Prevention Study (DPS); those who either early developed T2D (n = 96) or did not convert to T2D within the 15-year follow-up (n = 104). Several novel metabolites were associated with lower likelihood of developing T2D, including indole and lipid related metabolites. Higher indolepropionic acid was associated with reduced likelihood of T2D in the DPS. Interestingly, in those who remained free of T2D, indolepropionic acid and various lipid species were associated with better insulin secretion and sensitivity, respectively. Furthermore, these metabolites were negatively correlated with low-grade inflammation. We replicated the association between indolepropionic acid and T2D risk in one Finnish and one Swedish population. We suggest that indolepropionic acid, a gut microbiota-produced metabolite, is a potential biomarker for the development of T2D that may mediate its protective effect by preservation of β-cell function. Novel lipid metabolites associated with T2D may exert their effects partly through enhancing insulin sensitivity.</li>
</ul>
</li>
</ul>
<h2>05792</h2>
<ul>
<li>gene expression
<ul>
<li>
<p>x inactivation</p>
</li>
<li>
<p><em>diverse non-genetic, allele-specific expression effects shape genetic architecture at the cellular level in the mammalian brain</em><br />
wei-chao huang et al. 2017<br />
<a href="http://doi.org/10.1016/j.neuron.2017.01.033">doi.org/10.1016/j.neuron.2017.01.033</a></p>
<ul>
<li>
<p>•In vivo genome-wide screen uncovers diverse non-genetic allelic effects<br />
•Non-genetic allelic effects are prevalent in the neonatal mouse brain<br />
•Allelic effects cause mosaics of mutant and wild-type cells for heterozygous mutations<br />
•Allelic effects exist in the primate brain and impact genes linked to mental illness</p>
<p>Interactions between genetic and epigenetic effects shape brain function, behavior, and the risk for mental illness. Random X inactivation and genomic imprinting are epigenetic allelic effects that are well known to influence genetic architecture and disease risk. Less is known about the nature, prevalence, and conservation of other potential epigenetic allelic effects in vivo in the mouse and primate brain. Here we devise genomics, in situ hybridization, and mouse genetics strategies to uncover diverse allelic effects in the brain that are not caused by imprinting or genetic variation. We found allelic effects that are developmental stage and cell type specific, that are prevalent in the neonatal brain, and that cause mosaics of monoallelic brain cells that differentially express wild-type and mutant alleles for heterozygous mutations. Finally, we show that diverse non-genetic allelic effects that impact mental illness risk genes exist in the macaque and human brain. Our findings have potential implications for mammalian brain genetics.</p>
</li>
</ul>
</li>
<li>
<p>“silencing one gene copy may be a way in which cells fine tune their genetic program at specific times during the lifecycle of the animal, or in discrete places”<br />
<a href="https://www.sciencedaily.com/releases/2017/02/170223124227.htm">sciencedaily.com/releases/2017/02/170223124227.htm</a></p>
</li>
<li>
<p><em>impacts of neanderthal-introgressed sequences on the landscape of human gene expression</em><br />
rajiv c. mccoy, jon wakefield, joshua m. akey 2017<br />
<a href="http://doi.org/10.1016/j.cell.2017.01.038">doi.org/10.1016/j.cell.2017.01.038</a></p>
<ul>
<li>Regulatory variation influencing gene expression is a key contributor to phenotypic diversity, both within and between species. Unfortunately, RNA degrades too rapidly to be recovered from fossil remains, limiting functional genomic insights about our extinct hominin relatives. Many Neanderthal sequences survive in modern humans due to ancient hybridization, providing an opportunity to assess their contributions to transcriptional variation and to test hypotheses about regulatory evolution. We developed a flexible Bayesian statistical approach to quantify allele-specific expression (ASE) in complex RNA-seq datasets. We identified widespread expression differences between Neanderthal and modern human alleles, indicating pervasive cis-regulatory impacts of introgression. Brain regions and testes exhibited significant downregulation of Neanderthal alleles relative to other tissues, consistent with natural selection influencing the tissue-specific regulatory landscape. Our study demonstrates that Neanderthal-inherited sequences are not silent remnants of ancient interbreeding but have measurable impacts on gene expression that contribute to variation in modern human phenotypes.</li>
<li>identified people who had one copy of gene from modern humans and other copy from neanderthals</li>
</ul>
</li>
<li>
<p><em>dynamic genetic regulation of gene expression during cellular differentiation</em><br />
b. j. strober et al. 2019<br />
<a href="http://doi.org/10.1126/science.aaw0040">doi.org/10.1126/science.aaw0040</a></p>
<ul>
<li>
<p>analyzed RNA sequence data from 16 time points in human stem cells as they developed into cardiomyocytes, or heart muscle cells. In the process, they identified hundreds of expression quantitative trait loci (eQTLs), sections of DNA that are associated with differences in gene expression between individuals.</p>
<p>These differences in how genes are expressed may have functional significance and perhaps even explain varying risk for diseases. Since many of these differences in expression occur at intermediate points during development, however, scientists can't see them by studying only mature, fully-developed tissues.</p>
<p>&quot;Those associations are like shooting stars,&quot; said Yoav Gilad, PhD, Chief of Genetic Medicine at UChicago and senior author of the study. &quot;They appear at one point and never again during development, and they might actually be important to the phenotype of the mature tissue and maybe even disease. But unless you study those particular cell types at that particular time, you'll never see them.&quot;</p>
<p>The project was a collaboration between Gilad's lab at UChicago and the lab of Alexis Battle, PhD, an Associate Professor of Biomedical Engineering at Johns Hopkins University. They started with pluripotent stem cells, a type of artificial stem cells that can be coaxed into growing into many different cell types. For this study, the cells were differentiated into cardiomyocytes, the primary contracting muscle cells in the heart.</p>
<p>The researchers sampled RNA from the cells once a day over 16 days as they developed into cardiomyocytes. This allowed them to measure gene expression every single day in cell types that were not truly the beginning stem cells and not truly mature heart cells either. Instead of getting one snapshot of genetic activity by sampling adult tissues, they were able to see 16 additional snapshots leading up to that point as well.</p>
<p>The fleeting differences that occur during development could help explain differences in risk for complex diseases such as cancer, heart disease or diabetes that aren't caused by a single genetic mutation. Instead, these complex diseases are likely caused by dozens, if not hundreds, of subtle genetic mutations combined with interactions with lifestyle and the environment. On their own, these small mutations don't affect your overall health, but together they can elevate risk for particular diseases. The new research shows that these small genetic differences could impact gene expression at many points along the way.</p>
<p>&quot;We think a lot of the relevant molecular changes that can ultimately explain your risk profile are not going to occur in your mature tissues of the heart, liver or pancreas that we can sample from adults. They're probably occurring somewhere much earlier.&quot;</p>
<p>The process of taking stem cells from a person, coaxing them into different kinds of mature cells -- while periodically sampling RNA expression along the way -- is still time consuming and expensive, so the findings of this study won't immediately turn into a diagnostic tool. But Gilad says as researchers learn more about what these &quot;shooting star&quot; differences in gene expression mean, they may be able to spot the final genetic signatures they leave in mature tissues. That way, if they saw a certain pattern in the genome, for example, they might know it meant there was a particular level of expression on a certain day of development for those specific cell types.</p>
<p>&quot;We can now think in another dimension,&quot; Gilad said. &quot;Instead of taking a human and sampling everything you see in front of you, we now know there's a history of how we got here, and some of those differences can't be observed anymore.&quot;</p>
</li>
<li>
<p><em>abstract</em> Genetic regulation of gene expression is dynamic, as transcription can change during cell differentiation and across cell types. We mapped expression quantitative trait loci (eQTLs) throughout differentiation to elucidate the dynamics of genetic effects on cell type–specific gene expression. We generated time-series RNA sequencing data, capturing 16 time points during the differentiation of induced pluripotent stem cells to cardiomyocytes, in 19 human cell lines. We identified hundreds of dynamic eQTLs that change over time, with enrichment in enhancers of relevant cell types. We also found nonlinear dynamic eQTLs, which affect only intermediate stages of differentiation and cannot be found by using data from mature tissues. These fleeting genetic associations with gene regulation may explain some of the components of complex traits and disease. We highlight one example of a nonlinear eQTL that is associated with body mass index.</p>
</li>
</ul>
</li>
</ul>
</li>
</ul>
<h2>05892</h2>
<ul>
<li>
<p><em>legacy effects of developmental stages determine the functional role of predators</em><br />
volker h. w. rudolf &amp; b. g. van allen 2017<br />
<a href="http://doi.org/10.1038/s41559-016-0038">doi.org/10.1038/s41559-016-0038</a></p>
<ul>
<li>Predators are instrumental in structuring natural communities and ecosystem processes. The strong effects of predators are often attributed to their high trophic position in the food web. However, most predators have to grow and move up the food chain before reaching their final trophic position, and during this developmental process their traits, interactions and abundances change. Here, we show that this process of ‘moving up’ the food chain during development strongly determines the ecological role of a predator. By experimentally manipulating the succession of developmental stages of a predatory salamander in a seasonal aquatic ecosystem, we found that the effects of this apex predator on the ecosystem typically declined with age and size. Furthermore, younger, smaller predator stages had long-lasting effects on community structure and ecosystem function that determined the effects of subsequent older, larger stages. Consequently, the legacy effects of early stages largely shaped the impact of the predator on the ecosystem, which could not simply be inferred from its final trophic position. Our results highlight that accounting for all life stages when managing natural populations is crucial to preserve the functioning of natural ecosystems, especially given that early life stages of species are often particularly vulnerable to natural and anthropogenic disturbances.</li>
</ul>
</li>
</ul>
<h2>05992</h2>
<ul>
<li>cascade: continuous and punctuated
<ul>
<li>
<p><em>self-motion evokes precise spike timing in the primate vestibular system</em><br />
mohsen jamali, maurice j. chacron, kathleen e. cullen 2016<br />
<a href="http://doi.org/10.1038/ncomms13229">doi.org/10.1038/ncomms13229</a></p>
<ul>
<li>The accurate representation of self-motion requires the efficient processing of sensory input by the vestibular system. Conventional wisdom is that vestibular information is exclusively transmitted through changes in firing rate, yet under this assumption vestibular neurons display relatively poor detection and information transmission. Here, we carry out an analysis of the system’s coding capabilities by recording neuronal responses to repeated presentations of naturalistic stimuli. We find that afferents with greater intrinsic variability reliably discriminate between different stimulus waveforms through differential patterns of precise (∼6 ms) spike timing, while those with minimal intrinsic variability do not. A simple mathematical model provides an explanation for this result. Postsynaptic central neurons also demonstrate precise spike timing, suggesting that higher brain areas also represent self-motion using temporally precise firing. These findings demonstrate that two distinct sensory channels represent vestibular information: one using rate coding and the other that takes advantage of precise spike timing.</li>
</ul>
</li>
</ul>
</li>
</ul>
<h2>06092</h2>
<ul>
<li>commensualism
<ul>
<li>
<p><em>fishing down nutrients on coral reefs</em><br />
jacob e. allgeier 2016<br />
<a href="http://doi.org/10.1038/ncomms12461">doi.org/10.1038/ncomms12461</a></p>
<ul>
<li>Fishing is widely considered a leading cause of biodiversity loss in marine environments, but the potential effect on ecosystem processes, such as nutrient fluxes, is less explored. Here, we test how fishing on Caribbean coral reefs influences biodiversity and ecosystem functions provided by the fish community, that is, fish-mediated nutrient capacity. Specifically, we modelled five processes of nutrient storage (in biomass) and supply (via excretion) of nutrients, as well as a measure of their multifunctionality, onto 143 species of coral reef fishes across 110 coral reef fish communities. These communities span a gradient from extreme fishing pressure to protected areas with little to no fishing. We find that in fished sites fish-mediated nutrient capacity is reduced almost 50%, despite no substantial changes in the number of species. Instead, changes in community size and trophic structure were the primary cause of shifts in ecosystem function. These findings suggest that a broader perspective that incorporates predictable impacts of fishing pressure on ecosystem function is imperative for effective coral reef conservation and management.</li>
</ul>
</li>
</ul>
</li>
</ul>
<h2>06192</h2>
<ul>
<li>
<p><em>coupling between distant biofilms and emergence of nutrient time-sharing</em><br />
liu j et al. 2017<br />
<a href="http://doi.org/10.1126/science.aah4204">doi.org/10.1126/science.aah4204</a></p>
<ul>
<li>Bacteria within communities can interact to organize their behavior. It remains unclear whether such interactions extend beyond a single community to coordinate the behavior of distant populations. We discovered that two <em>Bacillus subtilis</em> biofilm communities undergoing metabolic oscillations become coupled through electrical signaling and synchronize their growth dynamics. Coupling increases competition by also synchronizing demand for limited nutrients. As predicted by mathematical modeling, we confirm that biofilms resolve this conflict by switching from in-phase to anti-phase oscillations. This results in time-sharing behavior where each community takes turns consuming nutrients. Time-sharing enables biofilms to counterintuitively increase growth under reduced nutrient supply. Distant biofilms can thus coordinate their behavior to resolve nutrient competition through time-sharing, a strategy used in engineered systems to allocate limited resources.</li>
</ul>
</li>
</ul>
<h2>06292</h2>
<ul>
<li>
<p><em>what does it mean to be human?</em><br />
gaia vince discovers that analysing the genetics of ancient humans means changing ideas about our evolution 2017</p>
<p><a href="https://mosaicscience.com/story/ancient-human-evolution-neanderthal-genetics">mosaicscience.com/story/ancient-human-evolution-neanderthal-genetics</a></p>
<ul>
<li>
<p>The things that we thought we understood about Europeans are coming unstuck as we examine the genes of more ancient people. For example, it was generally accepted that pale skin evolved so we could get more vitamin D after moving north to where there was little sun and people had to cover up against the cold. But it turns out that it was the Yamnaya people from much further south, tall and brown-eyed, who brought pale skins to Europe. Northern Europeans before then were dark-skinned and got plenty of vitamin D from eating fish.<br />
It is the same with lactose tolerance. Around 90 per cent of Europeans have a genetic mutation that allows them to digest milk into adulthood, and scientists had assumed that this gene evolved in farmers in northern Europe, giving them an additional food supply to help survive the long winters. But Eske’s research using the genomes of hundreds of Bronze Age people, who lived after the advent of farming, has cast doubt on this theory too: “We found that the genetic trait was almost non-existent in the European population. This trait only became abundant in the northern European population within the last 2,000 years,” he says.<br />
It turns out that lactose tolerance genes were also introduced by the Yamnaya. “They had a slightly higher tolerance to milk than the European farmers and must have introduced it to the European gene pool. Maybe there was a disaster around 2,000 years ago that caused a population bottleneck and allowed the gene to take off. The Viking sagas talk about the sun becoming black – a major volcanic eruption – that could have caused a massive drop in population size, which could have been where some of that stock takes off with lactose.”</p>
</li>
<li>
<p>yamnaya<br />
<a href="https://mobile.nytimes.com/2015/06/16/science/dna-deciphers-roots-of-modern-europeans.html?_r=0&amp;referer=https://duckduckgo.com/">mobile.nytimes.com/2015/06/16/science/dna-deciphers-roots-of-modern-europeans.html</a></p>
</li>
<li>
<p><a href="http://www.nature.com/news/european-languages-linked-to-migration-from-the-east-1.16919">nature.com/news/european-languages-linked-to-migration-from-the-east-1.16919</a></p>
</li>
<li>
<p><em>ancient human genomes suggest three ancestral populations for present–day humans</em><br />
<a href="http://www.nature.com/articles/nature13673.epdf?referrer_access_token=dUGFQ4p5aSftizrGNtcLtdRgN0jAjWel9jnR3ZoTv0MaJb0jOMpBA1y27HQGlCUiHaqx24ysNx_tIj9u3K68hIb67qNeR62pjGLMM_oheXxudlZ3ujrsQdcQ3y1Hv43qAtla3QkjTusatPWqibFrGLtYM3oNKg5nWyQD1eagFah-7e4zquIYKbz-elLhtQJkPiggMbPOOQtzFYi1DswNxZwPkSWHeMCYkX--l9qLsMH8I6o9psIi6HKUXdio5DylzrTjc4FaiOtO4VzrDa3EPg%3D%3D&amp;tracking_referrer=www.nature.com">nature.com/articles/nature13673.epdf</a></p>
</li>
<li>
<p><em>neanderthal behaviour, diet, and disease inferred from ancient dna in dental calculus</em><br />
laura s. weyrich et al. 2017<br />
<a href="http://doi.org/10.1038/nature21674">doi.org/10.1038/nature21674</a></p>
<ul>
<li>Recent genomic data have revealed multiple interactions between Neanderthals and modern humans<a href="http://www.nature.com/nature/journal/vaop/ncurrent/full/nature21674.html#ref1">29</a>, but there is currently little genetic evidence regarding Neanderthal behaviour, diet, or disease. Here we describe the shotgun-sequencing of ancient DNA from five specimens of Neanderthal calcified dental plaque (calculus) and the characterization of regional differences in Neanderthal ecology. At Spy cave, Belgium, Neanderthal diet was heavily meat based and included woolly rhinoceros and wild sheep (mouflon), characteristic of a steppe environment. In contrast, no meat was detected in the diet of Neanderthals from El Sidrón cave, Spain, and dietary components of mushrooms, pine nuts, and moss reflected forest gathering<a href="http://www.nature.com/nature/journal/vaop/ncurrent/full/nature21674.html#ref2">30</a>, <a href="http://www.nature.com/nature/journal/vaop/ncurrent/full/nature21674.html#ref3">31</a>. Differences in diet were also linked to an overall shift in the oral bacterial community (microbiota) and suggested that meat consumption contributed to substantial variation within Neanderthal microbiota. Evidence for self-medication was detected in an El Sidrón Neanderthal with a dental abscess<a href="http://www.nature.com/nature/journal/vaop/ncurrent/full/nature21674.html#ref4">32</a> and a chronic gastrointestinal pathogen (<em>Enterocytozoon bieneusi</em>). Metagenomic data from this individual also contained a nearly complete genome of the archaeal commensal <em>Methanobrevibacter oralis</em> (10.2× depth of coverage)—the oldest draft microbial genome generated to date, at around 48,000 years old. DNA preserved within dental calculus represents a notable source of information about the behaviour and health of ancient hominin specimens, as well as a unique system that is useful for the study of long-term microbial evolution.</li>
</ul>
</li>
</ul>
</li>
</ul>
<h2>06392</h2>
<ul>
<li>
<p><em>variable habitat conditions drive species covariation in the human microbiota</em><br />
charles k. fisher et al. 2017<br />
<a href="http://doi.org/10.1371/journal.pcbi.1005435">doi.org/10.1371/journal.pcbi.1005435</a></p>
<ul>
<li>
<p>Two species with similar resource requirements respond in a characteristic way to variations in their habitat—their abundances rise and fall in concert. We use this idea to learn how bacterial populations in the microbiota respond to habitat conditions that vary from person-to-person across the human population. Our mathematical framework shows that habitat fluctuations are sufficient for explaining intra-bodysite correlations in relative species abundances from the Human Microbiome Project. We explicitly show that the relative abundances of closely related species are positively correlated and can be predicted from taxonomic relationships. We identify a small set of functional pathways related to metabolism and maintenance of the cell wall that form the basis of a common resource sharing niche space of the human microbiota.</p>
<p>The human body is inhabited by a vast number of microorganisms comprising the human microbiota. The species composition of the microbiota varies considerably from person-to-person and the relative abundances of some species rise and fall in concert. We introduce a mathematical model where differences in habitat conditions cause most of the variability of the microbiota. A statistical analysis shows that variable habitat conditions are sufficient for explaining the patterns of variation observed across a healthy human population and, as a result, the correlation between the relative abundances of two species reflects how closely related they are rather than how they directly interact with each other.</p>
</li>
</ul>
</li>
</ul>
<h2>06492</h2>
<ul>
<li>
<p><em>antibiotic-producing symbionts dynamically transition between plant pathogenicity and insect-defensive mutualism</em><br />
laura v. flórez et al. 2017<br />
<a href="https://doi.org/10.1038/ncomms15172">doi.org/10.1038/ncomms15172</a></p>
<ul>
<li>Pathogenic and mutualistic bacteria associated with eukaryotic hosts often lack distinctive genomic features, suggesting regular transitions between these lifestyles. Here we present evidence supporting a dynamic transition from plant pathogenicity to insect-defensive mutualism in symbiotic <em>Burkholderia gladioli</em> bacteria. In a group of herbivorous beetles, these symbionts protect the vulnerable egg stage against detrimental microbes. The production of a blend of antibiotics by <em>B. gladioli</em>, including toxoflavin, caryoynencin and two new antimicrobial compounds, the macrolide lagriene and the isothiocyanate sinapigladioside, likely mediate this defensive role. In addition to vertical transmission, these insect symbionts can be exchanged via the host plant and retain the ability to initiate systemic plant infection at the expense of the plant’s fitness. Our findings provide a paradigm for the transition between pathogenic and mutualistic lifestyles and shed light on the evolution and chemical ecology of this defensive mutualism.</li>
</ul>
</li>
</ul>
<h2>06592</h2>
<ul>
<li>
<p><em>cycles of external dependency drive evolution of avian carotenoid networks</em><br />
alexander v. badyaev et al. 2019<br />
<a href="http://doi.org/10.1038/s41467-019-09579-y">doi.org/10.1038/s41467-019-09579-y</a></p>
<ul>
<li>
<p>All living things exist within communities, where they depend on resources or services provided by other species. As community members change, so do the products the species depend on and share. The late George Gaylord Simpson, who was a professor of geosciences at the UA and one of the most influential evolutionary thinkers of the last century, proposed that these fluctuating dependencies should determine the speed of evolution.</p>
<p>The theory has been notoriously difficult to test because species interactions are both ubiquitous and ephemeral, said UA ecology and evolutionary biology professor Alexander Badyaev. But he and his team think they've found a way by examining evolution of biochemical pathways that produce color diversity in birds.</p>
<p>Badyaev and his co-authors showed that the way biochemical processes are structured in birds holds the key to understanding how species gain and lose their reliance on others in their communities. Consequently, this dictates how quickly species can diversify and evolve.</p>
<p>The new study, which was published in Nature Communications earlier this month, both confirms this prediction and reveals the mechanisms that show how it works.</p>
<p>Badyaev studied the evolution of the pathways by which birds convert dietary carotenoids into molecules necessary for everything from vision to the immune system to feather pigmentation.</p>
<p>The team, which included undergraduate and graduate students, and a postdoctoral fellow in Badyaev's lab, built and tested the structure of thousands of carotenoid biochemical pathways in nearly 300 bird species. Then, they explored how the pathways had changed over the last 50 million of years.</p>
<p>&quot;The importance of carotenoids for multiple functions contrasts with birds' inability to create carotenoids themselves,&quot; Badyaev said. &quot;So a species deriving its dietary carotenoids from a single food source is hostage to the source's disappearance.&quot;</p>
<p>The solution resides in the structure of the biochemical pathways, where the same molecules might be interchangeably produced by different dietary carotenoids. Not only does this enable species to reliably receive their essential carotenoids despite environmental fluctuations, it also allows birds to explore additional biochemical pathways. Badyaev calls this &quot;internalizing control.&quot;</p>
<p>&quot;Think about hanging by a rope off a cliff. With one rope, if it disappears, you die. If you have two and one fails, you get to live. But having a third safety rope allows enough stability that you can make something out of the first two -- like a ladder -- and thus take control of your trajectory while the stability lasts,&quot; Badyaev said.</p>
<p>His team found that when species temporarily internalize control over their carotenoid production by capitalizing on multiple sources of carotenoids, they evolve at exceptionally high rates and produce some of the most extravagantly colored birds in the world.</p>
<p>&quot;But the moment you do this, you become susceptible to new external controls, and then the cycle repeats itself,&quot; he said. &quot;This is because both gains and losses of external controls occur with equal frequency.&quot;</p>
<p>This research builds on both Darwin's theory of evolution by natural selection and Simpson's idea that an organism's evolution is dependent on others in their community.</p>
<p>&quot;It shows how adaptation and evolutionary change are linked mechanistically,&quot; Badyaev said. &quot;It shows why gaining and losing internal control is a key feature of evolution.&quot;</p>
</li>
<li>
<p><em>abstract</em> All organisms depend on input of exogenous compounds that cannot be internally produced. Gain and loss of such dependencies structure ecological communities and drive species’ evolution, yet the evolution of mechanisms that accommodate these variable dependencies remain elusive. Here, we show that historical cycles of gains and losses of external dependencies in avian carotenoid-producing networks are linked to their evolutionary diversification. This occurs because internalization of metabolic controls—produced when gains in redundancy of dietary inputs coincide with increased branching of their derived products—enables rapid and sustainable exploration of an existing network by shielding it from environmental fluctuations in inputs. Correspondingly, loss of internal controls constrains evolution to the rate of the gains and losses of dietary precursors. Because internalization of a network’s controls necessarily bridges diet-specific enzymatic modules within a network, it structurally links local adaptation and continuous evolution even for traits fully dependent on contingent external inputs.</p>
</li>
</ul>
</li>
</ul>
<h2>06692</h2>
<ul>
<li>
<p><em>sexual dimorphism and retinal mosaic diversification following the evolution of a violet receptor in butterflies</em><br />
kyle j. mcculloch et al. 2017<br />
<a href="http://doi.org/10.1093/molbev/msx163">doi.org/10.1093/molbev/msx163</a></p>
<ul>
<li>Numerous animal lineages have expanded and diversified the opsin-based photoreceptors in their eyes underlying color vision behavior. However, the selective pressures giving rise to new photoreceptors and their spectral tuning remain mostly obscure. Previously, we identified a violet receptor (UV2) that is the result of a UV opsin gene duplication specific to <em>Heliconius</em> butterflies*.* At the same time the violet receptor evolved, <em>Heliconius</em> evolved UV-yellow coloration on their wings, due to the pigment 3-hydroxykynurenine (3-OHK) and the nanostructure architecture of the scale cells. In order to better understand the selective pressures giving rise to the violet receptor, we characterized opsin expression patterns using immunostaining (14 species) and RNA-Seq (18 species), and reconstructed evolutionary histories of visual traits in five major lineages within <em>Heliconius</em> and one species from the genus <em>Eueides.</em> Opsin expression patterns are hyperdiverse within <em>Heliconius</em>. We identified six unique retinal mosaics and three distinct forms of sexual dimorphism based on ommatidial types within the genus <em>Heliconius</em>. Additionally, phylogenetic analysis revealed independent losses of opsin expression, pseudogenization events, and relaxation of selection on <em>UVRh2</em> in one lineage. Despite this diversity, the newly evolved violet receptor is retained across most species and sexes surveyed. Discriminability modeling of behaviorally preferred 3-OHK yellow wing coloration suggests that the violet receptor may facilitate <em>Heliconius</em> color vision in the context of conspecific recognition. Our observations give insights into the selective pressures underlying the origins of new visual receptors.</li>
</ul>
</li>
</ul>
<h2>06792</h2>
<ul>
<li>
<p><em>on the thermodynamic origin of metabolic scaling</em><br />
fernando j. ballesteros et al. 2018<br />
<a href="http://doi.org/10.1038/s41598-018-19853-6">doi.org/10.1038/s41598-018-19853-6</a></p>
<ul>
<li>The origin and shape of metabolic scaling has been controversial since Kleiber found that basal metabolic rate of animals seemed to vary as a power law of their body mass with exponent 3/4, instead of 2/3, as a surface-to-volume argument predicts. The universality of exponent 3/4 -claimed in terms of the fractal properties of the nutrient network- has recently been challenged according to empirical evidence that observed a wealth of robust exponents deviating from 3/4. Here we present a conceptually simple thermodynamic framework, where the dependence of metabolic rate with body mass emerges from a trade-off between the energy dissipated as heat and the energy efficiently used by the organism to maintain its metabolism. This balance tunes the shape of an additive model from which different effective scalings can be recovered as particular cases, thereby reconciling previously inconsistent empirical evidence in mammals, birds, insects and even plants under a unified framework. This model is biologically motivated, fits remarkably well the data, and also explains additional features such as the relation between energy lost as heat and mass, the role and influence of different climatic environments or the difference found between endotherms and ectotherms.</li>
</ul>
</li>
</ul>
<h2>06892</h2>
<ul>
<li>
<p><em>improbable destinies: fate, chance, and the future of evolution</em><br />
jonathan losos 2017 9780399184932</p>
<ul>
<li>interesting accounts of evolutionary experiments</li>
</ul>
</li>
<li>
<p><em>debating darwin</em><br />
robert j. richards and michael ruse 2016<br />
<a href="http://doi.org/10.7208/chicago/9780226384399.001.0001">doi.org/10.7208/chicago/9780226384399.001.0001</a></p>
</li>
</ul>
<h2>06992</h2>
<h2>07092</h2>
<ul>
<li>transmit across host organisms
<ul>
<li>culture
<ul>
<li>
<p><em>social transmission of avoidance among predators facilitates the spread of novel prey</em><br />
rose thorogood et al. 2017<br />
<a href="http://doi.org/10.1038/s41559-017-0418-x">doi.org/10.1038/s41559-017-0418-x</a></p>
<ul>
<li>Warning signals are an effective defence strategy for aposematic prey, but only if they are recognized by potential predators. If predators must eat prey to associate novel warning signals with unpalatability, how can aposematic prey ever evolve? Using experiments with great tits (<em>Parus major</em>) as predators, we show that social transmission enhances the acquisition of avoidance by a predator population. Observing another predator’s disgust towards tasting one novel conspicuous prey item led to fewer aposematic than cryptic prey being eaten for the predator population to learn. Despite reduced personal encounters with unpalatable prey, avoidance persisted and increased over subsequent trials. Next we use a mathematical model to show that social transmission can shift the evolutionary trajectory of prey populations from fixation of crypsis to fixation of aposematism more easily than was previously thought. Therefore, social information use by predators has the potential to have evolutionary consequences across ecological communities.</li>
</ul>
</li>
</ul>
</li>
<li>niche construction</li>
</ul>
</li>
</ul>
<h2>07192</h2>
<h2>07292</h2>
<ul>
<li>
<p><em>dual-stressor selection alters eco-evolutionary dynamics in experimental communities</em><br />
teppo hiltunen et al. 2018<br />
<a href="http://doi.org/10.1038/s41559-018-0701-5">doi.org/10.1038/s41559-018-0701-5</a></p>
<ul>
<li>
<p>Bacteria rarely live alone; they are usually part of a community of species that is exposed to various stress factors. They can often react to these factors by adapting to new environmental conditions with astonishing speed. Antibiotics that enter soil and water via waste water and accumulate there in low concentrations can trigger the evolution of resistance in bacteria -- even though these concentrations are so low that they inhibit bacterial growth only slightly or not at all. However, bacteria do not only have to fight antibiotics; they also have to deal with predators. This is why they often grow in large colonies that cannot be consumed by predatory organisms.</p>
<p>Typically, scientists investigate the effects that a single stress factor has on an organism. Researchers at the Max Planck Institute for Evolutionary Biology in Plön and the Universities of Helsinki and Jyväskylä, Finland, have now investigated the question of how microorganisms behave when they are confronted with more than one stress factor. &quot;We simulated natural environmental conditions in the lab and exposed bacteria to both predators and antibiotics. This allows us to estimate how likely it is to find evolution of resistance to antibiotics outdoors,&quot; explains study leader Lutz Becks.</p>
<p>Antibiotics and predators</p>
<p>In the scientists' laboratory, the bacterium Pseudomonas fluorescence had to cope with both antibiotics and the predatory single-cell organism Tetrahymena thermophila. After just a short time, the team of researchers noticed that the bacterial population was changing: the bacteria were much slower and less effective in developing resistance and protecting themselves from being consumed than others of the species that were only exposed to one of these factors. Moreover, resistance against the antibiotic was much less common. &quot;The bacteria were clearly unable to optimize both attributes at the same time,&quot; says Becks.</p>
<p>In the next step, the scientists analysed the genetic basis of these adaptations. Their results show that mutations for improved protection from predators appear in the same numbers and at the same places in the bacterial genome if only the predatory ciliates are present. The same applies to mutations that cause resistance to antibiotics. However, other mutations occur as soon as both stress factors influence the bacteria and the bacteria have to fight both predators and antibiotics. This causes both the bacteria's protection against predators and resistance to antibiotics to evolve more slowly and be less efficient.</p>
<p>Because the bacteria are less able to protect themselves from predators if they are confronted by the predatory ciliates and antibiotics simultaneously, their numbers are fewer than when they only have to defend themselves from predators. Several stress factors therefore appear to have a strong influence on whether and how often resistance to antibiotics develops and how large the population of bacteria can become.</p>
<p>&quot;Microbial populations -- whether in a lake or in the gut -- are complex communities in which many species have to compete for resources. The various stress factors to which microbes are exposed have an enormous effect on their evolution and survival rate. It will take some time until we fully understand the interaction of all these factors and the influence of antibiotics and pesticides,&quot; explains Becks.</p>
</li>
<li>
<p><em>abstract</em> Recognizing when and how rapid evolution drives ecological change is fundamental for our understanding of almost all ecological and evolutionary processes such as community assembly, genetic diversification and the stability of communities and ecosystems. Generally, rapid evolutionary change is driven through selection on genetic variation and is affected by evolutionary constraints, such as tradeoffs and pleiotropic effects, all contributing to the overall rate of evolutionary change. Each of these processes can be influenced by the presence of multiple environmental stressors reducing a population’s reproductive output. Potential consequences of multistressor selection for the occurrence and strength of the link from rapid evolution to ecological change are unclear. However, understanding these is necessary for predicting when rapid evolution might drive ecological change. Here we investigate how the presence of two stressors affects this link using experimental evolution with the bacterium Pseudomonas fluorescens and its predator Tetrahymena thermophila. We show that the combination of predation and sublethal antibiotic concentrations delays the evolution of anti-predator defence and antibiotic resistance compared with the presence of only one of the two stressors. Rapid defence evolution drives stabilization of the predator–prey dynamics but this link between evolution and ecology is weaker in the two-stressor environment, where defence evolution is slower, leading to less stable population dynamics. Tracking the molecular evolution of whole populations over time shows further that mutations in different genes are favoured under multistressor selection. Overall, we show that selection by multiple stressors can significantly alter eco-evolutionary dynamics and their predictability.</p>
</li>
</ul>
</li>
</ul>
<h2>07392</h2>
<h2>07492</h2>
<ul>
<li><em>rapid hybrid speciation in darwin’s finches</em><br />
sangeet lamichhaney et al. 2017<br />
<a href="http://doi.org/10.1126/science.aao4593">doi.org/10.1126/science.aao4593</a></li>
</ul>
<h2>07592</h2>
<ul>
<li>
<p><em>an inverse latitudinal gradient in speciation rate for marine fishes</em><br />
daniel l. rabosky et al. 2018<br />
<a href="http://doi.org/10.1038/s41586-018-0273-1">doi.org/10.1038/s41586-018-0273-1</a></p>
<ul>
<li>perate and polar regions, but the evolutionary and ecological causes of this pattern remain controversial1,2. Tropical marine fish communities are much more diverse than cold-water fish communities found at higher latitudes3,4, and several explanations for this latitudinal diversity gradient propose that warm reef environments serve as evolutionary ‘hotspots’ for species formation5–8. Here we test the relationship between latitude, species richness and speciation rate across marine fishes. We assembled a time-calibrated phylogeny of all ray-finned fishes (31,526 tips, of which 11,638 had genetic data) and used this framework to describe the spatial dynamics of speciation in the marine realm. We show that the fastest rates of speciation occur in species-poor regions outside the tropics, and that high-latitude fish lineages form new species at much faster rates than their tropical counterparts. High rates of speciation occur in geographical regions that are characterized by low surface temperatures and high endemism. Our results reject a broad class of mechanisms under which the tropics serve as an evolutionary cradle for marine fish diversity and raise new questions about why the coldest oceans on Earth are present-day hotspots of species formation.</li>
</ul>
</li>
<li>
<p><em>macroecology and macroevolution of the latitudinal diversity gradient in ants</em><br />
evan p. economo et al. 2018<br />
<a href="http://doi.org/10.1038/s41467-018-04218-4">doi.org/10.1038/s41467-018-04218-4</a></p>
<ul>
<li>The latitudinal diversity gradient—the tendency for more species to occur toward the equator—is the dominant pattern of life on Earth, yet the mechanisms responsible for it remain largely unexplained. Recently, the analysis of global data has led to advances in understanding, but these advances have been mostly limited to vertebrates and trees and have not provided consensus answers. Here we synthesize large-scale geographic, phylogenetic, and fossil data for an exemplar invertebrate group—ants—and investigate whether the latitudinal diversity gradient arose due to higher rates of net diversification in the tropics, or due to a longer time period to accumulate diversity due to Earth’s climatic history. We find that latitudinal affinity is highly conserved, temperate clades are young and clustered within tropical clades, and diversification rate shows no systematic variation with latitude. These results indicate that diversification time—and not rate—is the main driver of the diversity gradient in ants.</li>
</ul>
</li>
</ul>
<h2>07692</h2>
<ul>
<li>
<p>genome</p>
<ul>
<li>sea slugs taking genes and chloroplasts from bacteria
<ul>
<li>
<p>usf paper<br />
<a href="http://biology.usf.edu/ib/data/flyers/SCHWARTZ-FISH-LABELING-1-2015.pdf">biology.usf.edu/ib/data/flyers/SCHWARTZ-FISH-LABELING-1-2015.pdf</a></p>
<ul>
<li>
<p>201412</p>
<p>Reference: Biol. Bull. 227: 300–312. (December 2014)</p>
<p>FISH Labeling Reveals a Horizontally Transferred Algal (Vaucheria litorea) Nuclear Gene on a Sea Slug (Elysia chlorotica) Chromosome</p>
<p>JULIE A. SCHWARTZ1, NICHOLAS E. CURTIS2, AND SIDNEY K. PIERCE1,3*<br />
1Department of Integrative Biology, University of South Florida, Tampa, Florida 33620; 2Department of Biology and Chemistry, Ave Maria University, Ave Maria, Florida 34142; and 3Department of Biology, University of Maryland, College Park, Maryland 20742</p>
</li>
</ul>
</li>
<li>
<p><a href="http://io9.com/confirmation-that-photosynthesizing-sea-slugs-steal-gen-1683702602">io9</a> article</p>
</li>
</ul>
</li>
</ul>
</li>
<li>
<p>horizontal gene transfer<br />
not only in simple organisms</p>
<ul>
<li>
<p><em>horizontal gene transfer is more frequent with increased heterotrophy and contributes to parasite adaptation</em><br />
zhenzhen yang et al. 2016<br />
<a href="http://doi.org/10.1073/pnas.1608765113">doi.org/10.1073/pnas.1608765113</a></p>
<ul>
<li>
<p>Horizontal gene transfer (HGT) is the nonsexual transfer and genomic integration of genetic materials between organisms. In eukaryotes, HGT appears rare, but parasitic plants may be exceptions, as haustorial feeding connections between parasites and their hosts provide intimate cellular contacts that could facilitate DNA transfer between unrelated species. Through analysis of genome-scale data, we identified &gt;50 expressed and likely functional HGT events in one family of parasitic plants. HGT reflected parasite preferences for different host plants and was much more frequent in plants with increasing parasitic dependency. HGT was strongly biased toward expression and protein types likely to contribute to haustorial function, suggesting that functional HGT of host genes may play an important role in adaptive evolution of parasites.<br />
Abstract</p>
<p>Horizontal gene transfer (HGT) is the transfer of genetic material across species boundaries and has been a driving force in prokaryotic evolution. HGT involving eukaryotes appears to be much less frequent, and the functional implications of HGT in eukaryotes are poorly understood. We test the hypothesis that parasitic plants, because of their intimate feeding contacts with host plant tissues, are especially prone to horizontal gene acquisition. We sought evidence of HGTs in transcriptomes of three parasitic members of Orobanchaceae, a plant family containing species spanning the full spectrum of parasitic capabilities, plus the free-living Lindenbergia. Following initial phylogenetic detection and an extensive validation procedure, 52 high-confidence horizontal transfer events were detected, often from lineages of known host plants and with an increasing number of HGT events in species with the greatest parasitic dependence. Analyses of intron sequences in putative donor and recipient lineages provide evidence for integration of genomic fragments far more often than retro-processed RNA sequences. Purifying selection predominates in functionally transferred sequences, with a small fraction of adaptively evolving sites. HGT-acquired genes are preferentially expressed in the haustorium—the organ of parasitic plants—and are strongly biased in predicted gene functions, suggesting that expression products of horizontally acquired genes are contributing to the unique adaptive feeding structure of parasitic plants.</p>
</li>
</ul>
</li>
</ul>
</li>
<li>
<p><em>an antifungal polyketide associated with horizontally acquired genes supports symbiont-mediated defense in lagria villosa beetles</em><br />
laura v. flórez et al. 2018<br />
<a href="http://doi.org/10.1038/s41467-018-04955-6">doi.org/10.1038/s41467-018-04955-6</a></p>
<ul>
<li>Microbial symbionts are often a source of chemical novelty and can contribute to host defense against antagonists. However, the ecological relevance of chemical mediators remains unclear for most systems. Lagria beetles live in symbiosis with multiple strains of Burkholderia bacteria that protect their offspring against pathogens. Here, we describe the antifungal polyketide lagriamide, and provide evidence supporting that it is produced by an uncultured symbiont, Burkholderia gladioli Lv-StB, which is dominant in field-collected Lagria villosa. Interestingly, lagriamide is structurally similar to bistramides, defensive compounds found in marine tunicates. We identify a gene cluster that is probably involved in lagriamide biosynthesis, provide evidence for horizontal acquisition of these genes, and show that the naturally occurring symbiont strains on the egg are protective in the soil environment. Our findings highlight the potential of microbial symbionts and horizontal gene transfer as influential sources of ecological innovation.</li>
</ul>
</li>
<li>
<p><em>neighbor predation linked to natural competence fosters the transfer of large genomic regions in vibrio cholerae</em><br />
noémie matthe et al. 2019<br />
<a href="http://doi.org/10.7554/elife.48212">doi.org/10.7554/elife.48212</a></p>
<ul>
<li>
<p>In 2015, EPFL researchers led by Melanie Blokesch published a seminal paper in Science showing that the bacterium responsible for cholera, Vibrio cholerae, uses a spring-loaded spear to literally stab neighboring bacteria and steal their DNA. They identified the spear mechanism to be the so-called &quot;type VI secretion system&quot; or T6SS, also used for interbacterial competition by many other bacteria.</p>
<p>V. cholerae uses its T6SS to compete with other bacteria in its aquatic environment and acquire new genetic material, which the pathogen absorbs and exchanges against some parts of its own genome. This mode of &quot;horizontal gene transfer&quot; leads to rapid evolution and pathogen emergence. The pathogen V. cholerae has caused seven major cholera pandemics since 1817 and, according to current WHO data, still kills more than 100,000 people each year and infects up to 4 million others, mostly in poor or underdeveloped countries.</p>
<p>Now, Blokesch's group has discovered the extent of DNA that V. cholerae can steal in a single attack: more than 150,000 nucleic acid base pairs, or roughly 150 genes in one go (the cholera bacterium carries around 4,000 genes in total). The researchers calculated this number by sequencing the entire genome of almost 400 V. cholerae strains before and after stealing DNA from their neighboring bacteria.</p>
<p>Previous studies have tried to establish just how much DNA competent bacteria can absorb and integrate into their genome by feeding them large quantities of purified DNA. However, this does not reflect the natural conditions in which the bacteria live and operate, as long stretches of free DNA are scarce in the environment. V. cholerae's T6SS-mediated DNA stealing, however, allows the bacterium to acquire fresly released and long DNA fragments. This bacterial behavior is tightly linked to the chitinous surfaces (e.g. shells) on which the bacterium usually lives in oceans and estuaries.</p>
<p>To address this, the EPFL researchers studied two unrelated (&quot;non-clonal&quot;) strains of V. cholerae in co-cultures on chitin. This allowed them to determine the frequency and extent of DNA exchanges between the two strains, under more natural conditions.</p>
<p>The study, led by PhD student Noémie Matthey, found that V. cholerae strains that carry a functional and chitin-inducible T6SS system are much more efficient at transferring DNA than bacteria that don't. The predatory bacterium acquired large genomic regions from its killed prey -- up to the 150,000 base pairs mentioned above.</p>
<p>The authors conclude that the environmental &quot;lifestyle&quot; of V. cholerae enables exchange of genetic material with enough coding capacity that it can significantly accelerate the evolution of the bacterium.</p>
<p>&quot;This finding is very relevant in the context of bacterial evolution,&quot; says Blokesch. &quot;It suggests that environmental bacteria might share a common gene pool, which could render their genomes highly flexible and the microbes prone to quick adaption.&quot;</p>
</li>
<li>
<p><em>abstract</em> Natural competence for transformation is a primary mode of horizontal gene transfer. Competent bacteria are able to absorb free DNA from their surroundings and exchange this DNA against pieces of their own genome when sufficiently homologous. However, the prevalence of non-degraded DNA with sufficient coding capacity is not well understood. In this context, we previously showed that naturally competent Vibrio cholerae use their type VI secretion system (T6SS) to actively acquire DNA from non-kin neighbors. Here, we explored the conditions of the DNA released through T6SS-mediated killing versus passive cell lysis and the extent of the transfers that occur due to these conditions. We show that competent V. cholerae acquire DNA fragments with a length exceeding 150 kbp in a T6SS-dependent manner. Collectively, our data support the notion that the environmental lifestyle of V. cholerae fosters the exchange of genetic material with sufficient coding capacity to significantly accelerate bacterial evolution.</p>
</li>
</ul>
</li>
<li>
<p><em>de novo emergence of adaptive membrane proteins from thymine-rich genomic sequences</em><br />
nikolaos vakirlis et al. 2020<br />
<a href="http://doi.org/10.1038/s41467-020-14500-z">doi.org/10.1038/s41467-020-14500-z</a></p>
<ul>
<li>
<p>Over time, genes change via random mutations. Some of these changes result in serious defects and are rarely passed on to the next generations, others have little impact, and others confer significant advantages, which become favoured due to natural selection and end up being passed on to future generations. This is the main source of genetic novelty and how organisms differ from each other. However, genetic novelty can also be generated by totally new genes evolving from scratch.<br />
In the <em>eLife</em> study, the scientists devised a way of assessing just how frequently genes seem to evolve from scratch. Their results were surprising.<br />
Explaining de novo genes, first author on the paper, Nikolaos Vakirlis, Trinity, said: “Most of the genes in a genome have ‘cousins’ in the genomes of other species; genes made up of similar DNA sequences that, once translated into proteins, perform similar functions. However, some genes are unique and can only be found in a single, or small number of closely related species. We call these ‘orphan genes’ because they appear to have no relatives and are often responsible for unique characteristics and abilities of organisms. For example, a gene that is unique to cod fish living in the arctic allows them to survive in sub-zero temperatures.”<br />
Orphan genes pose a tough evolutionary problem though. They don’t look like other genes, so where do they come from? One idea is that they can originate seemingly from nothing: over long, evolutionary timescales, a completely novel gene can emerge de novo out of a region in the genome that is made up of junk DNA. Alternatively, with enough time, two ‘cousin’ genes can diverge so much that we can no longer identify the relationship between them. Thus, a gene may at a glance appear to be an orphan without having really emerged de novo.</p>
<p><strong>A new approach to assessing de novo gene frequency</strong><br />
For a long time, scientists thought the majority of orphan genes were simply cases of ‘missing relatives’, which could be explained by the divergence of the sequences through mutations during evolution. The new research suggests this is not the case.<br />
Aoife McLysaght, professor in genetics at Trinity College Dublin, said: “To our surprise, at most, around one third of orphan genes result from divergence. So, in turn, this suggests that most unique genes in the species we looked at are the result of other processes, including de novo emergence, which is therefore much more frequent than scientists initially thought.”<br />
<strong>Are de novo emerging genes important?</strong><br />
In the second piece of research, published recently in leading journal <em>Nature Communications</em>, the scientists sought an answer to the obvious question: Are de novo emerging genes important?</p>
<p>This may seem a paradoxical question because something that has not yet emerged fully in the world of evolution wouldn’t be expected to be overly important. After all, how can a gene that was never used before suddenly appear and play a major role?<br />
This paradox can be resolved if emerging genes have high potential to be beneficial for the organism. So, while they are expected to play no particular role in their current form, random changes that affect their sequences or increase the amount of protein they produce when translated should lead to beneficial effects.<br />
The scientists tested whether this hypothesis may be true by doing a series of biological and computational experiments using baker’s yeast as a model organism. And when they artificially allowed emerging sequences to be expressed at higher levels than they are naturally, the cells tended to grow faster.<br />
Importantly, growth was not enhanced by overexpressing established genes. So, emerging sequences do indeed carry the potential to be important to the cells.<br />
 Anne-Ruxandra Carvunis, Ph.D., assistant professor of computational systems biology at the University of Pittsburgh, said:”Order seems like something that’s hard to achieve, but our results go completely opposite to that. We found that simple order is rampant everywhere in the genome. The propensity to make simple shapes that are stable is already there, waiting to be exposed. De novo gene birth is thus becoming less and less mysterious as we better understand molecular innovation.”</p>
</li>
<li>
<p><em>abstract</em> Recent evidence demonstrates that novel protein-coding genes can arise de novo from non-genic loci. This evolutionary innovation is thought to be facilitated by the pervasive translation of non-genic transcripts, which exposes a reservoir of variable polypeptides to natural selection. Here, we systematically characterize how these de novo emerging coding sequences impact fitness in budding yeast. Disruption of emerging sequences is generally inconsequential for fitness in the laboratory and in natural populations. Overexpression of emerging sequences, however, is enriched in adaptive fitness effects compared to overexpression of established genes. We find that adaptive emerging sequences tend to encode putative transmembrane domains, and that thymine-rich intergenic regions harbor a widespread potential to produce transmembrane domains. These findings, together with in-depth examination of the de novo emerging YBR196C-A locus, suggest a novel evolutionary model whereby adaptive transmembrane polypeptides emerge de novo from thymine-rich non-genic regions and subsequently accumulate changes molded by natural selection.</p>
</li>
</ul>
</li>
<li>
<p><em>the snap hypothesis: chromosomal rearrangements could emerge from positive selection during niche adaptation</em><br />
gerrit brandis, diarmaid hughes 2020<br />
<a href="http://doi.org/10.1371/journal.pgen.1008615">doi.org/10.1371/journal.pgen.1008615</a></p>
<ul>
<li>
<p>Current theory (that evolution involves mistakes made when replicating a gene) explains how genes can mutate over time and acquire new meanings. However, a mystery in biology is that the relative locations of genes on chromosomes also changes over time. This is very obvious in bacteria, where different species often have the same genes in very different relative locations. Since the origin of life, genes have apparently been changing location. The questions are, how and why do genes move their relative locations?<br />
Now, scientists at Uppsala University have proposed an addition to the theory of evolution that can explain how and why genes move on chromosomes. The hypothesis, called the SNAP Hypothesis, is based on the observation that tandem duplications of sections of chromosome occur very frequently in bacteria (more than a million times more frequently than most mutations). These duplications are lost spontaneously unless they are selected. Selection to maintain a duplication can occur whenever bacteria find themselves in a sub-optimal environment, where having two copies of a particular gene could increase fitness (for example, if the duplicated region includes a gene that increases growth rate on a poor nutrient).<br />
Duplications typically contain hundreds of genes, even if only one is selected. The scientists Gerrit Brandis and Diarmaid Hughes argue that mutations can quickly accumulate in the hundreds of non-selected genes, including genes that are normally essential when there is only a single copy in the chromosome. Once two different essential genes are inactivated, one in each copy of the duplication, the duplication can no longer be lost. From this point on, the bacteria will have many genes unnecessarily duplicated, and mutations to inactivate or delete them will be positively selected because they increase fitness.<br />
Over time, all of the unnecessary duplicated genes may be lost by mutation, but this will happen randomly in each copy of the duplication. By this process of random loss of unnecessary duplicated genes in each copy of the duplication, the relative order of the remaining genes can be completely changed. The SNAP process can rearrange gene order very rapidly and it may contribute to separating different species.</p>
</li>
<li>
<p><em>abstract</em> All life on earth has evolved from a universal common ancestor with a specific order of genes on the chromosome. This order is not maintained in modern species and the standard hypothesis is that changes reflect a lack of strong selection on gene order. Here, we propose an alternative hypothesis, SNAP. The occupation of a novel environment by bacteria is generally a trade-off situation. For example, while the bacteria may not be adapted to grow well under the new conditions, they may benefit by not having to share available resources with other microorganisms. Bacterial populations frequently acquire duplications of chromosomal segments containing genes that can help them adapt to a new environment. Other genes that are also duplicated are not required in two copies so that over time a superfluous copy can be lost. Eventually, the process of duplication and gene loss can lead to the rearrangement of the gene order in the chromosomal segment. The major benefit of this model over the standard hypothesis is that the process is driven by positive selection and can reach fixation rapidly.</p>
</li>
</ul>
</li>
</ul>
<h2>07792</h2>
<ul>
<li>cancer
<ul>
<li>
<p><em>extensive recovery of embryonic enhancer and gene memory stored in hypomethylated enhancer dna</em><br />
unmesh jadhav et al. 2019<br />
<a href="http://doi.org/10.1016/j.molcel.2019.02.024">doi.org/10.1016/j.molcel.2019.02.024</a></p>
<p>pretty much provides the evidence supporting davies and lineweaver 2011</p>
<ul>
<li>
<p>&quot;We discovered that adult cells maintain a catalog of all of the genes in use early in development -- a record of the stage in which organs and tissues are formed within the embryo,&quot; says the senior author of the new study, Ramesh Shivdasani, MD, PhD, of Dana-Farber, Brigham and Women's Hospital, Harvard Medical School, and the Harvard Stem Cell Institute. &quot;Beyond the sheer existence of this archive, we were surprised to find that it doesn't remain permanently locked away but can be accessed by cells under certain conditions. The implications of this discovery for how we think about cells' capabilities, and for the future treatment of degenerative and other diseases, are potentially profound.&quot;</p>
<p>The &quot;embryonic memory&quot; discovered by Shivdasani and his colleagues takes the form of molecules called methyl groups that bind to and detach from the DNA within cells. The placement of these methyl groups -- which portion of DNA they bind to, and in what numbers -- determines which genes are active and which are not. The arrangement of methyl groups in a given section of DNA is known as its methylation pattern.</p>
<p>In the new study, researchers focused on the methylation pattern of regions of DNA known as enhancers. Enhancers can be thought of as keys for switching genes on and off. To activate a gene, DNA forms a loop that brings an enhancer close to the coding portion of the gene -- the section that contains the blueprint for making a protein. Then, along with other regions of DNA and specialized proteins, the genetic code embedded in DNA is converted into RNA.</p>
<p>Over the course of embryonic and fetal development, as cells evolve to take on the specific characteristics of the hundreds of types of adult tissues, cells &quot;are constantly making choices about what kind of cell they will become,&quot; Shivdasani explains. &quot;This process, known as cell differentiation, involves cells flipping different genes on and off using different enhancers.&quot; At each stage of development, particular sets of enhancers become active, much as individual sections of an orchestra play during different portions of a symphony.</p>
<p>By the time a child is fully formed, the set of active enhancers remains largely unchanged for the remainder of life (although the liver, for example, becomes larger as a child grows, its identity as a liver is consistent). For the most part, enhancers that were used early in development but are now idle &quot;look like they've been shut down,&quot; Shivdasani says. &quot;They don't seem to have the features of activity.&quot;</p>
<p>One of the distinguishing features of enhancers is that certain sections of them -- where the C molecule of the genetic code is followed by the G molecule -- are largely shorn of methyl groups, a state known as hypomethylation. This is true even of enhancers that have been shut down after their role in embryonic development ended. Scientists didn't know, however, how extensively cells preserve this memory of their earliest incarnations, and whether these memories can be accessed.</p>
<p>The results of the new study were illuminating on both counts. In intestinal cells from adult mice, Shivdasani and his colleagues found a nearly complete archive of enhancers that were active in the formative stages of intestinal development. Moreover, they found that in the absence of a protein called Polycomb Repressive Complex 2 (PRC2), most of these mothballed enhancers returned to activity within two weeks' time. (PRC2 is one of the major proteins used by cells to turn off specific genes.)</p>
<p>&quot;We showed that adult cells not only retain a memory of the embryonic and fetal period but also that, under certain circumstances, this memory can be recovered,&quot; Shivdasani remarks. &quot;The archive is stored safely and can be recalled with remarkable specificity and accuracy.&quot;</p>
<p>At this point, researchers can only speculate about why adult cells preserve these molecular memories. One possibility is that they're simply relics of an earlier stage of cells' lineage -- fossils of their course of development. Another is that cells may need to summon these memories -- to bring them to life, in effect -- in order to generate fresh tissue to repair damage. &quot;If the body needs to regenerate tissue that is damaged, it may be necessary for cells within that tissue to replay what happened in the embryo,&quot; Shivdasani states.</p>
<p>The findings may open a new chapter in regenerative medicine, as scientists explore whether cell memory can be harnessed to generate replacement tissue for organs that are damaged or diseased, the study authors say. Since such tissue would be derived from patients' own cells, there would be no risk of rejection by the immune system.</p>
<p>The discovery may also hold promise for cancer treatment. It's thought that one of the ways cancer cells gain the ability to leave the original tumor and metastasize is by switching on genes that were active during fetal development but later became dormant. Knowing that cells keep a record of their once-active enhancers may suggest new targets for therapies aimed at halting or preventing metastasis in patients.</p>
</li>
<li>
<p><em>abstract</em> •Hypomethylated DNA preserves nearly complete catalogs of developmental enhancers<br />
•Adult H3K4me1+H3K27ac− enhancers are not poised but remnants of fetal gene activity<br />
•TFs relieved of PRC2 repression selectively reactivate hypomethylated enhancers<br />
•Recommissioned enhancers drive tissue-specific fetal and embryonic gene activity</p>
<p>Developing and adult tissues use different cis-regulatory elements. Although DNA at some decommissioned embryonic enhancers is hypomethylated in adult cells, it is unknown whether this putative epigenetic memory is complete and recoverable. We find that, in adult mouse cells, hypomethylated CpG dinucleotides preserve a nearly complete archive of tissue-specific developmental enhancers. Sites that carry the active histone mark H3K4me1, and are therefore considered “primed,” are mainly cis elements that act late in organogenesis. In contrast, sites decommissioned early in development retain hypomethylated DNA as a singular property. In adult intestinal and blood cells, sustained absence of polycomb repressive complex 2 indirectly reactivates most—and only—hypomethylated developmental enhancers. Embryonic and fetal transcriptional programs re-emerge as a result, in reverse chronology to cis element inactivation during development. Thus, hypomethylated DNA in adult cells preserves a “fossil record” of tissue-specific developmental enhancers, stably marking decommissioned sites and enabling recovery of this epigenetic memory.</p>
</li>
</ul>
</li>
<li>
<p><em>cancer tumors as metazoa 1.0: tapping genes of ancient ancestors</em><br />
p c w davies and c h lineweaver 2011<br />
<a href="http://doi.org/10.1088/1478-3975/8/1/015001">doi.org/10.1088/1478-3975/8/1/015001</a></p>
<ul>
<li>The genes of cellular cooperation that evolved with multicellularity about a billion years ago are the same genes that malfunction to cause cancer. We hypothesize that cancer is an atavistic condition that occurs when genetic or epigenetic malfunction unlocks an ancient 'toolkit' of pre-existing adaptations, re-establishing the dominance of an earlier layer of genes that controlled loose-knit colonies of only partially differentiated cells, similar to tumors. The existence of such a toolkit implies that the progress of the neoplasm in the host organism differs distinctively from normal Darwinian evolution. Comparative genomics and the phylogeny of basal metazoans, opisthokonta and basal multicellular eukaryotes should help identify the relevant genes and yield the order in which they evolved. This order will be a rough guide to the reverse order in which cancer develops, as mutations disrupt the genes of cellular cooperation. Our proposal is consistent with current understanding of cancer and explains the paradoxical rapidity with which cancer acquires a suite of mutually-supportive complex abilities. Finally we make several predictions and suggest ways to test this model.</li>
</ul>
</li>
<li>
<p><em>primordial germ cells as a potential shared cell of origin for mucinous cystic neoplasms of the pancreas and mucinous ovarian tumors</em><br />
kevin m elias et al. 2018<br />
<a href="http://doi.org/10.1002/path.5161">doi.org/10.1002/path.5161</a></p>
<ul>
<li>Mucinous ovarian tumors (MOTs) morphologically and epidemiologically resemble mucinous cystic neoplasms (MCNs) of the pancreas, sharing a similar stroma and both occurring disproportionately among young females. Additionally, MOTs and MCNs share similar clinical characteristics and immunohistochemical phenotypes. Exome sequencing has revealed frequent recurrent mutations in KRAS and RNF43 in both MOTs and MCNs. The cell of origin for these tumors remains unclear, but MOTs sometimes arise in the context of mature cystic teratomas and other primordial germ cell (PGC) tumors. We undertook the present study to investigate whether non‐teratoma‐associated MOTs and MCNs share a common cell of origin. Comparisons of the gene expression profiles of MOTs [including both the mucinous borderline ovarian tumors (MBOTs) and invasive mucinous ovarian carcinomas (MOCs)], high‐grade serous ovarian carcinomas, ovarian surface epithelium, Fallopian tube epithelium, normal pancreatic tissue, pancreatic duct adenocarcinomas, MCNs, and single‐cell RNA‐sequencing of PGCs revealed that both MOTs and MCNs are more closely related to PGCs than to either eutopic epithelial tumors or normal epithelia. We hypothesize that MCNs may arise from PGCs that stopped in the dorsal pancreas during their descent to the gonads during early human embryogenesis, while MOTs arise from PGCs in the ovary. Together, these data suggest a common pathway for the development of MCNs and MOTs, and suggest that these tumors may be more properly classified as germ cell tumor variants.</li>
</ul>
</li>
<li>
<p><em>single cell transcriptomes from human kidneys reveal the cellular identity of renal tumours</em><br />
matthew d young et al. 2018<br />
<a href="http://doi.org/10.1126/science.aat1699">doi.org/10.1126/science.aat1699</a></p>
<ul>
<li>Messenger RNA encodes cellular function and phenotype. In the context of human cancer, it defines the identities of malignant cells and the diversity of tumor tissue. We studied 72,501 single-cell transcriptomes of human renal tumors and normal tissue from fetal, pediatric, and adult kidneys. We matched childhood Wilms tumor with specific fetal cell types, thus providing evidence for the hypothesis that Wilms tumor cells are aberrant fetal cells. In adult renal cell carcinoma, we identified a canonical cancer transcriptome that matched a little-known subtype of proximal convoluted tubular cell. Analyses of the tumor composition defined cancer-associated normal cells and delineated a complex vascular endothelial growth factor (VEGF) signaling circuit. Our findings reveal the precise cellular identities and compositions of human kidney tumors.</li>
</ul>
</li>
<li>
<p>canine transmissible venereal tumour</p>
<ul>
<li>
<p><em>somatic evolution and global expansion of an ancient transmissible cancer lineage</em><br />
baez-ortega et al. 2019<br />
<a href="http://doi.org/10.1126/science.aau9923">doi.org/10.1126/science.aau9923</a></p>
<ul>
<li>
<p>A detailed genetic study, published today in Science, reveals some surprising -- and even mysterious -- findings about how this cancer, that has survived for thousands of years, has mutated and evolved over time.</p>
<p>'Canine transmissible venereal tumour' is a cancer that spreads between dogs through the transfer of living cancer cells, primarily during mating. The disease usually manifests as genital tumours in both male and female domestic dogs. It first arose in an individual dog, but survived beyond the death of the original dog by spreading to new dogs. The cancer is now found in dog populations worldwide, and is the oldest and most prolific cancer lineage known in nature.</p>
<p>One of the most remarkable aspects of these tumours is that their cells are those of the original dog in which the cancer arose, and not the carrier dog. The only differences between cells in the modern dogs' tumours and cells in the original tumour are those that have arisen over time either through spontaneous changes in the cells' DNA or through changes caused by carcinogens.</p>
<p>An international team of researchers, led by scientists at the Transmissible Cancer Group at the University of Cambridge, has compared differences in tumours taken from 546 dogs worldwide to try to understand how the disease arose and how it managed to spread around the world.</p>
<p>&quot;This tumour has spread to almost every continent, evolving as it spreads,&quot; says Adrian Baez-Ortega, a PhD student in the Transmissible Cancer Group, part of Cambridge's Department of Veterinary Medicine. &quot;Changes to its DNA tell a story of where it has been and when, almost like a historical travel journal.&quot;</p>
<p>Using the data, they created a phylogenetic tree -- a type of family tree of the different mutations in the tumours. This allowed them to estimate that the cancer first arose between 4,000 and 8,500 years ago, most likely in Asia or Europe. All of the modern tumours can be traced back to a common ancestor around 1,900 years ago.</p>
<p>The researchers say that the cancer first spread from Europe to the Americas around 500 years ago, when European settlers first arrived at the continent by sea. Almost all the tumours found today in North, Central and South America descend from this single introduction event.</p>
<p>From the Americas, the disease spread further, to Africa and back into the Indian subcontinent -- almost all places that were, at the time, European colonies. For example, the cancer is seen in Reunion, but this was where European travellers would stop off on the way to India. All of this evidence suggests that the tumour was spread by sea-faring dogs, transported through maritime activities.</p>
<p>While the findings related to the historical spread of the disease are interesting, it is the tumour's evolution that particularly excites the researchers.</p>
<p>Recent developments in cancer biology have enabled scientists to look at the mutations in tumour DNA and identify unique signatures left by carcinogens. This allows them to see, for example, the damage that ultraviolet (UV) light causes.</p>
<p>Using these techniques, the researchers identified signatures for five different biological processes that have damaged the canine tumour over its history. Four of these, including exposure to UV light, are known processes already linked to human cancers. However, one of them -- termed 'Signature A' -- has a very distinctive mutational signature, different to any seen previously: it caused mutations only in the tumour's distant past, several thousand years ago, and has never been seen since.</p>
<p>&quot;This is really exciting -- we've never seen anything like the pattern caused by this carcinogen before,&quot; says Dr Elizabeth Murchison, who leads the Transmissible Cancer Group at the University of Cambridge.</p>
<p>&quot;It looks like the tumour was exposed to something thousands of years ago that caused changes to its DNA for some length of time and then disappeared. It's a mystery what the carcinogen could be. Perhaps it was something present in the environment where the cancer first arose.&quot;</p>
<p>Another intriguing discovery related to how the tumours evolve. There are two main types of selection in evolutionary theory -- positive and negative. Positive selection is where mutations that provide an organism with a particular advantage are more likely to be passed down generations; negative selection is where mutations that are likely to have a deleterious effect are less likely to be passed on. Such selection tends to occur by way of sexual reproduction.</p>
<p>When the researchers analysed the tumours, they found no evidence of either positive or negative selection. This implies that the tumour will be accumulating more and more potentially damaging mutations over time, making it less and less fit to its environment.</p>
<p>Baez-Ortega explains: &quot;Normally, we see selection pressures acting on an organism's evolution. These canine tumours are foreign bodies, so one would expect to see a battle between them and the dog's immune system, leading to only the strongest tumours successfully being transmitted. This doesn't seem to be happening here.</p>
<p>&quot;This cancer 'parasite' has proved remarkably successful at surviving over thousands of years, yet is steadily deteriorating. It suggests that its days may be numbered -- but it's likely to be tens of thousands of years before it disappears.&quot;</p>
</li>
<li>
<p><em>abstract</em> The canine transmissible venereal tumor (CTVT) is a sexually transmitted cancer that manifests as genital tumors in dogs. This cancer first arose in an individual “founder dog” several thousand years ago and has since survived by transfer of living cancer cells to new hosts during coitus. Today, CTVT affects dogs around the world and is the oldest and most prolific known cancer lineage. CTVT thus provides an opportunity to explore the evolution of cancer over the long term and to track the unusual biological transition from multicellular organism to obligate conspecific asexual parasite. Furthermore, the CTVT genome, acting as a living biomarker, has recorded the changing mutagenic environments experienced by this cancer throughout millennia and across continents.</p>
</li>
</ul>
</li>
<li>
<p><em>the cancer which survived: insights from the genome of an 11 000 year-old cancer</em><br />
andrea strakova, elizabeth p murchison 2015<br />
<a href="http://doi.org/10.1016/j.gde.2015.03.005">doi.org/10.1016/j.gde.2015.03.005</a></p>
</li>
</ul>
</li>
<li>
<p>tasmanian devils</p>
</li>
<li>
<p><em>cancer cells enter dormancy after cannibalizing mesenchymal stem/stromal cells (mscs)</em><br />
thomas j. bartosh et al. 2016<br />
<a href="http://doi.org/10.1073/pnas.1612290113">doi.org/10.1073/pnas.1612290113</a></p>
<ul>
<li>Patients with breast cancer often develop malignant regrowth of residual drug-resistant dormant tumor cells years after primary treatment, a process defined as cancer relapse. Deciphering the causal basis of tumor dormancy therefore has obvious therapeutic significance. Because cancer cell behavior is strongly influenced by stromal cells, particularly the mesenchymal stem/stromal cells (MSCs) that are actively recruited into tumor-associated stroma, we assessed the impact of MSCs on breast cancer cell (BCC) dormancy. Using 3D cocultures to mimic the cellular interactions of an emerging tumor niche, we observed that MSCs sequentially surrounded the BCCs, promoted formation of cancer spheroids, and then were internalized/degraded through a process resembling the well-documented yet ill-defined clinical phenomenon of cancer cell cannibalism. This suspected feeding behavior was less appreciable in the presence of a rho kinase inhibitor and in 2D monolayer cocultures. Notably, cannibalism of MSCs enhanced survival of BCCs deprived of nutrients but suppressed their tumorigenicity, together suggesting the cancer cells entered dormancy. Transcriptome profiles revealed that the resulting BCCs acquired a unique molecular signature enriched in prosurvival factors and tumor suppressors, as well as inflammatory mediators that demarcate the secretome of senescent cells, also referred to as the senescence-associated secretory phenotype. Overall, our results provide intriguing evidence that cancer cells under duress enter dormancy after cannibalizing MSCs. Importantly, our practical 3D coculture model could provide a valuable tool to understand the antitumor activity of MSCs and cell cannibalism further, and therefore open new therapeutic avenues for the prevention of cancer recurrence.</li>
</ul>
</li>
<li>
<p><em>the actin cytoskeletal architecture of estrogen receptor positive breast cancer cells suppresses invasion</em><br />
marco padilla-rodriguez et al. 2018<br />
<a href="http://doi.org/10.1038/s41467-018-05367-2">doi.org/10.1038/s41467-018-05367-2</a></p>
<ul>
<li>
<p>They found that estrogen enhances the production of EVL, which seems to keep cancer cells contained to the original tumor site. As estrogen levels fall, so do levels of EVL, freeing cancer cells to invade neighboring tissues — the first step in metastasis. EVL’s role in regulating a cell’s actin cytoskeleton could be the key to its ability to suppress cell movement.</p>
<p>“Cells have skeletons, just like we have a skeleton,” Dr. Padilla-Rodriguez explained. “Actin is one type of skeleton. Unlike our skeleton, actin can be remodeled, like Lego pieces being shaped into different structures.”</p>
<p>Depending on how a cell rearranges its actin cytoskeleton, it might be more likely to stay in one place, adhering to adjacent cells, or it could have the ability to migrate, crawling away from other cells like a microscopic caterpillar. “Generally, cells like to cluster together,” Dr. Padilla-Rodriguez said. “When we treat them with anti-estrogenic drugs, actin allows the cell to break away and pull itself forward.”</p>
<p>One of the next steps is to learn how to leverage EVL’s interaction with estrogen to develop combination treatments for patients with ER-positive tumors. While anti-estrogenic drugs such as tamoxifen rein in tumor growth, they also might indirectly reduce EVL levels, accelerating remaining cancer cells’ invasion of neighboring tissues.</p>
<p>“With tamoxifen, you’re inhibiting the brakes,” Dr. Mouneimne said. “Now we want to go after the gas pedal to halt the cancer from progressing. Then we will be inhibiting both growth and invasion.”</p>
</li>
<li>
<p><em>abstract</em> Estrogen promotes growth of estrogen receptor-positive (ER+) breast tumors. However, epidemiological studies examining the prognostic characteristics of breast cancer in postmenopausal women receiving hormone replacement therapy reveal a significant decrease in tumor dissemination, suggesting that estrogen has potential protective effects against cancer cell invasion. Here, we show that estrogen suppresses invasion of ER+ breast cancer cells by increasing transcription of the Ena/VASP protein, EVL, which promotes the generation of suppressive cortical actin bundles that inhibit motility dynamics, and is crucial for the ER-mediated suppression of invasion in vitro and in vivo. Interestingly, despite its benefits in suppressing tumor growth, anti-estrogenic endocrine therapy decreases EVL expression and increases local invasion in patients. Our results highlight the dichotomous effects of estrogen on tumor progression and suggest that, in contrast to its established role in promoting growth of ER+ tumors, estrogen has a significant role in suppressing invasion through actin cytoskeletal remodeling.</p>
</li>
</ul>
</li>
<li>
<p><em>antibiotics that target mitochondria effectively eradicate cancer stem cells, across multiple tumor types: treating cancer like an infectious disease</em><br />
rebecca lamb et al. 2015<br />
<a href="https://dx.doi.org/10.18632%2Foncotarget.3174">doi.org/10.18632%2Foncotarget.3174</a></p>
<ul>
<li>Here, we propose a new strategy for the treatment of early cancerous lesions and advanced metastatic disease, via the selective targeting of cancer stem cells (CSCs), a.k.a., tumor-initiating cells (TICs). We searched for a global phenotypic characteristic that was highly conserved among cancer stem cells, across multiple tumor types, to provide a mutation-independent approach to cancer therapy. This would allow us to target cancer stem cells, effectively treating cancer as a single disease of “stemness”, independently of the tumor tissue type. Using this approach, we identified a conserved phenotypic weak point – a strict dependence on mitochondrial biogenesis for the clonal expansion and survival of cancer stem cells. Interestingly, several classes of FDA-approved antibiotics inhibit mitochondrial biogenesis as a known “side-effect”, which could be harnessed instead as a “therapeutic effect”. Based on this analysis, we now show that 4-to-5 different classes of FDA-approved drugs can be used to eradicate cancer stem cells, in 12 different cancer cell lines, across 8 different tumor types (breast, DCIS, ovarian, prostate, lung, pancreatic, melanoma, and glioblastoma (brain)). These five classes of mitochondrially-targeted antibiotics include: the erythromycins, the tetracyclines, the glycylcyclines, an anti-parasitic drug, and chloramphenicol. Functional data are presented for one antibiotic in each drug class: azithromycin, doxycycline, tigecycline, pyrvinium pamoate, as well as chloramphenicol, as proof-of-concept. Importantly, many of these drugs are non-toxic for normal cells, likely reducing the side effects of anti-cancer therapy. Thus, we now propose to treat cancer like an infectious disease, by repurposing FDA-approved antibiotics for anti-cancer therapy, across multiple tumor types. These drug classes should also be considered for prevention studies, specifically focused on the prevention of tumor recurrence and distant metastasis. Finally, recent clinical trials with doxycycline and azithromycin (intended to target cancer-associated infections, but not cancer cells) have already shown positive therapeutic effects in cancer patients, although their ability to eradicate cancer stem cells was not yet appreciated.</li>
</ul>
</li>
<li>
<p><em>is adaptive therapy natural?</em><br />
frédéric thomas et al. 2018<br />
<a href="http://doi.org/10.1371/journal.pbio.2007066">doi.org/10.1371/journal.pbio.2007066</a></p>
<ul>
<li>
<p>Cancer therapies, even when initially very effective, only rarely cure disseminated cancers. Intrinsic or acquired resistance by the cancer cells to treatment lead to relapse, progression, and death <a href="https://journals.plos.org/plosbiology/article/figure?id=10.1371/journal.pbio.2007066.g001">41</a>. We propose that the emergence of resistant cancer cells requires two steps: first, the cells must deploy the necessary molecular machinery to overcome the toxic effects of the treatment. Second, the resistant cells must be sufficiently proliferative to repopulate the tumor. These steps must be deeply related to the cost of resistance. The resources needed to develop a resistant phenotype likely reduce fitness in the absence of therapy. Thus, wittingly or unwittingly, therapies govern the survivorship and proliferation of different cancer cell phenotypes within their tumor ecosystem.<br />
Understanding the molecular basis of cancer drug resistance is a promising way to develop future treatments that could potentially circumvent or eliminate the problem of resistance [2–4]. Alternatively, strategies like adaptive therapy [5,6] focus on exploiting ecological (changes in the tumor size) and evolutionary dynamics (changes in the frequency of different cancer cell phenotypes) to delay or prevent the proliferation of resistant phenotypes (e.g., <a href="https://elifesciences.org/lookup/doi/10.7554/eLife.19466.002">7</a>). A critical issue arises when considering cancer therapy as an evolutionary process. The “brute force” therapy, aimed at killing the maximum number of malignant cells, can actually accelerate the evolution and proliferation of resistant cells. High-dose therapies allow resistant cancer cells to win twice. First, they are resistant. Second, they are free of competition from sensitive cancer cells. This leads to a proposed evolution-based strategy that enforces a stable tumor burden by permitting the persistence of a significant population of chemo-sensitive cells. In so doing, chemo-sensitive cells can outcompete chemo-resistant subpopulations, hence limiting their expansion. Since acquisition of chemo resistance generally requires significant investment of resources, cancer cells are subject to an evolutionary trade-off (due to the “cost” of phenotypic resistance) between resistance and proliferation [8,9].<br />
Adaptive therapy was developed through mathematical models and computer simulations. In preclinical mouse studies and in a clinical trial on castrate-resistant metastatic prostate cancer, adaptive therapy delays or even prevents cancer progression as compared to traditional therapies, particularly those involving maximum tolerable dose <a href="https://elifesciences.org/content/6/e19466/article-info">5</a>. Despite these successes, extensive further investigation is needed to define and understand evolutionarily optimal cancer treatment strategies [8,10].<br />
The premise behind adaptive therapy is simple but relies on understanding the ecology and evolution of the cancer cell communities and their diversity of phenotypes. When “treatment to kill” is not possible for the metastatic disease, the goal should be to “treat to contain” in a manner that keeps the tumor burden below the level that threatens loss of life or even quality of life. In adaptive therapy, the treatment is used sparingly and in a temporally dynamic fashion (Fig 1A). The onset of therapy both reduces the tumor burden, drives down the population size of sensitive cells, and releases resistant cells from competition. Prior to driving the sensitive cells to near extinction, therapy is then withdrawn to permit their recovery. In the absence of therapy, the recovery of sensitive cells now has a competitive advantage over the resistant ones, thus acting to suppress them. Upon the recovery of the sensitive cell population, therapy is resumed, and the cycle repeats itself. The keys to the success of adaptive therapy in controlling (though not eliminating) the cancer cells are 1) the competitive advantage of sensitive cells relative to resistant ones in the absence of therapy, 2) the sparing use of therapy below that which would achieve maximum short-term kill rates, and 3) the strategic timing of therapy in response to overall tumor sizes and the frequencies of sensitive and resistant cell phenotypes.</p>
<p>(A) Common strategies employed in chemotherapy and in adaptive therapy to deal with the proliferation of malignant cells. In adaptive therapy, treatment is used sparingly and in a temporally dynamic fashion, and this both increases the competitive advantage of chemo-sensitive cells and maintains a stable tumor burden. B) In a similar manner, when the immune system kills only some sensitive/visible tumor cells, it allows for maintenance of a stable tumor burden because this population competes with cells that are resistant to immune attack.<br />
doi:10.1371/journal.pbio.2007066.g001<br />
<a href="https://journals.plos.org/plosbiology/article/figure?id=10.1371/journal.pbio.2007066.g001">More »</a><br />
Here, we propose that the strategies employed in adaptive therapy may also be employed by multicellular organisms to deal with the inevitable development of malignant cells during growth and maintenance of normal tissue. Multicell organisms by way of natural selection over many generations may have evolved forms of natural adaptive therapy (NAT). Part of the organism’s anticancer adaptations may include containing rather than just eliminating or preventing cancers (Fig 1B). Such NAT could, for example, account for autopsy studies showing that small cancers are commonly present in people and animals who have died from noncancer causes. Such observations have led to a hypothesis that cancer emerges frequently, but host suppressive mechanisms, such as the immune system, successfully limit their proliferation in a manner that does not adversely affect host fitness. We also suggest explanations for some of the paradoxical relationships that seem to occur between the immune system and malignant cell populations. Why, for example, does the immune system appear to promote tumor growth under some conditions? [11,12]<br />
It is widely accepted that the immune system is integral to tumor suppression. However, immunotherapies to increase that response may be counterproductive <a href="http://darwin-online.org.uk/contents.html">13</a>. If our NAT is operating to contain the cancer cells and restrain their evolution of immune resistance, then forcing the immune system to increase kill rates may actually undermine this balance, quicken the evolution of resistant cancer cells, and reduce the time to cancer progression, as the resistant cancer cells are now free to proliferate independent of the immune system. By emphasizing short-term gains from the immune system, the host’s immune system may lose effectiveness and relevancy in the long term.</p>
</li>
</ul>
</li>
</ul>
</li>
</ul>
<h2>07892</h2>
<ul>
<li>immune system
<ul>
<li>
<p><em>shaping variation in the human immune system</em><br />
liston, linterman, and carr 2016<br />
<a href="http://doi.org/10.1016/j.it.2016.08.002">doi.org/10.1016/j.it.2016.08.002</a></p>
<ul>
<li>
<p>Immune responses demonstrate a high level of intra-species variation, compensating for the specialization capacity of pathogens. The recent advent of in-depth immune phenotyping projects in large-scale cohorts has allowed a first look into the factors that shape the inter-individual diversity of the human immune system. Genetic approaches have identified genetic diversity as drivers of 20–40% of the variation between the immune systems of individuals. The remaining 60–80% is shaped by intrinsic factors, with age being the predominant factor, as well as by environmental influences, where cohabitation and chronic viral infections were identified as key mediators. We review and integrate the recent in-depth large-scale studies on human immune diversity and its potential impact on health.</p>
<p>Trends<br />
Diversity within the human immune system is stable over the duration of months to years with an elastic response to immunological challenge, allowing the study of immune drivers.</p>
<p>Genetic variation accounts for 20–40% of immune variation, with enrichment of gene variants associated with autoimmunity, inflammatory disease, and susceptibility to infections among the identified genetic drivers.</p>
<p>Among the identified intrinsic drivers of immune variation, age is the most potent, driving a shift from a precursor-biased immune status to an inflammation-biased immune status.</p>
<p>A strong environmental effect on immune variation is observed, as revealed by cohabitation studies, with the strongest individual driver identified to date being chronic viral infection.</p>
</li>
</ul>
</li>
<li>
<p>bacterial immune system</p>
<ul>
<li>
<p><em>quorum sensing controls adaptive immunity through the regulation of multiple crispr-cas systems</em><br />
adrian g. patterson et al. 2016<br />
<a href="http://doi.org/10.1016/j.molcel.2016.11.012">doi.org/10.1016/j.molcel.2016.11.012</a></p>
<ul>
<li>
<p>Quorum sensing regulates the type I-E, I-F, and III-A CRISPR-Cas systems in <em>Serratia</em></p>
<p>SmaR represses <em>cas</em> gene and CRISPR expression in the absence of AHL signals</p>
<p>Both interference and adaptation are modulated by quorum sensing</p>
<p>Bacteria coordinate their defenses based on cell density and the risk of infection</p>
<p>Bacteria commonly exist in high cell density populations, making them prone to viral predation and horizontal gene transfer (HGT) through transformation and conjugation. To combat these invaders, bacteria possess an arsenal of defenses, such as CRISPR-Cas adaptive immunity. Many bacterial populations coordinate their behavior as cell density increases, using quorum sensing (QS) signaling. In this study, we demonstrate that QS regulation results in increased expression of the type I-E, I-F, and III-A CRISPR-Cas systems in <em>Serratia</em> cells in high-density populations. Strains unable to communicate via QS were less effective at defending against invaders targeted by any of the three CRISPR-Cas systems. Additionally, the acquisition of immunity by the type I-E and I-F systems was impaired in the absence of QS signaling. We propose that bacteria can use chemical communication to modulate the balance between community-level defense requirements in high cell density populations and host fitness costs of basal CRISPR-Cas activity.</p>
</li>
</ul>
</li>
</ul>
</li>
<li>
<p>immune system where we didn’t expect it</p>
<ul>
<li>brain
<ul>
<li>
<p><em>structural and functional features of central nervous system lymphatic vessels</em><br />
antoine louveau et al. 2015<br />
<a href="http://doi.org/10.1038/nature14432">doi.org/10.1038/nature14432</a></p>
<ul>
<li>One of the characteristics of the central nervous system is the lack of a classical lymphatic drainage system. Although it is now accepted that the central nervous system undergoes constant immune surveillance that takes place within the meningeal compartment<a href="http://www.nature.com/nature/journal/v523/n7560/full/nature14432.html#ref1">46</a>, <a href="http://www.nature.com/nature/journal/v523/n7560/full/nature14432.html#ref2">47</a>, <a href="http://www.nature.com/nature/journal/v523/n7560/full/nature14432.html#ref3">48</a>, the mechanisms governing the entrance and exit of immune cells from the central nervous system remain poorly understood<a href="http://www.nature.com/nature/journal/v523/n7560/full/nature14432.html#ref4">49</a>, <a href="http://www.nature.com/nature/journal/v523/n7560/full/nature14432.html#ref5">50</a>, <a href="http://www.nature.com/nature/journal/v523/n7560/full/nature14432.html#ref6">51</a>. In searching for T-cell gateways into and out of the meninges, we discovered functional lymphatic vessels lining the dural sinuses. These structures express all of the molecular hallmarks of lymphatic endothelial cells, are able to carry both fluid and immune cells from the cerebrospinal fluid, and are connected to the deep cervical lymph nodes. The unique location of these vessels may have impeded their discovery to date, thereby contributing to the long-held concept of the absence of lymphatic vasculature in the central nervous system. The discovery of the central nervous system lymphatic system may call for a reassessment of basic assumptions in neuroimmunology and sheds new light on the aetiology of neuroinflammatory and neurodegenerative diseases associated with immune system dysfunction.</li>
</ul>
</li>
</ul>
</li>
<li>testes
<ul>
<li>
<p><em>egress of sperm autoantigen from seminiferous tubules maintains systemic tolerance</em><br />
kenneth s.k. tung et al. 2017<br />
<a href="http://doi.org/10.1172/JCI89927">doi.org/10.1172/JCI89927</a></p>
<ul>
<li>Autoimmune responses to meiotic germ cell antigens (MGCA) that are expressed on sperm and testis occur in human infertility and after vasectomy. Many MGCA are also expressed as cancer/testis antigens (CTA) in human cancers, but the tolerance status of MGCA has not been investigated. MGCA are considered to be uniformly immunogenic and nontolerogenic, and the prevailing view posits that MGCA are sequestered behind the Sertoli cell barrier in seminiferous tubules. Here, we have shown that only some murine MGCA are sequestered. Nonsequestered MCGA (NS-MGCA) egressed from normal tubules, as evidenced by their ability to interact with systemically injected antibodies and form localized immune complexes outside the Sertoli cell barrier. NS-MGCA derived from cell fragments that were discarded by spermatids during spermiation. They egressed as cargo in residual bodies and maintained Treg-dependent physiological tolerance. In contrast, sequestered MGCA (S-MGCA) were undetectable in residual bodies and were nontolerogenic. Unlike postvasectomy autoantibodies, which have been shown to mainly target S-MGCA, autoantibodies produced by normal mice with transient Treg depletion that developed autoimmune orchitis exclusively targeted NS-MGCA. We conclude that spermiation, a physiological checkpoint in spermatogenesis, determines the egress and tolerogenicity of MGCA. Our findings will affect target antigen selection in testis and sperm autoimmunity and the immune responses to CTA in male cancer patients.</li>
</ul>
</li>
</ul>
</li>
</ul>
</li>
<li>
<p><em>regional differences in airway epithelial cells reveal tradeoff between defense against oxidative stress and defense against rhinovirus</em><br />
valia t. mihaylova et al. 2018<br />
<a href="http://doi.org/10.1016/j.celrep.2018.08.033">doi.org/10.1016/j.celrep.2018.08.033</a></p>
<ul>
<li>
<p>The research team used epithelial cells from healthy human donors. The cells were derived from either the nasal passages or the lungs. They exposed both cell types, maintained under the same conditions in cell culture, to rhinovirus. To their surprise, the researchers observed a more robust antiviral response in nasal cells.<br />
To investigate further, the researchers triggered the virus surveillance pathway — known as the RIG-I pathway — in both nasal and lung cells. They found that both cell types generated an antiviral response and a defense response against oxidative stress, a form of cell damage induced by viruses and other inhaled irritants such as cigarette smoke or tree pollen. In nasal cells, the antiviral response was stronger, but in bronchial cells, defense against oxidative stress was more pronounced.<br />
In additional experiments, the research team found evidence for a tradeoff: The defense response against oxidative stress shut off antiviral defenses. To probe this further, the team exposed nasal cells to oxidative stress in the form of cigarette smoke, and then to the cold virus, and found that the nasal cells were more susceptible to the virus. “They survive the cigarette smoke but can’t fight the virus as well,” Foxman said. “And the virus grows better.”<br />
This finding points to a delicate balance between the body’s different defense mechanisms, Foxman said. “Your airway lining protects against viruses but also other harmful substances that enter airways. The airway does pretty well if it encounters one stressor at a time. But when there are two different stressors, there’s a tradeoff,” Foxman explained. “What we found is that when your airway is trying to deal with another stress type, it can adapt but the cost is susceptibility to rhinovirus infection.”<br />
The study, she said, shows a mechanistic link between environmental exposures and susceptibility to the common cold</p>
</li>
<li>
<p><em>abstract</em> Activating RIG-I triggers host defense against viruses and oxidative stress<br />
These defenses are calibrated differently in nasal and bronchial epithelial cells<br />
Defense against oxidative stress antagonizes the antiviral interferon response</p>
<p>Rhinovirus is a leading cause of acute respiratory infections and asthma attacks, but infections are also frequently cleared from the nasal mucosa without causing symptoms. We sought to better understand host defense against rhinovirus by investigating antiviral defense in primary human nasal and bronchial airway epithelial cells cultured ex vivo. Surprisingly, upon rhinovirus infection or RIG-I stimulation, nasal-derived epithelial cells exhibited much more robust antiviral responses than bronchial-derived cells. Conversely, RIG-I stimulation triggered more robust activation of the NRF2-dependent oxidative stress response in bronchial cells compared to nasal cells. NRF2 activation dampened epithelial antiviral responses, whereas NRF2 knockdown enhanced antiviral responses and was protective during rhinovirus infection. These findings demonstrate a tradeoff in epithelial defense against distinct types of airway damage, namely, viral versus oxidative, and reveal differential calibration of defense responses in cells derived from different airway microenvironments.</p>
</li>
</ul>
</li>
</ul>
</li>
</ul>
<h2>07992</h2>
<ul>
<li>
<p><em>drastic genome reduction in an herbivore’s pectinolytic symbiont</em><br />
hassan salem et al. 2017<br />
<a href="http://doi.org/10.1016/j.cell.2017.10.029">doi.org/10.1016/j.cell.2017.10.029</a></p>
<ul>
<li>
<p>leafing through a 1953 edition of a book by the late Paul Buchner, a German scientist and one of the pioneers of systematic symbiosis research in insects. Buchner referenced a paper published in 1936 by one of his students, Hans-Jurgen Stammer, on <em>Cassida rubiginosa</em>.<br />
“Stammer wrote that, unlike most leaf-eating beetles that he had studied, this one had sac-like organs that he had never seen before and the organs were filled with micro-organisms,” says Salem, who looked up Stammer’s original paper in a now-obscure journal. “He didn’t have the high-powered microscopes that we have now, or genome sequencing technology, so he wasn’t able to comment on the functionality of the mysterious microbes. At that point, the idea that microbes could do anything beneficial for an animal was mushy science.”</p>
</li>
<li>
<p>“The most amazing thing to me is that we made this discovery because I read a really old book,” Salem says. “It speaks to the importance of natural history collections and libraries for old journals. We truly stand on the shoulders of giants, extending the work of those who came before us.”</p>
</li>
<li>
<p>Symbiosis is a strategy for an herbivore to gain pectinolytic metabolic enzymes<br />
•<em>Stammera</em> has the smallest genome of any known organism not living within a host cell<br />
•Specialized structures on the beetle house the symbiont and ensure vertical transfer<br />
•Symbiont acquisition likely relaxed selection for the host to endogenously maintain pectinases</p>
<p>Pectin, an integral component of the plant cell wall, is a recalcitrant substrate against enzymatic challenges by most animals. In characterizing the source of a leaf beetle’s (<em>Cassida rubiginosa</em>) pectin-degrading phenotype, we demonstrate its dependency on an extracellular bacterium housed in specialized organs connected to the foregut. Despite possessing the smallest genome (0.27 Mb) of any organism not subsisting within a host cell, the symbiont nonetheless retained a functional pectinolytic metabolism targeting the polysaccharide’s two most abundant classes: homogalacturonan and rhamnogalacturonan I. Comparative transcriptomics revealed pectinase expression to be enriched in the symbiotic organs, consistent with enzymatic buildup in these structures following immunostaining with pectinase-targeting antibodies. Symbiont elimination results in a drastically reduced host survivorship and a diminished capacity to degrade pectin. Collectively, our findings highlight symbiosis as a strategy for an herbivore to metabolize one of nature’s most complex polysaccharides and a universal component of plant tissues.</p>
</li>
</ul>
</li>
<li>
<p><em>reductions in complexity of mitochondrial genomes in lichen-forming fungi shed light on genome architecture of obligate symbioses</em><br />
cloe s. pogoda et al. 2018<br />
<a href="http://doi.org/10.1111/mec.14519">doi.org/10.1111/mec.14519</a></p>
<ul>
<li>Symbioses among co-evolving taxa are often marked by genome reductions such as a loss of protein-coding genes in at least one of the partners as a means of reducing redundancy or intergenomic conflict. To explore this phenomenon in an iconic yet under-studied group of obligate symbiotic organisms, mitochondrial genomes of 22 newly sequenced and annotated species of lichenized fungi were compared to 167 mitochondrial genomes of non-lichenized fungi. Our results demonstrate the first broad scale loss of <em>atp9</em> from mitochondria of lichenized fungi. Despite key functions in mitochondrial energy production, we show that <em>atp9</em> has been independently lost in three different lineages spanning 10 of the 22 studied species. A search for predicted, functional copies of <em>atp9</em> among genomes of other symbionts involved in each lichen revealed the full-length, presumably functional copies of <em>atp9</em> in either the photosynthetic algal partner and/or in other symbiotic fungi in all 10 instances. Together, these data yield evidence of an obligate symbiotic relationship in which core genomic processes have been streamlined, likely due to co-evolution.</li>
</ul>
</li>
</ul>
<h2>08092</h2>
<ul>
<li>
<p>transposons, “jumping genes”</p>
</li>
<li>
<p><em>ongoing transposon-mediated genome reduction in the luminous bacterial symbionts of deep-sea ceratioid anglerfishes</em><br />
tory a. hendry et al. 2018<br />
<a href="http://doi.org/10.1128/mBio.01033-18">doi.org/10.1128/mBio.01033-18</a></p>
<ul>
<li>Diverse marine fish and squid form symbiotic associations with extracellular bioluminescent bacteria. These symbionts are typically free-living bacteria with large genomes, but one known lineage of symbionts has undergone genomic reduction and evolution of host dependence. It is not known why distinct evolutionary trajectories have occurred among different luminous symbionts, and not all known lineages previously had genome sequences available. In order to better understand patterns of evolution across diverse bioluminescent symbionts, we <em>de novo</em> sequenced the genomes of bacteria from a poorly studied interaction, the extracellular symbionts from the “lures” of deep-sea ceratioid anglerfishes. Deep-sea anglerfish symbiont genomes are reduced in size by about 50% compared to free-living relatives. They show a striking convergence of genome reduction and loss of metabolic capabilities with a distinct lineage of obligately host-dependent luminous symbionts. These losses include reductions in amino acid synthesis pathways and abilities to utilize diverse sugars. However, the symbiont genomes have retained a number of categories of genes predicted to be useful only outside the host, such as those involved in chemotaxis and motility, suggesting that they may persist in the environment. These genomes contain very high numbers of pseudogenes and show massive expansions of transposable elements, with transposases accounting for 28 and 31% of coding sequences in the symbiont genomes. Transposon expansions appear to have occurred at different times in each symbiont lineage, indicating either independent evolutions of reduction or symbiont replacement. These results suggest ongoing genomic reduction in extracellular luminous symbionts that is facilitated by transposon proliferations.</li>
</ul>
</li>
<li>
<p><em>a robust transposon-endogenizing response from germline stem cells</em><br />
sungjin moon et al. 2018<br />
<a href="https://doi.org/10.1016/j.devcel.2018.10.011">doi.org/10.1016/j.devcel.2018.10.011</a></p>
<ul>
<li>
<p>“almost half of our DNA is made up of jumping genes—called transposons. Given their ability of jumping around the genome in developing sperm and egg cells, their invasion triggers DNA damage and mutations. This often leads to animal sterility or even death, threatening species survival. The high abundance of jumping genes implies that organisms have survived millions, if not billions, of transposon invasions. However, little is known about where this adaptability comes from. Now, a team of Carnegie researchers has discovered that, upon jumping gene invasion, reproductive stem cells boost production of non-coding RNA elements (piRNA) that suppress their activity and activates a DNA repair process allowing for normal egg development.”</p>
</li>
<li>
<p>•Shifting temperature can adjust the intensity of P element transposon invasion<br />
•Germline stem cells employ a robust adaptive response to endogenize invading transposons<br />
•Activation of the DNA damage checkpoint blocks oogenesis to tame transposons<br />
•Chk2-mediated adaptive response promotes piRNA production and silences transposons</p>
<p>The heavy occupancy of transposons in the genome implies that existing organisms have survived from multiple, independent rounds of transposon invasions. However, how and which host cell types survive the initial wave of transposon invasion remain unclear. We show that the germline stem cells can initiate a robust adaptive response that rapidly endogenizes invading P element transposons by activating the DNA damage checkpoint and piRNA production. We find that temperature modulates the P element activity in germline stem cells, establishing a powerful tool to trigger transposon hyper-activation. Facing vigorous invasion, Drosophila first shut down oogenesis and induce selective apoptosis. Interestingly, a robust adaptive response occurs in ovarian stem cells through activation of the DNA damage checkpoint. Within 4 days, the hosts amplify P element-silencing piRNAs, repair DNA damage, subdue the transposon, and reinitiate oogenesis. We propose that this robust adaptive response can bestow upon organisms the ability to survive recurrent transposon invasions throughout evolution.</p>
</li>
</ul>
</li>
</ul>
<h2>08192</h2>
<ul>
<li>
<p><em>ancient balancing selection on heterocyst function in a cosmopolitan cyanobacterium</em><br />
emiko b. sano et al. 2018<br />
<a href="http://doi.org/10.1038/s41559-017-0435-9">doi.org/10.1038/s41559-017-0435-9</a></p>
<ul>
<li>The conventional view of bacterial adaptation emphasizes the importance of rapidly evolved changes that are highly repeatable in response to similar environments and subject to loss in the absence of selection. Consequently, genetic variation is not expected to persist over long time scales for these organisms. Here, we show that a geographically widespread gene content polymorphism has surprisingly been maintained for tens of millions of years of diversification of the multicellular cyanobacterium <em>Fischerella thermalis</em>. The polymorphism affects gas permeability of the heterocyst—the oxygen-sensitive, nitrogen-fixing cell produced by these bacteria—and spatial variation in temperature favours alternative alleles due to thermodynamic effects on both heterocyst function and organism fitness at physiological temperature extremes. Whether or not ancient balancing selection plays a generally important role in the maintenance of microbial diversity remains to be investigated.</li>
</ul>
</li>
</ul>
<h2>08292</h2>
<ul>
<li>
<p><em>fine-scale characterization of genomic structural variation in the human genome reveals adaptive and biomedically relevant hotspots</em><br />
yen-lung lin, omer gokcumen 2019<br />
<a href="http://doi.org/10.1093/gbe/evz058">doi.org/10.1093/gbe/evz058</a></p>
<ul>
<li>
<p>Every person's genome is different, and the new study compared the DNA of more than 2,500 individuals.</p>
<p>Scientists zeroed in on the sections of the genome that differ most between people, identifying 1,148 areas that harbor unusually high numbers of structural variants, including chunks of duplicated, deleted, inserted, inverted or repeated sections of DNA.</p>
<p>New insights on the malleability of human DNA</p>
<p>An examination of these &quot;hot spots&quot; revealed a complex evolutionary story.</p>
<p>Most are found in gene-poor regions of the genome, as expected. (Altering genes can lead to devastating health problems, so it makes sense that gene-rich areas would tend to be more heavily conserved through evolution, Gokcumen explains.)</p>
<p>However, a small subset of structural variant hot spots is found in parts of the genome that harbor important genes. In these hubs, genes linked to our sense of smell, blood and skin function, and immunity to disease are overrepresented, according to the study.</p>
<p>Balancing selection -- in which dueling evolutionary forces drive a species to preserve an array of traits -- may help to explain why these gene-heavy hot spots exist.</p>
<p>One example: In the study, a DNA deletion that increases a person's risk for a blood disorder called thalassemia was found in about 16 percent of genes in sub-Saharan African populations. While evolution mostly weeded this genetic variation out of human societies in other parts of the world, the variation persists in sub-Saharan Africa because it's valuable there, Gokcumen says: The deletion may confer resistance to malaria, a major disease in the region.</p>
<p>&quot;There's an evolutionary reason why this mutation is lingering, despite its ill effects,&quot; he says. &quot;It's actually beneficial too, at least for some populations. Balancing selection is important for adaptation, and we think it contributes to the development of some structural variant hot spots.&quot;</p>
<p>If the findings on balancing selection showcase humanity's adaptability, a second result from the study hints at just how delicate we are -- at how easily problems can arise.</p>
<p>The conclusion has to do with the malleability of human DNA, and the possibility that some hot spots of variation may be located in sections of the genome that are, for biochemical reasons, more susceptible to being altered.</p>
<p>In most people, genetic mutations in these regions are not devastating. But in some cases, large genetic deletions that begin in one hotspot and end in another may result in the erasure of entire genes in between, leading to health complications, Gokcumen says.</p>
<p>One example: The study found that a number of consecutive structural variant hot spots lie on either side of the short stature homeobox (SHOX) gene, whose deletion can lead to a severe bone growth disorder that causes very short stature. In some people who are missing the SHOX gene, deletions of DNA began in one hotspot, spanned the entire SHOX gene, and ended in a second hotspot.</p>
<p>When Gokcumen and Lin ran statistical tests, they found that the start and end points of large genetic mutations with known medical relevance were found in structural variant hot spots more often than would be expected.</p>
</li>
<li>
<p><em>abstract</em> Genomic structural variants (SVs) are distributed nonrandomly across the human genome. The “hotspots” of SVs have been implicated in evolutionary innovations, as well as medical conditions. However, the evolutionary and biomedical features of these hotspots remain incompletely understood. Here, we analyzed data from 2,504 genomes to construct a refined map of 1,148 SV hotspots in human genomes. We confirmed that segmental-duplication related nonallelic homologous recombination is an important mechanistic driver of SV hotspot formation. However, to our surprise, we also found that a majority of SVs in hotspots do not form through such recombination-based mechanisms, suggesting diverse mechanistic and selective forces shaping hotspots. Indeed, our evolutionary analyses showed that the majority of SV hotspots are within gene-poor regions and evolve under relaxed negative selection or neutrality. However, we still found a small subset of SV hotspots harboring genes that are enriched for anthropologically crucial functions and evolve under geography-specific and balancing adaptive forces. These include two independent hotspots on different chromosomes affecting alpha and beta hemoglobin gene clusters. Biomedically, we found that the SV hotspots coincide with breakpoints of clinically relevant, large de novo SVs, significantly more often than genome-wide expectations. For example, we showed that the breakpoints of multiple large SVs, which lead to idiopathic short stature, coincide with SV hotspots. Therefore, the mutational instability in SV hotpots likely enables chromosomal breaks that lead to pathogenic structural variation formations. Overall, our study contributes to a better understanding of the mutational and adaptive landscape of the genome.</p>
</li>
</ul>
</li>
</ul>
<h2>08392</h2>
<h2>08492</h2>
<ul>
<li><em>tamed: ten species that changed our world</em><br />
alice roberts 2017 9781473538832</li>
</ul>
<h2>08592</h2>
<h2>08692</h2>
<ul>
<li>
<p><em>the genetic basis of adaptation following plastic changes in coloration in a novel environment</em><br />
ammon corl et al. 2018<br />
<a href="http://doi.org/10.1016/j.cub.2018.06.075">doi.org/10.1016/j.cub.2018.06.075</a></p>
<ul>
<li>
<pre><code>    Side-blotched lizards on a lava flow have melanic coloration that promotes crypsis  
      Ancestral plasticity in coloration promotes background matching to the lava habitat  
      Adaptive divergence in two genes that regulate melanin leads to darker lizards  
      Plasticity can help survival in new habitats, where selection then refines phenotypes  
</code></pre>
<strong>Summary</strong><br />
Phenotypic plasticity has been hypothesized to precede and facilitate adaptation to novel environments [1, 2, 3, 4, 5, 6, 7, 8], but examples of plasticity preceding adaptation in wild populations are rare (but see [9, 10]). We studied a population of side-blotched lizards, <em>Uta stansburiana</em>, living on a lava flow that formed 22,500 years ago <a href="http://wallace-online.org/">11</a> to understand the origin of their novel melanic phenotype that makes them cryptic on the black lava. We found that lizards living on and off of the lava flow exhibited phenotypic plasticity in coloration but also appeared to have heritable differences in pigmentation. We sequenced the exomes of 104 individuals and identified two known regulators of melanin production, PREP and PRKAR1A, which had markedly increased levels of divergence between lizards living on and off the lava flow. The derived variants in PREP and PRKAR1A were only found in the lava population and were associated with increased pigmentation levels in an experimental cohort of hatchling lizards. Simulations suggest that the derived variants in the PREP and PRKAR1A genes arose recently and were under strong positive selection in the lava population. Overall, our results suggest that ancestral plasticity for coloration facilitated initial survival in the lava environment and was followed by genetic changes that modified the phenotype in the direction of the induced plastic response, possibly through <em>de novo</em> mutations. These observations provide a detailed example supporting the hypothesis that plasticity aids in the initial colonization of a novel habitat, with natural selection subsequently refining the phenotype with genetic adaptations to the new environment.</li>
</ul>
</li>
</ul>
<h2>08792</h2>
<ul>
<li>
<p><em>a bacteriophage enzyme induces bacterial metabolic perturbation that confers a novel promiscuous function</em><br />
jon jerlström hultqvist et al. 2018<br />
<a href="http://doi.org/10.1038/s41559-018-0568-5">doi.org/10.1038/s41559-018-0568-5</a></p>
<ul>
<li>One key concept in the evolution of new functions is the ability of enzymes to perform promiscuous side-reactions that serve as a source of novelty that may become beneficial under certain conditions. Here, we identify a mechanism where a bacteriophage-encoded enzyme introduces novelty by inducing expression of a promiscuous bacterial enzyme. By screening for bacteriophage DNA that rescued an auxotrophic <em>Escherichia coli</em> mutant carrying a deletion of the <em>ilvA</em> gene, we show that bacteriophage-encoded S-adenosylmethionine (SAM) hydrolases reduce SAM levels. Through this perturbation of bacterial metabolism, expression of the promiscuous bacterial enzyme MetB is increased, which in turn complements the absence of IlvA. These results demonstrate how foreign DNA can increase the metabolic capacity of bacteria, not only by transfer of bona fide new genes, but also by bringing cryptic bacterial functions to light via perturbations of cellular physiology.</li>
</ul>
</li>
</ul>
<h2>08892</h2>
<ul>
<li>
<p>convergent evolution</p>
</li>
<li>
<p><em>agouti-related peptide 2 facilitates convergent evolution of stripe patterns across cichlid fish radiations</em><br />
claudius f. kratochwil et al. 2018<br />
<a href="http://doi.org/10.1126/science.aao6809">doi.org/10.1126/science.aao6809</a></p>
<ul>
<li>
<p>Which gene and which genetic mechanism are responsible for the cichlids' stripes to come and go has now been reconstructed in the laboratory through genome analyses, breeding and experiments, including CRISPR-Cas as &quot;gene scissors.&quot; Dr Claudius Kratochwil, early career researcher in Professor Meyers team and first author of the study in Science, explains: &quot;In breeding experiments we can exactly determine on which of the 22 chromosomes, even on which area of the chromosome of the fish, the genetic instruction for stripes is located.&quot; The relevant gene on this chromosome part is called agrp2. This &quot;stripe gene,&quot; its origin and distribution in other African lakes was described in comparative molecular studies. From an evolutionary point of view, the cichlids' stripes are rather unstable. Over the course of a few million years, they have been lost and re-emerged in the African lakes many times over. As these species (with and without stripes) are so young, they can be interbred in aquaria. Breeding and examining the cichlids with and without stripes in the laboratory shows that all cichlids carry the &quot;stripes gene,&quot; but the switches (regulatory elements) of this gene differ. &quot;This genetic switch causes the gene in species without stripes to be more activated. As a result, a lot of protein is produced. The &quot;stripes gene&quot; agrp2 works as a &quot;stripes inhibitor&quot;: if gene production is high, stripes will be suppressed, if production is low, they will remain. The researchers were able to demonstrate this by using modern genetic methods. &quot;If we use CRISPR-Cas to remove the gene from the genome of a species without stripes,&quot; Kratochwil explains, &quot;then even a &quot;stripeless&quot; fish will suddenly develop stripes, as we showed with a CRISPR-Cas mutant fish. This proves that the stripes gene is the decisive genetic factor.&quot;</p>
<p>The latest findings on this genetic mechanism, the activation or deactivation of stripes by the &quot;stripes gene,&quot; were published in the current issue of Science magazine. Interestingly, the cichlid's agrp2 gene is a copy of the agouti gene in mammals, which is responsible for the different coat colours of cats, dogs, horses and striped baby birds. &quot;The world of animals might be much less colourful without the agouti gene family,&quot; reflects Claudius Kratochwil. The mechanism of the &quot;stripes gene&quot; in cichlids clearly makes repeated evolution possible within the briefest of times, relatively speaking. If characteristics are lost during evolution, usually this loss is forever, as the Belgian palaeontologist Louis Dollo already realized exactly 125 years ago, and wrote up his conclusions in &quot;Dolls Law&quot; in 1893. The special aspect of the stripes gene agrp2 is that it makes repeated evolution of a characteristic possible in a simple way. If a cichlid loses its stripes that does not mean they will never return or vice versa. These molecular-biological studies also show that palaeontological rules and evolutionary rules have to be questioned once again.</p>
</li>
</ul>
</li>
</ul>
<h2>08992</h2>
<ul>
<li>
<p><em>evolutionary consequences of social isolation</em><br />
bailey &amp; moore 2018<br />
<a href="http://doi.org/10.1016/j.tree.2018.05.008">doi.org/10.1016/j.tree.2018.05.008</a></p>
<ul>
<li>The potential to experience social isolation is near-ubiquitous in animals, and research on its effects tends to focus on phenotypic consequences over an individual’s lifetime.<br />
A common, but not universal, finding is that social isolation negatively affects fitness measures such as health indicators, social competence, and competitiveness.<br />
Recent work suggests that social isolation can alter social selection and the genetic variance upon which selection acts, thereby altering evolutionary trajectories.<br />
Future research to evaluate evolutionary consequences of social isolation would benefit from (i) treating it as an experimental condition rather than a control, (ii) using quantitative indices of social isolation to enable standardised quantitative treatment, and (iii) testing evolutionary hypotheses using quantitative genetics, experimental evolution, and sociogenomics.<br />
Social isolation has profound impacts. Most animal research focuses on negative phenotypic consequences of social isolation within individual lifetimes. Less is known about how it affects genetics, selection, and evolution over longer timescales, though ample indirect evidence suggests that it might. We advocate that evolutionary consequences of social isolation be tested more directly. We suggest that the ‘index of social isolation’, the mismatch between actual and optimal social interaction experienced by individuals within a population, may play a key role in releasing cryptic genetic variation, adaptation rates, diversification patterns, and ecosystem-level processes. Evolutionary dynamics arising from social isolation could have significant impacts in applied settings such as conservation, animal breeding, control of biological invasions, and evolutionary resilience to anthropogenic change.</li>
</ul>
</li>
</ul>
<h2>09092</h2>
<ul>
<li>controversial theories
<ul>
<li>
<p><em>randomness in evolution</em><br />
john tyler bonner 2013</p>
<ul>
<li>
<p><em>Sewall Wright</em><br />
stochastic evolutionary change “drift.” the genetic makeup of a population could change simply because of those random events</p>
</li>
<li>
<p><em>C. E. Finch and T.B.L. Kirkwood</em><br />
many events during development are random and leave their imprint on the resulting adult. C. H. Waddington called this “developmental noise.”</p>
</li>
<li>
<p><em>Michael Lynch</em><br />
random molecular changes could give a directional push over time that does not involve natural selection</p>
</li>
<li>
<p><em>john tyler bonner</em><br />
effect of randomness differs for organisms of different sizes</p>
</li>
<li>
<p><em>Darwin; On the Origin of Species</em><br />
“Why have not the more highly developed forms everywhere supplanted and exterminated the lower? Lamarck, who believed in an innate and inevitable tendency towards perfection in all organic beings, seems to have felt this difficulty so strongly, that he was led to suppose that new and simple forms were continually being produced by spontaneous generation. I need hardly say that Science in her present state does not countenance the belief that living creatures are now ever produced from inorganic matter… If it were no advantage, these forms would be left by natural selection unimproved or but little improved; and might remain for indefinite ages in their present little advanced condition. And geology tells us that some of the lowest forms, as the infusoria and rhizopods, have remained for an enormous period in nearly their present state.”</p>
</li>
<li>
<p>“the idea that biological diversity could be explained by something other than natural selection approaches heresy”</p>
</li>
<li>
<p>“The key is the number of developmental steps: many, and randomness is suppressed; few, and the effect of randomness can come to the surface and bloom.”</p>
</li>
<li>
<p>evolved organisms as non–renewable resources, representing millions of years of evolution</p>
</li>
<li>
<p>“The neutral morphology hypothesis is a way of accounting for a number of features found among small organisms, in particular the incredible diversity in protists found within a single environment. This diversity among microbial phenotypes is a phenomenon at the fringe of Darwinian evolutionary biology that might be better understood from an unconventional perspective. Neutral morphologies are a possibility that cannot be ignored—it may provide valuable insights into the world of micro eukaryotes and their particular evolutionary processes.”</p>
</li>
<li>
<p>“Large organisms are unlikely to have an overall neutral morphology like small ones, and the reason is to be found in their elaborate development. The greater the size, the more developmental steps. The voyage from a single cell, a fertilized egg, to a large, mature organism with millions of cells is a process that cannot be chaotic, but must be controlled if it is to achieve a consistent ultimate shape from generation to generation. There can be no significant deviation from those set steps to get from one generation to the next.”</p>
</li>
<li>
<p>“multicellularity was invented a number of times. The great difficulty is that all those well-established origins occurred eons ago, and the only way we can reconstruct them is by hypothesis, by bald guesswork. (Of course there could be new inventions of multicellularity occurring today, but how could we ever find them?) Yet such thought experiments can give us some general idea of how complexity might have evolved and therefore some understanding of how it continues to do so.’</p>
</li>
<li>
<p>“The purpose of this essay has been to give a balanced view of evolution by showing how big a role randomness plays. I think randomness is necessary to counteract the tremendous power of natural selection that to some degree blinds our vision. Selection is the supreme mechanism that brings order out of chaos, and for that reason it is quite rightly foremost in our minds in all matters concerning evolution. It is for this reason that randomness is often ignored and sometimes rejected because of our natural selection mind-set. But, as I have pointed out, there could be no natural selection without randomness, for it is the foundation upon which natural selection is built. It provides the fodder for selection; without it everything would be the same and no change, no evolution would be possible. Randomness keeps appearing in different ways and at different times during the course of evolution. For many biologists, randomness is the skeleton that can be kept in the closet, and what I have done is bring it out.”</p>
</li>
</ul>
</li>
<li>
<p>evolution through loss is different from evolution through gain. currently evolution through gain.</p>
</li>
<li>
<p><em>aliens in our midst; the ctenophore’s brain suggests that, if evolution began again, intelligence would re-emerge because nature repeats itself</em><br />
douglas fox 2017<br />
<a href="https://aeon.co/essays/what-the-ctenophore-says-about-the-evolution-of-intelligence">aeon.co/essays/what-the-ctenophore-says-about-the-evolution-of-intelligence</a></p>
<ul>
<li>different nervous systems developed in ctenophores compared to other animals</li>
</ul>
</li>
<li>
<p>no genetic basis for race</p>
<ul>
<li>
<p><em>superior: the return of race science</em><br />
saini angela 2019</p>
</li>
<li>
<p>not recommended</p>
<ul>
<li>
<p>nicholas wade 2014 a troublesome inheritance: genes, race, and human history</p>
<ul>
<li>holds that east asian and jewish success comes from evolutionary pressure selecting for higher intelligence, e.g. emergence of Pharisees leading to conversion away from judaism of those who refused to become literate, leaving a self-selected founder population</li>
<li>holds that reason arose from religious debate, leading to reason and conscience and thence a legal system and thence science</li>
<li>holds that genetic differences between races leads to difference in industry, rather than difference in privilege</li>
</ul>
</li>
<li>
<p><em>the 10,000 year explosion; how civilization accelerated human evolution</em><br />
gregory cochran, henry harpending 2009<br />
978-0-465-00221-4</p>
<ul>
<li>contraversial, especially regarding IQ in Ashkenazim, saying that medieval European prejudice and restrictions to societal livelihood along with in–group marriage created evolutionary pressure for higher intelligence among Ashkenazim but not other Jewish populations</li>
</ul>
</li>
</ul>
</li>
</ul>
</li>
<li>
<p>cultural basis for healthy habits</p>
<ul>
<li>john durant 2013 the paleo manifesto: ancient wisdom for modern health
<ul>
<li>holds that better survival rate of jewish populations during plagues due to healthy practices encoded in cultural practices, e.g. hand washing valued compared to surrounding populations of christians who devalued cleanliness</li>
</ul>
</li>
</ul>
</li>
<li>
<p>geographic determinism</p>
<ul>
<li>jared diamond guns germs and steel
<ul>
<li>holds that &quot;history followed different courses for different peoples because of differences among peoples’ environments, not because of biological differences among peoples themselves.&quot;</li>
</ul>
</li>
</ul>
</li>
<li>
<p>better institutions</p>
<ul>
<li>niall ferguson
<ul>
<li>holds that open society led to innovation, led to rise of west
<ul>
<li>in which case west now in decline</li>
<li>where does open society come from?</li>
</ul>
</li>
</ul>
</li>
</ul>
</li>
<li>
<p><em>dna dispose, but subjects decide. learning and the extended synthesis</em><br />
markus lindholm 2015<br />
<a href="https://doi.org/10.1007/s12304-015-9242-3">doi.org/10.1007/s12304-015-9242-3</a></p>
<ul>
<li>Adaptation by means of natural selection depends on the ability of populations to maintain variation in heritable traits. According to the Modern Synthesis this variation is sustained by mutations and genetic drift. Epigenetics, evodevo, niche construction and cultural factors have more recently been shown to contribute to heritable variation, however, leading an increasing number of biologists to call for an extended view of speciation and evolution. An additional common feature across the animal kingdom is learning, defined as the ability to change behavior according to novel experiences or skills. Learning constitutes an additional source for phenotypic variation, and change in behavior may induce long lasting shifts in fitness, and hence favor evolutionary novelties. Based on published studies, I demonstrate how learning about food, mate choice and habitats has contributed substantially to speciation in the canonical story of Darwin’s finches on the Galapagos Islands. Learning cannot be reduced to genetics, because it demands decisions, which requires a subject. Evolutionary novelties may hence emerge both from shifts in allelic frequencies and from shifts in learned, subject driven behavior. The existence of two principally different sources of variation also prevents the Modern Synthesis from self-referring explanations.</li>
</ul>
</li>
<li>
<p>living space, not competition, is evolutionary driver?</p>
<ul>
<li>
<p>this is particularly important because &quot;evolutionary competition&quot; &quot;nature red in tooth and claw&quot; is a prime assumption of our society.</p>
</li>
<li>
<p>links between global taxonomic diversity, ecological diversity and the expansion of vertebrates on land<br />
sahney et al 2010<br />
<a href="https://doi.org/10.1098/rsbl.2009.1024">doi.org/10.1098/rsbl.2009.1024</a></p>
<ul>
<li>sahney<br />
<a href="http://bristol.academia.edu/SSahney/Papers">bristol.academia.edu/SSahney/Papers</a></li>
</ul>
</li>
</ul>
</li>
<li>
<p><em>evolutionary rescue can prevent extinction following environmental change</em><br />
graham bell and andrew gonzalez 2009<br />
<a href="http://doi.org/10.1111/j.1461-0248.2009.01350.x">doi.org/10.1111/j.1461-0248.2009.01350.x</a></p>
<ul>
<li>
<p>basically, presence of a few adapted individuals in a large population allows numbers to recover in laboratory i.e. stable conditions. changing conditions not investigated here, only single step change.</p>
</li>
<li>
<p>pdf<br />
Link: <a href="http://gonzalezlab.weebly.com/uploads/5/7/4/1/5741805/bell_and_gonzalez_09__el.pdf">gonzalezlab.weebly.com/uploads/5/7/4/1/5741805/bell_and_gonzalez_09__el.pdf</a></p>
</li>
<li>
<p>The ubiquity of global change and its impacts on biodiversity poses a clear and urgent challenge for evolutionary biologists. In many cases, environmental change is so widespread and rapid that individuals can neither accommodate to them physiologically nor migrate to a more favourable site. Extinction will ensue unless the population adapts fast enough to counter the rate of decline. According to theory, whether populations can be rescued by evolution depends upon several crucial variables: population size, the supply of genetic variation, and the degree of maladaptation to the new environment. Using techniques in experimental evolution we tested the conditions for evolutionary rescue (ER). Hundreds of yeast populations were exposed to normally lethal concentrations of salt in conditions, where the frequency of rescue mutations was estimated and population size was manipulated. In a striking match with theory, we show that ER is possible, and that the recovery of the population may occur within 25 generations. We observed a clear threshold in population size for ER whereby the ancestral population size must be sufficiently large to counter stochastic extinction and contain resistant individuals. These results demonstrate that rapid evolution is an important component of the response of small populations to environmental change.</p>
</li>
</ul>
</li>
<li>
<p>interface theory of perception<br />
<a href="http://www.cogsci.uci.edu/%7Eddhoff/interface.pdf">cogsci.uci.edu/~ddhoff/interface.pdf</a></p>
<ul>
<li>fits into our metaphor for evolution although we disagree with the tone and many of the arguments used.</li>
<li>he argues we have interfaces to the world and wonders why we do.</li>
<li>I say because this helps us to prepare for reality by enabling us to dream and think and tell stories!</li>
<li>beliefs are stories, stories are spells, spells are maps to the territory of reality. by understanding our maps we better navigate the territory.</li>
<li>absolutes do exist but only a few are useful. even the rule to not obey rules all the time should be broken sometimes so who is to say there might not be a few rules which might always apply? (likelihood low but not zero).</li>
<li>the beliefs in the world now are not static, they perpetuate. they are not only spells but become daemons. this is possession. the human world is possessed by some very evil daemons which we must face and defeat.</li>
<li>if the start of spells is the fall from paradise then the transcendence of spells is the return.</li>
<li>satisficing, what is good enough.</li>
<li>the author of the interface theory forgets that the theory of evolution is also an interface, according to his theory.</li>
<li>again his example of evolutionary game theory lacks the essential component of change which would render the theory useful. there may indeed be times when veracity outcompetes simple, even if those times are few and far between they would be significant for the survival of the less common variant. again another assumption is that there is only one outcome. organisms are not one–outcome values. it may be coincidence that different outcomes due to other factors enable survival when one factor changes.</li>
<li>so again the problem is reliance on stability as a criterion in deciding evolutionary fitness. he criticises others for conflating veracity with fitness but falls into a similar trap by conflating stability with fitness.</li>
</ul>
</li>
<li>
<p>is our next evolutionary step to learn to control these massive ponderous structures that we have built, of industry, of law, of everything? we are still operating as a society on the individual level and have poor control over the direction of our society.</p>
</li>
<li>
<p><em>balancing evolution</em><br />
occurrence of harmful gene variants could be the price we pay for the genetic diversity that is otherwise highly beneficial to our survival</p>
<ul>
<li>
<p><em>excess of deleterious mutations around hla genes reveals evolutionary cost of balancing selection</em><br />
tobias l. lenz et al. 2016<br />
<a href="http://doi.org/10.1093/molbev/msw127">doi.org/10.1093/molbev/msw127</a></p>
<ul>
<li>Deleterious mutations are expected to evolve under negative selection and are usually purged from the population. However, deleterious alleles segregate in the human population and some disease-associated variants are maintained at considerable frequencies. Here, we test the hypothesis that balancing selection may counteract purifying selection in neighboring regions and thus maintain deleterious variants at higher frequency than expected from their detrimental fitness effect. We first show in realistic simulations that balancing selection reduces the density of polymorphic sites surrounding a locus under balancing selection, but at the same time markedly increases the population frequency of the remaining variants, including even substantially deleterious alleles. To test the predictions of our simulations empirically, we then use whole-exome sequencing data from 6,500 human individuals and focus on the most established example for balancing selection in the human genome, the major histocompatibility complex (MHC). Our analysis shows an elevated frequency of putatively deleterious coding variants in nonhuman leukocyte antigen (non-HLA) genes localized in the MHC region. The mean frequency of these variants declined with physical distance from the classical HLA genes, indicating dependency on genetic linkage. These results reveal an indirect cost of the genetic diversity maintained by balancing selection, which has hitherto been perceived as mostly advantageous, and have implications both for the evolution of recombination and also for the epidemiology of various MHC-associated diseases.</li>
</ul>
</li>
</ul>
</li>
<li>
<p><em>social context, but not individual personality, alters immigrant viability in a spider with mixed social structure</em><br />
spencer j. ingley et al. 2016<br />
<a href="http://doi.org/10.1016/j.anbehav.2016.08.009">doi.org/10.1016/j.anbehav.2016.08.009</a></p>
<ul>
<li>Local adaptation in social animals can occur in individual- or group-level traits.<br />
•<br />
Misaligned traits at either level can reduce migration between environments.<br />
•<br />
We tested how social context and individual personality alter immigrant viability.<br />
•<br />
Social context, not personality, affected survival in foreign environments.<br />
•<br />
Spiders transplanted with their natal groups suffered the highest mortality rates.<br />
Immigrant viability is a major determinant of the realized rate of gene flow across populations. For social organisms, the social context in which immigrants disperse across contrasting environments may have important implications for their viability post dispersal. Here, we use social spiders whose individual personalities as well as group personality compositions vary across sites to test whether the strength of selection against immigrants (i.e. mortality rates) differs depending on whether spiders are transplanted (1) as individuals and remain alone, (2) join pre-existing colonies at their new non-native environment, or (3) move with their native group. We also tested for an effect of individual personality on survival. We found that social context, and not individual personality, affects individual survival in foreign environments with contrasting resource levels. Individuals that were transplanted with their native groups suffered higher mortality rates compared to individuals transplanted as singletons, regardless of whether or not they were assimilated into native colonies. Moving as individuals could thus provide an avenue for ongoing gene flow among populations from different resource environments. We found no depressed performance of control colonies that were transplanted across sites with resource levels similar to each colony's site of origin. These results are at odds with the intuition that dispersing as a group should generally enhance the viability of immigrants, at least in social species. We propose that these results could be explained by a mismatch in the ideal group compositions (personality compositions) favoured in different environments, despite a lack of selection on individual personality traits. These results provide a first glimpse into the relative roles of individual personality and social context in mediating gene flow among populations from divergent environments.</li>
</ul>
</li>
<li>
<p><em>demographic and reproductive plasticity across the depth distribution of a coral reef fish</em><br />
esther d. goldstein et al. 2016<br />
<a href="http://doi.org/10.1038/srep34077">doi.org/10.1038/srep34077</a></p>
<ul>
<li>As humans expand into natural environments, populations of wild organisms may become relegated to marginal habitats at the boundaries of their distributions. In the ocean, mesophotic coral ecosystems (30–150 m) at the depth limit of photosynthetic reefs are hypothesized to act as refuges that are buffered from anthropogenic and natural disturbances, yet the viability and persistence of subpopulations in these peripheral habitats remains poorly understood. To assess the potential for mesophotic reefs to support robust coral reef fish populations, we compared population density and structure, growth, size, and reproductive output of the bicolor damselfish (Stegastes partitus) from shallow (&lt;10 m), deep shelf (20–30 m), and mesophotic reefs (60–70 m) across the Florida Platform. Population densities decreased and size and age distributions shifted toward older and larger individuals in deeper habitats. Otolith-derived ages indicated that S. partitus found on mesophotic reefs reach larger asymptotic sizes and have longer lifespans than fish in shallower habitats. Based on measurements of oocyte area and batch fecundity, mesophotic fish also have higher reproductive investment. These demographic patterns indicate that mesophotic fish populations composed of large, fecund individuals produce high condition larvae and rely on longevity of individuals for population persistence and viability.</li>
</ul>
</li>
<li>
<p><em>archean coastal-plain paleosols and life on land</em><br />
gregory j. retallack et al. 2016<br />
<a href="http://doi.org/10.1016/j.gr.2016.08.003">doi.org/10.1016/j.gr.2016.08.003</a></p>
<ul>
<li>
<p>There are coastal-plain paleosols in 3.0 Ga Farrel Quartzite, Western Australia.</p>
<p>Paleosols have organic surface (A) and sulfate-rich subsurface (By) horizon.</p>
<p>Comparable profiles are known from deserts of Chile, Antarctica, and Mars.</p>
<p>Microfossils in paleosols include actinobacteria, sulfur bacteria, methanogens.</p>
<p>Coastal-plain paleosols in the 3.0 Ga Farrel Quartzite of Western Australia have organic surface (A horizon) and sulfate-rich subsurface (By) horizons, like soils of the Atacama Desert of Chile, Dry Valleys of Antarctica, and 3.7 Ga paleosols of Mars. Farrel Quartzite paleosols include previously described microfossils, permineralized by silica in a way comparable with the Devonian Rhynie Chert, a well known permineralized Histosol. Five microfossil morphotypes in the Farrel Quartzite include a variety of spheroidal cells (<em>Archaeosphaeroides</em>) as well as distinctive large spindles (new genus provisionally assigned to cf. <em>Eopoikilofusa</em>). Previously published cell-specific carbon isotopic analyses of the Farrel Quartzite microfossils, and unusually abundant sulfate considering a likely anoxic atmosphere, allow interpretation of these morphotypes as a terrestrial community of actinobacteria, purple sulfur bacteria, and methanogenic Archaea.</p>
</li>
</ul>
</li>
<li>
<p><em>the 'out of africa' hypothesis, human genetic diversity, and comparative economic development</em><br />
quamrul ashraf, oded galor 2013<br />
<a href="http://doi.org/10.1257/aer.103.1.1">doi.org/10.1257/aer.103.1.1</a></p>
<ul>
<li>This research advances and empirically establishes the hypothesis that, in the course of the prehistoric exodus of Homo sapiens out of Africa, variation in migratory distance to various settlements across the globe affected genetic diversity and has had a persistent hump-shaped effect on comparative economic development, reflecting the trade-off between the beneficial and the detrimental effects of diversity on productivity. While the low diversity of Native American populations and the high diversity of African populations have been detrimental for the development of these regions, the intermediate levels of diversity associated with European and Asian populations have been conducive for development.</li>
<li>“greater diversity are also characterized, on average, by a lower prevalence of interpersonal trust and a higher intensity of scientific knowledge creation.”</li>
</ul>
</li>
</ul>
</li>
</ul>
<h2>09192</h2>
<ul>
<li>
<p><em>human genealogy reveals a selective advantage to moderate fecundity</em><br />
oded galor, marc klemp 2019<br />
<a href="http://doi.org/10.1038/s41559-019-0846-x">doi.org/10.1038/s41559-019-0846-x</a></p>
<ul>
<li>
<p>genealogical records from Quebec's Saint Lawrence Valley dating from 1608 to 1800. Focusing on changes in families' fecundity, or predisposition toward fertility, they found that in those centuries, those who were able to conceive a child shortly after marriage -- a measure of fecundity -- had more surviving children.</p>
<p>However, the study found, those who conceived months after marriage -- a measure of more moderate fecundity -- had fewer children but a larger number of surviving descendants in future generations, giving them the evolutionary upper hand. The researchers also noted that the population they studied became increasingly less predisposed toward high fertility over the course of those two centuries.</p>
<p>Galor says the study results, published on Monday, April 1, in Nature Ecology &amp; Evolution, lend credence to what he and a colleague had surmised in a highly influential 2002 paper -- that during the pre-industrial era, the natural selection of those who were genetically predisposed toward having fewer children was instrumental in spurring industrialization and sustained economic growth.</p>
<p>&quot;The data suggest that over time, nature selected individuals who had a predisposition to invest in their children,&quot; Galor said. &quot;This contributed to the transition from an epoch of stagnation to an era of sustained economic growth.&quot;</p>
<p>Before the Industrial Revolution began in North America in the mid-1700s, Galor explained, humanity lived in what he calls the Malthusian epoch. For thousands of years, humanity had a predisposition toward high fertility. Galor and Kemp's study shows that the pattern began to change in the pre-industrial era, when those with more moderate levels of fecundity began to gain an evolutionary advantage. By the dawn of the Industrial Revolution, their advantage had grown so large that the high-fecundity population became the minority, while those with moderate fecundity started to dominate the population.</p>
<p>Galor argues that this change created ideal conditions for economic growth during the Industrial Revolution. As evolution began to favor families who were less fertile and thus had fewer children, those families had more resources to devote to each child. Children who came from these families became more educated -- an important trait, he says, in an era that demanded greater cognitive ability and creativity for technological advances. The population gradually became more educated, creating a &quot;positive feedback loop&quot; between education and technology and generating sustained economic growth.</p>
<p>&quot;The fundamental building block in our hypothesis, that natural selection was critical for the emergence of economic growth, is now supported by the evidence,&quot; Galor said of the 2002 paper. &quot;We show that although higher fecundity maximized the number of surviving children someone would have after one generation, moderate fecundity -- and therefore greater predisposition toward child quality -- generated higher reproductive success in the long run and was selected by nature in the pre-industrial period.&quot;</p>
<p>To reach their conclusion, the researchers chose to focus on an extensive genealogical record of nearly half a million individuals in a particular area of Quebec, where nearly every citizen's birth, marriage and death was recorded in Catholic parish registers between 1608 and 1800. Given the time period and the region's religious uniformity, the researchers could safely assume that for most, marriage signaled a deliberate attempt to conceive children. They ensured their findings weren't driven by stray exceptions in the dataset -- for example, those who married much later in life or whose genetics predisposed them to infertility.</p>
<p>The results of the analysis -- that those who successfully conceived a few months after marriage, rather than immediately afterward, had more surviving descendants in the long term -- mirror preliminary results from another analysis Galor and Klemp conducted using records in Britain between 1541 and 1871. Galor says this suggests the phenomenon may have extended beyond Quebec and Britain, as he posited in 2002.</p>
<p>&quot;My hope,&quot; he said, &quot;is that this study will spur further interest in exploring the role of evolutionary processes in economic development.&quot;</p>
</li>
<li>
<p><em>abstract</em> Life-history theory suggests that the level of fecundity of each organism reflects the effect of the trade-off between the quantity and quality of offspring on its long-run reproductive success. The present research provides evidence that moderate fecundity was conducive to long-run reproductive success in humans. Using a reconstructed genealogy for nearly half a million individuals in Quebec during the 1608–1800 period, the study establishes that, while high fecundity was associated with a larger number of children, perhaps paradoxically, moderate fecundity maximized the number of descendants after several generations. Moreover, the analysis further suggests that evolutionary forces decreased the level of fecundity in the population over this period, consistent with an additional finding that the level of fecundity that maximized long-run reproductive success was above the population mean. The research identifies several mechanisms that contributed to the importance of moderate fecundity for long-run reproductive success. It suggests that, while individuals with lower fecundity had fewer children, the observed hump-shaped effect of fecundity on long-run reproductive success reflects the beneficial effects of lower fecundity on various measures of child quality, such as marriageability and literacy, and thus on the reproductive success of each child.</p>
</li>
</ul>
</li>
</ul>
<h2>09292</h2>
<ul>
<li>
<p><em>on the origin of modern humans: asian perspectives</em><br />
christopher j. bae et al. 2017<br />
<a href="http://doi.org/10.1126/science.aai9067">doi.org/10.1126/science.aai9067</a></p>
<ul>
<li>The traditional “out of Africa” model, which posits a dispersal of modern <em>Homo sapiens</em> across Eurasia as a single wave at ~60,000 years ago and the subsequent replacement of all indigenous populations, is in need of revision. Recent discoveries from archaeology, hominin paleontology, geochronology, genetics, and paleoenvironmental studies have contributed to a better understanding of the Late Pleistocene record in Asia. Important findings highlighted here include growing evidence for multiple dispersals predating 60,000 years ago in regions such as southern and eastern Asia. Modern humans moving into Asia met Neandertals, Denisovans, mid-Pleistocene <em>Homo</em>, and possibly <em>H. floresiensis</em>, with some degree of interbreeding occurring. These early human dispersals, which left at least some genetic traces in modern populations, indicate that later replacements were not wholesale.</li>
</ul>
</li>
<li>
<p><em>did our species evolve in subdivided populations across africa, and why does it matter?</em><br />
eleanor m.l. scerri et al. 2018<br />
<a href="http://doi.org/10.1016/j.tree.2018.05.005">doi.org/10.1016/j.tree.2018.05.005</a></p>
<ul>
<li>The view that <em>Homo sapiens</em> evolved from a single region/population within Africa has been given primacy in studies of human evolution.<br />
However, developments across multiple fields show that relevant data are no longer consistent with this view.<br />
We argue instead that <em>Homo sapiens</em> evolved within a set of interlinked groups living across Africa, whose connectivity changed through time.<br />
Genetic models therefore need to incorporate a more complex view of ancient migration and divergence in Africa.<br />
We summarize this new framework emphasizing population structure, outline how this changes our understanding of human evolution, and identify new research directions.<br />
We challenge the view that our species, <em>Homo sapiens</em>, evolved within a single population and/or region of Africa. The chronology and physical diversity of Pleistocene human fossils suggest that morphologically varied populations pertaining to the <em>H. sapiens</em> clade lived throughout Africa. Similarly, the African archaeological record demonstrates the polycentric origin and persistence of regionally distinct Pleistocene material culture in a variety of paleoecological settings. Genetic studies also indicate that present-day population structure within Africa extends to deep times, paralleling a paleoenvironmental record of shifting and fractured habitable zones. We argue that these fields support an emerging view of a highly structured African prehistory that should be considered in human evolutionary inferences, prompting new interpretations, questions, and interdisciplinary research directions.</li>
</ul>
</li>
<li>
<p><em>the genomic history of southeastern europe</em><br />
iain mathieson et al. 2018<br />
<a href="http://doi.org/10.1038/nature25778">doi.org/10.1038/nature25778</a></p>
<ul>
<li>Farming was first introduced to Europe in the mid-seventh millennium BC, and was associated with migrants from Anatolia who settled in the southeast before spreading throughout Europe. Here, to understand the dynamics of this process, we analysed genome-wide ancient DNA data from 225 individuals who lived in southeastern Europe and surrounding regions between 12000 and 500 BC. We document a west–east cline of ancestry in indigenous hunter-gatherers and, in eastern Europe, the early stages in the formation of Bronze Age steppe ancestry. We show that the first farmers of northern and western Europe dispersed through southeastern Europe with limited hunter-gatherer admixture, but that some early groups in the southeast mixed extensively with hunter-gatherers without the sex-biased admixture that prevailed later in the north and west. We also show that southeastern Europe continued to be a nexus between east and west after the arrival of farmers, with intermittent genetic contact with steppe populations occurring up to 2,000 years earlier than the migrations from the steppe that ultimately replaced much of the population of northern Europe.</li>
</ul>
</li>
<li>
<p><em>the beaker phenomenon and the genomic transformation of northwest europe</em><br />
iñigo olalde et al. 2018<br />
<a href="http://doi.org/10.1038/nature25738">doi.org/10.1038/nature25738</a></p>
<ul>
<li>From around 2750 to 2500 BC, Bell Beaker pottery became widespread across western and central Europe, before it disappeared between 2200 and 1800 BC. The forces that propelled its expansion are a matter of long-standing debate, and there is support for both cultural diffusion and migration having a role in this process. Here we present genome-wide data from 400 Neolithic, Copper Age and Bronze Age Europeans, including 226 individuals associated with Beaker-complex artefacts. We detected limited genetic affinity between Beaker-complex-associated individuals from Iberia and central Europe, and thus exclude migration as an important mechanism of spread between these two regions. However, migration had a key role in the further dissemination of the Beaker complex. We document this phenomenon most clearly in Britain, where the spread of the Beaker complex introduced high levels of steppe-related ancestry and was associated with the replacement of approximately 90% of Britain’s gene pool within a few hundred years, continuing the east-to-west expansion that had brought steppe-related ancestry into central and northern Europe over the previous centuries.</li>
</ul>
</li>
<li>
<p><em>physiological and genetic adaptations to diving in sea nomads</em><br />
melissa a. ilardo et al. 2018<br />
<a href="https://doi.org/10.1016/j.cell.2018.03.054">doi.org/10.1016/j.cell.2018.03.054</a></p>
<ul>
<li>
<p>•The Bajau, or “Sea Nomads,” have engaged in breath-hold diving for thousands of years<br />
•Selection has increased Bajau spleen size, providing an oxygen reservoir for diving<br />
•We find evidence of additional diving-related phenotypes under selection<br />
•These findings have implications for hypoxia research, a pertinent medical issue</p>
<p>Understanding the physiology and genetics of human hypoxia tolerance has important medical implications, but this phenomenon has thus far only been investigated in high-altitude human populations. Another system, yet to be explored, is humans who engage in breath-hold diving. The indigenous Bajau people (“Sea Nomads”) of Southeast Asia live a subsistence lifestyle based on breath-hold diving and are renowned for their extraordinary breath-holding abilities. However, it is unknown whether this has a genetic basis. Using a comparative genomic study, we show that natural selection on genetic variants in the PDE10A gene have increased spleen size in the Bajau, providing them with a larger reservoir of oxygenated red blood cells. We also find evidence of strong selection specific to the Bajau on BDKRB2, a gene affecting the human diving reflex. Thus, the Bajau, and possibly other diving populations, provide a new opportunity to study human adaptation to hypoxia tolerance.</p>
</li>
</ul>
</li>
<li>
<p><em>environmental selection during the last ice age on the mother-to-infant transmission of vitamin d and fatty acids through breast milk</em><br />
leslea j. hlusko et al. 2018<br />
<a href="http://doi.org/10.1073/pnas.1711788115">doi.org/10.1073/pnas.1711788115</a></p>
<p>“human variation today reflects this dynamic process of ephemeral populations, rather than the traditional concept of geographic races with distinct differences between them”</p>
<ul>
<li>Because of the ubiquitous adaptability of our material culture, some human populations have occupied extreme environments that intensified selection on existing genomic variation. By 32,000 years ago, people were living in Arctic Beringia, and during the Last Glacial Maximum (LGM; 28,000–18,000 y ago), they likely persisted in the Beringian refugium. Such high latitudes provide only very low levels of UV radiation, and can thereby lead to dangerously low levels of biosynthesized vitamin D. The physiological effects of vitamin D deficiency range from reduced dietary absorption of calcium to a compromised immune system and modified adipose tissue function. The ectodysplasin A receptor (<em>EDAR</em>) gene has a range of pleiotropic effects, including sweat gland density, incisor shoveling, and mammary gland ductal branching. The frequency of the human-specific <em>EDAR V370A</em> allele appears to be uniquely elevated in North and East Asian and New World populations due to a bout of positive selection likely to have occurred <em>circa</em> 20,000 y ago. The dental pleiotropic effects of this allele suggest an even higher occurrence among indigenous people in the Western Hemisphere before European colonization. We hypothesize that selection on <em>EDAR V370A</em> occurred in the Beringian refugium because it increases mammary ductal branching, and thereby may amplify the transfer of critical nutrients in vitamin D-deficient conditions to infants via mothers’ milk. This hypothesized selective context for <em>EDAR V370A</em> was likely intertwined with selection on the fatty acid desaturase (<em>FADS</em>) gene cluster because it is known to modulate lipid profiles transmitted to milk from a vitamin D-rich diet high in omega-3 fatty acids.</li>
</ul>
</li>
<li>
<p><em>a common variation in edar is a genetic determinant of shovel-shaped incisors</em><br />
ryosuke kimura et al. 2009<br />
<a href="https://doi.org/10.1016/j.ajhg.2009.09.006">doi.org/10.1016/j.ajhg.2009.09.006</a></p>
<ul>
<li>Shovel shape of upper incisors is a common characteristic in Asian and Native American populations but is rare or absent in African and European populations. Like other common dental traits, genetic polymorphisms involved in the tooth shoveling have not yet been clarified. In <em>ectodysplasin A receptor</em> (<em>EDAR</em>), where dysfunctional mutations cause hypohidrotic ectodermal dysplasia, there is a nonsynonymous-derived variant, 1540C (rs3827760), that has a geographic distribution similar to that of the tooth shoveling. This allele has been recently reported to be associated with Asian-specific hair thickness. We aimed to clarify whether <em>EDAR</em> 1540C is also associated with dental morphology. For this purpose, we measured crown diameters and tooth-shoveling grades and analyzed the correlations between the dental traits and <em>EDAR</em> genotypes in two Japanese populations, inhabitants around Tokyo and in Sakishima Islands. The number of <em>EDAR</em> 1540C alleles in an individual was strongly correlated with the tooth-shoveling grade (p = 7.7 × 10−10). The effect of the allele was additive and explained 18.9% of the total variance in the shoveling grade, which corresponds to about one-fourth of the heritability of the trait reported previously. For data reduction of individual-level metric data, we applied a principal-component analysis, which yielded PC1-4, corresponding to four patterns of tooth size; this result implies that multiple factors are involved in dental morphology. The 1540C allele also significantly affected PC1 (p = 4.9 × 10−3), which denotes overall tooth size, and PC2 (p = 2.6 × 10−3), which denotes the ratio of mesiodistal diameter to buccolingual diameter.</li>
</ul>
</li>
<li>
<p><em>hominin occupation of the chinese loess plateau since about 2.1 million years ago</em><br />
zhaoyu zhu et al. 2018<br />
<a href="http://doi.org/10.1038/s41586-018-0299-4">doi.org/10.1038/s41586-018-0299-4</a></p>
<ul>
<li>Considerable attention has been paid to dating the earliest appearance of hominins outside Africa. The earliest skeletal and artefactual evidence for the genus Homo in Asia currently comes from Dmanisi, Georgia, and is dated to approximately 1.77–1.85 million years ago (Ma)<a href="https://www.nature.com/articles/s41586-018-0299-4#ref-CR1">66</a>. Two incisors that may belong to Homo erectus come from Yuanmou, south China, and are dated to 1.7 Ma<a href="https://www.nature.com/articles/s41586-018-0299-4#ref-CR2">67</a>; the next-oldest evidence is an H. erectus cranium from Lantian (Gongwangling)—which has recently been dated to 1.63 Ma<a href="https://www.nature.com/articles/s41586-018-0299-4#ref-CR3">68</a>—and the earliest hominin fossils from the Sangiran dome in Java, which are dated to about 1.5–1.6 Ma<a href="https://www.nature.com/articles/s41586-018-0299-4#ref-CR4">69</a>. Artefacts from Majuangou III<a href="https://www.nature.com/articles/s41586-018-0299-4#ref-CR5">70</a> and Shangshazui<a href="https://www.nature.com/articles/s41586-018-0299-4#ref-CR6">71</a> in the Nihewan basin, north China, have also been dated to 1.6–1.7 Ma. Here we report an Early Pleistocene and largely continuous artefact sequence from Shangchen, which is a newly discovered Palaeolithic locality of the southern Chinese Loess Plateau, near Gongwangling in Lantian county. The site contains 17 artefact layers that extend from palaeosol S15—dated to approximately 1.26 Ma—to loess L28, which we date to about 2.12 Ma. This discovery implies that hominins left Africa earlier than indicated by the evidence from Dmanisi.</li>
</ul>
</li>
</ul>
<h2>09392</h2>
<ul>
<li>
<p><em>type iv crispr–cas systems are highly diverse and involved in competition between plasmids</em><br />
shiraz a shah et al. 2020<br />
<a href="http://doi.org/10.1093/nar/gkz1197">doi.org/10.1093/nar/gkz1197</a></p>
<ul>
<li>
<p>“Until recently, CRISPR-Cas was believed to be a defense system used by bacteria to protect themselves against invading parasites such as viruses, much like our very own immune system protects us. However, it appears that CRISPR is a tool that can be used for different purposes by diverse biological entities,” according to 28-year-old Rafael Pinilla-Redondo, a PhD at UCPH’s Department of Biology who led the research.<br />
One of these biological entities are plasmids — small DNA molecules that often behave like parasites and, like viruses, require a host bacterium to survive.<br />
“Here we found evidence that certain plasmids use type IV CRISPR-Cas systems to fight other plasmids competing over the same bacterial host. This is remarkable because, in doing so, plasmids have managed to turn the system around. Instead of protecting bacteria from their parasites, CRISPR is exploited to perform another task,” says Pinilla-Redondo, adding:<br />
“This is similar to how some birds compete for the best nesting site in a tree, or how hermit crabs fight for ownership of a shell.”<br />
<strong>”A humbling realization”</strong><br />
The discovery challenges the notion that CRISPR-Cas systems have only one purpose in nature, that is, acting as immune systems in bacteria. According to Rafael Pinilla-Redondo, the discovery gives some additional perspective:<br />
“We humans have only recently begun to exploit nature’s CRISPR-Cas systems, but as it turns out, we are not the first. These ‘primitive parasites’ have been using them for millions of years, long before humans. It is quite a humbling realization”<br />
<strong>What can we use it for?</strong><br />
The researchers speculate that these systems could be used to combat one of the greatest threats to humanity: multi-drug resistant bacteria. Hundreds of thousands of people die from MDR bacteria every year.<br />
Bacteria become resistant to antibiotics by acquiring genes that make them resistant to antibiotic treatment. Very frequently, this occurs when plasmids transport antibiotic resistant genes from one bacterium to another.<br />
“As this system appears to have evolved to specifically attack plasmids, it is plausible that we could repurpose it to fight plasmids carrying antibiotic resistant genes</p>
</li>
<li>
<p><em>abstract</em> CRISPR–Cas systems provide prokaryotes with adaptive immune functions against viruses and other genetic parasites. In contrast to all other types of CRISPR–Cas systems, type IV has remained largely overlooked. Here, we describe a previously uncharted diversity of type IV gene cassettes, primarily encoded by plasmid-like elements from diverse prokaryotic taxa. Remarkably, via a comprehensive analysis of their CRISPR spacer content, these systems were found to exhibit a strong bias towards the targeting of other plasmids. Our data indicate that the functions of type IV systems have diverged from those of other host-related CRISPR–Cas immune systems to adopt a role in mediating conflicts between plasmids. Furthermore, we find evidence for cross-talk between certain type IV and type I CRISPR–Cas systems that co-exist intracellularly, thus providing a simple answer to the enigmatic absence of type IV adaptation modules. Collectively, our results lead to the expansion and reclassification of type IV systems and provide novel insights into the biological function and evolution of these elusive systems.</p>
</li>
</ul>
</li>
</ul>
<h2>09492</h2>
<h2>09592</h2>
<ul>
<li>
<p><em>genomic evidence of speciation reversal in ravens</em><br />
anna m. kearns et al. 2018<br />
<a href="http://doi.org/10.1038/s41467-018-03294-w">doi.org/10.1038/s41467-018-03294-w</a></p>
<ul>
<li>Many species, including humans, have emerged via complex reticulate processes involving hybridisation. Under certain circumstances, hybridisation can cause distinct lineages to collapse into a single lineage with an admixed mosaic genome. Most known cases of such ‘speciation reversal’ or ‘lineage fusion’ involve recently diverged lineages and anthropogenic perturbation. Here, we show that in western North America, Common Ravens (<em>Corvus corax</em>) have admixed mosaic genomes formed by the fusion of non-sister lineages (‘California’ and ‘Holarctic’) that diverged ~1.5 million years ago. Phylogenomic analyses and concordant patterns of geographic structuring in mtDNA, genome-wide SNPs and nuclear introns demonstrate long-term admixture and random interbreeding between the non-sister lineages. In contrast, our genomic data support reproductive isolation between Common Ravens and Chihuahuan Ravens (<em>C</em>. <em>cryptoleucus</em>) despite extensive geographic overlap and a sister relationship between Chihuahuan Ravens and the California lineage. These data suggest that the Common Raven genome was formed by secondary lineage fusion and most likely represents a case of ancient speciation reversal that occurred without anthropogenic causes.</li>
</ul>
</li>
</ul>
<h2>09692</h2>
<h2>09792</h2>
<ul>
<li>
<p><em>the kinetoplastid-infecting bodo saltans virus (bsv), a window into the most abundant giant viruses in the sea</em><br />
christoph m deeg et al. 2018<br />
<a href="http://doi.org/10.7554/eLife.33014">doi.org/10.7554/eLife.33014</a></p>
<ul>
<li>Giant viruses are ecologically important players in aquatic ecosystems that have challenged concepts of what constitutes a virus. Herein, we present the giant Bodo saltans virus (BsV), the first characterized representative of the most abundant group of giant viruses in ocean metagenomes, and the first isolate of a klosneuvirus, a subgroup of the <em>Mimiviridae</em> proposed from metagenomic data. BsV infects an ecologically important microzooplankton, the kinetoplastid <em>Bodo saltans</em>. Its 1.39 Mb genome encodes 1227 predicted ORFs, including a complex replication machinery. Yet, much of its translational apparatus has been lost, including all tRNAs. Essential genes are invaded by homing endonuclease-encoding self-splicing introns that may defend against competing viruses. Putative anti-host factors show extensive gene duplication via a genomic accordion indicating an ongoing evolutionary arms race and highlighting the rapid evolution and genomic plasticity that has led to genome gigantism and the enigma that is giant viruses.</li>
</ul>
</li>
</ul>
<h2>09892</h2>
<ul>
<li>to read next
<ul>
<li>
<p><em>lamarck’s revenge: how epigenetics is revolutionizing our understanding of evolution’s past and present</em><br />
peter ward 2018</p>
</li>
<li>
<p><em>purpose and desire: what makes something alive and why modern darwinism has failed to explain it</em><br />
j. scott turner 2017</p>
</li>
<li>
<p><em>darwin comes to town</em><br />
menno schilthuizen 2018 toreadnext</p>
</li>
<li>
<p><em>evolution of the learning brain or how you got to be so smart</em><br />
paul howard-jones 2018toreadnext</p>
</li>
<li>
<p><em>who we are and how we got here: ancient dna and the new science of the human past</em><br />
david reich 2018 to read next</p>
</li>
<li>
<p><em>the ends of the world: earth’s past mass extinctions</em><br />
peter brannen 2018 to read next</p>
</li>
<li>
<p><em>eco-evolutionary dynamics</em><br />
andrew p. hendry 2017 9780691145433</p>
</li>
<li>
<p><em>convergent evolution: limited forms most beautiful</em><br />
george mcghee 2011</p>
</li>
<li>
<p><em>the crucible of creation: the burgess shale and the rise of animals</em><br />
simon morris 1998</p>
</li>
<li>
<p><em>the structure of evolutionary theory</em><br />
stephen gould</p>
</li>
<li>
<p><em>debating darwin</em><br />
robert richards, michael ruse 2016</p>
</li>
<li>
<p><em>evolution driven by organismal behavior: a unifying view of life, function, form, mismatches and trends</em><br />
rui diogo 2017</p>
</li>
<li>
<p><em>a taste for the beautiful: the evolution of attraction</em><br />
michael ryan 2018</p>
</li>
<li>
<p><em>an essay on the principle of population</em><br />
thomas malthus</p>
</li>
<li>
<p><em>darwin’s dangerous idea: evolution and the meanings of life</em><br />
daniel dennett</p>
</li>
<li>
<p><em>species; the evolution of the idea, second edition</em><br />
john wilkins 2018</p>
</li>
<li>
<p><em>a brief history of everyone who ever lived: the human story retold through our genes</em><br />
adam rutherford 2017</p>
</li>
<li>
<p><em>darwin’s unfinished symphony: how culture made the human mind</em><br />
kevin laland 2017</p>
</li>
<li>
<p><em>evolutionaries; unlocking the spiritual and cultural potential of science’s greatest idea</em><br />
carter phipps 2012</p>
</li>
<li>
<p><em>the dominant animal: human evolution and the environment</em><br />
paul ehrlich &amp; anne ehrlich 2012</p>
</li>
<li>
<p><em>symbiotic planet: a new look at evolution</em><br />
lynn margulis 1999</p>
</li>
<li>
<p><em>how many friends does one person need?: dunbar’s number and other evolutionary quirks</em><br />
robin dunbar 2010</p>
</li>
<li>
<p><em>the next species: the future of evolution in the aftermath of man</em><br />
michael tennesen 2015</p>
</li>
<li>
<p><em>the vital question: energy, evolution, and the origins of complex life</em><br />
nick lane 2016</p>
</li>
<li>
<p><em>the evolution of beauty: how darwin’s forgotten theory of mate choice shapes the animal world-and us</em><br />
richard o. prum 2017</p>
</li>
<li>
<p><em>the equations of life: how physics shapes evolution</em><br />
charles cockell 2018</p>
</li>
<li>
<p><em>the tangled tree: a radical new history of life</em><br />
david quammen 2018</p>
</li>
<li>
<p><em>good reasons for bad feelings: insights from the frontier of evolutionary psychiatry</em><br />
randolph m. nesse 2019</p>
</li>
<li>
<p><em>the demon in the machine: how hidden webs of information are finally solving the mystery of life</em><br />
paul davies 2019</p>
</li>
<li>
<p><em>innate: how the wiring of our brains shapes who we are</em><br />
kevin j. mitchell 2019</p>
</li>
</ul>
</li>
</ul>
<h2>09992</h2>
<ul>
<li>
<p>fiction not yet read</p>
<ul>
<li>
<p><em>darwin’s radio</em><br />
greg bear</p>
<ul>
<li><em>darwin’s children</em><br />
greg bear</li>
</ul>
</li>
<li>
<p><em>the darwin variant</em><br />
kenneth c. johnson 2018</p>
</li>
<li>
<p><em>the second cure</em><br />
margaret morgan 2018</p>
</li>
</ul>
</li>
<li>
<p>not recommended</p>
<ul>
<li>
<p><em>runes of evolution</em><br />
simon conway morris 2015</p>
<ul>
<li>introduction, footnote 10 is interesting, regarding acquisition of chloroplasts from once free-living bacteria</li>
<li>Evolutionary inherency</li>
<li>published by templeton press</li>
</ul>
</li>
</ul>
</li>
</ul>
<h2>04992</h2>
<ul>
<li>
<p>yes let’s mess with genetics, what could possibly go wrong? (sarcasm)</p>
</li>
<li>
<p><em>activation of silent biosynthetic gene clusters using transcription factor decoys</em><br />
bin wang et al. 2018<br />
<a href="http://doi.org/10.1038/s41589-018-0187-0">doi.org/10.1038/s41589-018-0187-0</a></p>
<ul>
<li>
<p>&quot;There are so many undiscovered natural products lying unexpressed in genomes. We think of them as the dark matter of the cell,&quot; Zhao said. &quot;Anti-microbial resistance has become a global challenge, so clearly there's an urgent need for tools to aid the discovery of novel natural products. In this work, we found new compounds by activating silent gene clusters that have not been explored before.&quot;</p>
<p>The researchers previously demonstrated a technique to activate small silent gene clusters using CRISPR technology. However, large silent gene clusters have remained difficult to unmute. Those larger genes are of great interest to Zhao's group, since a number of them have sequences similar to regions that code for existing classes of antibiotics, such as tetracycline.</p>
<p>To unlock the large gene clusters of greatest interest, Zhao's group created clones of the DNA fragments they wanted to express and injected them into the bacteria in hopes of luring away the repressor molecules that were preventing gene expression. They called these clones transcription factor decoys.</p>
<p>&quot;Others have used this similar kind of decoys for therapeutic applications in mammalian cells, but we show here for the first time that it can be used for drug discovery by activating silent genes in bacteria,&quot; said Zhao, who is affiliated with the Carle Illinois College of Medicine, the Carl R. Woese Institute for Genomic Biology and the Center for Advanced Bioenergy and Bioproducts Innovation at Illinois.</p>
<p>To prove that the molecules they coded for were being expressed, researchers tested the decoy method first on two known gene clusters that synthesize natural products. Next, they created decoys for eight silent gene clusters that had been previously unexplored. In bacteria injected with the decoys, the targeted silent genes were expressed and the researchers harvested new products.</p>
<p>&quot;We saw that the method works well for these large clusters that are hard to target by other methods,&quot; Zhao said. &quot;It also has the advantage that it does not disturb the genome; it's just pulling away the repressors. Then the genes are expressed naturally from the native DNA.&quot;</p>
<p>In the search for drug candidates, each product needs to be isolated and then studied to determine what it does. Of the eight new molecules produced, the researchers purified and determined the structure of two molecules, and described one in detail in the study -- a novel type of oxazole, a class of molecules often used in drugs.</p>
<p>The researchers plan next to characterize the rest of the eight compounds and run various assays to find out whether they have any anti-microbial, anti-fungal, anti-cancer or other biological activities.</p>
<p>Zhao's group also plans to apply the decoy technique to explore more silent biosynthetic gene clusters of interest in Streptomyces and in other bacteria and fungi to find more undiscovered natural products. Other research groups are welcome to use the technique for gene clusters they are exploring, Zhao said.</p>
<p>&quot;The principle is the same, assuming that gene expression is repressed by transcription factors and we just need to release that expression by using decoy DNA fragments,&quot; Zhao said.</p>
</li>
<li>
<p><em>abstract</em> Here we report a transcription factor decoy strategy for targeted activation of eight large silent polyketide synthase and non-ribosomal peptide synthetase gene clusters, ranging from 50 to 134 kilobases (kb) in multiple streptomycetes, and characterization of a novel oxazole family compound produced by a 98-kb biosynthetic gene cluster. Owing to its simplicity and ease of use, this strategy can be scaled up readily for discovery of natural products in streptomycetes.</p>
</li>
</ul>
</li>
<li>
<p><em>unleashing meiotic crossovers in crops</em><br />
delphine mieulet et al. 2018<br />
<a href="http://doi.org/10.1038/s41477-018-0311-x">doi.org/10.1038/s41477-018-0311-x</a></p>
<ul>
<li>
<p>Recombination is a natural mechanism common to all organisms that reproduce sexually: plants, fungi or animals. The chromosome mix determines the genetic diversity of species. The plant breeding practised for the past ten thousand years, which consists in crossing two plants chosen for their complementary worthwhile characters, centres on that mechanism. For instance, to obtain a new tomato variety that is both tasty and pest- or disease-resistant, breeders cross and breed, via successive recombinations, plants that have the genes involved in taste and resistance. However, this is a lengthy process, as very few recombinations occur during reproduction. On average, there are just one to three genetic material crossover points between the chromosomes for every cross. It is therefore impossible to combine six worthwhile genes in a single generation, which is a major obstacle to crop improvement. So what is it that limits the number of recombinations?</p>
<p>To find out, researchers from INRA identified and studied the genes involved in controlling recombination in a model plant, Arabidopsis thaliana . They discovered that one gene, RECQ4, is particularly effective at preventing crossing-over. To the extent that inactivating it doubles to quadruples recombination frequency! What happens with crops? This is what the researchers in a consortium involving INRA and CIRAD set out to determine, by examining three agriculturally valuable species: pea, tomato and rice. And they succeeded. By &quot;switching off&quot; RECQ4, they trebled, on average, the number of crossovers, resulting in greater chromosome shuffling, hence increased diversity, with each generation. This will be a boon for future plant breeding operations at CIRAD and INRA</p>
</li>
<li>
<p>Improved plant varieties are important in our attempts to face the challenges of a growing human population and limited planet resources. Plant breeding relies on meiotic crossovers to combine favourable alleles into elite varieties1. However, meiotic crossovers are relatively rare, typically one to three per chromosome2, limiting the efficiency of the breeding process and related activities such as genetic mapping. Several genes that limit meiotic recombination were identified in the model species Arabidopsis thaliana2. Mutation of these genes in Arabidopsis induces a large increase in crossover frequency. However, it remained to be demonstrated whether crossovers could also be increased in crop species hybrids. We explored the effects of mutating the orthologues of FANCM3, RECQ44 or FIGL15 on recombination in three distant crop species, rice (Oryza sativa), pea (Pisum sativum) and tomato (Solanum lycopersicum). We found that the single recq4 mutation increases crossovers about three-fold in these crops, suggesting that manipulating RECQ4 may be a universal tool for increasing recombination in plants. Enhanced recombination could be used with other state-of-the-art technologies such as genomic selection, genome editing or speed breeding6 to enhance the pace and efficiency of plant improvement.</p>
</li>
</ul>
</li>
</ul>
<h2>04892</h2>
<h2>04792</h2>
<h2>04692</h2>
<h2>04592</h2>
<h2>04492</h2>
<h2>04392</h2>
<h2>04292</h2>
<h2>04192</h2>
<h2>04092</h2>
<h2>03992</h2>
<h2>03892</h2>
<h2>03792</h2>
<h2>03692</h2>
<h2>03592</h2>
<h2>03492</h2>
<h2>03392</h2>
<h2>03292</h2>
<h2>03192</h2>
<h2>03092</h2>
<h2>02992</h2>
<h2>02892</h2>
<ul>
<li>
<p>“junk” dna is not junk</p>
</li>
<li>
<p><em>human sex reversal is caused by duplication or deletion of core enhancers upstream of sox9</em><br />
brittany croft et al. 2018<br />
<a href="http://doi.org/10.1038/s41467-018-07784-9">doi.org/10.1038/s41467-018-07784-9</a></p>
<ul>
<li>
<p>&quot;The sex of a baby is determined by its chromosome make-up at conception. An embryo with two X chromosomes will become a girl, while an embryo with an X-Y combination results in a boy,&quot; Ms Croft said.</p>
<p>&quot;The Y chromosome carries a critical gene, called SRY, which acts on another gene called SOX9 to start the development of testes in the embryo. High levels of the SOX9 gene are needed for normal testis development.</p>
<p>&quot;However, if there is some disruption to SOX9 activity and only low levels are present, a testis will not develop resulting in a baby with a disorder of sex development.&quot;</p>
<p>Lead author of the study, Professor Andrew Sinclair, said that 90 percent of human DNA is made up of so called 'junk DNA or dark matter' which contains no genes but does carry important regulators that increase or decrease gene activity.</p>
<p>&quot;These regulatory segments of DNA are called enhancers,&quot; he said. If these enhancers that control testis genes are disrupted it may lead to a baby being born with a disorder of sex development.&quot;</p>
<p>Professor Sinclair, who is also a member of the Paediatrics Department of the University of Melbourne, said this study sought to understand how the SOX9 gene was regulated by enhancers and whether disruption of the enhancers would result in disorders of sex development.</p>
<p>&quot;We discovered three enhancers that, together ensure the SOX9 gene is turned on to a high level in an XY embryo, leading to normal testis and male development,&quot; he said.</p>
<p>&quot;Importantly, we identified XX patients who would normally have ovaries and be female but carried extra copies of these enhancers, (high levels of SOX9) and instead developed testes. In addition, we found XY patients who had lost these SOX9 enhancers, (low levels of SOX9) and developed ovaries instead of testes.&quot;</p>
<p>Ms Croft said human sex reversal such as seen in these cases is caused by gain or loss of these vital enhancers that regulate the SOX9 gene; consequently, these three enhancers are required for normal testes and male development.&quot;</p>
<p>&quot;This study is significant because in the past researchers have only looked at genes to diagnose these patients, but we have shown you need to look outside the genes to the enhancers,&quot; Ms Croft said.</p>
<p>Professor Sinclair said that across the human genome there were about one million enhancers controlling about 22,000 genes.</p>
<p>&quot;These enhancers lie on the DNA but outside genes, in regions previously referred to as junk DNA or dark matter,&quot; he said. &quot;The key to diagnosing many disorders may be found in these enhancers which hide in the poorly understood dark matter of our DNA.&quot;</p>
</li>
<li>
<p><em>abstract</em> Disorders of sex development (DSDs) are conditions affecting development of the gonads or genitalia. Variants in two key genes, SRY and its target SOX9, are an established cause of 46,XY DSD, but the genetic basis of many DSDs remains unknown. SRY-mediated SOX9 upregulation in the early gonad is crucial for testis development, yet the regulatory elements underlying this have not been identified in humans. Here, we identified four DSD patients with overlapping duplications or deletions upstream of SOX9. Bioinformatic analysis identified three putative enhancers for SOX9 that responded to different combinations of testis-specific regulators. All three enhancers showed synergistic activity and together drive SOX9 in the testis. This is the first study to identify SOX9 enhancers that, when duplicated or deleted, result in 46,XX or 46,XY sex reversal, respectively. These enhancers provide a hitherto missing link by which SRY activates SOX9 in humans, and establish SOX9 enhancer mutations as a significant cause of DSD.</p>
</li>
</ul>
</li>
<li>
<p><em>testing the retroelement invasion hypothesis for the emergence of the ancestral eukaryotic cell</em><br />
gloria lee et al. 2018<br />
<a href="http://doi.org/10.1073/pnas.1807709115">doi.org/10.1073/pnas.1807709115</a></p>
<ul>
<li>
<p>This discovery began with a curiosity for retrotransposons, known as “jumping genes,” which are DNA sequences that copy and paste themselves within the genome, multiplying rapidly. Nearly half of the human genome is made up of retrotransposons, but bacteria hardly have them at all.<br />
Nigel Goldenfeld, Swanlund Endowed Chair of Physics at the University of Illinois and Carl R. Woese Institute for Genomic Biology, and Thomas Kuhlman, a former physics professor at Illinois who is now at University of California, Riverside, wondered why this is.<br />
“We thought a really simple thing to try was to just take one (retrotransposon) out of my genome and put it into the bacteria just to see what would happen,” Kuhlman said. “And it turned out to be really quite interesting.”<br />
Their results, published in the <em>Proceedings of the National Academy of Sciences</em>, give more depth to the history of how advanced life may have emerged billions of years ago — and could also help determine the possibility and nature of life on other planets.<br />
Along the way to explaining life, the researchers first encountered death — bacterial death, that is. When they put retrotransposons in bacteria, the outcome was fatal.<br />
“As they jump around and make copies of themselves, they jump into genes that the bacteria need to survive,” Kuhlman said. “It’s incredibly lethal to them.”<br />
When retrotransposons copy themselves within the genome, they first find a spot in the DNA and cut it open. To survive, the organism then has to repair this cut. Some bacteria, like E. coli, only have one way to perform this repair, which usually ends up removing the new retrotransposon. But advanced organisms (eukaryotes) have an additional “trick” called nonhomologous end-joining, or NHEJ, that gives them another way to repair cuts in their DNA.<br />
Goldenfeld and Kuhlman decided to see what would happen if they gave bacteria the ability to do NHEJ, thinking that it would help them tolerate the damage to their DNA. But it just made the retrotransposons better at multiplying, causing even more damage than before.<br />
“It just completely killed everything,” Kuhlman said. “At the time, I thought I was just doing something wrong.”<br />
They realized that the interaction between NHEJ and retrotransposons may be more important than they previously thought.<br />
Eukaryotes typically have many retrotransposons in their genome, along with a lot of other “junk” DNA, which doesn’t have a well-understood function. Within the genome, there must be a constant interplay between NHEJ and retrotransposons, as NHEJ tries to control how rapidly the retrotransposons multiply. This gives the organism more power over their genome, and the presence of “junk” DNA is important.<br />
“As you get more and more junk in your DNA, you can start taking these pieces and combining them together in different ways, more ways than you could without all the junk in there,” Kuhlman said.<br />
These conditions — the accumulation of “junk” DNA, the presence of retrotransposons and their interactions with NHEJ — make the genome more complex. This is one feature that may distinguish advanced organisms, like humans, from simpler ones, like bacteria.<br />
Advanced organisms can also manage their genome by using their spliceosome, a molecular machine that sorts through the “junk” DNA and reconstructs the genes back to normal.<br />
Some parts of the spliceosome are similar to group II introns, bacteria’s primitive version of retrotransposons. Introns are also found in eukaryotes, and along with the spliceosome are evolutionarily derived from group II introns. Goldenfeld said this poses an evolutionary question.<br />
“What came first, the spliceosome or the group II introns? Clearly the group II introns,” he said. “So then you can ask: where did the eukaryotic cell first get those group II introns in order to build up the spliceosome early on?”<br />
This study suggests that group II introns, the ancestors of introns in the spliceosome and retrotransposons in eukaryotes, somehow invaded early eukaryotic cells. Then, their interactions with NHEJ created a “selection pressure” that helped lead to the emergence of the spliceosome, which helped life become advanced billions of years ago.<br />
The spliceosome helped life become advanced by enabling eukaryotes to do more with their DNA. For example, even though humans have roughly the same number of genes as C. elegans, a worm, humans can do more with those genes.<br />
“There’s not much difference between this very simple worm and humans, which is obviously insane,” Goldenfeld said. “What’s happening is that humans are able to take these genes and mix and match them in many combinations to do much more complicated functions than C. elegans does.”<br />
Not only did NHEJ and retrotransposons help with the creation of the spliceosome; this study suggests that they may also have assisted in making chromosomes — DNA molecules that contain genetic material — more advanced. Interactions between NHEJ and retrotransposons may have aided in the transition from circular chromosomes (which bacteria generally have) to linear ones (which more advanced organisms have), another indicator of advanced life.<br />
Goldenfeld said that before this research, many researchers studied the role of retrotransposons, but the importance of NHEJ was not fully appreciated. This research proves that it played a part, billions of years ago, in eukaryotes becoming the advanced organisms we know today.<br />
“This certainly was not the only thing that was going on,” Goldenfeld said. “But if it hadn’t happened, it’s hard to see how you could have complex life.”<br />
This study contributes to the larger questions that the Institute for Universal Biology, a NASA Astrobiology Institute that Goldenfeld directs, seeks to answer — questions like: what had to happen in order for life to become advanced?<br />
Answering this question in greater detail could help scientists determine the possibility of life on other planets.<br />
“If life exists on other planets, presumably one would expect it to be microbial. Could it ever have made this transition to complex life?” Goldenfeld said. “It’s not that you’re inevitably going to get advanced life, because there are a bunch of things that have to happen.”<br />
The physics perspective of this study helps to quantify these theoretical questions. This quantification comes from simply taking measurements in a laboratory and using those measurements to make models of evolution, as was done in this study.<br />
In doing so, basic measurements in a laboratory become a time machine to the past.<br />
“We’re doing laboratory evolution,” Goldenfeld said. “We’re looking at what evolutionary processes must have happened billions of years ago.”</p>
</li>
<li>
<p><em>abstract</em> Phylogenetic evidence suggests that the invasion and proliferation of retroelements, selfish mobile genetic elements that copy and paste themselves within a host genome, was one of the early evolutionary events in the emergence of eukaryotes. Here we test the effects of this event by determining the pressures retroelements exert on simple genomes. We transferred two retroelements, human LINE-1 and the bacterial group II intron Ll.LtrB, into bacteria, and find that both are functional and detrimental to growth. We find, surprisingly, that retroelement lethality and proliferation are enhanced by the ability to perform eukaryotic-like nonhomologous end-joining (NHEJ) DNA repair. We show that the only stable evolutionary consequence in simple cells is maintenance of retroelements in low numbers, suggesting how retrotransposition rates and costs in early eukaryotes could have been constrained to allow proliferation. Our results suggest that the interplay between NHEJ and retroelements may have played a fundamental and previously unappreciated role in facilitating the proliferation of retroelements, elements of which became the ancestors of the spliceosome components in eukaryotes.</p>
</li>
</ul>
</li>
<li>
<p><em>background selection and biased gene conversion affect more than 95% of the human genome and bias demographic inferences</em><br />
fanny pouyet et al. 2018<br />
<a href="http://doi.org/10.7554/elife.36317">doi.org/10.7554/elife.36317</a></p>
<ul>
<li>Disentangling the effect on genomic diversity of natural selection from that of demography is notoriously difficult, but necessary to properly reconstruct the history of species. Here, we use high-quality human genomic data to show that purifying selection at linked sites (i.e. background selection, BGS) and GC-biased gene conversion (gBGC) together affect as much as 95% of the variants of our genome. We find that the magnitude and relative importance of BGS and gBGC are largely determined by variation in recombination rate and base composition. Importantly, synonymous sites and non-transcribed regions are also affected, albeit to different degrees. Their use for demographic inference can lead to strong biases. However, by conditioning on genomic regions with recombination rates above 1.5 cM/Mb and mutation types (C↔G, A↔T), we identify a set of SNPs that is mostly unaffected by BGS or gBGC, and that avoids these biases in the reconstruction of human history.</li>
</ul>
</li>
<li>
<p><em>a zombie lif gene in elephants is upregulated by tp53 to induce apoptosis in response to dna damage</em><br />
juan manuel vazquez et al. 2018<br />
<a href="http://doi.org/10.1016/j.celrep.2018.07.042">doi.org/10.1016/j.celrep.2018.07.042</a></p>
<ul>
<li>
<p>Elephants have extra <em>LIF</em> genes; one (<em>LIF6</em>) is expressed in response to DNA damage<br />
<em>LIF6</em> encodes a separation of function isoform that is intracellular<br />
LIF6 induces Bak/Bax-dependent apoptosis<br />
Evolutionary analyses indicates that <em>LIF6</em> is a refunctionalized pseudogene</p>
<p>Large-bodied organisms have more cells that can potentially turn cancerous than small-bodied organisms, imposing an increased risk of developing cancer. This expectation predicts a positive correlation between body size and cancer risk; however, there is no correlation between body size and cancer risk across species (“Peto’s paradox”). Here, we show that elephants and their extinct relatives (proboscideans) may have resolved Peto’s paradox in part through refunctionalizing a leukemia inhibitory factor pseudogene (<em>LIF6</em>) with pro-apoptotic functions. <em>LIF6</em> is transcriptionally upregulated by TP53 in response to DNA damage and translocates to the mitochondria where it induces apoptosis. Phylogenetic analyses of living and extinct proboscidean <em>LIF6</em> genes indicates that its TP53 response element evolved coincident with the evolution of large body sizes in the proboscidean stem lineage. These results suggest that refunctionalizing of a pro-apoptotic LIF pseudogene may have been permissive (although not sufficient) for the evolution of large body sizes in proboscideans.</p>
</li>
</ul>
</li>
<li>
<p><em>genomic variation within alpha satellite dna influences centromere location on human chromosomes with metastable epialleles</em><br />
megan aldrup-macdonald et al. 2016;<br />
<a href="http://doi.org/10.1101/gr.206706.116">doi.org/10.1101/gr.206706.116</a></p>
<ul>
<li>Alpha satellite is a tandemly organized type of repetitive DNA that comprises 5% of the genome and is found at all human centromeres. A defined number of 171-bp monomers are organized into chromosome-specific higher-order repeats (HORs) that are reiterated thousands of times. At least half of all human chromosomes have two or more distinct HOR alpha satellite arrays within their centromere regions. We previously showed that the two alpha satellite arrays of Homo sapiens Chromosome 17 (HSA17), D17Z1 and D17Z1-B, behave as centromeric epialleles, that is, the centromere, defined by chromatin containing the centromeric histone variant CENPA and recruitment of other centromere proteins, can form at either D17Z1 or D17Z1-B. Some individuals in the human population are functional heterozygotes in that D17Z1 is the active centromere on one homolog and D17Z1-B is active on the other. In this study, we aimed to understand the molecular basis for how centromere location is determined on HSA17. Specifically, we focused on D17Z1 genomic variation as a driver of epiallele formation. We found that D17Z1 arrays that are predominantly composed of HOR size and sequence variants were functionally less competent. They either recruited decreased amounts of the centromere-specific histone variant CENPA and the HSA17 was mitotically unstable, or alternatively, the centromere was assembled at D17Z1-B and the HSA17 was stable. Our study demonstrates that genomic variation within highly repetitive, noncoding DNA of human centromere regions has a pronounced impact on genome stability and basic chromosomal function.</li>
</ul>
</li>
<li>
<p><em>l1-associated genomic regions are deleted in somatic cells of the healthy human brain</em><br />
jennifer erwin et al. 2016<br />
<a href="http://doi.org/10.1038/nn.4388">doi.org/10.1038/nn.4388</a></p>
<ul>
<li>The healthy human brain is a mosaic of varied genomes. Long interspersed element-1 (LINE-1 or L1) retrotransposition is known to create mosaicism by inserting L1 sequences into new locations of somatic cell genomes. Using a machine learning-based, single-cell sequencing approach, we discovered that somatic L1-associated variants (SLAVs) are composed of two classes: L1 retrotransposition insertions and retrotransposition-independent L1-associated variants. We demonstrate that a subset of SLAVs comprises somatic deletions generated by L1 endonuclease cutting activity. Retrotransposition-independent rearrangements in inherited L1s resulted in the deletion of proximal genomic regions. These rearrangements were resolved by microhomology-mediated repair, which suggests that L1-associated genomic regions are hotspots for somatic copy number variants in the brain and therefore a heritable genetic contributor to somatic mosaicism. We demonstrate that SLAVs are present in crucial neural genes, such as DLG2 (also called PSD93), and affect 44–63% of cells of the cells in the healthy brain.</li>
</ul>
</li>
<li>
<p><em>strains of bacterial species induce a greatly varied acute adaptive immune response: the contribution of the accessory genome</em><br />
uri sela et al. 2017<br />
<a href="http://doi.org/10.1371/journal.ppat.1006726">doi.org/10.1371/journal.ppat.1006726</a></p>
<ul>
<li>A fundamental question in human susceptibility to bacterial infections is to what extent variability is a function of differences in the pathogen species or in individual humans. To focus on the pathogen species, we compared in the same individual the human adaptive T and B cell immune response to multiple strains of two major human pathogens, <em>Staphylococcus aureus</em> and <em>Streptococcus pyogenes</em>. We found wide variability in the acute adaptive immune response induced by various strains of a species, with a unique combination of activation within the two arms of the adaptive response. Further, this was also accompanied by a dramatic difference in the intensity of the specific protective T helper (Th) response. Importantly, the same immune response differences induced by the individual strains were maintained across multiple healthy human donors. A comparison of isogenic phage KO strains, demonstrated that of the pangenome, prophages were the major contributor to inter-strain immune heterogeneity, as the T cell response to the remaining “core genome” was noticeably blunted. Therefore, these findings extend and modify the notion of an adaptive response to a pathogenic bacterium, by implying that the adaptive immune response signature of a bacterial species should be defined either per strain or alternatively to the species’ ‘core genome’, common to all of its strains. Further, our results demonstrate that the acquired immune response variation is as wide among different strains within a single pathogenic species as it is among different humans, and therefore may explain in part the clinical heterogeneity observed in patients infected with the same species.</li>
</ul>
</li>
<li>
<p><em>long noncoding RNA NEAT1 mediates neuronal histone methylation and age-related memory impairment</em><br />
anderson a. butler et al. 2019<br />
<a href="http://doi.org/10.1126/scisignal.aaw9277">doi.org/10.1126/scisignal.aaw9277</a></p>
<ul>
<li>
<p>non-coding RNA may play a more important role than originally believed.</p>
<p>&quot;NEAT1 is a tissue-specific, non-coding RNA found in the hippocampus region of the brain. This brain region is most associated with learning and memory,&quot; said Farah Lubin, Ph.D., associate professor in the Department of Neurobiology and primary investigator of the study. &quot;While it has some association with cancer in other parts of the body, we have discovered that, in the hippocampus, NEAT1 appears to regulate memory formation.&quot;</p>
<p>Lubin says that, when NEAT1 is on, or active, we do not learn as well. But when presented with an outside learning experience, it turns off, allowing the brain to learn from the outside stimulus. She uses a car analogy. The engine might be running; but when the brakes are on, the car does not move. You have to take off the brakes and hit the gas to get the car to move.</p>
<p>&quot;NEAT1 is the brake: When it is on, we aren't learning, at least not as much as we might with it off,&quot; Lubin said. &quot;In a younger brain, when presented with stimulus that promotes learning, NEAT1 turns off. Since one of the hallmarks of aging is a decline in memory, we wondered if NEAT1 was implicated in that decline.&quot;</p>
<p>Lubin says one of the genes that NEAT1 acts upon is c-FOS, which is necessary for memory formation. In an aging brain, NEAT1 is on more than it is in a younger brain, interfering with the epigenetic regulation of c-FOS, which disrupts its memory functions.</p>
<p>Using siRNA techniques in a mouse model, Lubin's team was able to turn off NEAT1 in older mice. With NEAT1 off, the mice demonstrated normal abilities in learning and memory.</p>
<p>The next step was to change the level of NEAT1 in younger mice, using CRISPR/dCas9 gene-activation technology. Boosting the presence of NEAT1 in younger mice caused a decline in their ability to learn and remember.</p>
<p>&quot;Turning NEAT1 off in older animals boosted memory, while increasing NEAT1 in younger animals diminished memory,&quot; Lubin said. &quot;This gives us very strong evidence that NEAT1 and its effects on the epigenetic control of c-FOS are one of the keys to memory formation. These are significant findings, for not only did we find a novel epigenetic initiator and regulator, we identified a new role for the NEAT1 non-coding RNA. This sets the stage for more research into the potential roles played by other non-coding RNAs.&quot;</p>
</li>
<li>
<p><em>abstract</em> Histone methylation is critical for the formation and maintenance of long-term memories. Long noncoding RNAs (lncRNAs) are regulators of histone methyltransferases and other chromatin-modifying enzymes (CMEs), thereby epigenetically modifying gene expression. Here, we investigated how the lncRNA NEAT1 may epigenetically contribute to hippocampus-dependent, long-term memory formation using a combination of transcriptomics, RNA-binding protein immunoprecipitation, CRISPR-mediated gene activation (CRISPRa), and behavioral approaches. Knockdown of the lncRNA Neat1 revealed widespread changes in gene transcription, as well as perturbations of histone 3 lysine 9 dimethylation (H3K9me2), a repressive histone modification mark that was increased in the hippocampus of aging rodents. We identified a NEAT1-dependent mechanism of transcriptional repression by H3K9me2 at the c-Fos promoter, corresponding with observed changes in c-Fos mRNA expression. Overexpression of hippocampal NEAT1 using CRISPRa was sufficient to impair memory formation in young adult mice, recapitulating observed memory deficits in old adult mice, whereas knocking down NEAT1 in both young and old adult mice improved behavior test–associated memory. These results suggest that the lncRNA NEAT1 is an epigenetic suppressor of hippocampus-dependent, long-term memory formation.</p>
</li>
</ul>
</li>
</ul>
<h2>02792</h2>
<h2>02692</h2>
<h2>02592</h2>
<ul>
<li>diversity through sex
<ul>
<li>
<p><em>sex-specific dominance reversal of genetic variation for fitness</em><br />
karl grieshop, göran arnqvist 2018<br />
<a href="http://doi.org/10.1371/journal.pbio.2006810">doi.org/10.1371/journal.pbio.2006810</a></p>
<ul>
<li>
<p>Evolutionary genetic theory shows that genetic variation can be maintained when selection favors different versions of the same genes in males and females -- an inevitable outcome of having separate sexes. That is, for many genes there may not be a universally 'best' version, but rather one is best for males and one is best for females. This is known as sexually antagonistic genetic variation, but it might only be maintained under a narrow set of conditions, limiting its prevalence in nature. However, Dr. Karl Grieshop and Professor Göran Arnqvist's study, published in PLoS Biology, may change this view.</p>
<p>&quot;One of the simplest ways for sexually antagonistic selection to maintain genetic variation in fitness is via sex-specific dominance reversal, where neither version of a gene is always dominant or recessive, but rather the version that benefits a given sex is also dominant in that sex. So, whether a given version of a gene is dominant or recessive to the other will depend upon which sex it is in,&quot; says Dr. Karl Grieshop.</p>
<p>This mechanism was met with early skepticism, but has seen recent theoretical and empirical support.</p>
<p>Grieshop and Arnqvist have now provided the first evidence of sex-specific dominance reversal for fitness. Using a panel of genetic strains of a seed beetle population that Grieshop studied throughout his PhD, and analyzing crosses among these strains, they could determine which strains harbored genetic variation that was dominant to the others'. Further, they could do this with regard to male fitness and female fitness separately. When they ranked the strains according to their relative dominance over one another they found that strains tending to be dominant over other strains with regard to male fitness also tended to be recessive to other strains with regard to female fitness, and vice versa. Thus, whether the genetic variation for fitness in each of their strains was dominant or recessive to that of other strains depended, oppositely, on whether it was in a male or a female.</p>
<p>The pattern suggests that sex-specific dominance reversal for fitness is a strong and common phenomenon throughout the genome in their study population</p>
</li>
<li>
<p><em>abstract</em> The maintenance of genetic variance in fitness represents one of the most longstanding enigmas in evolutionary biology. Sexually antagonistic (SA) selection may contribute substantially to maintaining genetic variance in fitness by maintaining alternative alleles with opposite fitness effects in the two sexes. This is especially likely if such SA loci exhibit sex-specific dominance reversal (SSDR)—wherein the allele that benefits a given sex is also dominant in that sex—which would generate balancing selection and maintain stable SA polymorphisms for fitness. However, direct empirical tests of SSDR for fitness are currently lacking. Here, we performed a full diallel cross among isogenic strains derived from a natural population of the seed beetle Callosobruchus maculatus that is known to exhibit SA genetic variance in fitness. We measured sex-specific competitive lifetime reproductive success (i.e., fitness) in &gt;500 sex-by-genotype F1 combinations and found that segregating genetic variation in fitness exhibited pronounced contributions from dominance variance and sex-specific dominance variance. A closer inspection of the nature of dominance variance revealed that the fixed allelic variation captured within each strain tended to be dominant in one sex but recessive in the other, revealing genome-wide SSDR for SA polymorphisms underlying fitness. Our findings suggest that SA balancing selection could play an underappreciated role in maintaining fitness variance in natural populations.</p>
</li>
</ul>
</li>
</ul>
</li>
</ul>
<h2>02492</h2>
<h2>02392</h2>
<ul>
<li>mutation
<ul>
<li>
<p><em>the rate and potential relevance of new mutations in a colonizing plant lineage</em><br />
moises exposito-alonso et al. 2018<br />
<a href="http://doi.org/10.1371/journal.pgen.1007155">doi.org/10.1371/journal.pgen.1007155</a></p>
<ul>
<li>By following the evolution of populations that are initially genetically homogeneous, much can be learned about core biological principles. For example, it allows for detailed studies of the rate of emergence of <em>de novo</em> mutations and their change in frequency due to drift and selection. Unfortunately, in multicellular organisms with generation times of months or years, it is difficult to set up and carry out such experiments over many generations. An alternative is provided by “natural evolution experiments” that started from colonizations or invasions of new habitats by selfing lineages. With limited or missing gene flow from other lineages, new mutations and their effects can be easily detected. North America has been colonized in historic times by the plant <em>Arabidopsis thaliana</em>, and although multiple intercrossing lineages are found today, many of the individuals belong to a single lineage, HPG1. To determine in this lineage the rate of substitutions—the subset of mutations that survived natural selection and drift–, we have sequenced genomes from plants collected between 1863 and 2006. We identified 73 modern and 27 herbarium specimens that belonged to HPG1. Using the estimated substitution rate, we infer that the last common HPG1 ancestor lived in the early 17th century, when it was most likely introduced by chance from Europe. Mutations in coding regions are depleted in frequency compared to those in other portions of the genome, consistent with purifying selection. Nevertheless, a handful of mutations is found at high frequency in present-day populations. We link these to detectable phenotypic variance in traits of known ecological importance, life history and growth, which could reflect their adaptive value. Our work showcases how, by applying genomics methods to a combination of modern and historic samples from colonizing lineages, we can directly study new mutations and their potential evolutionary relevance.</li>
</ul>
</li>
<li>
<p><em>evolution and unprecedented variants of the mitochondrial genetic code in a lineage of green algae</em><br />
david žihala, marek eliáš 2019<br />
<a href="http://doi.org/10.1093/gbe/evz210">doi.org/10.1093/gbe/evz210</a></p>
<ul>
<li>
<p>an analysis of the mitochondrial genomes of 51 green algae and land plants (Noutahi et al. 2019). This analysis relied on a newly expanded version of the bioinformatics tool CoreTracker, which was previously developed by this group (Noutahi et al. 2017). CoreTracker identifies differences between a DNA sequence and the expected amino acid based on the amino acids often found at that position in closely related species. Using this tool, Noutahi and colleagues identified 14 new codon reassignments involving the replacement of one amino acid with another, the vast majority of which were found in a group of algae known as the Sphaeropleales. These algae have an unusual mitochondrial genome organization that appears to be intermediate between the larger, ancestral genomes and the compact, derived genomes of some of their relatives.</p>
<p>According to the authors, the field of genetic code evolution is being fueled by a rapid increase in genomics data (genomes plus respective transcriptomes). Because of this, &quot;Comparative/evolutionary bioinformatics procedures such as CoreTracker are now in a position to not only predict deviations of the genetic code, but also provide clues with regard to the underlying mechanism.&quot; Indeed, in light of their results, the researchers proposed that the genetic code deviations in the Sphaeropleales mitochondria actually contributed to their unusual genome organization. Based on this theory, after the migration of some mitochondrial genes to the nuclear genome during the genome reduction process, &quot;UCA&quot; (normally encoding the amino acid serine) was reassigned to a termination codon. This would have made it impossible for additional mitochondrial genes to be transferred to the nucleus, resulting in a mitochondrial genome that was intermediate in terms of size.</p>
<p>Before the publication of the article by Noutahi et al., researchers David Zihala and Marek Elias from the University of Ostrava had also independently discovered the large number of changes to the genetic code in the Sphaeropleales. Following the purely coincidental discovery of novel genetic codes in several protists by Elias's research group, Zihala and Elias were motivated to &quot;embark on a systematic screening to find possible additional cases of organisms with novel genetic code variants or previously missed departures from the standard genetic code.&quot; Like Noutahi et al., their method involved the identification of discrepancies between DNA sequences and expected amino acids based on sequences present in related genomes, although they also performed a certain amount of manual curation. The analysis, published in the current issue of Genome Biology and Evolution (Zihala and Elias 2019), identified a few more codon reassignments in the Sphaeropleales, due to the fact that they included a broader sampling of this group. Otherwise, the results of the two studies were highly congruent, despite the somewhat different methods employed. In addition to the genetic code changes, Zihala and Elias also identified mutations in a mitochondrial release factor -- a protein that recognizes termination codons -- that, according to Elias, &quot;may be linked to the intriguing ability of some sphaeroplealean mitochondria to terminate translation at codons that are normally read as coding for an amino acid. We thus offer the first specific hypothesis for the molecular underpinnings of this unusual ability.&quot;</p>
<p>Overall, the findings of both studies highlight the need for a deeper awareness of genetic code discrepancies across the tree of life. Otherwise, use of an incorrect code when inferring protein sequences from DNA sequences could lead to inaccuracies in predicted protein sequences that are used for both phylogenetic and molecular biology analyses. Moreover, according to Noutahi and colleagues, &quot;The two publications have been successful in predicting specific changes in the meaning of codons by using publicly available data, without biochemical experimentation, yet with high confidence.&quot;</p>
<p>However, they also note that both studies are strictly computational in nature and that &quot;this type of 'paper biochemistry' has its limitations. Only evolutionarily well-established instances of codon evolution may be inferred (i.e., not including cases of initial or incomplete stages), and although changes to the tRNA repertoire, structure, and specificity may be inferred to some degree, biochemical confirmation is critically required.&quot; This limitation was also pointed out by Elias, who noted that his group plans to employ proteomics methods to verify some of their bioinformatic predictions concerning the various presumably reassigned codons. &quot;Unfortunately,&quot; continues Elias, &quot;non-standard genetic codes are generally found in organisms that are difficult to study by direct biochemical or genetic approaches, so it remains a challenge to get a deeper understanding of the molecular mechanisms behind the observed changes in codon meaning.&quot;</p>
<p>Future investigations will almost certainly uncover additional, as-yet-unidentified changes in the genetic code in various organisms. Indeed, notes Elias, &quot;We are also analyzing some exciting new cases of genetic code modification in nuclear genomes of certain obscure protists that were uncovered by our survey of publicly available sequence data.&quot; Furthermore, the authors of the Noutahi et al. study point out that, given &quot;how rapidly this field is evolving, due to an ever increasing number of reports on deviations from the standard genetic code -- particularly in eukaryotes and their organelles... the journey to understanding codon evolution with all of its mechanistic implications has just begun.&quot;</p>
</li>
<li>
<p><em>abstract</em> Mitochondria of diverse eukaryotes have evolved various departures from the standard genetic code, but the breadth of possible modifications and their phylogenetic distribution are known only incompletely. Furthermore, it is possible that some codon reassignments in previously sequenced mitogenomes have been missed, resulting in inaccurate protein sequences in databases. Here we show, considering the distribution of codons at conserved amino acid positions in mitogenome-encoded proteins, that mitochondria of the green algal order Sphaeropleales exhibit a diversity of codon reassignments, including previously missed ones and some that are unprecedented in any translation system examined so far, necessitating redefinition of existing translation tables and creating at least seven new ones. We resolve a previous controversy concerning the meaning the UAG codon in Hydrodictyaceae, which beyond any doubt encodes alanine. We further demonstrate that AGG, sometimes together with AGA, encodes alanine instead of arginine in diverse sphaeroplealeans. Further newly detected changes include Arg-to-Met reassignment of the AGG codon and Arg-to-Leu reassignment of the CGG codon in particular species. Analysis of tRNAs specified by sphaeroplealean mitogenomes provides direct support for and molecular underpinning of the proposed reassignments. Furthermore, we point to unique mutations in the mitochondrial release factor mtRF1a that correlate with changes in the use of termination codons in Sphaeropleales, including the two independent stop-to-sense UAG reassignments, the reintroduction of UGA in some Scenedesmaceae, and the sense-to-stop reassignment of UCA widespread in the group. Codon disappearance seems to be the main drive of the dynamic evolution of the mitochondrial genetic code in Sphaeropleales.</p>
</li>
</ul>
</li>
</ul>
</li>
</ul>
<h2>02292</h2>
<h2>02192</h2>
<ul>
<li>transmit to offspring
<ul>
<li>
<p>chromosome</p>
<ul>
<li>
<p>mutations and sexual recombination transmitted via DNA, distribution changes and also elimination via genetic drift and migration</p>
</li>
<li>
<p><em>defined chromosome structure in the genome-reduced bacterium mycoplasma pneumoniae</em><br />
marie trussart et al. 2017<br />
<a href="http://doi.org/10.1038/ncomms14665">doi.org/10.1038/ncomms14665</a></p>
<ul>
<li>DNA-binding proteins are central regulators of chromosome organization; however, in genome-reduced bacteria their diversity is largely diminished. Whether the chromosomes of such bacteria adopt defined three-dimensional structures remains unexplored. Here we combine Hi-C and super-resolution microscopy to determine the structure of the <em>Mycoplasma pneumoniae</em> chromosome at a 10 kb resolution. We find a defined structure, with a global symmetry between two arms that connect opposite poles, one bearing the chromosomal Ori and the other the midpoint. Analysis of local structures at a 3 kb resolution indicates that the chromosome is organized into domains ranging from 15 to 33 kb. We provide evidence that genes within the same domain tend to be co-regulated, suggesting that chromosome organization influences transcriptional regulation, and that supercoiling regulates local organization. This study extends the current understanding of bacterial genome organization and demonstrates that a defined chromosomal structure is a universal feature of living systems.</li>
<li>a higher 10 kb resolution Hi-C map of <em>C. crescentus</em><a href="http://www.nature.com/articles/ncomms14665#ref18">18</a> revealed that its genome is divided into 23 chromosome interacting domains (CIDs) or highly self-interacting regions, similar to the topologically associating domains (TADs) found in eukaryotes</li>
</ul>
</li>
<li>
<p><em>genome downsizing, physiological novelty, and the global dominance of flowering plants</em><br />
kevin a. simonin, adam b. roddy 2017<br />
<a href="http://doi.org/10.1371/journal.pbio.2003706">doi.org/10.1371/journal.pbio.2003706</a></p>
<ul>
<li>The abrupt origin and rapid diversification of the flowering plants during the Cretaceous has long been considered an “abominable mystery.” While the cause of their high diversity has been attributed largely to coevolution with pollinators and herbivores, their ability to outcompete the previously dominant ferns and gymnosperms has been the subject of many hypotheses. Common among these is that the angiosperms alone developed leaves with smaller, more numerous stomata and more highly branching venation networks that enable higher rates of transpiration, photosynthesis, and growth. Yet, how angiosperms pack their leaves with smaller, more abundant stomata and more veins is unknown but linked—we show—to simple biophysical constraints on cell size. Only angiosperm lineages underwent rapid genome downsizing during the early Cretaceous period, which facilitated the reductions in cell size necessary to pack more veins and stomata into their leaves, effectively bringing actual primary productivity closer to its maximum potential. Thus, the angiosperms’ heightened competitive abilities are due in no small part to genome downsizing.</li>
</ul>
</li>
<li>
<p><em>dynamic basis for dg•dt misincorporation via tautomerization and ionization</em><br />
isaac j. kimsey et al. 2018<br />
<a href="http://doi.org/10.1038/nature25487">doi.org/10.1038/nature25487</a></p>
<ul>
<li>Tautomeric and anionic Watson–Crick-like mismatches have important roles in replication and translation errors through mechanisms that are not fully understood. Here, using NMR relaxation dispersion, we resolve a sequence-dependent kinetic network connecting G•T/U wobbles with three distinct Watson–Crick mismatches: two rapidly exchanging tautomeric species (Genol•T/U￼G•Tenol/Uenol; population less than 0.4%) and one anionic species (G•T–/U–; population around 0.001% at neutral pH). The sequence-dependent tautomerization or ionization step was inserted into a minimal kinetic mechanism for correct incorporation during replication after the initial binding of the nucleotide, leading to accurate predictions of the probability of dG•dT misincorporation across different polymerases and pH conditions and for a chemically modified nucleotide, and providing mechanisms for sequence-dependent misincorporation. Our results indicate that the energetic penalty for tautomerization and/or ionization accounts for an approximately 10−2 to 10−3-fold discrimination against misincorporation, which proceeds primarily via tautomeric dGenol•dT and dG•dTenol, with contributions from anionic dG•dT– dominant at pH 8.4 and above or for some mutagenic nucleotides.</li>
</ul>
</li>
</ul>
</li>
<li>
<p>evodevo<br />
evolutionary development biology</p>
<ul>
<li>alters developmental processes to create new and novel structures from the old gene networks (such as bone structures of the jaw deviating to the ossicles of the middle ear) or will conserve (molecular economy) a similar program in a host of organisms such as eye development genes in molluscs, insects, and vertebrates</li>
</ul>
</li>
<li>
<p>neochromosome</p>
<ul>
<li>cancerous mutation e.g.<br />
<em>the architecture and evolution of cancer neochromosomes</em><br />
dale garsed et al. 2014<br />
<a href="http://dx.doi.org/10.1016/j.ccell.2014.09.010">dx.doi.org/10.1016/j.ccell.2014.09.010</a></li>
</ul>
</li>
<li>
<p>epigenome</p>
<ul>
<li>
<p>methylation during lifetime transmitted via germline</p>
</li>
<li>
<p><em>evolutionary persistence of dna methylation for millions of years after ancient loss of a de novo methyltransferase</em><br />
sandra catania et al. 2020<br />
<a href="http://doi.org/10.1016/j.cell.2019.12.012">doi.org/10.1016/j.cell.2019.12.012</a></p>
<ul>
<li>
<p>selection may also occur at the level of the epigenome — a term that refers to an assortment of chemical “annotations” to the genome that determine whether, when and to what extent genes are activated — and has done so for tens of millions of years. This unprecedented finding subverts the widely accepted notion that over geologic timescales, natural selection acts exclusively on variation in the genome sequence itself.<br />
In a study published Jan. 16, 2020 in the journal <em>Cell</em>, the researchers show that <em>Cryptococcus neoformans</em> — a pathogenic yeast that infects people with weakened immune systems and is responsible for about 20 percent of all HIV/AIDS-related deaths — contains a particular epigenetic “mark” on its DNA sequence, which, based on their lab experiments and statistical models, should have disappeared from the species sometime during the age of the dinosaurs.<br />
But the study shows that this methylation mark — so named because it’s created through a process that attaches a molecular tag called a methyl group to the genome — has managed to stick around for at least 50 million years — maybe as long as 150 million years — past its predicted expiration date. This amazing feat of evolutionary tenacity is made possible by an unusual enzyme and a hefty dose of natural selection.<br />
“What we’ve seen is that methylation can undergo natural variation and can be selected for over million-year time scales to drive evolution,” explained Hiten Madhani, MD, PhD, professor of biochemistry and biophysics at UCSF and senior author of the new study. “This is a previously unappreciated mode of evolution that’s not based on changes in the organism’s DNA sequence.”<br />
Though not seen in all life forms, DNA methylation isn’t uncommon either. It’s found in all vertebrates and plants, as well as many fungi and insects. In some species, however, methylation is nowhere to be found.<br />
“Methylation has a patchy evolutionary presence,” said Madhani, who is also a member of the UCSF Helen Diller Family Comprehensive Cancer Center and a Chan-Zuckerberg Biohub investigator. “Depending on what branch of the evolutionary tree you look at, different epigenetic mechanisms have been maintained or not maintained.”<br />
Many model organisms that are staples of the modern molecular biology lab — including the baker’s yeast <em>S. cerevisiae</em>, the roundworm <em>C. elegans</em>, and the fruit fly <em>D. melanogaster</em> — lack DNA methylation entirely. These species are descended from ancient ancestors that lost enzymes that were, until this study was published, thought to be essential for propagating methylation for generation upon generation. How <em>C. neoformans</em> managed to avoid the same fate was a mystery up to now.<br />
In the new study, Madhani and his collaborators show that hundreds of millions of years ago, the ancestor of <em>C. neoformans</em> had two enzymes that controlled DNA methylation. One was what’s known as a “de novo methyltransferase,” which was responsible for adding methylation marks to “naked” DNA that had none. The other was a “maintenance methyltransferase” that functioned a bit like a molecular Xerox. This enzyme copied existing methylation marks, which had been put in place by the de novo methyltransferase, onto unmethylated DNA during DNA replication. And like every other species with an epigenome that includes methylation, the ancestor of <em>C. neoformans</em> had both types of methyltransferase.<br />
But then, sometime during the age of the dinosaurs, the ancestor of <em>C. neoformans</em> lost its de novo enzyme. Its descendants have been living without one since then, making <em>C. neoformans</em> and its closest relatives the only species alive today known to have DNA methylation without a de novo methyltransferase. “We didn’t understand how methylation could still be in place since the Cretaceous period without a de novo enzyme,” said Madhani.<br />
Though the maintenance methyltransferase was still available to copy any existing methylation marks — and the new study clearly demonstrates that this enzyme is unique among such enzymes for a number of reasons, including its ability to propagate existing methylation marks with exceptionally high fidelity — the study also shows that unless natural selection were acting to preserve methylation, the ancient loss of the de novo methyltransferase should have resulted in the rapid demise and eventual disappearance of DNA methylation in <em>C. neoformans</em>.<br />
That’s because methylation marks can be randomly lost, which means that no matter how exquisitely a maintenance methyltransferase copies existing marks onto new strands of DNA, the accumulated loss of methylation would eventually leave the maintenance enzyme with no template to work from. Though it’s conceivable that these loss events might occur at a sluggish pace, experimental observations allowed the researchers to determine that each methylation mark in <em>C. neoformans</em> was likely to disappear from half of the population after just 7500 generations. Even assuming that for some reason <em>C. neoformans</em> might reproduce 100 times more slowly in the wild than in the lab, this would still be the equivalent of only 130 years.<br />
The rare and random acquisition of new methylation marks can’t account for the persistence of methylation in <em>C. neoformans</em> either. The researchers’ lab experiments demonstrated that new methylation marks arise by chance at a rate 20 times slower than methylation losses. Over evolutionary timescales, the losses would clearly predominate, and without a de novo enzyme to compensate, methylation would have vanished from <em>C. neoformans</em> around the time when dinosaurs disappeared had it not been for selection pressures favoring the marks.<br />
In fact, when the researchers compared a variety of <em>C. neoformans</em> strains that were known to have diverged from one another nearly 5 million years ago, they found that not only did all the strains still have DNA methylation, but the methylation marks were coating analogous regions of the genome, a finding which suggests that methylation marks at specific genomic sites confer some sort of survival advantage that’s being selected for.<br />
“Natural selection is maintaining methylation at much higher levels than would be expected from a neutral process of random gains and losses. This is the epigenetic equivalent of Darwinian evolution,” said Madhani.<br />
Asked why evolution would select for these particular marks, Madhani explained that “one of methylation’s major functions is genome defense. In this case we think it’s for silencing transposons.”<br />
Transposons, also known as jumping genes, are stretches of DNA that are able to extract themselves from one part of the genome and insert themselves into another. If a transposon were to insert itself into the middle of a gene needed for survival, that gene may no longer function and the cell would die. Therefore, transposon-silencing methylation provides an obvious survival advantage, which is exactly what’s needed to drive evolution.<br />
However, it remains to be seen how common this unappreciated form of natural selection is in other species.<br />
“Previously, there was no evidence of this kind of selection happening over these time scales. This is an entirely novel concept,” Madhani said. “But now the big question is ‘Is this happening outside of this exceptional circumstance, and if so, how do we find it?’”</p>
</li>
<li>
<p><em>abstract</em> •Exquisitely specific maintenance methylase enzyme drives all 5mC in <em>C. neoformans</em><br />
•Once lost, methylation is not efficiently restored mitotically or meiotically<br />
•The <em>de novo</em> enzyme DnmtX was lost in an ancestral species ∼50–150 mya<br />
•Persistence of 5mC for millions of years explained by Darwinian epigenome evolution</p>
<p>Cytosine methylation of DNA is a widespread modification of DNA that plays numerous critical roles. In the yeast <em>Cryptococcus neoformans</em>, CG methylation occurs in transposon-rich repeats and requires the DNA methyltransferase Dnmt5. We show that Dnmt5 displays exquisite maintenance-type specificity <em>in vitro</em> and <em>in vivo</em> and utilizes similar <em>in vivo</em> cofactors as the metazoan maintenance methylase Dnmt1. Remarkably, phylogenetic and functional analysis revealed that the ancestral species lost the gene for a <em>de novo</em> methylase, DnmtX, between 50–150 mya. We examined how methylation has persisted since the ancient loss of DnmtX. Experimental and comparative studies reveal efficient replication of methylation patterns in <em>C. neoformans</em>, rare stochastic methylation loss and gain events, and the action of natural selection. We propose that an epigenome has been propagated for &gt;50 million years through a process analogous to Darwinian evolution of the genome.</p>
</li>
</ul>
</li>
<li>
<p><em>piwi/prg-1 argonaute and tgf-β mediate transgenerational learned pathogenic avoidance</em><br />
rebecca s. moore et al. 2019<br />
<a href="http://doi.org/10.1016/j.cell.2019.05.024">doi.org/10.1016/j.cell.2019.05.024</a></p>
<ul>
<li>
<p>It's well known that an organism's characteristics are encoded in genes that are passed down from parent to progeny through the eggs and sperm of the germline. The inheritance of some traits is determined exclusively by whether the individual receives the dominant or recessive form of an associated gene from each parent. Other heritable traits are influenced both by genetic makeup and by factors such as nutrition, temperature or environmental stress, which can affect the expression levels of related genes. Features whose inheritance isn't driven exclusively by DNA sequence are termed &quot;epigenetic&quot; (the prefix &quot;epi&quot; means &quot;on top of&quot;).</p>
<p>An organism's phenotype can change during its lifetime due to epigenetic mechanisms. For example, in the microscopic roundworm Caenorhabditis elegans, starvation or heat stress prompts animals to adapt to these conditions by varying the expression of multiple genes. At the level of the genome, these changes can be made durable by altering how tightly the DNA that encodes a gene is packed, thereby regulating its accessibility to RNA transcription machinery. Alternatively, cells can engage mechanisms that destroy or sequester protein-coding RNA transcripts. When these modifications are made in germ cells, they can be passed down to future generations in a phenomenon is known as transgenerational epigenetic inheritance. Studies have shown that C. elegans adaptations to starvation and heat stress can be inherited for several generations. Might more complex phenotypes, such as behavioral changes, also be passed down in this way?</p>
<p>&quot;In their natural environment, worms come into contact with many different bacterial species. Some of these are nutritious food sources, while others will infect and kill them,&quot; said Murphy, a professor in Princeton's Department of Molecular Biology and the Lewis-Sigler Institute for Integrative Genomics. &quot;Worms are initially attracted to the pathogen Pseudomonas aeruginosa, but upon infection, they learn to avoid it. Otherwise they will die within a few days.&quot;</p>
<p>Moore and her colleagues investigated whether C. elegans can convey this learned avoidance behavior to their progeny. They found that when mother worms learned to avoid pathogenic P. aeruginosa, their progeny also knew to avoid the bacteria. The natural attraction of offspring to Pseudomonas was overridden even though they had never previously encountered the pathogen. Remarkably, this inherited aversive behavior lasted for four generations, but in the fifth generation the worms were once again attracted to Pseudomonas. In another surprise, the researchers observed that inheritance of learned avoidance was not universal for all pathogenic bacteria; although mother worms could learn to avoid the pathogenic bacterium Serratia marcescens, which is less abundant than Pseudomonas in C. elegans' environment, this aversion was not passed down to offspring. Intrigued, the researchers set out to explore what controls transmission of P. aeruginosa avoidance behavior across generations.</p>
<p>The authors showed that C. elegans mothers must actually become ill from ingesting P. aeruginosa in order to transmit avoidance to future generations; exposure to odors emitted by the pathogen wasn't sufficient to provoke avoidance. Nonetheless, neuronal sensory pathways are important for inherited avoidance, because avoidance behavior in both mothers and their progeny was associated with upregulated expression of several neuronally-associated genes. Among these, elevated expression of the TGF-β ligand daf -7 in mothers was needed for progeny to inherit pathogen aversion. Moore and her colleagues found that daf-7 expression in a certain type of sensory neuron, ASI neurons, correlated strongly with inherited avoidance behavior.</p>
<p>&quot;The process of inheriting this learned avoidance [also] requires the activity of small RNAs called piRNA,&quot; Murphy said. piRNAs have been implicated in other transgenerational epigenetic inheritance pathways in C. elegans, where they're thought to silence gene expression and indirectly regulate DNA packing. The researchers found that the piRNA-associated protein PRG-1, while not necessary for C. elegans mothers to learn avoidance of P. aeruginosa, was required for increased daf-7 expression in progeny, and for their inherited avoidance behavior. Whether piRNAs and PRG-1 operate primarily in the mother, the progeny, or both to promote inheritance of avoidance behavior isn't yet known.</p>
<p>Importantly, expression of daf-7 remains elevated in the ASI neurons of progeny for four generations, then returns to basal levels in the fifth generation, which is when the inherited avoidance behavior also disappears. As Murphy points out, although inheritance of avoidance behavior provides a survival advantage, it's also necessary for this avoidance behavior to eventually go away. That's because P. aeruginosa is only pathogenic at high temperatures; at lower temperatures, it's increasingly safe to eat, as are other Pseudomonas species. If the pathogenic threat is temporary, the eventual lapsing of inherited avoidance allows future generations to return to feasting on nutritious Pseudomonas.</p>
</li>
<li>
<p>•C. elegans transmit learned avoidance of P. aeruginosa for four generations<br />
•ASI and the TGF-β ligand daf-7 are required for transgenerational PA14 avoidance<br />
•Piwi/PRG-1 is required for transgenerational inheritance of P. aeruginosa avoidance<br />
•Transgenerational avoidance of P. aeruginosa provides fitness benefits to offspring</p>
<p>The ability to inherit learned information from parents could be evolutionarily beneficial, enabling progeny to better survive dangerous conditions. We discovered that, after C. elegans have learned to avoid the pathogenic bacteria Pseudomonas aeruginosa (PA14), they pass this learned behavior on to their progeny, through either the male or female germline, persisting through the fourth generation. Expression of the TGF-β ligand DAF-7 in the ASI sensory neurons correlates with and is required for this transgenerational avoidance behavior. Additionally, the Piwi Argonaute homolog PRG-1 and its downstream molecular components are required for transgenerational inheritance of both avoidance behavior and ASI daf-7 expression. Animals whose parents have learned to avoid PA14 display a PA14 avoidance-based survival advantage that is also prg-1 dependent, suggesting an adaptive response. Transgenerational epigenetic inheritance of pathogenic learning may optimize progeny decisions to increase survival in fluctuating environmental conditions.</p>
</li>
</ul>
</li>
<li>
<p><em>caenorhabditis elegans sperm carry a histone-based epigenetic memory of both spermatogenesis and oogenesis</em><br />
tomoko m. tabuchi et al. 2018<br />
<a href="http://doi.org/10.1038/s41467-018-06236-8">doi.org/10.1038/s41467-018-06236-8</a></p>
<ul>
<li>Paternal contributions to epigenetic inheritance are not well understood. Paternal contributions via marked nucleosomes are particularly understudied, in part because sperm in some organisms replace the majority of nucleosome packaging with protamine packaging. Here we report that in Caenorhabditis elegans sperm, the genome is packaged in nucleosomes and carries a histone-based epigenetic memory of genes expressed during spermatogenesis, which unexpectedly include genes well known for their expression during oogenesis. In sperm, genes with spermatogenesis-restricted expression are uniquely marked with both active and repressive marks, which may reflect a sperm-specific chromatin signature. We further demonstrate that epigenetic information provided by sperm is important and in fact sufficient to guide proper germ cell development in offspring. This study establishes one mode of paternal epigenetic inheritance and offers a potential mechanism for how the life experiences of fathers may impact the development and health of their descendants.</li>
</ul>
</li>
<li>
<p><em>maternal h3k27me3 controls dna methylation-independent imprinting</em><br />
azusa inoue, lan jiang, falong lu, tsukasa suzuki, yi zhang 2017<br />
<a href="http://doi.org/10.1038/nature23262">dx.doi.org/10.1038/nature23262</a></p>
<ul>
<li>Mammalian sperm and oocytes have different epigenetic landscapes and are organized in different fashions. After fertilization, the initially distinct parental epigenomes become largely equalized with the exception of certain loci, including imprinting control regions. How parental chromatin becomes equalized and how imprinting control regions escape from this reprogramming is largely unknown. Here we profile parental allele-specific DNase I hypersensitive sites in mouse zygotes and morula embryos, and investigate the epigenetic mechanisms underlying these allelic sites. Integrated analyses of DNA methylome and tri-methylation at lysine 27 of histone H3 (H3K27me3) chromatin immunoprecipitation followed by sequencing identify 76 genes with paternal allele-specific DNase I hypersensitive sites that are devoid of DNA methylation but harbour maternal allele-specific H3K27me3. Interestingly, these genes are paternally expressed in preimplantation embryos, and ectopic removal of H3K27me3 induces maternal allele expression. H3K27me3-dependent imprinting is largely lost in the embryonic cell lineage, but at least five genes maintain their imprinted expression in the extra-embryonic cell lineage. The five genes include all paternally expressed autosomal imprinted genes previously demonstrated to be independent of oocyte DNA methylation. Thus, our study identifies maternal H3K27me3 as a DNA methylation-independent imprinting mechanism.</li>
</ul>
</li>
<li>
<p><em>parent-of-origin dna methylation dynamics during mouse development</em><br />
yonatan stelzer et al. 2016<br />
<a href="http://doi.org/10.1016/j.celrep.2016.08.066">doi.org/10.1016/j.celrep.2016.08.066</a></p>
<ul>
<li>
<p>•In vivo tracing of parent-specific DNA methylation dynamics at single-cell resolution<br />
•Cell-type-specific methylation signatures at the Dlk-Dio3 IG-DMR during development<br />
•Dynamic parent- and cell-type-specific DNA methylation changes in the adult brain</p>
<p>Parent-specific differentially methylated regions (DMRs) are established during gametogenesis and regulate parent-specific expression of imprinted genes. Monoallelic expression of imprinted genes is essential for development, suggesting that imprints are faithfully maintained in embryos and adults. To test this hypothesis, we targeted a reporter for genomic methylation to the imprinted Dlk1-Dio3 intergenic DMR (IG-DMR) to assess the methylation of both parental alleles at single-cell resolution. Biallelic gain or loss of IG-DMR methylation occurred in a small fraction of mouse embryonic stem cells, significantly affecting developmental potency. Mice carrying the reporter in either parental allele showed striking parent-specific changes in IG-DMR methylation, causing substantial and consistent tissue- and cell-type-dependent signatures in embryos and postnatal animals. Furthermore, dynamics in DNA methylation persisted during adult neurogenesis, resulting in inter-individual diversity. This substantial cell-cell DNA methylation heterogeneity implies that dynamic DNA methylation variations in the adult may be of functional importance.</p>
</li>
</ul>
</li>
<li>
<p><em>krab zinc-finger proteins contribute to the evolution of gene regulatory networks</em><br />
michaël imbeault, pierre-yves helleboid, didier trono 2017<br />
<a href="http://doi.org/10.1038/nature21683">doi.org/10.1038/nature21683</a></p>
<ul>
<li>The human genome encodes some 350 Krüppel-associated box (KRAB) domain-containing zinc-finger proteins (KZFPs), the products of a rapidly evolving gene family that has been traced back to early tetrapods<a href="http://www.nature.com/nature/journal/vaop/ncurrent/full/nature21683.html#ref1">80</a>, <a href="http://www.nature.com/nature/journal/vaop/ncurrent/full/nature21683.html#ref2">81</a>. The function of most KZFPs is unknown, but a few have been demonstrated to repress transposable elements in embryonic stem (ES) cells by recruiting the transcriptional regulator TRIM28 and associated mediators of histone H3 Lys9 trimethylation (H3K9me3)-dependent heterochromatin formation and DNA methylation<a href="http://www.nature.com/nature/journal/vaop/ncurrent/full/nature21683.html#ref3">82</a>, <a href="http://www.nature.com/nature/journal/vaop/ncurrent/full/nature21683.html#ref4">83</a>, <a href="http://www.nature.com/nature/journal/vaop/ncurrent/full/nature21683.html#ref5">84</a>, <a href="http://www.nature.com/nature/journal/vaop/ncurrent/full/nature21683.html#ref6">85</a>, <a href="http://www.nature.com/nature/journal/vaop/ncurrent/full/nature21683.html#ref7">86</a>, <a href="http://www.nature.com/nature/journal/vaop/ncurrent/full/nature21683.html#ref8">87</a>, <a href="http://www.nature.com/nature/journal/vaop/ncurrent/full/nature21683.html#ref9">88</a>. Depletion of TRIM28 in human or mouse ES cells triggers the upregulation of a broad range of transposable elements<a href="http://www.nature.com/nature/journal/vaop/ncurrent/full/nature21683.html#ref4">83</a>, <a href="http://www.nature.com/nature/journal/vaop/ncurrent/full/nature21683.html#ref10">89</a>, <a href="http://www.nature.com/nature/journal/vaop/ncurrent/full/nature21683.html#ref11">90</a>, and recent data based on a few specific examples have pointed to an arms race between hosts and transposable elements as an important driver of KZFP gene selection<a href="http://www.nature.com/nature/journal/vaop/ncurrent/full/nature21683.html#ref5">84</a>. Here, to obtain a global view of this phenomenon, we combined phylogenetic and genomic studies to investigate the evolutionary emergence of KZFP genes in vertebrates and to identify their targets in the human genome. First, we unexpectedly reassigned the root of the family to a common ancestor of coelacanths and tetrapods. Second, although we confirmed that the majority of KZFPs bind transposable elements and pinpoint cases of ongoing co-evolution, we found that most of their transposable element targets have lost all transposition potential. Third, by examining the interplay between human KZFPs and other transcriptional modulators, we obtained evidence that KZFPs exploit evolutionarily conserved fragments of transposable elements as regulatory platforms long after the arms race against these genetic invaders has ended. Together, our results demonstrate that KZFPs partner with transposable elements to build a largely species-restricted layer of epigenetic regulation.</li>
</ul>
</li>
<li>
<p><em>germ line-inherited h3k27me3 restricts enhancer function during maternal-to-zygotic transition</em><br />
zenk f et al. 2017<br />
<a href="http://doi.org/10.1126/science.aam5339">doi.org/10.1126/science.aam5339</a></p>
<ul>
<li>Gametes carry parental genetic material to the next generation. Stress-induced epigenetic changes in the germ line can be inherited and can have a profound impact on offspring development. However, the molecular mechanisms and consequences of transgenerational epigenetic inheritance are poorly understood. We found that <em>Drosophila</em> oocytes transmit the repressive histone mark H3K27me3 to their offspring. Maternal contribution of the histone methyltransferase Enhancer of zeste, the enzymatic component of Polycomb repressive complex 2, is required for active propagation of H3K27me3 during early embryogenesis. H3K27me3 in the early embryo prevents aberrant accumulation of the active histone mark H3K27ac at regulatory regions and precocious activation of lineage-specific genes at zygotic genome activation. Disruption of the germ line–inherited Polycomb epigenetic memory causes embryonic lethality that cannot be rescued by late zygotic reestablishment of H3K27me3. Thus, maternally inherited H3K27me3, propagated in the early embryo, regulates the activation of enhancers and lineage-specific genes during development.</li>
</ul>
</li>
<li>
<p><em>evaluation of a validated methylation triage signature for human papillomavirus positive women in the hpv focal cervical cancer screening trial</em><br />
darrel a. cook et al. 2018<br />
<a href="http://doi.org/10.1002/ijc.31976">doi.org/10.1002/ijc.31976</a></p>
<p>&quot;In contrast to what most researchers and clinicians are saying, we are seeing more and more evidence that it is in fact epigenetics, and not DNA mutations, that drives a whole range of early cancers, including cervical, anal, oropharyngeal, colon, and prostate.&quot;</p>
<ul>
<li>
<p>The study, published in the International Journal of Cancer, compared a new 'epigenetics-based' cervical cancer test with Pap smear and HPV tests, and investigated how well it predicted the development of cervical cancer up to five years in advance in a large study of women aged 25-65 in Canada.</p>
<p>As opposed to checking for patterns in the DNA genetic code itself that are indicative of the HPV virus, the new test looks at the naturally-occurring chemical markers that appear on top of the DNA, making up its 'epigenetic profile'.</p>
<p>'An enormous development'</p>
<p>Lead researcher Professor Attila Lorincz from Queen Mary University of London, who also helped develop the world's first test for HPV in 1988, said: &quot;This is an enormous development. We're not only astounded by how well this test detects cervical cancer, but it is the first time that anyone has proven the key role of epigenetics in the development of a major solid cancer using data from patients in the clinic. Epigenetic changes are what this cervical cancer test picks up and is exactly why it works so well.</p>
<p>&quot;In contrast to what most researchers and clinicians are saying, we are seeing more and more evidence that it is in fact epigenetics, and not DNA mutations, that drives a whole range of early cancers, including cervical, anal, oropharyngeal, colon, and prostate.&quot;</p>
<p>Screening to prevent cervical cancer is typically done through the Pap smear, which involves the collection, staining and microscopic examination of cells from the cervix. Unfortunately, the Pap smear can detect only around 50 per cent of cervical pre-cancers.</p>
<p>A much more accurate cervical screening method involves testing for the presence of DNA from the human papillomavirus (HPV) -- the primary but indirect cause of cervical cancer. There are estimated to be around 10 million women in the UK who are infected by HPV.</p>
<p>However, the HPV test only identifies whether or not women are infected with a cancer-causing HPV, but not their actual risks of cancer, which remain quite low. This causes unnecessary worry for the majority of HPV-infected women who receive a positive result but will eventually clear the virus and not develop the disease.</p>
<p>Predicting a person's risk of cervical cancer</p>
<p>The new test was significantly better than either the Pap smear or HPV test. It detected 100 per cent of the eight invasive cervical cancers that developed in the 15,744 women during the trial. In comparison, the Pap smear only detected 25 per cent of the cancers, and the HPV test detected 50 per cent.</p>
<p>The study also looked more closely at a subset of 257 HPV-positive women which were representatively selected from the large study. The new test detected 93 per cent of pre-cancerous lesions in those women, compared to 86 per cent detected using a combination of the Pap smear and HPV test, and 61 per cent detected using the Pap smear on its own.</p>
<p>Reducing the number of screening appointments needed</p>
<p>Professor Lorincz added: &quot;This really is a huge advance in how to deal with HPV-infected women and men, numbering in the billions worldwide, and it is going to revolutionise screening.</p>
<p>&quot;We were surprised by how well this new test can detect and predict early cervical cancers years in advance, with 100 per cent of cancers detected, including adenocarcinomas, which is a type of cervical cancer that is very difficult to detect. The new test is much better than anything offered in the UK at present but could take at least five years to be established.&quot;</p>
<p>The authors say that using this test in the clinic would reduce the number of visits to the doctor and screening appointments, as high-grade disease would be detected from the start. They also say that if it was fully implemented, it would be cheaper than the Pap smear.</p>
</li>
<li>
<p><em>abstract</em> Human papillomavirus (HPV)‐based cervical cancer screening requires triage of HPV positive women to identify those at risk of cervical intraepithelial neoplasia grade 2 (CIN2) or worse. We conducted a blinded case‐control study within the HPV FOCAL randomized cervical cancer screening trial of women aged 25‐65 to examine whether baseline methylation testing using the S5 classifier provided triage performance similar to an algorithm relying on cytology and HPV genotyping. Groups were randomly selected from 257 women with known HPV/cytology results and pathology outcomes. Group 1: 104 HPV positive (HPV+), abnormal cytology (54 CIN2/3; 50 &lt;CIN2); Group 2: 103 HPV+, normal cytology with HPV persistence at 12 mo. (53 CIN2/3; 50 &lt;CIN2); Group 3: 50 HPV+, normal cytology with HPV clearance at 12 mo. (assumed &lt;CIN2). For the combined groups, S5 risk score CIN2/3 relative sensitivity, specificity and positive predictive value (PPV) were compared with other triage approaches. Methylation showed a highly significant increasing trend with disease severity. For CIN3, S5 relative sensitivity and specificity were: 93.2% (95%CI: 81.4‐98.0) and 41.8% (35.2‐48.8), compared to 86.4% (75.0‐95.7) and 49.8% (43.1‐56.6) respectively for combined abnormal cytology/HPV16/18 positivity (differences not significant); adjusted PPVs were 18.2% (16.2‐20.4) and 19.3% (16.6‐22.2) respectively. S5 was also positive in baseline specimens from eight cancers detected during or after trial participation. The S5 methylation score had high sensitivity and PPV for CIN3, compatible with US and European thresholds for colposcopy referral. Methylation signatures can identify most HPV positive women at increased risk of cervical cancer from their baseline screening specimens.</p>
</li>
</ul>
</li>
<li>
<p><em>does gender leave an epigenetic imprint on the brain?</em><br />
laura r. cortes et al. 2019<br />
<a href="http://doi.org/10.3389/fnins.2019.00173">doi.org/10.3389/fnins.2019.00173</a></p>
<ul>
<li>
<p>Though the terms 'sex' and 'gender' are often used interchangeably by the average person, for neuroscientists, they mean different things, Forger said.</p>
<p>&quot;Sex is based on biological factors such as sex chromosomes and gonads [reproductive organs],&quot; she said, &quot;whereas gender has a social component and involves expectations and behaviors based on an individual's perceived sex.&quot;</p>
<p>These behaviors and expectations around gender identity can be seen in &quot;epigenetic marks&quot; in the brain, which drive biological functions and features as diverse as memory, development and disease susceptibility. Forger explained that epigenetic marks help determine which genes are expressed and are sometimes passed on from cell to cell as they divide. They also can be passed down from one generation to the next, she said.</p>
<p>&quot;While we are accustomed to thinking about differences between the brains of males and females, we are much less used to thinking about the biological implications of gender identity,&quot; she said. &quot;There is now sufficient evidence to suggest that an epigenetic imprint for gender is a logical conclusion. It would be strange if this were not the case, because all environmental influences of any importance can epigenetically change the brain.&quot;</p>
<p>Forger, with doctoral student Laura Cortes and post-doctoral researcher Carla Daniela Cisternas, reviewed previous studies of epigenetics and sexual differentiation in rodents, along with new studies in which gendered experiences among humans have also been associated with changes in the brain.</p>
<p>In one example involving rats, the Georgia State authors cited a study by University of Wisconsin researchers who gave female rat pups extra attention designed to simulate the increased licking that mother rats normally perform on their male offspring. That treatment led to detectible changes in the brains of the female rats that received extra stimulation as compared to those who got the normal level of attention for female pups.</p>
<p>Among the studies involving humans, researchers considered the example of Chinese society during the Great Chinese Famine from 1959-1961, when many families preferred to spend their limited resources on boys, leading to higher rates of disability and illiteracy among female survivors in adulthood. This demonstrates, they said, that early life stress can be a gendered experience as it changes the neural epigenome.</p>
<p>&quot;Given our lifetimes of layered gendered experiences, and their inevitable, iterative interactions with sex, it may never be possible to completely disentangle the effects of sex and gender on the human brain,&quot; Forger said. &quot;We can start, however, by including gender in our thinking any time a difference between the brain functioning of men and women is reported.&quot;</p>
</li>
<li>
<p><em>abstract</em> The words “sex” and “gender” are often used interchangeably in common usage. In fact, the Merriam-Webster dictionary offers “sex” as the definition of gender. The authors of this review are neuroscientists, and the words “sex” and “gender” mean very different things to us: sex is based on biological factors such as sex chromosomes and gonads, whereas gender has a social component and involves differential expectations or treatment by conspecifics, based on an individual’s perceived sex. While we are accustomed to thinking about “sex” and differences between males and females in epigenetic marks in the brain, we are much less used to thinking about the biological implications of gender. Nonetheless, careful consideration of the field of epigenetics leads us to conclude that gender must also leave an epigenetic imprint on the brain. Indeed, it would be strange if this were not the case, because all environmental influences of any import can epigenetically change the brain. In the following pages, we explain why there is now sufficient evidence to suggest that an epigenetic imprint for gender is a logical conclusion. We define our terms for sex, gender, and epigenetics, and describe research demonstrating sex differences in epigenetic mechanisms in the brain which, to date, is mainly based on work in non-human animals. We then give several examples of how gender, rather than sex, may cause the brain epigenome to differ in males and females, and finally consider the myriad of ways that sex and gender interact to shape gene expression in the brain.</p>
</li>
</ul>
</li>
</ul>
</li>
<li>
<p>?</p>
<ul>
<li>
<p><em>a framework for parsing heritable information</em><br />
antony m. jose 2020<br />
<a href="http://doi.org/10.1098/rsif.2020.0154">doi.org/10.1098/rsif.2020.0154</a></p>
<ul>
<li>
<p>DNA is just the ingredient list, not the set of instructions used to build and maintain a living organism. The instructions, he says, are much more complicated, and they're stored in the molecules that regulate a cell's DNA and other functioning systems.<br />
Jose outlined a new theoretical framework for heredity, which was developed through 20 years of research on genetics and epigenetics, in peer-reviewed papers in the <em>Journal of the Royal Society Interface</em> and the journal <em>BioEssays</em>. Both papers were published on April 22, 2020.<br />
Jose's argument suggests that scientists may be overlooking important avenues for studying and treating hereditary diseases, and current beliefs about evolution may be overly focused on the role of the genome, which contains all of an organism's DNA.<br />
&quot;DNA cannot be seen as the 'blueprint' for life,&quot; Jose said. &quot;It is at best an overlapping and potentially scrambled list of ingredients that is used differently by different cells at different times.&quot;<br />
For example, the gene for eye color exists in every cell of the body, but the process that produces the protein for eye color only occurs during a specific stage of development and only in the cells that constitute the colored portion of the eyes. That information is not stored in the DNA.</p>
<p>In addition, scientists are unable to determine the complex shape of an organ such as an eye, or that a creature will have eyes at all, by reading the creature's DNA. These fundamental aspects of anatomy are dictated by something outside of the DNA.<br />
Jose argues that these aspects of development, which enable a fertilized egg to grow from a single cell into a complex organism, must be seen as an integral part of heredity. Jose's new framework recasts heredity as a complex, networked information system in which all the regulatory molecules that help the cell to function can constitute a store of hereditary information.<br />
Michael Levin, a professor of biology and director of the Tufts Center for Regenerative and Developmental Biology and the Allen Discovery Center at Tufts University, believes Jose's approach could help answer many questions not addressed by the current genome-centric view of biology. Levin was not involved with either of the published papers.<br />
&quot;Understanding the transmission, storage and encoding of biological information is a critical goal, not only for basic science but also for transformative advances in regenerative medicine,&quot; Levin said. &quot;In these two papers, Antony Jose masterfully applies a computer science approach to provide an overview and a quantitative analysis of possible molecular dynamics that could serve as a medium for heritable information.&quot;<br />
Jose proposes that instructions not coded in the DNA are contained in the arrangement of the molecules within cells and their interactions with one another. This arrangement of molecules is preserved and passed down from one generation to the next.</p>
<p>In his papers, Jose's framework recasts inheritance as the combined effects of three components: entities, sensors and properties.<br />
Entities include the genome and all the other molecules within a cell that are needed to build an organism. Entities can change over time, but they are recreated with their original structure, arrangement and interactions at the start of each generation.<br />
&quot;That aspect of heredity, that the arrangement of molecules is similar across generations, is deeply underappreciated, and it leads to all sorts of misunderstandings of how heredity works,&quot; Jose said.<br />
Sensors are specific entities that interact with and respond to other entities or to their environment. Sensors respond to certain properties, such as the arrangement of a molecule, its concentration in the cell or its proximity to another molecule.<br />
Together, entities, sensors and properties enable a living organism to sense or 'know' things about itself and its environment. Some of this knowledge is used along with the genome in every generation to build an organism.<br />
&quot;This framework is built on years of experimental research in many labs, including ours, on epigenetics and multi-generational gene silencing combined with our growing interest in theoretical biology,&quot; Jose said. &quot;Given how two people who contract the same disease do not necessarily show the same symptoms, we really need to understand all the places where two people can be different -- not just their genomes.&quot;<br />
The folly of maintaining a genome-centric view of heredity, according to Jose, is that scientists may be missing opportunities to combat heritable diseases and to understand the secrets of evolution.<br />
In medicine, for instance, research into why hereditary diseases affect individuals differently focuses on genetic differences and on chemical or physical differences in entities. But this new framework suggests researchers should be looking for non-genetic differences in the cells of individuals with hereditary diseases, such as the arrangement of molecules and their interactions. Scientists don't currently have methods to measure some of these things, so this work points to potentially important new avenues for research.<br />
In evolution, Jose's framework suggests that organisms could evolve through changes in the arrangement of molecules without changes in their DNA sequence. And in conservation science, this work suggests that attempts to preserve endangered species through DNA banks alone are missing critical information stored in non-DNA molecules.</p>
</li>
<li>
<p><em>abstract</em> Living systems transmit heritable information using the replicating gene sequences and the cycling regulators assembled around gene sequences. Here, I develop a framework for heredity and development that includes the cycling regulators parsed in terms of what an organism can sense about itself and its environment by defining entities, their sensors and the sensed properties. Entities include small molecules (ATP, ions, metabolites, etc.), macromolecules (individual proteins, RNAs, polysaccharides, etc.) and assemblies of molecules. While concentration may be the only relevant property measured by sensors for small molecules, multiple properties that include concentration, sequence, conformation and modification may all be measured for macromolecules and assemblies. Each configuration of these entities and sensors that is recreated in successive generations in a given environment thus specifies a potentially vast amount of information driving complex development in each generation. This entity–sensor–property framework explains how sensors limit the number of distinguishable states, how distinct molecular configurations can be functionally equivalent and how regulation of sensors prevents detection of some perturbations. Overall, this framework is a useful guide for understanding how life evolves and how the storage of information has itself evolved with complexity since before the origin of life.</p>
</li>
</ul>
</li>
<li>
<p><em>heritable epigenetic changes alter transgenerational waveforms maintained by cycling stores of information</em><br />
antony m. jose 2020<br />
<a href="http://doi.org/10.1002/bies.201900254">doi.org/10.1002/bies.201900254</a></p>
<ul>
<li><em>abstract</em> Our view of heredity can potentially be distorted by the ease of introducing heritable changes in the replicating gene sequences but not in the cycling assembly of regulators around gene sequences. Here, key experiments that have informed the understanding of heredity are reinterpreted to highlight this distortion and the possible variety of heritable changes are considered. Unlike heritable genetic changes, which are always associated with mutations in gene sequence, heritable epigenetic changes can be associated with physical or chemical changes in molecules or only changes in the system. The transmission of cycling stores along the continuous lineage of cells that connects successive generations creates waves of activity and localization of the molecules that together form the cell code for development in each generation. As a result, heritable epigenetic changes can include any that can alter a wave such as changes in form, midline, frequency, amplitude, or phase. Testing this integrated view of all heritable information will require the concerted application of multiple experimental approaches across generations.</li>
</ul>
</li>
<li>
<p>epigenetic-like transmission in archaea<br />
<em>nonmutational mechanism of inheritance in the archaeon sulfolobus solfataricus</em><br />
sophie payne et al. 2018<br />
<a href="http://doi.org/10.1073/pnas.1808221115">doi.org/10.1073/pnas.1808221115</a></p>
<ul>
<li>
<p>Species most often evolve through mutations in DNA that get inherited by successive generations. A few decades ago, researchers began discovering that multicellular species can also evolve through epigenetics: traits originating not from genetic changes but from the inheritance of cellular proteins that control access to an organism's DNA.</p>
<p>Because those proteins can respond to shifts in an organism's environment, epigenetics resides on the ever-thin border between nature and nurture. Evidence for it had emerged only in eukaryotes, the multicellular domain of life that comprises animals, plants and several other kingdoms.</p>
<p>But a series of experiments from Nebraska's Sophie Payne, Paul Blum and colleagues has shown that epigenetics can pass along extreme acid resistance in a species of archaea: microscopic, single-celled organisms that share features with both eukaryotes and bacteria.</p>
<p>&quot;The surprise is that it's in these relatively primitive organisms, which we know to be ancient,&quot; said Blum, Charles Bessey Professor of biological sciences at Nebraska. &quot;We've been thinking about this as something (evolutionarily) new. But epigenetics is not a newcomer to the planet.&quot;</p>
<p>The team discovered the phenomenon in Sulfolobus solfataricus, a sulfur-eating species that thrives in the boiling, vinegar-acidic springs of Yellowstone National Park. By exposing the species to increasing levels of acidity over several years, the researchers evolved three strains that exhibited a resistance 178 times greater than that of their Yellowstone ancestors.</p>
<p>One of those strains evolved the resistance despite no mutations in its DNA, while the other two underwent mutations in mutually exclusive genes that do not contribute to acid resistance. And when the team disrupted the proteins thought to control the expression of resistance-relevant genes -- leaving the DNA itself untouched -- that resistance abruptly disappeared in subsequent generations.</p>
<p>&quot;We predicted that they'd be mutated, and we'd follow the mutations, and that would teach us what caused the extreme acid resistance,&quot; Blum said. &quot;But that's not what we found.&quot;</p>
<p>Though epigenetics is essential to some of the most productive and destructive physiological processes in humans -- the differentiation of cells into roughly 200 types, the occurrence of cancers -- it remains difficult to study in eukaryotes.</p>
<p>The simplicity of archaea, combined with the fact that their cells resemble eukaryotes' in some important ways, should allow researchers to investigate epigenetic questions much faster and more cheaply than was possible before, Blum said.</p>
<p>&quot;We don't know what flips the switch in humans that changes epigenetic traits,&quot; Blum said. &quot;And we sure don't know how to reverse it very often. That's the first thing we'll go after: how to turn it on, how to turn it off, how to get it to switch. And that has benefits when you think about (managing) traits in us or traits in plants.&quot;</p>
<p>Yet the discovery also raises questions, Payne said, especially about how both eukaryotes and archaea came to adopt epigenetics as a method of inheritance.</p>
<p>&quot;Maybe both of them had it because they diverged from a common ancestor that had it,&quot; said Payne, a doctoral student in biological sciences. &quot;Or maybe it evolved twice. It's a really interesting concept from an evolutionary perspective.&quot;</p>
<p>Blum said the team is likewise curious about whether and how epigenetics might explain why no known archaea cause disease or wage antibiotic-armed warfare against their brethren, as bacteria do.</p>
<p>&quot;There are no antibiotics going on in that world,&quot; he said. &quot;Why is that? We're thinking (that) it's got something to do with epigenetics, and so their interactions among each other are fundamentally different than bacteria.&quot;</p>
<p>The discovery also introduces an even broader question, Blum said.</p>
<p>&quot;What was the benefit for them to have this? We don't know.&quot;</p>
</li>
<li>
<p><em>abstract</em> Archaea have considerable importance in ecology and evolution and have emerging roles in health. However, many of their cellular processes are under active study. Archaea are thought to acquire and inherit adaptive traits solely through mutation. Here, adaptive laboratory evolution of an extremophile trait revealed that an alternative nonmutational process was operative. When genes whose expression had been altered in a heritable manner were replaced by recombination using identical DNA, the evolved traits were changed. This implicated a regulatory role for chromatin proteins and was consistent with an epigenetic-like regulatory mechanism. This finding has evolutionary relevance for the origin of epigenetics, transcriptional regulation, and functional genome architecture.</p>
<p>Epigenetic phenomena have not yet been reported in archaea, which are presumed to use a classical genetic process of heritability. Here, analysis of independent lineages of Sulfolobus solfataricus evolved for enhanced fitness implicated a non-Mendelian basis for trait inheritance. The evolved strains, called super acid-resistant Crenarchaeota (SARC), acquired traits of extreme acid resistance and genome stability relative to their wild-type parental lines. Acid resistance was heritable because it was retained regardless of extensive passage without selection. Despite the hereditary pattern, in one strain, it was impossible for these SARC traits to result from mutation because its resequenced genome had no mutation. All strains also had conserved, heritable transcriptomes implicated in acid resistance. In addition, they had improved genome stability with absent or greatly decreased mutation and transposition relative to a passaged control. A mechanism that would confer these traits without DNA sequence alteration could involve posttranslationally modified archaeal chromatin proteins. To test this idea, homologous recombination with isogenic DNA was used to perturb native chromatin structure. Recombination at up-regulated loci from the heritable SARC transcriptome reduced acid resistance and gene expression in the majority of recombinants. In contrast, recombination at a control locus that was not part of the heritable transcriptome changed neither acid resistance nor gene expression. Variation in the amount of phenotypic and expression changes across individuals was consistent with Rad54-dependent chromatin remodeling that dictated crossover location and branch migration. These data support an epigenetic model implicating chromatin structure as a contributor to heritable traits.</p>
</li>
</ul>
</li>
<li>
<p>plant cell polarity is inherited from the mother, but exactly how is still unknown<br />
<em>mechanistic framework for cell-intrinsic re-establishment of pin2 polarity after cell division</em><br />
matouš glanc et al. 2018<br />
<a href="http://doi.org/10.1038/s41477-018-0318-3">doi.org/10.1038/s41477-018-0318-3</a></p>
<ul>
<li>
<p>The directional transport of the hormone auxin sets up polarization in plants, but this transport in turn depends on the polar distribution of PIN auxin transporters in each cell. This means that every single cell has to be polarly organized for the plant to distinguish up from down. Cell division, however, sets a challenge: At each division, trafficking of polar membrane proteins, such as PIN auxin transporters, is redirected to both newly formed membranes. Therefore, the PIN auxin transporter polarity is lost in one of the daughter cells after each division. How correct polarity is set up again was unknown. Using a new transgenic Arabidopsis plant line, in which fluorescent PIN auxin transporters can be followed exclusively in dividing cells, the researchers followed in real time what happens to PIN proteins and their polarity during cell division.</p>
<p>What they found was surprising, says first author Matouš Glanc. &quot;We thought that cells would need to communicate with their neighbors to correctly re-establish polarity. So we first looked for a signal that would be sent between cells. But we found no such thing. Instead, we found that polarity is communicated by the mother cell.&quot; Exactly how mother cells 'tell' their daughter cells where up and down are is not yet known, Glanc adds. &quot;We know that polarity information is not conveyed by signaling from neighbors, but is inherited from the mother cell -- we are still trying to understand how.&quot;</p>
<p>The researchers also found that endocytosis, which removes proteins from the cell surface, is crucial for this polarity re-establishment. Previously, it was thought that PIN auxin transporters that end up on the &quot;wrong&quot; cell side after division get removed by endocytosis and shuttled to the correct side. In the paper, the researchers show that instead of being ferried around, the wrongly placed transporters are endocytosed and destroyed. New PIN transporters are made and inserted in the correct side of the cell membrane.</p>
<p>A group of kinases, PINOID and its homologues WAG1 and WAG2, modify PIN transporters through a chemical reaction called phosphorylation, and are also crucial for determining their polarity. Plants in which all three kinases are no longer functional are unable to re-establish polarity after cell division. In these mutants, we see what happens when plants get polarity wrong: the roots don't grow down into the soil along gravity, but wave and turn instead.</p>
<p>While the study has provided some crucial answers as to how polarity is re-established, more questions remain open, says Glanc. &quot;We have identified endocytosis und phosphorylation as key steps in polarity establishment, and we have shown that polarity is inherited from the mother. But we still need to find the nature of the inherited information. It is something inherent to the cells, but what protein, lipid or sugar is involved remains to be seen.&quot;</p>
</li>
<li>
<p><em>abstract</em> Cell polarity, manifested by the localization of proteins to distinct polar plasma membrane domains, is a key prerequisite of multicellular life. In plants, PIN auxin transporters are prominent polarity markers crucial for a plethora of developmental processes. Cell polarity mechanisms in plants are distinct from other eukaryotes and still largely elusive. In particular, how the cell polarities are propagated and maintained following cell division remains unknown. Plant cytokinesis is orchestrated by the cell plate—a transient centrifugally growing endomembrane compartment ultimately forming the cross wall1. Trafficking of polar membrane proteins is typically redirected to the cell plate, and these will consequently have opposite polarity in at least one of the daughter cells2,3,4,5. Here, we provide mechanistic insights into post-cytokinetic re-establishment of cell polarity as manifested by the apical, polar localization of PIN2. We show that the apical domain is defined in a cell-intrinsic manner and that re-establishment of PIN2 localization to this domain requires de novo protein secretion and endocytosis, but not basal-to-apical transcytosis. Furthermore, we identify a PINOID-related kinase WAG1, which phosphorylates PIN2 in vitro6 and is transcriptionally upregulated specifically in dividing cells, as a crucial regulator of post-cytokinetic PIN2 polarity re-establishment.</p>
</li>
</ul>
</li>
<li>
<p><em>a heterochromatin-dependent transcription machinery drives piRNA expression</em><br />
peter refsing andersen et al. 2017<br />
<a href="http://doi.org/10.1038/nature23482">doi.org/10.1038/nature23482</a></p>
<ul>
<li>Nuclear small RNA pathways safeguard genome integrity by establishing transcription-repressing heterochromatin at transposable elements. This inevitably also targets the transposon-rich source loci of the small RNAs themselves. How small RNA source loci are efficiently transcribed while transposon promoters are potently silenced is not understood. Here we show that, in <em>Drosophila</em>, transcription of PIWI-interacting RNA (piRNA) clusters—small RNA source loci in animal gonads—is enforced through RNA polymerase II pre-initiation complex formation within repressive heterochromatin. This is accomplished through Moonshiner, a paralogue of a basal transcription factor IIA (TFIIA) subunit, which is recruited to piRNA clusters via the heterochromatin protein-1 variant Rhino. Moonshiner triggers transcription initiation within piRNA clusters by recruiting the TATA-box binding protein (TBP)-related factor TRF2, an animal TFIID core variant. Thus, transcription of heterochromatic small RNA source loci relies on direct recruitment of the core transcriptional machinery to DNA via histone marks rather than sequence motifs, a concept that we argue is a recurring theme in evolution.</li>
</ul>
</li>
<li>
<p><em>early post-conception maternal cortisol, children’s hpaa activity and dna methylation profiles</em><br />
c. k. barha et al. 2018<br />
<a href="http://doi.org/10.1017/s2040174418000880">doi.org/10.1017/s2040174418000880</a></p>
<ul>
<li>
<p>Using urine samples to measure reproductive hormones, the researchers identified the day children were conceived, as well as the moms' cortisol levels, a biomarker of physiological stress, during the first eight weeks after conception.</p>
<p>Twelve years later, they studied how these children reacted to the start of a new school year (a well known &quot;natural&quot; stressor) and to a public-speaking challenge (a frequently used &quot;experimental&quot; stressor).</p>
<p>Maternal cortisol following conception was associated with different facets of the children's cortisol responses to those challenges, and many of these associations differed between boys and girls.</p>
<p>Study lead author Cindy Barha, who worked under Nepomnaschy as a doctoral student, reports that sons of mothers who had higher cortisol in gestational week two had higher cortisol reactions to the experimental public-speaking challenge, but this association was not observed in daughters. In contrast, mothers with higher cortisol in gestational week five had daughters with higher 'basal' cortisol before the start of a new school term, but not sons.</p>
<p>However, both sons and daughters had higher cortisol responses to the start of a new school term, and in response to the experimental public speaking challenge, if their mothers had higher cortisol during gestational week five. The biological mechanisms mediating these associations are not yet known, but are likely to involve genetics and epigenetics as well as environmental and cultural factors shared by moms and their children.</p>
<p>&quot;Stress plays a critical role not only in children's ability to respond to social and academic challenges, but also in their development and health as adults,&quot; says Nepomnaschy. The researchers will continue their investigations into the connection between maternal and child stress from the moment of conception onwards. The findings will help to develop successful programs and interventions that prepare children to live healthy and fulfilling lives and realize their full potential.</p>
</li>
<li>
<p><em>abstract</em> The hypothalamic–pituitary–adrenal axis (HPAA) plays a critical role in the functioning of all other biological systems. Thus, studying how the environment may influence its ontogeny is paramount to understanding developmental origins of health and disease. The early post-conceptional (EPC) period could be particularly important for the HPAA as the effects of exposures on organisms’ first cells can be transmitted through all cell lineages. We evaluate putative relationships between EPC maternal cortisol levels, a marker of physiologic stress, and their children’s pre-pubertal HPAA activity (n=22 dyads). Maternal first-morning urinary (FMU) cortisol, collected every-other-day during the first 8 weeks post-conception, was associated with children’s FMU cortisol collected daily around the start of the school year, a non-experimental challenge, as well as salivary cortisol responses to an experimental challenge (all Ps&lt;0.05), with some sex-related differences. We investigated whether epigenetic mechanisms statistically mediated these links and, therefore, could provide cues as to possible biological pathways involved. EPC cortisol was associated with &gt;5% change in children’s buccal epithelial cells’ DNA methylation for 867 sites, while children’s HPAA activity was associated with five CpG sites. Yet, no CpG sites were related to both, EPC cortisol and children’s HPAA activity. Thus, these epigenetic modifications did not statistically mediate the observed physiological links. Larger, prospective peri-conceptional cohort studies including frequent bio-specimen collection from mothers and children will be required to replicate our analyses and, if our results are confirmed, identify biological mechanisms mediating the statistical links observed between maternal EPC cortisol and children’s HPAA activity</p>
</li>
</ul>
</li>
</ul>
</li>
<li>
<p>epitranscriptome</p>
<ul>
<li>
<p><em>the epitranscriptome of noncoding rnas in cancer</em><br />
manel esteller, pier paolo pandolfi 2017<br />
<a href="http://doi.org/10.1158/2159-8290.CD-16-1292">doi.org/10.1158/2159-8290.CD-16-1292</a></p>
<ul>
<li>The activity of RNA is controlled by different types of post-transcriptional modifications, such as the addition of methyl groups and other chemical and structural changes, that have been recently described in human cells by high-throughput sequencing. Herein, we will discuss how the so-called epitranscriptome is disrupted in cancer and what the contribution of its writers, readers, and erasers to the process of cellular transformation is, particularly focusing on the epigenetic modifications of ncRNAs.</li>
</ul>
</li>
<li>
<p><em>single-cell analysis of experience-dependent transcriptomic states in the mouse visual cortex</em><br />
sinisa hrvatin et al. 2018<br />
<a href="http://doi.org/10.1038/s41593-017-0029-5">doi.org/10.1038/s41593-017-0029-5</a></p>
<ul>
<li>Activity-dependent transcriptional responses shape cortical function. However, a comprehensive understanding of the diversity of these responses across the full range of cortical cell types, and how these changes contribute to neuronal plasticity and disease, is lacking. To investigate the breadth of transcriptional changes that occur across cell types in the mouse visual cortex after exposure to light, we applied high-throughput single-cell RNA sequencing. We identified significant and divergent transcriptional responses to stimulation in each of the 30 cell types characterized, thus revealing 611 stimulus-responsive genes. Excitatory pyramidal neurons exhibited inter- and intralaminar heterogeneity in the induction of stimulus-responsive genes. Non-neuronal cells showed clear transcriptional responses that may regulate experience-dependent changes in neurovascular coupling and myelination. Together, these results reveal the dynamic landscape of the stimulus-dependent transcriptional changes occurring across cell types in the visual cortex; these changes are probably critical for cortical function and may be sites of deregulation in developmental brain disorders.</li>
</ul>
</li>
<li>
<p><em>a placental mammal‐specific microrna cluster acts as a natural brake for sociability in mice</em><br />
martin lackinger et al. 2018<br />
<a href="http://doi.org/10.15252/embr.201846429">doi.org/10.15252/embr.201846429</a></p>
<ul>
<li>
<p>Textbook knowledge holds that DNA is first copied to make messenger RNA molecules (mRNAs) that are then translated into protein. MicroRNAs are short snippets of RNA that do not code for a protein. Rather, they function mainly by regulating the stability or translation rate of mRNAs, thereby inhibiting the production of particular proteins. These microRNAs form a whole new layer of gene regulation that has only been uncovered in the past 15 years. Each microRNA typically targets hundreds of different mRNAs, making them ideal for coordinating complex cellular processes.</p>
<p>In their work at the University of Marburg, Germany and later on at ETH Zurich, Switzerland, the research group of Gerhard Schratt and other laboratories have uncovered that a cluster of 38 microRNAs, termed miR379-410, plays an important role in neural development. Moreover, several hints pointed to the possibility that miR379-410 is involved in social behaviour. Schratt and colleagues now investigated this option in more detail and found that miR379-410 indeed regulates sociability in the brain of mice. The study opens up a new perspective on the molecular mechanisms behind social behaviour.</p>
<p>The researchers first observed that mice lacking a functional miR379-410 complex were more sociable than their littermates, indicating that miR379-410 functions to restrict sociability in healthy animals. Further investigation showed that neurons in the hippocampus of the brain in mice lacking miR379-410 formed more connections and were more likely to transmit electrical signals. &quot;Our study indicates that miR379-410 plays an important role in the development of neural circuitries that control social behaviour,&quot; explains Schratt.</p>
<p>At the molecular level, the miR379-410 complex targets thousands of genes, among them many that were known to function in regulating synaptic transmission. Moreover, a small subgroup of only five microRNAs from the cluster might explain to a large extent the upregulation of key synaptic proteins. These proteins are involved in a process termed homeostatic synaptic downscaling -- a feedback loop that kicks in when the brain becomes overactive because synaptic contacts are too strong.</p>
<p>While the current study uses mice as a model organism, there are indications that the miR379-410 complex is also involved in the regulation of social behaviour in humans. For example, several miR-379-410 members are dysregulated in the blood and brain of patients with neurodevelopmental disorders that affect social behaviour, such as schizophrenia or autism spectrum disorder. &quot;We hope that our study will contribute to the development of treatments to ameliorate social deficits in neuropsychiatric conditions in the future,&quot; says Schratt.</p>
</li>
<li>
<p><em>abstract</em> Aberrant synaptic function is thought to underlie social deficits in neurodevelopmental disorders such as autism and schizophrenia. Although microRNAs have been shown to regulate synapse development and plasticity, their potential involvement in the control of social behaviour in mammals remains unexplored. Here, we show that deletion of the placental mammal‐specific miR379‐410 cluster in mice leads to hypersocial behaviour, which is accompanied by increased excitatory synaptic transmission, and exaggerated expression of ionotropic glutamate receptor complexes in the hippocampus. Bioinformatic analyses further allowed us to identify five “hub” microRNAs whose deletion accounts largely for the upregulation of excitatory synaptic genes observed, including Cnih2, Dlgap3, Prr7 and Src. Thus, the miR379‐410 cluster acts a natural brake for sociability, and interfering with specific members of this cluster could represent a therapeutic strategy for the treatment of social deficits in neurodevelopmental disorders.</p>
</li>
</ul>
</li>
</ul>
</li>
<li>
<p>epigenetic memory</p>
<ul>
<li>
<p><em>reconsidering plant memory: intersections between stress recovery, rna turnover, and epigenetics</em><br />
peter crisp et al. 2016<br />
<a href="http://doi.org/10.1126/sciadv.1501340">doi.org/10.1126/sciadv.1501340</a></p>
<ul>
<li>plant memory and forgetting</li>
<li>some plants also seem to have short-term memory not using DNA or RNA</li>
<li>“unlike animals, in <em>Arabidopsis</em>, DNA methylation can be inherited through meiotic cell divisions”</li>
</ul>
</li>
<li>
<p><em>pathogenic infection in male mice changes sperm small rna profiles and transgenerationally alters offspring behavior</em><br />
shiraz tyebji et al. 2020<br />
<a href="http://doi.org/10.1016/j.celrep.2020.107573">doi.org/10.1016/j.celrep.2020.107573</a></p>
<ul>
<li>
<p>Studying mice infected with the common parasite <em>Toxoplasma</em>, the team discovered that sperm of infected fathers carried an altered 'epigenetic' signature which impacted the brains of resulting offspring. Molecules in the sperm called 'small RNA' appeared to influence the offspring's brain development and behaviour.<br />
'Intergenerational inheritance' of similar epigenetic changes from men exposed to extreme trauma has been well documented. This latest research, published in Cell Reports, has raised the question of whether <em>Toxoplasma</em> infections -- or even possibly other infections -- in men before conception could impact the health of subsequent generations.<br />
The research was led by Walter and Eliza Hall Institute researchers Dr Shiraz Tyebji and Associate Professor Chris Tonkin, in collaboration with Professor Anthony Hannan at the Florey Institute of Neuroscience and Mental Health.<br />
<strong>At a glance</strong><br />
•    Melbourne researchers have revealed that mice conceived from sperm of males infected with the parasite <em>Toxoplasma</em> displayed changes in their brain function and behaviour.<br />
•    The study showed <em>Toxoplasma</em> infection in male mice caused changes in levels of 'small RNA' molecules contained in their sperm, potentially altering gene expression in the resulting offspring.<br />
•    As trauma in men can also cause epigenetic changes in sperm and has been associated with neuropsychiatric changes in their children and grandchildren, the research raises the possibility that men might also transmit changes associated with a <em>Toxoplasma</em> infection to their children.<br />
<strong>Infectious inheritance</strong><br />
<em>Toxoplasma</em> is one of the world's most common parasites, estimated to be carried by between 25 and 80 per cent of the global population. <em>Toxoplasma</em> infection can cause an initial mild illness in most people, however, pregnant women, babies and people with weakened immunity experience more severe infections.</p>
<p>Associate Professor Tonkin said people could carry the dormant <em>Toxoplasma</em> parasite for decades, and that this had been associated with the appearance of symptoms of mental disorders such as schizophrenia and bipolar disorder.<br />
&quot;<em>Toxoplasma</em> infections have been shown to cause long-term epigenetic changes in a range of cells around our body. These are changes that do not alter the genetic sequence of DNA, but influence gene expression -- that is, which genes are switched on or off,&quot; he said.<br />
&quot;As other epigenetic changes in fathers -- such as those caused by trauma or smoking -- can influence their children, we decided to look at whether the effects of epigenetic changes caused by <em>Toxoplasma</em> infection could also be passed between generations.&quot;<br />
By studying male mice infected with <em>Toxoplasma</em>, the researchers were able to narrow their investigations down to the transmission of epigenetic information through sperm, Dr Tyebji said.<br />
&quot;We discovered that <em>Toxoplasma</em> infection alters levels of DNA-like molecules, called small RNA, that are carried by sperm,&quot; he said. &quot;These changes in small RNA levels affect gene expression, and so could potentially influence brain development and behaviour of offspring.</p>
<p>&quot;We were stunned to see that even the next generation -- the 'grandchildren' of the original infected male -- displayed changes in their behaviour,&quot; Dr Tyebji said.<br />
<strong>Impacts for public health</strong><br />
Professor Hannan said this was the first time it had been shown that an infection in a male can result in epigenetic changes being transmitted to subsequent generations. &quot;While our studies were in mice, it raises an important question about whether infections in human fathers before conception also impact their children,&quot; he said.<br />
&quot;We normally think more about how infectious diseases in women affect the developing fetus, but perhaps certain infections in men could have long-term impacts on subsequent generations' health.<br />
&quot;This is certainly something we are following up, both looking at what is happening in humans, as well as investigating infections other than <em>Toxoplasma</em>, including animal models of infection with the SARS-CoV-2 virus which causes COVID-19,&quot; Professor Hannan said.</p>
</li>
<li>
<p><em>abstract</em> Germline epigenetic factors influence transgenerational inheritance of behavioral traits upon changes in experience and environment. Immune activation due to infection can also modulate brain function, but whether this experience can be passed down to offspring remains unknown. Here, we show that infection of the male lineage with the common human parasite Toxoplasma results in transgenerational behavioral changes in offspring in a sex-dependent manner. Small RNA sequencing of sperm reveals significant tran- scriptional differences of infected animals compared to controls. Zygote microinjection of total small RNA from sperm of infected mice partially recapitulates the behavioral phenotype of naturally born offspring, sug- gesting an epigenetic mechanism of behavioral inheritance in the first generation. Our results demonstrate that sperm epigenetic factors can contribute to intergenerational inheritance of behavioral changes after pathogenic infection, which could have major public health implications</p>
</li>
</ul>
</li>
</ul>
</li>
<li>
<p>protein/prion</p>
<ul>
<li>
<p><em>intrinsically disordered proteins drive emergence and inheritance of biological traits</em><br />
sohini chakrabortee et al. 2016<br />
<a href="http://doi.org/10.1016/j.cell.2016.09.017">doi.org/10.1016/j.cell.2016.09.017</a></p>
<ul>
<li>•Transient overexpression of many yeast proteins creates heritable biological traits<br />
•The traits have prion-like inheritance patterns and can involve gains of function<br />
•The traits are transmissible by protein only but do not arise from amyloid fibers<br />
•The traits arise from disordered protein sequences conserved from yeast to man<br />
Summary<br />
Prions are a paradigm-shifting mechanism of inheritance in which phenotypes are encoded by self-templating protein conformations rather than nucleic acids. Here, we examine the breadth of protein-based inheritance across the yeast proteome by assessing the ability of nearly every open reading frame (ORF; ∼5,300 ORFs) to induce heritable traits. Transient overexpression of nearly 50 proteins created traits that remained heritable long after their expression returned to normal. These traits were beneficial, had prion-like patterns of inheritance, were common in wild yeasts, and could be transmitted to naive cells with protein alone. Most inducing proteins were not known prions and did not form amyloid. Instead, they are highly enriched in nucleic acid binding proteins with large intrinsically disordered domains that have been widely conserved across evolution. Thus, our data establish a common type of protein-based inheritance through which intrinsically disordered proteins can drive the emergence of new traits and adaptive opportunities.</li>
</ul>
</li>
<li>
<p><em>aβ and tau prion-like activities decline with longevity in the alzheimer’s disease human brain</em><br />
atsushi aoyagi et al. 2019<br />
<a href="http://doi.org/10.1126/scitranslmed.aat8462">doi.org/10.1126/scitranslmed.aat8462</a></p>
<ul>
<li>
<p>Using novel laboratory tests, the researchers were able to detect and measure specific, self-propagating prion forms of the proteins amyloid beta (A-β) and tau in postmortem brain tissue of 75 Alzheimer's patients. In a striking finding, higher levels of these prions in human brain samples were strongly associated with early-onset forms of the disease and younger age at death.</p>
<p>Alzheimer's disease is currently defined based on the presence of toxic protein aggregations in the brain known as amyloid plaques and tau tangles, accompanied by cognitive decline and dementia. But attempts to treat the disease by clearing out these inert proteins have been unsuccessful. The new evidence that active A-β and tau prions could be driving the disease -- published May 1, 2019 in Science Translational Medicine -- could lead researchers to explore new therapies that focus on prions directly.</p>
<p>&quot;I believe this shows beyond a shadow of a doubt that amyloid beta and tau are both prions, and that Alzheimer's disease is a double-prion disorder in which these two rogue proteins together destroy the brain,&quot; said Stanley Prusiner, MD, the study's senior author and director of the UCSF Institute for Neurodegenerative Diseases, part of the UCSF Weill Institute for Neurosciences. &quot;The fact that prion levels also appear linked to patient longevity should change how we think about the way forward for developing treatments for the disease. We need a sea change in Alzheimer's disease research, and that is what this paper does. This paper might catalyze a major change in AD research.&quot;</p>
<p>What are Prions?</p>
<p>Prions are misfolded versions of a protein that can spread like an infection by forcing normal copies of that protein into the same self-propagating, misfolded shape. The original prion protein, PrP, was identified by Prusiner in the 1980s as the cause of Creutzfeldt Jakob Disease (CJD) and spongiform bovine encephalopathy, also known as Mad Cow Disease, which spread through consumption of meat and bone meal tainted with PrP prions. This was the first time a disease had been shown to infect people not by an infestation of an organism such as a bacterium or a virus, but through an infectious protein, and Prusiner received a Nobel Prize for that discovery in 1997.</p>
<p>Prusiner and colleagues have long suspected that PrP was not the only protein capable of acting as a self-propagating prion, and that distinct types of prion could be responsible for other neurodegenerative diseases caused by the progressive toxic buildup of misfolded proteins. For example, Alzheimer's disease is defined by A-β plaques and tau tangles that gradually spread destruction through the brain. Over the past decade, laboratory studies at UCSF and elsewhere have begun to show that amyloid plaques and tau tangles from diseased brains can infect healthy brain tissue much like PrP, but considerably more slowly.</p>
<p>Many scientists have been reluctant to accept that A-β and tau are self-propagating prions -- instead referring to their spread as &quot;prion-like&quot; -- because unlike PrP prions, they were not thought to be infectious except in highly controlled laboratory studies. However, recent reports have documented rare cases of patients treated with growth hormone derived from human brain tissue, or given transplants of the brain's protective dura mater, who went on to develop A-β plaques in middle age, long before they should be seen in anyone without a genetic disorder. Prusiner contends that these findings argue that both Aß and tau are prions even though they propagate more slowly than highly aggressive PrP prions.</p>
<p>In response to these debates, Prusiner likes to quote from a 1969 lecture by neuroscientist Bernard Katz: &quot;There is a type of scientist who, if given the choice, would rather use his colleague's toothbrush than his terminology!&quot;</p>
<p>Laboratory Bioassays Reveal Aß and Tau Prions in Human Postmortem Brain Samples</p>
<p>In the new study, the researchers combined two recently developed laboratory tests to rapidly measure prions in human tissue samples: a new A-β detection system developed in the Prusiner lab and a tau prion assay previously developed by Marc Diamond, PhD, a former UCSF faculty member who is now director of the Center for Alzheimer's and Neurodegenerative Diseases at UT Southwestern Medical Center.</p>
<p>Unlike earlier animal models that could take months to reveal the slow spread of A-β and/or tau prions, these cell-based assays measure infectious prion levels in just three days, enabling the researchers to effectively quantify for the first time the levels of both tau and A-β prions in processed extracts from post-mortem brain samples. In the new study, they applied the technique to autopsied brain tissue from over 100 individuals who had died of Alzheimer's disease and other forms of neurodegeneration, which was collected from repositories in the United States, Europe, and Australia.</p>
<p>In assays comparing the samples from Alzheimer's patients with those who died of other diseases, prion activity corresponded exactly with the distinctive protein pathology that has been established in each disease: in 75 Alzheimer's disease brains, both A-β and tau prion activity was elevated; in 11 samples from patients with cerebral amyloid angiopathy (CAA), only A-β prions were seen; and in 10 tau-linked frontotemporal lobar degeneration (FTLD) samples, only tau prions were detected. Another recently developed bioassay for alpha-synuclein prions only found these infectious particles in the seven samples from patients with the synuclein-linked degenerative disorder multiple system atrophy (MSA).</p>
<p>&quot;These assays are a game-changer,&quot; said co-author and protein chemist William DeGrado, PhD, a professor of pharmaceutical chemistry and member of the UCSF Cardiovascular Research Institute, who contributed to the design and analysis of the current study. &quot;Previously Alzheimer's research has been stuck looking at collateral damage in the form of misfolded, dead proteins that form plaques and tangles. Now it turns out that it is prion activity that correlates with disease, rather than the amount of plaques and tangles at the time of autopsy. So if we are going to succeed in developing effective therapies and diagnostics, we need to target the active prion forms, rather than the large amount of protein in plaques and tangles.&quot;</p>
<p>A-β and Tau Prion Activity Linked to Alzheimer's Patients' Longevity</p>
<p>The most remarkable finding of the new study may be the discovery that the self-propagating prion forms of tau and A-β are most infectious in the brains of Alzheimer's patients who died at a young age from inherited, genetically driven forms of the disease, but much less prevalent in patients who died at a more advanced age.</p>
<p>In particular, when compared to measurements of overall tau buildup -- which is known to increase with age in Alzheimer's brains -- the researchers found a remarkable exponential decline in the relative abundance of the prion forms of tau with age. When the researchers plotted their data, they saw an extremely strong correlation between tau prions and patients' age at death: relative to overall tau levels, the quantity of tau prions in the brain of a patient who died at age 40 were on average 32 times higher than in a patient who died at 90.</p>
<p>&quot;I still remember where I was sitting and what time of day it was when I first saw this data over a year ago,&quot; said co-author and leading neurodegeneration researcher William Seeley, MD, a professor of neurology at the UCSF Memory and Aging Center who directs the UCSF Neurodegenerative Disease Brain Bank, which provided tissue used in the study. &quot;I've very rarely, if ever, seen this kind of correlation in human biological data. Now the job is to find out what the correlation means.&quot;</p>
<p>The research raises a number of questions that will need to be addressed by future studies, including whether differences in prion infectivity could explain the long-standing mystery of why Alzheimer's progresses at such different rates in different patients. Other open questions resulting from the research include whether higher prion levels in brain samples from younger patients are linked to the early onset of the disease or how quickly it progressed, and whether lower prion levels in older brains reflect less &quot;infective&quot; prion variants or instead some ability of these patients' brains to dispose of misfolded proteins.</p>
<p>The evidence that prion forms of A-β and tau play a specific role in Alzheimer's disease -- one that cannot be captured by simply counting amyloid plaques and tau tangles in patient brains -- also raises questions on current approaches to Alzheimer's diagnosis, clinical trial design, and drug discovery, say the authors, who hope their novel assays will spur renewed interest in developing therapies to target the now-measurable prion proteins.</p>
</li>
<li>
<p><em>abstract</em> The hallmarks of Alzheimer’s disease (AD) are the accumulation of Aβ plaques and neurofibrillary tangles composed of hyperphosphorylated tau. We developed sensitive cellular assays using human embryonic kidney–293T cells to quantify intracellular self-propagating conformers of Aβ in brain samples from patients with AD or other neurodegenerative diseases. Postmortem brain tissue from patients with AD had measurable amounts of pathological Aβ conformers. Individuals over 80 years of age had the lowest amounts of prion-like Aβ and phosphorylated tau. Unexpectedly, the longevity-dependent decrease in self-propagating tau conformers occurred in spite of increasing amounts of total insoluble tau. When corrected for the abundance of insoluble tau, the ability of postmortem AD brain homogenates to induce misfolded tau in the cellular assays showed an exponential decrease with longevity, with a half-life of about one decade over the age range of 37 to 99 years. Thus, our findings demonstrate an inverse correlation between longevity in patients with AD and the abundance of pathological tau conformers. Our cellular assays can be applied to patient selection for clinical studies and the development of new drugs and diagnostics for AD.</p>
</li>
</ul>
</li>
</ul>
</li>
<li>
<p>mitochondria</p>
<ul>
<li>
<p><em>genetic evidence for elevated pathogenicity of mitochondrial dna heteroplasmy in autism spectrum disorder</em><br />
wang y, picard m, gu z 2016<br />
<a href="http://doi.org/10.1371/journal.pgen.1006391">doi.org/10.1371/journal.pgen.1006391</a></p>
<ul>
<li>Mitochondria contain their own genome, the mitochondrial DNA (mtDNA), and are abundant in the brain where they produce energy and intracellular signals required for normal brain function and cognition. Mitochondrial dysfunction has been proposed as a cause of Autism Spectrum Disorder (ASD), but the genetic basis has not been established. By analyzing mtDNA sequences from 903 ASD children along with their unaffected siblings and mothers, we found a unique pattern of heteroplasmic mtDNA mutations (the coexistence of both mutant and wide-type mtDNA molecules in a cell) associated with increased risk of ASD. A large fraction of these mtDNA heteroplasmies are pathogenic and can be either inherited maternally or acquired during development. Our findings unveil an important issue about pathogenic mtDNA heteroplasmy transmission between mothers and children, which could underlie many neurodevelopmental disorders in children that are shown to involve mitochondrial dysfunction. Evaluating mtDNA heteroplasmies in high-risk families could help diagnosis and treatment of these diseases.</li>
</ul>
</li>
<li>
<p><em>classification and adaptive behavior prediction of children with autism spectrum disorder based upon multivariate data analysis of markers of oxidative stress and dna methylation</em><br />
daniel p. howsmon et al. 2017<br />
<a href="http://doi.org/10.1371/journal.pcbi.1005385">doi.org/10.1371/journal.pcbi.1005385</a></p>
<ul>
<li>The number of diagnosed cases of Autism Spectrum Disorders (ASD) has increased dramatically over the last four decades; however, there is still considerable debate regarding the underlying pathophysiology of ASD. This lack of biological knowledge restricts diagnoses to be made based on behavioral observations and psychometric tools. However, physiological measurements should support these behavioral diagnoses in the future in order to enable earlier and more accurate diagnoses. Stepping towards this goal of incorporating biochemical data into ASD diagnosis, this paper analyzes measurements of metabolite concentrations of the folate-dependent one-carbon metabolism and transulfuration pathways taken from blood samples of 83 participants with ASD and 76 age-matched neurotypical peers. Fisher Discriminant Analysis enables multivariate classification of the participants as on the spectrum or neurotypical which results in 96.1% of all neurotypical participants being correctly identified as such while still correctly identifying 97.6% of the ASD cohort. Furthermore, kernel partial least squares is used to predict adaptive behavior, as measured by the Vineland Adaptive Behavior Composite score, where measurement of five metabolites of the pathways was sufficient to predict the Vineland score with an <em>R</em>2 of 0.45 after cross-validation. This level of accuracy for classification as well as severity prediction far exceeds any other approach in this field and is a strong indicator that the metabolites under consideration are strongly correlated with an ASD diagnosis but also that the statistical analysis used here offers tremendous potential for extracting important information from complex biochemical data sets.<br />
Author summary<br />
Autism spectrum disorder (ASD) encompasses a family of neurological disorders characterized by limited social interaction and restricted repetitive behaviors. The number of children diagnosed with ASD has grown exponentially over the last four decades and is now estimated to affect ∼1.5% of children. Although ASD is currently diagnosed and treated based solely on psychometric tools, a biochemical view applicable to at least a subset of ASD cases is emerging. Abnormalities in folate-dependent one carbon metabolism and transsulfuration pathways can summarize a large number of observations of genetic and environmental effects that increase ASD predisposition. However, these complex, highly interconnected pathways require more advanced statistical models than the typical univariate models presented in the literature. Therefore, we developed multivariate statistical models that classify participants based on their neurological status and predict adaptive behavior in ASD. We emphasize that these models are cross-validated, helping to ensure that the results will generalize to new samples. The models developed herein have much stronger predictability than any existing approaches from the scientific literature.</li>
</ul>
</li>
<li>
<p><em>mitochondrial dysfunction in autism spectrum disorders: a systematic review and meta-analysis</em><br />
d a rossignol, r e frye 2011<br />
<a href="http://doi.org/10.1038/mp.2010.136">doi.org/10.1038/mp.2010.136</a></p>
<ul>
<li>A comprehensive literature search was performed to collate evidence of mitochondrial dysfunction in autism spectrum disorders (ASDs) with two primary objectives. First, features of mitochondrial dysfunction in the general population of children with ASD were identified. Second, characteristics of mitochondrial dysfunction in children with ASD and concomitant mitochondrial disease (MD) were compared with published literature of two general populations: ASD children without MD, and non-ASD children with MD. The prevalence of MD in the general population of ASD was 5.0% (95% confidence interval 3.2, 6.9%), much higher than found in the general population (~0.01%). The prevalence of abnormal biomarker values of mitochondrial dysfunction was high in ASD, much higher than the prevalence of MD. Variances and mean values of many mitochondrial biomarkers (lactate, pyruvate, carnitine and ubiquinone) were significantly different between ASD and controls. Some markers correlated with ASD severity. Neuroimaging, <em>in vitro</em> and post-mortem brain studies were consistent with an elevated prevalence of mitochondrial dysfunction in ASD. Taken together, these findings suggest children with ASD have a spectrum of mitochondrial dysfunction of differing severity. Eighteen publications representing a total of 112 children with ASD and MD (ASD/MD) were identified. The prevalence of developmental regression (52%), seizures (41%), motor delay (51%), gastrointestinal abnormalities (74%), female gender (39%), and elevated lactate (78%) and pyruvate (45%) was significantly higher in ASD/MD compared with the general ASD population. The prevalence of many of these abnormalities was similar to the general population of children with MD, suggesting that ASD/MD represents a distinct subgroup of children with MD. Most ASD/MD cases (79%) were not associated with genetic abnormalities, raising the possibility of secondary mitochondrial dysfunction. Treatment studies for ASD/MD were limited, although improvements were noted in some studies with carnitine, co-enzyme Q10 and B-vitamins. Many studies suffered from limitations, including small sample sizes, referral or publication biases, and variability in protocols for selecting children for MD workup, collecting mitochondrial biomarkers and defining MD. Overall, this evidence supports the notion that mitochondrial dysfunction is associated with ASD. Additional studies are needed to further define the role of mitochondrial dysfunction in ASD.</li>
</ul>
</li>
<li>
<p><em>mitochondrial function controls intestinal epithelial stemness and proliferation</em><br />
emanuel berger et al. 2016<br />
<a href="http://doi.org/10.1038/ncomms13171">doi.org/10.1038/ncomms13171</a></p>
<ul>
<li>Control of intestinal epithelial stemness is crucial for tissue homeostasis. Disturbances in epithelial function are implicated in inflammatory and neoplastic diseases of the gastrointestinal tract. Here we report that mitochondrial function plays a critical role in maintaining intestinal stemness and homeostasis. Using intestinal epithelial cell (IEC)-specific mouse models, we show that loss of HSP60, a mitochondrial chaperone, activates the mitochondrial unfolded protein response (MT-UPR) and results in mitochondrial dysfunction. HSP60-deficient crypts display loss of stemness and cell proliferation, accompanied by epithelial release of WNT10A and RSPO1. Sporadic failure of Cre-mediated <em>Hsp60</em> deletion gives rise to hyperproliferative crypt foci originating from OLFM4+ stem cells. These effects are independent of the MT-UPR-associated transcription factor CHOP. In conclusion, compensatory hyperproliferation of HSP60+ escaper stem cells suggests paracrine release of WNT-related factors from HSP60-deficient, functionally impaired IEC to be pivotal in the control of the proliferative capacity of the stem cell niche.</li>
</ul>
</li>
<li>
<p><em>pervasive within-mitochondrion single-nucleotide variant heteroplasmy as revealed by single-mitochondrion sequencing</em><br />
jacqueline morris et al. 2017<br />
<a href="http://doi.org/10.1016/j.celrep.2017.11.031">doi.org/10.1016/j.celrep.2017.11.031</a></p>
<ul>
<li>
<p>•Single-mitochondrion sequencing shows single-mitochondrion heteroplasmy<br />
•The distribution of SNV loci suggests inheritance of variants across generations<br />
•Analysis of SNVs in human and mouse suggests distinct modes of somatic segregation</p>
<p>A number of mitochondrial diseases arise from single-nucleotide variant (SNV) accumulation in multiple mitochondria. Here, we present a method for identification of variants present at the single-mitochondrion level in individual mouse and human neuronal cells, allowing for extremely high-resolution study of mitochondrial mutation dynamics. We identified extensive heteroplasmy between individual mitochondrion, along with three high-confidence variants in mouse and one in human that were present in multiple mitochondria across cells. The pattern of variation revealed by single-mitochondrion data shows surprisingly pervasive levels of heteroplasmy in inbred mice. Distribution of SNV loci suggests inheritance of variants across generations, resulting in Poisson jackpot lines with large SNV load. Comparison of human and mouse variants suggests that the two species might employ distinct modes of somatic segregation. Single-mitochondrion resolution revealed mitochondria mutational dynamics that we hypothesize to affect risk probabilities for mutations reaching disease thresholds.</p>
</li>
</ul>
</li>
<li>
<p><em>mitochondria are physiologically maintained at close to 50°C</em><br />
dominique chrétien et al. 2018<br />
<a href="http://doi.org/10.1371/journal.pbio.2003992">doi.org/10.1371/journal.pbio.2003992</a></p>
<ul>
<li>To ensure a stable internal temperature, endothermic species make use of the heat released during the final steps of food burning by the mitochondria present in all cells of the organism. Indeed, only a fraction of the energy released by the oxidation of respiratory substrates is used to generate ATP, while a substantial proportion is released as heat. Using a temperature-sensitive fluorescent probe targeted to mitochondria, we measured the temperature of active mitochondria in cultured intact human cells. Mitochondria were found to be more than 10 °C warmer when the respiratory chain was functional. This differential was abolished in cells depleted of mitochondrial DNA or by respiratory inhibitors but preserved or enhanced by the expression of thermogenic enzymes such as <em>Ciona</em> alternative oxidase or by uncoupling protein 1. The activity of various respiratory chain enzymes was found to be maximal near 50 °C. Note that in view of their potential consequences, the observations reported here need to be validated and explored further by independent methods.</li>
</ul>
</li>
</ul>
</li>
<li>
<p>genetic nurture</p>
<ul>
<li>
<p><em>the nature of nurture: effects of parental genotypes</em><br />
augustine kong et al. 2018<br />
<a href="http://doi.org/10.1126/science.aan6877">doi.org/10.1126/science.aan6877</a></p>
<ul>
<li>Sequence variants in the parental genomes that are not transmitted to a child (the proband) are often ignored in genetic studies. Here we show that nontransmitted alleles can affect a child through their impacts on the parents and other relatives, a phenomenon we call “genetic nurture.” Using results from a meta-analysis of educational attainment, we find that the polygenic score computed for the nontransmitted alleles of 21,637 probands with at least one parent genotyped has an estimated effect on the educational attainment of the proband that is 29.9% (P = 1.6 × 10−14) of that of the transmitted polygenic score. Genetic nurturing effects of this polygenic score extend to other traits. Paternal and maternal polygenic scores have similar effects on educational attainment, but mothers contribute more than fathers to nutrition- and heath-related traits.</li>
</ul>
</li>
</ul>
</li>
<li>
<p><em>somatic niche cells regulate the cep-1/p53-mediated dna damage response in primordial germ cells</em><br />
hui-ling ou et al. 2019<br />
<a href="http://doi.org/10.1016/j.devcel.2019.06.012">doi.org/10.1016/j.devcel.2019.06.012</a></p>
<ul>
<li>
<p>body cells which are in direct contact with the germ cells in the nematode Caenorhabditis elegans are responsible for controlling the stability of the genome in primordial germ cells (PGCs). All germ cells, including sperm and eggs, originate from primordial germ cells that form during early embryo development. Professor Dr. Björn Schumacher and his team at the UoC's Institute for Genome Stability in Aging and at CECAD discovered that somatic niche cells that surround the PGCs control their response to DNA damage. The study 'Somatic niche cells regulate the CEP-1/p53-mediated DNA damage response in primordial germ cells,' has now been published in Developmental Cell.</p>
<p>For more than hundred years, inheritance of genetic information was thought to be autonomously controlled by the germ cells, explaining why acquired traits cannot be genetically inherited. Scientists believed that mutations occurring only in germ cells were responsible for any heritable genetic changes -- be it during evolution or as cause of genetic disorders. Schumacher and his team now challenge this assertion.</p>
<p>The DNA of an organism constantly gets damaged. Not only environmental influences, but also by-products of the body's energy metabolism damage the molecular structure of the genome in every cell. The scientists investigated how the genome integrity of PGCs is controlled. PGCs need to survey their genomes particularly rigorously because they give rise to all sperm or eggs of the organism. Damaged PGCs are particularly dangerous because they are hereditary and can lead to serious genetic disorders. PGCs thus need to stop dividing when their genomes are damaged until the DNA is repaired. Special niche cells are responsible for signalling to the PGCs that they need to stop dividing and repair before generating further germ cells. If they fail to do so, the PGCs might pass on dangerous mutations to the next generation.</p>
<p>To fulfil this important function, the niche cells are in intimate contact with the PGCs and instruct them whether to divide and generate germ cells or whether to stay inactive. 'This means that the body is responsible for controlling the integrity of heritable genomes,' Schumacher remarked. 'The parental body thus has somatic control over the integrity of PGC genomes, controlling the quality of the heritable genetic information.' Since studying PGCs in mammals is a complicated endeavour, Schumacher's team used C. elegans as a simple animal model to shed new light on to how PGCs control the integrity of the genomes they will pass on to their offspring.</p>
</li>
<li>
<p><em>abstract</em> •CEP-1/p53 arrests primordial germ cell (PGC) proliferation in response to DNA damage<br />
•IFE-4 in somatic gonad precursor (PGC) niche cells controls CEP-1/p53 activity in PGCs<br />
•FGF-like signaling mediates somatic control of PGC arrest<br />
•eIF4E2 in niche cells controls p53 induction in hair follicle stem cells</p>
<p>Genome integrity in primordial germ cells (PGCs) is a prerequisite for fertility and species maintenance. In C. elegans, PGCs require global-genome nucleotide excision repair (GG-NER) to remove UV-induced DNA lesions. Failure to remove the lesions leads to the activation of the C. elegans p53, CEP-1, resulting in mitotic arrest of the PGCs. We show that the eIF4E2 translation initiation factor IFE-4 in somatic gonad precursor (SGP) niche cells regulates the CEP-1/p53-mediated DNA damage response (DDR) in PGCs. We determine that the IFE-4 translation target EGL-15/FGFR regulates the non-cell-autonomous DDR that is mediated via FGF-like signaling. Using hair follicle stem cells as a paradigm, we demonstrate that the eIF4E2-mediated niche cell regulation of the p53 response in stem cells is highly conserved in mammals. We thus reveal that the somatic niche regulates the CEP-1/p53-mediated DNA damage checkpoint in PGCs. Our data suggest that the somatic niche impacts the stability of heritable genomes.</p>
</li>
</ul>
</li>
</ul>
</li>
</ul>
<h2>02092</h2>
<ul>
<li><em>top of the world</em><br />
karen carpenter</li>
</ul>
<h2>01992</h2>
<h2>01892</h2>
<h2>01792</h2>
<h2>01692</h2>
<ul>
<li>genetics and epigenetics
<ul>
<li>epigenetic change may be constant, with the net effect from sustained trend in any direction
<ul>
<li>
<p>perhaps conscious change in particular direction can lead to epigenetic change</p>
<ul>
<li>could the same be true of insight and innovation, that by providing a suitable environment you set the trend towards certain methylations being more common</li>
</ul>
</li>
<li>
<p>they continue to look for pathways in which, for example, serotonin directly influences behavior via chemical changes in the brain. but what if the real mechanism is actually indirect, that epigenetic changes are constantly happening and what serotonin does is tip the balance towards a certain population of epigenetic variance, which in turn causes the behavior we observe</p>
</li>
<li>
<p>when I get enough sleep, ideas and connections and creativity just seem to flow. could this be through epigenetic change too?</p>
</li>
<li>
<p><em>adolescent binge-pattern alcohol exposure alters genome-wide dna methylation patterns in the hypothalamus of alcohol-naïve male offspring</em><br />
annadorothea animes 2016<br />
<a href="http://doi.org/10.1016/j.alcohol.2016.10.010">doi.org/10.1016/j.alcohol.2016.10.010</a></p>
<ul>
<li>Teenage binge drinking is a major health concern in the United States, with 21% of teenagers reporting binge-pattern drinking behavior in the last 30 days. Recently, our lab showed that alcohol-naïve offspring of rats exposed to alcohol during adolescence exhibited altered gene expression profiles in the hypo- thalamus, a brain region involved in stress regulation. We employed Enhanced Reduced Representation Bisulfite Sequencing as an unbiased approach to test the hypothesis that parental exposure to binge- pattern alcohol during adolescence alters DNA methylation profiles in their alcohol-naïve offspring. Wistar rats were administered a repeated binge-ethanol exposure paradigm during early (postnatal day (PND) 37e44) and late (PND 67e74) adolescent development. Animals were mated 24 h after the last ethanol dose and subsequent offspring were produced. Analysis of male PND7 offspring revealed that offspring of alcohol-exposed parents exhibited differential DNA methylation patterns in the hypothala- mus. The differentially methylated cytosines (DMCs) were distinct between offspring depending on which parent was exposed to ethanol. Moreover, novel DMCs were observed when both parents were exposed to ethanol and many DMCs from single parent ethanol exposure were not recapitulated with dual parent exposure. We also measured mRNA expression of several differentially methylated genes and some, but not all, showed correlative changes in expression. Importantly, methylation was not a direct predictor of expression levels, underscoring the complexity of transcriptional regulation. Overall, we demonstrate that adolescent binge ethanol exposure causes altered genome-wide DNA methylation patterns in the hypothalamus of alcohol-naïve offspring.</li>
</ul>
</li>
<li>
<p><em>epigenetic correlates of neonatal contact in humans</em><br />
sarah r. moore et al. 2017<br />
<a href="http://doi.org/10.1017/s0954579417001213">doi.org/10.1017/s0954579417001213</a></p>
<ul>
<li>Animal models of early postnatal mother–infant interactions have highlighted the importance of tactile contact for biobehavioral outcomes via the modification of DNA methylation (DNAm). The role of normative variation in contact in early human development has yet to be explored. In an effort to translate the animal work on tactile contact to humans, we applied a naturalistic daily diary strategy to assess the link between maternal contact with infants and epigenetic signatures in children 4–5 years later, with respect to multiple levels of child-level factors, including genetic variation and infant distress. We first investigated DNAm at four candidate genes: the glucocorticoid receptor gene, nuclear receptor subfamily 3, group C, member 1 (NR3C1), μ-opioid receptor M1 (OPRM1) and oxytocin receptor (OXTR; related to the neurobiology of social bonds), and brain-derived neurotrophic factor (BDNF; involved in postnatal plasticity). Although no candidate gene DNAm sites significantly associated with early postnatal contact, when we next examined DNAm across the genome, differentially methylated regions were identified between high and low contact groups. Using a different application of epigenomic information, we also quantified epigenetic age, and report that for infants who received low contact from caregivers, greater infant distress was associated with younger epigenetic age. These results suggested that early postnatal contact has lasting associations with child biology.</li>
</ul>
</li>
</ul>
</li>
<li>genetic sensitivity environment
<ul>
<li>(...) jed diamond in irritable male syndrome isbn 140507745x chap8 irritable male body quotes a study by remi cadoret, Iowa college medicine, comment by dean hamer, national cancer institute. saying that good genes or bad genes in good family, or good genes in bad family both fine. but bad genes in bad family not fine at all. &quot;shows that what is being inherited is not bad behaviour or aggression but rather a genetic sensitivity to the environment.&quot; he goes on to a conventional conclusion but what is important for us is that this sentence on its own, interpreted out of context, actually makes sense for our way of thinking about genes and development.
<ul>
<li>It is a genetic sensitivity to the environment that nature “selects” for, not a particular trait or set of traits. What will help organisms survive is not an unchanging adherence to a given pattern but an adaptability to the conditions in which they find themselves.</li>
<li>check for studies of cumulative genetic drift as opposed to single mutations – this may be missing in literature.</li>
</ul>
</li>
</ul>
</li>
</ul>
</li>
</ul>
<h2>01592</h2>
<h2>01492</h2>
<h2>01392</h2>
<h2>01292</h2>
<ul>
<li>invariant with time
<ul>
<li>scattershot
<ul>
<li>many outcomes, large range of abilities and differences</li>
<li>a few will be very well suited to the particular niche they happen to grow into</li>
</ul>
</li>
</ul>
</li>
</ul>
<h2>01192</h2>
<ul>
<li>varies with time
<ul>
<li>niche
<ul>
<li>a circumstance in the world to which a physical characteristic of an organism is advantageous</li>
<li>niches change so those that fit them will be different over time</li>
<li>at any particular time the crucial limiting niche may be different (the one circumstance that all organisms of a certain type in a certain area must endure to survive)</li>
<li>study cross-fostering blue tits and great tits finds learned behaviour from foster parents establishes the niche the offspring inhabit to a higher degree than expected http://rspb.royalsocietypublishing.org/content/274/1606/19.full</li>
</ul>
</li>
<li>survival
<ul>
<li>
<p>not selection</p>
</li>
<li>
<p>just those who happen to fit the current state of the current niche more likely to survive</p>
</li>
<li>
<p><em>opposite responses to selection and where to find them</em><br />
david n. fisher, jonathan n. pruitt 2019<br />
<a href="http://doi.org/10.1111/jeb.13432">doi.org/10.1111/jeb.13432</a></p>
<ul>
<li>
<p>increased evolution of selfless traits -- such as sharing food and keeping watch for one another -- is mathematically equivalent to the decreased evolution of individually beneficial traits.</p>
<p>&quot;They're two sides of the same coin,&quot; Fisher explains. &quot;On one side, traits evolve that benefit your kin, but don't benefit you, because you're helping your siblings or cousins. On the other side, traits that benefit you but cost your neighbours don't evolve, because you're causing damage to related individuals.&quot;</p>
<p>The work is part of the ongoing effort to understand the paradox of altruistic behaviour in the wild, explains Fisher, a research fellow in McMaster's Department of Psychology, Neuroscience and Behaviour.</p>
<p>Fisher goes on to show that another way evolution can go backwards is through the evolution of an individual's negative effects on neighbours and group members. For example, a fast-growing tree may take all the sunlight, water and nutrients out of the environment, causing its neighbours to grow slowly. In the next generation, more trees are fast-growing but are also nasty neighbours. As a result, negative social effects are much more prevalent, and so everyone's growth is reduced.</p>
<p>&quot;That means evolution has gone backwards. Even though growing quickly is beneficial, because of these negative social effects, the population, on average, grows more slowly,&quot; he says.</p>
<p>Fisher plans to travel to Ecuador this summer to study co-operative spiders, and whether changes in individual and group benefits can explain why co-operation diminishes at higher elevations.</p>
</li>
<li>
<p><em>abstract</em> We generally expect traits to evolve in the same direction as selection. However, many organisms possess traits that appear to be costly for individuals, while plant and animal breeding experiments reveal that selection may lead to no response or even negative responses to selection. We formalize both of these instances as cases of “opposite responses to selection.” Using quantitative genetic models for the response to selection, we outline when opposite responses to selection should be expected. These typically occur when social selection opposes direct selection, when individuals interact with others less related to them than a random member of the population, and if the genetic covariance between direct and indirect effects is negative. We discuss the likelihood of each of these occurring in nature and therefore summarize how frequent opposite responses to selection are likely to be. This links several evolutionary phenomena within a single framework.</p>
</li>
</ul>
</li>
</ul>
</li>
<li>sexual reproduction
<ul>
<li>illusion of choice</li>
<li>actually what may be happening is that the internal state of the selecting partner becomes effectively a niche</li>
</ul>
</li>
<li>fitness
<ul>
<li>I don't believe there is a single maximum for evolutionary fitness in a single location, because the conditions, however local, change over time. just as we were talking about another paper, where my viewpoint is that in some circumstances, viridical vision may be fit, whereas in others it may be unfit.</li>
<li>We are still assuming that natural selection operates on the level of individual organisms (even though chance and expression of genes are major factors) so how is it that there can be some creatures that are unfit for the current situation?</li>
<li>The answer that it is not a single-run game, where all the unfit are eliminated in one round. Simplifications of evolutionary theory assume that the unfit just die out, when in fact what is fit is not set, and so what is fit in one situation is unfit in another and it is evolutionarily effective to have a range of possible options.</li>
<li>So the distribution over time of so-called fitness is not going to a simple one, even if momentarily a sample indicates a peak distribution around some criterion.</li>
</ul>
</li>
</ul>
</li>
</ul>
<h2>01092</h2>
<ul>
<li>parasite eve theme, piano<br />
liliypichu 2014 <a href="https://www.youtube.com/watch?v=hCqdOCO7kMg">youtube</a></li>
</ul>
<h2>00992</h2>
<h2>00892</h2>
<ul>
<li>my view of evolution more tinged by meteors and supervolcanoes. i.e. were dinosaurs “maladapted” or are there other “solutions” to planet–killers? am starting to think our view of adaptation too species–specific, need to go “up a size class” as it were. inter–species diversity seems to be key there, but may be applies throughout the “size classes”, just salient at planet–killer scale.</li>
</ul>
<h2>00792</h2>
<ul>
<li>darwin’s theory of evolution did much to end the iron grip of religion on the world, but the abusers simply switched their tools and started using a corrupted version of the theory as their tool of oppression instead. there is no perfect theory that can avoid abuse, abusers will try to find a way to use the zeitgeist to abuse others.</li>
</ul>
<h2>00692</h2>
<ul>
<li>villains force our evolution — if we do not overcome them, we go extinct.</li>
</ul>
<h2>00592</h2>
<ul>
<li>evolutionary biology is like the “antiques roadshow” — high on speculation, short on fact</li>
</ul>
<h2>00492</h2>
<ul>
<li>no longer appropriate to say “genetic inheritance”. more “inherited transmission” or better just “transmission”</li>
</ul>
<h2>00392</h2>
<ul>
<li>firstly it is an illusion that polymath is rarer than specialist. both are common. polymath like swarm behaviour, self–correcting. moves in manner not easily predictable using methods for individual movement.</li>
</ul>
<h2>00292</h2>
<ul>
<li>evolution is example of a method of preparation and adaptation to possible and current conditions</li>
</ul>
<h2>00192</h2>
<ul>
<li>evolution</li>
</ul>
<h2>00092</h2>
<ul>
<li>uranium U (οὐρανός ouranos, uranus)</li>
<li>eve brea
<ul>
<li>controlling a stinger</li>
<li>stinger</li>
</ul>
</li>
<li>IB</li>
</ul>

</div>

</body>
</html>
