<html>
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
<title>12b6ddcbb4ef1e022f47434b81bdc8ca</title>
<script type="text/javascript">
function showhide(id)
{
    var e = document.getElementById(id);
    e.style.display = (e.style.display == 'block') ? 'none' : 'block';
}
</script>
<style>
rect {
transition: .6s fill;
fill: #D3D3D3;
opacity: 0;
}
rect:hover {
    fill: #D3D3D3;
    opacity: 0.2;
}

.outline {
    clear: both;
}

.svg-container {
    width: 100%;
    max-width: 696px;
    margin-left: auto;
    margin-right: auto;
    display: block;
}

.svg-content {
    width: 100%;
}

.container {
    width: 100%;
}
</style>

</head>

<body>

<div class="container"><div class="spacer">&nbsp;</div><div class="svg-container"><svg version="1.1"  preserveAspectRatio="xMinYMin meet" class="svg-content" viewBox="0 0 696 8192" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
<image width="696" height="8192" xlink:href="00094-12b6ddcbb4ef1e022f47434b81bdc8ca-assets/00094-12b6ddcbb4ef1e022f47434b81bdc8ca.png"></image>
<a xlink:href="http://doi.org/10.1098/rstb.2018.0092" xlink:title="http://doi.org/10.1098/rstb.2018.0092"><rect x="318" y="60" width="89" height="7"/></a>
<a xlink:href="http://doi.org/10.1038/s41586-019-1065-y" xlink:title="http://doi.org/10.1038/s41586-019-1065-y"><rect x="318" y="98" width="103" height="7"/></a>
<a xlink:href="http://doi.org/10.1073/pnas.1218525110" xlink:title="http://doi.org/10.1073/pnas.1218525110"><rect x="318" y="135" width="98" height="7"/></a>
<a xlink:href="http://doi.org/10.1128/mSystems.00055-18" xlink:title="http://doi.org/10.1128/mSystems.00055-18"><rect x="318" y="172" width="105" height="7"/></a>
<a xlink:href="https://aeon.co/essays/gut-worms-were-once-a-cause-of-disease-now-they-are-a-cure" xlink:title="https://aeon.co/essays/gut-worms-were-once-a-cause-of-disease-now-they-are-a-cure"><rect x="318" y="291" width="133" height="7"/></a>
<a xlink:href="https://aeon.co/essays/gut-worms-were-once-a-cause-of-disease-now-they-are-a-cure" xlink:title="https://aeon.co/essays/gut-worms-were-once-a-cause-of-disease-now-they-are-a-cure"><rect x="318" y="299" width="86" height="7"/></a>
<a xlink:href="https://aeon.co/ideas/i-holobiont-are-you-and-your-microbes-a-community-or-a-single-entity" xlink:title="https://aeon.co/ideas/i-holobiont-are-you-and-your-microbes-a-community-or-a-single-entity"><rect x="318" y="396" width="134" height="7"/></a>
<a xlink:href="https://aeon.co/ideas/i-holobiont-are-you-and-your-microbes-a-community-or-a-single-entity" xlink:title="https://aeon.co/ideas/i-holobiont-are-you-and-your-microbes-a-community-or-a-single-entity"><rect x="318" y="403" width="110" height="7"/></a>
<a xlink:href="http://doi.org/10.1371/journal.pgen.1006297" xlink:title="http://doi.org/10.1371/journal.pgen.1006297"><rect x="318" y="488" width="107" height="7"/></a>
<a xlink:href="http://doi.org/10.1038/ncomms14040" xlink:title="http://doi.org/10.1038/ncomms14040"><rect x="318" y="555" width="88" height="7"/></a>
<a xlink:href="http://doi.org/10.1186/s40168-018-0432-5" xlink:title="http://doi.org/10.1186/s40168-018-0432-5"><rect x="318" y="600" width="104" height="7"/></a>
<a xlink:href="http://doi.org/10.1371/journal.pbio.2000633" xlink:title="http://doi.org/10.1371/journal.pbio.2000633"><rect x="318" y="646" width="105" height="7"/></a>
<a xlink:href="http://doi.org/10.1371/journal.pbio.3000298" xlink:title="http://doi.org/10.1371/journal.pbio.3000298"><rect x="318" y="676" width="105" height="7"/></a>
<a xlink:href="http://doi.org/10.1038/nature21725" xlink:title="http://doi.org/10.1038/nature21725"><rect x="318" y="714" width="81" height="7"/></a>
<a xlink:href="http://doi.org/10.1073/pnas.1707009114" xlink:title="http://doi.org/10.1073/pnas.1707009114"><rect x="318" y="759" width="98" height="7"/></a>
<a xlink:href="http://doi.org/10.1016/j.cell.2016.11.003" xlink:title="http://doi.org/10.1016/j.cell.2016.11.003"><rect x="318" y="797" width="96" height="7"/></a>
<a xlink:href="http://doi.org/10.1098/rspb.2017.2274" xlink:title="http://doi.org/10.1098/rspb.2017.2274"><rect x="318" y="834" width="91" height="7"/></a>
<a xlink:href="http://doi.org/10.1073/pnas.1806834116" xlink:title="http://doi.org/10.1073/pnas.1806834116"><rect x="318" y="872" width="98" height="7"/></a>
<a xlink:href="http://doi.org/10.1038/s41467-017-02651-5" xlink:title="http://doi.org/10.1038/s41467-017-02651-5"><rect x="468" y="936" width="107" height="7"/></a>
<a xlink:href="http://doi.org/10.1016/j.cell.2015.11.004" xlink:title="http://doi.org/10.1016/j.cell.2015.11.004"><rect x="468" y="974" width="96" height="7"/></a>
<a xlink:href="http://doi.org/10.1038/s41586-018-0634-9" xlink:title="http://doi.org/10.1038/s41586-018-0634-9"><rect x="468" y="1012" width="104" height="7"/></a>
<a xlink:href="http://doi.org/10.1016/j.cmet.2018.05.026" xlink:title="http://doi.org/10.1016/j.cmet.2018.05.026"><rect x="468" y="1057" width="101" height="7"/></a>
<a xlink:href="http://doi.org/10.1016/j.cell.2017.12.033" xlink:title="http://doi.org/10.1016/j.cell.2017.12.033"><rect x="468" y="1095" width="96" height="7"/></a>
<a xlink:href="http://doi.org/10.1371/journal.pone.0169303" xlink:title="http://doi.org/10.1371/journal.pone.0169303"><rect x="468" y="1140" width="107" height="7"/></a>
<a xlink:href="http://doi.org/10.1016/j.molcel.2016.10.25" xlink:title="http://doi.org/10.1016/j.molcel.2016.10.25"><rect x="468" y="1186" width="101" height="7"/></a>
<a xlink:href="http://doi.org/10.1038/NCOMMS13329" xlink:title="http://doi.org/10.1038/NCOMMS13329"><rect x="468" y="1224" width="93" height="7"/></a>
<a xlink:href="http://doi.org/10.1038/s41590-019-0316-2" xlink:title="http://doi.org/10.1038/s41590-019-0316-2"><rect x="468" y="1269" width="104" height="7"/></a>
<a xlink:href="http://doi.org/10.1016/j.celrep.2019.05.105" xlink:title="http://doi.org/10.1016/j.celrep.2019.05.105"><rect x="468" y="1315" width="104" height="7"/></a>
<a xlink:href="http://doi.org/10.1016/j.celrep.2019.05.105" xlink:title="http://doi.org/10.1016/j.celrep.2019.05.105"><rect x="468" y="1360" width="104" height="7"/></a>
<a xlink:href="http://doi.org/10.1038/s41564-019-0540-4" xlink:title="http://doi.org/10.1038/s41564-019-0540-4"><rect x="468" y="1398" width="104" height="7"/></a>
<a xlink:href="http://doi.org/10.1126/science.aaw3134" xlink:title="http://doi.org/10.1126/science.aaw3134"><rect x="468" y="1443" width="95" height="7"/></a>
<a xlink:href="http://doi.org/10.1073/pnas.1809349115" xlink:title="http://doi.org/10.1073/pnas.1809349115"><rect x="397" y="1473" width="98" height="7"/></a>
<a xlink:href="http://doi.org/10.1016/j.cell.2017.05.036" xlink:title="http://doi.org/10.1016/j.cell.2017.05.036"><rect x="397" y="1511" width="96" height="7"/></a>
<a xlink:href="http://doi.org/10.14309/ctg.0000000000000122" xlink:title="http://doi.org/10.14309/ctg.0000000000000122"><rect x="388" y="1548" width="118" height="7"/></a>
<a xlink:href="http://doi.org/10.1161/circulationaha.118.036652" xlink:title="http://doi.org/10.1161/circulationaha.118.036652"><rect x="521" y="1586" width="120" height="7"/></a>
<a xlink:href="http://doi.org/10.1126/science.aao5774" xlink:title="http://doi.org/10.1126/science.aao5774"><rect x="388" y="1623" width="94" height="7"/></a>
<a xlink:href="http://doi.org/10.1016/j.cell.2018.04.027" xlink:title="http://doi.org/10.1016/j.cell.2018.04.027"><rect x="388" y="1661" width="96" height="7"/></a>
<a xlink:href="http://doi.org/10.3389/fmicb.2016.01230" xlink:title="http://doi.org/10.3389/fmicb.2016.01230"><rect x="400" y="1766" width="98" height="7"/></a>
<a xlink:href="http://doi.org/10.7554/eLife.18678" xlink:title="http://doi.org/10.7554/eLife.18678"><rect x="400" y="1812" width="79" height="7"/></a>
<a xlink:href="http://doi.org/10.1186/s40168-018-0416-5" xlink:title="http://doi.org/10.1186/s40168-018-0416-5"><rect x="400" y="1858" width="104" height="7"/></a>
<a xlink:href="http://doi.org/10.1126/sciadv.aao4502" xlink:title="http://doi.org/10.1126/sciadv.aao4502"><rect x="400" y="1896" width="90" height="7"/></a>
<a xlink:href="http://doi.org/10.1038/s41598-017-04197-4" xlink:title="http://doi.org/10.1038/s41598-017-04197-4"><rect x="400" y="1933" width="107" height="7"/></a>
<a xlink:href="http://doi.org/10.1038/s41467-019-09525-y" xlink:title="http://doi.org/10.1038/s41467-019-09525-y"><rect x="400" y="1978" width="106" height="7"/></a>
<a xlink:href="http://doi.org/10.1016/j.chom.2019.03.004" xlink:title="http://doi.org/10.1016/j.chom.2019.03.004"><rect x="400" y="2024" width="103" height="7"/></a>
<a xlink:href="http://doi.org/10.1249/mss.0000000000001495" xlink:title="http://doi.org/10.1249/mss.0000000000001495"><rect x="400" y="2062" width="118" height="7"/></a>
<a xlink:href="http://doi.org/10.1086/698021" xlink:title="http://doi.org/10.1086/698021"><rect x="399" y="2099" width="68" height="7"/></a>
<a xlink:href="http://abcnews.go.com/Health/anxiety-head-gut/story?id=20229136" xlink:title="http://abcnews.go.com/Health/anxiety-head-gut/story?id=20229136"><rect x="387" y="2129" width="129" height="7"/></a>
<a xlink:href="http://doi.org/10.1016/j.molmet.2016.10.003" xlink:title="http://doi.org/10.1016/j.molmet.2016.10.003"><rect x="318" y="2190" width="107" height="7"/></a>
<a xlink:href="http://doi.org/10.1186/s40168-017-0321-3" xlink:title="http://doi.org/10.1186/s40168-017-0321-3"><rect x="318" y="2228" width="104" height="7"/></a>
<a xlink:href="http://doi.org/10.1038/s41586-018-0119-x" xlink:title="http://doi.org/10.1038/s41586-018-0119-x"><rect x="318" y="2265" width="103" height="7"/></a>
<a xlink:href="http://doi.org/10.1016/j.cell.2018.07.048" xlink:title="http://doi.org/10.1016/j.cell.2018.07.048"><rect x="318" y="2303" width="96" height="7"/></a>
<a xlink:href="http://https://doi.org/10.1016/j.tree.2019.11.004" xlink:title="http://https://doi.org/10.1016/j.tree.2019.11.004"><rect x="318" y="2348" width="98" height="7"/></a>
<a xlink:href="http://doi.org/10.1038/s41586-020-2745-3" xlink:title="http://doi.org/10.1038/s41586-020-2745-3"><rect x="318" y="2406" width="104" height="7"/></a>
<a xlink:href="http://doi.org/10.1371/journal.pcbi.1006960" xlink:title="http://doi.org/10.1371/journal.pcbi.1006960"><rect x="318" y="2443" width="105" height="7"/></a>
<a xlink:href="http://doi.org/10.1038/s41598-019-42183-0" xlink:title="http://doi.org/10.1038/s41598-019-42183-0"><rect x="318" y="2488" width="107" height="7"/></a>
<a xlink:href="http://doi.org/10.1016/j.cell.2019.05.004" xlink:title="http://doi.org/10.1016/j.cell.2019.05.004"><rect x="318" y="2534" width="96" height="7"/></a>
<a xlink:href="http://doi.org/10.4049/jimmunol.1701755" xlink:title="http://doi.org/10.4049/jimmunol.1701755"><rect x="318" y="2580" width="98" height="7"/></a>
<a xlink:href="http://doi.org/10.3389/fncel.2017.00120" xlink:title="http://doi.org/10.3389/fncel.2017.00120"><rect x="318" y="2617" width="95" height="7"/></a>
<a xlink:href="http://doi.org/10.1038/nature25973" xlink:title="http://doi.org/10.1038/nature25973"><rect x="318" y="2655" width="81" height="7"/></a>
<a xlink:href="http://doi.org/10.1038/s41591-019-0450-2" xlink:title="http://doi.org/10.1038/s41591-019-0450-2"><rect x="318" y="2684" width="104" height="7"/></a>
<a xlink:href="http://doi.org/10.1038/s41586-018-0617-x" xlink:title="http://doi.org/10.1038/s41586-018-0617-x"><rect x="318" y="2722" width="103" height="7"/></a>
<a xlink:href="http://doi.org/10.1038/s41586-018-0620-2" xlink:title="http://doi.org/10.1038/s41586-018-0620-2"><rect x="318" y="2759" width="104" height="7"/></a>
<a xlink:href="http://doi.org/10.1128/mbio.01751-18" xlink:title="http://doi.org/10.1128/mbio.01751-18"><rect x="318" y="2804" width="89" height="7"/></a>
<a xlink:href="http://doi.org/10.1016/j.chom.2019.01.011" xlink:title="http://doi.org/10.1016/j.chom.2019.01.011"><rect x="318" y="2850" width="103" height="7"/></a>
<a xlink:href="http://doi.org/10.1016/j.ajog.2019.04.036" xlink:title="http://doi.org/10.1016/j.ajog.2019.04.036"><rect x="318" y="2896" width="99" height="7"/></a>
<a xlink:href="http://doi.org/10.1038/srep31023" xlink:title="http://doi.org/10.1038/srep31023"><rect x="359" y="2926" width="76" height="7"/></a>
<a xlink:href="http://doi.org/10.1016/j.immuni.2017.02.016" xlink:title="http://doi.org/10.1016/j.immuni.2017.02.016"><rect x="359" y="2972" width="107" height="7"/></a>
<a xlink:href="http://doi.org/10.1183/13993003.00280-2017" xlink:title="http://doi.org/10.1183/13993003.00280-2017"><rect x="359" y="3010" width="112" height="7"/></a>
<a xlink:href="http://doi.org/10.1038/s41591-019-0469-4" xlink:title="http://doi.org/10.1038/s41591-019-0469-4"><rect x="359" y="3047" width="104" height="7"/></a>
<a xlink:href="http://doi.org/10.1038/s41467-019-13698-x" xlink:title="http://doi.org/10.1038/s41467-019-13698-x"><rect x="359" y="3084" width="106" height="7"/></a>
<a xlink:href="http://doi.org/10.1038/s41467-019-09284-w" xlink:title="http://doi.org/10.1038/s41467-019-09284-w"><rect x="318" y="3122" width="108" height="7"/></a>
<a xlink:href="http://doi.org/10.1080/19490976.2017.1353849" xlink:title="http://doi.org/10.1080/19490976.2017.1353849"><rect x="318" y="3167" width="119" height="7"/></a>
<a xlink:href="http://doi.org/10.1038/s41598-018-20953-6" xlink:title="http://doi.org/10.1038/s41598-018-20953-6"><rect x="318" y="3205" width="107" height="7"/></a>
<a xlink:href="http://doi.org/10.1016/j.celrep.2016.12.010" xlink:title="http://doi.org/10.1016/j.celrep.2016.12.010"><rect x="318" y="3250" width="104" height="7"/></a>
<a xlink:href="http://doi.org/10.1016/j.chom.2017.11.004" xlink:title="http://doi.org/10.1016/j.chom.2017.11.004"><rect x="318" y="3296" width="103" height="7"/></a>
<a xlink:href="http://doi.org/10.1016/j.chom.2017.11.003" xlink:title="http://doi.org/10.1016/j.chom.2017.11.003"><rect x="318" y="3341" width="103" height="7"/></a>
<a xlink:href="http://doi.org/10.1503/cmaj.170809" xlink:title="http://doi.org/10.1503/cmaj.170809"><rect x="318" y="3387" width="83" height="7"/></a>
<a xlink:href="http://doi.org/10.1038/s41598-017-16606-9" xlink:title="http://doi.org/10.1038/s41598-017-16606-9"><rect x="318" y="3433" width="107" height="7"/></a>
<a xlink:href="http://doi.org/10.1016/j.chom.2019.03.007" xlink:title="http://doi.org/10.1016/j.chom.2019.03.007"><rect x="318" y="3504" width="103" height="7"/></a>
<a xlink:href="http://doi.org/10.1016/j.cell.2018.10.029" xlink:title="http://doi.org/10.1016/j.cell.2018.10.029"><rect x="318" y="3542" width="96" height="7"/></a>
<a xlink:href="http://doi.org/10.1126/science.aan4834" xlink:title="http://doi.org/10.1126/science.aan4834"><rect x="318" y="3579" width="94" height="7"/></a>
<a xlink:href="http://doi.org/10.1103/PhysRevLett.121.098101" xlink:title="http://doi.org/10.1103/PhysRevLett.121.098101"><rect x="318" y="3624" width="116" height="7"/></a>
<a xlink:href="http://doi.org/10.1128/mSphere.00327-17" xlink:title="http://doi.org/10.1128/mSphere.00327-17"><rect x="318" y="3662" width="101" height="7"/></a>
<a xlink:href="http://doi.org/10.1128/mSphere.00193-18" xlink:title="http://doi.org/10.1128/mSphere.00193-18"><rect x="318" y="3707" width="101" height="7"/></a>
<a xlink:href="http://doi.org/10.1038/s41586-019-1058-x" xlink:title="http://doi.org/10.1038/s41586-019-1058-x"><rect x="318" y="3745" width="103" height="7"/></a>
<a xlink:href="http://dx.doi.org/10.1016/j.celrep.2015.03.049" xlink:title="http://dx.doi.org/10.1016/j.celrep.2015.03.049"><rect x="318" y="3790" width="112" height="7"/></a>
<a xlink:href="http://doi.org/10.1038/s41598-018-29032-2" xlink:title="http://doi.org/10.1038/s41598-018-29032-2"><rect x="318" y="3836" width="107" height="7"/></a>
<a xlink:href="http://doi.org/10.3389/fmicb.2018.02323" xlink:title="http://doi.org/10.3389/fmicb.2018.02323"><rect x="318" y="3882" width="98" height="7"/></a>
<a xlink:href="http://doi.org/10.1038/s41598-017-17487-8" xlink:title="http://doi.org/10.1038/s41598-017-17487-8"><rect x="318" y="3935" width="107" height="7"/></a>
<a xlink:href="http://doi.org/10.1093/jn/nxy004" xlink:title="http://doi.org/10.1093/jn/nxy004"><rect x="318" y="3989" width="73" height="7"/></a>
<a xlink:href="http://doi.org/10.1186/s40168-018-0559-4" xlink:title="http://doi.org/10.1186/s40168-018-0559-4"><rect x="318" y="4026" width="104" height="7"/></a>
<a xlink:href="http://doi.org/10.1016/j.cels.2018.08.003" xlink:title="http://doi.org/10.1016/j.cels.2018.08.003"><rect x="318" y="4064" width="98" height="7"/></a>
<a xlink:href="http://doi.org/10.1016/j.cell.2018.10.020" xlink:title="http://doi.org/10.1016/j.cell.2018.10.020"><rect x="318" y="4101" width="96" height="7"/></a>
<a xlink:href="http://doi.org/10.3389/fmicb.2019.01849" xlink:title="http://doi.org/10.3389/fmicb.2019.01849"><rect x="318" y="4138" width="98" height="7"/></a>
<a xlink:href="http://doi.org/10.1038/s41467-018-07275-x" xlink:title="http://doi.org/10.1038/s41467-018-07275-x"><rect x="318" y="4176" width="106" height="7"/></a>
<a xlink:href="http://doi.org/10.1016/j.cell.2018.09.055" xlink:title="http://doi.org/10.1016/j.cell.2018.09.055"><rect x="318" y="4227" width="96" height="7"/></a>
<a xlink:href="http://doi.org/10.1126/scisignal.aav5938" xlink:title="http://doi.org/10.1126/scisignal.aav5938"><rect x="318" y="4272" width="95" height="7"/></a>
<a xlink:href="http://doi.org/10.1038/s41598-019-45348-z" xlink:title="http://doi.org/10.1038/s41598-019-45348-z"><rect x="318" y="4318" width="106" height="7"/></a>
<a xlink:href="http://doi.org/10.1038/s41586-020-2047-9" xlink:title="http://doi.org/10.1038/s41586-020-2047-9"><rect x="318" y="4375" width="104" height="7"/></a>
<a xlink:href="http://doi.org/10.1038/s41586-019-1521-8" xlink:title="http://doi.org/10.1038/s41586-019-1521-8"><rect x="401" y="4413" width="104" height="7"/></a>
<a xlink:href="http://doi.org/10.1001/jamapsychiatry.2018.3428" xlink:title="http://doi.org/10.1001/jamapsychiatry.2018.3428"><rect x="401" y="4466" width="119" height="7"/></a>
<a xlink:href="http://doi.org/10.1001/jamapsychiatry.2019.0029" xlink:title="http://doi.org/10.1001/jamapsychiatry.2019.0029"><rect x="401" y="4551" width="119" height="7"/></a>
<a xlink:href="http://doi.org/10.1016/j.chom.2017.10.020" xlink:title="http://doi.org/10.1016/j.chom.2017.10.020"><rect x="401" y="4636" width="103" height="7"/></a>
<a xlink:href="http://doi.org/10.1038/s41564-018-0257-9" xlink:title="http://doi.org/10.1038/s41564-018-0257-9"><rect x="401" y="4674" width="104" height="7"/></a>
<a xlink:href="http://doi.org/10.1093/nar/gky793" xlink:title="http://doi.org/10.1093/nar/gky793"><rect x="401" y="4719" width="77" height="7"/></a>
<a xlink:href="https://doi.org/10.7717/peerj.4452" xlink:title="https://doi.org/10.7717/peerj.4452"><rect x="401" y="4773" width="76" height="7"/></a>
<a xlink:href="http://doi.org/10.1126/scisignal.aat5750" xlink:title="http://doi.org/10.1126/scisignal.aat5750"><rect x="401" y="4818" width="94" height="7"/></a>
<a xlink:href="http://doi.org/10.1002/anie.201707737" xlink:title="http://doi.org/10.1002/anie.201707737"><rect x="559" y="4864" width="92" height="7"/></a>
<a xlink:href="http://doi.org/10.1186/s40168-019-0632-7" xlink:title="http://doi.org/10.1186/s40168-019-0632-7"><rect x="401" y="4910" width="104" height="7"/></a>
<a xlink:href="http://doi.org/10.1093/eurheartj/ehz231" xlink:title="http://doi.org/10.1093/eurheartj/ehz231"><rect x="401" y="4948" width="92" height="7"/></a>
<a xlink:href="http://doi.org/10.1002/mds.27924" xlink:title="http://doi.org/10.1002/mds.27924"><rect x="401" y="4993" width="78" height="7"/></a>
<a xlink:href="http://doi.org/10.1177/1535370218782139" xlink:title="http://doi.org/10.1177/1535370218782139"><rect x="421" y="5039" width="104" height="7"/></a>
<a xlink:href="https://doi.org/10.1073/pnas.1803880115" xlink:title="https://doi.org/10.1073/pnas.1803880115"><rect x="421" y="5076" width="98" height="7"/></a>
<a xlink:href="http://doi.org/10.3390/e15041416" xlink:title="http://doi.org/10.3390/e15041416"><rect x="421" y="5129" width="79" height="7"/></a>
<a xlink:href="https://doi.org/10.3389/fmicb.2017.00004" xlink:title="https://doi.org/10.3389/fmicb.2017.00004"><rect x="417" y="5175" width="98" height="7"/></a>
<a xlink:href="http://doi.org/10.1128/mbio.02282-18" xlink:title="http://doi.org/10.1128/mbio.02282-18"><rect x="417" y="5228" width="89" height="7"/></a>
<a xlink:href="http://doi.org/10.1038/nm.4174" xlink:title="http://doi.org/10.1038/nm.4174"><rect x="417" y="5274" width="71" height="7"/></a>
<a xlink:href="http://doi.org/10.1371/journal.ppat.1007957" xlink:title="http://doi.org/10.1371/journal.ppat.1007957"><rect x="417" y="5312" width="105" height="7"/></a>
<a xlink:href="http://doi.org/10.1039/c7fo02038d" xlink:title="http://doi.org/10.1039/c7fo02038d"><rect x="417" y="5349" width="80" height="7"/></a>
<a xlink:href="http://doi.org/10.3390/molecules23102454" xlink:title="http://doi.org/10.3390/molecules23102454"><rect x="417" y="5386" width="103" height="7"/></a>
<a xlink:href="http://doi.org/10.1017/s0954579419000087" xlink:title="http://doi.org/10.1017/s0954579419000087"><rect x="443" y="5432" width="108" height="7"/></a>
<a xlink:href="http://doi.org/10.1038/s41396-018-0241-7" xlink:title="http://doi.org/10.1038/s41396-018-0241-7"><rect x="443" y="5477" width="104" height="7"/></a>
<a xlink:href="https://doi.org/10.1038/s41467-018-06552-z" xlink:title="https://doi.org/10.1038/s41467-018-06552-z"><rect x="443" y="5523" width="106" height="7"/></a>
<a xlink:href="https://www.sciencedaily.com/releases/2016/11/161114124958.htm" xlink:title="https://www.sciencedaily.com/releases/2016/11/161114124958.htm"><rect x="443" y="5585" width="135" height="7"/></a>
<a xlink:href="https://www.sciencedaily.com/releases/2016/11/161114124958.htm" xlink:title="https://www.sciencedaily.com/releases/2016/11/161114124958.htm"><rect x="443" y="5593" width="81" height="7"/></a>
<a xlink:href="https://doi.org/10.1038/s41380-018-0105-6" xlink:title="https://doi.org/10.1038/s41380-018-0105-6"><rect x="397" y="5638" width="104" height="7"/></a>
<a xlink:href="http://doi.org/10.1128/mbio.00355-19" xlink:title="http://doi.org/10.1128/mbio.00355-19"><rect x="397" y="5691" width="89" height="7"/></a>
<a xlink:href="http://doi.org/10.1016/j.chembiol.2019.09.010" xlink:title="http://doi.org/10.1016/j.chembiol.2019.09.010"><rect x="397" y="5745" width="112" height="7"/></a>
<a xlink:href="http://doi.org/10.1016/j.mehy.2019.109421" xlink:title="http://doi.org/10.1016/j.mehy.2019.109421"><rect x="397" y="5821" width="104" height="7"/></a>
<a xlink:href="http://doi.org/10.1126/sciadv.aau3333" xlink:title="http://doi.org/10.1126/sciadv.aau3333"><rect x="404" y="5867" width="90" height="7"/></a>
<a xlink:href="http://doi.org/10.1038/s41582-020-0323-9" xlink:title="http://doi.org/10.1038/s41582-020-0323-9"><rect x="404" y="5904" width="104" height="7"/></a>
<a xlink:href="http://doi.org/10.1126/scitranslmed.aar5280" xlink:title="http://doi.org/10.1126/scitranslmed.aar5280"><rect x="404" y="5942" width="105" height="7"/></a>
<a xlink:href="http://dx.doi.org/10.1016/j.cell.2016.11.018" xlink:title="http://dx.doi.org/10.1016/j.cell.2016.11.018"><rect x="404" y="5987" width="105" height="7"/></a>
<a xlink:href="http://doi.org/10.1371/journal.ppat.1006654" xlink:title="http://doi.org/10.1371/journal.ppat.1006654"><rect x="404" y="6025" width="105" height="7"/></a>
<a xlink:href="http://doi.org/10.1038/srep41802" xlink:title="http://doi.org/10.1038/srep41802"><rect x="404" y="6070" width="76" height="7"/></a>
<a xlink:href="http://doi.org/10.18632/oncotarget.21490" xlink:title="http://doi.org/10.18632/oncotarget.21490"><rect x="360" y="6108" width="99" height="7"/></a>
<a xlink:href="http://doi.org/10.1126/science.aah3648" xlink:title="http://doi.org/10.1126/science.aah3648"><rect x="360" y="6153" width="94" height="7"/></a>
<a xlink:href="http://doi.org/10.1016/j.chom.2018.01.007" xlink:title="http://doi.org/10.1016/j.chom.2018.01.007"><rect x="360" y="6199" width="103" height="7"/></a>
<a xlink:href="http://doi.org/10.1038/s41568-018-0015-6" xlink:title="http://doi.org/10.1038/s41568-018-0015-6"><rect x="360" y="6237" width="104" height="7"/></a>
<a xlink:href="http://doi.org/10.1053/j.gastro.2018.07.028" xlink:title="http://doi.org/10.1053/j.gastro.2018.07.028"><rect x="360" y="6282" width="104" height="7"/></a>
<a xlink:href="http://doi.org/10.1038/s41586-020-2080-8" xlink:title="http://doi.org/10.1038/s41586-020-2080-8"><rect x="360" y="6320" width="104" height="7"/></a>
<a xlink:href="http://doi.org/10.1038/s41594-019-0353-4" xlink:title="http://doi.org/10.1038/s41594-019-0353-4"><rect x="442" y="6357" width="104" height="7"/></a>
<a xlink:href="http://doi.org/10.1038/s41467-018-06125-0" xlink:title="http://doi.org/10.1038/s41467-018-06125-0"><rect x="442" y="6395" width="107" height="7"/></a>
<a xlink:href="http://doi.org/10.1126/science.aar7201" xlink:title="http://doi.org/10.1126/science.aar7201"><rect x="442" y="6432" width="92" height="7"/></a>
<a xlink:href="http://doi.org/10.3389/fmicb.2018.01995" xlink:title="http://doi.org/10.3389/fmicb.2018.01995"><rect x="442" y="6477" width="98" height="7"/></a>
<a xlink:href="http://doi.org/10.1016/j.chom.2017.03.002" xlink:title="http://doi.org/10.1016/j.chom.2017.03.002"><rect x="442" y="6523" width="103" height="7"/></a>
<a xlink:href="http://doi.org/10.1126/science.aao5610" xlink:title="http://doi.org/10.1126/science.aao5610"><rect x="442" y="6561" width="94" height="7"/></a>
<a xlink:href="http://doi.org/10.1007/s11103-017-0590-z" xlink:title="http://doi.org/10.1007/s11103-017-0590-z"><rect x="442" y="6598" width="103" height="7"/></a>
<a xlink:href="http://doi.org/10.1016/j.chom.2018.01.001" xlink:title="http://doi.org/10.1016/j.chom.2018.01.001"><rect x="442" y="6643" width="103" height="7"/></a>
<a xlink:href="http://doi.org/10.1016/j.chom.2020.01.021" xlink:title="http://doi.org/10.1016/j.chom.2020.01.021"><rect x="442" y="6681" width="103" height="7"/></a>
<a xlink:href="http://doi.org/10.1038/s41591-018-0061-3" xlink:title="http://doi.org/10.1038/s41591-018-0061-3"><rect x="399" y="6726" width="104" height="7"/></a>
<a xlink:href="http://doi.org/10.1016/j.steroids.2017.06.002" xlink:title="http://doi.org/10.1016/j.steroids.2017.06.002"><rect x="399" y="6764" width="109" height="7"/></a>
<a xlink:href="http://doi.org/10.1093/ajcn/nqx072" xlink:title="http://doi.org/10.1093/ajcn/nqx072"><rect x="399" y="6825" width="80" height="7"/></a>
<a xlink:href="http://doi.org/10.1038/nature22075" xlink:title="http://doi.org/10.1038/nature22075"><rect x="384" y="6862" width="81" height="7"/></a>
<a xlink:href="http://doi.org/10.1186/s40168-017-0260-z" xlink:title="http://doi.org/10.1186/s40168-017-0260-z"><rect x="384" y="6907" width="103" height="7"/></a>
<a xlink:href="http://doi.org/10.1016/j.bbi.2017.06.018" xlink:title="http://doi.org/10.1016/j.bbi.2017.06.018"><rect x="384" y="6953" width="96" height="7"/></a>
<a xlink:href="http://doi.org/10.1038/s41564-018-0307-3" xlink:title="http://doi.org/10.1038/s41564-018-0307-3"><rect x="384" y="6991" width="104" height="7"/></a>
<a xlink:href="http://doi.org/10.1016/j.chom.2017.06.006" xlink:title="http://doi.org/10.1016/j.chom.2017.06.006"><rect x="400" y="7036" width="103" height="7"/></a>
<a xlink:href="http://doi.org/10.1128/msphere.00199-16" xlink:title="http://doi.org/10.1128/msphere.00199-16"><rect x="368" y="7074" width="100" height="7"/></a>
<a xlink:href="http://doi.org/10.1073/pnas.1912129116" xlink:title="http://doi.org/10.1073/pnas.1912129116"><rect x="368" y="7119" width="98" height="7"/></a>
<a xlink:href="http://doi.org/10.1111/joim.12805" xlink:title="http://doi.org/10.1111/joim.12805"><rect x="318" y="7193" width="77" height="7"/></a>
<a xlink:href="http://doi.org/10.1172/jci.insight.95997" xlink:title="http://doi.org/10.1172/jci.insight.95997"><rect x="318" y="7230" width="91" height="7"/></a>
<a xlink:href="http://doi.org/10.1016/j.tins.2016.09.002" xlink:title="http://doi.org/10.1016/j.tins.2016.09.002"><rect x="318" y="7267" width="97" height="7"/></a>
<a xlink:href="https://www.sciencedaily.com/releases/2017/04/170403123508.htm" xlink:title="https://www.sciencedaily.com/releases/2017/04/170403123508.htm"><rect x="318" y="7352" width="135" height="7"/></a>
<a xlink:href="https://www.sciencedaily.com/releases/2017/04/170403123508.htm" xlink:title="https://www.sciencedaily.com/releases/2017/04/170403123508.htm"><rect x="318" y="7360" width="81" height="7"/></a>
<a xlink:href="http://doi.org/10.1038/s41586-018-0092-4" xlink:title="http://doi.org/10.1038/s41586-018-0092-4"><rect x="318" y="7398" width="104" height="7"/></a>
<a xlink:href="http://doi.org/10.1016/j.cell.2018.07.016" xlink:title="http://doi.org/10.1016/j.cell.2018.07.016"><rect x="318" y="7451" width="96" height="7"/></a>
<a xlink:href="http://doi.org/10.1038/s41467-019-10430-7" xlink:title="http://doi.org/10.1038/s41467-019-10430-7"><rect x="318" y="7508" width="107" height="7"/></a>
<a xlink:href="http://doi.org/10.1186/s12916-016-0698-z" xlink:title="http://doi.org/10.1186/s12916-016-0698-z"><rect x="318" y="7561" width="103" height="7"/></a>
<a xlink:href="http://doi.org/10.1001/jama.2017.17077" xlink:title="http://doi.org/10.1001/jama.2017.17077"><rect x="318" y="7615" width="95" height="7"/></a>
<a xlink:href="http://doi.org/10.1016/j.watres.2018.03.032" xlink:title="http://doi.org/10.1016/j.watres.2018.03.032"><rect x="318" y="7660" width="105" height="7"/></a>
<a xlink:href="http://doi.org/10.1073/pnas.1613843113" xlink:title="http://doi.org/10.1073/pnas.1613843113"><rect x="318" y="7725" width="98" height="7"/></a>
<a xlink:href="http://doi.org/10.1038/s41396-017-0042-4" xlink:title="http://doi.org/10.1038/s41396-017-0042-4"><rect x="318" y="7763" width="104" height="7"/></a>
<a xlink:href="http://doi.org/10.1016/j.chom.2018.07.006" xlink:title="http://doi.org/10.1016/j.chom.2018.07.006"><rect x="318" y="7801" width="103" height="7"/></a>
<a xlink:href="http://doi.org/10.1016/j.chom.2019.05.001" xlink:title="http://doi.org/10.1016/j.chom.2019.05.001"><rect x="318" y="7846" width="103" height="7"/></a>
<a xlink:href="https://doi.org/10.1016/j.cels.2018.06.005" xlink:title="https://doi.org/10.1016/j.cels.2018.06.005"><rect x="47" y="4471" width="98" height="7"/></a>
<a xlink:href="http://doi.org/10.1038/s41564-019-0475-9" xlink:title="http://doi.org/10.1038/s41564-019-0475-9"><rect x="40" y="3873" width="104" height="7"/></a>
<a xlink:href="http://doi.org/10.1126/science.aat9691" xlink:title="http://doi.org/10.1126/science.aat9691"><rect x="22" y="3761" width="92" height="7"/></a>
<a xlink:href="http://doi.org/10.1371/journal.ppat.1007597" xlink:title="http://doi.org/10.1371/journal.ppat.1007597"><rect x="28" y="3814" width="105" height="7"/></a>
</svg>
</div>
</div>

<br/>
<a href="javascript:showhide('outlineDiv')">Outline show/hide</a>
<div id="outlineDiv" style="display:none;">
<h1>00094</h1>
<p>le petit prince</p>
<h2>microbiome</h2>
<h2>05094</h2>
<ul>
<li>
<p><em>natural diversity of crispr spacers of thermus evidence of local spacer acquisition and global spacer exchange</em><br />
anna lopatina et al. 2019<br />
<a href="http://doi.org/10.1098/rstb.2018.0092">doi.org/10.1098/rstb.2018.0092</a></p>
<ul>
<li>
<p>planet-wide mechanism that ensures the exchange of bacteria between faraway places,&quot; said senior author Konstantin Severinov, a principal investigator at the Waksman Institute of Microbiology and professor of molecular biology and biochemistry in the School of Arts and Sciences at Rutgers University-New Brunswick.</p>
<p>&quot;Because the bacteria we study live in very hot water -- about 160 degrees Fahrenheit -- in remote places, it is not feasible to imagine that animals, birds or humans transport them,&quot; Severinov said. &quot;They must be transported by air and this movement must be very extensive so bacteria in isolated places share common characteristics.&quot;</p>
<p>Severinov and other researchers studied the &quot;molecular memories&quot; of bacteria from their encounters with viruses, with the memories stored in bacterial DNA, according to a study in the journal Philosophical Transactions of the Royal Society B.</p>
<p>Bacteriophages -- viruses of bacteria -- are the most abundant and ubiquitous forms of life on the planet, the study notes. The viruses have a profound influence on microbial populations, community structure and evolution.</p>
<p>The scientists collected heat-loving Thermus thermophilus bacteria in hot gravel on Mount Vesuvius and hot springs on Mount Etna in Italy; hot springs in the El Tatio region in northern Chile and southern Chile's Termas del Flaco region; and hot springs in the Uzon caldera in Kamchatka, Russia.</p>
<p>In bacterial cells infected by viruses, molecular memories are stored in special regions of bacterial DNA called CRISPR arrays. Cells that survive infections pass the memories -- small pieces of viral DNA -- to their offspring. The order of these memories allows scientists to follow the history of bacterial interaction with viruses over time.</p>
<p>Initially, the scientists thought that bacteria of the same species living in hot springs thousands of miles apart -- and therefore isolated from each other -- would have very different memories of their encounters with viruses. That's because the bacteria all should have independent histories of viral infections. The scientists also thought that bacteria should be evolving very rapidly and become different, much like the famous finches Charles Darwin observed on the Galapagos Islands.</p>
<p>&quot;What we found, however, is that there were plenty of shared memories -- identical pieces of viral DNA stored in the same order in the DNA of bacteria from distant hot springs,&quot; Severinov said. &quot;Our analysis may inform ecological and epidemiological studies of harmful bacteria that globally share antibiotic resistance genes and may also get dispersed by air instead of human travelers.&quot;</p>
<p>The scientists want to test their air bridge hypothesis by sampling air at different altitudes and locations around the world and by identifying the bacteria there, he said. They would need access to planes, drones or research balloons.</p>
</li>
<li>
<p><em>abstract</em> We investigated the diversity of CRISPR spacers of Thermus communities from two locations in Italy, two in Chile and one location in Russia. Among the five sampling sites, a total of more than 7200 unique spacers belonging to different CRISPR-Cas systems types and subtypes were identified. Most of these spacers are not found in CRISPR arrays of sequenced Thermus strains. Comparison of spacer sets revealed that samples within the same area (separated by few to hundreds of metres) have similar spacer sets, which appear to be largely stable at least over the course of several years. While at further distances (hundreds of kilometres and more) the similarity of spacer sets is decreased, there are still multiple common spacers in Thermus communities from different continents. The common spacers can be reconstructed in identical or similar CRISPR arrays, excluding their independent appearance and suggesting an extensive migration of thermophilic bacteria over long distances. Several new Thermus phages were isolated in the sampling sites. Mapping of spacers to bacteriophage sequences revealed examples of local acquisition of spacers from some phages and distinct patterns of targeting of phage genomes by different CRISPR-Cas systems.</p>
</li>
</ul>
</li>
<li>
<p><em>structural variation in the gut microbiome associates with host health</em><br />
david zeevi et al. 2019<br />
<a href="http://doi.org/10.1038/s41586-019-1065-y">doi.org/10.1038/s41586-019-1065-y</a></p>
<ul>
<li>
<p>&quot;What if the same microbe is different in different people?&quot;</p>
<p>It has been long known that the genomes of microbes are not fixed from birth, as ours are. They are able to lose some of their genes, exchange genes with other microorganisms, or gain new ones from their environment. Thus, a detailed comparison of the genomes of seemingly identical bacteria will reveal sequences of DNA that occur in one genome and not others, or possibly sequences that appear just once in one and several times over in others. These differences are called structural variants. Structural variants -- even tiny ones -- can translate into huge differences in the ways that microbes interact with their human hosts. A variant might be the difference between a benign presence and a pathogenic one, or it could give bacteria resistance to antibiotics.</p>
<p>Drs. David Zeevi and Tal Korem, initially in the lab of Prof. Eran Segal in the Weizmann Institute of Science and then in their present positions in Rockefeller and Columbia Universities, developed algorithms that systematically identify structural variants across human gut microbiomes. The researchers began with microbiomes from nearly 900 Israeli subjects, in which they succeeded in identifying over 7,000 variants. Next, they formed a collaboration with Dutch researchers from the University of Groningen, in the Netherlands, and they looked for these variants in the microbiomes of a large group of Dutch subjects. Most of the structural variants they had identified in the Israeli subjects could also be found among the Dutch ones, despite the differences in genetics and lifestyle between the groups.</p>
<p>The scientists next asked whether any of the structural variants they had identified are associated with health or disease. The group turned up more than 100 that were associated with risk factors for disease. Many of these associations were again replicated in the Dutch cohort.</p>
<p>In one case, individuals who had a certain variant present in the genome of a particular microbial species in their microbiome were 6 kg thinner and had a 4 cm narrower waist, on average, than individuals who had the same microbe -- but one that did not harbor that particular variant. The scientists then analyzed the genes encoded on this variant and found that it gives the bacterium the potential ability to turn certain sugars into a substance called butyrate. Butyrate is a small fatty acid that smells like rancid butter (thus its name, from the Ancient Greek for &quot;butter&quot;); despite its odor, butyrate has been shown to have anti-inflammatory effects and a positive influence on metabolism. This ability, say the scientists, could help explain the weight difference between those carrying bacteria with and those without the structural variant.</p>
<p>The finding suggests the method the group developed could help researchers pinpoint the connections between our microbiome, health and disease in significant ways that might be missed with other means. &quot;The real potential of this approach,&quot; says Zeevi, &quot;is that it allows us to look for the actual mechanisms behind the associations we find.&quot;</p>
<p>Segal estimates there may be tens of thousands of structural variants within the human gut microbiome and thousands of these could be associated with disease and disease risk. Since the makeup of the microbiome has been implicated in so many different syndromes and disorders, this research could have a lasting impact on the search for better, more targeted probiotics for treating disease.</p>
</li>
<li>
<p><em>abstract</em> Differences in the presence of even a few genes between otherwise identical bacterial strains may result in critical phenotypic differences. Here we systematically identify microbial genomic structural variants (SVs) and find them to be prevalent in the human gut microbiome across phyla and to replicate in different cohorts. SVs are enriched for CRISPR-associated and antibiotic-producing functions and depleted from housekeeping genes, suggesting that they have a role in microbial adaptation. We find multiple associations between SVs and host disease risk factors, many of which replicate in an independent cohort. Exploring genes that are clustered in the same SV, we uncover several possible mechanistic links between the microbiome and its host, including a region in Anaerostipes hadrus that encodes a composite inositol catabolism-butyrate biosynthesis pathway, the presence of which is associated with lower host metabolic disease risk. Overall, our results uncover a nascent layer of variability in the microbiome that is associated with microbial adaptation and host health.</p>
</li>
</ul>
</li>
</ul>
<h2>05194</h2>
<ul>
<li>
<p><em>animals in a bacterial world, a new imperative for the life sciences</em><br />
mcfall–ngai et al 2013<br />
<a href="http://doi.org/10.1073/pnas.1218525110">doi.org/10.1073/pnas.1218525110</a></p>
<ul>
<li>In the last two decades, the widespread application of genetic and genomic approaches has revealed a bacterial world astonishing in its ubiquity and diversity. This review examines how a growing knowledge of the vast range of animal–bacterial interactions, whether in shared ecosystems or intimate symbioses, is fundamentally altering our understanding of animal biology. Specifically, we highlight recent techno-logical and intellectual advances that have changed our thinking about five questions: how have bacteria facilitated the origin and evolution of animals; how do animals and bacteria affect each other's genomes; how does normal animal development depend on bacterial partners; how is homeostasis maintained between animals and their symbionts; and how can ecological approaches deepen our understanding of the multiple levels of animal–bacterial interaction. As answers to these fundamental questions emerge, all biologists will be challenged to broaden their appreciation of these interactions and to include investigations of the relationships between and among bacteria and their animal partners as we seek a better understanding of the natural world.</li>
<li>the New Synthesis of the 1930s and beyond must be reconsidered in terms of three areas in which it has proven weakest: symbiosis, de-velopment, and microbiology (115). One of these areas, microbiology, presents particular challenges both to the species concept, as formulated by Ernst Mayr in 1942, and to the concept that vertical transmission of genetic information is the only motor of selectable evolutionary change.</li>
</ul>
</li>
<li>
<p><em>phylogenetically novel uncultured microbial cells dominate earth microbiomes</em><br />
karen g. lloyd et al. 2018<br />
<a href="http://doi.org/10.1128/mSystems.00055-18">doi.org/10.1128/mSystems.00055-18</a></p>
<ul>
<li>To describe a microbe’s physiology, including its metabolism, environmental roles, and growth characteristics, it must be grown in a laboratory culture. Unfortunately, many phylogenetically novel groups have never been cultured, so their physiologies have only been inferred from genomics and environmental characteristics. Although the diversity, or number of different taxonomic groups, of uncultured clades has been studied well, their global abundances, or numbers of cells in any given environment, have not been assessed. We quantified the degree of similarity of 16S rRNA gene sequences from diverse environments in publicly available metagenome and metatranscriptome databases, which we show have far less of the culture bias present in primer-amplified 16S rRNA gene surveys, to those of their nearest cultured relatives. Whether normalized to scaffold read depths or not, the highest abundances of metagenomic 16S rRNA gene sequences belong to phylogenetically novel uncultured groups in seawater, freshwater, terrestrial subsurface, soil, hypersaline environments, marine sediment, hot springs, hydrothermal vents, nonhuman hosts, snow, and bioreactors (22% to 87% uncultured genera to classes and 0% to 64% uncultured phyla). The exceptions were human and human-associated environments, which were dominated by cultured genera (45% to 97%). We estimate that uncultured genera and phyla could comprise 7.3 × 1029 (81%) and 2.2 × 1029 (25%) of microbial cells, respectively. Uncultured phyla were overrepresented in metatranscriptomes relative to metagenomes (46% to 84% of sequences in a given environment), suggesting that they are viable. Therefore, uncultured microbes, often from deeply phylogenetically divergent groups, dominate nonhuman environments on Earth, and their undiscovered physiologies may matter for Earth systems.</li>
</ul>
</li>
</ul>
<h2>05294</h2>
<ul>
<li>
<p><em>i, superorganism: learning to love your inner ecosystem</em><br />
jon turney 2015 9781848318229</p>
<ul>
<li>diverse ecosystems but similar collective metabolism</li>
<li>has full story of helicobacter pylori</li>
</ul>
</li>
<li>
<p><em>gut: the inside story of the body’s most under–rated organ</em><br />
giulia enders 2015 9781771641500</p>
</li>
<li>
<p><em>we need worms: you might think they are disgusting. but our war against intestinal worms has damaged our immune systems and mental health</em><br />
william parker 2019<br />
<a href="https://aeon.co/essays/gut-worms-were-once-a-cause-of-disease-now-they-are-a-cure">aeon.co/essays/gut-worms-were-once-a-cause-of-disease-now-they-are-a-cure</a></p>
</li>
</ul>
<h2>05394</h2>
<ul>
<li><em>i contain multitudes: the microbes within us and a grander view of life</em><br />
ed jong 2016 9780062368621</li>
</ul>
<h2>05494</h2>
<ul>
<li><em>missing microbes: how the overuse of antibiotics is fuelling our modern plagues</em><br />
martin blaser 2014 978-1250069276</li>
</ul>
<h2>05594</h2>
<ul>
<li>
<p><em>i, holobiont. are you and your microbes a community or a single entity?</em><br />
derek skillings 2018<br />
<a href="https://aeon.co/ideas/i-holobiont-are-you-and-your-microbes-a-community-or-a-single-entity">aeon.co/ideas/i-holobiont-are-you-and-your-microbes-a-community-or-a-single-entity</a></p>
<p>“natural selection doesn’t just act on the genome of individual organisms: it acts on the hologenome of holobionts, which are seen as single units that can evolve at the level of the holobiont.”</p>
</li>
<li>
<p><em>drosophila adaptation to viral infection through defensive symbiont evolution</em><br />
vitor g. faria et al. 2016<br />
<a href="http://doi.org/10.1371/journal.pgen.1006297">doi.org/10.1371/journal.pgen.1006297</a></p>
<ul>
<li>
<p>Microbial symbionts can modulate host interactions with biotic and abiotic factors. Such interactions may affect the evolutionary trajectories of both host and symbiont. Wolbachia protects Drosophila melanogaster against several viral infections and the strength of the protection varies between variants of this endosymbiont. Since Wolbachia is maternally transmitted, its fitness depends on the fitness of its host. Therefore, Wolbachia populations may be under selection when Drosophila is subjected to viral infection. Here we show that in D. melanogaster populations selected for increased survival upon infection with Drosophila C virus there is a strong selection coefficient for specific Wolbachia variants, leading to their fixation. Flies carrying these selected Wolbachia variants have higher survival and fertility upon viral infection when compared to flies with the other variants. These findings demonstrate how the interaction of a host with pathogens shapes the genetic composition of symbiont populations. Furthermore, host adaptation can result from the evolution of its symbionts, with host and symbiont functioning as a single evolutionary unit.</p>
<p>Author Summary</p>
<p>Animals live in close association with microbial partners that can shape many aspects of their lives. For instance, several insects carry bacteria that defend them against parasites and infectious diseases. The intracellular bacterium Wolbachia protects the fruit fly Drosophila melanogaster against viral infection. Natural populations of Drosophila carry different variants of Wolbachia, which differ from one another in the strength of this protection. Here we show that a population of Drosophila infected with viruses during several generations adapts to this challenge through turnover in Wolbachia composition. The Wolbachia variants that give higher protection to viruses, by increasing fly survival and fecundity upon infection, are strongly selected. This work demonstrates that the interaction of an animal with a pathogen can shape its associated microbial populations. We show that adaptation to pathogens can be achieved not only through selection of resistance on the host proper but also through the evolutionary shaping of its microbial community.</p>
</li>
</ul>
</li>
</ul>
<h2>05694</h2>
<ul>
<li>
<p><em>brain maker: the power of gut microbes to heal and protect your brain — for life</em><br />
david perlmutter 2015</p>
<ul>
<li>more controversial</li>
</ul>
</li>
<li>
<p><em>microbes can help explain the evolution of host altruism</em><br />
ohad lewin-epstein et al. 2017<br />
<a href="http://doi.org/10.1038/ncomms14040">doi.org/10.1038/ncomms14040</a></p>
<ul>
<li>The evolution of altruistic behaviour, which is costly to the donor but beneficial for the recipient, is among the most intriguing questions in evolutionary biology. Several theories have been proposed to explain it, including kin selection, group selection and reciprocity. Here we propose that microbes that manipulate their hosts to act altruistically could be favoured by selection, and may play a role in the widespread occurrence of altruism. Using computational models, we find that microbe-induced altruism can explain the evolution of host altruistic behaviour under wider conditions than host-centred theories, including in a fully mixed host population, without repeating interactions or individual recognition. Our results suggest that factors such as antibiotics that kill microbes might negatively affect cooperation in a wide range of organisms.</li>
</ul>
</li>
</ul>
<h2>05794</h2>
<ul>
<li>
<p><em>trees, fungi and bacteria: tripartite metatranscriptomics of a root microbiome responding to soil contamination</em><br />
e. gonzalez et al. 2018<br />
<a href="http://doi.org/10.1186/s40168-018-0432-5">doi.org/10.1186/s40168-018-0432-5</a></p>
<ul>
<li>From gene expression alone, the major responses of <em>Salix purpurea</em> cv. ‘Fish Creek’ to soil contamination do not seem to be direct degradation, immobilisation or exclusion of contaminants but rather widespread, extraordinarily complex, alterations to interactions with microbiota. The extremely diverse fungal community, revealed using a bioinformatics approach unconstrained by the requirement of <em>a priori</em> nucleotide sequence, responds dynamically to contamination both with changes to gene expression but also with substantial shifts in the community makeup. Much like the host plant, fungal gene expression was dominated by alterations in pathways associated to microbiome interactions as opposed to direct hydrocarbon degradation. Surprisingly, gene expression representing increased petroleum hydrocarbon metabolism equipment came from an <em>Enterobacteriaceae</em> species whose total expression was uniformly more abundant in roots of contaminated trees. It seems possible that, given the successful contamination tolerance, willows may depend on symbiosis to tolerate stressful environmental conditions instead of relying solely on their own metabolism.<br />
While polyadenylated bacterial sequences are often not quantified across experimental systems investigated with eukaryote expression in mind, the bacterial functionality observed here is thought-provoking, revealing a plausible set of expressed genes indicating tripartite mutualism. A finding which is useful in providing a convincing reminder that observing the entirety of generated data, even that which is commonly discarded, is always of potential value for exposing the unexpected. Most importantly, with regard to the attempted all-inclusive strategy of observing all expressed sequences within a biological sample, regardless of origin, is how extensively interpretation of expression could be confounded if plant, fungal or bacterial expression were investigated as responsive to treatment (here being contamination) in isolation, and thus how crucial it is to attempt observation of the entire microbiome and wider metaorganism. This metatranscriptomic approach, which marries an understanding of the uncertainty of microbiome biology with the strengths and limitations of current bioinformatics, should be broadly transferable through utility for biological samples containing expressed sequences from well-characterised, poorly characterised or entirely unknown organisms at a diversity of low or high complexity.</li>
</ul>
</li>
</ul>
<h2>05894</h2>
<ul>
<li>
<p><em>stochastic assembly produces heterogeneous communities in the caenorhabditis elegans intestine</em><br />
nicole m. vega, jeff gore 2017<br />
<a href="http://doi.org/10.1371/journal.pbio.2000633">doi.org/10.1371/journal.pbio.2000633</a></p>
<ul>
<li>
<p>Host-associated bacterial communities vary extensively between individuals, but it can be very difficult to determine the sources of this heterogeneity. Here, we demonstrate that stochastic bacterial community assembly in the <em>Caenorhabditis elegans</em> intestine is sufficient to produce strong interworm heterogeneity in community composition. When worms are fed with two neutrally competing, fluorescently labeled bacterial strains, we observe stochastically driven bimodality in community composition, in which approximately half of the worms are dominated by each bacterial strain. A simple model incorporating stochastic colonization suggests that heterogeneity between worms is driven by the low rate at which bacteria successfully establish new intestinal colonies. We can increase this rate experimentally by feeding worms at high bacterial density; in these conditions, the bimodality disappears. These results demonstrate that demographic noise is a potentially important driver of diversity in bacterial community formation and suggest a role for <em>C</em>. <em>elegans</em> as a model system for ecology of host-associated communities.<br />
Author summary<br />
Host-associated bacterial communities—also known as microbiomes—vary extensively between individuals, even among clones exposed to the same environment. The sources of this variation are not entirely understood and can be very difficult to determine. In this manuscript, we demonstrate experimentally how randomness in bacterial colonization can result in large differences in the composition of host-associated bacterial communities, using the nematode worm <em>Caenorhabditis elegans</em> as a tractable host model. We find that the amount of variation between individual communities is a function of two rates relevant to how bacteria colonize the host intestine: the colonization rate and the birth rate. We can manipulate the degree of variation between communities by altering the colonization rate, using the amount of bacteria presented to the worms to control the rate at which migrants enter the intestine. When worms are fed with two neutrally competing, fluorescently labeled bacterial strains at low colonization rates, we are able to produce noise-induced bistability in this system, in which each community is dominated by bacteria of only one color. These results demonstrate the potential importance of noise and randomness as a driver of variation between communities and highlight the utility of the simple model organism <em>C</em>. <em>elegans</em> for studying questions relevant to host-associated microbial communities.</p>
</li>
<li>
<p>system that is stable without diffusion becomes unstable in the presence of diffusion. Turing was motivated to understand morphogenesis with this example of instability</p>
</li>
</ul>
</li>
<li>
<p><em>neutrality in the metaorganism</em><br />
michael sieber et al. 2019<br />
<a href="http://doi.org/10.1371/journal.pbio.3000298">doi.org/10.1371/journal.pbio.3000298</a></p>
<ul>
<li>
<p>Theoretical models offer one way to make the highly complex, individual microbiome composition manageable. A fundamental model in evolutionary research is the so-called neutral null model. This is used to predict how populations would develop without any selection pressure whatsoever. The research team at the CRC 1182 has now applied this model to several model organisms from threadworms to house mice and compared the predictions with experimentally collected data. &quot;Theory and experimental data match surprisingly well for many organisms. The predicted composition in the house mouse, for example, is found in the actual microbial species community,&quot; summarised Dr Michael Sieber, research associate at the MPI-EB and member of the CRC 1182. &quot;t is possible that selection plays a lesser role in the microbiome's composition than we previously assumed, while this does not mean that the microbiome has no important functions for the organism, it could be an indication that many different compositions of the microbiome can perform these functions equally well. And which specific composition actually forms in a single organism is then driven by chance.&quot;</p>
<p>A map for further exploration of the microbiome</p>
<p>The researchers did notice some significant deviations between the neutral model and the real compositions of the microbiome, however. For example, individual bacterial species in the mouse microbiome did not match the neutral prediction. And the microbial species composition of the Caenorhabditis elegans thread worm did not match the neutral model at all.</p>
<p>&quot;We assume that these deviations between model and reality could indicate specific functions of certain microorganisms,&quot; Sieber emphasised. Investigating the systematic deviations from the neutral model therefore holds the potential to discover key functions of certain bacterial species within the microbiome.</p>
<p>First explanations for the deviations from the neutral model are already being discussed. Some non-neutral bacteria in the mouse microbiome, for example, are involved in digestion and their presence may therefore be the result of a targeted selection process. On the other hand, Caenorhabditis elegans, with its very fast generational change, might not live long enough to develop a stable, mainly neutral composition of its microbiome.</p>
</li>
<li>
<p><em>abstract</em> Almost all animals and plants are inhabited by diverse communities of microorganisms, the microbiota, thereby forming an integrated entity, the metaorganism. Natural selection should favor hosts that shape the community composition of these microbes to promote a beneficial host-microbe symbiosis. Indeed, animal hosts often pose selective environments, which only a subset of the environmentally available microbes are able to colonize. How these microbes assemble after colonization to form the complex microbiota is less clear. Neutral models are based on the assumption that the alternatives in microbiota community composition are selectively equivalent and thus entirely shaped by random population dynamics and dispersal. Here, we use the neutral model as a null hypothesis to assess microbiata composition in host organisms, which does not rely on invoking any adaptive processes underlying microbial community assembly. We show that the overall microbiota community structure from a wide range of host organisms, in particular including previously understudied invertebrates, is in many cases consistent with neutral expectations. Our approach allows to identify individual microbes that are deviating from the neutral expectation and are therefore interesting candidates for further study. Moreover, using simulated communities, we demonstrate that transient community states may play a role in the deviations from the neutral expectation. Our findings highlight that the consideration of neutral processes and temporal changes in community composition are critical for an in-depth understanding of microbiota-host interactions.</p>
</li>
</ul>
</li>
</ul>
<h2>05994</h2>
<ul>
<li>
<p><em>complex pectin metabolism by gut bacteria reveals novel catalytic functions</em><br />
didier ndeh et al. 2017<br />
<a href="http://doi.org/10.1038/nature21725">doi.org/10.1038/nature21725</a></p>
<ul>
<li>
<p>single microorganisms in the human gut have the ability to disassemble the most complex of carbohydrates in our diet.<br />
It is the first time such a discovery has been made and it is hoped that this may be used to one day identify new pre- and pro-biotic products to enhance people’s health.<br />
Led by Professor Harry Gilbert, from the Institute for Cell and Molecular Biosciences at Newcastle University, UK, the study is published today (Wednesday) in the leading academic journal, <em>Nature</em>.<br />
Bacteria in the large bowel — the human gut — has a major impact on health and physiology as they help to disintegrate substances in food that we cannot digest, such as starches and fibre.<br />
The main source of nutrients available to the gut bacteria are carbohydrates from the human diet, which the body is unable to metabolise.<br />
The most complex of these carbohydrates is the plant polysaccharide, ‘rhamnogalacturonan II (RG-II)’, which can also be found at elevated levels in red wine.<br />
Previously it was thought that only groups of bacteria would be able to metabolise and breakdown RG-II, reflecting its complex structure. However, this research shows that single organisms present in the gut also have the ability to do this.<br />
Professor Gilbert said: “Our research reports how a highly complex biological process in the body is achieved.<br />
“This is an exciting step forward in the understanding of how human gut bacteria work and has implications for future research.”<br />
The team of international scientists found that RG-II is metabolised through the action of a type of bacterial enzyme, known as glycoside hydrolases, which target the complex carbohydrates sugars in the large bowel.<br />
The bacteria that can metabolise RG-II contain several genes that encode proteins that previously had no known action until now. The group have shown that seven of these genes produce glycoside hydrolases — which split the glycosidic linkage that joins sugars together in polysaccharides — and contribute to the breakdown of RG-II.<br />
Each of these seven glycoside hydrolases are founding members of a novel enzyme family. Three of the glycoside hydrolases that contribute to RG-II degradation break glycosidic linkages that have not previously been shown to be susceptible to biological attack, and these enzymes display novel catalytic functions.<br />
Professor Gilbert said: “This study has potential applications as understanding how this highly complex carbohydrate, which is an integral component of our diet, is utilised offers opportunities for developing new pre- and pro-biotic strategies to improve human health.</p>
</li>
<li>
<p><em>abstract</em> The metabolism of carbohydrate polymers drives microbial diversity in the human gut microbiota. It is unclear, however, whether bacterial consortia or single organisms are required to depolymerize highly complex glycans. Here we show that the gut bacterium <em>Bacteroides thetaiotaomicron</em> uses the most structurally complex glycan known: the plant pectic polysaccharide rhamnogalacturonan-II, cleaving all but 1 of its 21 distinct glycosidic linkages. The deconstruction of rhamnogalacturonan-II side chains and backbone are coordinated to overcome steric constraints, and the degradation involves previously undiscovered enzyme families and catalytic activities. The degradation system informs revision of the current structural model of rhamnogalacturonan-II and highlights how individual gut bacteria orchestrate manifold enzymes to metabolize the most challenging glycan in the human diet.</p>
</li>
</ul>
</li>
</ul>
<h2>06094</h2>
<ul>
<li>
<p><em>numerous uncharacterized and highly divergent microbes which colonize humans are revealed by circulating cell-free dna</em><br />
mark kowarsky et al. 2017<br />
<a href="http://doi.org/10.1073/pnas.1707009114">doi.org/10.1073/pnas.1707009114</a></p>
<ul>
<li>
<p>Through massive shotgun sequencing of circulating cell-free DNA from the blood of more than 1,000 independent samples, we identified hundreds of new bacteria and viruses which represent previously unidentified members of the human microbiome. Previous studies targeted specific niches such as feces, skin, or the oral cavity, whereas our approach of using blood effectively enables sampling of the entire body and reveals the colonization of niches which have been previously inaccessible. We were thus able to discover that the human body contains a vast and unexpected diversity of microbes, many of which have highly divergent relationships to the known tree of life.</p>
<p>Blood circulates throughout the human body and contains molecules drawn from virtually every tissue, including the microbes and viruses which colonize the body. Through massive shotgun sequencing of circulating cell-free DNA from the blood, we identified hundreds of new bacteria and viruses which represent previously unidentified members of the human microbiome. Analyzing cumulative sequence data from 1,351 blood samples collected from 188 patients enabled us to assemble 7,190 contiguous regions (contigs) larger than 1 kbp, of which 3,761 are novel with little or no sequence homology in any existing databases. The vast majority of these novel contigs possess coding sequences, and we have validated their existence both by finding their presence in independent experiments and by performing direct PCR amplification. When their nearest neighbors are located in the tree of life, many of the organisms represent entirely novel taxa, showing that microbial diversity within the human body is substantially broader than previously appreciated.</p>
</li>
</ul>
</li>
<li>
<p><em>microbiota diurnal rhythmicity programs host transcriptome oscillations</em><br />
christoph a. thaiss et al. 2016<br />
<a href="http://doi.org/10.1016/j.cell.2016.11.003">doi.org/10.1016/j.cell.2016.11.003</a></p>
<ul>
<li>
<p>Intestinal microbiota biogeography and metabolome undergo diurnal oscillations</p>
<p>Circadian oscillations of serum metabolites are regulated by the microbiota</p>
<p>Microbiota rhythms program the circadian epigenetic and transcriptional landscape</p>
<p>The microbiota regulates the circadian liver transcriptome and detoxification pattern</p>
<p>The intestinal microbiota undergoes diurnal compositional and functional oscillations that affect metabolic homeostasis, but the mechanisms by which the rhythmic microbiota influences host circadian activity remain elusive. Using integrated multi-omics and imaging approaches, we demonstrate that the gut microbiota features oscillating biogeographical localization and metabolome patterns that determine the rhythmic exposure of the intestinal epithelium to different bacterial species and their metabolites over the course of a day. This diurnal microbial behavior drives, in turn, the global programming of the host circadian transcriptional, epigenetic, and metabolite oscillations. Surprisingly, disruption of homeostatic microbiome rhythmicity not only abrogates normal chromatin and transcriptional oscillations of the host, but also incites genome-wide de novo oscillations in both intestine and liver, thereby impacting diurnal fluctuations of host physiology and disease susceptibility. As such, the rhythmic biogeography and metabolome of the intestinal microbiota regulates the temporal organization and functional outcome of host transcriptional and epigenetic programs.</p>
</li>
</ul>
</li>
<li>
<p><em>hierarchical social networks shape gut microbial composition in wild verreaux’s sifaka</em><br />
amanda c. perofsky et al. 2017<br />
<a href="http://doi.org/10.1098/rspb.2017.2274">doi.org/10.1098/rspb.2017.2274</a></p>
<ul>
<li>In wild primates, social behaviour influences exposure to environmentally acquired and directly transmitted microorganisms. Prior studies indicate that gut microbiota reflect pairwise social interactions among chimpanzee and baboon hosts. Here, we demonstrate that higher-order social network structure—beyond just pairwise interactions—drives gut bacterial composition in wild lemurs, which live in smaller and more cohesive groups than previously studied anthropoid species. Using 16S rRNA gene sequencing and social network analysis of grooming contacts, we estimate the relative impacts of hierarchical (i.e. multilevel) social structure, individual demographic traits, diet, scent-marking, and habitat overlap on bacteria acquisition in a wild population of Verreaux’s sifaka (<em>Propithecus verreauxi</em>) consisting of seven social groups. We show that social group membership is clearly reflected in the microbiomes of individual sifaka, and that social groups with denser grooming networks have more homogeneous gut microbial compositions. Within social groups, adults, more gregarious individuals, and individuals that scent-mark frequently harbour the greatest microbial diversity. Thus, the community structure of wild lemurs governs symbiotic relationships by constraining transmission between hosts and partitioning environmental exposure to microorganisms. This social cultivation of mutualistic gut flora may be an evolutionary benefit of tight-knit group living.</li>
</ul>
</li>
<li>
<p><em>symbiotic skin bacteria as a source for sex-specific scents in frogs</em><br />
andrés e. brunetti et al. 2019<br />
<a href="http://doi.org/10.1073/pnas.1806834116">doi.org/10.1073/pnas.1806834116</a></p>
<ul>
<li>
<p>&quot;Frogs emit a pungent odor. Sometimes a particular species can be recognized by its scent, but until now, the function of this odor was unknown. It was typically assumed to be an aposematic smell, meaning a chemical warning sign that served to repel predators, as in the case of skunks [Mephitis mephitis] among mammals, for example,&quot; said Célio Haddad, a professor at São Paulo State University's Rio Claro Bioscience Institute (IBRC-UNESP) in Brazil and a coauthor of the article.</p>
<p>According to Haddad, who is also affiliated with the university's Aquaculture Center (CAUNESP) in Jaboticabal, this hypothesis was considered plausible because many amphibian species, especially when poisonous, are brightly colored, and this serves as a visual alert to frighten predators. &quot;We thought odor might play a similar role among anurans [frogs and toads],&quot; he said.</p>
<p>The new study resulted from the postdoctoral research of Argentinean biologist Andrés Eduardo Brunetti, supervised by Professor Norberto Peporine Lopes. Conducted at the University of São Paulo's Ribeirão Preto School of Pharmaceutical Sciences (FCFRP-USP), the research was supported by FAPESP.</p>
<p>&quot;The importance and originality of Brunetti's research is that for the first time it shows a pronounced difference in the odors emitted by frogs of opposite sexes,&quot; Haddad said. &quot;No other studies of anurans have ever described this type of behavior. The results suggest that the odor serves to permit mutual recognition between males and females of the same species for mating purposes.&quot;</p>
<p>The research was also supported by the FAPESP Research Program on Biodiversity Characterization, Conservation, Restoration and Sustainable Use (BIOTA-FAPESP) and by the University of São Paulo (USP), the National Council for Scientific and Technological Development (CNPq) and Brazil's Coordination for the Improvement of Higher Education Personnel (CAPES).</p>
<p>&quot;In anurans, you often see different species sharing a lake or marsh. In such places, there are 30 male frogs for every female of the same species on average. The question is how the females recognize males of their own species among a multitude of males belonging to several species while they're all vocalizing at the same time,&quot; Brunetti said.</p>
<p>&quot;It's well-known that the function of the call of anuran males is to attract females and that every species has a characteristic song. Our findings suggest that odor appears to play a similar role, serving as an olfactory signal that enables females to recognize males of their own species.&quot;</p>
<p>Biologists were also unaware of a difference in the scents of male and female frogs. Brunetti discovered this difference during his research, whose primary goal was to understand the chemical composition of the volatile components emitted by the skin of various frog species.</p>
<p>His working hypothesis suggested that smell was a chemical warning sign that served to repel predators. To verify the hypothesis, Brunetti conducted field surveys at several sites in São Paulo state and Rio de Janeiro state, collecting specimens of the tree frog Boana prasina.</p>
<p>&quot;It's very hard to collect females in the wild. Initially, we managed to collect only males. When we noticed what appeared to be a sexual difference in their odors, I went into the field again with the specific aim of capturing females for comparison,&quot; he said.</p>
<p>&quot;During my doctoral research at the Argentinian Natural Science Museum in Buenos Aires, while investigating the volatile compounds in two other frog species, I discovered that the secretions were made up of a blend of 35 to 42 compounds in nine different chemical classes. We then realized that some of the compounds had the specific signature of compounds produced by bacteria.&quot;</p>
<p>Brunetti came to Brazil to investigate whether the selected tree frogs had skin bacteria that produced the characteristic odor of each species, and if so, which compounds they produced. His laboratory research proceeded on two fronts: analysis of the volatile compounds released by the skin of these frogs and identification of the bacteria on their skin.</p>
<p>Brunetti and colleagues used gas chromatography and mass spectrometry to analyze the diversity of the volatile components secreted by the skin of B. prasina. They found that adult males and females secrete a blend of 60-80 compounds, including alcohols, aldehydes, alkenes, ethers, ketones, methoxypyrazines, terpenes and thioethers.</p>
<p>The compounds were exactly the same in both males and females, but the researchers were surprised to find a pronounced sexual difference in the levels of methoxypyrazines, terpenes, and thioethers.</p>
<p>&quot;These three components were responsible for the difference between males and females. Thioethers and methoxypyrazines are typically produced by microorganisms,&quot; Brunetti said.</p>
<p>They decided to determine whether microorganisms were the source of these compounds in B. prasina. To do so, they isolated, cultivated and identified bacteria associated with the skin of these frogs and analyzed their volatile components.</p>
<p>No fewer than 128 different components were detected. Analysis of each component revealed that four methoxypyrazines present in males and females were produced by a single bacterium of the genus Pseudomonas.</p>
<p>In B. prasina, Brunetti discovered, methoxypyrazines were much more abundant in females than in males. Two of the four types of methoxypyrazines were measured at higher levels in females, while two were found at higher levels in males.</p>
<p>Symbiotic relationship</p>
<p>&quot;The interesting thing about Pseudomonas sp. is that these bacteria live on the skin of males and females, where they metabolize the same volatile compounds but at different levels of concentration according to the sex of the host,&quot; Brunetti said.</p>
<p>The levels of methoxypyrazine measured in these frogs, he added, suggest the existence of a complex mechanism of metabolic interactions that creates a different environment on the skin of males and females, favoring the synthesis of characteristic methoxypyrazines in each sex.</p>
<p>&quot;These frogs and bacteria have a symbiotic relationship. In exchange for the service provided by the bacteria, entailing sexual differentiation by scent, the frogs provide an environment -- their own skin -- on which the bacteria can proliferate,&quot; he explained.</p>
<p>The function of this sexual difference in methoxypyrazine levels is unknown. &quot;However, we assume that the difference in scent helps male frogs of this species recognize females of the same species in places inhabited by other frog species,&quot; Brunetti said.</p>
<p>&quot;We know that many anurans use visual communication [bright skin colors] to repel predators as well as acoustic communication [vocalization] to attract female mates. Perhaps B. prasina uses a form of olfactory communication for the same purpose.&quot;</p>
<p>Brunetti will attempt to confirm this hypothesis in future research. If correct, it will have major repercussions. &quot;Only one anuran, in Madagascar, is currently known to communicate by odor. Among amphibians, salamanders, which are distant relatives of anurans, are known to use this form of communication,&quot; Haddad said.</p>
<p>&quot;If B. prasina uses scent as a form of communication, it may well be the case that other species also use olfactory communication, given that each species has a characteristic odor. Brunetti's discovery, if confirmed, opens up a new field of investigation in herpetology, which will now focus on olfactory communication among anurans, rather than just visual and acoustic communication.&quot;</p>
</li>
<li>
<p><em>abstract</em> Amphibians are known to possess a wide variety of compounds stored in their skin glands. While significant progress has been made in understanding the chemical diversity and biological relevance of alkaloids, amines, steroids, and peptides, most aspects of the odorous secretions are completely unknown. In this study, we examined sexual variations in the volatile profile from the skin of the tree frog Boana prasina and combined culture and culture-independent methods to investigate if microorganisms might be a source of these compounds. We found that sesquiterpenes, thioethers, and methoxypyrazines are major contributors to the observed sex differences. We also observed that each sex has a distinct profile of methoxypyrazines, and that the chemical origin of these compounds can be traced to a Pseudomonas sp. strain isolated from the frog’s skin. This symbiotic bacterium was present in almost all individuals examined from different sites and was maintained in captive conditions, supporting its significance as the source of methoxypyrazines in these frogs. Our results highlight the potential relevance of bacteria as a source of chemical signals in amphibians and contribute to increasing our understanding of the role that symbiotic associations have in animals.</p>
</li>
</ul>
</li>
</ul>
<h2>homeostasis</h2>
<h2>06194</h2>
<ul>
<li>communication between microbiome and host
<ul>
<li>
<p><em>microbiota derived short chain fatty acids promote histone crotonylation in the colon through histone deacetylases</em><br />
rachel fellows et al. 2017<br />
<a href="http://doi.org/10.1038/s41467-017-02651-5">doi.org/10.1038/s41467-017-02651-5</a></p>
<ul>
<li>The recently discovered histone post-translational modification crotonylation connects cellular metabolism to gene regulation. Its regulation and tissue-specific functions are poorly understood. We characterize histone crotonylation in intestinal epithelia and find that histone H3 crotonylation at lysine 18 is a surprisingly abundant modification in the small intestine crypt and colon, and is linked to gene regulation. We show that this modification is highly dynamic and regulated during the cell cycle. We identify class I histone deacetylases, HDAC1, HDAC2, and HDAC3, as major executors of histone decrotonylation. We show that known HDAC inhibitors, including the gut microbiota-derived butyrate, affect histone decrotonylation. Consistent with this, we find that depletion of the gut microbiota leads to a global change in histone crotonylation in the colon. Our results suggest that histone crotonylation connects chromatin to the gut microbiota, at least in part, via short-chain fatty acids and HDACs.</li>
</ul>
</li>
<li>
<p><em>gut microbiota orchestrates energy homeostasis during cold</em><br />
chevalier et al 2015<br />
<a href="http://doi.org/10.1016/j.cell.2015.11.004">doi.org/10.1016/j.cell.2015.11.004</a></p>
<ul>
<li>Cold exposure markedly shifts the composition of the gut microbiota. This ‘‘cold microbiota’’ mediates remodeling of the fat and intestinal tissues, helping the host to withstand periods of high energy demand.</li>
<li>Cold exposure leads to marked changes in the gut microbiota composition</li>
<li>Cold microbiota transplantation increases insulin sensitivity and WAT browning</li>
<li>Cold exposure or cold transplantation increase the gut size and absorptive capacity</li>
<li>Reconstitution of cold-suppressed A. muciniphila reverts the increased caloric uptake</li>
</ul>
</li>
<li>
<p><em>a gut microbial factor modulates locomotor behaviour in drosophila</em><br />
catherine e. schretter et al. 2018<br />
<a href="http://doi.org/10.1038/s41586-018-0634-9">doi.org/10.1038/s41586-018-0634-9</a></p>
<ul>
<li>
<p>“This study provides additional evidence for a connection between the gut and the brain, and in particular outlines how gut bacteria may influence behavior, including movement,” said Margaret Sutherland, Ph.D., program director at NINDS.<br />
Researchers led by Sarkis K. Mazmanian, Ph.D., professor of microbiology at the California Institute of Technology in Pasadena, and graduate student Catherine E. Schretter, observed that germ-free flies, which did not carry bacteria, were hyperactive. For instance, they walked faster, over greater distances, and took shorter rests than flies that had normal levels of microbes. Dr. Mazmanian and his team investigated ways in which gut bacteria may affect behavior in fruit flies.<br />
“Locomotion is important for a number of activities such as mating and searching for food. It turns out that gut bacteria may be critical for fundamental behaviors in animals,” said Dr. Mazmanian.<br />
Fruit flies carry between five and 20 different species of bacteria and Dr. Mazmanian’s team treated the germ-free animals with individual strains of those microbes. When the flies received Lactobacillus brevis, their movements slowed down to normal speed. L. brevis was one of only two species of bacteria that restored normal behavior in the germ-free flies.<br />
Dr. Mazmanian’s group also discovered that the molecule xylose isomerase (Xi), a protein that breaks down sugar and is found in L. brevis, may be critical to this process. Isolating the molecule and treating germ-free flies with it was sufficient to slow down the speedwalkers.<br />
Additional experiments showed that Xi may regulate movement by fine-tuning levels of certain carbohydrates, such as trehalose, which is the main sugar found in flies and is similar to mammalian glucose. Flies that were given Xi had lower levels of trehalose than did untreated germ-free flies. When Xi-treated flies, which showed normal behavior, were given trehalose alone, they resumed fast movements suggesting that the sugar was able to reverse the effects of Xi.<br />
Next, the researchers looked into the flies’ nervous system to see what cells were involved in bacteria-directed movement. When Dr. Mazmanian’s team turned on neurons that produce the chemical octopamine, that activation canceled out the effect of L. brevis on the germ-free flies. As a result, the flies, which had previously slowed down after receiving the bacterium or Xi, resumed their speedwalking behavior. Turning on octopamine-producing nerve cells in flies with normal levels of bacteria also caused them to move faster. However, activating neurons that produce other brain chemicals did not influence the flies’ movements.<br />
According to Dr. Mazmanian, Schretter and their colleagues, Xi may be monitoring the flies’ metabolic state, including levels of nutrients, and then signaling to octopamine neurons whether they should turn on or off, resulting in changes in behavior.<br />
Instead of octopamine, mammals produce a comparable chemical called noradrenaline, which has been shown to control movement.<br />
“Gut bacteria may play a similar role in mammalian locomotion, and even in movement disorders such as Parkinson’s disease,” said Dr. Mazmanian.</p>
</li>
<li>
<p><em>abstract</em> While research into the biology of animal behaviour has primarily focused on the central nervous system, cues from peripheral tissues and the environment have been implicated in brain development and function<a href="https://www.nature.com/articles/s41586-018-0634-9#ref-CR1">9</a>. There is emerging evidence that bidirectional communication between the gut and the brain affects behaviours including anxiety, cognition, nociception and social interaction1,2,3,4,5,6,7,8,<a href="https://www.nature.com/articles/s41586-018-0634-9#ref-CR9">9</a>. Coordinated locomotor behaviour is critical for the survival and propagation of animals, and is regulated by internal and external sensory inputs<a href="https://www.nature.com/articles/s41586-018-0634-9#ref-CR10">10</a>,<a href="https://www.nature.com/articles/s41586-018-0634-9#ref-CR11">11</a>. However, little is known about how the gut microbiome influences host locomotion, or the molecular and cellular mechanisms involved. Here we report that germ-free status or antibiotic treatment results in hyperactive locomotor behaviour in the fruit fly Drosophila melanogaster. Increased walking speed and daily activity in the absence of a gut microbiome are rescued by mono-colonization with specific bacteria, including the fly commensal Lactobacillus brevis. The bacterial enzyme xylose isomerase from L. brevis recapitulates the locomotor effects of microbial colonization by modulating sugar metabolism in flies. Notably, thermogenetic activation of octopaminergic neurons or exogenous administration of octopamine, the invertebrate counterpart of noradrenaline, abrogates the effects of xylose isomerase on Drosophila locomotion. These findings reveal a previously unappreciated role for the gut microbiome in modulating locomotion, and identify octopaminergic neurons as mediators of peripheral microbial cues that regulate motor behaviour in animals.</p>
</li>
</ul>
</li>
<li>
<p><em>the drosophila immune deficiency pathway modulates enteroendocrine function and host metabolism</em><br />
layla kamareddine et al. 2018<br />
<a href="http://doi.org/10.1016/j.cmet.2018.05.026">doi.org/10.1016/j.cmet.2018.05.026</a></p>
<ul>
<li>
<p>•The innate immune deficiency (IMD) pathway regulates tachykinin transcription<br />
•Microbial activation of enteroendocrine IMD signaling mediates metabolic homeostasis<br />
•Acetate restores IMD pathway activation and metabolic homeostasis in axenic flies<br />
•Acetate signaling through IMD depends on the membrane-associated receptor PGRP-LC</p>
<p>Enteroendocrine cells (EEs) are interspersed between enterocytes and stem cells in the Drosophila intestinal epithelium. Like enterocytes, EEs express components of the immune deficiency (IMD) innate immune pathway, which activates transcription of genes encoding antimicrobial peptides. The discovery of large lipid droplets in intestines of IMD pathway mutants prompted us to investigate the role of the IMD pathway in the host metabolic response to its intestinal microbiota. Here we provide evidence that the short-chain fatty acid acetate is a microbial metabolic signal that activates signaling through the enteroendocrine IMD pathway in a PGRP-LC-dependent manner. This, in turn, increases transcription of the gene encoding the endocrine peptide Tachykinin (Tk), which is essential for timely larval development and optimal lipid metabolism and insulin signaling. Our findings suggest innate immune pathways not only provide the first line of defense against infection but also afford the intestinal microbiota control over host development and metabolism.</p>
</li>
</ul>
</li>
<li>
<p><em>non-classical immunity controls microbiota impact on skin immunity and tissue repair</em><br />
jonathan l. linehan et al. 2018<br />
<a href="http://doi.org/10.1016/j.cell.2017.12.033">doi.org/10.1016/j.cell.2017.12.033</a></p>
<ul>
<li>
<p>•Non-classical MHC class I molecules promote homeostatic immunity to the microbiota<br />
•Commensal-specific T cells express immunoregulatory and tissue repair signatures<br />
•Commensal-specific T cells accelerate wound closure</p>
<p>Mammalian barrier surfaces are constitutively colonized by numerous microorganisms. We explored how the microbiota was sensed by the immune system and the defining properties of such responses. Here, we show that a skin commensal can induce T cell responses in a manner that is restricted to non-classical MHC class I molecules. These responses are uncoupled from inflammation and highly distinct from pathogen-induced cells. Commensal-specific T cells express a defined gene signature that is characterized by expression of effector genes together with immunoregulatory and tissue-repair signatures. As such, non-classical MHCI-restricted commensal-specific immune responses not only promoted protection to pathogens, but also accelerated skin wound closure. Thus, the microbiota can induce a highly physiological and pleiotropic form of adaptive immunity that couples antimicrobial function with tissue repair. Our work also reveals that non-classical MHC class I molecules, an evolutionarily ancient arm of the immune system, can promote homeostatic immunity to the microbiota.</p>
</li>
</ul>
</li>
<li>
<p><em>intestinal serotonin transporter inhibition by toll-like receptor 2 activation. a feedback modulation</em><br />
latorre e et al. 2016<br />
<a href="http://doi.org/10.1371/journal.pone.0169303">doi.org/10.1371/journal.pone.0169303</a></p>
<ul>
<li>
<p>a protein known as TLR2, a critical detector of the microbiota found in the intestine. They found that it regulates levels of serotonin — a neurotransmitter which carries messages to the brain, and is also found in the gut, where it regulates our bowel routines.<br />
The research, carried out in cell cultures and verified in mice, provides strong evidence that microbiota can interfere with human physiology by modulating the serotonin transporter activity. Serotonin transporter is a target for numerous diseases and it seems that microbiota living in our guts is able to interfere with this transporter, controlling our serotonin levels.<br />
The finding, published in <em>PLOS ONE</em>, comes as scientists across the world are working to understand the complicated interactions between the “invisible world” of the microbiota in our bodies and the impact they have on our health and even our moods. Recently, scientists in California found evidence that the bacteria in the gut play a role in causing Parkinson’s Disease.<br />
It may also help explain how the microbiota in our guts affect our physiology. Inflammatory bowel disease is thought to be triggered when TLR2 is not functioning properly, but so far, the mechanisms behind this have not been fully understood. This study aimed to further this understanding, and was supported the Foundation for the Study of Inflammatory Bowel Diseases in Aragón (ARAINF), in Spain.<br />
Dr Eva Latorre, a postdoctoral researcher at the University of Exeter Medical School, said the new finding helped to further understanding in a fast-growing research area. She said: “This paper has concluded that the protein TLR2 alters the availability of serotonin, which is important in a range of conditions from depression to inflammatory bowel disease.</p>
</li>
<li>
<p><em>abstract</em> TLR2 is a microbiota recognition receptor that has been described to contribute to intestinal homeostasis and to ameliorate inflammatory intestinal injury. In this context, serotonin (5-HT) has shown to be an essential intestinal physiological neuromodulator that is also involved in intestinal inflammatory diseases. Since the interaction between TLR2 activation and the intestinal serotoninergic system remains non-investigated, our main aim was to analyze the effect of TLR2 on intestinal serotonin transporter (SERT) activity and expression and the intracellular pathways involved. Caco-2/TC7 cells were used to analyze SERT and TLR2 molecular expression and SERT activity by measuring 5-HT uptake. The results showed that apical TLR2 activation inhibits SERT activity in Caco-2/TC7 cells mainly by reducing SERT protein level either in the plasma membrane, after short-term TLR2 activation or in both the plasma membrane and cell lysate, after long-term activation. cAMP/PKA pathway appears to mediate short-term inhibitory effect of TLR2 on SERT; however, p38 MAPK pathway has been shown to be involved in both short- and long-term TLR2 effect. Reciprocally, 5-HT long-term treatment yielded TLR2 down regulation in Caco-2/TC7 cells. Finally, results from <em>in vivo</em> showed an augmented intestinal SERT expression in mice <em>Tlr2-/-</em>, thus confirming our inhibitory effect of TLR2 on intestinal SERT <em>in vitro</em>. The present work infers that TLR2 may act in intestinal pathophysiology, not only by its inherent innate immune role, but also by regulating the intestinal serotoninergic system.</p>
</li>
</ul>
</li>
<li>
<p><em>diet-microbiota interactions mediate global epigenetic programming in multiple host tissues</em><br />
kimberly a. krautkramer et al. 2016<br />
<a href="http://doi.org/10.1016/j.molcel.2016.10.25">doi.org/10.1016/j.molcel.2016.10.25</a></p>
<ul>
<li>
<p>A plant-based diet, according to Federico Rey, a UW-Madison professor of bacteriology and also a co-corresponding author of the new report, yields a richer microbiome: “A good diet translates to a beautifully complex microbiome,” Rey says.<br />
“And we see that the gut microbiome affects the host epigenome in a diet-dependent manner. A plant-based diet seems to favor host-microbe communication.”<br />
The new Wisconsin study shows that a small set of short-chain fatty acids produced as the gut bacteria consume, metabolize and ferment nutrients from plants are important chemical messengers, communicating with the cells of the host through the epigenome. “One of the findings here is that microbial metabolism or fermentation of plant fiber results in the production of short-chain fatty acids. These molecules, and potentially many others, are partially responsible for the communication” with the epigenome, says Denu.<br />
In the study, the gut microbiota of the animals that were fed a diet rich in sugar and fat have a diminished capacity to communicate with host cells. According to the Wisconsin team, that may be a hint that the template for a healthy human microbiome was set in the distant past, when food from plants made up a larger portion of diet and sugar and fat were less available than in contemporary diets with more meat and processed foods.<br />
“As we move away from plant-based diets, we may be losing some of that communication between microbes and host,” notes Rey. “With a Western-type diet, it seems like the communication between microbes and host gets lost.”<br />
Foods rich in fat and sugar, especially processed foods, are more easily digested by the host, but are not necessarily a good source of food for the flora inhabiting the gut. The result is a less diverse microbiome and less communication to the host, according to the researchers.<br />
A surprising finding in the study is that the chemical communication between the microbiome and host cells is far reaching. In addition to talking to cells in the colon, the microbiome also seems to be communicating with cells in the liver and in fatty tissue far removed from the gut. That, says Denu, is more evidence of the importance of the microbiome to the well-being of its host.<br />
The kicker experiment in the study, says Denu, was providing mice raised in a germ-free environment with three different short-chain fatty acids that the study showed to be important messengers to the epigenome. The supplement was enough to promote the kind of healthy interplay between microbiota and host cells seen in mice given a diet high in plant fiber.<br />
“It helps show that the collection of three short-chain fatty acids produced in the plant-based diet are likely major communicators,” adds Denu. “We see that it is not just the microbe. It’s microbial metabolism.”</p>
</li>
<li>
<p><em>abstract</em> Gut microbiota alter host histone acetylation and methylation in multiple tissues</p>
<p>Western diet suppresses microbiota-driven SCFA production and chromatin effects</p>
<p>SCFAs recapitulate microbiota-driven chromatin and transcriptional effects</p>
<p>Histone-modifying enzymes regulate transcription and are sensitive to availability of endogenous small-molecule metabolites, allowing chromatin to respond to changes in environment. The gut microbiota produces a myriad of metabolites that affect host physiology and susceptibility to disease; however, the underlying molecular events remain largely unknown. Here we demonstrate that microbial colonization regulates global histone acetylation and methylation in multiple host tissues in a diet-dependent manner: consumption of a “Western-type” diet prevents many of the microbiota-dependent chromatin changes that occur in a polysaccharide-rich diet. Finally, we demonstrate that supplementation of germ-free mice with short-chain fatty acids, major products of gut bacterial fermentation, is sufficient to recapitulate chromatin modification states and transcriptional responses associated with colonization. These findings have profound implications for understanding the complex functional interactions between diet, gut microbiota, and host health.</p>
</li>
</ul>
</li>
<li>
<p><em>akkermansia muciniphila mediates negative effects of ifnγ on glucose metabolism</em><br />
renee l. greer 2016<br />
<a href="http://doi.org/10.1038/NCOMMS13329">doi.org/10.1038/NCOMMS13329</a></p>
<ul>
<li>Cross-talk between the gut microbiota and the host immune system regulates host metabolism, and its dysregulation can cause metabolic disease. Here, we show that the gut microbe <em>Akkermansia muciniphila</em> can mediate negative effects of IFNγ on glucose tolerance. In IFNγ-deficient mice, <em>A. muciniphila</em> is significantly increased and restoration of IFNγ levels reduces <em>A. muciniphila</em> abundance. We further show that IFNγ-knockout mice whose microbiota does not contain <em>A. muciniphila</em> do not show improvement in glucose tolerance and adding back <em>A. muciniphila</em> promoted enhanced glucose tolerance. We go on to identify Irgm1 as an IFNγ-regulated gene in the mouse ileum that controls gut <em>A. muciniphila</em> levels. <em>A. muciniphila</em> is also linked to IFNγ-regulated gene expression in the intestine and glucose parameters in humans, suggesting that this trialogue between IFNγ, <em>A. muciniphila</em> and glucose tolerance might be an evolutionally conserved mechanism regulating metabolic health in mice and humans.</li>
<li>our study revealed a new homeostatic regulatory process in mammalian organisms, where a member of different kingdom, <em>A. muciniphila,</em> constitutes an integral part of the interaction between the supposedly functionally distinct and distant systems of immunity and glucose metabolism. Furthermore, our results and other published work in mouse models and human subjects suggest that this transkingdom interaction may be common in mammals<a href="http://www.nature.com/articles/ncomms13329#ref25">25</a>,<a href="http://www.nature.com/articles/ncomms13329#ref26">26</a>,<a href="http://www.nature.com/articles/ncomms13329#ref27">27</a>. Over the years, biologists have drawn boundaries between systems and kingdoms. Our results highlight the fact that these boundaries must be crossed to fully understand the complexity of living organisms.</li>
</ul>
</li>
<li>
<p><em>c-maf-dependent treg cell control of intestinal th17 cells and iga establishes host–microbiota homeostasis</em><br />
christian neumann et al. 2019<br />
<a href="http://doi.org/10.1038/s41590-019-0316-2">doi.org/10.1038/s41590-019-0316-2</a></p>
<ul>
<li>
<p>The immune system protects against the spread of pathogenic germs in the intestine. At the same time, it allows the colonisation of beneficial microorganisms. Conversely, the composition of the microorganisms in the intestine, the so-called microbiota, has an influence on the quality of the immune reaction. An international research group led by Prof. Dr. Alexander Scheffold of Kiel University (CAU) and the Cluster of Excellence &quot;Precision Medicine in Chronic Inflammation&quot; has uncovered a critical mechanism that establishes the balance between immune system and microbiota.</p>
<p>The researchers Dr. Christian Neumann (Charité), Dr. Sascha Rutz (Genentech, San Francisco), Prof. Dr. Axel Kallies (University of Melbourne and Walter and Eliza Hall Institute of Medical Research, Melbourne), Prof. Scheffold and colleagues studied molecular regulators of immune-microbiome interactions in mice. The team focused on so-called regulatory T cells. These are immune cells that prevent harmless or even useful microorganisms in the intestine from being attacked by the immune system. &quot;We have identified a molecule, c-Maf, which is critical for the development and function of specific regulatory T cells in the gut,&quot; explains Scheffold. C-Maf prevents the immune system from attacking the microbiota. &quot;If this molecule is missing, the gut's immune system overreacts and the microbiota composition changes considerably,&quot; added first author Dr. Neumann of Charité's Institute of Microbiology, Infectious Diseases and Immunology. This change in composition proved remarkably stable: When the researchers transferred the altered microbiota to mice with intact c-Maf-dependent regulatory T cells, they also developed an overreaction of the intestinal immune system.</p>
</li>
<li>
<p><em>abstract</em> Foxp3+ regulatory T cells (Treg cells) are crucial for the maintenance of immune homeostasis both in lymphoid tissues and in non-lymphoid tissues. Here we demonstrate that the ability of intestinal Treg cells to constrain microbiota-dependent interleukin (IL)-17–producing helper T cell (TH17 cell) and immunoglobulin A responses critically required expression of the transcription factor c-Maf. The terminal differentiation and function of several intestinal Treg cell populations, including RORγt+ Treg cells and follicular regulatory T cells, were c-Maf dependent. c-Maf controlled Treg cell–derived IL-10 production and prevented excessive signaling via the kinases PI(3)K (phosphatidylinositol-3-OH kinase) and Akt and the metabolic checkpoint kinase complex mTORC1 (mammalian target of rapamycin) and expression of inflammatory cytokines in intestinal Treg cells. c-Maf deficiency in Treg cells led to profound dysbiosis of the intestinal microbiota, which when transferred to germ-free mice was sufficient to induce exacerbated intestinal TH17 responses, even in a c-Maf-competent environment. Thus, c-Maf acts to preserve the identity and function of intestinal Treg cells, which is essential for the establishment of host–microbe symbiosis.</p>
</li>
</ul>
</li>
<li>
<p><em>microbiota-driven tonic interferon signals in lung stromal cells protect from influenza virus infection</em><br />
konrad c. bradley et al. 2019<br />
<a href="http://doi.org/10.1016/j.celrep.2019.05.105">doi.org/10.1016/j.celrep.2019.05.105</a></p>
<ul>
<li>
<p>signals from gut bacteria help to maintain a first line of defence in the lining of the lung. When mice with healthy gut bacteria were infected with the flu, around 80% of them survived. However, only a third survived if they were given antibiotics before being infected.</p>
<p>&quot;We found that antibiotics can wipe out early flu resistance, adding further evidence that they should not be taken or prescribed lightly,&quot; explains Dr Andreas Wack, who led the research at the Francis Crick Institute. &quot;Inappropriate use not only promotes antibiotic resistance and kills helpful gut bacteria, but may also leave us more vulnerable to viruses. This could be relevant not only in humans but also livestock animals, as many farms around the world use antibiotics prophylactically. Further research in these environments is urgently needed to see whether this makes them more susceptible to viral infections.&quot;</p>
<p>The study found that type I interferon signalling, which is known to regulate immune responses, was key to early defence. Among the genes switched on by interferon is a mouse gene, Mx1, which is the equivalent of the human MxA gene. This antiviral gene produces proteins that can interfere with influenza virus replication. Although often studied in immune cells, the researchers found that microbiota-driven interferon signals also keep antiviral genes in the lung lining active, preventing the virus from gaining a foothold.</p>
<p>&quot;We were surprised to discover that the cells lining the lung, rather than immune cells, were responsible for early flu resistance induced by microbiota,&quot; says Andreas. &quot;Previous studies have focused on immune cells, but we found that the lining cells are more important for the crucial early stages of infection. They are the only place that the virus can multiply, so they are the key battleground in the fight against flu. Gut bacteria send a signal that keeps the cells lining the lung prepared, preventing the virus from multiplying so quickly.</p>
<p>&quot;It takes around two days for immune cells to mount a response, in which time the virus is multiplying in the lung lining. Two days after infection, antibiotic-treated mice had five times more virus in their lungs. To face this bigger threat, the immune response is much stronger and more damaging, leading to more severe symptoms and worse outcomes.&quot;</p>
<p>To test whether the protective effect was related to gut bacteria rather than local processes in the lung, the researchers treated mice with antibiotics and then repopulated their gut bacteria through faecal transplant. This restored interferon signalling and associated flu resistance, suggesting that gut bacteria play a crucial role in maintaining defences.</p>
<p>&quot;Taken together, our findings show that gut bacteria help to keep non-immune cells elsewhere in the body prepared for attack,&quot; says Andreas. &quot;They are better protected from flu because antiviral genes are already switched on when the virus arrives. So when the virus infects a prepared organism, it has almost lost before the battle starts. By contrast, without gut bacteria, the antiviral genes won't come on until the immune response kicks in. This is sometimes too late as the virus has already multiplied many times, so a massive, damaging immune response is inevitable.&quot;</p>
</li>
<li>
<p><em>abstract</em> •Microbiota drive an interferon (IFN) signature in lung stroma cells<br />
•Increased IFN signature impedes early influenza virus replication in lung epithelia<br />
•IFN receptor levels fine-tune the IFN signature<br />
•Antibiotics reduce the IFN signature and facilitate early virus replication</p>
<p>Type I interferon (IFNα/β) pathways are fine-tuned to elicit antiviral protection while minimizing immunopathology; however, the initiating stimuli, target tissues, and underlying mechanisms are unclear. Using models of physiological and dysregulated IFNα/β receptor (IFNAR1) surface expression, we show here that IFNAR1-dependent signals set the steady-state IFN signature in both hematopoietic and stromal cells. Increased IFNAR1 levels promote a lung environment refractory to early influenza virus replication by elevating the baseline interferon signature. Commensal microbiota drive the IFN signature specifically in lung stroma, as shown by antibiotic treatment and fecal transplantation. Bone marrow chimera experiments identify lung stromal cells as crucially important for early antiviral immunity and stroma-immune cell interaction for late antiviral resistance. We propose that the microbiota-driven interferon signature in lung epithelia impedes early virus replication and that IFNAR1 surface levels fine-tune this signature. Our findings highlight the interplay between bacterial and viral exposure, with important implications for antibiotic use.</p>
</li>
</ul>
</li>
<li>
<p><em>microbiota-driven tonic interferon signals in lung stromal cells protect from influenza virus infection</em><br />
konrad c. bradley et al. 2019<br />
<a href="http://doi.org/10.1016/j.celrep.2019.05.105">doi.org/10.1016/j.celrep.2019.05.105</a></p>
<ul>
<li>
<p>signals from gut bacteria help to maintain a first line of defence in the lining of the lung. When mice with healthy gut bacteria were infected with the flu, around 80% of them survived. However, only a third survived if they were given antibiotics before being infected.<br />
“We found that antibiotics can wipe out early flu resistance, adding further evidence that they should not be taken or prescribed lightly,” explains Dr Andreas Wack, who led the research at the Francis Crick Institute. “Inappropriate use not only promotes antibiotic resistance and kills helpful gut bacteria, but may also leave us more vulnerable to viruses. This could be relevant not only in humans but also livestock animals, as many farms around the world use antibiotics prophylactically. Further research in these environments is urgently needed to see whether this makes them more susceptible to viral infections.”<br />
The study found that type I interferon signalling, which is known to regulate immune responses, was key to early defence. Among the genes switched on by interferon is a mouse gene, Mx1, which is the equivalent of the human MxA gene. This antiviral gene produces proteins that can interfere with influenza virus replication. Although often studied in immune cells, the researchers found that microbiota-driven interferon signals also keep antiviral genes in the lung lining active, preventing the virus from gaining a foothold.<br />
“We were surprised to discover that the cells lining the lung, rather than immune cells, were responsible for early flu resistance induced by microbiota,” says Andreas. “Previous studies have focused on immune cells, but we found that the lining cells are more important for the crucial early stages of infection. They are the only place that the virus can multiply, so they are the key battleground in the fight against flu. Gut bacteria send a signal that keeps the cells lining the lung prepared, preventing the virus from multiplying so quickly.<br />
“It takes around two days for immune cells to mount a response, in which time the virus is multiplying in the lung lining. Two days after infection, antibiotic-treated mice had five times more virus in their lungs. To face this bigger threat, the immune response is much stronger and more damaging, leading to more severe symptoms and worse outcomes.”<br />
To test whether the protective effect was related to gut bacteria rather than local processes in the lung, the researchers treated mice with antibiotics and then repopulated their gut bacteria through faecal transplant. This restored interferon signalling and associated flu resistance, suggesting that gut bacteria play a crucial role in maintaining defences.<br />
“Taken together, our findings show that gut bacteria help to keep non-immune cells elsewhere in the body prepared for attack,” says Andreas. “They are better protected from flu because antiviral genes are already switched on when the virus arrives. So when the virus infects a prepared organism, it has almost lost before the battle starts. By contrast, without gut bacteria, the antiviral genes won’t come on until the immune response kicks in. This is sometimes too late as the virus has already multiplied many times, so a massive, damaging immune response is inevitable.”</p>
</li>
<li>
<p><em>abstract</em> •Microbiota drive an interferon (IFN) signature in lung stroma cells<br />
•Increased IFN signature impedes early influenza virus replication in lung epithelia<br />
•IFN receptor levels fine-tune the IFN signature<br />
•Antibiotics reduce the IFN signature and facilitate early virus replication</p>
<p>Type I interferon (IFNα/β) pathways are fine-tuned to elicit antiviral protection while minimizing immunopathology; however, the initiating stimuli, target tissues, and underlying mechanisms are unclear. Using models of physiological and dysregulated IFNα/β receptor (IFNAR1) surface expression, we show here that IFNAR1-dependent signals set the steady-state IFN signature in both hematopoietic and stromal cells. Increased IFNAR1 levels promote a lung environment refractory to early influenza virus replication by elevating the baseline interferon signature. Commensal microbiota drive the IFN signature specifically in lung stroma, as shown by antibiotic treatment and fecal transplantation. Bone marrow chimera experiments identify lung stromal cells as crucially important for early antiviral immunity and stroma-immune cell interaction for late antiviral resistance. We propose that the microbiota-driven interferon signature in lung epithelia impedes early virus replication and that IFNAR1 surface levels fine-tune this signature. Our findings highlight the interplay between bacterial and viral exposure, with important implications for antibiotic use.</p>
</li>
</ul>
</li>
<li>
<p><em>intestinal serotonin and fluoxetine exposure modulate bacterial colonization in the gut</em><br />
thomas c. fung et al. 2019<br />
<a href="http://doi.org/10.1038/s41564-019-0540-4">doi.org/10.1038/s41564-019-0540-4</a></p>
<ul>
<li>
<p>Hsiao and her research group reported in the journal Cell in 2015 that in mice, a specific mixture of bacteria, consisting mainly of Turicibacter sanguinis and Clostridia, produces molecules that signal to gut cells to increase production of serotonin. When Hsiao's team raised mice without the bacteria, more than 50% of their gut serotonin was missing. The researchers then added the bacteria mixture of mainly Turicibacter and Clostridia, and their serotonin increased to a normal level.</p>
<p>That study got the team wondering why bacteria signal to our gut cells to make serotonin. Do microbes use serotonin, and if so, for what?</p>
<p>In this new study, the researchers added serotonin to the drinking water of some mice and raised others with a mutation (created by altering a specific serotonin transporter gene) that increased the levels of serotonin in their guts. After studying the microbiota of the mice, the researchers discovered that the bacteria Turicibacter and Clostridia increased significantly when there was more serotonin in the gut.</p>
<p>If these bacteria increase in the presence of serotonin, perhaps they have some cellular machinery to detect serotonin, the researchers speculated. Together with study co-author Lucy Forrest and her team at the National Institutes of Health's National Institute of Neurological Disorders and Stroke, the researchers found a protein in multiple species of Turicibacter that has some structural similarity to a protein that transports serotonin in mammals. When they grew Turicibacter sanguinis in the lab, they found that the bacterium imports serotonin into the cell.</p>
<p>In another experiment, the researchers added the antidepressant fluoxetine, which normally blocks the mammalian serotonin transporter, to a tube containing Turicibacter sanguinis. They found the bacterium transported significantly less serotonin.</p>
<p>The team found that exposing Turicibacter sanguinis to serotonin or fluoxetine influenced how well the bacterium could thrive in the gastrointestinal tract. In the presence of serotonin, the bacterium grew to high levels in mice, but when exposed to fluoxetine, the bacterium grew to only low levels in mice.</p>
<p>&quot;Previous studies from our lab and others showed that specific bacteria promote serotonin levels in the gut,&quot; Fung said. &quot;Our new study tells us that certain gut bacteria can respond to serotonin and drugs that influence serotonin, like anti-depressants. This is a unique form of communication between bacteria and our own cells through molecules traditionally recognized as neurotransmitters.&quot;</p>
<p>The team's research on Turicibacter aligns with a growing number of studies reporting that anti-depressants can alter the gut microbiota. &quot;For the future,&quot; Hsiao said, &quot;we want to learn whether microbial interactions with antidepressants have consequences for health and disease.&quot;</p>
</li>
<li>
<p><em>abstract</em> The gut microbiota regulates levels of serotonin (5-hydroxytryptamine (5-HT)) in the intestinal epithelium and lumen1,2,3,4,5. However, whether 5-HT plays a functional role in bacteria from the gut microbiota remains unknown. We demonstrate that elevating levels of intestinal lumenal 5-HT by oral supplementation or genetic deficiency in the host 5-HT transporter (SERT) increases the relative abundance of spore-forming members of the gut microbiota, which were previously reported to promote host 5-HT biosynthesis. Within this microbial community, we identify Turicibacter sanguinis as a gut bacterium that expresses a neurotransmitter sodium symporter-related protein with sequence and structural homology to mammalian SERT. T. sanguinis imports 5-HT through a mechanism that is inhibited by the selective 5-HT reuptake inhibitor fluoxetine. 5-HT reduces the expression of sporulation factors and membrane transporters in T. sanguinis, which is reversed by fluoxetine exposure. Treating T. sanguinis with 5-HT or fluoxetine modulates its competitive colonization in the gastrointestinal tract of antibiotic-treated mice. In addition, fluoxetine reduces the membership of T. sanguinis in the gut microbiota of conventionally colonized mice. Host association with T. sanguinis alters intestinal expression of multiple gene pathways, including those important for lipid and steroid metabolism, with corresponding reductions in host systemic triglyceride levels and inguinal adipocyte size. Together, these findings support the notion that select bacteria indigenous to the gut microbiota signal bidirectionally with the host serotonergic system to promote their fitness in the intestine.</p>
</li>
</ul>
</li>
<li>
<p><em>the intestinal microbiota programs diurnal rhythms in host metabolism through histone deacetylase 3</em><br />
zheng kuang et al. 2019<br />
<a href="http://doi.org/10.1126/science.aaw3134">doi.org/10.1126/science.aaw3134</a></p>
<ul>
<li>
<p>commensal, or good, bacteria that live in the guts of mammals program the metabolic rhythms that govern the body's absorption of dietary fat. Dr. Hooper, Chair of Immunology and a Howard Hughes Medical Institute Investigator, is senior author of the study.</p>
<p>The study also found that microbes program these so-called circadian rhythms by activating a protein named histone deacetylase 3 (HDAC3), which is made by cells that line the gut. Those cells act as intermediaries between bacteria that aid in digestion of food and proteins that enable absorption of nutrients.</p>
<p>The study, done in mice, revealed that HDAC3 turns on genes involved in the absorption of fat. They found that HDAC3 interacts with the biological clock machinery within the gut to refine the rhythmic ebb and flow of proteins that enhance absorption of fat. This regulation occurs in the daytime in humans, who eat during the day, and at night in mice, which eat at night.</p>
<p>&quot;The microbiome actually communicates with our metabolic machinery to make fat absorption more efficient. But when fat is overabundant, this communication can result in obesity. Whether the same thing is going on in other mammals, including humans, is the subject of future studies,&quot; added lead author Dr. Zheng Kuang, a postdoctoral fellow in the Hooper laboratory.</p>
<p>To go back in time, the story really starts with a few mice and crosstalk between two laboratories at UT Southwestern.</p>
<p>Dr. Hooper, who runs the University's colony of germ-free mice, which are raised in environments that have no microbes, is also a Professor of Immunology and Microbiology and a member of the Center for the Genetics of Host Defense. She holds the Jonathan W. Uhr, M.D. Distinguished Chair in Immunology, and is a Nancy Cain and Jeffrey A. Marcus Scholar in Medical Research, in Honor of Bill S. Vowell.</p>
<p>Histone modifications -- which are made by enzymes like HDAC3 -- control the expression of genes that in turn make proteins that carry out the work of the cell. Not long ago, the Hooper laboratory decided to do a mouse study of histone modifications that seemed to rise and fall along with circadian rhythms.</p>
<p>In comparing normal, bacteria-laden mice with germ-free ones, researchers discovered some histone modifications -- including those made by HDAC3 -- were circadian in normal mice, but held steady at a flat level in germ-free mice.</p>
<p>That's when Dr. Hooper contacted Dr. Eric Olson, Chair of Molecular Biology and Director of the Hamon Center for Regenerative Science and Medicine, who had done studies on HDAC3 in a different tissue, the heart. The two laboratories collaborated to develop a mouse that lacked HDAC3 only in the gut lining.</p>
<p>The mice they generated seemed unremarkable while eating a normal chow diet. However, when the researchers fed the mice a high fat, high sugar diet similar to one commonly consumed in the United States -- they found something very different.</p>
<p>&quot;We call it the junk food diet. I describe it as like driving through a fast food restaurant for a burger and fries and then stopping off at the donut shop,&quot; she said. &quot;Most mice on that diet become obese. To our surprise, those that had no HDAC3 in their intestinal lining were able to eat a high fat, high sugar diet and stay lean.&quot;</p>
<p>Next, they compared the HDAC3-deficient mice to the germ-free mice. The researchers found that both groups of mice showed the same flat, nonrhythmic histone modifications, confirming HDAC3's importance in circadian rhythms.</p>
<p>Every cell in the body has a molecular clock that governs bodily processes. The mouse study revealed that HDAC3 attaches to that cellular clock machinery to ensure absorption of fat is highest when mammals are awake and eating.</p>
<p>&quot;Our results suggest that the microbiome and the circadian clock have evolved to work together to regulate metabolism,&quot; she said.</p>
<p>Why would a system evolve to make us fat? Dr. Hooper believes it could have evolved to enable mammals to use energy efficiently in order to boost immunity in an environment with food scarcity.</p>
<p>&quot;This regulatory interaction probably didn't evolve to make us obese, but when combined with today's calorie-rich diets, obesity arises,&quot; she said, adding that this is speculation and the team is still working to understand all the components of the pathway.</p>
<p>&quot;Our results also suggest that disrupting the interactions between the microbiota and the body's clock could make us more likely to become obese. These disruptions happen frequently in modern life when we take antibiotics, work overnight shifts, or travel internationally. But we think that our findings might eventually lead to new treatments for obesity -- and possibly malnutrition -- by altering the bacteria in our guts.&quot;</p>
</li>
<li>
<p><em>abstract</em> Circadian rhythmicity is a defining feature of mammalian metabolism that synchronizes metabolic processes to day-night light cycles. Here, we show that the intestinal microbiota programs diurnal metabolic rhythms in the mouse small intestine through histone deacetylase 3 (HDAC3). The microbiota induced expression of intestinal epithelial HDAC3, which was recruited rhythmically to chromatin, and produced synchronized diurnal oscillations in histone acetylation, metabolic gene expression, and nutrient uptake. HDAC3 also functioned noncanonically to coactivate estrogen-related receptor α, inducing microbiota-dependent rhythmic transcription of the lipid transporter gene Cd36 and promoting lipid absorption and diet-induced obesity. Our findings reveal that HDAC3 integrates microbial and circadian cues for regulation of diurnal metabolic rhythms and pinpoint a key mechanism by which the microbiota controls host metabolism.</p>
</li>
</ul>
</li>
</ul>
</li>
<li>fitness and longevity
<ul>
<li>
<p><em>microbiome interactions shape host fitness</em><br />
alison l. gould et al. 2018<br />
<a href="http://doi.org/10.1073/pnas.1809349115">doi.org/10.1073/pnas.1809349115</a></p>
<ul>
<li>
<p>UC Santa Barbara physicists Eric Jones and Jean Carlson have developed a mathematical approach to analyze and model interactions between gut bacteria in fruit flies. This method could lead to a more sophisticated understanding of the complex interactions between human gut microbes.</p>
<p>Their finding appear in the Proceedings of the National Academy of Sciences.</p>
<p>&quot;Especially over the past 20 years or so, scientists have been finding that the microbiome interacts with the rest of your body, with your immune system, with your brain,&quot; said Jones, a graduate student researcher in Carlson's lab. &quot;Many diseases are associated with certain microbial compositions in the gut.&quot;</p>
<p>The human gut microbiome as yet is too diverse to fully analyze. Instead, the research team, led by Carnegie Institution for Science biologist Will Ludington, used the fruit fly as a model organism to tease apart how the presence of particular gut bacteria could lead to physical and behavioral effects in the host organism.</p>
<p>In their paper, &quot;Microbiome interactions shape host fitness,&quot; Carlson, Jones, Ludington and colleagues examine the interactions between five core species of bacteria found in the fly gut, and calculate how the presence or absence of individual species influences aspects of the fly's fitness, including lifespan, fertility and development. &quot;The classic way we think about bacterial species is in a black-and-white context as agents of disease -- either you have it or you don't,&quot; Ludington said. &quot;Our work shows that isn't the case for the microbiome. The effects of a particular species depend on the context of which other species are also present.&quot;</p>
<p>Building on previous research that found the presence versus the absence of bacteria affected the longevity of an organism (sterile hosts lived longer), the researchers' work on this project revealed that the situation is far more nuanced. For example, the presence of certain bacteria might increase the host's fecundity, while others might decrease longevity. &quot;As we examined the total of what we call a fly's fitness -- it's chances of surviving and creating offspring -- we found that there was a tradeoff between having a short lifespan with lots of offspring, versus having a long lifespan with few offspring,&quot; Ludington explained. &quot;This tradeoff was mediated by microbiome interactions.&quot;</p>
<p>To decipher these interactions, Ludington performed a combinatorial assay, rearing 32 batches of flies each inhabited by a unique combination of the five bacteria. For each bacterial combination, Ludington measured the fly's development, fecundity and longevity. The analysis of the interactions required Carlson and Jones to develop new mathematical approaches.</p>
<p>&quot;One model that often would be a starting point would be to consider the interactions between pairs of bacteria,&quot; said Carlson, whose research delves into the physics of complex systems. &quot;This research shows us that a strictly pairwise model does not capture all of the observed fly traits.&quot;</p>
<p>What the study shows, the researchers said, is that the interactions between the bacterial populations are as significant to the host's overall fitness as their presence -- the microbiome's influence cannot be solely attributed to the presence or absence of individual species. &quot;In a sense,&quot; said Jones, &quot;the microbiome's influence on the host is more than the sum of its parts.&quot;</p>
<p>The newly developed models could be extended to better understand the interactions of the thousands of different species of bacteria in the human microbiome, which could, in turn, shed light on the many connections to microbiome-affiliated diseases including mood disorders, neurological dysfunctions, autoimmune diseases and antibiotic-resistant superbugs.</p>
<p>&quot;In many cases infections are caused by bacteria that we all have in ourselves all the time, and are kept in check by native gut bacteria,&quot; Carlson said. It's not so much that the infection is some new, horrible bacteria, she explained, but that the populations of other bacteria have changed, resulting in unrestricted growth for the infectious bacteria.</p>
<p>&quot;It's really about understanding the population dynamics of these systems,&quot; she said.</p>
</li>
<li>
<p><em>abstract</em> Gut bacteria can affect key aspects of host fitness, such as development, fecundity, and lifespan, while the host, in turn, shapes the gut microbiome. However, it is unclear to what extent individual species versus community interactions within the microbiome are linked to host fitness. Here, we combinatorially dissect the natural microbiome of <em>Drosophila melanogaster</em> and reveal that interactions between bacteria shape host fitness through life history tradeoffs. Empirically, we made germ-free flies colonized with each possible combination of the five core species of fly gut bacteria. We measured the resulting bacterial community abundances and fly fitness traits, including development, reproduction, and lifespan. The fly gut promoted bacterial diversity, which, in turn, accelerated development, reproduction, and aging: Flies that reproduced more died sooner. From these measurements, we calculated the impact of bacterial interactions on fly fitness by adapting the mathematics of genetic epistasis to the microbiome. Development and fecundity converged with higher diversity, suggesting minimal dependence on interactions. However, host lifespan and microbiome abundances were highly dependent on interactions between bacterial species. Higher-order interactions (involving three, four, and five species) occurred in 13–44% of possible cases depending on the trait, with the same interactions affecting multiple traits, a reflection of the life history tradeoff. Overall, we found these interactions were frequently context-dependent and often had the same magnitude as individual species themselves, indicating that the interactions can be as important as the individual species in gut microbiomes.</p>
</li>
</ul>
</li>
<li>
<p><em>microbial genetic composition tunes host longevity</em><br />
bing han et al. 2017<br />
<a href="http://doi.org/10.1016/j.cell.2017.05.036">doi.org/10.1016/j.cell.2017.05.036</a></p>
<ul>
<li>
<pre><code>    •Systematic analysis of longevity-promoting microbial genetic variations  
      •Colanic acid as a pro-longevity natural compound effective in different species  
      •Bacterial metabolites regulate host mitochondrial dynamics and UPRmit  
</code></pre>
<p>Homeostasis of the gut microbiota critically influences host health and aging. Developing genetically engineered probiotics holds great promise as a new therapeutic paradigm to promote healthy aging. Here, through screening 3,983 <em>Escherichia coli</em> mutants, we discovered that 29 bacterial genes, when deleted, increase longevity in the host <em>Caenorhabditis elegans</em>. A dozen of these bacterial mutants also protect the host from age-related progression of tumor growth and amyloid-beta accumulation. Mechanistically, we discovered that five bacterial mutants promote longevity through increased secretion of the polysaccharide colanic acid (CA), which regulates mitochondrial dynamics and unfolded protein response (UPRmt) in the host. Purified CA polymers are sufficient to promote longevity via ATFS-1, the host UPRmt-responsive transcription factor. Furthermore, the mitochondrial changes and longevity effects induced by CA are conserved across different species. Together, our results identified molecular targets for developing pro-longevity microbes and a bacterial metabolite acting on host mitochondria to promote longevity.</p>
</li>
<li>
<p>&quot;These findings are also interesting and have implications from the biological point of view in the way we understand host-microbe communication,&quot; Wang said. &quot;Mitochondria seem to have evolved from bacteria that millions of years ago entered primitive cells. Our finding suggests that products from bacteria today can still chime in the communication between mitochondria in our cells. We think that this type of communication is very important and here we have provided the first evidence of this. Fully understanding microbe-mitochondria communication can help us understand at a deeper level the interactions between microbes and their hosts.&quot;</p>
</li>
</ul>
</li>
</ul>
</li>
</ul>
<h2>06294</h2>
<ul>
<li>promoted by diet
<ul>
<li>
<p><em>effects of high-fiber diets and macronutrient substitution on bloating</em><br />
mingyu zhang et al. 2020<br />
<a href="http://doi.org/10.14309/ctg.0000000000000122">doi.org/10.14309/ctg.0000000000000122</a></p>
<ul>
<li>analyzed data from a clinical trial of high fiber diets. They determined that when the 164 trial participants ate versions of a heart-healthy, high-fiber diet that was relatively rich in plant protein, they were about 40 percent more likely to report bloating symptoms than when eating a carbohydrate-rich version of the same high fiber diet.<br />
The study suggests that people who want to eat a high fiber diet would be less likely to experience bloating if the diet were relatively carb-rich versus protein-rich.<br />
High-fiber diets are believed to cause bloating by boosting certain populations of healthful fiber-digesting gut bacteria species, which produce gas as a byproduct. The findings thus also hint at a role for “macronutrients” such as carbs and proteins in modifying the gut bacteria population — the microbiome.<br />
“It’s possible that in this study, the protein-rich version of the diet caused more bloating because it caused more of a healthy shift in the composition of the microbiome,” says study co-senior author Noel Mueller, PhD, an assistant professor in the Department of Epidemiology at the Bloomberg School. “Notably, the protein in these diets was mostly from vegetable sources such as beans, legumes, and nuts.”<br />
Bloating affects about 20 percent of the U.S. adult population, and is so common as a side effect of high fiber diets that it deters many people from adopting such diets. Mueller and colleagues in recent years have been re-examining data from past, high quality clinical trials to find dietary factors that might modify bloating frequency in the context of a high fiber diet. Last year, Mueller and colleagues reported that salt appeared to be one such factor. It was associated with more bloating in a trial of a heart-healthy, high-fiber diet, suggesting that cutting back on salt could be one easy way to reduce bloating.<br />
In the new study, the researchers examined a dietary clinical trial that was conducted in 2003 and 2005 at the Johns Hopkins ProHealth Clinical Research Unit in Baltimore and at the Brigham and Women’s Hospital in Boston. Known as the Optimal Macronutrient Intake Trial to Prevent Heart Disease (OmniHeart), it included 164 participants who had above-normal blood pressure. They were assigned to three different diets over consecutive six-week periods separated by two-week “washout” intervals during which participants returned to regular eating habits.<br />
The diets were all considered high-fiber, low-sodium “DASH” diets, and had the same number of calories, but varied in their macronutrient emphases: a carbohydrate-rich version was, by calories, 58 percent carbohydrate, 15 percent protein, and 27 percent fat; a plant-protein-rich version was 48 percent carbs, 25 percent protein, 27 percent fat; and a fat-rich version was 48 percent carbohydrate, 15 percent protein, and 37 percent fat.<br />
The primary results of the OmniHeart trial, published in 2005, suggested that the plant-protein-rich and fat-rich diets were the most effective in reducing blood pressure and improving measures of blood cholesterol (reducing LDL and triglycerides; increasing HDL).<br />
In their new analysis of this data, Mueller and colleagues examined how participants’ reports of bloating — which were among the secondary data collected in that trial — varied as participants ate the three OmniHeart diets. A key finding was that the prevalence of bloating went from 18 percent before the diets to 24, 33, and 30 percent, respectively, on the carb-, protein-, and fat-rich diets — indicating that these high fiber diets did indeed appear to increase bloating.<br />
The researchers also analyzed the relative changes among the diets, and linked the protein-rich diet to a significantly greater chance of bloating — roughly 40 percent greater — in comparison with the carb-rich diet.<br />
The results suggest that substituting high quality carb calories, such as whole grain, for protein calories might reduce bloating for those on high fiber diets, making such diets more tolerable.<br />
There is the possibility, however, that making high fiber diets more tolerable in this way would also make them less healthy. The plant- protein- and fat-rich diets in the study, which led to higher bloating prevalence, also appeared, in the original OmniHeart trial analysis, to lower blood pressure and improve blood lipid measures more than the carb-rich diet.<br />
Researchers have been uncovering evidence that many of the effects of a healthier diet come from shifts in the gut microbiome that result in greater microbial production of health-promoting molecules called metabolites. Mueller suspects that the plant-protein-rich diet caused more bloating because it caused a greater and healthier shift in the microbiome.<br />
“Bloating may be just a consequence of a healthy shift in the microbiome, so that if somebody is able to put up with the bloating caused by a high-protein, high-fiber diet, they may ultimately benefit more in other health measures,” Mueller says.<br />
He and his colleagues are working on a follow-on study of the effects of similar dietary patterns on the gut microbiome.</li>
</ul>
</li>
<li>
<p>human microbiome produces protective propionate when fed fibre</p>
<ul>
<li>
<p><em>the short-chain fatty acid propionate protects from hypertensive cardiovascular damage</em><br />
hendrik bartolomaeus et al. 2018<br />
<a href="http://doi.org/10.1161/circulationaha.118.036652">doi.org/10.1161/circulationaha.118.036652</a></p>
<ul>
<li>
<p>&quot;You are what you eat,&quot; as the proverb goes. But to a large extent our well-being also depends on what bacterial guests in our digestive tract consume. That's because gut flora help the human body to utilize food and produce essential micronutrients, including vitamins.</p>
<p>Beneficial gut microbes can produce metabolites from dietary fiber, including a fatty acid called propionate. This substance protects against the harmful consequences of high blood pressure. A Berlin research team from the Experimental and Clinical Research Center (ECRC), a joint institution of the Max Delbrück Center for Molecular Medicine (MDC) and Charité -- Universitätsmedizin Berlin, shows why this is the case. Their study has been published in advance online in the journal Circulation.</p>
<p>The researchers fed propionate to mice with elevated blood pressure. Afterwards, the animals had less pronounced damage to the heart or abnormal enlargement of the organ, making them less susceptible to cardiac arrhythmia. Vascular damage, such as atherosclerosis, also decreased in mice. &quot;Propionate works against a range of impairments in cardiovascular function caused by high blood pressure,&quot; says MDC researcher and research group leader Professor Dominik N. Müller. &quot;This may be a promising treatment option, particularly for patients who have too little of this fatty acid.&quot;</p>
<p>Detour via the immune system</p>
<p>&quot;Our study made it clear that the substance takes a detour via the immune system and thus affects the heart and blood vessels,&quot; say Dr. Nicola Wilck and Hendrik Bartolomaeus from the ECRC, who have been working together on the project for nearly five years. In particular, T helper cells, which enhance inflammatory processes and contribute to high blood pressure, were calmed.</p>
<p>This has a direct effect on the functional ability of the heart, for example. The research team triggered heart arrhythmia in 70 percent of the untreated mice through targeted electrical stimuli. However, only one-fifth of the animals treated with the fatty acid were susceptible to an irregular heartbeat. Further investigations with ultrasound, tissue sections, and single-cell analyses showed that propionate also reduced blood pressure-related damage to the animals' cardiovascular system, significantly increasing their survival rate.</p>
<p>But when researchers deactivated a certain T cell subtype in the mice's bodies, known as regulatory T cells, the positive effects of propionate disappeared. The immune cells are therefore indispensable for the substance's beneficial effect. A research group under Johannes Stegbauer, an adjunct professor at Düsseldorf University Hospital, confirmed the team's findings in a second animal model.</p>
<p>Short-chain fatty acid as a therapeutic option</p>
<p>The results explain why a diet rich in fiber, which has been recommended by nutrition organizations for many years, helps prevent cardiovascular diseases. Whole-grain products and fruits, for example, contain cellulose and inulin fibers, from which gut bacteria produce the beneficial molecules like propionate, a short-chain fatty acid with a backbone of just three carbon atoms.</p>
<p>&quot;Previously, it had not been clear which fatty acid is behind the positive effects and how it works,&quot; says Wilck. The study opens up new avenues in the treatment of cardiovascular diseases. &quot;It might make sense to administer propionate or a chemical precursor directly as a drug&quot; -- for example, when the blood of those affected contains too little of the substance.</p>
<p>Propionate still has to prove itself in everyday clinical practice. The research team now hopes to validate their findings by examining the substance's effects on human subjects. It is already known that propionate is safe for human consumption and can also be produced economically: The substance has been used for centuries as a preservative, for example. It is already approved as a food additive. &quot;With these favorable conditions, hopefully propionate will soon make the leap from the lab to patients who need it,&quot; says Wilck.</p>
</li>
<li>
<p><em>abstract</em> <strong>Background:</strong> Arterial hypertension and its organ sequelae show characteristics of T cell mediated inflammatory diseases. Experimental anti-inflammatory therapies have been shown to ameliorate hypertensive end-organ damage. Recently, the CANTOS study targeting interleukin-1β demonstrated that anti-inflammatory therapy reduces cardiovascular risk. The gut microbiome plays pivotal role in immune homeostasis and cardiovascular health. Short-chain fatty acids (SCFA) are produced from dietary fiber by gut bacteria and affect host immune homeostasis. Here, we investigated effects of the SCFA propionate in two different mouse models of hypertensive cardiovascular damage.<br />
<strong>Methods:</strong> To investigate the effect of SCFA on hypertensive cardiac damage and atherosclerosis, wild-type NMRI (WT) or ApoE-/- deficient mice received propionate (200mM) or control in the drinking water. To induce hypertension, WT mice were infused with Angiotensin (Ang)II (1.44mg/kg/d s.c.) for 14 days. To accelerate the development of atherosclerosis, ApoE-/- mice were infused with AngII (0.72mg/kg/d s.c.) for 28 days. Cardiac damage and atherosclerosis were assessed using histology, echocardiography, <em>in vivo</em> electrophysiology, immunofluorescence, and flow cytometry. Blood pressure was measured by radiotelemetry. Regulatory T cell (Treg) depletion using PC61 antibody was used to examine the mode of action of propionate.<br />
<strong>Results:</strong> Propionate significantly attenuated cardiac hypertrophy, fibrosis, vascular dysfunction, and hypertension in both models. Susceptibility to cardiac ventricular arrhythmias was significantly reduced in propionate-treated AngII-infused WT mice. Aortic atherosclerotic lesion area was significantly decreased in propionate-treated ApoE-/-. Systemic inflammation was mitigated by propionate treatment, quantified as a reduction in splenic effector memory T cell frequencies and splenic T helper 17 cells in both models, and a decrease in local cardiac immune cell infiltration in WT mice. Cardioprotective effects of propionate were abrogated in Treg-depleted AngII-infused mice, suggesting the effect is Treg-dependent.<br />
<strong>Conclusions:</strong> Our data emphasize an immune-modulatory role of SCFAs and their importance for cardiovascular health. The data suggest that lifestyle modifications leading to augmented SCFA production could be a beneficial non-pharmacological preventive strategy for patients with hypertensive cardiovascular disease.</p>
</li>
</ul>
</li>
</ul>
</li>
<li>
<p><em>gut bacteria selectively promoted by dietary fibers alleviate type 2 diabetes</em><br />
liping zhao et al. 2018<br />
<a href="http://doi.org/10.1126/science.aao5774">doi.org/10.1126/science.aao5774</a></p>
<ul>
<li>The gut microbiota benefits humans via short-chain fatty acid (SCFA) production from carbohydrate fermentation, and deficiency in SCFA production is associated with type 2 diabetes mellitus (T2DM). We conducted a randomized clinical study of specifically designed isoenergetic diets, together with fecal shotgun metagenomics, to show that a select group of SCFA-producing strains was promoted by dietary fibers and that most other potential producers were either diminished or unchanged in patients with T2DM. When the fiber-promoted SCFA producers were present in greater diversity and abundance, participants had better improvement in hemoglobin A1c levels, partly via increased glucagon-like peptide-1 production. Promotion of these positive responders diminished producers of metabolically detrimental compounds such as indole and hydrogen sulfide. Targeted restoration of these SCFA producers may present a novel ecological approach for managing T2DM.</li>
</ul>
</li>
<li>
<p><em>the gut microbiota mediates the anti-seizure effects of the ketogenic diet</em><br />
christine a. olson et al. 2018<br />
<a href="http://doi.org/10.1016/j.cell.2018.04.027">doi.org/10.1016/j.cell.2018.04.027</a></p>
<ul>
<li>
<p>•Changes in the gut microbiota are required for the anti-seizure effects of the KD<br />
•Specific KD-associated bacteria mediate and confer the anti-seizure effects of the KD<br />
•KD microbiota regulate amino acid γ-glutamylation and hippocampal GABA/glutamate</p>
<p>The ketogenic diet (KD) is used to treat refractory epilepsy, but the mechanisms underlying its neuroprotective effects remain unclear. Here, we show that the gut microbiota is altered by the KD and required for protection against acute electrically induced seizures and spontaneous tonic-clonic seizures in two mouse models. Mice treated with antibiotics or reared germ free are resistant to KD-mediated seizure protection. Enrichment of, and gnotobiotic co-colonization with, KD-associated Akkermansia and Parabacteroides restores seizure protection. Moreover, transplantation of the KD gut microbiota and treatment with Akkermansia and Parabacteroides each confer seizure protection to mice fed a control diet. Alterations in colonic lumenal, serum, and hippocampal metabolomic profiles correlate with seizure protection, including reductions in systemic gamma-glutamylated amino acids and elevated hippocampal GABA/glutamate levels. Bacterial cross-feeding decreases gamma-glutamyltranspeptidase activity, and inhibiting gamma-glutamylation promotes seizure protection in vivo. Overall, this study reveals that the gut microbiota modulates host metabolism and seizure susceptibility in mice.</p>
</li>
</ul>
</li>
<li>
<p><em>lifelong choline supplementation ameliorates alzheimer’s disease pathology and associated cognitive deficits by attenuating microglia activation</em><br />
ramon velazquez et al. 2019<br />
[doi.org/10.1111/acel.13037](http://doi.org/10.1111/acel.13037</p>
<ul>
<li>
<p>The study focuses on female mice bred to develop AD-like symptoms. Given the higher prevalence of AD in human females, the study sought to establish the findings in female mice. Results showed that when these mice are given high choline in their diet throughout life, they exhibit improvements in spatial memory, compared with those receiving a normal choline regimen.</p>
<p>Notably, findings published in July 2019 from a group in China found benefits of lifelong choline supplementation in male mice with AD-like symptoms. &quot;Our results nicely replicate findings by this group in females,&quot; Velazquez says.</p>
<p>Intriguingly, the beneficial effects of lifelong choline supplementation reduce the activation of microglia. Microglia are specialized cells that rid the brain of deleterious debris. Although they naturally occur to keep the brain healthy, if they are overactivated, brain inflammation and neuronal death, common symptoms of AD, will occur.</p>
<p>The observed reductions in disease-associated microglia, which are present in various neurodegenerative diseases, offer exciting new avenues of research and suggest ways of treating a broad range of disorders, including traumatic brain injuries, multiple sclerosis and Parkinson's disease.</p>
<p>The findings appear in the current issue of the journal Aging Cell.</p>
<p>Supplementing the brain with additional choline</p>
<p>Choline acts to protect the brain from Alzheimer's disease in at least two ways, both of which are explored in the new study. First, choline blocks the production of amyloid-beta plaques. Amyloid-beta plaques are the hallmark pathology observed in Alzheimer's disease.</p>
<p>Secondly, choline supplementation reduces the activation of microglia. Over-activation of microglia causes brain inflammation and can eventually lead to neuronal death, thereby compromising cognitive function. Choline supplementation reduces the activation of microglia, offering further protection from the ravages of AD.</p>
<p>Mechanistically, the reductions in microglia activation are driven by alteration of two key receptors, the alpha7 nicotinic acetylcholine and Sigma-1 receptor. A new report this year found that choline can act as an agonist for Sigma-1 receptors. These results confirm that lifelong choline supplementation can alter the expression of the Sigma-1 receptor, which thereby attenuates microglia activation. (An agonist is a substance that activates a given receptor.)</p>
<p>The devastating decline</p>
<p>In the scientific community, it is well understood that Alzheimer's disease causes harm to the brain long before clinical symptoms are made evident. And once these symptoms are identified, it is too late -- the disease has become irreversible. In addition to causing disorientation and memory loss, the disease causes loss of motor control in those who are afflicted.</p>
<p>Approximately 6 million individuals are living with AD in the U.S. currently, and the disease is projected to afflict 14 million Americans in the next four decades. Economically, the costs associated with managing Alzheimer's are expected to exceed $20 trillion in the same time span.</p>
<p>To develop more effective treatments, we first need to understand the disease itself, which is one of the tallest orders facing modern medicine today.</p>
<p>Women are at a particular increased risk of developing Alzheimer's disease. This study shows that the simple addition of choline in the diet throughout life may reduce AD pathology in those most affected by the disease. Additionally, these results have implications for other neurodegenerative afflictions where activated microglia are rampant says Velazquez.</p>
<p>Guidelines for dietary choline</p>
<p>Prior research concerning Alzheimer's has indicated that there is no one factor at play. Rather, a multitude of factors that are believed to contribute to the development of the disease, including genetics, age and lifestyle. Additionally, studies suggest that diet can have a significant effect in increasing or lowering the risk of cognitive decline.</p>
<p>A recent report suggested that plant-based diets may be determinantal due to the lack of important nutrients, including choline. Another recent report found that the increase in cases of dementia in the United Kingdom may be associated with a lack of recommendations for choline in the diet throughout life. In fact, as of August 2019, AD and other forms of dementia are now the leading cause of death in England and Wales.</p>
<p>The current established adequate intake level of choline for adult women (&gt;19yrs of age) is 425mg/day, and 550mg/day for adult men. A converging line of evidence indicates that even the current recommended daily intake (RDI) may not be optimal for a proper aging process, especially in women. This is relevant, given the higher incidence of AD seen in women. This suggests that additional choline in diet may be beneficial in preventing neuropathological changes associated with the aging brain.</p>
<p>The tolerable upper limit (TUL) of choline unlikely to cause side effects for adult females and males (&gt;19yrs of age) is 3500mg/day, which is 8.24 times higher than the 425mg/day recommendation for females and 6.36 times higher than the 550mg/day recommendation for males. &quot;Our choline supplemented diet regimen was only 4.5 times the RDI, which is well below the TUL and makes this a safe strategy,&quot; Velazquez says.</p>
<p>Choline can be found in various foods. According to the United States Department of Agriculture (USDA), high levels of choline are found in chicken liver (3oz; 247mg), eggs (1 large egg with yolk;147mg), beef grass-fed steak (3oz; 55mg), wheat germ (1oz toast; 51mg), milk (8oz; 38mg), and Brussel sprouts (1/2 cup; 32mg). Additionally, vitamin supplements containing choline, for example choline bitartrate and choline chloride, are widely available at affordable costs. The vitamin supplements containing choline are particularly relevant for those who are on plant-based diets.</p>
<p>Effects of choline</p>
<p>All plant and animal cells require choline to maintain their structural integrity. It has long been recognized that choline is particularly important for brain function.</p>
<p>The human body uses choline to produce acetylcholine, a neurotransmitter responsible for functioning memory, muscle control and mood. Choline also is used to build cell membranes and plays a vital role in regulating gene expression. Additionally, a new report in Jan 2019 found that choline acts as an agonist for Sigma-1 receptors, which are implicated in AD pathogenesis.</p>
<p>In this study, researchers used a water maze to determine whether the mice with AD-like symptoms that received lifelong supplemental choline exhibited improvements in spatial memory. It was found that this was indeed the case, and subsequent examination of mouse tissue extracted from the hippocampus, a brain region known to play a central role in memory formation, confirmed changes in toxic amyloid-beta and reductions in microglia activation, which reduces brain inflammation.</p>
<p>Due to alterations of key microglia receptors induced by choline, the improvements in behavior may be attributed to reduced microglia activation. &quot;We found that lifelong choline supplementation altered the alpha7 nicotinic acetylcholine and Sigma-1 receptor, which may have resulted in the reduction of diseased associated activated microglia,&quot; Velazquez said. These receptors regulate CNS immune response and their dysregulation contributes to AD pathogenesis.</p>
<p>The study's significance establishes beneficial effects of nutrient supplementation in females throughout life. &quot;Our work nicely complements recent work showing benefits in male AD-mice on a lifelong choline supplementation regimen.&quot; &quot;No one has shown lifelong benefits of choline supplementation in female AD-mice.&quot; &quot;That's what is novel about our work.&quot;</p>
<p>Choline is an attractive candidate for prevention of AD as it is considered a very safe alternative, compared with many pharmaceuticals. &quot;At 4.5 times the RDI (recommended daily intake), we are well under the tolerable upper limit, making this a safe preventive therapeutic strategy.&quot;</p>
<p>Although the results improve the understanding of the disease, the authors suggest that clinical trials will be necessary to confirm whether choline can be used as a viable treatment in the future.</p>
</li>
<li>
<p><em>abstract</em> Currently, there are no effective therapies to ameliorate the pathological progression of Alzheimer's disease (AD). Evidence suggests that environmental factors may contribute to AD. Notably, dietary nutrients are suggested to play a key role in mediating mechanisms associated with brain function. Choline is a B‐like vitamin nutrient found in common foods that is important in various cell functions. It serves as a methyl donor and as a precursor for production of cell membranes. Choline is also the precursor for acetylcholine, a neurotransmitter which activates the alpha7 nicotinic acetylcholine receptor (α7nAchR), and also acts as an agonist for the Sigma‐1 R (σ1R). These receptors regulate CNS immune response, and their dysregulation contributes to AD pathogenesis. Here, we tested whether dietary choline supplementation throughout life reduces AD‐like pathology and rescues memory deficits in the APP/PS1 mouse model of AD. We exposed female APP/PS1 and NonTg mice to either a control choline (1.1 g/kg choline chloride) or a choline‐supplemented diet (5.0 g/kg choline chloride) from 2.5 to 10 months of age. Mice were tested in the Morris water maze to assess spatial memory followed by neuropathological evaluation. Lifelong choline supplementation significantly reduced amyloid‐β plaque load and improved spatial memory in APP/PS1 mice. Mechanistically, these changes were linked to a decrease of the amyloidogenic processing of APP, reductions in disease‐associated microglial activation, and a downregulation of the α7nAch and σ1 receptors. Our results demonstrate that lifelong choline supplementation produces profound benefits and suggest that simply modifying diet throughout life may reduce AD pathology.</p>
</li>
</ul>
</li>
</ul>
</li>
</ul>
<h2>06394</h2>
<ul>
<li>
<p>promoted by diversity</p>
<ul>
<li>
<p><em>staphylococcus aureus shifts toward commensalism in response to corynebacterium species</em><br />
matthew m. ramsey et al. 2016<br />
<a href="http://doi.org/10.3389/fmicb.2016.01230">doi.org/10.3389/fmicb.2016.01230</a></p>
<ul>
<li>Staphylococcus aureus–human interactions result in a continuum of outcomes from commensalism to pathogenesis. S. aureus is a clinically important pathogen that asymptomatically colonizes ~25% of humans as a member of the nostril and skin microbiota, where it resides with other bacteria including commensal Corynebacterium species. Commensal Corynebacterium spp. are also positively correlated with S. aureus in chronic polymicrobial diabetic foot infections, distinct from acute monomicrobial S. aureus infections. Recent work by our lab and others indicates that microbe–microbe interactions between S. aureus and human skin/nasal commensals, including Corynebacterium species, affect S. aureus behavior and fitness. Thus, we hypothesized that S. aureus interactions with Corynebacterium spp. diminish S. aureus virulence. We tested this by assaying for changes in S. aureus gene expression during in vitro mono- versus coculture with Corynebacterium striatum, a common skin and nasal commensal. We observed a broad shift in S. aureus gene transcription during in vitro growth with C. striatum, including increased transcription of genes known to exhibit increased expression during human nasal colonization and decreased transcription of virulence genes. S. aureus uses several regulatory pathways to transition between commensal and pathogenic states. One of these, the quorum signal accessory gene regulator (agr) system, was strongly inhibited in response to Corynebacterium spp. Phenotypically, S. aureus exposed to C. striatum exhibited increased adhesion to epithelial cells, reflecting a commensal state, and decreased hemolysin activity, reflecting an attenuation of virulence. Consistent with this, S. aureus displayed diminished fitness in experimental in vivo coinfection with C. striatum when compared to monoinfection. These data support a model in which S. aureus shifts from virulence toward a commensal state when exposed to commensal Corynebacterium species.</li>
</ul>
</li>
<li>
<p><em>harbouring public good mutants within a pathogen population can increase both fitness and virulence</em><br />
richard j lindsay et al 2016<br />
<a href="http://doi.org/10.7554/eLife.18678">doi.org/10.7554/eLife.18678</a></p>
<ul>
<li>“paradoxic” increase of virulence in diverse population versus “pure” population</li>
<li>Existing theory, empirical, clinical and field research all predict that reducing the virulence of individuals within a pathogen population will reduce the overall virulence, rendering disease less severe. Here, we show that this seemingly successful disease management strategy can fail with devastating consequences for infected hosts. We deploy cooperation theory and a novel synthetic system involving the rice blast fungus <em>Magnaporthe oryzae</em>. In vivo infections of rice demonstrate that <em>M. oryzae</em> virulence is enhanced, quite paradoxically, when a public good mutant is present in a population of high-virulence pathogens. We reason that during infection, the fungus engages in multiple cooperative acts to exploit host resources. We establish a multi-trait cooperation model which suggests that the observed failure of the virulence reduction strategy is caused by the interference between different social traits. Multi-trait cooperative interactions are widespread, so we caution against the indiscriminant application of anti-virulence therapy as a disease-management strategy.</li>
</ul>
</li>
<li>
<p><em>differential human gut microbiome assemblages during soil-transmitted helminth infections in indonesia and liberia</em><br />
bruce a. rosa et al. 2018<br />
<a href="http://doi.org/10.1186/s40168-018-0416-5">doi.org/10.1186/s40168-018-0416-5</a></p>
<ul>
<li><em>abstract</em> we identify specific members of the gut microbiome that discriminate between moderately/heavily STH-infected and non-infected states across very diverse geographical regions using two different statistical methods. We also identify microbiome-encoded biological functions associated with the STH infections, which are associated potentially with STH survival strategies, and changes in the host environment. These results provide a novel insight of the cross-kingdom interactions in the human gut ecosystem by unlocking the microbiome assemblages at taxonomic, genetic, and functional levels so that advances towards key mechanistic studies can be made.</li>
<li>Studying communities in Liberia and Indonesia, the researchers found that the gut microbiomes of people able to clear the infections without drugs were more alike and differed markedly from the microbiomes of those who could not clear the infections without treatment.<br />
“People who have sustained infections or who experience multiple infections have a different microbiome to start with compared with those who do not have as much trouble with infection,” said senior author Makedonka Mitreva, PhD, an associate professor of medicine. “Our work included samples from a placebo-controlled trial of drugs against these parasitic worms. It suggests that the microbiomes of people who retained infection are somehow compromised to begin with. Something about their microbiomes makes them more prone to getting infected and to maintaining a chronic infection.”<br />
According to the World Health Organization, about one quarter of the world’s population — over 1.5 billion people — is infected with parasitic worms called helminths. These worm infections are most common in tropical and subtropical areas with poor sanitation.<br />
The researchers identified 12 microbes associated with worm-infected individuals and one type of bacteria associated with uninfected individuals. In particular, gut bacteria associated with increased inflammation were linked to healthy uninfected individuals, perhaps because this type of inflammatory environment makes it harder for the worms to establish themselves in the gut.<br />
The researchers, including first author Bruce A. Rosa, PhD, a staff scientist, analyzed about 400 fecal samples from 250 people living in different villages in Liberia and Indonesia. Mitreva said one of the study’s strengths is that it established common gut microbiome patterns associated with high and low risk of worm infections in two distant geographic regions — West Africa and Southeast Asia, where such infections are prevalent.<br />
The infections spread easily when soil is contaminated with human feces that contain worm eggs. Children are particularly vulnerable because the infections can cause malnutrition, leading to impaired growth and development. Even when infections are treated with appropriate drugs, they often recur because the infective stages of the parasite are ever present in the soil. The main species that affect human health are large roundworm, whipworm and hookworm.<br />
“There are good drugs against most — but not all — of these infections. We are beginning to see more responses to treatment that are less than optimal, indicating the worms are starting to develop resistance to some of these treatments,” said Mitreva, also an assistant director of the university’s McDonnell Genome Institute. “The big problem is reinfection. Even if the therapy works and the infection is cleared, the exposure to contaminated soil is so pervasive that new infections are extremely common. Building on the information from this study, our ultimate long-term goal is to develop a way to alter the gut microbiome so that it protects people from reinfection.”<br />
Currently, antihelminthic drugs are used to deworm people who are infected, especially school-age children and women of reproductive age. Since the study found characteristics of the microbiome that are discriminative of infection, Mitreva said this information could be used to predict who is most likely to develop severe and chronic infections and direct more preventive efforts to those individuals.<br />
“Ideally, we would like to be in a position to suggest some local fermented foods that could alter the microbiome and result in a decreased rate of reinfection,” Mitreva said. “Rather than give more antihelminthic drugs, we want to help people fight off the infection by themselves. Right now, we are looking at the bacteria that we found were associated with protection against infection and studying their effects in mice with intestinal worm infections.”</li>
</ul>
</li>
<li>
<p><em>a commensal strain of staphylococcus epidermidis protects against skin neoplasia</em><br />
teruaki nakatsuji et al. 2018<br />
<a href="http://doi.org/10.1126/sciadv.aao4502">doi.org/10.1126/sciadv.aao4502</a></p>
<ul>
<li>We report the discovery that strains of <em>Staphylococcus epidermidis</em> produce 6-<em>N</em>-hydroxyaminopurine (6-HAP), a molecule that inhibits DNA polymerase activity. In culture, 6-HAP selectively inhibited proliferation of tumor lines but did not inhibit primary keratinocytes. Resistance to 6-HAP was associated with the expression of mitochondrial amidoxime reducing components, enzymes that were not observed in cells sensitive to this compound. Intravenous injection of 6-HAP in mice suppressed the growth of B16F10 melanoma without evidence of systemic toxicity. Colonization of mice with an <em>S. epidermidis</em> strain producing 6-HAP reduced the incidence of ultraviolet-induced tumors compared to mice colonized by a control strain that did not produce 6-HAP. <em>S. epidermidis</em> strains producing 6-HAP were found in the metagenome from multiple healthy human subjects, suggesting that the microbiome of some individuals may confer protection against skin cancer. These findings show a new role for skin commensal bacteria in host defense.</li>
</ul>
</li>
<li>
<p><em>human age and skin physiology shape diversity and abundance of archaea on skin</em><br />
christine moissl-eichinger et al. 2017<br />
<a href="http://doi.org/10.1038/s41598-017-04197-4">doi.org/10.1038/s41598-017-04197-4</a></p>
<ul>
<li>The human skin microbiome acts as an important barrier protecting our body from pathogens and other environmental influences. Recent investigations have provided evidence that Archaea are a constant but highly variable component of the human skin microbiome, yet factors that determine their abundance changes are unknown. Here, we tested the hypothesis that the abundance of archaea on human skin is influenced by human age and skin physiology by quantitative PCR of 51 different skin samples taken from human subjects of various age. Our results reveal that archaea are more abundant in human subjects either older than 60 years or younger than 12 years as compared to middle-aged human subjects. These results, together with results obtained from spectroscopy analysis, allowed us gain first insights into a potential link of lower sebum levels and lipid content and thus reduced skin moisture with an increase in archaeal signatures. Amplicon sequencing of selected samples revealed the prevalence of specific eury- and mainly thaumarchaeal taxa, represented by a core archaeome of the human skin.</li>
</ul>
</li>
<li>
<p><em>gut microbiota dependent anti-tumor immunity restricts melanoma growth in rnf5−/− mice</em><br />
yan li et al. 2019<br />
<a href="http://doi.org/10.1038/s41467-019-09525-y">doi.org/10.1038/s41467-019-09525-y</a></p>
<ul>
<li>
<p>11 bacterial strains that activated the immune system and slowed the growth of melanoma in mice. The study also points to the role of unfolded protein response (UPR), a cellular signaling pathway that maintains protein health (homeostasis). Reduced UPR was seen in melanoma patients who are responsive to immune checkpoint therapy, revealing potential markers for patient stratification. The study was published in Nature Communications.</p>
<p>&quot;Immunotherapies have extended the lives of many cancer patients. However, the incredible effects we are seeing today are only the tip of the iceberg. By studying mechanisms of treatment response versus resistance, we can eventually expand the number of people who benefit from immunotherapy,&quot; says Thomas Gajewski, M.D., Ph.D., the AbbVie Foundation Professor of Cancer Immunotherapy at the University of Chicago Medicine. &quot;This study provides an important step toward this goal. The investigators have pinpointed the UPR as an important link between the gut microbiota and anti-tumor immunity. Given previous work indicating a causal role for the host microbiota in the efficacy of checkpoint blockade immunotherapy, this additional mechanistic insight should help select patients who will respond to treatment and also help to guide new therapeutic development.&quot;</p>
<p>Although immune checkpoint therapies have significantly improved patient survival rates, metastatic melanoma remains the deadliest form of skin cancer, according to the American Cancer Society. Even when used as part of combination therapy, immune checkpoint inhibitors only benefit about half of patients, and these responses may involve autoimmune-related side effects, limited durability (the length of time a patient responds to treatment) and, at times, resistance to therapy. Accumulating evidence supports the role of the gut microbiome in effective immune therapy: Antibiotics and select probiotics reduce treatment efficacy, while certain bacterial strains enhance efficacy. This study sheds new light on these observations.</p>
<p>&quot;Our study establishes a formal link between the microbiome and anti-tumor immunity and points to the role of the UPR in this process, answering a long-sought question for the field,&quot; says Ze'ev Ronai, Ph.D., senior author of the study and a professor at Sanford Burnham Prebys' NCI-designated Cancer Center. &quot;These results also identify a collection of bacterial strains that could turn on anti-tumor immunity and biomarkers that could be used to stratify people with melanoma for treatment with select checkpoint inhibitors.&quot;</p>
<p>&quot;Boring&quot; mice yield exciting results</p>
<p>Ronai has dedicated much of his lab's efforts to understanding how cancer responds to stress and becomes treatment resistant. As part of this work, he and his team are studying a genetic mouse model that lacks the gene for RING finger protein 5 (RNF5), a ubiquitin ligase that helps remove inappropriately folded or damaged proteins. While these molecular traits are critical for the current study, the mice don't show any outward signs of disease.</p>
<p>&quot;We call them the 'boring mice' because they don't have a notable phenotype,&quot; says Ronai.</p>
<p>However, the RNF5-lacking mice were able to inhibit the growth of melanoma tumors, provided they had an intact immune system and gut microbiome. Treating these mice with a cocktail of antibiotics or housing the mice with their regular (wildtype) littermates abolished the anti-tumor immunity phenotype and consequently, tumor rejection -- indicating the important role of the gut microbiome in anti-tumor immunity. Mapping the immune components engaged in the process revealed several immune system components, including Toll-like receptors and select dendritic cells, within the gut intestinal environment. Reduced UPR was commonly identified in immune and intestinal epithelial cells and was sufficient for immune cell activation. Reduced UPR signaling was also associated with the altered gut microbiomes seen in the mice.</p>
<p>Advanced bioinformatics techniques allowed the scientists to identify 11 bacterial strains that were enriched in the guts of the RNF5-lacking mice. Transferring these 11 bacterial strains to regular mice that lack intestinal bacteria (germ-free) induced anti-tumor immune response and slowed tumor growth.</p>
<p>To confirm that the results were relevant in human disease, the scientists obtained tissue samples from three cohorts of people with metastatic melanoma who subsequently received checkpoint inhibitor treatment. Indeed, reduced expression of UPR components (sXBP1, ATF4 and BiP) correlated with responsiveness to treatment, suggesting that there are potentially predictive biomarkers for the selection of patients who should receive immune checkpoint therapy.</p>
<p>Next, the scientists plan to determine what the bacteria are producing that slows tumor growth. These products, called metabolites, could then be tested to determine their ability to enhance anti-tumor immunity but also to define possible prebiotics that may be used to enrich their presence in the gut of melanoma patients.</p>
<p>&quot;We believe this research applies to another fundamental question pertaining to the balance between anti-tumor immunity and autoimmunity,&quot; says Ronai. &quot;Because mice that lack RNF5 are also prone to developing gut inflammation -- a side effect seen for certain immune checkpoint therapies -- we can exploit this powerful model to study how we may tilt the balance between autoimmunity and anti-tumor immunity, which could help more people benefit from these remarkable therapies.&quot;</p>
</li>
<li>
<p><em>abstract</em> Accumulating evidence points to an important role for the gut microbiome in anti-tumor immunity. Here, we show that altered intestinal microbiota contributes to anti-tumor immunity, limiting tumor expansion. Mice lacking the ubiquitin ligase RNF5 exhibit attenuated activation of the unfolded protein response (UPR) components, which coincides with increased expression of inflammasome components, recruitment and activation of dendritic cells and reduced expression of antimicrobial peptides in intestinal epithelial cells. Reduced UPR expression is also seen in murine and human melanoma tumor specimens that responded to immune checkpoint therapy. Co-housing of Rnf5−/− and WT mice abolishes the anti-tumor immunity and tumor inhibition phenotype, whereas transfer of 11 bacterial strains, including B. rodentium, enriched in Rnf5−/− mice, establishes anti-tumor immunity and restricts melanoma growth in germ-free WT mice. Altered UPR signaling, exemplified in Rnf5−/− mice, coincides with altered gut microbiota composition and anti-tumor immunity to control melanoma growth.</p>
</li>
</ul>
</li>
<li>
<p><em>mucispirillum schaedleri antagonizes salmonella virulence to protect mice against colitis</em><br />
simone herp et al. 2019<br />
<a href="http://doi.org/10.1016/j.chom.2019.03.004">doi.org/10.1016/j.chom.2019.03.004</a></p>
<ul>
<li>
<p>In healthy humans and mice, the gastrointestinal tract is colonized by multiple species of bacteria and other microorganisms. This 'microbiome' can protect its hosts from infection with species of Salmonella, because the endogenous members produce inhibitory substances, occupy all the ecological niches available, and consume most of the relevant nutrients, such as sugars or proteins, but also oxygen. In an effort to determine the composition of a healthy gut microbiome which confers optimal protection against bacterial infections, co-authors Sandrine Brugiroux and Debora Garzetti compared the microbiota of several groups of mice. One group had been shown to be resistant to Salmonella infection, while the others were susceptible to the pathogen.</p>
<p>The researchers discovered that the microbiomes of the protected mice included bacteria belonging to the species Mucispirillum schaedleri, which were absent from the other groups. M. schaedleri belongs to a large group of bacterial species whose representatives primarily inhabit environments rich in mud or sediments. Of these, only Mucispirillum spp. occurs in the gastrointestinal tracts of warm-blooded animals like mice and humans. &quot;It had been thought to be relatively rare in human microbiomes, as it is usually not found in stool samples,&quot; says Stecher. &quot;However, this is because it preferentially colonizes mucous layer of the gut. In studies in which the mucosal biopsies were examined, M. schaedleri was discovered in 50% of subjects.&quot;</p>
<p>To test directly whether the bacterium in fact is causally linked to protection against Salmonella, Simone Herp, who has just completed her doctoral thesis at the Max von Pettenkofer-Institute, took advantage of a gnotobiotic (gnotos, greek: &quot;known&quot;) mouse model, which microbiota can be manipulated. Selected bacterial species can therefore be introduced into these mice, such that the composition of their microbiota is defined and known. &quot;We generated two groups of mice, one of which contains Mucispirillum schaedleri, while the other specifically lacks it. We experimentally infected both groups with Salmonella, and were able to confirm that M. schaedleri is causally associated with protection against Salmonella infections,&quot; says Stecher.</p>
<p>Further investigations revealed that the protective effect of M. schaedleri likely depends on its ability to successfully compete with Salmonella for certain essential nutrients, such as nitrate. This competition does not necessarily mean that the growth rate of latter is reduced. However, without adequate amounts of nitrate, Salmonella enterica serovar Typhimurium is unable to express its most important virulence factor, a Type III secretion system. As a result, their ability to induce pathogenic changes in the lining of the gut is significantly reduced. The primary virulence factor involved is essentially a molecular machine that acts as syringe, and allows the bacteria to inject toxic proteins into the cells of the gut epithelium. This system enables the bacteria to invade these cells, which in turn leads to inflammation and gastroenteritis.</p>
<p>The new results could, in the long term, lead to the development of new strategies for the prevention of bacterial infections of the gastrointestinal tract. &quot;But that will require a great deal of further work,&quot; as Stecher points out. &quot;For example, we still do not know whether or not M. schaedleri has other -- and possibly deleterious -- effects on the gut and human health.&quot;</p>
</li>
<li>
<p><em>abstract</em> •Mucispirillum schaedleri confers protection against Salmonella colitis in mice<br />
•Salmonella and M. schaedleri compete for anaerobic respiration substrates in the gut<br />
•M. schaedleri restricts Salmonella infection and inhibits virulence factor expression<br />
•Mucispirillum spp. are enriched in human gut mucosal samples</p>
<p>The microbiota and the gastrointestinal mucus layer play a pivotal role in protection against non-typhoidal Salmonella enterica serovar Typhimurium (S. Tm) colitis. Here, we analyzed the course of Salmonella colitis in mice lacking a functional mucus layer in the gut. Unexpectedly, in contrast to mucus-proficient littermates, genetically deficient mice were protected against Salmonella-induced gut inflammation in the streptomycin colitis model. This correlated with microbiota alterations and enrichment of the bacterial phylum Deferribacteres. Using gnotobiotic mice associated with defined bacterial consortia, we causally linked Mucispirillum schaedleri, currently the sole known representative of Deferribacteres present in the mammalian microbiota, to host protection against S. Tm colitis. Inhibition by M. schaedleri involves interference with S. Tm invasion gene expression, partly by competing for anaerobic electron acceptors. In conclusion, this study establishes M. schaedleri, a core member of the murine gut microbiota, as a key antagonist of S. Tm virulence in the gut.</p>
</li>
</ul>
</li>
</ul>
</li>
<li>
<p>promoted by exercise</p>
<ul>
<li>
<p><em>exercise alters gut microbiota composition and function in lean and obese humans</em><br />
jacob m. allen et al. 2017<br />
<a href="http://doi.org/10.1249/mss.0000000000001495">doi.org/10.1249/mss.0000000000001495</a></p>
<ul>
<li>Exercise is associated with altered gut microbial composition, but studies have not investigated whether the gut microbiota and associated metabolites are modulated by exercise training in humans. We explored the impact of six weeks of endurance exercise on the composition, functional capacity, and metabolic output of the gut microbiota in lean and obese adults with multiple-day dietary controls prior to outcome variable collection.<br />
METHODS<br />
Thirty-two lean (n=18 [9 female]) and obese (n=14 [11 female]), previously sedentary subjects participated in six weeks of supervised, endurance-based exercise training (3 days/wk) that progressed from 30 to 60 minutes/day and from moderate (60% of heart rate reserve [HRR]) to vigorous intensity (75% HRR). Subsequently, participants subsequently returned to a sedentary lifestyle activity for a six week washout period. Fecal samples were collected before and after six weeks of exercise, as well as after the sedentary washout period, with 3-day dietary controls in place prior to each collection.<br />
RESULTS<br />
β-diversity analysis revealed that exercise-induced alterations of the gut microbiota were dependent on obesity status. Exercise increased fecal concentrations of short chain fatty acids (SCFAs) in lean, but not obese, participants. Exercise-induced shifts in metabolic output of the microbiota paralleled changes in bacterial genes and taxa capable of SCFA production. Lastly, exercise-induced changes in the microbiota were largely reversed once exercise training ceased.<br />
CONCLUSION<br />
These findings suggest that exercise training induces compositional and functional changes in the human gut microbiota that are dependent on obesity status, independent of diet and contingent on the sustainment of exercise.</li>
</ul>
</li>
</ul>
</li>
<li>
<p>ecological restoration</p>
<ul>
<li>
<p><em>gut microbiota and human health: insights from ecological restoration</em><br />
matthew r. orr et al. 2018<br />
<a href="http://doi.org/10.1086/698021">doi.org/10.1086/698021</a></p>
<ul>
<li>Dysbiosis of the gut facilitates chronic diseases such as cancer, obesity, and autoimmune disorders. Successful treatment may require restoration of functions performed by symbiotic microbes. The field of ecological restoration has restored species diversity and ecosystem function of natural habitats since the latter part of the 20th century, but its possible applications to medicine are unexplored. Here we show that approaches developed to restore natural ecosystems may aid in the treatment of chronic disease by helping to characterize target conditions for healthy microbe communities, by identifying the importance of donor diversity in fecal microbiota transplants, and by elevating the importance of diet, patient involvement, and personalized medicine in gut health. Dialogue between the fields of restoration ecology and medicine may help to guide future directions in gut research and clinical care.</li>
</ul>
</li>
</ul>
</li>
</ul>
<h2>06494</h2>
<ul>
<li>
<p>mental resilience</p>
<ul>
<li><em>anxiety in your head could come from your gut</em><br />
susan donaldson James 2013<br />
<a href="http://abcnews.go.com/Health/anxiety-head-gut/story?id=20229136">abcnews.go.com/Health/anxiety-head-gut/story</a></li>
</ul>
</li>
<li>
<p><em>gut microbiota and glucometabolic alterations in response to recurrent partial sleep deprivation in normal-weight young individuals</em><br />
christian benedict, heike vogel, wenke jonas, anni woting, michael blaut, annette schürmann, jonathan cedernaes 2016<br />
<a href="http://doi.org/10.1016/j.molmet.2016.10.003">doi.org/10.1016/j.molmet.2016.10.003</a></p>
<ul>
<li>very basic first study to reveal changes in gut microbiota due to changes in sleep regime
<ul>
<li>Objective: Changes to the microbial community in the human gut have been proposed to promote metabolic disturbances that also occur after short periods of sleep loss (including insulin resistance). However, whether sleep loss affects the gut microbiota remains unknown.<br />
Methods: In a randomized within-subject crossover study utilizing a standardized in-lab protocol (with fixed meal times and exercise schedules), we studied nine normal-weight men at two occasions: after two nights of partial sleep deprivation (PSD; sleep opportunity 02:45-07:00h), and after two nights of normal sleep (NS; sleep opportunity 22:30-07:00h). Fecal samples were collected within 24 hours before, and after two in-lab nights, of either NS or PSD. In addition, participants underwent an oral glucose tolerance test following each sleep intervention.<br />
Results: Microbiota composition analysis (V4 16S rRNA gene sequencing) revealed that after two days of PSD vs. after two days of NS, individuals exhibited an increased Firmicutes:Bacteroidetes ratio and of the families Coriobacteriaceae and Erysipelotrichaceae, with decreases in Tenericutes (all P&lt;0.05) – previously all associated with metabolic perturbations in animal or human models. However, no PSD vs. NS effect on beta diversity or on fecal short-chain fatty acid concentrations was found. Fasting and postprandial insulin sensitivity decreased after PSD (all P&lt;0.05).<br />
Discussion: Our findings demonstrate that short-term sleep loss induces subtle effects on human microbiota. To what extent changes to the microbial community contribute to metabolic consequences of sleep loss warrants further investigations in larger and more prolonged sleep studies, to also assess how sleep loss impacts the microbiota in individuals who already are metabolically compromised.</li>
</ul>
</li>
</ul>
</li>
<li>
<p><em>microbial regulation of microrna expression in the amygdala and prefrontal cortex</em><br />
alan e. hoban et al. 2017<br />
<a href="http://doi.org/10.1186/s40168-017-0321-3">doi.org/10.1186/s40168-017-0321-3</a></p>
<ul>
<li>One of the most exciting findings in the past decade in relation to psychiatric disorders has been the discovery that the gut microbiota may act as a key regulator of the brain and behaviour [1, 2, 3]. Proof-of-principal experiments using microbiota-deficient or germ-free (GF) rodents have been extremely useful in investigating the impact of the absence of the gut microbiota on brain development and behaviour <a href="http://doi.org/10.1371/journal.pbio.2000633">4</a>, especially regarding the ability of the gut microbiota to influence normal anxiety and fear behaviours [5, 6, 7]. GF animals not only display changes in basal anxiety but also have deficits in sociability, cognition and increased depressive-like behaviours [8, 9, 10]. Some studies that use antibiotic exposure in order to deplete the gut microbiota have also shown similarly altered behavioural phenotypes [11, 12, 13]. Preclinical evidence also highlights that certain probiotics and prebiotics have anxiolytic-like activity [14, 15] which highlights the potential of microbiota-mediated therapeutic approaches for treating anxiety-related disorders.<br />
The most reproducible finding with regard to behavioural effects in rodent models of microbial disturbances is a marked influence on anxiety-related behaviours <a href="http://doi.org/10.1371/journal.pbio.2000633">4</a>. The neural systems that govern such behaviours are complex, with an array of brain regions that interconnect in order to process emotional stimuli and allow for appropriate responses <a href="http://www.nature.com/articles/ncomms13329#ref26">16</a>. Specifically, the amygdala and PFC are key loci that control anxiety and the response to fearful stimuli <a href="http://www.nature.com/articles/ncomms13329#ref27">17</a>. Both regions display abnormalities in GF animals including hypermyelination in the PFC <a href="http://doi.org/10.1016/j.cell.2017.05.036">18</a>, altered morphology of the amygdala <a href="http://doi.org/10.7554/eLife.18678">19</a> and impaired amygdala-dependent fear memory recall <a href="http://doi.org/10.1016/j.cell.2016.11.003">7</a>. A large body of evidence implicates dysfunction of the neural circuits connecting these two brain regions in the pathophysiology of fear- and anxiety-like disorders [20, 21].<br />
Over the past few years, attempts to understand the mechanisms underlying psychiatric disorders have focused attention on the role of miRNAs <a href="http://doi.org/10.1016/j.molmet.2016.10.003">22</a>. miRNAs are an expanding class of endogenous small non-coding RNAs, which function as post-transcriptional regulators of gene expression, primarily through translational repression. It has been demonstrated that brain-specific miRNA candidates within the amygdala and PFC regulate anxiety- and fear-related behaviours in mice [23, 24, 25, 26].<br />
We have previously shown that the absence of microbial exposure throughout life results in altered transcriptional regulation in both the amygdala and PFC [18, 27]. We therefore hypothesized that alterations in miRNA expression may be associated with some of the noted molecular changes in these animals. To this end, we used a number of complementary approaches. Firstly, we conducted unbiased Illumina® NGS for miRNAs in the amygdala and PFC of conventionally raised (CON), GF and colonized GF mice (exGF) to investigate whether miRNA expression can be influenced by the gut microbiota. To delineate specific miRNAs that may be influenced by the gut microbiota during adulthood following normal assembly patterns in early life, we then examined miRNA expression in the central nervous system (CNS) of adult rodents following sustained microbiota knockdown with antibiotics. We hypothesized that there would be extensive reorganization of miRNA expression in these two key brain regions (amygdala and PFC) of microbiota-deficient rodents.</li>
</ul>
</li>
<li>
<p><em>microglial control of astrocytes in response to microbial metabolites</em><br />
veit rothhammer et al. 2018<br />
<a href="http://doi.org/10.1038/s41586-018-0119-x">doi.org/10.1038/s41586-018-0119-x</a></p>
<ul>
<li>Microglia and astrocytes modulate inflammation and neurodegeneration in the central nervous system (CNS). Microglia modulate pro-inflammatory and neurotoxic activities in astrocytes, but the mechanisms involved are not completely understood. Here we report that TGFα and VEGF-B produced by microglia regulate the pathogenic activities of astrocytes in the experimental autoimmune encephalomyelitis (EAE) mouse model of multiple sclerosis. Microglia-derived TGFα acts via the ErbB1 receptor in astrocytes to limit their pathogenic activities and EAE development. Conversely, microglial VEGF-B triggers FLT-1 signalling in astrocytes and worsens EAE. VEGF-B and TGFα also participate in the microglial control of human astrocytes. Furthermore, expression of TGFα and VEGF-B in CD14+ cells correlates with the multiple sclerosis lesion stage. Finally, metabolites of dietary tryptophan produced by the commensal flora control microglial activation and TGFα and VEGF-B production, modulating the transcriptional program of astrocytes and CNS inflammation through a mechanism mediated by the aryl hydrocarbon receptor. In summary, we identified positive and negative regulators that mediate the microglial control of astrocytes. Moreover, these findings define a pathway through which microbial metabolites limit pathogenic activities of microglia and astrocytes, and suppress CNS inflammation. This pathway may guide new therapies for multiple sclerosis and other neurological disorders.</li>
</ul>
</li>
<li>
<p><em>self-maintaining gut macrophages are essential for intestinal homeostasis</em><br />
sebastiaan de schepper et al. 2018<br />
<a href="http://doi.org/10.1016/j.cell.2018.07.048">doi.org/10.1016/j.cell.2018.07.048</a></p>
<ul>
<li>
<p>Self-maintaining macrophages (gMacs) colonize distinct niches of the gut<br />
gMacs have a unique transcriptional profile depending on the niche in which they reside<br />
gMacs support enteric neurons and submucosal vasculature<br />
Loss of gMacs results in vascular leakage, reduced intestinal secretion, and transit</p>
<p>Macrophages are highly heterogeneous tissue-resident immune cells that perform a variety of tissue-supportive functions. The current paradigm dictates that intestinal macrophages are continuously replaced by incoming monocytes that acquire a pro-inflammatory or tissue-protective signature. Here, we identify a self-maintaining population of macrophages that arise from both embryonic precursors and adult bone marrow-derived monocytes and persists throughout adulthood. Gene expression and imaging studies of self-maintaining macrophages revealed distinct transcriptional profiles that reflect their unique localization (i.e., closely positioned to blood vessels, submucosal and myenteric plexus, Paneth cells, and Peyer’s patches). Depletion of self-maintaining macrophages resulted in morphological abnormalities in the submucosal vasculature and loss of enteric neurons, leading to vascular leakage, impaired secretion, and reduced intestinal motility. These results provide critical insights in intestinal macrophage heterogeneity and demonstrate the strategic role of self-maintaining macrophages in gut homeostasis and intestinal physiology.</p>
</li>
</ul>
</li>
</ul>
<h2>06594</h2>
<ul>
<li>
<p><em>the reproductive microbiome: an emerging driver of sexual selection, sexual conflict, mating systems, and reproductive isolation</em><br />
melissah rowe et al. 2020<br />
<a href="http://https://doi.org/10.1016/j.tree.2019.11.004">doi.org/10.1016/j.tree.2019.11.004</a></p>
<ul>
<li>
<p>“Microbes appear to influence fertility, reproduction, and the evolution of animal species in so many ways,” says Rowe. Sperm quality, mate choice, sexual health, success at producing offspring, the balance between female and male mating interests, general health, and even the origin of new species. “And yet, almost nobody is studying this, especially in non-human animals.”<br />
The ecology and evolution of reproductive biology and behaviour in animals, especially birds, is Rowe’s field of research. Recently, she has started as a scientist in the Department of Animal Ecology with the Netherlands Institute of Ecology. She is interested in the impact of the reproductive microbiome. Together with colleagues from the University of Oslo (Norway), Oxford and Exeter (United Kingdom), Rowe composed an overview of all available scientific data.<br />
<strong>Bacteria as jury</strong><br />
A couple of examples. Men with large amounts of certain bacteria in their semen are more likely to be infertile. Female bedbugs ramp up their immunological defences ahead of mating, as males will pierce their abdomen with their genitalia during mating. The resulting infections from bacteria transmitted via the genitalia can be fatal. In mallard ducks, males with a more colourful bill produce semen that is better able to kill bacteria, thereby possibly influencing the females’ partner choice for a ‘safe’ male.<br />
Rowe: “I think that reproductive microbiomes may be an important, yet relatively overlooked, evolutionary force.” Natural selection, but with bacteria and other microbes as the ‘jury’ and sometimes the ‘executioner’.<br />
As this research field is new and unexplored, many questions are still waiting to be answered. For example, are the positive or negative effects of the microbiome caused by specific species or is it the composition of the whole microbial community that is important? Do these reproductive microbiomes evolve differently in females compared to males? And can knowledge of these microbial communities improve our success at breeding endangered species, and reintroducing them back into the wild? “Let’s find out.”</p>
</li>
<li>
<p><em>abstract</em> Although it is clear that the microbiome can have major influences on host biology, relatively little is known about the microbial communities in the reproductive systems of males and females.<br />
The reproductive microbiome can have significant effects on the reproductive function and fitness of males and females, although these effects are rarely considered from an ecological or evolutionary perspective.<br />
Recent research indicates that the reproductive microbiome can influence the outcomes of pre- and postcopulatory sexual selection, as well as generate conflict between the sexes.<br />
Thus, complex coevolutionary dynamics can emerge between the host mating system and the reproductive microbiome, to shape patterns of reproductive skew in the host, microbial diversity across individuals and species, and the ecology and evolution of reproductive microbes. Consequently, reproductive microbiomes also have strong potential to influence reproductive barriers between host populations.</p>
</li>
</ul>
</li>
</ul>
<h2>development</h2>
<h2>06694</h2>
<ul>
<li>
<p><em>the maternal microbiome modulates fetal neurodevelopment in mice</em><br />
h.e. vuong et al. 2020<br />
<a href="http://doi.org/10.1038/s41586-020-2745-3">doi.org/10.1038/s41586-020-2745-3</a></p>
<ul>
<li>
<p>New findings in mice suggest yet another role for gut microbes, even before birth.<br />
The microbes residing in a female mouse’s gut <a href="https://www.nature.com/articles/s41586-020-2745-3">help shape the wiring of her offspring’s brain</a>, researchers report September 23 in <em>Nature.</em> While mouse and human development are worlds apart, the study hints at how a mother’s microbiome may have long-term consequences for her offspring.<br />
Scientists have previously found links between a mouse mother’s microbiome and her young’s brain and behavior, but many of those studies worked with <a href="https://www.sciencenews.org/article/vaginal-microbes-mice-transfer-stress-their-pups">animals that were stressed</a> (<em>SN: 7/9/18</em>) or sick. Instead, Helen Vuong, a neurobiologist at UCLA, and her colleagues looked at what a mother’s microbial mix normally does for her pups’ brains.<br />
The new results point to the influence of specific microbes and the small molecules they produce, called metabolites. “Metabolites from the microbiome of the mother can influence the developing brain of the fetus,” says Cathryn Nagler, an immunologist at the University of Chicago who was not involved with the study. The metabolites do this by reaching a developing pup’s brain where they affect the growth of axons, she says. Axons are the threadlike signal-transmitters of nerve cells.</p>
<p>Vuong and her team looked at the brains of fetuses from pregnant mice — some with their usual gut bugs, some raised without microbes and others ridded of their gut bacteria with antibiotics. When a mother’s microbes were missing, fetuses had shorter and fewer axons extending from the brain’s “relay station” to the cortex, Vuong says. These connections are important for processing sensory information.<br />
Those brain differences appear to have consequences for mice later in life. As adults, mice born to microbe-deficient mothers were less sensitive to touch than mice from mothers with a typical microbiome. For instance, in one of several sensory tests, mice from microbe-deficient mothers took longer to notice a small piece of tape stuck to one of their paws. But when microbe-lacking females were given <em>Clostridia</em> bacteria, their offspring’s brain and behavior developed normally. <em>Clostridia</em> are common gut microbes in humans and in mice, Nagler says, and their absence has been linked to some noncommunicable conditions, <a href="https://www.sciencenews.org/article/gut-bacteria-may-prevent-food-allergies">such as food allergies</a> (<em>SN: 8/26/14</em>).<br />
Small molecules made by the gut bugs may account for this effect. The researchers found that levels of several metabolites in mom’s blood were linked to levels in the fetal blood and brain. “It’s kind of cool that it’s crossing different sites from the mom all the way to the fetus,” Vuong says. That suggests mom shares her gut metabolites with her young.<br />
When pregnant mice with altered microbiomes received supplements of some of those metabolites, their pups’ behavior developed normally. It’s not clear yet how gut microbes and metabolites could be involved in human brain development. Still, this “points now to a way that one might think about intervening,” if pregnant women have deficient microbiomes, says Nagler, who is president of ClostraBio, a company that’s exploring metabolite treatments for diseases related to the immune system. Instead of trying to alter those microbiomes, which can be difficult, pregnant women could receive the needed metabolites directly.<br />
“It will be really important to understand whether these negative effects also happen in humans and whether they lead to long-term medical concerns,” says Carolina Tropini, a microbiologist and biomedical engineer at the University of British Columbia in Vancouver who was not part of the work. Researchers will need to study how the short-term benefits of antibiotics stack up against potential risks, she says, but such research may also lead to therapies for pregnant women who need antibiotics.</p>
</li>
<li>
<p><em>abstract</em> ‘Dysbiosis’ of the maternal gut microbiome, in response to challenges such as infection<a href="https://www.nature.com/articles/s41586-020-2745-3#ref-CR1">27</a>, altered diet<a href="https://www.nature.com/articles/s41586-020-2745-3#ref-CR2">28</a> and stress<a href="https://www.nature.com/articles/s41586-020-2745-3#ref-CR3">29</a> during pregnancy, has been increasingly associated with abnormalities in brain function and behaviour of the offspring<a href="https://www.nature.com/articles/s41586-020-2745-3#ref-CR4">30</a>. However, it is unclear whether the maternal gut microbiome influences neurodevelopment during critical prenatal periods and in the absence of environmental challenges. Here we investigate how depletion and selective reconstitution of the maternal gut microbiome influences fetal neurodevelopment in mice. Embryos from antibiotic-treated and germ-free dams exhibited reduced brain expression of genes related to axonogenesis, deficient thalamocortical axons and impaired outgrowth of thalamic axons in response to cell-extrinsic factors. Gnotobiotic colonization of microbiome-depleted dams with a limited consortium of bacteria prevented abnormalities in fetal brain gene expression and thalamocortical axonogenesis. Metabolomic profiling revealed that the maternal microbiome regulates numerous small molecules in the maternal serum and the brains of fetal offspring. Select microbiota-dependent metabolites promoted axon outgrowth from fetal thalamic explants. Moreover, maternal supplementation with these metabolites abrogated deficiencies in fetal thalamocortical axons. Manipulation of the maternal microbiome and microbial metabolites during pregnancy yielded adult offspring with altered tactile sensitivity in two aversive somatosensory behavioural tasks, but no overt differences in many other sensorimotor behaviours. Together, our findings show that the maternal gut microbiome promotes fetal thalamocortical axonogenesis, probably through signalling by microbially modulated metabolites to neurons in the developing brain.</p>
</li>
</ul>
</li>
<li>
<p><em>modeling the temporal dynamics of the gut microbial community in adults and infants</em><br />
liat shenhav et al. 2019<br />
<a href="http://doi.org/10.1371/journal.pcbi.1006960">doi.org/10.1371/journal.pcbi.1006960</a></p>
<ul>
<li>
<p>developed Microbial community Temporal Variability Linear Mixed Model (MTV-LMM), a new method for modeling temporal changes in the microbial composition of the gut. When tested against real-world data, the new tool makes more accurate predictions than do other models previously developed for the same purpose.</p>
<p>The researchers then used MTV-LMM to surface new insights into microbiome dynamics. For instance, they demonstrated that, in both infants and adults, gut microbiome community composition can indeed be accurately predicted based on earlier observations of the community. They also applied the model to data from 39 infants and revealed a key shift around the age of 9 months in how the gut microbiome changes over time.</p>
<p>Looking forward, MTV-LMM could be applied to explore temporal dynamics of the gut microbiome in the context of disease, which could lead to improved diagnosis and treatment. It could also be useful for understanding other types of temporal microbiome processes, such as those occurring during digestion.</p>
</li>
<li>
<p><em>abstract</em> Given the highly dynamic and complex nature of the human gut microbial community, the ability to identify and predict time-dependent compositional patterns of microbes is crucial to our understanding of the structure and functions of this ecosystem. One factor that could affect such time-dependent patterns is microbial interactions, wherein community composition at a given time point affects the microbial composition at a later time point. However, the field has not yet settled on the degree of this effect. Specifically, it has been recently suggested that only a minority of taxa depend on the microbial composition in earlier times. To address the issue of identifying and predicting temporal microbial patterns we developed a new model, MTV-LMM (Microbial Temporal Variability Linear Mixed Model), a linear mixed model for the prediction of microbial community temporal dynamics. MTV-LMM can identify time-dependent microbes (i.e., microbes whose abundance can be predicted based on the previous microbial composition) in longitudinal studies, which can then be used to analyze the trajectory of the microbiome over time. We evaluated the performance of MTV-LMM on real and synthetic time series datasets, and found that MTV-LMM outperforms commonly used methods for microbiome time series modeling. Particularly, we demonstrate that the effect of the microbial composition in previous time points on the abundance of taxa at later time points is underestimated by a factor of at least 10 when applying previous approaches. Using MTV-LMM, we demonstrate that a considerable portion of the human gut microbiome, both in infants and adults, has a significant time-dependent component that can be predicted based on microbiome composition in earlier time points. This suggests that microbiome composition at a given time point is a major factor in defining future microbiome composition and that this phenomenon is considerably more common than previously reported for the human gut microbiome.</p>
</li>
</ul>
</li>
<li>
<p><em>long-term benefit of microbiota transfer therapy on autism symptoms and gut microbiota</em><br />
dae-wook kang et al. 2019<br />
<a href="http://doi.org/10.1038/s41598-019-42183-0">doi.org/10.1038/s41598-019-42183-0</a></p>
<ul>
<li>
<p>The apparent rise in autism spectrum disorder (ASD) and its stubborn resistance to treatment has spurred a legion of researchers to enter the field and explore the disability in innovative ways.</p>
<p>Currently, effective treatments for ASD include behavioral therapy, speech and social therapy, psychiatric medications, and dietary and nutritional approaches. However, no medical treatments have been approved to treat core symptoms of ASD such as social communication difficulties and repetitive behaviors.</p>
<p>One promising avenue of autism research involves the gut microbiome, which is the collection of microbes that lives in our intestines and helps us in many ways including digestion of our food, training our immune system and preventing overgrowth of harmful bacteria. Recent research suggests our gut microbiomes also affect brain communication and neurological health. Worldwide, interest is growing in the idea that changes in normal gut microbiota may be responsible for triggering a vast range of diseases.</p>
<p>In a new study, &quot;Long-term benefit of Microbiota Transfer Therapy in Autism Symptoms and Gut Microbiota,&quot; published in Scientific Reports, Arizona State University researchers Rosa Krajmalnik-Brown, Ph.D., James Adams, Ph.D, and lead author Dae-Wook Kang, Ph.D, demonstrate long-term beneficial effects for children diagnosed with ASD through a revolutionary technique known as Microbiota Transfer Therapy (MTT), a special type of fecal transplant originally pioneered by Dr. Thomas Borody, an Australian gastroenterologist. Remarkably, improvements in gut health and autism symptoms appear to persist long after treatment.</p>
<p>At two years post-treatment, most of the initial improvements in gut symptoms remained. In addition, parents reported a slow steady reduction of ASD symptoms during treatment and over the next two years. A professional evaluator found a 45% reduction in core ASD symptoms (language, social interaction and behavior) at two years post-treatment compared to before treatment began.</p>
<p>&quot;We are finding a very strong connection between the microbes that live in our intestines and signals that travel to the brain,&quot; said Krajmalnik-Brown, a professor at the Biodesign Swette Center for Environmental Biotechnology at the Biodesign Institute and ASU's School for Sustainable Engineering and the Built Environment. &quot;Two years later, the children are doing even better, which is amazing.&quot;</p>
<p>&quot;Many kids with autism have gastrointestinal problems, and some studies, including ours, have found that those children also have worse autism-related symptoms,&quot; said Krajmalnik-Brown. &quot;In many cases, when you are able to treat those gastrointestinal problems, their behavior improves.&quot;</p>
<p>Roughly 30-50% of all people with autism have chronic gastrointestinal (GI) problems, primarily constipation and/or diarrhea that can last for many years. That chronic discomfort and pain can cause irritability, decreased attention and learning, and negatively impact behavior.</p>
<p>An earlier study with only vancomycin (an antibiotic) had found major temporary improvements in GI and autism symptoms, but the benefits were lost a few weeks after treatment stopped despite use of over-the-counter probiotics.</p>
<p>So, the question at hand was what's going on in the gut, and how does it affect both physical and behavioral symptoms of autism, and how can we develop a long-lasting treatment?</p>
<p>Krajmalnik-Brown, Kang and Adams have shown that by transferring healthy microbiota to individuals lacking certain gut bacteria, it is possible to &quot;donate&quot; a more diverse set of bacteria into the patient and improve gut health.</p>
<p>In Australia, Fecal Microbiota Transplantation (FMT) was initially developed by Borody. At his Centre for Digestive Diseases in Sydney, Borody has overseen more than 18,000 FMTs for various disorders since 1987. He pioneered in Australia the use of FMT for colitis and Clostridium difficile infection, and was the first to use oral FMT to treat children with ASD. Only one dose of FMT is usually enough to cure C. Difficile infections, but his patients with autism were far harder to treat. He discovered that three months of daily FMT was required to treat his autism patients, but eventually resulted in significant improvements in both GI and autism symptoms.</p>
<p>Based on his experience with his patients, Borody led the design of the clinical treatment used at ASU for this study. The MTT approach involves 10 weeks of treatment, including pre-treatment with vancomycin, a bowel cleanse, a stomach acid suppressant, and fecal microbiota transfer daily for seven to eight weeks.</p>
<p>The initial open-label study, led by Krajmalnik-Brown and Adams, and published in the journal Microbiome in 2017, concluded that &quot;this exploratory, extended-duration treatment protocol thus appears to be a promising approach to alter the gut microbiome and improve GI and behavioral symptoms of ASD. Improvements in GI symptoms, ASD symptoms, and the microbiome all persisted for at least eight weeks after treatment ended, suggesting a long-term impact.&quot; The present study now shows the benefits are extended beyond eight weeks to at least two years post-treatment.</p>
<p>The ASU team compared differences in the microbiome of children with autism compared to typically developing children. At the start of the study, children with autism were found to have lower diversity in their respective gut microbes and were depleted of certain strains of helpful bacteria, such as Bifidobacteria and Prevotella. &quot;Kids with autism are lacking important beneficial bacteria, and have fewer options in the bacterial menu of important functions that bacteria provide to the gut than typically developing kids,&quot; Krajmalnik-Brown said.</p>
<p>FMT treatment substantially increased microbial diversity and the presence of helpful bacteria in the gut, such as Bifidobacteria and Prevotella. After two years, diversity was even higher and the presence of beneficial microbes remained.</p>
<p>&quot;We originally hypothesized that our therapy would be efficient to transform the dysbiotic gut microbiome toward a healthy one. In our original paper in 2017, we reported an increase in gut diversity together with beneficial bacteria after MTT, and after two years, we observed diversity was even higher and the presence of beneficial microbes remained,&quot; Kang said. He added that this may be one of the reasons for success in improving the gut health, but further mechanistic studies are warranted to define specific roles of gut microbes in the context of autism.</p>
<p>The work done at ASU is not only about treating patients but also about learning from the treatment in order to develop better formulations and optimize dosing.</p>
<p>&quot;Understanding which microbes and chemicals produced by the microbes are driving these behavioral changes is at the heart of our work,&quot; Krajmalnik-Brown said. The team's new publication reports that the study demonstrated that two years after treatment stopped the participants still had an average of a 58% reduction in GI symptoms compared to baseline. In addition, the parents of most participants reported &quot;a slow but steady improvement in core ASD symptoms.&quot;</p>
<p>&quot;Every family completed the study, and every family returned two years later for a follow-up evaluation,&quot; said Adams, citing the families' dedication to the research. &quot;The treatment was generally well-tolerated with minimal adverse effects.&quot;</p>
<p>&quot;This is a world-first discovery that when we treated the gut bacteria in these children during our clinical trial two years ago to reset their microbiome with FMT, positive results are still continuing to be improving two years from the original treatments. I would call it the highest improvement in a cohort that anyone has achieved for autism symptoms,&quot; said Borody.</p>
<p>Professional evaluation revealed a 45% decrease in ASD symptoms compared to baseline. Researchers note that although there may be some placebo effect, much of that effect appears to be real. At the start of the study, 83% of participants were rated as &quot;severe&quot; autism. At the end of the study, only 17% were &quot;severe,&quot; 39% were &quot;mild/moderate,&quot; and 44% were below the cut-off for mild ASD.</p>
<p>Greg Caporaso, at Northern Arizona University, a leading expert in microbiome data science and a co-author on these studies, helped to analyze the microbiome data to better understand bacterial changes as a result of MTT.</p>
<p>&quot;Drs. Krajmalnik-Brown, Kang and I are excited about the results, but we want to caution the public that we need larger clinical trials for this to become an FDA-approved treatment,&quot; said Adams. Professional expertise is required for safe and effective treatment.</p>
<p>MTT improves GI distress by introducing key strains of beneficial bacteria and helping to raise levels of biodiversity within the gut, boosting health overall.</p>
<p>Adams has both professional and personal reasons for doggedly pursuing ways to help children with autism because he knows the situation first-hand. His daughter was diagnosed with autism just before her third birthday. Adams, a President's Professor at ASU's School for Engineering of Matter, Transport and Energy, and the chair of Materials Sciences, is also president of the Autism Society of Greater Phoenix, the largest parent support group in Arizona.</p>
<p>&quot;Dr. James Adams is the reason why I started working on autism,&quot; Krajmalnik-Brown said. &quot;I had the methods to do all of the measurements and assessments in the microbiome part of the work, and he had the autism knowledge.&quot;</p>
<p>Adams recruited patients, supervised clinical work and ASD assessments, and guided the patients through the trials, and Krajmalnik-Brown led the microbiome evaluations and helped plan the study.</p>
<p>All of the participants in the study exhibited chronic GI symptoms from infancy, including chronic constipation and/or chronic diarrhea. The treatment benefits extended beyond their physical symptoms, even causing some parents to note how much their children's behavior had improved over time.</p>
<p>&quot;It is very unusual to see steady gradual improvement after the conclusion of any treatment,&quot; said Adams. &quot;We only conducted the long-term follow-up study after several families told us that their child was continuing to improve significantly.&quot; Krajmalnik-Brown stated that the data suggests that the MTT intervention transformed the gut environment into a healthier status, leading to long-term benefit on both GI and ASD symptoms.</p>
<p>Adams said many of the participants in the trial shared common traits, including birth by C-section, reduced breastfeeding, increased antibiotics, and low fiber intake by the mother and child, all of which lead to limited biodiversity in their gut bacteria. Due to the open label nature of the study and the small sample size used, more research is needed in order to verify the usefulness of MTT as a therapeutic.</p>
<p>The initial study involved a &quot;first-generation&quot; estimate as to optimal dose and duration of treatment, and it was enough for 90% of the children to have substantial benefit. The team is now working on optimizing the dosing and duration to try to improve benefits even more, and to determine if booster doses may be needed in some cases.</p>
</li>
<li>
<p><em>abstract</em> Many studies have reported abnormal gut microbiota in individuals with Autism Spectrum Disorders (ASD), suggesting a link between gut microbiome and autism-like behaviors. Modifying the gut microbiome is a potential route to improve gastrointestinal (GI) and behavioral symptoms in children with ASD, and fecal microbiota transplant could transform the dysbiotic gut microbiome toward a healthy one by delivering a large number of commensal microbes from a healthy donor. We previously performed an open-label trial of Microbiota Transfer Therapy (MTT) that combined antibiotics, a bowel cleanse, a stomach-acid suppressant, and fecal microbiota transplant, and observed significant improvements in GI symptoms, autism-related symptoms, and gut microbiota. Here, we report on a follow-up with the same 18 participants two years after treatment was completed. Notably, most improvements in GI symptoms were maintained, and autism-related symptoms improved even more after the end of treatment. Important changes in gut microbiota at the end of treatment remained at follow-up, including significant increases in bacterial diversity and relative abundances of Bifidobacteria and Prevotella. Our observations demonstrate the long-term safety and efficacy of MTT as a potential therapy to treat children with ASD who have GI problems, and warrant a double-blind, placebo-controlled trial in the future.</p>
</li>
</ul>
</li>
<li>
<p><em>human gut microbiota from autism spectrum disorder promote behavioral symptoms in mice</em><br />
gil sharon et al. 2019<br />
<a href="http://doi.org/10.1016/j.cell.2019.05.004">doi.org/10.1016/j.cell.2019.05.004</a></p>
<ul>
<li>
<p>•Mice harboring human ASD, but not TD, microbiomes exhibit ASD-like behaviors<br />
•ASD and TD microbiota produce differential metabolome profiles in mice<br />
•Extensive alternative splicing of risk genes in brains of mice with ASD microbiota<br />
•BTBR mice treated with 5AV or taurine improved repetitive and social behaviors</p>
<p>Autism spectrum disorder (ASD) manifests as alterations in complex human behaviors including social communication and stereotypies. In addition to genetic risks, the gut microbiome differs between typically developing (TD) and ASD individuals, though it remains unclear whether the microbiome contributes to symptoms. We transplanted gut microbiota from human donors with ASD or TD controls into germ-free mice and reveal that colonization with ASD microbiota is sufficient to induce hallmark autistic behaviors. The brains of mice colonized with ASD microbiota display alternative splicing of ASD-relevant genes. Microbiome and metabolome profiles of mice harboring human microbiota predict that specific bacterial taxa and their metabolites modulate ASD behaviors. Indeed, treatment of an ASD mouse model with candidate microbial metabolites improves behavioral abnormalities and modulates neuronal excitability in the brain. We propose that the gut microbiota regulates behaviors in mice via production of neuroactive metabolites, suggesting that gut-brain connections contribute to the pathophysiology of ASD.</p>
</li>
</ul>
</li>
<li>
<p><em>cutting edge: critical roles for microbiota-mediated regulation of the immune system in a prenatal immune activation model of autism</em><br />
catherine r. lammert et al. 2018<br />
<a href="http://doi.org/10.4049/jimmunol.1701755">doi.org/10.4049/jimmunol.1701755</a></p>
<ul>
<li>Recent studies suggest that autism is often associated with dysregulated immune responses and altered microbiota composition. This has led to growing speculation about potential roles for hyperactive immune responses and the microbiome in autism. Yet how microbiome–immune cross-talk contributes to neurodevelopmental disorders currently remains poorly understood. In this study, we report critical roles for prenatal microbiota composition in the development of behavioral abnormalities in a murine maternal immune activation (MIA) model of autism that is driven by the viral mimetic polyinosinic-polycytidylic acid. We show that preconception microbiota transplantation can transfer susceptibility to MIA-associated neurodevelopmental disease and that this is associated with modulation of the maternal immune response. Furthermore, we find that ablation of IL-17a signaling provides protection against the development of neurodevelopmental abnormalities in MIA offspring. Our findings suggest that microbiota landscape can influence MIA-induced neurodevelopmental disease pathogenesis and that this occurs as a result of microflora-associated calibration of gestational IL-17a responses.</li>
</ul>
</li>
<li>
<p><em>the gut microbiota and autism spectrum disorders</em><br />
qinrui li et al. 2017<br />
<a href="http://doi.org/10.3389/fncel.2017.00120">doi.org/10.3389/fncel.2017.00120</a></p>
<ul>
<li>Gastrointestinal (GI) symptoms are a common comorbidity in patients with autism spectrum disorder (ASD), but the underlying mechanisms are unknown. Many studies have shown alterations in the composition of the fecal flora and metabolic products of the gut microbiome in patients with ASD. The gut microbiota influences brain development and behaviors through the neuroendocrine, neuroimmune and autonomic nervous systems. In addition, an abnormal gut microbiota is associated with several diseases, such as inflammatory bowel disease (IBD), ASD and mood disorders. Here, we review the bidirectional interactions between the central nervous system and the gastrointestinal tract (brain-gut axis) and the role of the gut microbiota in the central nervous system (CNS) and ASD. Microbiome-mediated therapies might be a safe and effective treatment for ASD.</li>
</ul>
</li>
<li>
<p><em>environment dominates over host genetics in shaping human gut microbiota</em><br />
daphna rothschild et al. 2018<br />
<a href="http://doi.org/10.1038/nature25973">doi.org/10.1038/nature25973</a></p>
<ul>
<li>Human gut microbiome composition is shaped by multiple factors but the relative contribution of host genetics remains elusive. Here we examine genotype and microbiome data from 1,046 healthy individuals with several distinct ancestral origins who share a relatively common environment, and demonstrate that the gut microbiome is not significantly associated with genetic ancestry, and that host genetics have a minor role in determining microbiome composition. We show that, by contrast, there are significant similarities in the compositions of the microbiomes of genetically unrelated individuals who share a household, and that over 20% of the inter-person microbiome variability is associated with factors related to diet, drugs and anthropometric measurements. We further demonstrate that microbiome data significantly improve the prediction accuracy for many human traits, such as glucose and obesity measures, compared to models that use only host genetic and environmental data. These results suggest that microbiome alterations aimed at improving clinical outcomes may be carried out across diverse genetic backgrounds.</li>
</ul>
</li>
<li>
<p><em>the vaginal microbiome and preterm birth</em><br />
jennifer m. fettweis et al. 2019<br />
<a href="http://doi.org/10.1038/s41591-019-0450-2">doi.org/10.1038/s41591-019-0450-2</a></p>
<ul>
<li>
<p>vaginal microbiomes of 45 women who gave birth prematurely and 90 whose pregnancies extended through term. About 78 percent of the women in each group were African American.</p>
<p>&quot;Given where VCU is -- in the Richmond area -- you have a predominantly African American population, and so that's who shows up at the VCU clinics,&quot; Huzurbazar said. &quot;That's a really useful thing because a lot of other studies are mostly done with large groups of people of European descent.&quot;</p>
<p>Another useful thing was the timespan the data represented. Because the data spanned all trimesters of pregnancy -- and even extended into the postpartum period -- the researchers could track how the bacterial communities changed over time.</p>
<p>They found that by the second trimester, women who would eventually give birth prematurely harbored much larger populations of certain vaginal bacteria than their full-term counterparts did. In addition, they had smaller populations of Lactobacillus crispatus. That's noteworthy because a Lactobacillus-dominated microbiome is a hallmark of a healthy reproductive tract. These differences only grew more pronounced as the pregnancies progressed.</p>
<p>The researchers also discovered that the microbial profile associated with preterm birth correlated to a greater presence of proinflammatory proteins -- called cytokines -- in vaginal fluid.</p>
<p>&quot;Microbiome data is relatively new -- especially the vaginal microbiome -- so a lot of it is looking at the data and figuring out what you see in terms of correlation,&quot; Huzurbazar said. &quot;The next steps are doing the basic science to find out, for example, how the cytokines potentially affect preterm birth.&quot;</p>
<p>Their findings may influence how pregnant women's risk of preterm birth is assessed. For example, perhaps their vaginal microbiomes could be screened for the bacteria at issue.</p>
<p>&quot;We don't know why the microbiome looks different. A large part of the microbiome -- whether it's gut or vaginal or whatever -- might have to do with access to healthcare, socioeconomic factors and so on,&quot; Huzurbazar said. &quot;The bigger goal is to try to figure out early on which women are more likely to have preterm births. Then you can do something about it because you have some time for intervention.&quot;</p>
</li>
<li>
<p><em>abstract</em> The incidence of preterm birth exceeds 10% worldwide. There are significant disparities in the frequency of preterm birth among populations within countries, and women of African ancestry disproportionately bear the burden of risk in the United States. In the present study, we report a community resource that includes ‘omics’ data from approximately 12,000 samples as part of the integrative Human Microbiome Project. Longitudinal analyses of 16S ribosomal RNA, metagenomic, metatranscriptomic and cytokine profiles from 45 preterm and 90 term birth controls identified harbingers of preterm birth in this cohort of women predominantly of African ancestry. Women who delivered preterm exhibited significantly lower vaginal levels of Lactobacillus crispatus and higher levels of BVAB1, Sneathia amnii, TM7-H1, a group of Prevotella species and nine additional taxa. The first representative genomes of BVAB1 and TM7-H1 are described. Preterm-birth-associated taxa were correlated with proinflammatory cytokines in vaginal fluid. These findings highlight new opportunities for assessment of the risk of preterm birth.</p>
</li>
</ul>
</li>
</ul>
<h2>06794</h2>
<ul>
<li>
<p><em>temporal development of the gut microbiome in early childhood from the teddy study</em><br />
christopher j. stewart et al. 2018<br />
<a href="http://doi.org/10.1038/s41586-018-0617-x">doi.org/10.1038/s41586-018-0617-x</a></p>
<ul>
<li>The development of the microbiome from infancy to childhood is dependent on a range of factors, with microbial–immune crosstalk during this time thought to be involved in the pathobiology of later life diseases1,2,3,4,5,6,7,8,9 such as persistent islet autoimmunity and type 1 diabetes10,11,12. However, to our knowledge, no studies have performed extensive characterization of the microbiome in early life in a large, multi-centre population. Here we analyse longitudinal stool samples from 903 children between 3 and 46 months of age by 16S rRNA gene sequencing (n = 12,005) and metagenomic sequencing (n = 10,867), as part of the The Environmental Determinants of Diabetes in the Young (TEDDY) study. We show that the developing gut microbiome undergoes three distinct phases of microbiome progression: a developmental phase (months 3–14), a transitional phase (months 15–30), and a stable phase (months 31–46). Receipt of breast milk, either exclusive or partial, was the most significant factor associated with the microbiome structure. Breastfeeding was associated with higher levels of Bifidobacterium species (B. breve and B. bifidum), and the cessation of breast milk resulted in faster maturation of the gut microbiome, as marked by the phylum Firmicutes. Birth mode was also significantly associated with the microbiome during the developmental phase, driven by higher levels of Bacteroides species (particularly B. fragilis) in infants delivered vaginally. Bacteroides was also associated with increased gut diversity and faster maturation, regardless of the birth mode. Environmental factors including geographical location and household exposures (such as siblings and furry pets) also represented important covariates. A nested case–control analysis revealed subtle associations between microbial taxonomy and the development of islet autoimmunity or type 1 diabetes. These data determine the structural and functional assembly of the microbiome in early life and provide a foundation for targeted mechanistic investigation into the consequences of microbial–immune crosstalk for long-term health.</li>
</ul>
</li>
<li>
<p><em>the human gut microbiome in early-onset type 1 diabetes from the teddy study</em><br />
tommi vatanen et al. 2018<br />
<a href="http://doi.org/10.1038/s41586-018-0620-2">doi.org/10.1038/s41586-018-0620-2</a></p>
<ul>
<li>Type 1 diabetes (T1D) is an autoimmune disease that targets pancreatic islet beta cells and incorporates genetic and environmental factors<a href="https://www.nature.com/articles/s41586-018-0620-2#ref-CR1">32</a>, including complex genetic elements<a href="https://www.nature.com/articles/s41586-018-0620-2#ref-CR2">33</a>, patient exposures<a href="https://www.nature.com/articles/s41586-018-0620-2#ref-CR3">34</a> and the gut microbiome<a href="https://www.nature.com/articles/s41586-018-0620-2#ref-CR4">35</a>. Viral infections<a href="https://www.nature.com/articles/s41586-018-0620-2#ref-CR5">36</a> and broader gut dysbioses<a href="https://www.nature.com/articles/s41586-018-0620-2#ref-CR6">37</a> have been identified as potential causes or contributing factors; however, human studies have not yet identified microbial compositional or functional triggers that are predictive of islet autoimmunity or T1D. Here we analyse 10,913 metagenomes in stool samples from 783 mostly white, non-Hispanic children. The samples were collected monthly from three months of age until the clinical end point (islet autoimmunity or T1D) in the The Environmental Determinants of Diabetes in the Young (TEDDY) study, to characterize the natural history of the early gut microbiome in connection to islet autoimmunity, T1D diagnosis, and other common early life events such as antibiotic treatments and probiotics. The microbiomes of control children contained more genes that were related to fermentation and the biosynthesis of short-chain fatty acids, but these were not consistently associated with particular taxa across geographically diverse clinical centres, suggesting that microbial factors associated with T1D are taxonomically diffuse but functionally more coherent. When we investigated the broader establishment and development of the infant microbiome, both taxonomic and functional profiles were dynamic and highly individualized, and dominated in the first year of life by one of three largely exclusive Bifidobacterium species (B. bifidum, B. breve or B. longum) or by the phylum Proteobacteria. In particular, the strain-specific carriage of genes for the utilization of human milk oligosaccharide within a subset of B. longum was present specifically in breast-fed infants. These analyses of TEDDY gut metagenomes provide, to our knowledge, the largest and most detailed longitudinal functional profile of the developing gut microbiome in relation to islet autoimmunity, T1D and other early childhood events. Together with existing evidence from human cohorts<a href="https://www.nature.com/articles/s41586-018-0620-2#ref-CR7">38</a>,<a href="https://www.nature.com/articles/s41586-018-0620-2#ref-CR8">39</a> and a T1D mouse model<a href="https://www.nature.com/articles/s41586-018-0620-2#ref-CR9">40</a>, these data support the protective effects of short-chain fatty acids in early-onset human T1D.</li>
</ul>
</li>
<li>
<p><em>gut microbiota in the first 2 years of life and the association with body mass index at age 12 in a norwegian birth cohort</em><br />
maggie a. stanislawski et al. 2018<br />
<a href="http://doi.org/10.1128/mbio.01751-18">doi.org/10.1128/mbio.01751-18</a></p>
<ul>
<li>Childhood obesity is a growing problem worldwide. Recent research suggests that the gut microbiota may play an important and potentially causal role in the development of obesity and may be one mechanism that explains the transgenerational transmission of obesity risk. Here we examine the early-life gut microbiota at days 4, 10, 30, 120, 365, and 730 and the association with body mass index (BMI) z-scores at age 12 in a Norwegian prospective cohort (<em>n</em> = 165), and evaluate how these BMI-associated taxa relate to maternal overweight/obesity (Ow/Ob) and excessive gestational weight gain (GWG). We performed 16S rRNA gene sequencing on the gut microbiota samples. Taxonomic phylogeny at days 10 and 730 was significantly associated with childhood BMI, and the gut microbiota taxa at two years of age explained over 50% of the variation in childhood BMI in this cohort. The subset of the early-life taxa within the gut microbiota that best predicted later childhood BMI showed substantial overlap with the maternal taxa most strongly associated with maternal Ow/Ob and excessive GWG. Our results show an association between the infant gut microbiota and later BMI, and they offer preliminary evidence that the infant gut microbiota, particularly at 2 years of age, may have potential to help identify children at risk for obesity.<br />
<strong>IMPORTANCE</strong> Understanding the role of the early-life gut microbiota in obesity is important because there may be opportunities for preventive strategies. We examined the relationships between infant gut microbiota at six times during the first two years of life and BMI at age 12 in a birth cohort of 165 children and their mothers. We found that the gut microbiota from early life to two years shows an increasingly strong association with childhood BMI. This study provides preliminary evidence that the gut microbiome at 2 years of age may offer useful information to help to identify youth who are at risk for obesity, which could facilitate more-targeted early prevention efforts.</li>
</ul>
</li>
<li>
<p><em>composition and variation of the human milk microbiota are influenced by maternal and early-life factors</em><br />
shirin moossavi et al. 2019<br />
<a href="http://doi.org/10.1016/j.chom.2019.01.011">doi.org/10.1016/j.chom.2019.01.011</a></p>
<ul>
<li>
<p>Although previously considered sterile, breastmilk is now known to contain a low abundance of bacteria. While the complexities of how maternal microbiota influence the infant microbiota are still unknown, this complex community of bacteria in breastmilk may help to establish the infant gut microbiota. Disruptions in this process could alter the infant microbiota, causing predisposition to chronic diseases such as allergies, asthma, and obesity. Although recent studies on human milk microbiota suggest that it might be affected by various factors, these findings have not been reproduced in large-scale studies, and the determinants of milk microbiota are still mostly unknown.</p>
<p>To address this gap in knowledge, Azad and her collaborators carried out bacterial gene sequencing on milk samples from 393 healthy mothers three to four months after giving birth. They used this information to examine how the milk microbiota composition is affected by maternal factors, early life events, breastfeeding practices, and other milk components. The researchers found a high degree of variability in the milk microbiota across mothers. Among the many factors analyzed, the mode of breastfeeding -- whether mothers provided milk with or without a pump -- was the only consistent factor directly associated with the milk microbiota composition.</p>
<p>Specifically, indirect breastfeeding (defined as at least one serving of pumped milk in the preceding two weeks) was associated with a higher abundance of potential opportunistic pathogens, such as Stenotrophomonas and Pseudomonadaceae.&quot;Increased exposure to potential pathogens in breastmilk could pose a risk of respiratory infection in the infant, potentially explaining why infants fed pumped milk are at increased risk for pediatric asthma compared to those fed exclusively at the breast,&quot; says first author Shirin Moossavi of the University of Manitoba. To determine if this is the case, there will need to be additional research into how changes in the milk microbiota affect colonization of the infant gut microbiome, which influences health.</p>
<p>By contrast, direct breastfeeding without a pump was associated with microbes typically found in the mouth, as well as higher overall bacterial richness and diversity. Taken together, the findings suggest that direct breastfeeding facilitates the acquisition of oral microbiota from infants, whereas indirect breastfeeding leads to enrichment with environmental (pump-associated) bacteria.</p>
<p>&quot;Our study contributes new evidence to the ongoing debate regarding the origins of milk microbiota,&quot; Azad says. &quot;Contrary or in addition to the hypothesis that milk bacteria come from the mother's gut, our results suggest that the infant's oral bacteria are important in shaping the milk microbiota. Mechanistic studies are needed to confirm this, but if true, it could provide exciting new opportunities for understanding and modifying the milk microbiota.&quot;</p>
<p>Intriguingly, the researchers also observed differences in the milk microbiota depending on the infant sex. These findings further support the idea that the milk microbiota is partially derived from the infant oral cavity, which may differ in male versus female infants.</p>
<p>According to the authors, this topic is ripe for further investigation. Collectively, the many factors they evaluated explained less than a third of the total variation observed in milk microbiota composition, indicating that other unmeasured factors contribute to the large inter-individual variation in milk microbiota profiles.</p>
<p>In future studies, the researchers will further explore the composition and function of the milk microbiota. In addition to bacteria, they will profile fungi in the milk samples. They also plan to investigate how the milk microbiota influences both the gut microbiota of infants and infant development and health. Specifically, their projects will examine the association of milk microbiota with infant growth, asthma, and allergies. &quot;This work could have important implications for microbiota-based strategies for early-life prevention of chronic conditions,&quot; Azad says.</p>
</li>
<li>
<p><em>abstract</em> •Milk microbiota variability is affected by maternal factors and other milk components<br />
•Some factors have phylum-specific effects<br />
•Some variations in milk microbiota are sex-specific<br />
•Feeding method (at the breast versus pumped) was strongly associated with milk microbiota</p>
<p>Breastmilk contains a complex community of bacteria that may help seed the infant gut microbiota. The composition and determinants of milk microbiota are poorly understood. Among 393 mother-infant dyads from the CHILD cohort, we found that milk microbiota at 3–4 months postpartum was dominated by inversely correlated Proteobacteria and Firmicutes, and exhibited discrete compositional patterns. Milk microbiota composition and diversity were associated with maternal factors (BMI, parity, and mode of delivery), breastfeeding practices, and other milk components in a sex-specific manner. Causal modeling identified mode of breastfeeding as a key determinant of milk microbiota composition. Specifically, providing pumped breastmilk was consistently associated with multiple microbiota parameters including enrichment of potential pathogens and depletion of bifidobacteria. Further, these data support the retrograde inoculation hypothesis, whereby the infant oral cavity impacts the milk microbiota. Collectively, these results identify features and determinants of human milk microbiota composition, with potential implications for infant health and development.</p>
</li>
</ul>
</li>
<li>
<p><em>visualization of microbes by 16s in situ hybridization in term and preterm placentae without intraamniotic infection</em><br />
maxim d. seferovic et al. 2019<br />
<a href="http://doi.org/10.1016/j.ajog.2019.04.036">doi.org/10.1016/j.ajog.2019.04.036</a></p>
<ul>
<li>
<p>&quot;There has been some debate about our and others' findings in the placenta. Because it is a sparse, or low biomass, community, it is a fair question to ask how much of what we identify as the microbiome is actually bacteria and how much is potentially environmental contamination, or maternal blood in the placenta,&quot; said senior author Dr. Kjersti Aagaard, professor and the Henry and Emma Meyer Chair of obstetrics and gynecology at Baylor.</p>
<p>Visual confirmation</p>
<p>&quot;Previously, bacteria were found using metagenomics or microbiome sequencing, and now we have confirmed that signal based on our ability to label the bacterial RNA with a florescent 'tag' and actually see them,&quot; said Dr. Maxim Seferovic, instructor in obstetrics and gynecology at Baylor and lead author in the study. &quot;We leveraged a powerful new imaging technology to add greater specificity in the signal of bacterial RNA, which helped us to see bacteria within the microarchitecture of the placental tissue.&quot;</p>
<p>Researchers examined microbes in term and preterm gestations using a signal amplified 16S universal in situ hybridization probe designed for bacterial rRNA, along with several other histologic methods. Seferovic said the study was carefully designed to control for contamination as best as possible, so that these sparse bacteria could be accurately attributed to their location in the placenta.</p>
<p>&quot;We did not see quantitative or numerical differences between preterm or full-term births, nor did we see them localizing to different substrata. But we do see differences in what genera of bacteria are there in preterm or full term, and this supported our and other's past findings as well,&quot; said Aagaard.</p>
<p>A sparse community</p>
<p>Seferovic said the study was designed to determine if past studies were in fact accurate and truly did look at a low biomass community of microbes that could be reliably distinguished from environmental contamination. This work, when combined with that of several other labs, should give researchers confidence that not only can they sequence these microbes but also that they can see the bacteria in very predictable locations in different placentas.</p>
<p>Seferovic and Aagaard suggest that this boosts their team's and others' confidence that they can begin to look more toward the role of microbes in the intrauterine environment in shaping the developing immune system in the fetus, and what role things like the mom's diet or preterm birth may play in that development.</p>
<p>&quot;At some point we all acquire trillions of bacteria in our bodies that we do not reject with an immune inflammatory response. We are speculating that these low biomass communities may play a key role in shaping the developing fetal immune system to help educate it on which microbes may be beneficial and which might not,&quot; Aagaard said.</p>
<p>Both Aagaard and Seferovic agree that there is still a lot of work ahead to be done in this exciting area of the developing microbiome and microbiome science. It is their hope that the techniques and tools developed for this study will lend a hand to other researchers similarly working in challenging low biomass communities.</p>
</li>
</ul>
</li>
</ul>
<h2>06894</h2>
<ul>
<li>airway
<ul>
<li>
<p><em>the airway microbiome at birth</em><br />
charitharth vivek lal et al 2016<br />
<a href="http://doi.org/10.1038/srep31023">doi.org/10.1038/srep31023</a></p>
<ul>
<li>Alterations of pulmonary microbiome have been recognized in multiple respiratory disorders. It is critically important to ascertain if an airway microbiome exists at birth and if so, whether it is associated with subsequent lung disease. We found an established diverse and similar airway microbiome at birth in both preterm and term infants, which was more diverse and different from that of older preterm infants with established chronic lung disease (bronchopulmonary dysplasia). Consistent temporal dysbiotic changes in the airway microbiome were seen from birth to the development of bronchopulmonary dysplasia in extremely preterm infants. Genus Lactobacillus was decreased at birth in infants with chorioamnionitis and in preterm infants who subsequently went on to develop lung disease. Our results, taken together with previous literature indicating a placental and amniotic fluid microbiome, suggest fetal acquisition of an airway microbiome. We speculate that the early airway microbiome may prime the developing pulmonary immune system, and dysbiosis in its development may set the stage for subsequent lung disease.</li>
</ul>
</li>
<li>
<p><em>exposure to bacterial cpg dna protects from airway allergic inflammation by expanding regulatory lung interstitial macrophages</em><br />
catherine sabatel et al. 2017<br />
<a href="http://doi.org/10.1016/j.immuni.2017.02.016">doi.org/10.1016/j.immuni.2017.02.016</a></p>
<ul>
<li>Living in a microbe-rich environment reduces the risk of developing asthma. Exposure of humans or mice to unmethylated CpG DNA (CpG) from bacteria reproduces these protective effects, suggesting a major contribution of CpG to microbe-induced asthma resistance. However, how CpG confers protection remains elusive. We found that exposure to CpG expanded regulatory lung interstitial macrophages (IMs) from monocytes infiltrating the lung or mobilized from the spleen. Trafficking of IM precursors to the lung was independent of CCR2, a chemokine receptor required for monocyte mobilization from the bone marrow. Using a mouse model of allergic airway inflammation, we found that adoptive transfer of IMs isolated from CpG-treated mice recapitulated the protective effects of CpG when administered before allergen sensitization or challenge. IM-mediated protection was dependent on IL-10, given that <em>Il10−/−</em> CpG-induced IMs lacked regulatory effects. Thus, the expansion of regulatory lung IMs upon exposure to CpG might underlie the reduced risk of asthma development associated with a microbe-rich environment.</li>
</ul>
</li>
<li>
<p><em>sex-specific impact of asthma during pregnancy on infant gut microbiota</em><br />
petya t. koleva et al. 2017<br />
<a href="http://doi.org/10.1183/13993003.00280-2017">doi.org/10.1183/13993003.00280-2017</a></p>
<ul>
<li>Asthma during pregnancy is associated with retardation of fetal growth in a sex-specific manner. Lactobacilli microbes influence infant growth. This study aimed to determine whether lactobacilli and other microbes are reduced in the gut of infants born to an asthmatic mother, and whether this differs by the sex of the infant.<br />
Mother-infant pairs (N=1021) from the Canadian Healthy Infant Longitudinal Development full-term cohort were studied. The abundance of infant faecal microbiota at 3–4 months, profiled by gene sequencing, was compared between both women with and without asthma treatment during pregnancy. Infant sex, maternal ethnicity, pre-pregnancy overweight and atopy status, birth mode, breastfeeding status and intrapartum antibiotic treatment were tested as covariates.<br />
Independent of birth mode and other covariates, male, Caucasian infants born to women with prenatal asthma harboured fewer lactobacilli in the gut at 3–4 months of age. If asthmatic mothers had pre-pregnancy overweight, the abundance of <em>Lactobacillus</em> in males was further reduced in the infant gut, whereas the microbiota of female infants was enriched with <em>Bacteroidaceae</em>. Similar differences in infant gut microbial composition according to maternal prenatal asthma status were also more evident among women with food or environmental allergies.<br />
Gut lactobacilli were less abundant in male infants, but <em>Bacteroidaceae</em> were more abundant in female infants at 3–4 months of age, following maternal asthma during pregnancy.</li>
</ul>
</li>
<li>
<p><em>farm-like indoor microbiota in non-farm homes protects children from asthma development</em><br />
pirkka v. kirjavainen et al. 2019<br />
<a href="http://doi.org/10.1038/s41591-019-0469-4">doi.org/10.1038/s41591-019-0469-4</a></p>
<ul>
<li>
<p>the presence of farm-like microbiota in an early-life home seemed to protect from asthma also in urban homes. The effect was not based on the presence of large number of different microbial species but rather differences in the relative abundance of certain bacterial groups,&quot; says Pirkka Kirjavainen, Senior Researcher at the Finnish National Institute for Health and Welfare.</p>
<p>Wearing outdoor shoes indoors, the number of siblings and the age of the house played a role</p>
<p>The study found that the microbiota in homes protecting from asthma contained a wealth of bacteria typical of the outdoor environment, including bacteria in soil. On the other hand, the proportion of microbes normally occurring in the human respiratory tract and associated with respiratory tract infections was small.</p>
<p>&quot;The key characteristic of microbiota in homes protecting from asthma appears to be large abundance of bacteria which originate from the outdoor environment and are beneficial or harmless to health, relative to bacteria that are a potential threat to health,&quot; Kirjavainen comments.</p>
<p>In urban homes, factors that increased the farm-like features in the microbiota included wearing outdoor shoes indoors, the number of siblings and the age of the house; all factors that may increase transport of outdoor microbes into the home.</p>
<p>Asthma is the most common chronic disease in children in Finland -- can new cases be prevented in the future?</p>
<p>&quot;It is interesting to see how clear of a protective effect indoor microbiota can have against the development of asthma. In contrast, it has been considerably more difficult to identify microbiota that would explain the detrimental effect of moisture damage on asthma,&quot; says Professor Juha Pekkanen.</p>
<p>Asthma is the most common chronic disease in children in Finland as well as in many other countries, and its prevalence is increasing with urbanisation. The new study supports the view that children's early exposure to 'right cocktail' of microbes may help their bodies to develop mechanisms protecting from asthma.</p>
<p>&quot;The results suggest that asthma could be prevented in the future by modifying children's early microbial exposures,&quot; says Pekkanen</p>
</li>
<li>
<p><em>abstract</em> Asthma prevalence has increased in epidemic proportions with urbanization, but growing up on traditional farms offers protection even today1. The asthma-protective effect of farms appears to be associated with rich home dust microbiota2,3, which could be used to model a health-promoting indoor microbiome. Here we show by modeling differences in house dust microbiota composition between farm and non-farm homes of Finnish birth cohorts4 that in children who grow up in non-farm homes, asthma risk decreases as the similarity of their home bacterial microbiota composition to that of farm homes increases. The protective microbiota had a low abundance of Streptococcaceae relative to outdoor-associated bacterial taxa. The protective effect was independent of richness and total bacterial load and was associated with reduced proinflammatory cytokine responses against bacterial cell wall components ex vivo. We were able to reproduce these findings in a study among rural German children2 and showed that children living in German non-farm homes with an indoor microbiota more similar to Finnish farm homes have decreased asthma risk. The indoor dust microbiota composition appears to be a definable, reproducible predictor of asthma risk and a potential modifiable target for asthma prevention.</p>
</li>
</ul>
</li>
<li>
<p><em>the upper-airway microbiota and loss of asthma control among asthmatic children</em><br />
yanjiao zhou et al. 2019<br />
<a href="http://doi.org/10.1038/s41467-019-13698-x">doi.org/10.1038/s41467-019-13698-x</a></p>
<ul>
<li>
<p>The researchers found that children who experienced early warning signs that their asthma was going to flare up were more likely to have bacteria associated with disease — including Staphylococcus, Streptococcus and Moraxella bacterial groups — living in their upper airways. In contrast, airway microbes dominated by Corynebacterium and Dolosigranulum bacteria were associated with periods of good health, when asthma was well-controlled.<br />
Beigelman and his colleagues also found that children whose airway microbial communities switched from being dominated by Corynebacterium and Dolosigranulum bacteria to being dominated by Moraxella bacteria were at the highest risk of worsening asthma symptoms compared with children whose microbial communities made any other kind of shift.</p>
</li>
<li>
<p><em>abstract</em> The airway microbiome has an important role in asthma pathophysiology. However, little is known on the relationships between the airway microbiome of asthmatic children, loss of asthma control, and severe exacerbations. Here we report that the microbiota’s dynamic patterns and compositions are related to asthma exacerbations. We collected nasal blow samples (n = 319) longitudinally during a clinical trial at 2 time-points within one year: randomization when asthma is under control, and at time of early loss of asthma control (yellow zone (YZ)). We report that participants whose microbiota was dominated by the commensal Corynebacterium + Dolosigranulum cluster at RD experience the lowest rates of YZs (p = 0.005) and have longer time to develop at least 2 episodes of YZ (p = 0.03). The airway microbiota have changed from randomization to YZ. A switch from the Corynebacterium + Dolosigranulum cluster at randomization to the Moraxella- cluster at YZ poses the highest risk of severe asthma exacerbation (p = 0.04). Corynebacterium’s relative abundance at YZ is inversely associated with severe exacerbation (p = 0.002).</p>
</li>
</ul>
</li>
</ul>
</li>
</ul>
<h2>06994</h2>
<ul>
<li>
<p><em>foliar-feeding insects acquire microbiomes from the soil rather than the host plant</em><br />
s. emilia hannula et al. 2019<br />
<a href="http://doi.org/10.1038/s41467-019-09284-w">doi.org/10.1038/s41467-019-09284-w</a></p>
<ul>
<li>
<p>Earlier NIOO research had found that belowground and aboveground insects can communicate with each other using plants as a kind of 'green telephone'. Messages can even be left in the soil to be retrieved later, like voicemail. This new research by a team of four ecologists shows that surprisingly, aboveground insects such as caterpillars can retrieve these voicemails from the soil without any mediation from plants.</p>
<p>Emilia Hannula, Feng Zhu, Robin Heinen and Martijn Bezemer conducted experiments that focused on the 'microbiome': a community of micro-organisms that you'll find in almost any place. Lining grains of sand and roots for example, but also on leaves, on teeth and in intestines.</p>
<p>Microbial ecologist Emilia Hannula explains: &quot;the composition of the microbiome in the intestines of the caterpillars we studied was astonishingly similar to that of the soil itself. There was a 75% overlap.&quot; That is indeed surprising, as expectations were that the species of bacteria and fungi found in herbivorous insects would be most similar to those on the plant.</p>
<p>First evidence</p>
<p>There's more: the soil the researchers used was from a field experiment in the Veluwe natural area, with various combinations of herbs and grasses. The differences between these plants in growth form and rate also prompted differences in the composition of the soil bacteria and soil fungi, which were still traceable in the insects in the greenhouse experiment.</p>
<p>According to the researchers, it's the first evidence that these legacy effects in the soil can have such a strong impact on the microbiome of insects. Heinen observed that if caterpillars had the option, they would &quot;move actively from the plant to the ground. They regularly spend time on the soil.&quot;</p>
<p>Caterpillars that were put on an exclusive diet of cut leaves from the same plant (dandelion) instead, without being able to move to the ground, had a much simpler intestinal flora, which was three times less varied: much more in line with that of the micro-organisms on the leaves.</p>
<p>Self-medicating</p>
<p>This knowledge could be useful not just for explaining certain scientific effects, but also to farmers and growers. &quot;The history of the soil is visible not just in plants but also in insects. So in order to control pests, for example, you'll have to take into account the soil in which you work,&quot; says Martijn Bezemer. Healthy and biodiverse soil life can contribute greatly to pest control, and provide food for healthy crops and a species-rich nature.&quot;</p>
<p>So do caterpillars really play doctor? The researchers confirm that they may be actively searching the soil for beneficial substances and micro-organisms: &quot;you could think of it as self-medicating.&quot; Among the things they observed during their experiments was an abundance of soil bacteria that are known to have a symbiotic relationship with the intestines of insects and even humans.</p>
<p>Some of these produce antibiotics: great against pathogens, which are also present in the soil of course. &quot;It's one of the reasons why we're pursuing this research further.&quot;</p>
</li>
<li>
<p><em>abstract</em> Microbiomes of soils and plants are linked, but how this affects microbiomes of aboveground herbivorous insects is unknown. We first generated plant-conditioned soils in field plots, then reared leaf-feeding caterpillars on dandelion grown in these soils, and then assessed whether the microbiomes of the caterpillars were attributed to the conditioned soil microbiomes or the dandelion microbiome. Microbiomes of caterpillars kept on intact plants differed from those of caterpillars fed detached leaves collected from plants growing in the same soil. Microbiomes of caterpillars reared on detached leaves were relatively simple and resembled leaf microbiomes, while those of caterpillars from intact plants were more diverse and resembled soil microbiomes. Plant-mediated changes in soil microbiomes were not reflected in the phytobiome but were detected in caterpillar microbiomes, however, only when kept on intact plants. Our results imply that insect microbiomes depend on soil microbiomes, and that effects of plants on soil microbiomes can be transmitted to aboveground insects feeding later on other plants.</p>
</li>
</ul>
</li>
<li>
<p><em>serum cortisol mediates the relationship between fecal ruminococcus and brain n-acetylaspartate in the young pig</em><br />
austin t. mudd et al. 2017<br />
<a href="http://doi.org/10.1080/19490976.2017.1353849">doi.org/10.1080/19490976.2017.1353849</a></p>
<ul>
<li>A dynamic relationship between the gut microbiota and brain is pivotal in neonatal development. Dysbiosis of the microbiome may result in altered neurodevelopment; however, it is unclear which specific members of microbiota are most influential and what factors might mediate the relationship between the gut and the brain. Twenty-four vaginally-derived male piglets were subjected to magnetic resonance spectroscopy at 30 d of age. Ascending colon contents, feces, and blood were collected and analyzed for volatile fatty acids, microbiota relative abundance by 16s rRNA, and serum metabolites, respectively. A mediation analysis was performed to assess the mediatory effect of serum biomarkers on the relationship between microbiota and neurometabolites. Results indicated fecal <em>Ruminococcus</em> and <em>Butyricimonas</em> predicted brain N-acetylaspartate (NAA). Analysis of serum biomarkers indicated <em>Ruminococcus</em> independently predicted serum serotonin and cortisol. A 3-step mediation indicated: i) <em>Ruminococcus</em> negatively predicted NAA, ii) <em>Ruminococcus</em> negatively predicted cortisol, and iii) a significant indirect effect (i.e., the effect of fecal <em>Ruminococcus</em> through cortisol on NAA) was observed and the direct effect became insignificant. Thus, serum cortisol fully mediated the relationship between fecal <em>Ruminococcus</em> and brain NAA. Using magnetic resonance spectroscopy, this study used a statistical mediation analysis and provides a novel perspective into the potential underlying mechanisms through which the microbiota may shape brain development. This is the first study to link <em>Ruminococcus</em>, cortisol, and NAA <em>in vivo</em>, and these findings are substantiated by previous literature indicating these factors may be influential in the etiology of neurodevelopmental disorders.</li>
</ul>
</li>
<li>
<p><em>pyrazines from bacteria and ants: convergent chemistry within an ecological niche</em><br />
eduardo a. silva-junior et al. 2018<br />
<a href="http://doi.org/10.1038/s41598-018-20953-6">doi.org/10.1038/s41598-018-20953-6</a></p>
<ul>
<li>Ants use pheromones to coordinate their communal activity. Volatile pyrazines, for instance, mediate food resource gathering and alarm behaviors in different ant species. Here we report that leaf-cutter ant-associated bacteria produce a family of pyrazines that includes members previously identified as ant trail and alarm pheromones. We found that L-threonine induces the bacterial production of the trail pheromone pyrazines, which are common for the host leaf-cutter ants. Isotope feeding experiments revealed that L-threonine along with sodium acetate were the biosynthetic precursors of these natural products and a biosynthetic pathway was proposed.</li>
</ul>
</li>
</ul>
<h2>07094</h2>
<ul>
<li>
<p><em>trim28 controls a gene regulatory network based on endogenous retroviruses in human neural progenitor cells</em><br />
per ludvik brattås et al. 2017<br />
<a href="http://doi.org/10.1016/j.celrep.2016.12.010">doi.org/10.1016/j.celrep.2016.12.010</a></p>
<ul>
<li>
<p>•Stage- and region-specific expression of ERVs during human brain development<br />
•TRIM28 binds to ERVs and induces hetereochromatin in human neural progenitor cells<br />
•Knockdown of TRIM28 in hNPCs results in the upregulation of ERV expression<br />
•Protein-coding genes located near upregulated ERVs are upregulated</p>
<p>Endogenous retroviruses (ERVs), which make up 8% of the human genome, have been proposed to participate in the control of gene regulatory networks. In this study, we find a region- and developmental stage-specific expression pattern of ERVs in the developing human brain, which is linked to a transcriptional network based on ERVs. We demonstrate that almost 10,000, primarily primate-specific, ERVs act as docking platforms for the co-repressor protein TRIM28 in human neural progenitor cells, which results in the establishment of local heterochromatin. Thereby, TRIM28 represses ERVs and consequently regulates the expression of neighboring genes. These results uncover a gene regulatory network based on ERVs that participates in control of gene expression of protein-coding transcripts important for brain development.</p>
</li>
</ul>
</li>
</ul>
<h2>07194</h2>
<ul>
<li>
<p><em>bifidobacteria or fiber protects against diet-induced microbiota-mediated colonic mucus deterioration</em><br />
bjoern o. schroeder et al. 2017<br />
<a href="http://doi.org/10.1016/j.chom.2017.11.004">doi.org/10.1016/j.chom.2017.11.004</a></p>
<ul>
<li>
<p>•Western style diet impairs penetrability and growth of the colonic mucus layer in mice<br />
Mucus defects are accompanied by distinct alterations in microbiota composition<br />
•Microbiota transplant from chow-fed mice prevents mucus aberrations in mice fed a WSD<br />
•Mucus aberrations can be prevented by <em>Bifidobacterium longum</em> and inulin treatment</p>
<p>Diet strongly affects gut microbiota composition, and gut bacteria can influence the colonic mucus layer, a physical barrier that separates trillions of gut bacteria from the host. However, the interplay between a Western style diet (WSD), gut microbiota composition, and the intestinal mucus layer is less clear. Here we show that mice fed a WSD have an altered colonic microbiota composition that causes increased penetrability and a reduced growth rate of the inner mucus layer. Both barrier defects can be prevented by transplanting microbiota from chow-fed mice. In addition, we found that administration of <em>Bifidobacterium longum</em> was sufficient to restore mucus growth, whereas administration of the fiber inulin prevented increased mucus penetrability in WSD-fed mice. We hypothesize that the presence of distinct bacteria is crucial for proper mucus function. If confirmed in humans, these findings may help to better understand diseases with an affected mucus layer, such as ulcerative colitis.</p>
</li>
</ul>
</li>
<li>
<p><em>mediated nourishment of gut microbiota protects against diet-induced obesity by restoring il-22-mediated colonic health</em><br />
jun zou et al. 2017<br />
<a href="http://doi.org/10.1016/j.chom.2017.11.003">doi.org/10.1016/j.chom.2017.11.003</a></p>
<ul>
<li>
<p>•The fermentable fiber inulin prevented high-fat diet (HFD)-induced metabolic syndrome<br />
•HFD enriched with inulin increased gut epithelial proliferation, prevented colon atrophy<br />
•Inulin restored HFD-induced microbiota depletion and microbiota-mucosa separation<br />
•Inulin effects are microbiota and IL-22, but not short-chain fatty acid, dependent</p>
<p>Dietary supplementation with fermentable fiber suppresses adiposity and the associated parameters of metabolic syndrome. Microbiota-generated fiber-derived short-chain fatty acids (SCFAs) and free fatty acid receptors including GPR43 are thought to mediate these effects. We find that while fermentable (inulin), but not insoluble (cellulose), fiber markedly protected mice against high-fat diet (HFD)-induced metabolic syndrome, the effect was not significantly impaired by either inhibiting SCFA production or genetic ablation of GPR43. Rather, HFD decimates gut microbiota, resulting in loss of enterocyte proliferation, leading to microbiota encroachment, low-grade inflammation (LGI), and metabolic syndrome. Enriching HFD with inulin restored microbiota loads, interleukin-22 (IL-22) production, enterocyte proliferation, and antimicrobial gene expression in a microbiota-dependent manner, as assessed by antibiotic and germ-free approaches. Inulin-induced IL-22 expression, which required innate lymphoid cells, prevented microbiota encroachment and protected against LGI and metabolic syndrome. Thus, fermentable fiber protects against metabolic syndrome by nourishing microbiota to restore IL-22-mediated enterocyte function.</p>
</li>
</ul>
</li>
<li>
<p><em>postnatal exposure to household disinfectants, infant gut microbiota and subsequent risk of overweight in children</em><br />
mon h. tun et al. 2018<br />
<a href="http://doi.org/10.1503/cmaj.170809">doi.org/10.1503/cmaj.170809</a></p>
<ul>
<li>Antibacterial cleaning products have the capacity to change the environmental microbiome and alter risk for child overweight. Our study provides novel information regarding the impact of these products on infant gut microbial composition and outcomes of overweight in the same population. We found <em>Lachnospiraceae</em> to be enriched in infant gut microbiota with frequent postnatal use of domestic disinfectants but not eco-friendly products; genus <em>Clostridium</em> and <em>Haemophilus</em> were reduced concurrently. Evidence of statistical mediation with <em>Lachnospiraceae</em> abundance showed a role for this disinfectant-related change to gut microbiota in causing overweight. We did not observe mediation for infant fecal <em>Enterobacteriaceae</em>, suggesting an alternate pathway for the association between postnatal eco-friendly product use and reduced child overweight. Further study is required on the mechanisms through which household cleaning products alter gut microbial composition and the subsequent role this might have on metabolic disease</li>
</ul>
</li>
</ul>
<h2>07294</h2>
<ul>
<li>
<p><em>intrapartum antibiotics for gbs prophylaxis alter colonization patterns in the early infant gut microbiome of low risk infants</em><br />
jennifer c. stearns et al. 2017<br />
<a href="http://doi.org/10.1038/s41598-017-16606-9">doi.org/10.1038/s41598-017-16606-9</a></p>
<ul>
<li>Early life microbial colonization and succession is critically important to healthy development with impacts on metabolic and immunologic processes throughout life. A longitudinal prospective cohort was recruited from midwifery practices to include infants born at full term gestation to women with uncomplicated pregnancies. Here we compare bacterial community succession in infants born vaginally, with no exposure to antibiotics (n = 53), with infants who were exposed to intrapartum antibiotic prophylaxis (IAP) for Group B <em>Streptococcus</em> (GBS; n = 14), and infants born by C-section (n = 7). Molecular profiles  of the 16 S rRNA genes indicate that there is a delay in the expansion of <em>Bifidobacterium</em>, which was the dominate infant gut colonizer, over the first 12 weeks and a persistence of <em>Escherichia</em> when IAP for GBS exposure is present during vaginal labour. Longer duration of IAP exposure increased the magnitude of the effect on <em>Bifidobacterium</em> populations, suggesting a longer delay in microbial community maturation. As with prior studies, we found altered gut colonisation following C-section that included a notable lack of Bacteroidetes. This study found that exposure of infants to IAP for GBS during vaginal birth affected aspects of gut microbial ecology that, although dramatic at early time points, disappeared by 12 weeks of age in most infants.</li>
</ul>
</li>
</ul>
<h2>07394</h2>
<h2>comparative studies</h2>
<h2>07494</h2>
<ul>
<li>
<p><em>adaptive evolution within gut microbiomes of healthy people</em><br />
shijie zhao et al. 2019<br />
<a href="http://doi.org/10.1016/j.chom.2019.03.007">doi.org/10.1016/j.chom.2019.03.007</a></p>
<ul>
<li>
<p>&quot;The strains of B. fragilis that are growing in humans have been in that gut-like environment for millions of years, so the idea that encountering a new host's gut would induce a bunch of new adaptive mutations, and that these commensals would still be rapidly evolving, was surprising to us,&quot; says Eric Alm, a professor of biological engineering and co-director of the Center for Microbiome Informatics and Therapeutics at MIT.</p>
<p>Alm and his colleagues' analysis revealed at least sixteen genes undergo within-person evolution, and the majority of mutations targeted pathways involved with fiber uptake in the cell and biosynthesis of the cell envelope. While the mutations they observed occurred in the same types of genes over and over, and within similar locations in those genes, the particular gene that acquired mutations in each person was often different, suggesting ongoing personalization of the bacteria. &quot;Bacteroides species are helping digest complex fibers in the large intestine, which are coming from the food you eat, so adaptations in them may be related to the personalized diet,&quot; Shijie Zhao, co-first author and member of Alm's lab, says.</p>
<p>The explanation as to why those genes are being targeted may not be limited to dietary choices alone, however. For example, it is possible that rapid evolution is a tactic needed to avoid phages and immune cells. &quot;These mutations might be responsible for changing the way B. fragilis interacts with the immune system,&quot; says Tami Lieberman (@contaminatedsci), co-first author and now an assistant professor in MIT's Department of Civil and Environmental Engineering. &quot;Alternately, it could be a technique to enhance their ability to fend off attacks from other members of the microbiome.&quot;</p>
<p>Additionally, the authors found differences in mutations between Western and Eastern metagenomes, with one allele highly selected for within Western participants -- with independent mutation in each person -- but rare in Chinese subjects. This striking difference is in a gene with an unknown biological role, but the authors believe that figuring out the source of different pressures across continents may yield new discoveries.</p>
<p>Typically, when researchers consider changes in the microbiome, they are looking to see if the species composition of the microbiome is remaining constant, not what might be changing within one particular species. But in light of these findings, keeping an eye on the adaptive abilities of individual commensals, particularly as it concerns probiotic choices and microbial therapies, could become more of a focus.</p>
<p>&quot;It's not a scary thing -- I think we just need to be aware of and thoughtful about the fact that you can do extensive testing in a lab, but once a commensal gets out there and into a real person in the real world, it can evolve -- its phenotype can change -- and that's something that might be hard to anticipate,&quot; Alm says.</p>
<p>Moving forward, the authors plan to undertake more mechanistic studies and conduct metagenomic and genomic analyses in different species of commensals, under both disease and healthy conditions, to better understand the extent of evolution within the microbiome.</p>
<p>&quot;If this pattern of gut bacteria evolving really quickly after introduction into a new host holds up, then we'll probably see entirely different pathways in different types of bacteria, and that's going to be pretty exciting,&quot; says Alm.</p>
</li>
<li>
<p><em>abstract</em> •Bacteroides fragilis adapts via de novo mutations within healthy people<br />
•Polysaccharide utilization and capsule synthesis pathways change during colonization<br />
•B. fragilis diversifies into coexisting sublineages within individuals<br />
•An adaptive mutation emerges with different likelihood between human populations</p>
<p>Natural selection shapes bacterial evolution in all environments. However, the extent to which commensal bacteria diversify and adapt within the human gut remains unclear. Here, we combine culture-based population genomics and metagenomics to investigate the within-microbiome evolution of Bacteroides fragilis. We find that intra-individual B. fragilis populations contain substantial de novo nucleotide and mobile element diversity, preserving years of within-person history. This history reveals multiple signatures of within-person adaptation, including parallel evolution in sixteen genes. Many of these genes are implicated in cell-envelope biosynthesis and polysaccharide utilization. Tracking evolutionary trajectories using near-daily metagenomic sampling, we find evidence for years-long coexistence in one subject despite adaptive dynamics. We used public metagenomes to investigate one adaptive mutation common in our cohort and found that it emerges frequently in Western, but not Chinese, microbiomes. Collectively, these results demonstrate that B. fragilis adapts within individual microbiomes, pointing to factors that promote long-term gut colonization.</p>
</li>
</ul>
</li>
<li>
<p><em>us immigration westernizes the human gut microbiome</em><br />
pajau vangay et al. 2018<br />
<a href="http://doi.org/10.1016/j.cell.2018.10.029">doi.org/10.1016/j.cell.2018.10.029</a></p>
<ul>
<li>
<p>•US immigration is associated with loss of gut microbiome diversity<br />
•US immigrants lose bacterial enzymes associated with plant fiber degradation<br />
•Bacteroides strains displace Prevotella strains according to time spent in the USA<br />
•Loss of diversity increases with obesity and is compounded across generations</p>
<p>Many US immigrant populations develop metabolic diseases post immigration, but the causes are not well understood. Although the microbiome plays a role in metabolic disease, there have been no studies measuring the effects of US immigration on the gut microbiome. We collected stool, dietary recalls, and anthropometrics from 514 Hmong and Karen individuals living in Thailand and the United States, including first- and second-generation immigrants and 19 Karen individuals sampled before and after immigration, as well as from 36 US-born European American individuals. Using 16S and deep shotgun metagenomic DNA sequencing, we found that migration from a non-Western country to the United States is associated with immediate loss of gut microbiome diversity and function in which US-associated strains and functions displace native strains and functions. These effects increase with duration of US residence and are compounded by obesity and across generations.</p>
</li>
</ul>
</li>
<li>
<p><em>seasonal cycling in the gut microbiome of the hadza hunter-gatherers of tanzania</em><br />
samuel a. smits et al. 2017<br />
<a href="http://doi.org/10.1126/science.aan4834">doi.org/10.1126/science.aan4834</a></p>
<ul>
<li>Although humans have cospeciated with their gut-resident microbes, it is difficult to infer features of our ancestral microbiome. Here, we examine the microbiome profile of 350 stool samples collected longitudinally for more than a year from the Hadza hunter-gatherers of Tanzania. The data reveal annual cyclic reconfiguration of the microbiome, in which some taxa become undetectable only to reappear in a subsequent season. Comparison of the Hadza data set with data collected from 18 populations in 16 countries with varying lifestyles reveals that gut community membership corresponds to modernization: Notably, the taxa within the Hadza that are the most seasonally volatile similarly differentiate industrialized and traditional populations. These data indicate that some dynamic lineages of microbes have decreased in prevalence and abundance in modernized populations.</li>
</ul>
</li>
<li>
<p><em>frequency- and amplitude-dependent microbial population dynamics during cycles of feast and famine</em><br />
jason merritt, seppe kuehn 2018<br />
<a href="http://doi.org/10.1103/PhysRevLett.121.098101">doi.org/10.1103/PhysRevLett.121.098101</a></p>
<ul>
<li>In nature microbial populations are subject to fluctuating nutrient levels. Nutrient fluctuations are important for evolutionary and ecological dynamics in microbial communities since they impact growth rates, population sizes, and biofilm formation. Here we use automated continuous-culture devices and high-throughput imaging to show that when populations of <em>Escherichia coli</em> are subjected to cycles of nutrient excess (feasts) and scarcity (famine) their abundance dynamics during famines depend on the frequency and amplitude of feasts. We show that frequency and amplitude dependent dynamics in planktonic populations arise from nutrient and history dependent rates of aggregation and dispersal. A phenomenological model recapitulates our experimental observations. Our results show that the statistical properties of environmental fluctuations have substantial impacts on spatial structure in bacterial populations driving large changes in abundance dynamics.</li>
</ul>
</li>
</ul>
<h2>07594</h2>
<ul>
<li>
<p><em>the gut microbiota of healthy aged chinese is similar to that of the healthy young</em><br />
gaorui bian 2017<br />
<a href="http://doi.org/10.1128/mSphere.00327-17">doi.org/10.1128/mSphere.00327-17</a></p>
<ul>
<li>The microbiota of the aged is variously described as being more or less diverse than that of younger cohorts, but the comparison groups used and the definitions of the aged population differ between experiments. The differences are often described by null hypothesis statistical tests, which are notoriously irreproducible when dealing with large multivariate samples. We collected and examined the gut microbiota of a cross-sectional cohort of more than 1,000 very healthy Chinese individuals who spanned ages from 3 to over 100 years. The analysis of 16S rRNA gene sequencing results used a compositional data analysis paradigm coupled with measures of effect size, where ordination, differential abundance, and correlation can be explored and analyzed in a unified and reproducible framework. Our analysis showed several surprising results compared to other cohorts. First, the overall microbiota composition of the healthy aged group was similar to that of people decades younger. Second, the major differences between groups in the gut microbiota profiles were found before age 20. Third, the gut microbiota differed little between individuals from the ages of 30 to &gt;100. Fourth, the gut microbiota of males appeared to be more variable than that of females. Taken together, the present findings suggest that the microbiota of the healthy aged in this cross-sectional study differ little from that of the healthy young in the same population, although the minor variations that do exist depend upon the comparison cohort.</li>
</ul>
</li>
<li>
<p><em>changes in the gut microbiota of urban subjects during an immersion in the traditional diet and lifestyle of a rainforest village</em><br />
kelly v. ruggles et al. 2018<br />
<a href="http://doi.org/10.1128/mSphere.00193-18">doi.org/10.1128/mSphere.00193-18</a></p>
<ul>
<li>
<p>Because the children’s gut microbiota exhibited more plasticity, these results raise an interesting possibility that urban children who eat a more traditional, high-fiber, low-fat and low-processed diet early in life might cultivate a more diverse set of gut microbes. Conversely, adults may have a limited response due to their low microbiome plasticity.<br />
Dominguez-Bello was not terribly surprised that the adults’ gut and other microbiota changed so little: “If you take traditional people and bring them to New York, give them antibiotics and McDonald’s to eat everyday, it’s not surprising that they lose diversity,” she says. “But if, as an urban dweller, you’ve already lost that gut microbe diversity and you move to a high-diversity diet, maybe you cannot ‘bloom’ diversity because you simply don’t have those microbes present anymore.”</p>
</li>
<li>
<p>People living traditional lifestyles have higher gut microbiota diversity than urban subjects. We hypothesized that shifting lifestyles from an urban environment to a traditional rainforest village would lead to changes in the microbiota of visitors, which would become more similar to the microbiota of villagers. Here, we characterized at different time points the microbiota of 7 urban visitors (5 adults and 2 children) staying in a rainforest Amerindian village for 16 days and compared them with a reference collection of samples from age-matched local villagers. We performed a 16S rRNA gene survey of samples from multiple body sites (including fecal, oral, nasal, and skin samples) using Illumina MiSeq sequencing. The main factor segregating the microbiotas of each body site was the human group (i.e., visitors versus villagers), with the visitor microbiota tending to have lower alpha diversity; the lowered alpha diversity was statistically significant in the microbiota of skin and in the children’s fecal and oral microbiota. During the rainforest period, all visitors experienced microbiota changes within their personal cloud of variation. For all body sites, the microbiota conformations in the visitor children better matched the microbiota conformations in villagers of the same age than did those of the visitor adults, which showed a lower “microbiota age” than the microbiota of the villagers. The results suggest higher stability in the adult microbiota, with the less resilient children’s microbiota responding more to dietary changes.</p>
</li>
</ul>
</li>
<li>
<p><em>novel insights from uncultivated genomes of the global human gut microbiome</em><br />
stephen nayfach et al. 2019<br />
<a href="http://doi.org/10.1038/s41586-019-1058-x">doi.org/10.1038/s41586-019-1058-x</a></p>
<ul>
<li>
<p>nearly 61,000 microbial genomes that were computationally reconstructed from 3,810 publicly available human gut metagenomes, which are datasets of all the genetic material present in a microbiome sample. The metagenome-assembled genomes (MAGs) included 2,058 previously unknown species, thereby bringing the number of known human gut species to 4,558 and increasing the phylogenetic diversity of sequenced gut bacteria by 50 percent.</p>
<p>A model community for large-scale culturing efforts</p>
<p>This work helps answer the question of why certain microbes have not been cultivated in the lab. Scientists have previously used metagenomics and single-cell genomics to discern the specific metabolic capabilities of uncultured microbes present in environmental samples. &quot;However, many environmental communities are poorly studied, so it's not clear whether or not uncultivated organisms are really uncultivable,&quot; said Stephen Nayfach, a scientist in Berkeley Lab's Environmental Genomics and Systems Biology (EGSB) division and the study's first author. &quot;The human gut, in contrast, is intensely studied with many large-scale culturing efforts, which suggests that the many of the 'wild,' uncultivated species in the human gut are difficult to culture using current approaches.&quot;</p>
<p>By comparing the reconstructed genomes of uncultivated species versus those that have been cultivated, the team found that uncultivated species' genomes are roughly 20 percent smaller, on average, and are missing numerous pathways for biosynthesis of fatty acids, amino acids, and vitamins. &quot;Genes that are commonly missing from uncultivated gut bacteria may point towards important growth factors that have been overlooked in previous culture-based studies,&quot; Nayfach said.</p>
<p>Improving genomic resources for global populations</p>
<p>With the help of a new tool called IGGsearch, the team compared the microbiomes of people with 10 different diseases to those of healthy individuals and found that nearly 40 percent of microbe-disease associations involve a species that did not previously have a genome. &quot;These disease links used to be invisible or hard to detect,&quot; said Katie Pollard, a senior investigator at the Gladstone Institutes and Biohub, and contributing author on the study.</p>
<p>One new species in the Negativicutes class, for example, was strongly depleted in people with the spinal inflammatory condition ankylosing spondylitis (AS). &quot;As an AS patient, I am thrilled that we are finally gaining a more complete picture of how the microbiome changes in this disease,&quot; she added. Additionally, the team used IGGsearch microbiome profiles to build predictive models for disease and found that prediction accuracy was &quot;significantly improved&quot; compared to existing tools that primarily quantified the abundance of cultivated species.</p>
<p>Pollard, who is also a professor at UC San Francisco, added that, until now, microbiome genomic resources have been particularly sparse for individuals living outside North America, Europe, or China. &quot;By assembling genomes from metagenomes of diverse people, we have helped to close this gap,&quot; she said.</p>
<p>Extending technologies across microbiome areas</p>
<p>EGSB senior scientist and team lead Nikos Kyrpides said that several of the computational methods and analyses Nayfach developed for this research are currently being used to enable one of the Joint Genome Institute (JGI) groundbreaking projects: analyzing a massive collection of other JGI-sequenced MAGs from diverse environments. He added that this type of analysis hinges on several critical factors: the availability of the microbiome data; the availability of the sequence data in public archives; and the lack of any data utilization restrictions from the community, as referenced in a recent Science policy paper on which he and Katie Pollard are co-authors.</p>
</li>
<li>
<p><em>abstract</em> Largely due to challenges cultivating microbes under laboratory conditions, the genome sequence of many species in the human gut microbiome remains unknown. To address this problem, we reconstructed 60,664 prokaryotic draft genomes from 3,810 faecal metagenomes from geographically and phenotypically diverse human subjects. These genomes provide reference points for 2,058 previously unknown species-level operational taxonomic units (OTUs), representing a 50% increase in the phylogenetic diversity of sequenced gut bacteria. On average, new OTUs comprise 33% of richness and 28% of species abundance per individual and are enriched in humans from rural populations. A meta-analysis of clinical gut microbiome studies pinpointed numerous disease associations for new OTUs, which have the potential to improve predictive models. Finally, our analysis revealed that uncultured gut species have undergone genome reduction with loss of certain biosynthetic pathways, which may offer clues for improving cultivation strategies in the future.</p>
</li>
</ul>
</li>
</ul>
<h2>07694</h2>
<ul>
<li>
<p><em>the gut microbiota of rural papua new guineans: composition, diversity patterns, and ecological processes</em><br />
inés martínez et al. 2015<br />
<a href="http://dx.doi.org/10.1016/j.celrep.2015.03.049">dx.doi.org/10.1016/j.celrep.2015.03.049</a></p>
<ul>
<li>
<p>•The fecal microbiota in PNG is more diverse but less individualized than in the US<br />
•Most bacterial species are shared among PNG and the US, but abundance profiles differ<br />
•Impact of lifestyle on ecological assembly processes might explain these patterns<br />
•Westernization may decrease bacterial dispersal rates, altering microbiota structure</p>
<p>Although recent research revealed an impact of westernization on diversity and composition of the human gut microbiota, the exact consequences on metacommunity characteristics are insufficiently understood, and the underlying ecological mechanisms have not been elucidated. Here, we have compared the fecal microbiota of adults from two non-industrialized regions in Papua New Guinea (PNG) with that of United States (US) residents. Papua New Guineans harbor communities with greater bacterial diversity, lower inter-individual variation, vastly different abundance profiles, and bacterial lineages undetectable in US residents. A quantification of the ecological processes that govern community assembly identified bacterial dispersal as the dominant process that shapes the microbiome in PNG but not in the US. These findings suggest that the microbiome alterations detected in industrialized societies might arise from modern lifestyle factors limiting bacterial dispersal, which has implications for human health and the development of strategies aimed to redress the impact of westernization.</p>
</li>
</ul>
</li>
<li>
<p><em>impact of edible cricket consumption on gut microbiota in healthy adults, a double-blind, randomized crossover trial</em><br />
valerie j. stull et al. 2018<br />
<a href="http://doi.org/10.1038/s41598-018-29032-2">doi.org/10.1038/s41598-018-29032-2</a></p>
<ul>
<li>Edible insects are often considered a nutritious, protein-rich, environmentally sustainable alternative to traditional livestock with growing popularity among North American consumers. While the nutrient composition of several insects is characterized, all potential health impacts have not been evaluated. In addition to high protein levels, crickets contain chitin and other fibers that may influence gut health. In this study, we evaluated the effects of consuming 25 grams/day whole cricket powder on gut microbiota composition, while assessing safety and tolerability. Twenty healthy adults participated in this six-week, double-blind, crossover dietary intervention. Participants were randomized into two study arms and consumed either cricket-containing or control breakfast foods for 14 days, followed by a washout period and assignment to the opposite treatment. Blood and stool samples were collected at baseline and after each treatment period to assess liver function and microbiota changes. Results demonstrate cricket consumption is tolerable and non-toxic at the studied dose. Cricket powder supported growth of the probiotic bacterium, Bifidobacterium animalis, which increased 5.7-fold. Cricket consumption was also associated with reduced plasma TNF-α. These data suggest that eating crickets may improve gut health and reduce systemic inflammation; however, more research is needed to understand these effects and underlying mechanisms.</li>
</ul>
</li>
<li>
<p><em>six-week endurance exercise alters gut metagenome that is not reflected in systemic metabolism in over-weight women</em><br />
eveliina munukka et al. 2018<br />
<a href="http://doi.org/10.3389/fmicb.2018.02323">doi.org/10.3389/fmicb.2018.02323</a></p>
<ul>
<li>Recent studies suggest that exercise alters the gut microbiome. We determined whether six-weeks endurance exercise, without changing diet, affected the gut metagenome and systemic metabolites of overweight women. Previously sedentary overweight women (<em>n</em> = 19) underwent a six-weeks endurance exercise intervention, but two were excluded due to antibiotic therapy. The gut microbiota composition and functions were analyzed by 16S rRNA gene amplicon sequencing and metagenomics. Body composition was analyzed with DXA X-ray densitometer and serum metabolomics with NMR metabolomics. Total energy and energy-yielding nutrient intakes were analyzed from food records using Micro-Nutrica software. Serum clinical variables were determined with KONELAB instrument. Soluble Vascular Adhesion Protein 1 (VAP-1) was measured with ELISA and its’ enzymatic activity as produced hydrogen peroxide. The exercise intervention was effective, as maximal power and maximum rate of oxygen consumption increased while android fat mass decreased. No changes in diet were observed. Metagenomic analysis revealed taxonomic shifts including an increase in <em>Akkermansia</em> and a decrease in <em>Proteobacteria</em>. These changes were independent of age, weight, fat % as well as energy and fiber intake. Training slightly increased Jaccard distance of genus level β-diversity. Training did not alter the enriched metagenomic pathways, which, according to Bray Curtis dissimilarity analysis, may have been due to that only half of the subjects’ microbiomes responded considerably to exercise. Nevertheless, tranining decreased the abundance of several genes including those related to fructose and amino acid metabolism. These metagenomic changes, however, were not translated into major systemic metabolic changes as only two metabolites, phospholipids and cholesterol in large VLDL particles, decreased after exercise. Training also decreased the amine oxidase activity of pro-inflammatory VAP-1, whereas no changes in CRP were detected. All clinical blood variables were within normal range, yet exercise slightly increased glucose and decreased LDL and HDL. In conclusion, exercise training modified the gut microbiome without greatly affecting systemic metabolites or body composition. Based on our data and existing literature, we propose that especially <em>Akkermansia</em> and <em>Proteobacteria</em> are exercise-responsive taxa. Our results warrant the need for further studies in larger cohorts to determine whether exercise types other than endurance exercise also modify the gut metagenome.</li>
</ul>
</li>
</ul>
<h2>07794</h2>
<ul>
<li>
<p><em>heat-killed lactobacillus casei confers broad protection against influenza a virus primary infection and develops heterosubtypic immunity against future secondary infection</em><br />
yu-jin jung et al. 2017<br />
<a href="http://doi.org/10.1038/s41598-017-17487-8">doi.org/10.1038/s41598-017-17487-8</a></p>
<ul>
<li>Lactic acid bacteria (LAB) are the common probiotics. Here, we investigated the antiviral protective effects of heat-killed LAB strain <em>Lactobacillus casei</em> DK128 (DK128) on influenza viruses. Intranasal treatment of mice with DK128 conferred protection against different subtypes of influenza viruses by lessening weight loss and lowering viral loads. Protection via heat-killed DK128 was correlated with an increase in alveolar macrophage cells in the lungs and airways, early induction of virus specific antibodies, reduced levels of pro-inflammatory cytokines and innate immune cells. Importantly, the mice that were protected against primary viral infection as a result of heat-killed DK128 pretreatment developed subsequent heterosubtypic immunity against secondary virus infection. For protection against influenza virus via heat-killed DK128 pretreatment, B cells and partially CD4 T cells but not CD8 T cells were required as inferred from studies using knockout mouse models. Our study provides insight into how hosts can be equipped with innate and adaptive immunity via heat-killed DK128 treatment to protect against influenza virus, supporting that heat-killed LAB may be developed as anti-virus probiotics.</li>
</ul>
</li>
<li>
<p><em>walnut consumption alters the gastrointestinal microbiota, microbially derived secondary bile acids, and health markers in healthy adults: a randomized controlled trial</em><br />
hannah holscher et al. 2018<br />
<a href="http://doi.org/10.1093/jn/nxy004">doi.org/10.1093/jn/nxy004</a></p>
<ul>
<li>Walnut consumption affected the composition and function of the human gastrointestinal microbiota, increasing the relative abundances of Firmicutes species in butyrate-producing Clostridium clusters XIVa and IV, including <em>Faecalibacterium</em> and <em>Roseburia</em>, and reducing microbially derived, proinflammatory secondary bile acids and LDL cholesterol. These results suggest that the gastrointestinal microbiota may contribute to the underlying mechanisms of the beneficial health effects of walnut consumption.</li>
</ul>
</li>
<li>
<p><em>daylight exposure modulates bacterial communities associated with household dust</em><br />
ashkaan k. fahimipour et al. 2018<br />
<a href="http://doi.org/10.1186/s40168-018-0559-4">doi.org/10.1186/s40168-018-0559-4</a></p>
<ul>
<li>Microbial communities associated with indoor dust abound in the built environment. The transmission of sunlight through windows is a key building design consideration, but the effects of light exposure on dust communities remain unclear. We report results of an experiment and computational models designed to assess the effects of light exposure and wavelengths on the structure of the dust microbiome. Specifically, we placed household dust in replicate model “rooms” with windows that transmitted visible, ultraviolet, or no light and measured taxonomic compositions, absolute abundances, and viabilities of the resulting bacterial communities.<br />
Results<br />
Light exposure per se led to lower abundances of viable bacteria and communities that were compositionally distinct from dark rooms, suggesting preferential inactivation of some microbes over others under daylighting conditions. Differences between communities experiencing visible and ultraviolet light wavelengths were relatively minor, manifesting primarily in abundances of dead human-derived taxa. Daylighting was associated with the loss of a few numerically dominant groups of related microorganisms and apparent increases in the abundances of some rare groups, suggesting that a small number of microorganisms may have exhibited modest population growth under lighting conditions. Although biological processes like population growth on dust could have generated these patterns, we also present an alternate statistical explanation using sampling models from ecology; simulations indicate that artefactual, apparent increases in the abundances of very rare taxa may be a null expectation following the selective inactivation of dominant microorganisms in a community.<br />
Conclusions<br />
Our experimental and simulation-based results indicate that dust contains living bacterial taxa that can be inactivated following changes in local abiotic conditions and suggest that the bactericidal potential of ordinary window-filtered sunlight may be similar to ultraviolet wavelengths across dosages that are relevant to real buildings.</li>
</ul>
</li>
<li>
<p><em>inferring metabolic mechanisms of interaction within a defined gut microbiota</em><br />
gregory l. medlock et al. 2018<br />
<a href="http://doi.org/10.1016/j.cels.2018.08.003">doi.org/10.1016/j.cels.2018.08.003</a></p>
<ul>
<li>
<p>•<em>In vitro</em> monoculture and co-culture of six members of a defined murine microbiota<br />
•Growth and metabolome profiling to identify ecological and metabolic interactions<br />
•Development and application of a constant yield expectation (ConYE) model<br />
•Testing of inferred cross-feeding that contributed to commensal interaction</p>
<p>The diversity and number of species present within microbial communities create the potential for a multitude of interspecies metabolic interactions. Here, we develop, apply, and experimentally test a framework for inferring metabolic mechanisms associated with interspecies interactions. We perform pairwise growth and metabolome profiling of co-cultures of strains from a model mouse microbiota. We then apply our framework to dissect emergent metabolic behaviors that occur in co-culture. Based on one of the inferences from this framework, we identify and interrogate an amino acid cross-feeding interaction and validate that the proposed interaction leads to a growth benefit <em>in vitro</em>. Our results reveal the type and extent of emergent metabolic behavior in microbial communities composed of gut microbes. We focus on growth-modulating interactions, but the framework can be applied to interspecies interactions that modulate any phenotype of interest within microbial communities.</p>
</li>
</ul>
</li>
<li>
<p><em>microbial interkingdom interactions in roots promote arabidopsis survival</em><br />
paloma durán et al. 2018<br />
<a href="http://doi.org/10.1016/j.cell.2018.10.020">doi.org/10.1016/j.cell.2018.10.020</a></p>
<ul>
<li>
<p>•Roots of healthy plants are colonized by multi-kingdom microbial consortia<br />
•Bacterial Root Commensals (BRCs) shape fungal and oomycetal community structure<br />
•BRCs protect plants against fungi and oomycetes<br />
•Biocontrol activity of BRCs is a redundant trait and essential for plant survival</p>
<p>Roots of healthy plants are inhabited by soil-derived bacteria, fungi, and oomycetes that have evolved independently in distinct kingdoms of life. How these microorganisms interact and to what extent those interactions affect plant health are poorly understood. We examined root-associated microbial communities from three Arabidopsis thaliana populations and detected mostly negative correlations between bacteria and filamentous microbial eukaryotes. We established microbial culture collections for reconstitution experiments using germ-free A. thaliana. In plants inoculated with mono- or multi-kingdom synthetic microbial consortia, we observed a profound impact of the bacterial root microbiota on fungal and oomycetal community structure and diversity. We demonstrate that the bacterial microbiota is essential for plant survival and protection against root-derived filamentous eukaryotes. Deconvolution of 2,862 binary bacterial-fungal interactions ex situ, combined with community perturbation experiments in planta, indicate that biocontrol activity of bacterial root commensals is a redundant trait that maintains microbial interkingdom balance for plant health.</p>
</li>
</ul>
</li>
<li>
<p><em>salt-tolerant halophyte rhizosphere bacteria stimulate growth of alfalfa in salty soil</em><br />
jennifer kearl et al. 2019<br />
<a href="http://doi.org/10.3389/fmicb.2019.01849">doi.org/10.3389/fmicb.2019.01849</a></p>
<ul>
<li>
<p>&quot;We've long wondered if increasingly salty land was just a losing battle or if there was something we could do about it,&quot; Nielsen said. &quot;Now we have shown there is something we can do about it.&quot;</p>
</li>
<li>
<p>scientists have used bacteria found in the roots of salt-tolerant plants to successfully inoculate alfalfa plants against overly salty soil.</p>
<p>&quot;We take the roots of these salt-tolerant plants (called halophytes), grind them up and grow the bacteria in a petri dish in the lab,&quot; Nielsen said. &quot;Doing this, we isolated over 40 different bacteria isolates, some of which can tolerate ocean-level salt content.&quot;</p>
<p>The team then applied the bacteria isolates to alfalfa seeds through a solution and tested the alfalfa's ability to grow in high-saline conditions. They saw significant growth of the alfalfa both in their lab and in greenhouse experiments carried out by collaborators at the Institute for Advanced Learning and Research in Virginia.</p>
<p>The study identifies two specific bacteria isolates -- Halomonas and Bacillus -- that worked to stimulate plant growth in the presence of 1 percent sodium chloride (salt), a level that significantly inhibits growth of uninoculated plants. This discovery is significant since soils throughout areas of China, Australia and the Middle East have grown increasingly salty, as well as major farmland in the southwest United States.</p>
<p>&quot;As an area of land is repeatedly used for farming, the salinity rises; the irrigation water has salt in it and when it evaporates or is taken up by the plants, the salt is left behind,&quot; said student Caitlyn McNary, one of six BYU undergraduate co-authors on the paper. &quot;With what we've found, lands that are now unable to sustain plant life due to high salinity could once again be used for crops.&quot;</p>
<p>In addition to the work on alfalfa, America's No. 4 crop, the research team has already started to conduct lab and greenhouse experiments on rice, green beans and lettuce. The next step is to carry out field trials on the inoculated crops.</p>
</li>
<li>
<p><em>abstract</em> Halophytes are plants that are adapted to grow in saline soils, and have been widely studied for their physiological and molecular characteristics, but little is known about their associated microbiomes. Bacteria were isolated from the rhizosphere and as root endophytes of Salicornia rubra, Sarcocornia utahensis, and Allenrolfea occidentalis, three native Utah halophytes. A total of 41 independent isolates were identified by 16S rRNA gene sequencing analysis. Isolates were tested for maximum salt tolerance, and some were able to grow in the presence of up to 4 M NaCl. Pigmentation, Gram stain characteristics, optimal temperature for growth, and biofilm formation of each isolate aided in species identification. Some variation in the bacterial population was observed in samples collected at different times of the year, while most of the genera were present regardless of the sampling time. Halomonas, Bacillus, and Kushneria species were consistently isolated both from the soil and as endophytes from roots of all three plant species at all collection times. Non-culturable bacterial species were analyzed by Illumina DNA sequencing. The most commonly identified bacteria were from several phyla commonly found in soil or extreme environments: Acidobacteria, Actinobacteria, Bacteroidetes, Chloroflexi, and Gamma- and Delta-Proteobacteria. Isolates were tested for the ability to stimulate growth of alfalfa under saline conditions. This screening led to the identification of one Halomonas and one Bacillus isolate that, when used to inoculate young alfalfa seedlings, stimulate plant growth in the presence of 1% NaCl, a level that significantly inhibits growth of uninoculated plants. The same bacteria used in the inoculation were recovered from surface sterilized alfalfa roots, indicating the ability of the inoculum to become established as an endophyte. The results with these isolates have exciting promise for enhancing the growth of inoculated alfalfa in salty soil.</p>
</li>
</ul>
</li>
<li>
<p><em>coral-associated bacteria demonstrate phylosymbiosis and cophylogeny</em><br />
f. joseph pollock et al. 2018<br />
<a href="http://doi.org/10.1038/s41467-018-07275-x">doi.org/10.1038/s41467-018-07275-x</a></p>
<ul>
<li>Scleractinian corals’ microbial symbionts influence host health, yet how coral microbiomes assembled over evolution is not well understood. We survey bacterial and archaeal communities in phylogenetically diverse Australian corals representing more than 425 million years of diversification. We show that coral microbiomes are anatomically compartmentalized in both modern microbial ecology and evolutionary assembly. Coral mucus, tissue, and skeleton microbiomes differ in microbial community composition, richness, and response to host vs. environmental drivers. We also find evidence of coral-microbe phylosymbiosis, in which coral microbiome composition and richness reflect coral phylogeny. Surprisingly, the coral skeleton represents the most biodiverse coral microbiome, and also shows the strongest evidence of phylosymbiosis. Interactions between bacterial and coral phylogeny significantly influence the abundance of four groups of bacteria–including Endozoicomonas-like bacteria, which divide into host-generalist and host-specific subclades. Together these results trace microbial symbiosis across anatomy during the evolution of a basal animal lineage.</li>
</ul>
</li>
<li>
<p>causal</p>
</li>
<li>
<p><em>microbially produced imidazole propionate impairs insulin signaling through mtorc1</em><br />
ara koh et al. 2018<br />
<a href="http://doi.org/10.1016/j.cell.2018.09.055">doi.org/10.1016/j.cell.2018.09.055</a></p>
<ul>
<li>
<p>•Imidazole propionate levels are increased in subjects with type 2 diabetes (T2D)<br />
•Imidazole propionate is produced from histidine by T2D-associated bacteria<br />
•Imidazole propionate impairs glucose tolerance and insulin signaling<br />
•Imidazole propionate inhibits IRS via activation of p38γ/p62/mTORC1</p>
<p>Interactions between the gut microbiota, diet, and the host potentially contribute to the development of metabolic diseases. Here, we identify imidazole propionate as a microbially produced histidine-derived metabolite that is present at higher concentrations in subjects with versus without type 2 diabetes. We show that imidazole propionate is produced from histidine in a gut simulator at higher concentrations when using fecal microbiota from subjects with versus without type 2 diabetes and that it impairs glucose tolerance when administered to mice. We further show that imidazole propionate impairs insulin signaling at the level of insulin receptor substrate through the activation of p38γ MAPK, which promotes p62 phosphorylation and, subsequently, activation of mechanistic target of rapamycin complex 1 (mTORC1). We also demonstrate increased activation of p62 and mTORC1 in liver from subjects with type 2 diabetes. Our findings indicate that the microbial metabolite imidazole propionate may contribute to the pathogenesis of type 2 diabetes.</p>
</li>
</ul>
</li>
<li>
<p><em>the short-chain fatty acid propionate increases glucagon and fabp4 production, impairing insulin action in mice and humans</em><br />
amir tirosh et al. 2019<br />
<a href="http://doi.org/10.1126/scisignal.aav5938">doi.org/10.1126/scisignal.aav5938</a></p>
<ul>
<li>
<p>combined data from a randomized placebo-controlled trial in humans and mouse studies, indicated that propionate can trigger a cascade of metabolic events that leads to insulin resistance and hyperinsulinemia -- a condition marked by excessive levels of insulin. The findings also showed that in mice, chronic exposure to propionate resulted in weight gain and insulin resistance.</p>
<p>The study will be published online in Science Translational Medicine on April 24, 2019.</p>
<p>&quot;Understanding how ingredients in food affect the body's metabolism at the molecular and cellular level could help us develop simple but effective measures to tackle the dual epidemics of obesity and diabetes,&quot; said Gökhan S. Hotam??l?gil, James Stevens Simmons Professor of Genetics and Metabolism and Director of the Sabri Ülker Center for Metabolic Research at Harvard Chan School.</p>
<p>More than 400 million people worldwide suffer from diabetes, and the rate of diabetes incidence is projected to increase 40% by 2040 despite extensive efforts to curb the disease. The surging rates of diabetes, as well as obesity, in the last 50 years indicate that environmental and dietary factors must be influencing the growth of this epidemic. Researchers have suggested that dietary components including ingredients used for preparation or preservation of food may be a contributing factor, but there is little research evaluating these molecules.</p>
<p>For this study, the researchers focused on propionate, a naturally occurring short-chain fatty acid that helps prevents mold from forming on foods. They first administered this short chain fatty acid to mice and found that it rapidly activated the sympathetic nervous system, which led to a surge in hormones, including glucagon, norepinephrine, and a newly discovered gluconeogenic hormone called fatty acid-binding protein 4 (FABP4). This in turn led the mice to produce more glucose from their liver cells, leading to hyperglycemia -- a defining trait of diabetes. Moreover, the researchers found that chronic treatment of mice with a dose of propionate that was equivalent to the amount typically consumed by humans led to significant weight gain in the mice, as well as insulin resistance.</p>
<p>To determine how the findings in mice may translate to humans, the researchers established a double-blinded placebo-controlled study that included 14 healthy participants. The participants were randomized into two groups: One group received a meal that contained one gram of propionate as an additive and the other group was given a meal that contained a placebo. Blood samples were collected before the meal, within 15 minutes of eating the meal, and every 30 minutes thereafter for four hours.</p>
<p>The researchers found that people who consumed the meal containing propionate had significant increases in norepinephrine as well as increases in glucagon and FABP4 soon after eating the meal. The findings indicate that propionate may act as a &quot;metabolic disruptor&quot; that potentially increases the risk for diabetes and obesity in humans. The researchers noted that while propionate is generally recognized as safe by the U.S. Food and Drug Administration, these new findings warrant further investigation into propionate and potential alternatives that could be used in food preparation.</p>
</li>
</ul>
</li>
<li>
<p><em>propionic acid induces gliosis and neuro-inflammation through modulation of pten/akt pathway in autism spectrum disorder</em><br />
latifa s. abdell et al. 2019<br />
<a href="http://doi.org/10.1038/s41598-019-45348-z">doi.org/10.1038/s41598-019-45348-z</a></p>
<ul>
<li>
<p>Drs. Saleh Naser, Latifa Abdelli and UCF undergraduate research assistant Aseela Samsam have identified the molecular changes that happen when neural stem cells are exposed to high levels of an acid commonly found in processed foods. In a study published June 19 in Scientific Reports, a Nature journal, the UCF scientists discovered how high levels of Propionic Acid (PPA), used to increase the shelf life of packaged foods and inhibit mold in commercially processed cheese and bread, reduce the development of neurons in fetal brains.</p>
<p>Dr. Naser, who specializes in gastroenterology research at the College of Medicine's Burnett School of Biomedical Sciences, began the study after reports showed that autistic children often suffer from gastric issues such as irritable bowel syndrome. He wondered about a possible link between the gut and the brain and began examining how the microbiome -- or gut bacteria -- differed between people with autism and those who do not have the condition.</p>
<p>&quot;Studies have shown a higher level of PPA in stool samples from children with autism and the gut microbiome in autistic children is different,&quot; Dr. Naser said. &quot;I wanted to know what the underlying cause was.&quot;</p>
<p>In the lab, the scientists found exposing neural stem cells to excessive PPA damages brain cells in several ways. First, the acid disrupts the natural balance between brain cells by reducing the number of neurons and over-producing glial cells. While glial cells help develop and protect neuron function, too many glia cells disturb connectivity between neurons. They also cause inflammation, which has been noted in the brains of autistic children.</p>
<p>Excessive amounts of the acid also shorten and damage pathways that neurons use to communicate with the rest of the body. The combination of reduced neurons and damaged pathways impede the brain's ability to communicate, resulting in behaviors that are often found in children with autism, including repetitive behavior, mobility issues and inability to interact with others.</p>
<p>Previous studies have proposed links between autism and environmental and genetic factors, but Drs. Naser and Abdelli say their study is the first to discover the molecular link between elevated levels of PPA, proliferation of glial cells, disturbed neural circuitry and autism. The 18-month study was self-funded by UCF.</p>
<p>PPA occurs naturally in the gut and a mother's microbiome changes during pregnancy and can cause increases in the acid. But Drs. Naser and Abdelli said eating packaged foods containing the acid can further increase PPA in the woman's gut, which then crosses to the fetus.</p>
<p>More research needs to be done before drawing clinical conclusions. Next, the research team will attempt to validate its findings in mice models by seeing if a high PPA maternal diet causes autism in mice genetically predisposed to the condition. There is no cure for autism, which affects about 1 in 59 children, but the scientists hope their findings will advance studies for ways to prevent the disorder.</p>
</li>
<li>
<p><em>abstract</em> Autism spectrum disorder (ASD) is a neurodevelopmental disorder characterized by glia over-proliferation, neuro-inflammation, perturbed neural circuitry, and gastrointestinal symptoms. The role of gut dys-biosis in ASD is intriguing and should be elucidated. We investigated the effect of Propionic acid (PPA), a short-chain fatty acid (SCFA) and a product of dys-biotic ASD gut, on human neural stem cells (hNSCs) proliferation, differentiation and inflammation. hNSCs proliferated to 66 neuropsheres when exposed to PPA versus 45 in control. The neurosphere diameter also increased at day 10 post PPA treatment to (Mean: 193.47 um ± SEM: 6.673 um) versus (154.16 um ± 9.95 um) in control, p &lt; 0.001. Pre-treatment with β-HB, SCFA receptor inhibitor, hindered neurosphere expansion (p &lt; 0.001). While hNSCs spontaneously differentiated to (48.38% ± 6.08%) neurons (Tubulin-IIIβ positive) and (46.63% ± 2.5%) glia (GFAP positive), PPA treatment drastically shifted differentiation to 80% GFAP cells (p &lt; 0.05). Following 2 mM PPA exposure, TNF-α transcription increased 4.98 fold and the cytokine increased 3.29 fold compared to control (P &lt; 0.001). Likewise, GPR41 (PPA receptor) and pro-survival p-Akt protein were elevated (p &lt; 0.001). PTEN (Akt inhibitor) level decreased to (0.42 ug/ul ± 0.04 ug/ul) at 2 mM PPA compared to (0.83 ug/ul ± 0.09 ug/ul) in control (p &lt; 0.001). PPA at 2 mM decreased neurite outgrowth to (80.70 um ± 5.5 um) compared to (194.93 um ± 19.7 um) in control. Clearly, the data supports a significant role for PPA in modulating hNSC patterning leading to gliosis, disturbed neuro-circuitry, and inflammatory response as seen in ASD.</p>
</li>
</ul>
</li>
</ul>
<h2>dysbiosis</h2>
<h2>07894</h2>
<ul>
<li>
<p><em>global chemical effects of the microbiome include new bile-acid conjugations</em><br />
robert a. quinn et al. 2020<br />
<a href="http://doi.org/10.1038/s41586-020-2047-9">doi.org/10.1038/s41586-020-2047-9</a></p>
<ul>
<li>
<p>Much of our knowledge about bile hasn’t changed in many decades. It’s produced in the liver, stored in our gall bladder and injected into our intestine when we eat, where it breaks down fats in our gut. In fact, the first bile acid was discovered in 1848, and the scientists who revealed the structure of bile acids in 1928 won the Nobel Prize. That’s a long time ago.<br />
“Since then, our understanding of the chemistry of bile production in the liver was that the cholesterol backbone of the bile acid structure is linked to the amino acids glycine or taurine to produce our primary bile acids,” said Robert Quinn, assistant professor of biochemistry and molecular biology and Global Impact researcher, and lead author of the study. “It begs the question of how the new bile acids we’ve discovered have remained hidden during the last 170 years of bile acid chemical research.”<br />
These new bile acids are not produced by our enzymes; they’re made by microbes in our gut. This discovery will change how medical textbooks address digestion, and it contributes to an ever-growing body of knowledge supporting the importance of the microbiome, the collective community of bacteria and other microorganisms living in our guts.<br />
Quinn’s team, comprised of scientists from MSU, the University of California San Diego and a number of collaborating institutions, showed that microbes in the gut, members of the microbiome, produce unique bile acids by conjugating the cholesterol backbone with myriad other amino acids.<br />
This represents a fifth mechanism of bile acid metabolism by the microbiome that greatly expands our understanding of mammalian bile.<br />
While much of the study was conducted in mice, these novel bile acids were also found in humans. And here’s the kicker that will guide future research: They’re particularly abundant in the guts of people suffering with gastrointestinal diseases, such as Crohn’s disease and cystic fibrosis.<br />
“These molecules can alter signaling pathways in the human gut that result in a reduction of overall bile acid production, representing a new mechanism where our gut bacteria can manipulate our own physiology,” Quinn said.</p>
</li>
<li>
<p><em>abstract</em> A mosaic of cross-phylum chemical interactions occurs between all metazoans and their microbiomes. A number of molecular families that are known to be produced by the microbiome have a marked effect on the balance between health and disease1,2,3,4,5,6,7,8,<a href="https://www.nature.com/articles/s41586-020-2047-9#ref-CR9">9</a>. Considering the diversity of the human microbiome (which numbers over 40,000 operational taxonomic units<a href="https://www.nature.com/articles/s41586-020-2047-9#ref-CR10">10</a>), the effect of the microbiome on the chemistry of an entire animal remains underexplored. Here we use mass spectrometry informatics and data visualization approaches11,12,<a href="https://www.nature.com/articles/s41586-020-2047-9#ref-CR13">13</a> to provide an assessment of the effects of the microbiome on the chemistry of an entire mammal by comparing metabolomics data from germ-free and specific-pathogen-free mice. We found that the microbiota affects the chemistry of all organs. This included the amino acid conjugations of host bile acids that were used to produce phenylalanocholic acid, tyrosocholic acid and leucocholic acid, which have not previously been characterized despite extensive research on bile-acid chemistry<a href="https://www.nature.com/articles/s41586-020-2047-9#ref-CR14">14</a>. These bile-acid conjugates were also found in humans, and were enriched in patients with inflammatory bowel disease or cystic fibrosis. These compounds agonized the farnesoid X receptor in vitro, and mice gavaged with the compounds showed reduced expression of bile-acid synthesis genes in vivo. Further studies are required to confirm whether these compounds have a physiological role in the host, and whether they contribute to gut diseases that are associated with microbiome dysbiosis.</p>
</li>
</ul>
</li>
<li>
<p>induced by antibiotics</p>
<ul>
<li>
<p><em>salmonella persisters promote the spread of antibiotic resistance plasmids in the gut</em><br />
erik bakkeren et al. 2019<br />
<a href="http://doi.org/10.1038/s41586-019-1521-8">doi.org/10.1038/s41586-019-1521-8</a></p>
<ul>
<li>
<p>It's a common assumption that the resistance genes spread primarily when antibiotics are used, a rationale backed up by Darwin's theory: only in cases where antibiotics are actually being used does a resistant bacterium have an advantage over other bacteria. In an antibiotic-free environment, resistant bacteria have no advantage. This explains why health experts are concerned about the excessive use of antibiotics and call for more restrictions on their use.</p>
<p>However, a team of researchers led by scientists from ETH Zurich and the University of Basel have now discovered an additional, previously unknown mechanism that spreads resistance in intestinal bacteria that is independent of the use of antibiotics. &quot;Restricting the use of antibiotics is important and the indeed the right thing to do, but this measure alone is not sufficient to prevent the spread of resistance,&quot; says Médéric Diard, who until recently held a post at ETH Zurich and is now a professor at the Biozentrum of the University of Basel. He continues, &quot;If you want to control the spread of resistance genes, you have to start with the resistant microorganisms themselves and prevent these from spreading through, say, more effective hygiene measures or vaccinations.&quot; Diard led the research project together with Wolf-Dietrich Hardt, Professor for Microbiology at ETH Zurich.</p>
<p>Two resistance mechanisms combined</p>
<p>Persistent bacteria, also known as persisters, are responsible for this newly discovered resistance-spreading mechanism. Scientists have known for some time that, just like bacteria that carry resistance genes, persisters can survive antibiotic treatment. They fall into a temporary, dormant state and can reduce their metabolism to a minimum, which prevents the antibiotics from killing them. In the case of salmonella, the bacteria become dormant when they penetrate the body tissue from inside the gut. Once they have invaded the tissue, the persisters can live there undetected for months before awakening from their dormant state. If the conditions are conducive to bacterial survival, the infection can flare up again.</p>
<p>Even if the persisters don't cause a new infection, they can still have an adverse effect, as the scientists report in the journal Nature. In salmonella, a combination of the two resistance mechanisms is common: persisters that also carry small DNA molecules (plasmids) containing resistance genes.</p>
<p>Reservoir of genetic information</p>
<p>In experiments with mice, the researchers demonstrated that dormant salmonella in the gut can pass their resistance genes on to other individual bacteria of the same species and even to other species, such as E. coli from the normal intestinal flora. Their experiments showed that persisters are very efficient at sharing their resistance genes as soon as they awaken from their dormant state and encounter other bacteria that are susceptible to gene transfer. &quot;By exploiting their persistent host bacterium, the resistance plasmids can survive for a prolonged period in one host before transferring into other bacteria. This speeds up their spread,&quot; ETH professor Hardt explains. It's important to note here that this transfer happens regardless of whether antibiotics are present or not.</p>
<p>The researchers now want to take their findings in mice and explore these more closely in livestock that frequently suffer from salmonella infections, such as pigs. The scientists also want to investigate whether it's possible to control the spread of resistance in livestock populations with probiotics or with a vaccination against salmonella.</p>
</li>
<li>
<p><em>abstract</em> The emergence of antibiotic-resistant bacteria through mutations or the acquisition of genetic material such as resistance plasmids represents a major public health issue1,2. Persisters are subpopulations of bacteria that survive antibiotics by reversibly adapting their physiology3,4,5,6,7,8,9,10, and can promote the emergence of antibiotic-resistant mutants11. We investigated whether persisters can also promote the spread of resistance plasmids. In contrast to mutations, the transfer of resistance plasmids requires the co-occurrence of both a donor and a recipient bacterial strain. For our experiments, we chose the facultative intracellular entero-pathogen Salmonella enterica serovar Typhimurium (S. Typhimurium) and Escherichia coli, a common member of the microbiota12. S. Typhimurium forms persisters that survive antibiotic therapy in several host tissues. Here we show that tissue-associated S. Typhimurium persisters represent long-lived reservoirs of plasmid donors or recipients. The formation of reservoirs of S. Typhimurium persisters requires Salmonella pathogenicity island (SPI)-1 and/or SPI-2 in gut-associated tissues, or SPI-2 at systemic sites. The re-seeding of these persister bacteria into the gut lumen enables the co-occurrence of donors with gut-resident recipients, and thereby favours plasmid transfer between various strains of Enterobacteriaceae. We observe up to 99% transconjugants within two to three days of re-seeding. Mathematical modelling shows that rare re-seeding events may suffice for a high frequency of conjugation. Vaccination reduces the formation of reservoirs of persisters after oral infection with S. Typhimurium, as well as subsequent plasmid transfer. We conclude that—even without selection for plasmid-encoded resistance genes—small reservoirs of pathogen persisters can foster the spread of promiscuous resistance plasmids in the gut.</p>
</li>
</ul>
</li>
<li>
<p><em>a nationwide study in denmark of the association between treated infections and the subsequent risk of treated mental disorders in children and adolescents</em><br />
ole köhler-forsberg et al. 2018<br />
<a href="http://doi.org/10.1001/jamapsychiatry.2018.3428">doi.org/10.1001/jamapsychiatry.2018.3428</a></p>
<p>effect of antibiotics was oddly minimised in abstract, conclusions, and media attention of this article, but if you know microbiome studies the incidental revelations in the paper itself are smoking guns.</p>
<ul>
<li>Antibiotics were the anti-infective agents associated with the largest risk increase in mental disorders (Table 1), whereas antivirals and antimycotics did not increase the risk estimates.</li>
<li>the risks were increased for schizophrenia spectrum disorders, OCD, personality and behavior disorders, mental retardation, ASD, ADHD, ODD/CD, and tic disorders. The results for anti-infective agents were higher among individuals with prior hospitalizations due to infections and increased in a dose-response association.</li>
<li>To our knowledge, the present study is the first to indicate that any treated infection, including less severe infection, is associated with an increased risk of a wide range of childhood and adolescent mental disorders.</li>
</ul>
</li>
<li>
<p><em>long-term risk of neuropsychiatric disease after exposure to infection in utero</em><br />
benjamin j. s. al-haddad et al. 2019<br />
<a href="http://doi.org/10.1001/jamapsychiatry.2019.0029">doi.org/10.1001/jamapsychiatry.2019.0029</a></p>
<p>yet another study missing the point that it could be exposure to hospital environments or medical intervention that is causative rather than infection itself</p>
<ul>
<li>
<p>&quot;The results indicate that safeguarding against and preventing infection during pregnancy as far as possible by, for instance, following flu vaccination recommendations, may be called for,&quot; says Verena Sengpiel, Associate Professor of Obstetrics and Gynecology at Sahlgrenska Academy, University of Gothenburg, and last author of the study, published in the journal JAMA Psychiatry.</p>
<p>Maternal infection with certain infectious agents, such as cytomegalovirus (CMV) or the herpes virus, are already known to be capable of harming fetal brain development and boosting the risk of certain psychiatric disorders.</p>
<p>The findings of the current study, however, also show that infection in general during pregnancy, too -- including when the actual infectious agent does not reach the fetal brain -- is related to elevated risk of the child developing autism or depression later in life.</p>
<p>More autism and depression</p>
<p>The study is based on data on all children, totaling almost 1.8 million, born in Sweden during the years 1973-2014. The particulars from the Swedish Medical Birth Register were linked to the national inpatient register, which records whether the mother was treated in hospital with an infection diagnosis during the pregnancy concerned.</p>
<p>Using the inpatient register, the researchers also monitored these children's mental health until 2014, when the oldest were aged 41.</p>
<p>It was found that if, during pregnancy, a mother with an infection diagnosis received hospital treatment, there was a marked rise in the risk of her child needing hospital care later in life, with a diagnosis of either autism or depression. The increase in risk was 79 percent for autism and 24 percent for depression.</p>
<p>In contrast, there was no association between the mothers being in hospital with an infection diagnosis during pregnancy and two other psychiatric diagnoses studied in their children: bipolar disorder and psychosis, including schizophrenia.</p>
<p>Increased risk even after mild infection</p>
<p>The pregnant women in the study may have been hospitalized with diagnoses other than infections, but then had infections diagnosed during their stay as well. The elevated risk of mental ill-health in the child was also evident after infections in the pregnant women that are usually considered mild, such as a common urinary tract infection.</p>
<p>The study, which was observational, provides no answer on how maternal infection during pregnancy affects fetal brain development. However, other studies have shown that an infection in the mother leads to an inflammatory reaction, and that some inflammatory proteins can affect gene expression in fetal brain cells.</p>
<p>Other research shows that inflammation in the mother boosts production of the neurotransmitter serotonin in the placenta, which may conceivably affect the unborn child's brain development.</p>
</li>
<li>
<p><em>abstract</em> Question  Does exposure to maternal infection during pregnancy increase the long-term risk for major psychiatric disorders in the child?</p>
<p>Findings  In this Swedish population-based cohort study of children born between 1973 and 2014, exposure to infection in pregnancy significantly increased the risk for autism spectrum disorder and depression.</p>
<p>Meaning  Maternal infection during pregnancy may be responsible for some portion of autism and depression in childhood and adulthood among the exposed offspring.</p>
<p>Abstract<br />
Importance  The developmental origins of mental illness are incompletely understood. Although the development of autism and schizophrenia are linked to infections during fetal life, it is unknown whether more common psychiatric conditions such as depression might begin in utero.</p>
<p>Objective  To estimate the risk of psychopathologic conditions imparted from fetal exposure to any maternal infection while hospitalized during pregnancy.</p>
<p>Design, Setting, and Participants  A total of 1 791 520 Swedish children born between January 1, 1973, and December 31, 2014, were observed for up to 41 years using linked population-based registries. Children were excluded if they were born too late to contribute person-time, died before being at risk for the outcome, or were missing particular model data. Infection and psychiatric diagnoses were derived using codes from hospitalizations. Directed acyclic graphs were developed from a systematic literature review to determine Cox proportional hazards regression models for risk of psychopathologic conditions in the children. Results were evaluated using probabilistic and simple bias analyses. Statistical analysis was conducted from February 10 to October 17, 2018.</p>
<p>Exposures  Hospitalization during pregnancy with any maternal infection, severe maternal infection, and urinary tract infection.</p>
<p>Main Outcomes and Measures  Inpatient diagnosis of autism, depression, bipolar disorder, or psychosis among offspring.</p>
<p>Results  A total of 1 791 520 Swedish-born children (48.6% females and 51.4% males) were observed from birth up to age 41 years, with a total of 32 125 813 person-years. Within the directed acyclic graph framework of assumptions, fetal exposure to any maternal infection increased the risk of an inpatient diagnosis in the child of autism (hazard ratio [HR], 1.79; 95% CI, 1.34-2.40) or depression (HR, 1.24; 95% CI, 1.08-1.42). Effect estimates for autism and depression were similar following a severe maternal infection (autism: HR, 1.81; 95% CI, 1.18-2.78; depression: HR, 1.24; 95% CI, 0.88-1.73) or urinary tract infection (autism: HR, 1.89; 95% CI, 1.23-2.90; depression: HR, 1.30; 95% CI, 1.04-1.61) and were robust to moderate unknown confounding. Within the directed acyclic graph framework of assumptions, the relationship between infection and depression was vulnerable to bias from loss to follow-up, but separate data from the Swedish Death Registry demonstrated increased risk of suicide among individuals exposed to pregnancy infection. No evidence was found for increased risk of bipolar disorder or psychosis among children exposed to infection in utero.</p>
<p>Conclusions and Relevance  These findings suggest that fetal exposure to a maternal infection while hospitalized increased the risk for autism and depression, but not bipolar or psychosis, during the child’s life. These results emphasize the importance of avoiding infections during pregnancy, which may impart subtle fetal brain injuries contributing to development of autism and depression.</p>
</li>
</ul>
</li>
<li>
<p><em>antibiotic-induced changes to the host metabolic environment inhibit drug efficacy and alter immune function</em><br />
jason h. yang et al. 2017<br />
<a href="http://doi.org/10.1016/j.chom.2017.10.020">doi.org/10.1016/j.chom.2017.10.020</a></p>
<ul>
<li>
<p>Antibiotic treatment depletes central metabolism intermediates in the peritoneum<br />
Antibiotic treatment elicits microbiome-independent changes in host metabolites<br />
Metabolites altered by antibiotic treatment during infection inhibit drug efficacy<br />
Antibiotic treatment impairs phagocytic killing by inhibiting respiratory activity</p>
<p>Bactericidal antibiotics alter microbial metabolism as part of their lethality and can damage mitochondria in mammalian cells. In addition, antibiotic susceptibility is sensitive to extracellular metabolites, but it remains unknown whether metabolites present at an infection site can affect either treatment efficacy or immune function. Here, we quantify local metabolic changes in the host microenvironment following antibiotic treatment for a peritoneal <em>Escherichia coli</em> infection. Antibiotic treatment elicits microbiome-independent changes in local metabolites, but not those distal to the infection site, by acting directly on host cells. The metabolites induced during treatment, such as AMP, reduce antibiotic efficacy and enhance phagocytic killing. Moreover, antibiotic treatment impairs immune function by inhibiting respiratory activity in immune cells. Collectively, these results highlight the immunomodulatory potential of antibiotics and reveal the local metabolic microenvironment to be an important determinant of infection resolution.</p>
</li>
</ul>
</li>
<li>
<p><em>recovery of gut microbiota of healthy adults following antibiotic exposure</em><br />
albert palleja et al. 2018<br />
<a href="http://doi.org/10.1038/s41564-018-0257-9">doi.org/10.1038/s41564-018-0257-9</a></p>
<ul>
<li>“Interestingly, Clostridium spp. were particularly likely to colonize after the antibiotic intervention, perhaps having survived as spores. This is relevant as Clostridium difficile infections are associated with the use of broad-spectrum antibiotics in acute care settings, which reduce host resistance to colonization and expansion of this pathogen.”</li>
<li><em>abstract</em> To minimize the impact of antibiotics, gut microorganisms harbour and exchange antibiotics resistance genes, collectively called their resistome. Using shotgun sequencing-based metagenomics, we analysed the partial eradication and subsequent regrowth of the gut microbiota in 12 healthy men over a 6-month period following a 4-day intervention with a cocktail of 3 last-resort antibiotics: meropenem, gentamicin and vancomycin. Initial changes included blooms of enterobacteria and other pathobionts, such as Enterococcus faecalis and Fusobacterium nucleatum, and the depletion of Bifidobacterium species and butyrate producers. The gut microbiota of the subjects recovered to near-baseline composition within 1.5 months, although 9 common species, which were present in all subjects before the treatment, remained undetectable in most of the subjects after 180 days. Species that harbour β-lactam resistance genes were positively selected for during and after the intervention. Harbouring glycopeptide or aminoglycoside resistance genes increased the odds of de novo colonization, however, the former also decreased the odds of survival. Compositional changes under antibiotic intervention in vivo matched results from in vitro susceptibility tests. Despite a mild yet long-lasting imprint following antibiotics exposure, the gut microbiota of healthy young adults are resilient to a short-term broad-spectrum antibiotics intervention and their antibiotics resistance gene carriage modulates their recovery processes.</li>
</ul>
</li>
<li>
<p><em>ciprofloxacin impairs mitochondrial dna replication initiation through inhibition of topoisomerase 2</em><br />
anu hangas et al. 2018<br />
<a href="http://doi.org/10.1093/nar/gky793">doi.org/10.1093/nar/gky793</a></p>
<ul>
<li>
<p>Fluoroquinolones are some of the most frequently used broad-spectrum antibacterial antibiotics, and usually prescribed for respiratory, urinary or ear infections. While they are generally well tolerated, some patients receiving fluoroquinolones develop severe health problems, among them tendon rupture, permanent nerve damage or depression. The reasons for these side effects are yet unclear.<br />
A study carried out at the University of Eastern Finland and published in <em>Nucleic Acids Research</em> investigated the effect of ciprofloxacin on mitochondria, the important cell organelles in our body that produce the energy for cellular function. Mitochondria possess their own small circular genome, which requires topoisomerase enzymes for its maintenance. Topoisomerases regulate the topology of DNA and untangle for instance knots and overwound stretches of a genome by cutting and reconnecting the DNA sequence. While fluoroquinolones are designed to inhibit the bacterial topoisomerase gyrase, which leads to the death of the bacterium, they also inhibit the topoisomerase 2 of our own cells.<br />
“We noticed that topoisomerase 2 is especially important in the replication of the mitochondrial genome, as it regulates the winding of this small DNA molecule by removing positive twists,” Academy Research Fellow Steffi Goffart from the University of Eastern Finland says.<br />
Ciprofloxacin stopped this normal maintenance and transcription of mitochondrial DNA by changing mtDNA topology, causing impaired mitochondrial energy production and blocking cellular growth and differentiation. This dramatic impact on mitochondrial DNA is the likely cause for most negative side effects experienced by patients, and also a reason to use fluoroquinolone antibiotics with great caution.</p>
</li>
<li>
<p><em>abstract</em> Maintenance of topological homeostasis is vital for gene expression and genome replication in all organisms. Similar to other circular genomes, also mitochondrial DNA (mtDNA) is known to exist in various different topological forms, although their functional significance remains unknown. We report here that both known type II topoisomerases Top2α and Top2β are present in mammalian mitochondria, with especially Top2β regulating the supercoiling state of mtDNA. Loss of Top2β or its inhibition by ciprofloxacin results in accumulation of positively supercoiled mtDNA, followed by cessation of mitochondrial transcription and replication initiation, causing depletion of mtDNA copy number. These mitochondrial effects block both cell proliferation and differentiation, possibly explaining some of the side effects associated with fluoroquinolone antibiotics. Our results show for the first time the importance of topology for maintenance of mtDNA homeostasis and provide novel insight into the mitochondrial effects of fluoroquinolones.</p>
</li>
</ul>
</li>
<li>
<p><em>characterization of shifts of koala (phascolarctos cinereus) intestinal microbial communities associated with antibiotic treatment</em><br />
katherine e. dahlhausen et al. 2018<br />
<a href="https://doi.org/10.7717/peerj.4452">doi.org/10.7717/peerj.4452</a></p>
<ul>
<li>Koalas (<em>Phascolarctos cinereus</em>) are arboreal marsupials native to Australia that eat a specialized diet of almost exclusively eucalyptus leaves. Microbes in koala intestines are known to break down otherwise toxic compounds, such as tannins, in eucalyptus leaves. Infections by <em>Chlamydia</em>, obligate intracellular bacterial pathogens, are highly prevalent in koala populations. If animals with <em>Chlamydia</em> infections are received by wildlife hospitals, a range of antibiotics can be used to treat them. However, previous studies suggested that koalas can suffer adverse side effects during antibiotic treatment. This study aimed to use 16S rRNA gene sequences derived from koala feces to characterize the intestinal microbiome of koalas throughout antibiotic treatment and identify specific taxa associated with koala health after treatment. Although differences in the alpha diversity were observed in the intestinal flora between treated and untreated koalas and between koalas treated with different antibiotics, these differences were not statistically significant. The alpha diversity of microbial communities from koalas that lived through antibiotic treatment versus those who did not was significantly greater, however. Beta diversity analysis largely confirmed the latter observation, revealing that the overall communities were different between koalas on antibiotics that died versus those that survived or never received antibiotics. Using both machine learning and OTU (operational taxonomic unit) co-occurrence network analyses, we found that OTUs that are very closely related to <em>Lonepinella koalarum</em>, a known tannin degrader found by culture-based methods to be present in koala intestines, was correlated with a koala’s health status. This is the first study to characterize the time course of effects of antibiotics on koala intestinal microbiomes. Our results suggest it may be useful to pursue alternative treatments for <em>Chlamydia</em> infections without the use of antibiotics or the development of <em>Chlamydia</em>-specific antimicrobial compounds that do not broadly affect microbial communities.</li>
</ul>
</li>
<li>
<p><em>the kinases hipa and hipa7 phosphorylate different substrate pools in escherichia coli to promote multidrug tolerance</em><br />
maja semanjski et al. 2018<br />
<a href="http://doi.org/10.1126/scisignal.aat5750">doi.org/10.1126/scisignal.aat5750</a></p>
<ul>
<li>The bacterial serine-threonine protein kinase HipA promotes multidrug tolerance by phosphorylating the glutamate-tRNA ligase (GltX), leading to a halt in translation, inhibition of growth, and induction of a physiologically dormant state (persistence). The HipA variant HipA7 substantially increases persistence despite being less efficient at inhibiting cell growth. We postulated that this phenotypic difference was caused by differences in the substrates targeted by both kinases. We overproduced HipA and HipA7 in Escherichia coli and identified their endogenous substrates by SILAC-based quantitative phosphoproteomics. We confirmed that GltX was the main substrate of both kinase variants and likely the primary determinant of persistence. When HipA and HipA7 were moderately overproduced from plasmids, HipA7 targeted only GltX, but HipA phosphorylated several additional substrates involved in translation, transcription, and replication, such as ribosomal protein L11 (RplK) and the negative modulator of replication initiation, SeqA. HipA7 showed reduced kinase activity compared to HipA and targeted a substrate pool similar to that of HipA only when produced from a high–copy number plasmid. The kinase variants also differed in autophosphorylation, which was substantially reduced for HipA7. When produced endogenously from the chromosome, HipA showed no activity because of inhibition by the antitoxin HipB, whereas HipA7 phosphorylated GltX and phage shock protein PspA. Initial testing did not reveal a connection between HipA-induced phosphorylation of RplK and persistence or growth inhibition, suggesting that other HipA-specific substrates were likely responsible for growth inhibition. Our results contribute to the understanding of HipA7 action and present a resource for elucidating HipA-related persistence.</li>
</ul>
</li>
<li>
<p>antibiotic-associated hemorrhagic colitis (AAHC)<br />
one mechanism for antibiotics to cause damage, through imbalance of toxin-producing part of microbiome</p>
<ul>
<li>
<p><em>biosynthesis of the enterotoxic pyrrolobenzodiazepine natural product tilivalline</em><br />
elisabeth dornisch et al. 2017<br />
<a href="http://doi.org/10.1002/anie.201707737">doi.org/10.1002/anie.201707737</a></p>
<ul>
<li>The nonribosomal enterotoxin tilivalline was the first naturally occurring pyrrolobenzodiazepine to be linked to disease in the human intestine. Since the producing organism <em>Klebsiella oxytoca</em> is part of the intestinal microbiota and the pyrrolobenzodiazepine causes the pathogenesis of colitis it is important to understand the biosynthesis and regulation of tilivalline activity. Here we report the biosynthesis of tilivalline and show that this nonribosomal peptide assembly pathway initially generates tilimycin, a simple pyrrolobenzodiazepine with cytotoxic properties. Tilivalline results from the non-enzymatic spontaneous reaction of tilimycin with biogenetically generated indole. Through a chemical total synthesis of tilimycin we could corroborate the predictions made about the biosynthesis. Production of two cytotoxic pyrrolobenzodiazepines with distinct functionalities by human gut resident <em>Klebsiella oxytoca</em> has important implications for intestinal disease.</li>
<li>The enteric bacterium <em>Klebsiella oxytoca</em> presents a striking example. In some patients treated with penicillin antibiotics, rapid proliferation of this organism results in antibiotic-associated hemorrhagic colitis (AAHC).<a href="http://doi.org/10.1016/j.cell.2016.11.003">7</a> We showed in previous work that the indol-3-yl-substituted pyrrolobenzodiazepine (PBD) tilivalline (1) produced by <em>K. oxytoca</em> was required for the pathogenesis of colitis in an animal model of AAHC.<a href="http://doi.org/10.1016/j.cell.2015.11.004">8</a> Naturally occurring PBDs form a family of antitumor antibiotics produced by Gram-positive soil bacteria.<a href="https://www.nature.com/articles/s41586-018-0634-9#ref-CR1">9</a> Synthesis of a natural PBD by a Gram-negative resident of the gut is thus surprising. We identified the toxin biosynthetic gene cluster by genetic mutation and localized the region to a unique pathogenicity island on the genome of cytotoxic <em>K. oxytoca</em> strains.<a href="http://doi.org/10.1016/j.cell.2015.11.004">8</a></li>
</ul>
</li>
</ul>
</li>
<li>
<p><em>oral administration of antibiotics increased the potential mobility of bacterial resistance genes in the gut of the fish piaractus mesopotamicus</em><br />
johan s. sáenz et al. 2019<br />
<a href="http://doi.org/10.1186/s40168-019-0632-7">doi.org/10.1186/s40168-019-0632-7</a></p>
<ul>
<li>
<p>they studied Piaractus mesopotamicus, a South American species known as pacu that is often raised in aquaculture. The fish received the antibiotic florfenicol in their food for 34 days. During this time and after the application period, the researchers took samples from the digestive tract of the fish and looked for relevant genetic changes in the gut bacteria.</p>
<p>Resistance genes hop around the genome</p>
<p>&quot;As expected, administration of the antibiotic induced an increase in the genes responsible for resistance to that antibiotic,&quot; explains COMI doctoral student Johan Sebastian Sáenz Medina, lead author of the paper. &quot;One example are genes for pump proteins, which simply remove the active substance from the bacteria again. However, we were particularly surprised by the different mechanisms that we could detect by which antibiotic resistance genes are spread amongst gut bacteria of the fish,&quot; Sáenz Medina explains. &quot;This suggests that the bacteria also exchange resistance through viruses, known as phages, and transposons.&quot;</p>
<p>Further metagenomic studies confirmed that these mobile genetic elements induce a fast distribution of resistance genes among genomes of different organisms. So far it has been postulated that only plasmids (in essence, easily transferable mini-chromosomes) are mainly responsible for the exchange of resistance genes.</p>
<p>&quot;The finding that resistance is also extensively transferred between bacteria without the involvement of plasmids is really quite surprising,&quot; says Michael Schloter.</p>
</li>
<li>
<p><em>abstract</em> Aquaculture is on the rise worldwide, and the use of antibiotics is fostering higher production intensity. However, recent findings suggest that the use of antibiotics comes at the price of increased antibiotic resistance. Yet, the effect of the oral administration of antibiotics on the mobility of microbial resistance genes in the fish gut is not well understood. In the present study, Piaractus mesopotamicus was used as a model to evaluate the effect of the antimicrobial florfenicol on the diversity of the gut microbiome as well as antibiotic resistance genes (ARGs) and mobile genetic elements (MGEs) using a metagenomic approach.</p>
</li>
</ul>
</li>
<li>
<p><em>duration and life-stage of antibiotic use and risk of cardiovascular events in women</em><br />
yoriko heianza et al. 2019<br />
<a href="http://doi.org/10.1093/eurheartj/ehz231">doi.org/10.1093/eurheartj/ehz231</a></p>
<ul>
<li>
<p>Women who take antibiotics over a long period of time are at increased risk of heart attack or stroke, according to research carried out in nearly 36,500 women.</p>
<p>The study, published in the European Heart Journal <a href="http://doi.org/10.1073/pnas.1218525110">1</a> today (Thursday), found that women aged 60 or older who took antibiotics for two months or more had the greatest risk of cardiovascular disease, but long duration of antibiotic use was also associated with an increased risk if taken during middle age (aged 40-59). The researchers could find no increased risk from antibiotic use by younger adults aged between 20-39.</p>
<p>Professor Lu Qi, director of the Tulane University Obesity Research Centre, Tulane University, New Orleans, and adjunct professor of nutrition at Harvard T.C. Chan School of Public Health, Boston, USA, who led the research, says that a possible reason why antibiotic use is linked to an increased risk of cardiovascular disease is because antibiotics alter the balance of the micro-environment in the gut, destroying &quot;good&quot; probiotic bacteria and increasing the prevalence of viruses, bacteria or other micro-organisms that can cause disease.</p>
<p>&quot;Antibiotic use is the most critical factor in altering the balance of microorganisms in the gut. Previous studies have shown a link between alterations in the microbiotic environment of the gut and inflammation and narrowing of the blood vessels, stroke and heart disease,&quot; he said.</p>
<p>The researchers studied 36,429 women who took part in the Nurses' Health Study, which has been running in the USA since 1976. The current study looked at data from 2004 to June 2012. In 2004 the women were aged 60 or older, and they were asked about their use of antibiotics when they were young (20-39), middle-aged (40-59) or older (60 and older). The researchers categorised them into four groups: those who had never taken antibiotics, those who had taken them for time periods of less than 15 days, 15 days to two months, or for two months or longer.</p>
<p>During an average follow-up period of nearly eight years, during which time the women continued to complete questionnaires every two years, 1056 participants developed cardiovascular disease.</p>
<p>After adjustments to take account of factors that could affect their results, such as age, race, sex, diet and lifestyle, reasons for antibiotic use, overweight or obesity, other diseases and medication use, the researchers found that women who used antibiotics for periods of two months or longer in late adulthood were 32% more likely to develop cardiovascular disease than women who did not use antibiotics. Women who took antibiotics for longer than two months in middle age had a 28% increased risk compared to women who did not.</p>
<p>These findings mean that among women who take antibiotics for two months or more in late adulthood, six women per 1,000 would develop a cardiovascular disease, compared to three per 1,000 among women who had not taken antibiotics.</p>
<p>The first author of the study is Dr Yoriko Heianza. a research fellow at Tulane University. She said: &quot;By investigating the duration of antibiotic use in various stages of adulthood we have found an association between long-term use in middle age and later life and an increased risk of stroke and heart disease during the following eight years. As these women grew older they were more likely to need more antibiotics, and sometimes for longer periods of time, which suggests a cumulative effect may be the reason for the stronger link in older age between antibiotic use and cardiovascular disease.&quot;</p>
<p>The most common reasons for antibiotic use were respiratory infections, urinary tract infections and dental problems.</p>
<p>The study is the largest prospective study to investigate the link between antibiotic use and risk of heart disease and stroke, and this is one of the strengths of the study, as well as the long follow-up and comprehensive information on factors that could affect the results such as life style, diet, age, other diseases and medication use.</p>
<p>Limitations include the fact that the participants reported their use of antibiotics and so this could be mis-remembered. However, as they were all health professionals, they were able to provide more accurate information on medication use than the general population. The researchers did not have information on the different classes of antibiotics used, but believe that the most common type of prescription tends to depend on the infections it is treating, and information on these was included in their analysis. As the study only looked at middle-aged and elderly women, the results cannot necessarily be extrapolated to younger ages and to men.</p>
<p>Prof Qi concluded: &quot;This is an observational study and so it cannot show that antibiotics cause heart disease and stroke, only that there is a link between them. It's possible that women who reported more antibiotic use might be sicker in other ways that we were unable to measure, or there may be other factors that could affect the results that we have not been able take account of.</p>
<p>&quot;Our study suggests that antibiotics should be used only when they are absolutely needed. Considering the potentially cumulative adverse effects, the shorter time of antibiotic use the better.&quot;</p>
</li>
<li>
<p><em>abstract</em> Growing data suggest that antibiotic exposure is associated with a long-lasting alteration in gut microbiota, and may be related to subsequent cardiovascular disease (CVD). We investigated associations of life-stage and duration of antibiotic exposure during adulthood with subsequent CVD events.<br />
Methods and results<br />
This study included 36 429 women initially free of CVD and cancer from the Nurses’ Health Study. We estimated hazard ratios (HRs) for CVD (a composite endpoint of coronary heart disease or stroke) according to duration of antibiotic use in young (age 20–39), middle (age 40–59), and late (age 60 and older) adulthood. During an average of 7.6 years of follow-up, 1056 participants developed CVD. Women with long-term use of antibiotics (for ≥2 months) in late adulthood had a significantly increased risk of CVD (HR 1.32, 95% confidence interval 1.03–1.70) after adjustment for covariates (such as demographic factors, diet and lifestyle, reasons for antibiotic use, overweight or obesity, disease status, and other medication use), as compared to women who did not use antibiotics in this life-stage. Longer duration of antibiotic use in middle adulthood was also related to higher risk of CVD (P trend = 0.003) after controlling for these covariates. There was no significant relationship between the use in young adulthood and the risk of CVD.<br />
Conclusion<br />
In this study which examined the antibiotic use in different life-stages, longer duration of exposure to antibiotics in the middle and older adulthood was related to an increased risk of future CVD events among elderly women at usual risk.</p>
</li>
</ul>
</li>
<li>
<p><em>antibiotic exposure and risk of parkinson's disease in finland: a nationwide case‐control study</em><br />
tuomas h. mertsalmi et al. 2019<br />
<a href="http://doi.org/10.1002/mds.27924">doi.org/10.1002/mds.27924</a></p>
<ul>
<li>
<p>Higher exposure to commonly used oral antibiotics is linked to an increased risk of Parkinson’s disease according to a recently published study by researchers form the Helsinki University Hospital, Finland.<br />
The strongest associations were found for broad spectrum antibiotics and those that act against against anaerobic bacteria and fungi. The timing of antibiotic exposure also seemed to matter.<br />
The study suggests that excessive use of certain antibiotics can predispose to Parkinson’s disease with a delay of up to 10 to 15 years. This connection may be explained by their disruptive effects on the gut microbial ecosystem.<br />
“The link between antibiotic exposure and Parkinson’s disease fits the current view that in a significant proportion of patients the pathology of Parkinson’s may originate in the gut, possibly related to microbial changes, years before the onset of typical Parkinson motor symptoms such as slowness, muscle stiffness and shaking of the extremities. It was known that the bacterial composition of the intestine in Parkinson’s patients is abnormal, but the cause is unclear. Our results suggest that some commonly used antibiotics, which are known to strongly influence the gut microbiota, could be a predisposing factor,” says research team leader, neurologist Filip Scheperjans MD, PhD from the Department of Neurology of Helsinki University Hospital.<br />
In the gut, pathological changes typical of Parkinson’s disease have been observed up to 20 years before diagnosis. Constipation, irritable bowel syndrome and inflammatory bowel disease have been associated with a higher risk of developing Parkinson’s disease. Exposure to antibiotics has been shown to cause changes in the gut microbiome and their use is associated with an increased risk of several diseases, such as psychiatric disorders and Crohn’s disease. However, these diseases or increased susceptibility to infection do not explain the now observed relationship between antibiotics and Parkinson’s.<br />
“The discovery may also have implications for antibiotic prescribing practices in the future. In addition to the problem of antibiotic resistance, antimicrobial prescribing should also take into account their potentially long-lasting effects on the gut microbiome and the development of certain diseases,” says Scheperjans.<br />
The possible association of antibiotic exposure with Parkinson’s disease was investigated in a case-control study using data extracted from national registries. The study compared antibiotic exposure during the years 1998-2014 in 13,976 Parkinson’s disease patients and compared it with 40,697 non-affected persons matched for the age, sex and place of residence.<br />
Antibiotic exposure was examined over three different time periods: 1-5, 5-10, and 10-15 years prior to the index date, based on oral antibiotic purchase data. Exposure was classified based on number of purchased courses. Exposure was also examined by classifying antibiotics according to their chemical structure, antimicrobial spectrum, and mechanism of action.</p>
</li>
<li>
<p><em>abstract</em> Gut microbiota alterations have been found in prodromal and established Parkinson's disease (PD). Antibiotic exposure can have long‐term effects on the composition of human intestinal microbiota, but a potential connection between antibiotic exposure and risk of PD has not been studied previously.</p>
<p>Objective</p>
<p>To evaluate the impact of antibiotic exposure on the risk of PD in a nationwide, register‐based, case‐control study.</p>
<p>Methods</p>
<p>We identified all patients who were diagnosed with PD in Finland during the years 1998 to 2014. Information was obtained on individual purchases of orally administered antibiotics during the years 1993 to 2014. We assessed the association between prior antibiotic exposure and PD using conditional logistic regression.</p>
<p>Results</p>
<p>The study population consisted of 13,976 PD cases and 40,697 controls. The strongest connection with PD risk was found for oral exposure to macrolides and lincosamides (adjusted odds ratio up to 1.416; 95% confidence interval, 1.053–1.904). After correction for multiple comparisons, exposure to antianaerobics and tetracyclines 10 to 15 years before the index date, sulfonamides and trimethoprim 1 to 5 years before the index date, and antifungal medications 1 to 5 years before the index date were positively associated with PD risk. In post hoc analyses, further positive associations were found for broad‐spectrum antibiotics.</p>
<p>Conclusions</p>
<p>Exposure to certain types of oral antibiotics seems to be associated with an elevated risk of PD with a delay that is consistent with the proposed duration of a prodromal period. The pattern of associations supports the hypothesis that effects on gut microbiota could link antibiotics to PD, but further studies are needed to confirm this.</p>
</li>
</ul>
</li>
</ul>
</li>
</ul>
<h2>07994</h2>
<ul>
<li>induced by pesticides, toxins
<ul>
<li>
<p><em>bisphenol-a alters microbiota metabolites derived from aromatic amino acids and worsens disease activity during colitis</em><br />
jennifer aa deluca et al. 2018<br />
<a href="http://doi.org/10.1177/1535370218782139">doi.org/10.1177/1535370218782139</a></p>
<ul>
<li>Inflammatory bowel disease is a complex collection of disorders. Microbial dysbiosis as well as exposure to toxins including xenoestrogens are thought to be risk factors for inflammatory bowel disease development and relapse. Bisphenol-A has been shown to exert estrogenic activity in the colon and alter intestinal function, but the role that xenoestrogens, such as bisphenol-A , play in colonic inflammation has been previously described but with conflicting results. We investigated the ability of bisphenol-A to exacerbate colonic inflammation and alter microbiota metabolites derived from aromatic amino acids in an acute dextran sulfate sodium-induced colitis model. Female C57BL/6 mice were ovariectomized and exposed to bisphenol-A daily for 15 days. Disease activity measures include body weight, fecal consistency, and rectal bleeding. Colons were scored for inflammation, injury, and nodularity. Alterations in the levels of microbiota metabolites derived from aromatic amino acids known to reflect phenotypic changes in the gut microbiome were analyzed. Bisphenol-A exposure increased mortality and worsened disease activity as well as inflammation and nodularity scores in the middle colon region following dextran sulfate sodium exposure. Unique patterns of metabolites were associated with bisphenol-A consumption. Regardless of dextran sulfate sodium treatment, bisphenol-A reduced levels of tryptophan and several metabolites associated with decreased inflammation in the colon. This is the first study to show that bisphenol-A treatment alone can reduce microbiota metabolites derived from aromatic amino acids in the colon which may be associated with increased colonic inflammation and inflammatory bowel disease.</li>
</ul>
</li>
<li>
<p><em>glyphosate perturbs the gut microbiota of honey bees</em><br />
erick v. s. motta et al. 2018<br />
<a href="https://doi.org/10.1073/pnas.1803880115">doi.org/10.1073/pnas.1803880115</a></p>
<ul>
<li>Glyphosate, the primary herbicide used globally for weed control, targets the 5-enolpyruvylshikimate-3-phosphate synthase (EPSPS) enzyme in the shikimate pathway found in plants and some microorganisms. Thus, glyphosate may affect bacterial symbionts of animals living near agricultural sites, including pollinators such as bees. The honey bee gut microbiota is dominated by eight bacterial species that promote weight gain and reduce pathogen susceptibility. The gene encoding EPSPS is present in almost all sequenced genomes of bee gut bacteria, indicating that they are potentially susceptible to glyphosate. We demonstrated that the relative and absolute abundances of dominant gut microbiota species are decreased in bees exposed to glyphosate at concentrations documented in the environment. Glyphosate exposure of young workers increased mortality of bees subsequently exposed to the opportunistic pathogen <em>Serratia marcescens</em>. Members of the bee gut microbiota varied in susceptibility to glyphosate, largely corresponding to whether they possessed an EPSPS of class I (sensitive to glyphosate) or class II (insensitive to glyphosate). This basis for differences in sensitivity was confirmed using in vitro experiments in which the EPSPS gene from bee gut bacteria was cloned into <em>Escherichia coli</em>. All strains of the core bee gut species, <em>Snodgrassella alvi</em>, encode a sensitive class I EPSPS, and reduction in <em>S. alvi</em> levels was a consistent experimental result. However, some <em>S. alvi</em> strains appear to possess an alternative mechanism of glyphosate resistance. Thus, exposure of bees to glyphosate can perturb their beneficial gut microbiota, potentially affecting bee health and their effectiveness as pollinators.</li>
</ul>
</li>
<li>
<p><em>glyphosate’s suppression of cytochrome p450 enzymes and amino acid biosynthesis by the gut microbiome: pathways to modern diseases</em><br />
anthony samsel and stephanie seneff 2013<br />
<a href="http://doi.org/10.3390/e15041416">doi.org/10.3390/e15041416</a></p>
<ul>
<li>Glyphosate, the active ingredient in Roundup®, is the most popular herbicide used worldwide. The industry asserts it is minimally toxic to humans, but here we argue otherwise. Residues are found in the main foods of the Western diet, comprised primarily of sugar, corn, soy and wheat. Glyphosate's inhibition of cytochrome P450 (CYP) enzymes is an overlooked component of its toxicity to mammals. CYP enzymes play crucial roles in biology, one of which is to detoxify xenobiotics. Thus, glyphosate enhances the damaging effects of other food borne chemical residues and environmental toxins. Negative impact on the body is insidious and manifests slowly over time as inflammation damages cellular systems throughout the body. Here, we show how interference with CYP enzymes acts synergistically with disruption of the biosynthesis of aromatic amino acids by gut bacteria, as well as impairment in serum sulfate transport. Consequences are most of the diseases and conditions associated with a Western diet, which include gastrointestinal disorders, obesity, diabetes, heart disease, depression, autism, infertility, cancer and Alzheimer’s disease. We explain the documented effects of glyphosate and its ability to induce disease, and we show that glyphosate is the “textbook example” of exogenous semiotic entropy: the disruption of homeostasis by environmental toxins.</li>
</ul>
</li>
</ul>
</li>
</ul>
<h2>08094</h2>
<ul>
<li>induced by other medicines
<ul>
<li>
<p><em>the active component of aspirin, salicylic acid, promotes staphylococcus aureus biofilm formation in a pia-dependent manner</em><br />
cristian dotto et al. 2017<br />
<a href="https://doi.org/10.3389/fmicb.2017.00004">doi.org/10.3389/fmicb.2017.00004</a></p>
<ul>
<li>Aspirin has provided clear benefits to human health. But salicylic acid (SAL) -the main aspirin biometabolite- exerts several effects on eukaryote and prokaryote cells. SAL can affect, for instance, the expression of <em>Staphylococcus aureus</em> virulence factors. SAL can also form complexes with iron cations and it has been shown that different iron chelating molecules diminished the formation of <em>S. aureus</em> biofilm. The aim of this study was to elucidate whether the iron content limitation caused by SAL can modify the <em>S. aureus</em> metabolism and/or metabolic regulators thus changing the expression of the main polysaccharides involved in biofilm formation. The exposure of biofilm to 2 mM SAL induced a 27% reduction in the intracellular free Fe2+ concentration compared with the controls. In addition, SAL depleted 23% of the available free Fe2+ cation in culture media. These moderate iron-limited conditions promoted an intensification of biofilms formed by strain Newman and by <em>S. aureus</em> clinical isolates related to the USA300 and USA100 clones. The slight decrease in iron bioavailability generated by SAL was enough to induce the increase of PIA expression in biofilms formed by methicillin-resistant as well as methicillin-sensitive <em>S. aureus</em> strains. <em>S. aureus</em> did not produce capsular polysaccharide (CP) when it was forming biofilms under any of the experimental conditions tested. Furthermore, SAL diminished aconitase activity and stimulated the lactic fermentation pathway in bacteria forming biofilms. The polysaccharide composition of <em>S. aureus</em> biofilms was examined and FTIR spectroscopic analysis revealed a clear impact of SAL in a <em>codY</em>-dependent manner. Moreover, SAL negatively affected <em>codY</em> transcription in mature biofilms thus relieving the CodY repression of the <em>ica</em> operon. Treatment of mice with SAL induced a significant increase of <em>S aureus</em> colonization. It is suggested that the elevated PIA expression induced by SAL might be responsible for the high nasal colonization observed in mice. SAL-induced biofilms may contribute to <em>S. aureus</em> infection persistence in vegetarian individuals as well as in patients that frequently consume aspirin.</li>
</ul>
</li>
<li>
<p><em>nonsteroidal anti-inflammatory drugs alter the microbiota and exacerbate clostridium difficile colitis while dysregulating the inflammatory response</em><br />
damian maseda et al. 2019<br />
<a href="http://doi.org/10.1128/mbio.02282-18">doi.org/10.1128/mbio.02282-18</a></p>
<ul>
<li>
<p>The researchers followed two groups of antibiotic-treated mice for one week after infection with C. difficile. One group had been treated with an NSAID called indomethacin prior to infection, and the other hadn't. Only about 20 percent of the mice treated with the NSAID survived to the end of the observation period, compared to about 80 percent of the mice that hadn't been exposed to the NSAID.</p>
<p>Aronoff and his collaborators determined that even brief exposure to the NSAID prior to C. difficile inoculation increased the severity of infections and shortened survival. Further cellular and genetic analyses revealed that the NSAID exposure altered the gut microbiota and depleted the production of prostaglandins, hormone-like substances known to play an important role in gastrointestinal health. Those observations align with previous studies reporting that NSAIDs can cause or exacerbate an inflammatory disease called colitis, also by inhibiting the body's production of prostaglandins.</p>
<p>In the new study, the researchers conclude that NSAID-driven changes worsened C. difficile infections by impairing epithelial cells -- the main defense system in the intestine against infectious taxa -- and by disturbing the normal immune response. They studied at the impact of only one NSAID, indomethacin, but Aronoff says he thinks the findings might extend to other common NSAIDs, including ibuprofen and aspirin, since they all have roughly the same biological mechanism.</p>
<p>&quot;Ultimately, these new results might guide how we treat people with C. diff, particularly with pain management,&quot; says Aronoff. &quot;Right now, it's too early for our results to guide clinical care, but they should be a stimulus for future studies.&quot;</p>
</li>
<li>
<p><em>abstract</em> Clostridium difficile infection (CDI) is a major public health threat worldwide. The use of nonsteroidal anti-inflammatory drugs (NSAIDs) is associated with enhanced susceptibility to and severity of CDI; however, the mechanisms driving this phenomenon have not been elucidated. NSAIDs alter prostaglandin (PG) metabolism by inhibiting cyclooxygenase (COX) enzymes. Here, we found that treatment with the NSAID indomethacin prior to infection altered the microbiota and dramatically increased mortality and the intestinal pathology associated with CDI in mice. We demonstrated that in C. difficile-infected animals, indomethacin treatment led to PG deregulation, an altered proinflammatory transcriptional and protein profile, and perturbed epithelial cell junctions. These effects were paralleled by increased recruitment of intestinal neutrophils and CD4+ cells and also by a perturbation of the gut microbiota. Together, these data implicate NSAIDs in the disruption of protective COX-mediated PG production during CDI, resulting in altered epithelial integrity and associated immune responses.</p>
</li>
</ul>
</li>
<li>
<p><em>dietary zinc alters the microbiota and decreases resistance to clostridium difficile infection</em><br />
joseph p zackular et al. 2016<br />
<a href="http://doi.org/10.1038/nm.4174">doi.org/10.1038/nm.4174</a></p>
<ul>
<li>Clostridium difficile is the most commonly reported nosocomial pathogen in the United States and is an urgent public health concern worldwide1. Over the past decade, incidence, severity and costs associated with C. difficile infection (CDI) have increased dramatically2. CDI is most commonly initiated by antibiotic-mediated disruption of the gut microbiota; however, non-antibiotic-associated CDI cases are well documented and on the rise3, 4. This suggests that unexplored environmental, nutrient and host factors probably influence CDI. Here we show that excess dietary zinc (Zn) substantially alters the gut microbiota and, in turn, reduces the minimum amount of antibiotics needed to confer susceptibility to CDI. In mice colonized with C. difficile, excess dietary Zn severely exacerbated C. difficile–associated disease by increasing toxin activity and altering the host immune response. In addition, we show that the Zn-binding S100 protein calprotectin has antimicrobial effects against C. difficile and is an essential component of the innate immune response to CDI. Taken together, these data suggest that nutrient Zn levels have a key role in determining susceptibility to CDI and severity of disease, and that calprotectin-mediated metal limitation is an important factor in the host immune response to C. difficile.</li>
</ul>
</li>
<li>
<p><em>dietary zinc and the control of streptococcus pneumoniae infection</em><br />
bart a. eijkelkamp et al. 2019<br />
<a href="http://doi.org/10.1371/journal.ppat.1007957">doi.org/10.1371/journal.ppat.1007957</a></p>
<ul>
<li>
<p>&quot;Dietary zinc is associated with immune function and resistance to bacterial infection, but how it provides protection has remained elusive,&quot; Dr Eijkelkamp said.</p>
<p>&quot;Our work shows that zinc is mobilised to sites of infection where it stresses the invading bacteria and helps specific immune cells kill Streptococcus pneumoniae.&quot;</p>
</li>
<li>
<p>This work also translated its findings by showing that specific human immune cells could use zinc to enhance their killing of invading Streptococcus pneumoniae.</p>
</li>
<li>
<p><em>abstract</em> Human zinc deficiency increases susceptibility to bacterial infection. Although zinc supplementation therapies can reduce the impact of disease, the molecular basis for protection remains unclear. Streptococcus pneumoniae is a major cause of bacterial pneumonia, which is prevalent in regions of zinc deficiency. We report that dietary zinc levels dictate the outcome of S. pneumoniae infection in a murine model. Dietary zinc restriction impacts murine tissue zinc levels with distribution post-infection altered, and S. pneumoniae virulence and infection enhanced. Although the activation and infiltration of murine phagocytic cells was not affected by zinc restriction, their efficacy of bacterial control was compromised. S. pneumoniae was shown to be highly sensitive to zinc intoxication, with this process impaired in zinc restricted mice and isolated phagocytic cells. Collectively, these data show how dietary zinc deficiency increases sensitivity to S. pneumoniae infection while revealing a role for zinc as a component of host antimicrobial defences.</p>
</li>
</ul>
</li>
<li>
<p><em>zno nanoparticles affect intestinal function in an in vitro model</em><br />
fabiola moreno-olivas et al. 2018<br />
<a href="http://doi.org/10.1039/c7fo02038d">doi.org/10.1039/c7fo02038d</a></p>
<ul>
<li>Zinc oxide nanoparticles (ZnO NP) may be present in food packaging, which would put consumers at risk of NP ingestion. There is little information on the amount of ZnO NP that are present in food packaging and the effects of ZnO exposure on intestinal function. To estimate physiologically relevant ZnO exposures, foods that are naturally low in zinc (Zn), but are commonly packaged with ZnO NP, such as tuna, corn, and asparagus, were analyzed with inductively coupled plasma mass spectrometry (ICP-MS). It was found that the Zn present in a serving of these foods is approximately one hundred times higher than the recommended dietary allowance. An <em>in vitro</em> model of the small intestine composed of Caco-2 and HT29-MTX cells was used to investigate the effects of ZnO NP exposure. Cells were exposed to physiologically realistic doses of pristine NP in culture medium and to NP subjected to an <em>in vitro</em> digestion to better reflect the transformation that the NP may undergo once they enter the human GI tract. Uptake and/or transport of iron (Fe), Zn, glucose, and fatty acids were assessed and intestinal alkaline phosphatase (IAP) levels were measured before and after NP exposure. The findings show that there is a 75% decrease in Fe transport and a 30% decrease in glucose transport following ZnO NP exposure. These decreases were consistent with gene expression changes for their transporters. There is also evidence that the ZnO NP affect the microvilli of the intestinal cells, therefore reducing the amount of surface area available to absorb nutrients. These results suggest that the ingestion of physiologically relevant doses of ZnO NP can alter intestinal function in an <em>in vitro</em> model of the human small intestine.</li>
</ul>
</li>
<li>
<p><em>measuring artificial sweeteners toxicity using a bioluminescent bacterial panel</em><br />
dorin harpaz et al. 2018<br />
<a href="http://doi.org/10.3390/molecules23102454">doi.org/10.3390/molecules23102454</a></p>
<ul>
<li><em>abstract</em> Artificial sweeteners have become increasingly controversial due to their questionable influence on consumers’ health. They are introduced in most foods and many consume this added ingredient without their knowledge. Currently, there is still no consensus regarding the health consequences of artificial sweeteners intake as they have not been fully investigated. Consumption of artificial sweeteners has been linked with adverse effects such as cancer, weight gain, metabolic disorders, type-2 diabetes and alteration of gut microbiota activity. Moreover, artificial sweeteners have been identified as emerging environmental pollutants, and can be found in receiving waters, i.e., surface waters, groundwater aquifers and drinking waters. In this study, the relative toxicity of six FDA-approved artificial sweeteners (aspartame, sucralose, saccharine, neotame, advantame and acesulfame potassium-k (ace-k)) and that of ten sport supplements containing these artificial sweeteners, were tested using genetically modified bioluminescent bacteria from <em>E. coli</em>. The bioluminescent bacteria, which luminesce when they detect toxicants, act as a sensing model representative of the complex microbial system. Both induced luminescent signals and bacterial growth were measured. Toxic effects were found when the bacteria were exposed to certain concentrations of the artificial sweeteners. In the bioluminescence activity assay, two toxicity response patterns were observed, namely, the induction and inhibition of the bioluminescent signal. An inhibition response pattern may be observed in the response of sucralose in all the tested strains: TV1061 (MLIC = 1 mg/mL), DPD2544 (MLIC = 50 mg/mL) and DPD2794 (MLIC = 100 mg/mL). It is also observed in neotame in the DPD2544 (MLIC = 2 mg/mL) strain. On the other hand, the induction response pattern may be observed in its response in saccharin in TV1061 (MLIndC = 5 mg/mL) and DPD2794 (MLIndC = 5 mg/mL) strains, aspartame in DPD2794 (MLIndC = 4 mg/mL) strain, and ace-k in DPD2794 (MLIndC = 10 mg/mL) strain. The results of this study may help in understanding the relative toxicity of artificial sweeteners on <em>E. coli</em>, a sensing model representative of the gut bacteria. Furthermore, the tested bioluminescent bacterial panel can potentially be used for detecting artificial sweeteners in the environment, using a specific mode-of-action pattern</li>
</ul>
</li>
</ul>
</li>
</ul>
<h2>08194</h2>
<ul>
<li>stress, mechanical and psychological
<ul>
<li>
<p><em>mind and gut: associations between mood and gastrointestinal distress in children exposed to adversity</em><br />
bridget l. callaghan et al. 2019<br />
<a href="http://doi.org/10.1017/s0954579419000087">doi.org/10.1017/s0954579419000087</a></p>
<ul>
<li>
<p>&quot;One common reason children show up at doctors' offices is intestinal complaints,&quot; said Nim Tottenham, a professor of psychology at Columbia and senior author on the study. &quot;Our findings indicate that gastrointestinal symptoms in young children could be a red flag to primary care physicians for future emotional health problems.&quot;</p>
<p>Scientists have long noted the strong connection between the gut and brain. Previous research has demonstrated that a history of trauma or abuse has been reported in up to half of adults with irritable bowel syndrome (IBS), at a prevalence twice that of patients without IBS.</p>
<p>&quot;The role of trauma in increasing vulnerability to both gastrointestinal and mental health symptoms is well established in adults but rarely studied in childhood,&quot; said study lead author Bridget Callaghan, a post-doctoral research fellow in Columbia's psychology department. In addition, she said, animal studies have demonstrated that adversity-induced changes in the gut microbiome -- the community of bacteria in the body that regulates everything from digestion to immune system function-influence neurological development, but no human studies have done so.</p>
<p>&quot;Our study is among the first to link disruption of a child's gastrointestinal microbiome triggered by early-life adversity with brain activity in regions associated with emotional health,&quot; Callaghan said.</p>
<p>The researchers focused on development in children who experienced extreme psychosocial deprivation due to institutional care before international adoption. Separation of a child from a parent is known to be a powerful predictor of mental health issues in humans. That experience, when modeled in rodents, induces fear and anxiety, hinders neurodevelopment and alters microbial communities across the lifespan.</p>
<p>The researchers drew upon data from 115 children adopted from orphanages or foster care on or before approximately they were 2 years old, and from 229 children raised by a biological caregiver. The children with past caregiving disruptions showed higher levels of symptoms that included stomach aches, constipation, vomiting and nausea.</p>
<p>From that sample of adoptees, the researchers then selected eight participants, ages 7 to 13, from the adversity exposed group and another eight who'd been in the group raised by their biological parents. Tottenham and Callaghan collected behavioral information, stool samples and brain images from all the children. They used gene sequencing to identify the microbes present in the stool samples and examined the abundance and diversity of bacteria in each participant's fecal matter.</p>
<p>The children with a history of early caregiving disruptions had distinctly different gut microbiomes from those raised with biological caregivers from birth. Brain scans of all the children also showed that brain activity patterns were correlated with certain bacteria. For example, the children raised by parents had increased gut microbiome diversity, which is linked to the prefrontal cortex, a region of the brain known to help regulate emotions.</p>
<p>&quot;It is too early to say anything conclusive, but our study indicates that adversity-associated changes in the gut microbiome are related to brain function, including differences in the regions of the brain associated with emotional processing,&quot; says Tottenham, an expert in emotional development.</p>
</li>
<li>
<p><em>abstract</em> Gastrointestinal and mental disorders are highly comorbid, and animal models have shown that both can be caused by early adversity (e.g., parental deprivation). Interactions between the brain and bacteria that live within the gastrointestinal system (the microbiome) underlie adversity–gastrointestinal–anxiety interactions, but these links have not been investigated during human development. In this study, we utilized data from a population of 344 youth (3–18 years old) who were raised with their biological parents or were exposed to early adverse caregiving experiences (i.e., institutional or foster care followed by international adoption) to explore adversity–gastrointestinal–anxiety associations. In Study 1, we demonstrated that previous adverse care experiences were associated with increased incidence of gastrointestinal symptoms in youth. Gastrointestinal symptoms were also associated with concurrent and future anxiety (measured across 5 years), and those gastrointestinal symptoms mediated the adversity–anxiety association at Time 1. In a subsample of children who provided both stool samples and functional magnetic resonance imaging of the brain (Study 2, which was a “proof-of-principle”), adversity was associated with changes in diversity (both alpha and beta) of microbial communities, and bacteria levels (adversity-associated and adversity-independent) were correlated with prefrontal cortex activation to emotional faces. Implications of these data for supporting youth mental health are discussed.</p>
</li>
</ul>
</li>
<li>
<p><em>stressor interaction networks suggest antibiotic resistance co-opted from stress responses to temperature</em><br />
mauricio cruz-loya et al. 2018<br />
<a href="http://doi.org/10.1038/s41396-018-0241-7">doi.org/10.1038/s41396-018-0241-7</a></p>
<ul>
<li>Environmental factors like temperature, pressure, and pH partly shaped the evolution of life. As life progressed, new stressors (e.g., poisons and antibiotics) arose as part of an arms race among organisms. Here we ask if cells co-opted existing mechanisms to respond to new stressors, or whether new responses evolved de novo. We use a network-clustering approach based purely on phenotypic growth measurements and interactions among the effects of stressors on population growth. We apply this method to two types of stressors—temperature and antibiotics—to discover the extent to which their cellular responses overlap in Escherichia coli. Our clustering reveals that responses to low and high temperatures are clearly separated, and each is grouped with responses to antibiotics that have similar effects to cold or heat, respectively. As further support, we use a library of transcriptional fluorescent reporters to confirm heat-shock and cold-shock genes are induced by antibiotics. We also show strains evolved at high temperatures are more sensitive to antibiotics that mimic the effects of cold. Taken together, our results strongly suggest that temperature stress responses have been co-opted to deal with antibiotic stress.</li>
</ul>
</li>
<li>
<p><em>self-induced mechanical stress can trigger biofilm formation in uropathogenic escherichia coli</em><br />
eric k. chu  et al. 2018<br />
<a href="https://doi.org/10.1038/s41467-018-06552-z">doi.org/10.1038/s41467-018-06552-z</a></p>
<ul>
<li>Bacterial biofilms represent an important medical problem; however, the mechanisms of the onset of biofilm formation are poorly understood. Here, using new controlled methods allowing high-throughput and reproducible biofilm growth, we show that biofilm formation is linked to self-imposed mechanical stress. In growing uropathogenic Escherichia coli colonies, we report that mechanical stress can initially emerge from the physical stress accompanying colony confinement within micro-cavities or hydrogel environments reminiscent of the cytosol of host cells. Biofilm formation can then be enhanced by a nutrient access-modulated feedback loop, in which biofilm matrix deposition can be particularly high in areas of increased mechanical and biological stress, with the deposited matrix further enhancing the stress levels. This feedback regulation can lead to adaptive and diverse biofilm formation guided by the environmental stresses. Our results suggest previously unappreciated mechanisms of the onset and progression of biofilm growth</li>
</ul>
</li>
<li>
<p>prenatal exposure to a mother’s stress contributes to anxiety and cognitive problems that persist into adulthood, a phenomenon that could be explained by lasting – and potentially damaging – changes in the microbiome, according to new research in mice<br />
<a href="https://www.sciencedaily.com/releases/2016/11/161114124958.htm">sciencedaily.com/releases/2016/11/161114124958.htm</a></p>
</li>
</ul>
</li>
</ul>
<h2>08294</h2>
<ul>
<li>induced by nutrients
<ul>
<li>
<p><em>nitrated meat products are associated with mania in humans and altered behavior and brain gene expression in rats</em><br />
seva g. khambadkone et al. 2018<br />
<a href="https://doi.org/10.1038/s41380-018-0105-6">doi.org/10.1038/s41380-018-0105-6</a></p>
<ul>
<li>Mania is a serious neuropsychiatric condition associated with significant morbidity and mortality. Previous studies have suggested that environmental exposures can contribute to mania pathogenesis. We measured dietary exposures in a cohort of individuals with mania and other psychiatric disorders as well as in control individuals without a psychiatric disorder. We found that a history of eating nitrated dry cured meat but not other meat or fish products was strongly and independently associated with current mania (adjusted odds ratio 3.49, 95% confidence interval (CI) 2.24–5.45, p &lt; 8.97 × 10−8). Lower odds of association were found between eating nitrated dry cured meat and other psychiatric disorders. We further found that the feeding of meat preparations with added nitrate to rats resulted in hyperactivity reminiscent of human mania, alterations in brain pathways that have been implicated in human bipolar disorder, and changes in intestinal microbiota. These findings may lead to new methods for preventing mania and for developing novel therapeutic interventions.</li>
</ul>
</li>
<li>
<p><em>exposure of the host-associated microbiome to nutrient-rich conditions may lead to dysbiosis and disease development—an evolutionary perspective</em><br />
tim lachnit et al. 2019<br />
<a href="http://doi.org/10.1128/mbio.00355-19">doi.org/10.1128/mbio.00355-19</a></p>
<ul>
<li>
<p>Since the end of the Second World War, along with the growing prosperity and the associated changes in lifestyle, numerous new and civilisation-related disease patterns have developed in today's industrialised nations. Examples of the so-called &quot;environmental diseases&quot; are different bowel inflammations like Crohn's disease or ulcerative colitis. Common causes include disruptions to the human microbiome, i.e. the natural microbial colonisation of the body, and in particular of the intestine. To date, scientists have explained this disrupted cooperation between host body and microbes with different hypotheses: for example, they postulated that excessive hygiene, the intensive use of antibiotics, or certain genetic factors permanently disrupt the microbiome, thus making people vulnerable to illnesses. However, these explanation attempts have so far been incomplete. A team from the Collaborative Research Centre (CRC) 1182 &quot;Origin and Function of Metaorganisms&quot; at Kiel University (CAU) has now formulated a new and more comprehensive ecological-evolutionary theory on the development of environmental diseases. The Kiel researchers suggest that an unnatural and particularly comprehensive nutrient supply decouples bacteria from their host organisms, and thus destroys the delicate balance of the microbiome. The, to some extent, over-fed bacteria in the gut thus promote disease development. The Kiel scientists published this fundamental new approach towards a more complete explanation of environmental diseases yesterday in the journal mBio.</p>
<p>The origin lies in the oceans</p>
<p>The starting point for the Kiel research team was the ecology of marine habitats: research on coral and algae dying off, and the associated effects on important ecosystems in the oceans, suggests that in addition to other factors such as climate change or overfishing, the nutrient conditions in the seawater may be the cause of the problem. As soon as there is an oversupply of food due to human influences, bacteria living in a community with corals begin to decouple from their hosts. They then no longer feed off the metabolic products of the host, but prefer the richer nutrient supply of the surrounding waters. The balance of the coral microbiome is disrupted because of the exodus of its symbiotic partner, and diseases occur as a result. &quot;In this connection between nutrient availability and the balance of bacteria-host relationships, we see a universal principle which goes way beyond the very specific example of corals,&quot; explained Dr Tim Lachnit, research associate at the CRC 1182 and first author of the study. &quot;In studies of our model organism, the freshwater polyp Hydra, we were able to experimentally confirm this connection,&quot; continued Lachnit. These small cnidarians also showed clear signs of disease as soon as their normal nutrient uptake was disturbed and an over-supply of food was available instead.</p>
<p>What do corals and cnidarians have to do with people?</p>
<p>With a high degree of probability, the knowledge gained in the experiment can also be transferred to human health. Similar to in seawater, or in the simple body cavity of a freshwater polyp, which during the course of evolution has decoupled from its external environment and a direct food supply, the nutrient supply in the human gut is also changing along with the civilisation-induced changes in eating habits -- towards an unbalanced, energy-rich and low-fibre diet. In addition to direct negative health consequences, a permanently high, easy to process supply of nutrients not only affects the human metabolism it feeds, but also the bacterial colonisation of the intestine, which is also &quot;fed.&quot; The microbes switch from the metabolites of the host as their staple food to the abundantly available nutrients from the human food and thus decouple from their interactions with the host organism. &quot;This over-feeding of the bacteria promotes their growth as a whole, and certain species of bacteria proliferate to the detriment of other members of the microbiome in an increased and uncontrolled manner,&quot; emphasised Professor Thomas Bosch, spokesperson of the CRC 1182. &quot;Thus, along with the change in the composition of the bacterial colonisation, the interactions between bacteria and host organism also change, and a serious maladaptation -- known as dysbiosis -- occurs,&quot; explained Dr Peter Deines, research associate at the Kiel metaorganism CRC.</p>
<p>Other civilisation-related factors increase this imbalance of the microbiome. The elimination of periodic fasting resulting from food sources not always being available, the only very rare occurrence of diarrhea leading to episodic reductions of the intestinal bacterial colonisers and the diet-related impoverishment of the microbial diversity in the gut are just a few examples. The first two of these represent very fundamental mechanisms, which since the early development of humankind right up to the pre-industrial era enabled the microbiome to return to a normal state at regular intervals, and thus regain a healthy and natural composition.</p>
<p>Does the microbiome heal itself?</p>
<p>The &quot;over-feeding hypothesis&quot; proposed by researchers from the Kiel CRC 1182, in close cooperation with the CAU Cluster of Excellence &quot;Precision Medicine in Chronic Inflammation,&quot; offers valuable approaches for further research, right through to potential transfer to future treatments: to date, scientists were particularly looking for ways to correct a disturbed microbiome through external interventions such as probiotics, i.e. the addition of certain types of helpful bacteria, or even faecal transplants to restore the balance. Now, the ecological-evolutionary perspective has added another dimension. More than ever before, it incorporates the natural ability of the microbiome to readjust itself, and to restore a healthy composition. Therefore, future research approaches lie in the specific mechanisms that balance the microbiome, and the question of whether the &quot;overfeeding&quot; of the bacteria can be reduced by changed eating habits. &quot;An interesting question will be whether the original evolutionary processes which ensure the balance of the microbiome also have therapeutic potential,&quot; said Lachnit. &quot;In the future we will, for example, not only consider the known health benefits of fasting, but also its effects on the composition and function of the microbiome, and thus on the development of inflammatory diseases,&quot; continued Lachnit.</p>
</li>
<li>
<p><em>abstract</em> Inflammatory diseases, such as inflammatory bowel diseases, are dramatically increasing worldwide, but an understanding of the underlying factors is lacking. We here present an ecoevolutionary perspective on the emergence of inflammatory diseases. We propose that adaptation has led to fine-tuned host-microbe interactions, which are maintained by secreted host metabolites nourishing the associated microbes. A constant elevation of nutrients in the gut environment leads to an increased activity and changed functionality of the microbiota, thus severely disturbing host-microbe interactions and leading to dysbiosis and disease development. In the past, starvation and pathogen infections, causing diarrhea, were common incidences that reset the gut bacterial community to its “human-specific-baseline.” However, these natural clearing mechanisms have been virtually eradicated in developed countries, allowing a constant uncontrolled growth of bacteria. This leads to an increase of bacterial products that stimulate the immune system and ultimately might initiate inflammatory reactions.</p>
</li>
</ul>
</li>
<li>
<p><em>a catalytic trisulfide in human sulfide quinone oxidoreductase catalyzes coenzyme a persulfide synthesis and inhibits butyrate oxidation</em><br />
aaron p. landry et al. 2019<br />
<a href="http://doi.org/10.1016/j.chembiol.2019.09.010">doi.org/10.1016/j.chembiol.2019.09.010</a><br />
“Sulfide is a byproduct of digestion that our bodies have to continuously contend with. If you have a diet without enough fiber, it could potentially aggravate the effects of hydrogen sulfide or our ability to detoxify it.”</p>
<ul>
<li>
<p>one of numerous byproducts of microbial chemical processes taking place in the human gut and must be detoxified. In another process, the trillions of bacteria that call our colon home, make a helpful chemical called butyrate by fermenting insoluble fiber remnants from our diet, such as fiber from whole grains and vegetables. The cells lining the colon use butyrate for energy and reduce inflammation.</p>
<p>A team of Michigan Medicine researchers has determined how the two processes -- sulfide removal and butyrate utilization -- which compete for the same cellular resources, could be held in balance inside the colon.</p>
<p>&quot;Our colon cells are well suited to detoxifying hydrogen sulfide because they are exposed to higher concentrations of it than other tissues in the body,&quot; says Aaron Landry, Ph.D., a postdoctoral fellow in the lab of Ruma Banerjee, Ph.D. in the department of Biological Chemistry, and co-first author of a new paper in Cell Chemical Biology.</p>
<p>Establishing priorities?</p>
<p>Mitochondria, which are best known as the powerhouses of the cell, contain several enzymes that contribute to energy production. One such enzyme is SQR (short for sulfide quinone oxidoreductase), which performs the first and critical step in detoxifying hydrogen sulfide and clearing it from the body.</p>
<p>SQR oxidizes hydrogen sulfide by removing electrons from it and dumping them into coenzyme Q10 (CoQ10), a compound that is naturally present in the body. (Some people take CoQ10 as a dietary supplement, though studies are currently inconclusive as to whether it is effective at preventing or treating disease.) Meanwhile, our colon cells use butyrate, produced by bacteria, for energy production using an enzyme called ACADS (short-chain acyl-CoA dehydrogenase). Since ACADS also uses CoQ10 for dumping electrons, it gives rise to a dilemma, as there is only so much CoQ10 to go around.</p>
<p>How then does the cell prioritize clearing a poisonous gas over making energy?</p>
<p>A clue came from another molecule in colon cells called coenzyme A, which is needed when butyrate is used for energy production. Landry and colleagues noted that for decades, ACADS was known to contain CoA with an extra sulfur bound to it, called CoA persulfide.</p>
<p>&quot;The additional sulfur suggested that this ligand came from somewhere, but no one knew the source of it,&quot; says Landry.</p>
<p>When CoA persulfide is bound to ACADS, it essentially blocks its function, says Landry, preventing the utilization of butyrate. They showed that SQR can convert CoA and hydrogen sulfide to CoA persulfide, which would allow prioritization of poisonous hydrogen sulfide clearance over energy production by colon cells.</p>
<p>Says Landry, &quot;Sulfide is a byproduct of digestion that our bodies have to continuously contend with. If you have a diet without enough fiber, it could potentially aggravate the effects of hydrogen sulfide or our ability to detoxify it.&quot;</p>
</li>
<li>
<p><em>abstract</em> •Human SQR utilizes CoA as an alternate sulfur acceptor generating CoA-persulfide<br />
•SQR-derived CoA-persulfide inhibits ACADS and butyrate oxidation<br />
•Four crystal structures of human SQR provide snapshots of its reaction coordinate<br />
•Cysteine trisulfide is the active cofactor and reforms at the end of turnover</p>
<p>Mitochondrial sulfide quinone oxidoreductase (SQR) catalyzes the oxidation of H 2S to glutathione persulfide with concomitant reduction of CoQ 10. We report herein that the promiscuous activity of human SQR supported the conversion of CoA to CoA-SSH (CoA-persulfide), a potent inhibitor of butyryl-CoA dehydrogenase, and revealed a molecular link between sulfide and butyrate metabolism, which are known to interact. Three different CoQ 1-bound crystal structures furnished insights into how diverse substrates access human SQR, and provided snapshots of the reaction coordinate. Unexpectedly, the active site cysteines in SQR are configured in a bridging trisulfide at the start and end of the catalytic cycle, and the presence of sulfane sulfur was confirmed biochemically. Importantly, our study leads to a mechanistic proposal for human SQR in which sulfide addition to the trisulfide cofactor eliminates 201Cys-SSH, forming an intense charge-transfer complex with flavin adenine dinucleotide, and 379Cys-SSH, which transfers sulfur to an external acceptor.</p>
</li>
</ul>
</li>
<li>
<p><em>the depressogenic potential of added dietary sugars</em><br />
daniel j. reis et al. 2019<br />
<a href="http://doi.org/10.1016/j.mehy.2019.109421">doi.org/10.1016/j.mehy.2019.109421</a></p>
<ul>
<li><em>abstract</em> Added sugars are ubiquitous in contemporary Western diets. Although excessive sugar consumption is now robustly associated with an array of adverse health consequences, comparatively little research has thus far addressed its impact on the risk of mental illness. But ample evidence suggests that high-dose sugar intake can perturb numerous metabolic, inflammatory, and neurobiological processes. Many such effects are of particular relevance to the onset and maintenance of depressive illness, among them: systemic inflammation, gut microbiota disruption, perturbed dopaminergic reward signaling, insulin resistance, oxidative stress, and the generation of toxic advanced glycation end-products (AGEs). Accordingly, we hypothesize that added dietary sugars carry the potential to increase vulnerability to major depressive disorder, particularly at high levels of consumption. The present paper: (a) summarizes the existing experimental and epidemiological research regarding sugar consumption and depression vulnerability; (b) examines the impact of sugar ingestion on known depressogenic physiological processes; and (c) outlines the clinical and theoretical implications of the apparent sugar-depression link. We conclude that the extant literature supports the hypothesized depressogenic impact of added dietary sugars, and propose that an improved understanding of the effects of sugar on body and mind may aid in the development of novel therapeutic and preventative measures for depression.</li>
<li>Coupled with dwindling light in wintertime and corresponding changes in sleep patterns, high sugar consumption could result in a “perfect storm” that adversely affects mental health, according to the researchers.<br />
“For many people, reduced sunlight exposure during the winter will throw off circadian rhythms, disrupting healthy sleep and pushing five to 10% of the population into a full-blown episode of clinical depression,” said Stephen Ilardi, KU associate professor of clinical psychology.<br />
Ilardi, who coauthored the study with KU graduate students Daniel Reis (lead author), Michael Namekata, Erik Wing and Carina Fowler (now of Duke University), said these symptoms of “winter-onset depression” could prompt people to consume more sweets.<br />
“One common characteristic of winter-onset depression is craving sugar,” he said. “So, we’ve got up to 30% of the population suffering from at least some symptoms of winter-onset depression, causing them to crave carbs — and now they’re constantly confronted with holiday sweets.”<br />
Ilardi said avoidance of added dietary sugar might be especially challenging because sugar offers an initial mood boost, leading some with depressive illness to seek its temporary emotional lift.<br />
“When we consume sweets, they act like a drug,” said the KU researcher, who also is author of “The Depression Cure” (First De Capo Press, 2009). “They have an immediate mood-elevating effect, but in high doses they can also have a paradoxical, pernicious longer-term consequence of making mood worse, reducing well-being, elevating inflammation and causing weight gain.”<br />
The investigators reached their conclusions by analyzing a wide range of research on the physiological and psychological effects of consuming added sugar, including the Women’s Health Initiative Observational Study, the NIH-AARP Diet and Health Study, a study of Spanish university graduates, and studies of Australian and Chinese soda-drinkers.<br />
Ilardi cautioned it might be appropriate to view added sugar, at high enough levels, as physically and psychologically harmful, akin to drinking a little too much liquor.<br />
“We have pretty good evidence that one alcoholic drink a day is safe, and it might have beneficial effect for some people,” he said. “Alcohol is basically pure calories, pure energy, non-nutritive and super toxic at high doses. Sugars are very similar. We’re learning when it comes to depression, people who optimize their diet should provide all the nutrients the brain needs and mostly avoid these potential toxins.”<br />
The researchers found inflammation is the most important physiological effect of dietary sugar related to mental health and depressive disorder.<br />
“A large subset of people with depression have high levels of systemic inflammation,” said Ilardi. “When we think about inflammatory disease we think about things like diabetes and rheumatoid arthritis — diseases with a high level of systemic inflammation. We don’t normally think about depression being in that category, but it turns out that it really is — not for everyone who’s depressed, but for about half. We also know that inflammatory hormones can directly push the brain into a state of severe depression. So, an inflamed brain is typically a depressed brain. And added sugars have a pro-inflammatory effect on the body and brain.”<br />
Ilardi and his collaborators also identify sugar’s impact on the microbiome as a potential contributor to depression.<br />
“Our bodies host over 10 trillion microbes and many of them know how to hack into the brain,” Ilardi said. “The symbiotic microbial species, the beneficial microbes, basically hack the brain to enhance our well-being. They want us to thrive so they can thrive. But there are also some opportunistic species that can be thought of as more purely parasitic — they don’t have our best interest in mind at all. Many of those parasitic microbes thrive on added sugars, and they can produce chemicals that push the brain in a state of anxiety and stress and depression. They’re also highly inflammatory.”</li>
</ul>
</li>
</ul>
</li>
</ul>
<h2>08394</h2>
<ul>
<li>parkinsons, alzheimers
<ul>
<li>
<p><em>porphyromonas gingivalis in alzheimer’s disease brains: evidence for disease causation and treatment with small-molecule inhibitors</em><br />
stephen s. dominy et al. 2019<br />
<a href="http://doi.org/10.1126/sciadv.aau3333">doi.org/10.1126/sciadv.aau3333</a></p>
<ul>
<li>
<p><em>abstract</em> Porphyromonas gingivalis, the keystone pathogen in chronic periodontitis, was identified in the brain of Alzheimer’s disease patients. Toxic proteases from the bacterium called gingipains were also identified in the brain of Alzheimer’s patients, and levels correlated with tau and ubiquitin pathology. Oral P. gingivalis infection in mice resulted in brain colonization and increased production of Aβ1–42, a component of amyloid plaques. Further, gingipains were neurotoxic in vivo and in vitro, exerting detrimental effects on tau, a protein needed for normal neuronal function. To block this neurotoxicity, we designed and synthesized small-molecule inhibitors targeting gingipains. Gingipain inhibition reduced the bacterial load of an established P. gingivalis brain infection, blocked Aβ1–42 production, reduced neuroinflammation, and rescued neurons in the hippocampus. These data suggest that gingipain inhibitors could be valuable for treating P. gingivalis brain colonization and neurodegeneration in Alzheimer’s disease.</p>
</li>
<li>
<p>In 2016, researchers discovered that amyloid seems to function as a sticky defence against bacteria. They found that the protein can act as an anti-microbial compound that kills bacteria, and when they injected bacteria into the brains of mice engineered to make Alzheimer’s proteins, plaques developed round bacterial cells overnight.</p>
<p>At the time, the team said it still believed that amyloid itself went on to cause the brain damage of Alzheimer’s, not bacteria. But a spate of subsequent studies have <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3073380/">looked at microbes</a>. Bacteria have been found in the brains of people who had Alzheimer’s when they were alive. But it hasn’t been clear whether the bacteria caused the disease or were simply able to enter brains damaged by Alzheimer’s.<br />
Multiple teams have been researching Porphyromonas gingivalis, the main bacterium involved in <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4399973/">gum disease</a>, which is a known risk factor for Alzheimer’s. So far, teams have found that P. gingivalis <a href="https://content.iospress.com/articles/journal-of-alzheimers-disease/jad140315">invades and inflames brain regions</a> affected by Alzheimer’s; that gum infections can <a href="https://www.nature.com/articles/s41514-017-0015-x.pdf">worsen symptoms in mice</a> genetically engineered to have Alzheimer’s; and that it can cause Alzheimer’s-like brain inflammation, neural damage and <a href="https://doi.org/10.1371/journal.pone.0204941">amyloid plaques in healthy mice</a>.<br />
<strong>A whole new hypothesis</strong><br />
“When science converges from multiple independent laboratories like this, it is very compelling,” says Casey Lynch of Cortexyme, a pharmaceutical firm in San Francisco.<br />
Now researchers from Cortexyme and several universities have reported finding the two toxic enzymes that <em>P. gingivalis</em> uses to feed on human tissue in 99 and 96 per cent of 54 human Alzheimer’s brain samples taken from the hippocampus – a brain area important for memory (<em>Science Advances</em>, <a href="http://dx.doi.org/gftvdt">doi.org/gftvdt</a>). These protein-degrading enzymes are called gingipains, and they were found in higher levels in brain tissue that also had more tau fragments and thus more cognitive decline.<br />
The team also found genetic material from <em>P. gingivalis</em> in the cerebral cortex – a region involved in conceptual thinking – in all three Alzheimer’s brains they looked for it in.<br />
“This is the first report showing <em>P. gingivalis</em> DNA in human brains, and the associated gingipains co-localising with plaques,” says Sim Singhrao at the University of Central Lancashire, UK, who wasn’t involved in the study. Her team has previously found that <em>P. gingivalis</em> <a href="https://content.iospress.com/articles/journal-of-alzheimers-disease/jad140315">actively invades the brains of mice</a> with gum infections.<br />
When Lynch and her colleagues looked at brain samples from people without Alzheimer’s, they saw that some had <em>P. gingivalis</em> and protein accumulations, but at low levels. We already know that amyloid and tau can accumulate in the brain for 10 or 20 years before Alzheimer’s symptoms begin. This, says the team, shows that <em>P. gingivalis</em> doesn’t get into the brain as a result of Alzheimer’s – but could be the cause.<br />
￼<br />
<strong>The Porphyromonas gingivalis bacteria that can cause gum disease</strong><br />
A. Dowsett, Public Health England/Science Photo Library<br />
When the team gave <em>P. gingivalis</em> gum disease to mice, it led to brain infection, amyloid production, tangles of tau protein and neural damage in the regions and nerves normally affected by Alzheimer’s. This suggests causation, says Lynch.<br />
She adds that <em>P. gingivalis</em> fulfils an updated set of criteria for attributing a disease to a particular pathogen. These conditions are named Koch’s postulates, after Robert Koch, a founder of the germ theory of disease.<br />
“The study does address most of Koch’s postulates,” says Robert Genco of the University at Buffalo, New York. “Future studies need to be in humans to be convincing.”<br />
We don’t know how <em>P. gingivalis</em> gets into the brain, but there are plausible routes it could take. Your mouth normally hosts a diverse and relatively stable community of bacteria, but when dental plaque builds under the edge of your gums, it can form inflamed pockets in which <em>P. gingivalis</em> can thrive and release toxins.</p>
<p>This inflammation can lead to <a href="https://www.nature.com/articles/s41586-018-0634-9#ref-CR9">chronic periodontitis and tooth loss</a>, and some studies have shown that people with <a href="https://jada.ada.org/article/S0002-8177(14)63215-3/abstract">fewer teeth</a> are <a href="https://www.alzheimersanddementia.com/article/S1552-5260(06)00003-3/abstract">more likely to have dementia</a>. The inflammation and toxins caused by <em>P. gingivalis</em> damage the lining of your mouth, which may make it possible for oral bacteria to enter the bloodstream and then other organs. Even if you don’t have gum disease, transient damage to your mouth lining from eating or tooth-brushing can let mouth bacteria into your blood, says Lynch.<br />
The blood-brain barrier should protect your brain from microbes, but <em>P. gingivalis</em> can invade white blood cells and the cells lining blood vessels, so might cross it that way. It may also invade cranial nerves near the mouth, then spread from cell to cell towards the brain over a period of years.</p>
<p>As to how <em>P. gingivalis</em> might cause dementia after it arrives in the brain, there are two clear possibilities. It may trigger the release of amyloid, the brain’s method of trying to contain the infection, and this may then kill neurons.<br />
Or <em>P. gingivalis</em> may directly damage the brain. We already know that Alzheimer’s involves inflammation, an excessive immune response that ends up killing neurons instead of protecting them. <em>P. gingivalis</em> is known to cause inflammation in gum tissue, and it may do so in the brain as well.<br />
In response to the new findings, David Reynolds of the Alzheimer’s UK charity said he is dubious that <em>P. gingivalis</em> causes Alzheimer’s, because of the evidence showing that a person’s genes play a crucial role in the disease. “Strong genetic evidence indicates that factors other than bacterial infections are central to the development of Alzheimer’s, so these new findings need to be taken in the context of this existing research,” he said in a statement.</p>
<p>But a bacterial hypothesis for Alzheimer’s doesn’t conflict with genetic evidence. The human body’s propensity for inflammation can vary according to genetic variations that affect our immune systems, and this may influence how much damage <em>P. gingivalis</em> induces in a brain.<br />
The biggest genetic risk factor for Alzheimer’s is a variant of the gene that makes the ApoE immune protein. Last year, a team in Sweden found that the gingipains released by <em>P. gingivalis</em> break up the ApoE protein into fragments, cleaving it at the site of a particular amino acid within the protein, and that these fragments may harm nerves. The ApoE4 variant of this protein contains more of this amino acid, suggesting that the reason people who make this variant are at a higher risk of developing Alzheimer’s may be because harmful levels of ApoE protein fragments build up more quickly in their brains than in those of other people.<br />
<strong>Hope for treatments</strong><br />
The speed at which damage accumulates is a key factor in the disease. Although many people harbour <em>P. gingivalis</em> in their mouths, only some develop Alzheimer’s. Because it can be decades before Alzheimer’s symptoms appear, whether a person develops the condition could come down to how much damage occurs before they die of other causes.<br />
“Alzheimer’s strikes people who accumulate gingipains and damage in the brain fast enough to develop symptoms during their lifetimes,” says Lynch. She says her team’s findings are a “universal hypothesis of pathogenesis”, fully explaining the causes of Alzheimer’s disease.</p>
</li>
</ul>
</li>
<li>
<p><em>do infections have a role in the pathogenesis of alzheimer disease?</em><br />
ruth f. itzhaki et al. 2020<br />
<a href="http://doi.org/10.1038/s41582-020-0323-9">doi.org/10.1038/s41582-020-0323-9</a></p>
<ul>
<li>
<p>plaques and tangles are regarded as diagnostic signposts of Alzheimer’s disease, many believe they are late arrivals in the pitiless course of the ailment, rather than primary instigators of the illness. Recently, a pair or promising experimental drugs, gantenerumab, made by Roche, and solanezumab, made by Eli Lilly, were tested on a unique group of participants. Still young and healthy at the time of the drug trials, each carried a rare mutation that guaranteed they would develop dementia over time, making them ideal candidates to test if amyloid-fighting drugs, given well in advance of AD symptoms, could prove beneficial.</p>
<p>The results, reported a month ago, confirmed the drugs failed to prevent or slow mental decline associated with dementia. It was the latest stunning blow to the amyloid hypothesis — the reigning theory describing the mechanisms of Alzheimer’s onset and progression.<br />
<strong>Another path to AD</strong><br />
Even before the amyloid hypothesis came under attack as a potential blind alley, alternate theories of the disease had been proposed. One of the more intriguing is described in the Viewpoint discussion. Perhaps Alzheimer’s is caused not by accumulations of inanimate protein but rather by microorganisms, the way so many infectious diseases are.<br />
Readhead and others have tracked the presence of various infectious agents that appear to be associated with Alzheimer’s disease. The Viewpoint discussion highlights much of the circumstantial evidence suggesting that microbes may indeed be crucial players in Alzheimer’s pathology, while emphasizing a number of confounding factors and the serious challenges involved in proving a pathogen link to the disease.<br />
In earlier research, Readhead and his colleagues at the Icahn School of Medicine at Mount Sinai used large data sets in order to explore the prevalence of two common herpesviruses sometimes found in Alzheimer’s brain tissue. The study demonstrated that three viral strains, HSV-1, HHV-6A and 7 appeared in greater abundance in brain samples derived from Alzheimer’s patients, compared with normal brains.<br />
The viruses also seem to be implicated in the AD-related genetic networks associated with classic Alzheimer’s pathology, including cell death, accumulation of amyloid-? and production of neurofibrillary tangles.<br />
In the current article, Readhead is joined by Ruth Itzhaki, Emeritus Professor at Manchester University and a Visiting Professor at Oxford University, UK; Todd E. Golde, Professor of Neuroscience and Director of the Evelyn F. and William L. McKnight Brain Institute at the University of Florida, and director of the NIH-funded Florida Alzheimer’s Disease Research Center; and Michael T. Heneka, currently the Director of the Department of Neurodegenerative Disease and Geriatric Psychiatry at the University of Bonn Medical Center.<br />
All were participants in the 2019 AAIC debate.<br />
<strong>Brain bugs</strong><br />
The Viewpoint discussion explores some of the leading evidence both for and against the infectious theory of Alzheimer’s, highlighting both viral and bacterial correlates. It also offers suggestions for future research and drug development.<br />
The panelists cite a number of reasons the pathogen theory has met with some hostility. Researchers may have insufficient background in microbiology or may inaccurately associate infectious agents solely with acute rather than chronic afflictions, though a number of microbial infections can indeed linger in the body asymptomatically for decades.<br />
Perhaps the greatest resistance to the pathogen theory comes from proponents of the amyloid hypothesis, some of whom believe that it will diminish research into amyloid plaques and tau tangles. The Viewpoint article stresses that a microbial link with AD and the amyloid hypothesis may be complementary rather than exclusionary. It is still possible that deposition of amyloid instigates a process of neurological decline, followed by opportunistic infections, or that the reverse is the case, with amyloid deposits representing a defense response to infection, trapping invasive microbes in sticky concentrations of amyloid like insects entombed in tree resin.</p>
</li>
<li>
<p><em>abstract</em> The idea that infectious agents in the brain have a role in the pathogenesis of Alzheimer disease (AD) was proposed nearly 30 years ago. However, this theory failed to gain substantial traction and was largely disregarded by the AD research community for many years. Several recent discoveries have reignited interest in the infectious theory of AD, culminating in a debate on the topic at the Alzheimer’s Association International Conference (AAIC) in July 2019. In this Viewpoint article, experts who participated in the AAIC debate weigh up the evidence for and against the infectious theory of AD and suggest avenues for future research and drug development.</p>
</li>
</ul>
</li>
<li>
<p><em>the vermiform appendix impacts the risk of developing parkinson’s disease</em><br />
bryan a. killinger et al. 2018<br />
<a href="http://doi.org/10.1126/scitranslmed.aar5280">doi.org/10.1126/scitranslmed.aar5280</a></p>
<ul>
<li>
<p>“ “Our results point to the appendix as a site of origin for Parkinson’s and provide a path forward for devising new treatment strategies that leverage the gastrointestinal tract’s role in the development of the disease,” said Viviane Labrie, Ph.D., an assistant professor at Van Andel Research Institute (VARI) and senior author of the study. “Despite having a reputation as largely unnecessary, the appendix actually plays a major part in our immune systems, in regulating the makeup of our gut bacteria and now, as shown by our work, in Parkinson’s disease.”<br />
The reduced risk for Parkinson’s was only apparent when the appendix and the alpha-synuclein contained within it were removed early in life, years before the onset of Parkinson’s, suggesting that the appendix may be involved in disease initiation. Removal of the appendix after the disease process starts, however, had no effect on disease progression.<br />
In a general population, people who had an appendectomy were 19 percent less likely to develop Parkinson’s. This effect was magnified in people who live in rural areas, with appendectomies resulting in a 25 percent reduction in disease risk. Parkinson’s often is more prevalent in rural populations, a trend that has been associated with increased exposure to pesticides.<br />
The study also demonstrated that appendectomy can delay disease progression in people who go on to develop Parkinson’s, pushing back diagnosis by an average of 3.6 years. Because there are no definitive tests for Parkinson’s, people often are diagnosed after motor symptoms such as tremor or rigidity arise. By then, the disease typically is quite advanced, with significant damage to the area of the brain that regulates voluntary movement.<br />
Conversely, appendectomies had no apparent benefit in people whose disease was linked to genetic mutations passed down through their families, a group that comprises fewer than 10 percent of cases.<br />
“Our findings today add a new layer to our understanding of this incredibly complex disease,” said Bryan Killinger, Ph.D., the study’s first author and a postdoctoral fellow in Labrie’s laboratory. “We have shown that the appendix is a hub for the accumulation of clumped forms of alpha-synuclein proteins, which are implicated in Parkinson’s disease. This knowledge will be invaluable as we explore new prevention and treatment strategies.”<br />
Labrie and her team also found clumps of alpha-synuclein in the appendixes of healthy people of all ages as well as people with Parkinson’s, raising new questions about the mechanisms that give rise to the disease and propel its progression. Clumped alpha-synuclein is considered to be a key hallmark of Parkinson’s; previously, it was thought to only be present in people with the disease.<br />
“We were surprised that pathogenic forms of alpha-synuclein were so pervasive in the appendixes of people both with and without Parkinson’s. It appears that these aggregates — although toxic when in the brain — are quite normal when in the appendix. This clearly suggests their presence alone cannot be the cause of the disease,” Labrie said. “Parkinson’s is relatively rare — less than 1 percent of the population — so there has to be some other mechanism or confluence of events that allows the appendix to affect Parkinson’s risk. That’s what we plan to look at next; which factor or factors tip the scale in favor of Parkinson’s?”<br />
Data for the study were gleaned from an in-depth characterization and visualization of alpha-synuclein forms in the appendix, which bore a remarkable resemblance to those found in the Parkinson’s disease brain, as well as analyses of two large health-record databases. The first dataset was garnered from the Swedish National Patient Registry, a one-of-a-kind database that contains de-identified medical diagnoses and surgical histories for the Swedish population beginning in 1964, and Statistics Sweden, a Swedish governmental agency responsible for official national statistics. The team at VARI collaborated with researchers at Lund University, Sweden, to comb through records for 1,698,000 people followed up to 52 years, a total of nearly 92 million person-years. The second dataset was from the Parkinson’s Progression Marker Initiative (PPMI), which includes details about patient diagnosis, age of onset, demographics and genetic information. ”</p>
</li>
<li>
<p>“Misfolded α-synuclein is a pathological hallmark of Parkinson’s disease (PD). Killinger et al. now report that the human appendix contains an abundance of misfolded α-synuclein and that removal of the appendix decreased the risk of developing PD. The appendix of both PD cases and healthy individuals contained abnormally cleaved and aggregated forms of α-synuclein, analogous to those found in postmortem brain tissue from patients with PD. Furthermore, α-synuclein derived from the appendix seeded rapid aggregation of recombinant α-synuclein in vitro. In two large-scale epidemiological studies, the authors demonstrated that an appendectomy occurring decades prior reduced the risk of developing PD, suggesting that the appendix may be implicated in PD initiation.”</p>
</li>
<li>
<p><em>abstract</em> The pathogenesis of Parkinson’s disease (PD) involves the accumulation of aggregated α-synuclein, which has been suggested to begin in the gastrointestinal tract. Here, we determined the capacity of the appendix to modify PD risk and influence pathogenesis. In two independent epidemiological datasets, involving more than 1.6 million individuals and over 91 million person-years, we observed that removal of the appendix decades before PD onset was associated with a lower risk for PD, particularly for individuals living in rural areas, and delayed the age of PD onset. We also found that the healthy human appendix contained intraneuronal α-synuclein aggregates and an abundance of PD pathology–associated α-synuclein truncation products that are known to accumulate in Lewy bodies, the pathological hallmark of PD. Lysates of human appendix tissue induced the rapid cleavage and oligomerization of full-length recombinant α-synuclein. Together, we propose that the normal human appendix contains pathogenic forms of α-synuclein that affect the risk of developing PD.</p>
</li>
</ul>
</li>
<li>
<p><em>gut microbiota regulate motor deficits and neuroinflammation in a model of parkinson’s disease</em><br />
timothy r. sampson et al. 2016<br />
<a href="http://dx.doi.org/10.1016/j.cell.2016.11.018">dx.doi.org/10.1016/j.cell.2016.11.018</a></p>
<ul>
<li>
<p>Gut microbes promote α-synuclein-mediated motor deficits and brain pathology</p>
<p>Depletion of gut bacteria reduces microglia activation</p>
<p>SCFAs modulate microglia and enhance PD pathophysiology</p>
<p>Human gut microbiota from PD patients induce enhanced motor dysfunction in mice</p>
<p>The intestinal microbiota influence neurodevelopment, modulate behavior, and contribute to neurological disorders. However, a functional link between gut bacteria and neurodegenerative diseases remains unexplored. Synucleinopathies are characterized by aggregation of the protein α-synuclein (αSyn), often resulting in motor dysfunction as exemplified by Parkinson’s disease (PD). Using mice that overexpress αSyn, we report herein that gut microbiota are required for motor deficits, microglia activation, and αSyn pathology. Antibiotic treatment ameliorates, while microbial re-colonization promotes, pathophysiology in adult animals, suggesting that postnatal signaling between the gut and the brain modulates disease. Indeed, oral administration of specific microbial metabolites to germ-free mice promotes neuroinflammation and motor symptoms. Remarkably, colonization of αSyn-overexpressing mice with microbiota from PD-affected patients enhances physical impairments compared to microbiota transplants from healthy human donors. These findings reveal that gut bacteria regulate movement disorders in mice and suggest that alterations in the human microbiome represent a risk factor for PD.</p>
</li>
</ul>
</li>
<li>
<p><em>the role of microbial amyloid in neurodegeneration</em><br />
robert p. friedland, matthew r. chapman 2017<br />
<a href="http://doi.org/10.1371/journal.ppat.1006654">doi.org/10.1371/journal.ppat.1006654</a></p>
<ul>
<li>It has become apparent that the intestinal microbiota orchestrates important aspects of our metabolism, immunity, and development. Recent work has demonstrated that the microbiota also influences brain function in healthy and diseased individuals. Of great interest are reports that intestinal bacteria play a role in the pathogenic cascade of both Parkinson and Alzheimer diseases. These neurodegenerative disorders both involve misfolding of endogenous proteins that spreads from one region of the body to another in a manner analogous to prions. The mechanisms of how the microbiota influences or is correlated with disease require elaboration. Microbial proteins or metabolites may influence neurodegeneration through the promotion of amyloid formation by human proteins or by enhancing inflammatory responses to endogenous neuronal amyloids. We review the current knowledge concerning bacterial amyloids and their potential to influence cerebral amyloid aggregation and neuroinflammation. We propose the term “mapranosis” to describe the process of microbiota-associated proteopathy and neuroinflammation. The study of amyloid proteins made by the microbiota and their influence on health and disease is in its infancy. This is a promising area for therapeutic intervention because there are many ways to alter our microbial partners and their products, including amyloid proteins.</li>
</ul>
</li>
<li>
<p><em>reduction of abeta amyloid pathology in appps1 transgenic mice in the absence of gut microbiota</em><br />
t. harach et al. 2017<br />
<a href="http://doi.org/10.1038/srep41802">doi.org/10.1038/srep41802</a></p>
<ul>
<li>Alzheimer’s disease is the most common form of dementia in the western world, however there is no cure available for this devastating neurodegenerative disorder. Despite clinical and experimental evidence implicating the intestinal microbiota in a number of brain disorders, its impact on Alzheimer’s disease is not known. To this end we sequenced bacterial 16S rRNA from fecal samples of Aβ precursor protein (APP) transgenic mouse model and found a remarkable shift in the gut microbiota as compared to non-transgenic wild-type mice. Subsequently we generated germ-free APP transgenic mice and found a drastic reduction of cerebral Aβ amyloid pathology when compared to control mice with intestinal microbiota. Importantly, colonization of germ-free APP transgenic mice with microbiota from conventionally-raised APP transgenic mice increased cerebral Aβ pathology, while colonization with microbiota from wild-type mice was less effective in increasing cerebral Aβ levels. Our results indicate a microbial involvement in the development of Abeta amyloid pathology, and suggest that microbiota may contribute to the development of neurodegenerative diseases.</li>
</ul>
</li>
</ul>
</li>
</ul>
<h2>08494</h2>
<ul>
<li>cancer
<ul>
<li>
<p><em>breast tissue, oral and urinary microbiomes in breast cancer</em><br />
hannah wang et al. 2017<br />
<a href="http://doi.org/10.18632/oncotarget.21490">doi.org/10.18632/oncotarget.21490</a></p>
<ul>
<li>It has long been proposed that the gut microbiome contributes to breast carcinogenesis by modifying systemic estrogen levels. This is often cited as a possible mechanism linking breast cancer and high-fat, low-fiber diets as well as antibiotic exposure, associations previously identified in population-based studies. More recently, a distinct microbiome has been identified within breast milk and tissue, but few studies have characterized differences in the breast tissue microbiota of patients with and without cancer, and none have investigated distant body-site microbiomes outside of the gut. We hypothesize that cancerous breast tissue is associated with a microbiomic profile distinct from that of benign breast tissue, and that microbiomes of more distant sites, the oral cavity and urinary tract, will reflect dysbiosis as well. Fifty-seven women with invasive breast cancer undergoing mastectomy and 21 healthy women undergoing cosmetic breast surgery were enrolled. The bacterial 16S rRNA gene was amplified from urine, oral rinse and surgically collected breast tissue, sequenced, and processed through a QIIME-based bioinformatics pipeline. Cancer patient breast tissue microbiomes clustered significantly differently from non-cancer patients (<em>p</em>=0.03), largely driven by decreased relative abundance of <em>Methylobacterium</em> in cancer patients (median 0.10 vs. 0.24, <em>p</em>=0.03). There were no significant differences in oral rinse samples. Differences in urinary microbiomes were largely explained by menopausal status, with peri/postmenopausal women showing decreased levels of <em>Lactobacillus</em>. Independent of menopausal status, however, cancer patients had increased levels of gram-positive organisms including <em>Corynebacterium</em> (<em>p</em>&lt;0.01), <em>Staphylococcus</em> (<em>p</em>=0.02)<em>, Actinomyces</em> (<em>p</em>&lt;0.01), and Propionibacteriaceae (<em>p</em>&lt;0.01). Our observations suggest that the local breast microbiota differ in patients with and without breast cancer. Cancer patient urinary microbiomes were characterized by increased levels of gram-positive organisms in this study, but need to be further studied in larger cohorts.</li>
</ul>
</li>
<li>
<p><em>patients with familial adenomatous polyposis harbor colonic biofilms containing tumorigenic bacteria</em><br />
christine m. dejea et al. 2018<br />
<a href="http://doi.org/10.1126/science.aah3648">doi.org/10.1126/science.aah3648</a></p>
<ul>
<li>Individuals with sporadic colorectal cancer (CRC) frequently harbor abnormalities in the composition of the gut microbiome; however, the microbiota associated with precancerous lesions in hereditary CRC remains largely unknown. We studied colonic mucosa of patients with familial adenomatous polyposis (FAP), who develop benign precursor lesions (polyps) early in life. We identified patchy bacterial biofilms composed predominately of <em>Escherichia coli</em> and <em>Bacteroides fragilis</em>. Genes for colibactin (<em>clbB</em>) and <em>Bacteroides fragilis</em> toxin (<em>bft</em>), encoding secreted oncotoxins, were highly enriched in FAP patients’ colonic mucosa compared to healthy individuals. Tumor-prone mice cocolonized with <em>E. coli</em> (expressing colibactin), and enterotoxigenic <em>B. fragilis</em> showed increased interleukin-17 in the colon and DNA damage in colonic epithelium with faster tumor onset and greater mortality, compared to mice with either bacterial strain alone. These data suggest an unexpected link between early neoplasia of the colon and tumorigenic bacteria.</li>
</ul>
</li>
<li>
<p><em>bacteroides fragilis toxin coordinates a pro-carcinogenic inflammatory cascade via targeting of colonic epithelial cells</em><br />
liam chung et al. 2018<br />
<a href="http://doi.org/10.1016/j.chom.2018.01.007">doi.org/10.1016/j.chom.2018.01.007</a></p>
<ul>
<li>
<p>•<em>B. fragilis</em> toxin-induced tumorigenesis requires epithelial IL17 and Stat3 signaling<br />
•IL17 targets colonic epithelial cells (CECs) to promote ETBF-mediated carcinogenesis<br />
•IL17-activated NF-κb signaling in CECs triggers C-X-C chemokine expression<br />
•NF-κB-induced chemokines direct pro-tumoral myeloid infiltration to distal colon</p>
<p>Pro-carcinogenic bacteria have the potential to initiate and/or promote colon cancer, in part via immune mechanisms that are incompletely understood. Using <em>ApcMin</em> mice colonized with the human pathobiont enterotoxigenic <em>Bacteroides fragilis</em> (ETBF) as a model of microbe-induced colon tumorigenesis, we show that the <em>Bacteroides fragilis</em> toxin (BFT) triggers a pro-carcinogenic, multi-step inflammatory cascade requiring IL-17R, NF-κB, and Stat3 signaling in colonic epithelial cells (CECs). Although necessary, Stat3 activation in CECs is not sufficient to trigger ETBF colon tumorigenesis. Notably, IL-17-dependent NF-κB activation in CECs induces a proximal to distal mucosal gradient of C-X-C chemokines, including CXCL1, that mediates the recruitment of CXCR2-expressing polymorphonuclear immature myeloid cells with parallel onset of ETBF-mediated distal colon tumorigenesis. Thus, BFT induces a pro-carcinogenic signaling relay from the CEC to a mucosal Th17 response that results in selective NF-κB activation in distal colon CECs, which collectively triggers myeloid-cell-dependent distal colon tumorigenesis.</p>
</li>
</ul>
</li>
<li>
<p><em>a causal mechanism for childhood acute lymphoblastic leukaemia</em><br />
mel greaves 2018<br />
<a href="http://doi.org/10.1038/s41568-018-0015-6">doi.org/10.1038/s41568-018-0015-6</a></p>
<ul>
<li>In this Review, I present evidence supporting a multifactorial causation of childhood acute lymphoblastic leukaemia (ALL), a major subtype of paediatric cancer. ALL evolves in two discrete steps. First, in utero initiation by fusion gene formation or hyperdiploidy generates a covert, pre-leukaemic clone. Second, in a small fraction of these cases, the postnatal acquisition of secondary genetic changes (primarily V(D)J recombination-activating protein (RAG) and activation-induced cytidine deaminase (AID)-driven copy number alterations in the case of ETS translocation variant 6 (<em>ETV6</em>)–runt-related transcription factor 1 (<em>RUNX1</em>)+ ALL) drives conversion to overt leukaemia. Epidemiological and modelling studies endorse a dual role for common infections. Microbial exposures earlier in life are protective but, in their absence, later infections trigger the critical secondary mutations. Risk is further modified by inherited genetics, chance and, probably, diet. Childhood ALL can be viewed as a paradoxical consequence of progress in modern societies, where behavioural changes have restrained early microbial exposure. This engenders an evolutionary mismatch between historical adaptations of the immune system and contemporary lifestyles. Childhood ALL may be a preventable cancer.</li>
</ul>
</li>
<li>
<p><em>activated atf6 induces intestinal dysbiosis and innate immune response to promote colorectal tumorigenesis</em><br />
o.i. coleman et al. 2018<br />
<a href="http://doi.org/10.1053/j.gastro.2018.07.028">doi.org/10.1053/j.gastro.2018.07.028</a></p>
<ul>
<li>In patients with CRC, ATF6 was associated with reduced time of disease-free survival. In studies of nATF6IEC mice, we found sustained intestinal activation of ATF6 in the colon to promote dysbiosis and microbiota-dependent tumorigenesis.</li>
</ul>
</li>
<li>
<p><em>mutational signature in colorectal cancer caused by genotoxic pks e. coli</em><br />
cayetano pleguezuelos-manzano et al. 2020<br />
<a href="http://doi.org/10.1038/s41586-020-2080-8">doi.org/10.1038/s41586-020-2080-8</a></p>
<ul>
<li>
<p>focused on one strain of <em>E. coli</em> producing a toxin called colibactin, and which is more often present in the stool samples of people with bowel cancer compared to healthy people. Because colibactin can cause DNA damage in cells grown in the lab, they thought the toxin might be doing the same to cells lining the gut.<br />
The team used human intestinal organoids, miniature replicas of the gut grown in the lab, and exposed them to colibactin-producing <em>E. coli</em>. They analysed the DNA sequence of the gut cells in the organoids after 5 months and found about double the DNA damage in them, compared to organoids exposed to ‘regular’ <em>E. coli</em> that didn’t produce the colibactin.<br />
The researchers also found that the DNA damage caused by colibactin followed two very specific patterns — like fingerprints — which were unique to the toxin.<br />
To determine whether the DNA damage caused by the bacterium played a role in bowel cancer, the researchers then analysed the DNA sequences of more than 5500 tumour samples from the UK and Netherlands, with the help of Dr Henry Wood and Professor Philip Quirke from the University of Leeds.<br />
First, they checked for the two colibactin DNA damage fingerprints in over 3600 Dutch samples of various cancer types. The fingerprints were present in multiple tumours, and much more often in bowel cancers than other cancer types.</p>
<p>The researchers then refined their investigation on bowel cancer tumours specifically, and analysed over 2000 bowel cancer samples from the UK, collected as part of the 100,000 Genomes Project run by Genomics England. Among these samples, the colibactin fingerprints were present in 4-5% of patients. This suggests that colibactin-producing <em>E. coli</em> may contribute to 1 in 20 bowel cancer cases in the UK. It will be up to further studies to shed light on just how much of a role the toxin could play in these cases, and what other components of the microbiome may be involved in the early stages of bowel cancer.</p>
</li>
<li>
<p><em>abstract</em> Various species of the intestinal microbiota have been associated with the development of colorectal cancer (CRC)1,2, yet a direct role of bacteria in the occurrence of oncogenic mutations has not been established. Escherichia coli can carry the pathogenicity island pks, which encodes a set of enzymes that synthesize colibactin3. This compound is believed to alkylate DNA on adenine residues4,5 and induces double-strand breaks in cultured cells3. Here, we expose human intestinal organoids to genotoxic pks+ E. coli by repeated luminal injection over a period of 5 months. Whole-genome sequencing of clonal organoids before and after this exposure reveals a distinct mutational signature, absent from organoids injected with isogenic pks-mutant bacteria. The same mutational signature is detected in a subset of 5,876 human cancer genomes from two independent cohorts, predominantly in CRC. Our study describes a distinct mutational signature in CRC and implies that the underlying mutational process directly results from past exposure to bacteria carrying the colibactin-producing pks pathogenicity island.</p>
</li>
</ul>
</li>
</ul>
</li>
</ul>
<h2>08594</h2>
<ul>
<li>immune system, inflammation, allergy
<ul>
<li>
<p><em>t cell receptor cross-reactivity between gliadin and bacterial peptides in celiac disease</em><br />
jan petersen et al. 2020<br />
<a href="http://doi.org/10.1038/s41594-019-0353-4">doi.org/10.1038/s41594-019-0353-4</a></p>
<ul>
<li>
<p>at the molecular level, how receptors isolated from immune T cells from celiac disease patients can recognise protein fragments from certain bacteria that mimic those fragments from gluten.<br />
Exposure to such bacterial proteins may be involved in the generation of aberrant recognition of gluten by these same T cells when susceptible individuals eat cereals containing gluten, he said.</p>
</li>
<li>
<p><em>abstract</em> The human leukocyte antigen (HLA) locus is strongly associated with T cell-mediated autoimmune disorders. HLA-DQ2.5-mediated celiac disease (CeD) is triggered by the ingestion of gluten, although the relative roles of genetic and environmental risk factors in CeD is unclear. Here we identify microbially derived mimics of gliadin epitopes and a parental bacterial protein that is naturally processed by antigen-presenting cells and activated gliadin reactive HLA-DQ2.5-restricted T cells derived from CeD patients. Crystal structures of T cell receptors in complex with HLA-DQ2.5 bound to two distinct bacterial peptides demonstrate that molecular mimicry underpins cross-reactivity toward the gliadin epitopes. Accordingly, gliadin reactive T cells involved in CeD pathogenesis cross-react with ubiquitous bacterial peptides, thereby suggesting microbial exposure as a potential environmental factor in CeD.</p>
</li>
</ul>
</li>
<li>
<p><em>nlrp1 restricts butyrate producing commensals to exacerbate inflammatory bowel disease</em><br />
hazel tye et al. 2018<br />
<a href="http://doi.org/10.1038/s41467-018-06125-0">doi.org/10.1038/s41467-018-06125-0</a></p>
<ul>
<li><em>abstract</em> Anti-microbial signaling pathways are normally triggered by innate immune receptors when detecting pathogenic microbes to provide protective immunity. Here we show that the inflammasome sensor Nlrp1 aggravates DSS-induced experimental mouse colitis by limiting beneficial, butyrate-producing Clostridiales in the gut. The colitis-protective effects of Nlrp1 deficiency are thus reversed by vancomycin treatment, but recapitulated with butyrate supplementation in wild-type mice. Moreover, an activating mutation in Nlrp1a increases IL-18 and IFNγ production, and decreases colonic butyrate to exacerbate colitis. We also show that, in patients with ulcerative colitis, increased NLRP1 in inflamed regions of the colon is associated with increased IFN-γ. In this context, NLRP1, IL-18 or IFN-γ expression negatively correlates with the abundance of Clostridiales in human rectal mucosal biopsies. Our data identify the NLRP1 inflammasome to be a key negative regulator of protective, butyrate-producing commensals, which therefore promotes inflammatory bowel disease.</li>
</ul>
</li>
<li>
<p><em>translocation of a gut pathobiont drives autoimmunity in mice and humans</em><br />
s. manfredo vieira et al. 2018<br />
<a href="http://doi.org/10.1126/science.aar7201">doi.org/10.1126/science.aar7201</a></p>
<ul>
<li>Despite multiple associations between the microbiota and immune diseases, their role in autoimmunity is poorly understood. We found that translocation of a gut pathobiont, <em>Enterococcus gallinarum</em>, to the liver and other systemic tissues triggers autoimmune responses in a genetic background predisposing to autoimmunity. Antibiotic treatment prevented mortality in this model, suppressed growth of <em>E. gallinarum</em> in tissues, and eliminated pathogenic autoantibodies and T cells. Hepatocyte–<em>E. gallinarum</em> cocultures induced autoimmune-promoting factors. Pathobiont translocation in monocolonized and autoimmune-prone mice induced autoantibodies and caused mortality, which could be prevented by an intramuscular vaccine targeting the pathobiont. <em>E. gallinarum</em>–specific DNA was recovered from liver biopsies of autoimmune patients, and cocultures with human hepatocytes replicated the murine findings; hence, similar processes apparently occur in susceptible humans. These discoveries show that a gut pathobiont can translocate and promote autoimmunity in genetically predisposed hosts.</li>
</ul>
</li>
<li>
<p><em>role of short chain fatty acids in controlling tregs and immunopathology during mucosal infection</em><br />
natarajan bhaskaran et al. 2018<br />
<a href="http://doi.org/10.3389/fmicb.2018.01995">doi.org/10.3389/fmicb.2018.01995</a></p>
<ul>
<li>Interactions between mucosal tissues and commensal microbes control appropriate host immune responses and inflammation, but very little is known about these interactions. Here we show that the depletion of resident bacteria using antibiotics (Abx) causes oral and gut immunopathology during oropharyngeal candidiasis (OPC) infection. Antibiotic treatment causes reduction in the frequency of Foxp3+ regulatory cells (Tregs) and IL-17A producers, with a concomitant increase in oral tissue pathology. While <em>C. albicans</em> (CA) is usually controlled in the oral cavity, antibiotic treatment led to CA dependent oral and gut inflammation. A combination of short chain fatty acids (SCFA) controlled the pathology in Abx treated mice, correlating to an increase in the frequency of Foxp3+, IL-17A+, and Foxp3+IL-17A+ double positive (Treg17) cells in tongue and oral draining lymph nodes. However, SCFA treatment did not fully reverse the gut inflammation suggesting that resident microbiota have SCFA independent homeostatic mechanisms in gut mucosa. We also found that SCFA potently induce Foxp3 and IL-17A expression in CD4+ T cells, depending on the cytokine milieu <em>in vitro</em>. Depletion of Tregs alone in FDTR mice recapitulated oral inflammation in CA infected mice, showing that Abx mediated reduction of Tregs was involved in infection induced pathology. SCFA did not control inflammation in Treg depleted mice in CA infected FDTR mice, showing that Foxp3+ T cell induction was required for the protective effect mediated by SCFA. Taken together, our data reveal that SCFA derived from resident bacteria play a critical role in controlling immunopathology by regulating T cell cytokines during mucosal infections. This study has broader implications on protective effects of resident microbiota in regulating pathological infections.</li>
</ul>
</li>
<li>
<p><em>age-associated microbial dysbiosis promotes intestinal permeability, systemic inflammation, and macrophage dysfunction</em><br />
netusha thevaranjan et al. 2017<br />
<a href="http://doi.org/10.1016/j.chom.2017.03.002">doi.org/10.1016/j.chom.2017.03.002</a></p>
<ul>
<li>
<p>•Age-associated inflammation drives macrophage dysfunction and tissue damage<br />
•Mice under germ-free conditions are protected from age-associated inflammation<br />
•Co-housing germ-free mice with old, but not young, mice increases age-related inflammation<br />
•Age-related microbiota changes can be reversed by reducing TNF levels</p>
<p>Levels of inflammatory mediators in circulation are known to increase with age, but the underlying cause of this age-associated inflammation is debated. We find that, when maintained under germ-free conditions, mice do not display an age-related increase in circulating pro-inflammatory cytokine levels. A higher proportion of germ-free mice live to 600 days than their conventional counterparts, and macrophages derived from aged germ-free mice maintain anti-microbial activity. Co-housing germ-free mice with old, but not young, conventionally raised mice increases pro-inflammatory cytokines in the blood. In tumor necrosis factor (TNF)-deficient mice, which are protected from age-associated inflammation, age-related microbiota changes are not observed. Furthermore, age-associated microbiota changes can be reversed by reducing TNF using anti-TNF therapy. These data suggest that aging-associated microbiota promote inflammation and that reversing these age-related microbiota changes represents a potential strategy for reducing age-associated inflammation and the accompanying morbidity.</p>
</li>
</ul>
</li>
<li>
<p><em>recurrent infection progressively disables host protection against intestinal inflammation</em><br />
won ho yang et al. 2017<br />
<a href="http://doi.org/10.1126/science.aao5610">doi.org/10.1126/science.aao5610</a></p>
<ul>
<li>We have discovered an environmental and pathogenic origin of chronic intestinal inflammation using a model of recurrent human food poisoning. We further show that this disease can be prevented by either IAP augmentation or Neu inhibition, both of which may represent prophylactic and therapeutic approaches to human colitis and the IBDs.</li>
</ul>
</li>
<li>
<p><em>identification of the ligand of pru p 3, a peach ltp</em><br />
nuria cubells-baeza et al. 2017<br />
<a href="http://doi.org/10.1007/s11103-017-0590-z">doi.org/10.1007/s11103-017-0590-z</a></p>
<ul>
<li>
<p>certain compounds carried by allergic proteins would be collaborating agents needed in the processes that trigger immune responses with the appearance of allergic symptoms.<br />
This study was carried out by researchers from Mount Sinai Hospital in New York, Swiss Allergy Centre in Zurich and Institute of Applied Molecular Medicine of Universidad San Pablo CEU in Madrid. The obtained results will help develop preventive methods and effective treatments for allergies.<br />
In spite of a great effort, fundamental questions on the molecular and immunological origin of allergies are still ignored. Many studies have sought solutions to an ancient enigma: why some proteins cause allergies despite being very similar to others that are harmless. Researchers achieved to identify the proteins which are common allergens are in pollen, mites, domestic animals, food…<br />
Unfortunately, the characteristics of these proteins linked to allergens have been not found yet. However, in recent years a hypothesis developed in collaboration with the allergens group of CBGP has gained importance: certain compounds carried by the allergen proteins, known as ligands, would act as necessary collaborator agents in the allergic sensitization phase.<br />
Pru p 3 was the protein selected for this study. This protein is responsible for the peach allergy, being very common in Mediterranean countries. The CBGP team has recently identified in a study the natural ligand of Pru p 3 as a compound formed by an alkaloid attached to a hydrocarbon tail. In the current study, the team has found direct evidence of the participation of the Pru p 3 ligand in the processes of the immune system recognition in the allergic sensitization phase.<br />
Results reveal that the ligand is recognized by a type of cellular receptor called CD1d in the cell surface where the antigens appear, that is, substances able to provoke a response of the immune system to produce antibodies. The CD1d expressions are responsible for presenting lipid antigens activating cells of the immune system called iNKT (invariant natural killer T-cells). Once activated, these iNKT cells produce an enormous amount of substances that cause the characteristic symptoms of allergic disorders.<br />
Since many allergens transport diverse compounds, the discovery of Pru p 3 lipid-ligand as an adjuvant to promote allergic sensitization through its recognition by CD1d expressions open new horizons. This new discovery could be a general essential feature of the mechanism underlying the phenomenon of allergenicity.</p>
</li>
<li>
<p><em>abstract</em> The allergen Pru p 3, a peach lipid transfer protein, has been well studied. However, its physiological function remains to be elucidated. Our results showed that Pru p 3 usually carries a lipid ligand that play an essential role in its function in plants. Using ESI-qToF, we observed that the ligand was a derivative of camptothecin binding to phytosphingosine, wich that is inserted into the hydrophobic tunnel of the protein. In addition, the described ligand displayed topoisomerase I activity inhibition and self-fluorescence, both recognized as camptothecin properties. During flower development, the highest expression of <em>Pru p 3</em> was detected in the styles of pollinated flowers, in contrast to its non-expression in unpollinated pistils, where expression decreased after anthesis. During ripening, the expression of Pru p 3 were observed mainly in peel but not in pulp. In this sense, Pru p 3 protein was also localized in trichomes covering the fruit epidermis.</p>
</li>
</ul>
</li>
<li>
<p><em>loss of paneth cell autophagy causes acute susceptibility to toxoplasma gondii-mediated inflammation</em><br />
elise burger et al. 2018<br />
<a href="http://doi.org/10.1016/j.chom.2018.01.001">doi.org/10.1016/j.chom.2018.01.001</a></p>
<ul>
<li>
<p>•Basal Paneth cell autophagy is driven by the microbiota and IFN-γ<br />
•Paneth cell autophagy protects against acute infection<br />
•Loss of Paneth cell autophagy results in impaired intestinal permeability<br />
•Loss of Paneth cell autophagy leads to TNF- and IFN-γ mediated intestinal pathology</p>
<p>The protozoan parasite <em>Toxoplasma gondii</em> triggers severe small intestinal immunopathology characterized by IFN-γ- and intestinal microbiota-mediated inflammation, Paneth cell loss, and bacterial dysbiosis. Paneth cells are a prominent secretory epithelial cell type that resides at the base of intestinal crypts and releases antimicrobial peptides. We demonstrate that the microbiota triggers basal Paneth cell-specific autophagy via induction of IFN-γ, a known trigger of autophagy, to maintain intestinal homeostasis. Deletion of the autophagy protein Atg5 specifically in Paneth cells results in exaggerated intestinal inflammation characterized by complete destruction of the intestinal crypts resembling that seen in pan-epithelial Atg5-deficient mice. Additionally, lack of functional autophagy in Paneth cells within intestinal organoids and <em>T. gondii</em>-infected mice causes increased sensitivity to the proinflammatory cytokine TNF along with increased intestinal permeability, leading to exaggerated microbiota- and IFN-γ-dependent intestinal immunopathology. Thus, Atg5 expression in Paneth cells is essential for tissue protection against cytokine-mediated immunopathology during acute gastrointestinal infection.</p>
</li>
</ul>
</li>
<li>
<p><em>dysbiosis-induced secondary bile acid deficiency promotes intestinal inflammation</em><br />
sidhartha r. sinha et al. 2020<br />
<a href="http://doi.org/10.1016/j.chom.2020.01.021">doi.org/10.1016/j.chom.2020.01.021</a></p>
<ul>
<li>
<p>Ulcerative colitis is an inflammatory condition in which the immune system attacks tissue in the rectum or colon. Patients can suffer from heavy bleeding, diarrhea, weight loss and, if the colon becomes sufficiently perforated, life-threatening sepsis.<br />
There is no known cure. While immunosuppressant drugs can keep ulcerative colitis at bay, they put patients at increased risk for cancer and infection. Moreover, not all patients respond, and even when an immunosuppressant drug works initially, its effectiveness can fade with time. About one in five ulcerative colitis patients progress to the point where they require total colectomy, the surgical removal of the colon and rectum, followed by the repositioning of the lower end of the small intestine to form a J-shaped pouch that serves as a rectum.<br />
These “pouch patients” can lead quite normal lives. However, as many as half will develop pouchitis, a return of the inflammation and symptoms they experienced in their initial condition.</p>
<p>The new study began with a clinical observation. “Patients with a rare genetic condition called familial adenomatous polyposis, or FAP, are at extremely high risk for colon cancer,” Habtezion said. “To prevent this, they undergo the exact same surgical procedure patients with refractory ulcerative colitis do.” Yet FAP pouch patients rarely if ever experience the inflammatory attacks on their remaining lower digestive tract that ulcerative-colitis patients with a pouch do, she said.<br />
The Stanford scientists decided to find out why. Their first clue lay in a large difference in levels of a group of substances called secondary bile acids in the intestines of seven FAP patients compared with 17 patients with ulcerative colitis who had undergone the pouch surgery. The investigators measured these metabolite levels by examining the participants’ stool samples.<br />
Primary bile acids are produced in the liver, stored in the gallbladder and released into the digestive tract to help emulsify fats. The vast majority of secreted primary bile acids are taken up in the intestine, where resident bacteria perform a series of enzymatic operations to convert them to secondary bile acids.<br />
Prior research has suggested, without much elaboration or follow-up, that secondary bile acids are depleted in ulcerative colitis patients and in those with a related condition, Crohn’s disease, in which tissue-destroying inflammation can occur in both the colon and the small intestine.<br />
The researchers confirmed that levels of the two most prominent secondary bile acids, deoxycholic acid and lithocholic acid, were much lower in stool specimens taken from the ulcerative colitis pouch patients than from FAP pouch patients. Clearly, the surgical procedure hadn’t caused the depletion.<br />
<strong>Diminished microbial diversity</strong><br />
These findings were mirrored by the scientists’ observation that microbial diversity in the specimens from ulcerative colitis pouch patients was diminished. Moreover, the investigators showed that a single bacterial family — Ruminococcaceae — was markedly underrepresented in ulcerative colitis pouch patients compared with FAP pouch patients. A genomic analysis of all the gut bacteria in the participants showed that the genes for making enzymes that convert primary bile acids to secondary bile acids were underrepresented, too. Ruminococcaceae, but few other gut bacteria, carry those genes.<br />
“All healthy people have Ruminococcaceae in their intestines,” Habtezion said. “But in the UC pouch patients, members of this family were significantly depleted.”<br />
Incubating primary bile acids with stool samples from FAP pouch patients, but not from ulcerative colitis pouch patients, resulted in those substances’ effective conversion to secondary bile acids.<br />
In three different mouse models of colitis, supplementation with lithocholic acid and deoxycholic acid reduced infiltration by inflammatory immune cells and levels of several inflammatory signaling proteins and chemicals in the mice’s intestines, the researchers showed. The supplements also mitigated the classic symptoms of colitis in the mice, such as weight loss or signs of colon pathology.<br />
All three mouse models are considered representative of not just ulcerative colitis but inflammatory bowel disease in general, a category that also includes Crohn’s disease. So the findings may apply to Crohn’s disease patients, as well, Habtezion said.</p>
</li>
<li>
<p><em>abstract</em> •Secondary bile acids (SBAs) are reduced in UC pouch patients, relative to FAP patients<br />
•Reduced Ruminococcaceae in UC pouches is associates with SBA deficiency<br />
•SBA supplementation ameliorates inflammation in animal models of colitis<br />
•The protective effect of SBAs is in part dependent on the TGR5 bile acid receptor</p>
<p>Secondary bile acids (SBAs) are derived from primary bile acids (PBAs) in a process reliant on biosynthetic capabilities possessed by few microbes. To evaluate the role of BAs in intestinal inflammation, we performed metabolomic, microbiome, metagenomic, and transcriptomic profiling of stool from ileal pouches (surgically created resevoirs) in colectomy-treated patients with ulcerative colitis (UC) versus controls (familial adenomatous polyposis [FAP]). We show that relative to FAP, UC pouches have reduced levels of lithocholic acid and deoxycholic acid (normally the most abundant gut SBAs), genes required to convert PBAs to SBAs, and Ruminococcaceae (one of few taxa known to include SBA-producing bacteria). In three murine colitis models, SBA supplementation reduces intestinal inflammation. This anti-inflammatory effect is in part dependent on the TGR5 bile acid receptor. These data suggest that dysbiosis induces SBA deficiency in inflammatory-prone UC patients, which promotes a pro-inflammatory state within the intestine that may be treated by SBA restoration.</p>
</li>
</ul>
</li>
</ul>
</li>
</ul>
<h2>08694</h2>
<ul>
<li>obesity, hypertension
<ul>
<li>
<p><em>molecular phenomics and metagenomics of hepatic steatosis in non-diabetic obese women</em><br />
lesley hoyles et al. 2018<br />
<a href="http://doi.org/10.1038/s41591-018-0061-3">doi.org/10.1038/s41591-018-0061-3</a></p>
<ul>
<li>Hepatic steatosis is a multifactorial condition that is often observed in obese patients and is a prelude to non-alcoholic fatty liver disease. Here, we combine shotgun sequencing of fecal metagenomes with molecular phenomics (hepatic transcriptome and plasma and urine metabolomes) in two well-characterized cohorts of morbidly obese women recruited to the FLORINASH study. We reveal molecular networks linking the gut microbiome and the host phenome to hepatic steatosis. Patients with steatosis have low microbial gene richness and increased genetic potential for the processing of dietary lipids and endotoxin biosynthesis (notably from Proteobacteria), hepatic inflammation and dysregulation of aromatic and branched-chain amino acid metabolism. We demonstrated that fecal microbiota transplants and chronic treatment with phenylacetic acid, a microbial product of aromatic amino acid metabolism, successfully trigger steatosis and branched-chain amino acid metabolism. Molecular phenomic signatures were predictive (area under the curve = 87%) and consistent with the gut microbiome having an effect on the steatosis phenome (&gt;75% shared variation) and, therefore, actionable via microbiome-based therapies.</li>
</ul>
</li>
<li>
<p><em>glucocorticoids and gut bacteria: “the galf hypothesis” in the metagenomic era</em><br />
david j. morris, jason m. ridlon 2017<br />
<a href="http://doi.org/10.1016/j.steroids.2017.06.002">doi.org/10.1016/j.steroids.2017.06.002</a></p>
<ul>
<li>
<p>The gut microbiota are implicated causally in essential hypertension.<br />
A proposed mechanism of essential hypertension is production of endogenous of glycerrhetinic acid-like factors (GALFs)<br />
GALFs act as competitive inhibitors of renal 11β-hydroxysteroid dehydrogenase.<br />
Gut bacteria are capable of converting host glucocorticoids into GALFs.</p>
<p>A new concept is emerging in biomedical sciences: the gut microbiota is a virtual ‘organ’ with endocrine function. Here, we explore the literature pertaining to the role of gut microbial metabolism of endogenous adrenocorticosteroids as a contributing factor in the etiology of essential hypertension. A body of literature demonstrates that bacterial products of glucocorticoid metabolism are absorbed into the portal circulation, and pass through the kidney before excretion into urine. Apparent mineralocorticoid excess (AME) syndrome patients were found to have congenital mutations resulting in non-functional renal 11β-hydroxysteroid dehydrogenase-2 (11β-HSD2) and severe hypertension often lethal in childhood. 11β-HSD2 acts as a “guardian” enzyme protecting the mineralocorticoid receptor from excess cortisol, preventing sodium and water retention in the normotensive state. Licorice root, whose active ingredient, glycerrhetinic acid (GA), inhibits renal 11β-HSD2, and thereby causes hypertension in some individuals. Bacterially derived glucocorticoid metabolites may cause hypertension in some patients by a similar mechanism. Parallel observations in gut microbiology coupled with screening of endogenous steroids as inhibitors of 11β-HSD2 have implicated particular gut bacteria in essential hypertension through the production of glycerrhetinic acid-like factors (GALFs). A protective role of GALFs produced by gut bacteria in the etiology of colorectal cancer is also explored.</p>
</li>
</ul>
</li>
<li>
<p><em>characterization of metabolic responses to healthy diets and association with blood pressure: application to the optimal macronutrient intake trial for heart health (omniheart), a randomized controlled study</em><br />
ruey leng loo et al. 2018<br />
<a href="http://doi.org/10.1093/ajcn/nqx072">doi.org/10.1093/ajcn/nqx072</a></p>
<ul>
<li>we identified 2 gut microbial-host co-metabolites associated with BP: phenylacetylglutamine and 4-cresyl sulfate, deriving from phenylalanine and tyrosine, respectively, resulting from bacterial putrefaction of protein in the distal colon. The gut microbiota, in particular <em>Firmicutes</em> and <em>Bacteroidetes</em>, can adapt to dietary changes and induce changes in host metabolism (35): an increase of <em>Firmicutes</em> to <em>Bacteroidetes</em> ratio has been demonstrated in spontaneous hypertensive rats (36). Other researchers have manipulated gut microbiota balance via probiotic administration with consequent beneficial effects on BP levels (37). More recently, blood concentrations of phenylacetylglutamine were found to be strongly anti-correlated with BP, consistent with our results, and with carotid-femoral pulse-wave velocity, a measure of aortic stiffness (38). Although 4-cresyl sulfate has never been formally linked to BP, its dietary excretion has been shown to be highly correlated with that of phenylacetylglutamine (16).</li>
<li>The association between gut-microbial co-metabolites and BP is further evidenced in the direct association we found between BP and hippurate, originating from the conversion of benzoic acid by gut microflora via the shikimate pathway (39). In contrast to our results, hypertensive rats showed an anti-correlation between hippurate and BP (40) but interpolation from animal data to human must be performed with care due to the differences in gut microbiomes between species. An inverse association between excretion of hippurate and BP has been reported in humans but this association was not significant after adjusting for BMI, alcohol intake, and urinary excretion of sodium and potassium (41). A controlled feeding study by Wu et al. (35) showed that changes in the gut microbiome occurred within 24 h of initiating a change in diet and that BMI and weight loss can also influence the gut microbiome composition. However, in our dietary intervention study, all participants consumed a consistent healthy dietary pattern for 6 wk and maintained their body weight, with micronutrients being indexed to the energy level of their diets. Our data, therefore, suggest modulation of diets can affect gut microbiome activity and that this may lead to a direct effect on BP regulation.</li>
<li>We observed an inverse association of <em>N</em>-methyl-2-pyridone-5-carboxamide (tryptophan–NAD metabolite) and BP. Bartus et al. (42) showed that ingestion of 1*-<em>methylnicotinamide in hypertriglyceridemic rats resulted in an increase of 1</em>-<em>methylnicotinamide and its metabolites such as <em>N</em>-methyl-2-pyridone-5-carboxamide and found that ingestion of 1</em>-<em>methylnicotinamide in both the diabetic and hypertriglyceridemic rats can ameliorate the nitric oxide-dependent vasodilation, a surrogate marker for atherosclerosis. Others have found that 1</em>-*methylnicotinamide demonstrates anti-thrombotic activity (43). Our findings further support the beneficial impact of <em>N</em>-methyl-2-pyridone-5-carboxamide on CVD health. We suggest the tryptophan–NAD pathway may offer a new target for pharmacological treatment of hypertension.</li>
<li>We also confirmed the association of dietary markers with BP including: a direct association between BP and carnitine (a marker for protein ingestion); and an inverse association with proline-betaine (citrus fruit ingestion). Our results are consistent with previous studies linking hypertension with blood concentration of carnitine (44) and variations in BP following carnitine treatment in rats (45). Similarly our results support the previously postulated benefit of citrus fruit intake in reduction of BP (34). Specifically for the OmniProt diet, despite the increased excretion of carnitine, a marker which was linked to higher BP, overall beneficial reductions in CVD risk factors (both BP levels and lipid profiles) were elicited and these benefits persisted for those who were considered as typical (HDR) as well as variable (VDR) responders. The specific mechanisms for this remain unclear although it may be hypothesized that the altered large-bowel microbiome following protein-rich dietary intervention may play a significant role.</li>
</ul>
</li>
</ul>
</li>
</ul>
<h2>08794</h2>
<ul>
<li>brain interaction
<ul>
<li>
<p><em>endothelial tlr4 and the microbiome drive cerebral cavernous malformations</em><br />
alan t. tang et al. 2017<br />
<a href="http://doi.org/10.1038/nature22075">doi.org/10.1038/nature22075</a></p>
<ul>
<li>Cerebral cavernous malformations (CCMs) are a cause of stroke and seizure for which no effective medical therapies yet exist. CCMs arise from the loss of an adaptor complex that negatively regulates MEKK3–KLF2/4 signalling in brain endothelial cells, but upstream activators of this disease pathway have yet to be identified. Here we identify endothelial Toll-like receptor 4 (TLR4) and the gut microbiome as critical stimulants of CCM formation. Activation of TLR4 by Gram-negative bacteria or lipopolysaccharide accelerates CCM formation, and genetic or pharmacologic blockade of TLR4 signalling prevents CCM formation in mice. Polymorphisms that increase expression of the <em>TLR4</em> gene or the gene encoding its co-receptor CD14 are associated with higher CCM lesion burden in humans. Germ-free mice are protected from CCM formation, and a single course of antibiotics permanently alters CCM susceptibility in mice. These studies identify unexpected roles for the microbiome and innate immune signalling in the pathogenesis of a cerebrovascular disease, as well as strategies for its treatment.</li>
</ul>
</li>
<li>
<p><em>differences in gut microbial composition correlate with regional brain volumes in irritable bowel syndrome</em><br />
jennifer s. labus et al. 2017<br />
<a href="http://doi.org/10.1186/s40168-017-0260-z">doi.org/10.1186/s40168-017-0260-z</a></p>
<ul>
<li>The results confirm previous reports of gut microbiome-based IBS subgroups and identify for the first time brain structural alterations associated with these subgroups. They provide preliminary evidence for the involvement of specific microbes and their predicted metabolites in these correlations.</li>
</ul>
</li>
<li>
<p><em>bidirectional brain-gut interactions and chronic pathological changes after traumatic brain injury in mice</em><br />
elise l. ma et al. 2017<br />
<a href="http://doi.org/10.1016/j.bbi.2017.06.018">doi.org/10.1016/j.bbi.2017.06.018</a></p>
<ul>
<li>
<p>•Experimental TBI induces chronic structural and functional changes in the colon.<br />
•Barrier dysfunction after TBI is linked to decreased colonic claudin-1 expression.<br />
•TBI alters the number of activated enteric glial cells in the colon mucosa.<br />
•Enteric <em>C. rodentium</em> infection weeks after TBI exacerbates brain neuropathology.</p>
<p>Objectives<br />
<a href="http://www.sciencedirect.com/topics/neuroscience/traumatic-brain-injury">Traumatic brain injury</a> (TBI) has complex effects on the gastrointestinal tract that are associated with TBI-related morbidity and mortality. We examined changes in <a href="http://www.sciencedirect.com/topics/neuroscience/mucous-membrane">mucosal</a> barrier properties and enteric <a href="http://www.sciencedirect.com/topics/neuroscience/glial-cells">glial cell</a> response in the gut after experimental TBI in mice, as well as effects of the enteric pathogen <a href="http://www.sciencedirect.com/topics/immunology-and-microbiology/citrobacter-rodentium"><em>Citrobacter rodentium</em></a> (<em>Cr</em>) on both gut and brain after injury.<br />
Methods<br />
Moderate-level TBI was induced in C57BL/6 mice by controlled cortical impact (CCI). Mucosal barrier function was assessed by transepithelial resistance, fluorescent-labelled dextran flux, and quantification of <a href="http://www.sciencedirect.com/topics/neuroscience/tight-junction">tight junction</a> proteins. Enteric glial cell number and activation were measured by Sox10 expression and <a href="http://www.sciencedirect.com/topics/neuroscience/glial-fibrillary-acidic-protein">GFAP</a> reactivity, respectively. Separate groups of mice were challenged with <em>Cr</em> infection during the chronic phase of TBI, and <a href="http://www.sciencedirect.com/topics/immunology-and-microbiology/host-immune-response">host immune response</a>, barrier integrity, enteric glial cell reactivity, and progression of brain injury and inflammation were assessed.<br />
Results<br />
Chronic CCI induced changes in colon morphology, including increased mucosal depth and smooth muscle thickening. At day 28 post-CCI, increased paracellular permeability and decreased claudin-1 mRNA and protein expression were observed in the absence of inflammation in the colon. Colonic glial cell GFAP and Sox10 expression were significantly increased 28 days after brain injury. Clearance of <em>Cr</em> and <a href="http://www.sciencedirect.com/topics/neuroscience/downregulation-and-upregulation">upregulation</a> of Th1/Th17 <a href="http://www.sciencedirect.com/topics/neuroscience/cytokines">cytokines</a> in the colon were unaffected by CCI; however, colonic paracellular flux and enteric glial cell GFAP expression were significantly increased. Importantly, <em>Cr</em> infection in chronically-injured mice worsened the brain lesion injury and increased astrocyte- and microglial-mediated inflammation.<br />
Conclusion<br />
These experimental studies demonstrate chronic and bidirectional <a href="http://www.sciencedirect.com/topics/immunology-and-microbiology/gutbrain-axis">brain-gut</a> interactions after TBI, which may negatively impact late outcomes after brain injury.</p>
</li>
</ul>
</li>
<li>
<p><em>gaba-modulating bacteria of the human gut microbiota</em><br />
philip strandwitz et al. 2019<br />
<a href="http://doi.org/10.1038/s41564-018-0307-3">doi.org/10.1038/s41564-018-0307-3</a></p>
<ul>
<li>
<p>Gut microbiota, the entire collection of microorganisms found in that habitat, affect many important functions, including the immune response and the nervous system. Nevertheless, many microorganisims residing in the human gut remain uncultured, which the research team called &quot;an obstacle for understanding their biological roles&quot; in the Nature Microbiology article.</p>
<p>More such microorganisms probably remain uncultured because they require key growth factors that are provided by neighboring bacteria in their natural environments, but not under artificial laboratory conditions. During an extensive screening process, the team found that KLE1738 required the presence of Bacteroides fragilis, a common human gut bacterium, to grow.</p>
<p>Further biological testing and purification led to the isolation of GABA as the growth factor produced by Bacteroides fragilis. GABA was, in fact, the only nutrient tested during the experiments that supported the growth of KLE1738.</p>
<p>In the next research phase, the team explored the possible connection between Bacteroides and depression. Stool samples and functional magnetic resonance imaging measurements of brain activity were collected from 23 subjects suffering from clinically diagnosed depression.</p>
<p>The researchers found an inverse relationship between the relative abundance of fecal Bacteroidesand functional connectivity in a part of the brain associated with elevated activity during depression. This means that low abundance of Bacteroides was associated with high activity in that part of the brain, and vice versa.</p>
<p>&quot;A good first step is to repeat our findings in additional human cohorts, which we are actively exploring,&quot; said Strandwitz of further research. &quot;When it comes to depression, animal models are often difficult to translate, which is why we are so excited about human studies. &quot;</p>
<p>Recent work published in the journals Science and Cell have identified the presence of sensory neurons in the gut that are hard-wired to the brain. &quot;It would be great to explore whether microbial GABA can act as a signal via that pathway,&quot; said Anukriti Sharma, a co-author of the Nature Microbiology article and a postdoctoral scholar at Argonne</p>
</li>
<li>
<p><em>abstract</em> The gut microbiota affects many important host functions, including the immune response and the nervous system1. However, while substantial progress has been made in growing diverse microorganisms of the microbiota2, 23–65% of species residing in the human gut remain uncultured3,4, which is an obstacle for understanding their biological roles. A likely reason for this unculturability is the absence in artificial media of key growth factors that are provided by neighbouring bacteria in situ5,6. In the present study, we used co-culture to isolate KLE1738, which required the presence of Bacteroides fragilis to grow. Bioassay-driven purification of B. fragilis supernatant led to the isolation of the growth factor, which, surprisingly, is the major inhibitory neurotransmitter GABA (γ-aminobutyric acid). GABA was the only tested nutrient that supported the growth of KLE1738, and a genome analysis supported a GABA-dependent metabolism mechanism. Using growth of KLE1738 as an indicator, we isolated a variety of GABA-producing bacteria, and found that Bacteroides ssp. produced large quantities of GABA. Genome-based metabolic modelling of the human gut microbiota revealed multiple genera with the predicted capability to produce or consume GABA. A transcriptome analysis of human stool samples from healthy individuals showed that GABA-producing pathways are actively expressed by Bacteroides, Parabacteroides and Escherichia species. By coupling 16S ribosmal RNA sequencing with functional magentic resonance imaging in patients with major depressive disorder, a disease associated with an altered GABA-mediated response, we found that the relative abundance levels of faecal Bacteroides are negatively correlated with brain signatures associated with depression.</p>
</li>
</ul>
</li>
</ul>
</li>
</ul>
<h2>08894</h2>
<ul>
<li>transmissible disease
<ul>
<li>
<p><em>cutaneous leishmaniasis induces a transmissible dysbiotic skin microbiota that promotes skin inflammation</em><br />
ciara gimblet et al. 2017<br />
<a href="http://doi.org/10.1016/j.chom.2017.06.006">doi.org/10.1016/j.chom.2017.06.006</a></p>
<ul>
<li>
<p>Leishmania infection alters the skin microbiota of both humans and mice<br />
Dysbiosis is characterized by a dominance of <em>Staphylococcus</em> and/or <em>Streptococcus</em><br />
Naive mice acquire dysbiosis when co-housed with leishmania-infected mice<br />
Acquiring a dysbiotic microbiota prior to infection exacerbates skin inflammation</p>
<p>Skin microbiota can impact allergic and autoimmune responses, wound healing, and anti-microbial defense. We investigated the role of skin microbiota in cutaneous leishmaniasis and found that human patients infected with <em>Leishmania braziliensis</em> develop dysbiotic skin microbiota, characterized by increases in the abundance of <em>Staphylococcus</em> and/or <em>Streptococcus</em>. Mice infected with <em>L. major</em> exhibit similar changes depending upon disease severity. Importantly, this dysbiosis is not limited to the lesion site, but is transmissible to normal skin distant from the infection site and to skin from co-housed naive mice. This observation allowed us to test whether a pre-existing dysbiotic skin microbiota influences disease, and we found that challenging dysbiotic naive mice with <em>L. major</em> or testing for contact hypersensitivity results in exacerbated skin inflammatory responses. These findings demonstrate that a dysbiotic skin microbiota is not only a consequence of tissue stress, but also enhances inflammation, which has implications for many inflammatory cutaneous diseases.</p>
</li>
</ul>
</li>
</ul>
</li>
</ul>
<h2>08994</h2>
<ul>
<li>diagnosis
<ul>
<li>
<p><em>extreme dysbiosis of the microbiome in critical illness</em><br />
daniel mcdonald et al 2016<br />
<a href="http://doi.org/10.1128/msphere.00199-16">doi.org/10.1128/msphere.00199-16</a></p>
<ul>
<li>Critical illness is hypothesized to associate with loss of “health- promoting” commensal microbes and overgrowth of pathogenic bacteria (dysbiosis). This dysbiosis is believed to increase susceptibility to nosocomial infections, sepsis, and organ failure. A trial with prospective monitoring of the intensive care unit (ICU) patient microbiome using culture-independent techniques to confirm and character- ize this dysbiosis is thus urgently needed. Characterizing ICU patient microbiome changes may provide first steps toward the development of diagnostic and thera- peutic interventions using microbiome signatures. To characterize the ICU patient microbiome, we collected fecal, oral, and skin samples from 115 mixed ICU patients across four centers in the United States and Canada. Samples were collected at two time points: within 48 h of ICU admission, and at ICU discharge or on ICU day 10. Sample collection and processing were performed according to Earth Microbiome Project protocols. We applied SourceTracker to assess the source composition of ICU patient samples by using Qiita, including samples from the American Gut Project (AGP), mammalian corpse decomposition samples, childhood (Global Gut study), and house surfaces. Our results demonstrate that critical illness leads to significant and rapid dysbiosis. Many taxons significantly depleted from ICU patients versus AGP healthy controls are key “health-promoting” organisms, and overgrowth of known pathogens was frequent. Source compositions of ICU patient samples are largely un- characteristic of the expected community type. Between time points and within a patient, the source composition changed dramatically. Our initial results show great promise for microbiome signatures as diagnostic markers and guides to therapeutic interventions in the ICU to repopulate the normal, “health-promoting” microbiome and thereby improve patient outcomes.</li>
</ul>
</li>
<li>
<p><em>colorectal cancer-associated microbiota contributes to oncogenic epigenetic signatures</em><br />
iradj sobhani et al. 2019<br />
<a href="http://doi.org/10.1073/pnas.1912129116">doi.org/10.1073/pnas.1912129116</a></p>
<ul>
<li>
<p>Sporadic colorectal cancer can develop in patients without any known risk factors. It occurs as a result of complex interactions between individuals and their environment. The increasing incidence of this condition reflects negative environmental developments, which can trigger alterations to the genetic and epigenetic DNA of host cells, thereby promoting the onset of sporadic colorectal cancer.<br />
Several studies have investigated the role of the microbiota as a mediator of these interactions. The team in the Gastroenterology Department at Henri-Mondor AP-HP Hospital and University Paris-Est Créteil, together with a team composed of members from Inserm and the Institut Pasteur Molecular Microbial Pathogenesis Unit (U1202), and the Chair in Microbiology and Infectious Diseases at the Collège de France, demonstrated in mice that the epigenetic mechanism triggered by some bacteria in the microbiota contributes to the onset or promotion of sporadic colorectal cancer. They subsequently validated their findings in humans.<br />
The 136 mice in the study were transplanted with either fresh stools from nine patients with sporadic colorectal cancer or fresh stools from nine patients with no colon disorders. The procedure was carried out at Henri-Mondor AP-HP Hospital. The colons of the mice were examined 7 and 14 weeks after the human fecal microbiota transplant. The teams particularly investigated the number and development of aberrant crypt foci (or ACF, a type of precancerous lesion), the microbial profile and damage to colonic DNA. They also took into account the animals’ food intake, weight and blood indicators.<br />
A link between fecal dysbiosis (an imbalance in gut bacteria composition) and the genetic and epigenetic DNA signature in the animals’ tissues was identified using statistical tests. Mice who had received fresh stools from patients with sporadic colorectal cancer developed precancerous lesions known as aberrant crypt foci (ACF) without any significant genetic changes to the colon, but they had a greater number of hypermethylated genes — which have been significantly linked to the incidence of ACF in the colonic mucosa.<br />
After verifying links between fecal dysbiosis and DNA anomalies (methylation) in the patients with sporadic colorectal cancer who took part in the fecal transplants, a pilot study was carried out in humans with the aim of developing a simple, reproducible blood test that can be used for early-stage diagnosis of colorectal tumors in asymptomatic patients. Prospective validation of the test was performed on 1,000 asymptomatic patients who were due to be given a colonoscopy. To identify the bacteria involved, their entire bacterial genome was sequenced. The level of hypermethylation of three genes was defined as a cumulative methylation index (CMI). The patients were classified according to their CMI (positive or negative). An analysis identified a positive CMI as a predictive factor for the onset of sporadic colorectal cancer.<br />
This research shows that the microbiota of subjects with sporadic colorectal cancer induces precancerous colonic lesions in animals by the hypermethylation of a small number of genes. The CMI and/or methylating bacteria could therefore be used as diagnostic markers for this type of cancer.</p>
</li>
<li>
<p><em>abstract</em> This study advances our appreciation and understanding of the role of colon dysbiosis in the pathogenesis of colorectal cancer. In a human pilot study of 266 individuals, greater epigenomic (methylation) DNA alterations correlated with CRC and microbiota composition. Beyond this correlative evidence, when germ-free mice received fresh feces from CRC patients and their healthy controls, the former animals developed colon epithelial renewal, more precancerous lesions, and increased tissue and blood DNA methylation in intestinal tissues. Confirmation was obtained in a larger cohort of 1,000 patients, indicating that CRC-associated dysbiosis may promote colon carcinogenesis via epigenome dysregulation. Gene methylation can therefore serve as a marker for CRC and likely for predicting efficacy of prebiotic supplementation in average-risk individuals.</p>
<p>Sporadic colorectal cancer (CRC) is a result of complex interactions between the host and its environment. Environmental stressors act by causing host cell DNA alterations implicated in the onset of cancer. Here we investigate the stressor ability of CRC-associated gut dysbiosis as causal agent of host DNA alterations. The epigenetic nature of these alterations was investigated in humans and in mice. Germ-free mice receiving fecal samples from subjects with normal colonoscopy or from CRC patients were monitored for 7 or 14 wk. Aberrant crypt foci, luminal microbiota, and DNA alterations (colonic exome sequencing and methylation patterns) were monitored following human feces transfer. CRC-associated microbiota induced higher numbers of hypermethylated genes in murine colonic mucosa (vs. healthy controls’ microbiota recipients). Several gene promoters including SFRP1,2,3, PENK, NPY, ALX4, SEPT9, and WIF1 promoters were found hypermethylated in CRC but not in normal tissues or effluents from fecal donors. In a pilot study (n = 266), the blood methylation levels of 3 genes (Wif1, PENK, and NPY) were shown closely associated with CRC dysbiosis. In a validation study (n = 1,000), the cumulative methylation index (CMI) of these genes was significantly higher in CRCs than in controls. Further, CMI appeared as an independent risk factor for CRC diagnosis as shown by multivariate analysis that included fecal immunochemical blood test. Consequently, fecal bacterial species in individuals with higher CMI in blood were identified by whole metagenomic analysis. Thus, CRC-related dysbiosis induces methylation of host genes, and corresponding CMIs together with associated bacteria are potential biomarkers for CRC.</p>
</li>
</ul>
</li>
</ul>
</li>
</ul>
<h2>probiotics</h2>
<h2>09094</h2>
<ul>
<li>
<p><em>lactobacillus reuteri reduces bone loss in older women with low bone mineral density - a randomized, placebo-controlled, double-blind, clinical trial</em><br />
anna g. nilsson et al. 2018<br />
<a href="http://doi.org/10.1111/joim.12805">doi.org/10.1111/joim.12805</a></p>
<ul>
<li>The importance of the gut microbiome for bone metabolism in mice has recently been demonstrated, but no studies are available in humans. <em>Lactobacillus reuteri</em> ATCCPTA 6475 (<em>L. reuteri</em> 6475) has been reported to increase bone mineral density (BMD) in mice but its effect on the human skeleton is unknown. The objective of this trial was to investigate if <em>L. reuteri</em> 6475 affects bone loss in older women with low BMD. In this double‐blind, placebo‐controlled study, women from the population who were 75 to 80 years old and had low BMD were randomized to orally receive 1010 colony‐forming units of <em>L. reuteri</em> 6475 daily or placebo. The predefined primary end‐point was relative change after 12 months in tibia total volumetric BMD (vBMD).<br />
Ninety women were included and 70 completed the study. <em>L. reuteri</em> 6475 reduced loss of total vBMD compared to placebo both in the intention‐to‐treat (ITT) analysis (‐0.83% [95% confidence interval [CI], ‐1.47 to ‐0.19%] vs. ‐1.85% [95% CI, ‐2.64 to ‐1.07%]; mean difference 1.02% [95% CI, 0.02 to 2.03]) and per protocol analysis (‐0.93% [95% CI, ‐1.45 to ‐0.40] vs. ‐1.86% [95% CI, ‐2.35 to ‐1.36]; mean difference 0.93% [95% CI, 0.21 to 1.65]. In general, similar but smaller effects were observed in the secondary bone variable outcomes, but these differences did not reach statistical significance in the ITT population. Adverse events did not differ between groups. In conclusion, supplementation with <em>L. reuteri</em> 6475 should be further explored as a novel approach to prevent age‐associated bone loss and osteoporosis.</li>
</ul>
</li>
<li>
<p><em>targeting the gut microbiome to treat the osteoarthritis of obesity</em><br />
eric m. schott et al. 2018<br />
<a href="http://doi.org/10.1172/jci.insight.95997">doi.org/10.1172/jci.insight.95997</a></p>
<ul>
<li>Obesity is a risk factor for osteoarthritis (OA), the greatest cause of disability in the US. The impact of obesity on OA is driven by systemic inflammation, and increased systemic inflammation is now understood to be caused by gut microbiome dysbiosis. Oligofructose, a nondigestible prebiotic fiber, can restore a lean gut microbial community profile in the context of obesity, suggesting a potentially novel approach to treat the OA of obesity. Here, we report that — compared with the lean murine gut — obesity is associated with loss of beneficial <em>Bifidobacteria</em>, while key proinflammatory species gain in abundance. A downstream systemic inflammatory signature culminates with macrophage migration to the synovium and accelerated knee OA. Oligofructose supplementation restores the lean gut microbiome in obese mice, in part, by supporting key commensal microflora, particularly <em>Bifidobacterium pseudolongum</em>. This is associated with reduced inflammation in the colon, circulation, and knee and protection from OA. This observation of a gut microbiome–OA connection sets the stage for discovery of potentially new OA therapeutics involving strategic manipulation of specific microbial species inhabiting the intestinal space.</li>
</ul>
</li>
</ul>
<h2>09194</h2>
<ul>
<li>
<p><em>psychobiotics and the manipulation of bacteria-brain signals</em><br />
sarkar et al. 2016<br />
<a href="http://doi.org/10.1016/j.tins.2016.09.002">doi.org/10.1016/j.tins.2016.09.002</a></p>
<ul>
<li>Psychobiotics were previously defined as live bacteria (probiotics) which, when ingested, confer mental health benefits through interactions with commensal gut bacteria. We expand this definition to encompass prebiotics, which enhance the growth of beneficial gut bacteria. We review probiotic and prebiotic effects on emotional, cognitive, systemic, and neural variables relevant to health and disease. We discuss gut–brain signalling mechanisms enabling psychobiotic effects, such as metabolite production. Overall, knowledge of how the microbiome responds to exogenous influence remains limited. We tabulate several important research questions and issues, exploration of which will generate both mechanistic insights and facilitate future psychobiotic development. We suggest the definition of psychobiotics be expanded beyond probiotics and prebiotics to include other means of influencing the microbiome.</li>
</ul>
</li>
<li>
<p>A new study finds that a noninvasive electromagnetic brain stimulation technique helps obese people lose weight, partly by changing the composition of their intestinal bacteria -- the so-called gut microbiota. Results of the technique, called deep transcranial magnetic stimulation (dTMS), will be presented Sunday at ENDO 2017, the Endocrine Society's 99th annual meeting in Orlando, Fla.<br />
<a href="https://www.sciencedaily.com/releases/2017/04/170403123508.htm">sciencedaily.com/releases/2017/04/170403123508.htm</a></p>
</li>
</ul>
<h2>09294</h2>
<ul>
<li>
<p><em>an exclusive metabolic niche enables strain engraftment in the gut microbiota</em><br />
elizabeth stanley shepherd et al. 2018<br />
<a href="http://doi.org/10.1038/s41586-018-0092-4">doi.org/10.1038/s41586-018-0092-4</a></p>
<ul>
<li>The dense microbial ecosystem in the gut is intimately connected to numerous facets of human biology, and manipulation of the gut microbiota has broad implications for human health. In the absence of profound perturbation, the bacterial strains that reside within an individual are mostly stable over time<a href="https://www.nature.com/articles/s41586-018-0092-4#ref-CR1">89</a>. By contrast, the fate of exogenous commensal and probiotic strains applied to an established microbiota is variable, generally unpredictable and greatly influenced by the background microbiota<a href="https://www.nature.com/articles/s41586-018-0092-4#ref-CR2">90</a>,<a href="https://www.nature.com/articles/s41586-018-0092-4#ref-CR3">91</a>. Therefore, analysis of the factors that govern strain engraftment and abundance is of critical importance to the emerging field of microbiome reprogramming. Here we generate an exclusive metabolic niche in mice via administration of a marine polysaccharide, porphyran, and an exogenous <em>Bacteroides</em> strain harbouring a rare gene cluster for porphyran utilization. Privileged nutrient access enables reliable engraftment of the exogenous strain at predictable abundances in mice harbouring diverse communities of gut microbes. This targeted dietary support is sufficient to overcome priority exclusion by an isogenic strain<a href="https://www.nature.com/articles/s41586-018-0092-4#ref-CR4">92</a>, and enables strain replacement. We demonstrate transfer of the 60-kb porphyran utilization locus into a naive strain of <em>Bacteroides</em>, and show finely tuned control of strain abundance in the mouse gut across multiple orders of magnitude by varying porphyran dosage. Finally, we show that this system enables the introduction of a new strain into the colonic crypt ecosystem. These data highlight the influence of nutrient availability in shaping microbiota membership, expand the ability to perform a broad spectrum of investigations in the context of a complex microbiota, and have implications for cell-based therapeutic strategies in the gut.</li>
</ul>
</li>
<li>
<p><em>cooperative metabolic adaptations in the host can favor asymptomatic infection and select for attenuated virulence in an enteric pathogen</em><br />
karina k. sanchez et al. 2018<br />
<a href="http://doi.org/10.1016/j.cell.2018.07.016">doi.org/10.1016/j.cell.2018.07.016</a></p>
<ul>
<li>
<p>Lethal dose 50 can be used to identify cooperative defense mechanisms<br />
Dietary iron promotes cooperative defenses against a lethal enteric infection<br />
Insulin resistance reduces gut glucose absorption<br />
Cooperative defenses drive selection for attenuated pathogen strains</p>
<p>Pathogen virulence exists on a continuum. The strategies that drive symptomatic or asymptomatic infections remain largely unknown. We took advantage of the concept of lethal dose 50 (LD50) to ask which component of individual non-genetic variation between hosts defines whether they survive or succumb to infection. Using the enteric pathogen Citrobacter, we found no difference in pathogen burdens between healthy and symptomatic populations. Iron metabolism-related genes were induced in asymptomatic hosts compared to symptomatic or naive mice. Dietary iron conferred complete protection without influencing pathogen burdens, even at 1000× the lethal dose of Citrobacter. Dietary iron induced insulin resistance, increasing glucose levels in the intestine that were necessary and sufficient to suppress pathogen virulence. A short course of dietary iron drove the selection of attenuated Citrobacter strains that can transmit and asymptomatically colonize naive hosts, demonstrating that environmental factors and cooperative metabolic strategies can drive conversion of pathogens toward commensalism.</p>
</li>
</ul>
</li>
</ul>
<h2>transplantation</h2>
<h2>09394</h2>
<ul>
<li>
<p><em>heterochronic faecal transplantation boosts gut germinal centres in aged mice</em><br />
marisa stebegg et al. 2019<br />
<a href="http://doi.org/10.1038/s41467-019-10430-7">doi.org/10.1038/s41467-019-10430-7</a></p>
<ul>
<li>
<p>The gut is one of the organs that is most severely affected by ageing and age-dependent changes to the human gut microbiome have been linked to increased frailty, inflammation and increased susceptibility to intestinal disorders. These age-dependent changes to the gut microbiome happen in parallel with a decrease in function of the gut immune system but, until now, it was unknown whether the two changes were linked.</p>
<p>&quot;Our gut microbiomes are made up of hundreds of different types of bacteria and these are essential to our health, playing a role in our metabolism, brain function and immune response,&quot; explains lead researcher Dr Marisa Stebegg. &quot;Our immune system is constantly interacting with the bacteria in the gastrointestinal tract. As immunologists who study why our immune system doesn't work as well as we age, we were interested to explore whether the make-up of the gut microbiome might influence the strength of the gut immune response.&quot;</p>
<p>Co-housing young and aged mice (mice naturally like to sample the faecal pellets of other mice!) or more directly performing faecal transfer from young to aged mice boosted the gut immune system in the aged mice, partly correcting the age-related decline.</p>
<p>&quot;To our surprise, co-housing rescued the reduced gut immune response in aged mice. Looking at the numbers of the immune cells involved, the aged mice possessed gut immune responses that were almost indistinguishable from those of the younger mice.&quot; commented Dr Michelle Linterman, group leader in the Immunology programme at the Babraham Institute.</p>
<p>The results show that the poor gut immune response is not irreversible and that the response can be strengthened by challenging with appropriate stimuli, essentially turning back the clock on the gut immune system to more closely resemble the situation in a young mouse.</p>
<p>The results of the study have relevance for treating age-related symptoms, confirming a link between the effects of the ageing immune system and age-associated changes in the gut microbiome. By demonstrating the effectiveness of interventions that have a positive impact on the composition of the gut microbiome, this research suggests that faecal transplants, probiotics, co-habitation and diet might all prove to be ways to facilitate healthy ageing</p>
</li>
<li>
<p><em>abstract</em> Ageing is a complex multifactorial process associated with a plethora of disorders, which contribute significantly to morbidity worldwide. One of the organs significantly affected by age is the gut. Age-dependent changes of the gut-associated microbiome have been linked to increased frailty and systemic inflammation. This change in microbial composition with age occurs in parallel with a decline in function of the gut immune system; however, it is not clear whether there is a causal link between the two. Here we report that the defective germinal centre reaction in Peyer’s patches of aged mice can be rescued by faecal transfers from younger adults into aged mice and by immunisations with cholera toxin, without affecting germinal centre reactions in peripheral lymph nodes. This demonstrates that the poor germinal centre reaction in aged animals is not irreversible, and that it is possible to improve this response in older individuals by providing appropriate stimuli.</p>
</li>
</ul>
</li>
<li>
<p><em>long-term effects on luminal and mucosal microbiota and commonly acquired taxa in faecal microbiota transplantation for recurrent clostridium difficile infection</em><br />
jonna jalanka et al. 2016<br />
<a href="http://doi.org/10.1186/s12916-016-0698-z">doi.org/10.1186/s12916-016-0698-z</a></p>
<ul>
<li>
<p>Background<br />
Faecal microbiota transplantation (FMT) is an effective treatment for recurrent Clostridium difficile infection (rCDI). It restores the disrupted intestinal microbiota and subsequently suppresses C. difficile. The long-term stability of the intestinal microbiota and the recovery of mucosal microbiota, both of which have not been previously studied, are assessed herein. Further, the specific bacteria behind the treatment efficacy are also investigated.</p>
<p>Methods<br />
We performed a high-throughput microbiota profiling using a phylogenetic microarray analysis of 131 faecal and mucosal samples from 14 rCDI patients pre- and post-FMT during a 1-year follow-up and 23 samples from the three universal donors over the same period.</p>
<p>Results<br />
The FMT treatment was successful in all patients. FMT reverted the patients’ bacterial community to become dominated by Clostridium clusters IV and XIVa, the major anaerobic bacterial groups of the healthy gut. In the mucosa, the amount of facultative anaerobes decreased, whereas Bacteroidetes increased. Post-FMT, the patients’ microbiota profiles were more similar to their own donors than what is generally observed for unrelated subjects and this striking similarity was retained throughout the 1-year follow-up. Furthermore, the universal donor approach allowed us to identify bacteria commonly established in all CDI patients and revealed a commonly acquired core microbiota consisting of 24 bacterial taxa.</p>
<p>Conclusions<br />
FMT induces profound microbiota changes, therefore explaining the high clinical efficacy for rCDI. The identification of commonly acquired bacteria could lead to effective bacteriotherapeutic formulations. FMT can affect microbiota in the long-term and offers a means to modify it relatively permanently for the treatment of microbiota-associated diseases.</p>
</li>
</ul>
</li>
</ul>
<h2>09494</h2>
<ul>
<li>
<p><em>effect of oral capsule– vs colonoscopy-delivered fecal microbiota transplantation on recurrent clostridium difficile infection: a randomized clinical trial</em><br />
dina kao et al. 2017<br />
<a href="http://doi.org/10.1001/jama.2017.17077">doi.org/10.1001/jama.2017.17077</a></p>
<ul>
<li>Question  Does the clinical efficacy of fecal microbiota transplantation (FMT) in treating recurrent <em>Clostridium difficile</em> infection (RCDI) depend on the route of delivery?<br />
Findings  In this noninferiority randomized clinical trial that included 116 adults with RCDI, the proportion without recurrence over 12 weeks was 96.2% after a single treatment in a group treated with oral capsules and in a group treated via colonoscopy, meeting the noninferiority margin of 15%.<br />
Meaning  FMT by oral capsules may be an effective approach to treating RCDI.<br />
Abstract<br />
Importance  Fecal microbiota transplantation (FMT) is effective in preventing recurrent <em>Clostridium difficile</em> infection (RCDI). However, it is not known whether clinical efficacy differs by route of delivery.<br />
Objective  To determine whether FMT by oral capsule is noninferior to colonoscopy delivery in efficacy.<br />
Design, Setting, and Participants  Noninferiority, unblinded, randomized trial conducted in 3 academic centers in Alberta, Canada. A total of 116 adult patients with RCDI were enrolled between October 2014 and September 2016, with follow-up to December 2016. The noninferiority margin was 15%.<br />
Interventions  Participants were randomly assigned to FMT by capsule or by colonoscopy at a 1:1 ratio.<br />
Main Outcomes and Measures  The primary outcome was the proportion of patients without RCDI 12 weeks after FMT. Secondary outcomes included (1) serious and minor adverse events, (2) changes in quality of life by the 36-Item Short Form Survey on a scale of 0 (worst possible quality of life) to 100 (best quality of life), and (3) patient perception on a scale of 1 (not at all unpleasant) to 10 (extremely unpleasant) and satisfaction on a scale of 1 (best) to 10 (worst).<br />
Results  Among 116 patients randomized (mean [SD] age, 58 <a href="http://doi.org/10.7554/eLife.18678">19</a> years; 79 women [68%]), 105 (91%) completed the trial, with 57 patients randomized to the capsule group and 59 to the colonoscopy group. In per-protocol analysis, prevention of RCDI after a single treatment was achieved in 96.2% in both the capsule group (51/53) and the colonoscopy group (50/52) (difference, 0%; 1-sided 95% CI, −6.1% to infinity; <em>P</em> &lt; .001), meeting the criterion for noninferiority. One patient in each group died of underlying cardiopulmonary illness unrelated to FMT. Rates of minor adverse events were 5.4% for the capsule group vs 12.5% for the colonoscopy group. There was no significant between-group difference in improvement in quality of life. A significantly greater proportion of participants receiving capsules rated their experience as “not at all unpleasant” (66% vs 44%; difference, 22% [95% CI, 3%-40%]; <em>P</em> = .01).<br />
Conclusions and Relevance  Among adults with RCDI, FMT via oral capsules was not inferior to delivery by colonoscopy for preventing recurrent infection over 12 weeks. Treatment with oral capsules may be an effective approach to treating RCDI.</li>
</ul>
</li>
</ul>
<h2>09594</h2>
<ul>
<li>
<p><em>monitoring biofilm function in new and matured full-scale slow sand filters using flow cytometric histogram image comparison (chic)</em><br />
sandy chan et al. 2018<br />
<a href="http://doi.org/10.1016/j.watres.2018.03.032">doi.org/10.1016/j.watres.2018.03.032</a></p>
<ul>
<li>
<p>•Removal of the schmutzdecke did not affect the function of established SSF.<br />
•Use of preconditioned sand accelerated deep sand biofilm maturation in new SSF.<br />
•FCM bacterial profiles correlated with the presence of microbial indicators.<br />
•FCM is useful for process control of SSFs.</p>
<p>While slow sand filters (SSFs) have produced drinking water for more than a hundred years, understanding of their associated microbial communities is limited. In this study, bacteria in influent and effluent water from full-scale SSFs were explored using flow cytometry (FCM) with cytometric histogram image comparison (CHIC) analysis; and routine microbial counts for heterotrophs, total coliforms and Escherichia coli. To assess if FCM can monitor biofilm function, SSFs differing in age and sand composition were compared. FCM profiles from two established filters were indistinguishable. To examine biofilm in the deep sand bed, SSFs were monitored during a scraping event, when the top layer of sand and the schmutzdecke are removed to restore flow through the filter. The performance of an established SSF was stable: total organic carbon (TOC), pH, numbers of heterotrophs, coliforms, E. coli, and FCM bacterial profile were unaffected by scraping. However, the performance of two newly-built SSFs containing new and mixed sand was compromised: breakthrough of both microbial indicators and TOC occurred following scraping. The compromised performance of the new SSFs was reflected in distinct effluent bacterial communities; and, the presence of microbial indicators correlated to influent bacterial communities. This demonstrated that FCM can monitor SSF performance. Removal of the top layer of sand did not alter the effluent water from the established SSF, but did affect that of the SSFs containing new sand. This suggests that the impact of the surface biofilm on effluent water is greater when the deep sand bed biofilm is not established.</p>
</li>
</ul>
</li>
</ul>
<h2>virus/phage</h2>
<h2>09694</h2>
<ul>
<li>
<p><em>predators catalyze an increase in chloroviruses by foraging on the symbiotic hosts of zoochlorellae</em><br />
john p. delong et al. 2017 al. 2016<br />
<a href="http://doi.org/10.1073/pnas.1613843113">doi.org/10.1073/pnas.1613843113</a></p>
<ul>
<li>
<p>Reproduction and growth of viruses depend on successful encounters with appropriate hosts. However, some hosts are difficult to encounter. In particular, chloroviruses cannot reach their target zoochlorellae hosts, because zoochlorellae are endosymbionts, living inside the cell of a protist that protects the zoochlorellae from the chlorovirus. The protist host is subject to predation, and we show that copepods foraging on zoochlorellae-bearing protists can disrupt the mutualism and pass endosymbiontic zoochlorellae through their guts, exposing them to chloroviruses. In this way, predators can catalyze the virus population growth by breaking down physical barriers between viruses and their endosymbiont hosts.</p>
<p>Virus population growth depends on contacts between viruses and their hosts. It is often unclear how sufficient contacts are made between viruses and their specific hosts to generate spikes in viral abundance. Here, we show that copepods, acting as predators, can bring aquatic viruses and their algal hosts into contact. Specifically, predation of the protist <em>Paramecium bursaria</em> by copepods resulted in a &gt;100-fold increase in the number of chloroviruses in 1 d. Copepod predation can be seen as an ecological “catalyst” by increasing contacts between chloroviruses and their hosts, zoochlorellae (endosymbiotic algae that live within paramecia), thereby facilitating viral population growth. When feeding, copepods passed <em>P. bursaria</em> through their digestive tract only partially digested, releasing endosymbiotic algae that still supported viral reproduction and resulting in a virus population spike. A simple predator–prey model parameterized for copepods consuming protists generates cycle periods for viruses consistent with those observed in natural ponds. Food webs are replete with similar symbiotic organisms, and we suspect the predator catalyst mechanism is capable of generating blooms for other endosymbiont-targeting viruses.</p>
</li>
</ul>
</li>
<li>
<p><em>deposition rates of viruses and bacteria above the atmospheric boundary layer</em><br />
isabel reche et al. 2018<br />
<a href="http://doi.org/10.1038/s41396-017-0042-4">doi.org/10.1038/s41396-017-0042-4</a></p>
<ul>
<li>Aerosolization of soil-dust and organic aggregates in sea spray facilitates the long-range transport of bacteria, and likely viruses across the free atmosphere. Although long-distance transport occurs, there are many uncertainties associated with their deposition rates. Here, we demonstrate that even in pristine environments, above the atmospheric boundary layer, the downward flux of viruses ranged from 0.26 × 109 to &gt;7 × 109 m−2 per day. These deposition rates were 9–461 times greater than the rates for bacteria, which ranged from 0.3 × 107 to &gt;8 × 107 m−2 per day. The highest relative deposition rates for viruses were associated with atmospheric transport from marine rather than terrestrial sources. Deposition rates of bacteria were significantly higher during rain events and Saharan dust intrusions, whereas, rainfall did not significantly influence virus deposition. Virus deposition rates were positively correlated with organic aerosols &lt;0.7 μm, whereas, bacteria were primarily associated with organic aerosols &gt;0.7 μm, implying that viruses could have longer residence times in the atmosphere and, consequently, will be dispersed further. These results provide an explanation for enigmatic observations that viruses with very high genetic identity can be found in very distant and different environments.</li>
</ul>
</li>
<li>
<p><em>vesicle-cloaked virus clusters are optimal units for inter-organismal viral transmission</em><br />
marianita santiana et al. 2018<br />
<a href="http://doi.org/10.1016/j.chom.2018.07.006">doi.org/10.1016/j.chom.2018.07.006</a></p>
<ul>
<li>
<p>Rotaviruses and noroviruses are shed in stool as viral clusters inside vesicles<br />
Vesicles containing virus clusters remain intact during fecal-oral transmission<br />
Vesicles achieve a high MOI and induce severe disease<br />
Vesicle-cloaked viral clusters are more virulent units than free viral particles</p>
<p>In enteric viral infections, such as those with rotavirus and norovirus, individual viral particles shed in stool are considered the optimal units of fecal-oral transmission. We reveal that rotaviruses and noroviruses are also shed in stool as viral clusters enclosed within vesicles that deliver a high inoculum to the receiving host. Cultured cells non-lytically release rotaviruses and noroviruses inside extracellular vesicles. In addition, stools of infected hosts contain norovirus and rotavirus within vesicles of exosomal or plasma membrane origin. These vesicles remain intact during fecal-oral transmission and thereby transport multiple viral particles collectively to the next host, enhancing both the MOI and disease severity. Vesicle-cloaked viruses are non-negligible populations in stool and have a disproportionately larger contribution to infectivity than free viruses. Our findings indicate that vesicle-cloaked viruses are highly virulent units of fecal-oral transmission and highlight a need for antivirals targeting vesicles and virus clustering.</p>
</li>
</ul>
</li>
</ul>
<h2>09794</h2>
<ul>
<li>
<p><em>dynamic modulation of the gut microbiota and metabolome by bacteriophages in a mouse model</em><br />
bryan b. hsu et al. 2019<br />
<a href="http://doi.org/10.1016/j.chom.2019.05.001">doi.org/10.1016/j.chom.2019.05.001</a></p>
<ul>
<li>
<p>added phages, tracking the growth of each microbe. Using high-throughput sequencing and computational analyses, the team found that the phage caused attritions of the species they preyed upon as expected, but with a rippling effect on the rest of the ecosystem including blooms of non-targeted species.</p>
<p>In addition to looking at the effects on microbes, the team also looked for effects on the metabolome -- chemical substances that can come from both the host and the bacteria present. They found that when they modulated the microbiome with phage, they could see targeted changes in the metabolome, including changes in neurotransmitter levels and bile acids.</p>
<p>&quot;This finding fascinates me for followup and raises significant questions: Could we use phage to modulate these activities? Could this be an intervention for conditions, such as depression, where you'd want to change neurotransmitter levels?&quot; said Gerber. &quot;Even if they aren't used as a direct therapeutic, our study suggests that phage may be a good tool for understanding the potential effects of other therapeutics that alter the microbiome.&quot;</p>
<p>Gerber and colleagues are especially interested in looking at the intersection of phage and malnutrition in the developing world, given the profound effects on the metabolome and microbiome that malnutrition can have.</p>
<p>&quot;We hope that our work will provide a framework to guide future investigations to elucidate the interplay between phage, the microbiota, and host health and disease,&quot; said Gerber.</p>
</li>
<li>
<p><em>abstract</em> •Phages can coexist over time with targeted gut bacteria<br />
•Phages induce cascading effects on microbiota species that are not directly targeted<br />
•Phage-induced bacterial modulation impacts the gut metabolome<br />
•Phages can modulate metabolites, which are known to affect the mammalian host</p>
<p>The human gut microbiome is comprised of densely colonizing microorganisms including bacteriophages, which are in dynamic interaction with each other and the mammalian host. To address how bacteriophages impact bacterial communities in the gut, we investigated the dynamic effects of phages on a model microbiome. Gnotobiotic mice were colonized with defined human gut commensal bacteria and subjected to predation by cognate lytic phages. We found that phage predation not only directly impacts susceptible bacteria but also leads to cascading effects on other bacterial species via interbacterial interactions. Metabolomic profiling revealed that shifts in the microbiome caused by phage predation have a direct consequence on the gut metabolome. Our work provides insight into the ecological importance of phages as modulators of bacterial colonization, and it additionally suggests the potential impact of gut phages on the mammalian host with implications for their therapeutic use to precisely modulate the microbiome.</p>
</li>
</ul>
</li>
</ul>
<h2>09894</h2>
<ul>
<li>not yet read
<ul>
<li>
<p><em>microbia: a journey into the unseen world around you</em><br />
eugenia bone 2018</p>
</li>
<li>
<p><em>the hidden half of nature: the microbial roots of life and health</em><br />
david montgomery, anne biklé 2015</p>
</li>
<li>
<p><em>the mind-gut connection: how the hidden conversation within our bodies impacts our mood, our choices, and our overall health</em><br />
emeran mayer 2016</p>
</li>
<li>
<p><em>follow your gut: the enormous impact of tiny microbes</em><br />
rob knight 2015</p>
</li>
<li>
<p><em>the psychobiotic revolution: mood, food and the new science of the gut-brain connection</em><br />
scott anderson et al. 2017</p>
</li>
<li>
<p><em>deadly companions how microbes shaped our history</em><br />
dorothy crawford 2018</p>
</li>
<li>
<p><em>human microbes the power within health healing and beyond</em><br />
vasu appanna 2018</p>
</li>
<li>
<p><em>natural defense: enlisting bugs and germs to protect our food and health</em><br />
emily monosson 2017</p>
</li>
<li>
<p><em>your baby’s microbiome: the critical role of vaginal birth and breastfeeding for lifelong health</em><br />
toni harman, alex wakeford 2018</p>
</li>
<li>
<p><em>clean: the new science of skin and the beauty of doing less</em><br />
james hamblin 2020</p>
</li>
</ul>
</li>
</ul>
<h2>09994</h2>
<h2>04994</h2>
<ul>
<li><em>bugs as drugs: therapeutic microbes for the prevention and treatment of disease</em><br />
robert britton, patrice cani 2018 to read next</li>
</ul>
<h2>04894</h2>
<h2>04794</h2>
<h2>04694</h2>
<ul>
<li>
<p><em>signal percolation within a bacterial community</em><br />
joseph w. larkin et al. 2018<br />
<a href="https://doi.org/10.1016/j.cels.2018.06.005">doi.org/10.1016/j.cels.2018.06.005</a></p>
<ul>
<li>
<p>•Biofilm cells exhibit heterogeneity (firing/non-firing) in electrical signaling<br />
•Firing cells arrange themselves into a theoretically predicted percolated network<br />
•Percolation model accounts for signal transmission in wild-type and mutant biofilms<br />
•Cost-benefit balance can organize the community at the critical percolation point</p>
<p>Signal transmission among cells enables long-range coordination in biological systems. However, the scarcity of quantitative measurements hinders the development of theories that relate signal propagation to cellular heterogeneity and spatial organization. We address this problem in a bacterial community that employs electrochemical cell-to-cell communication. We developed a model based on percolation theory, which describes how signals propagate through a heterogeneous medium. Our model predicts that signal transmission becomes possible when the community is organized near a critical phase transition between a disconnected and a fully connected conduit of signaling cells. By measuring population-level signal transmission with single-cell resolution in wild-type and genetically modified communities, we confirm that the spatial distribution of signaling cells is organized at the predicted phase transition. Our findings suggest that at this critical point, the population-level benefit of signal transmission outweighs the single-cell level cost. The bacterial community thus appears to be organized according to a theoretically predicted spatial heterogeneity that promotes efficient signal transmission.</p>
</li>
</ul>
</li>
</ul>
<h2>04594</h2>
<h2>04494</h2>
<h2>04394</h2>
<h2>04294</h2>
<h2>04194</h2>
<h2>04094</h2>
<h2>03994</h2>
<h2>03894</h2>
<h2>03794</h2>
<h2>03694</h2>
<h2>03594</h2>
<h2>03494</h2>
<h2>03394</h2>
<h2>03294</h2>
<h2>03194</h2>
<h2>03094</h2>
<h2>02994</h2>
<h2>02894</h2>
<h2>02794</h2>
<h2>02694</h2>
<h2>02594</h2>
<h2>02494</h2>
<h2>02394</h2>
<h2>02294</h2>
<h2>02194</h2>
<h2>02094</h2>
<h2>01994</h2>
<h2>01894</h2>
<h2>01794</h2>
<h2>01694</h2>
<h2>01594</h2>
<h2>01494</h2>
<h2>01394</h2>
<h2>01294</h2>
<h2>01194</h2>
<h2>01094</h2>
<h2>00994</h2>
<h2>00894</h2>
<h2>00794</h2>
<h2>00694</h2>
<h2>00594</h2>
<ul>
<li>
<p><em>two intracellular and cell type-specific bacterial symbionts in the placozoan trichoplax h2</em><br />
harald r. gruber-vodicka et al. 2019<br />
<a href="http://doi.org/10.1038/s41564-019-0475-9">doi.org/10.1038/s41564-019-0475-9</a></p>
<ul>
<li>
<p>Trichoplax, together with sponges and jellyfish, belongs to one of the most basal lineages of the animal kingdom. Until the 70ies, it was not even clear if Trichoplax is a proper, fully-grown animal or just the juvenile stage of a jellyfish. Only about a half a millimetre in diameter, these animals lack a mouth, gut and any other organs, and are made up of only six different kinds of cells. Its simplicity makes it a popular model organism for biologists.</p>
<p>Scientists from the Max Planck Institute for Marine Microbiology in Bremen, Germany, the University of Hawaii and North Carolina State University have now discovered that Trichoplax is not as simple as it looks. It lives in a remarkably sophisticated symbiosis with highly unusual bacteria.</p>
<p>Simple is beautiful</p>
<p>The first observation of bacteria in Trichoplax was nearly 50 years ago by the German zoologist Karl Grell. But no one has really taken a closer look since then. An international group of scientists around Harald Gruber-Vodicka, Niko Leisch and Nicole Dubilier from the Max Planck Institute for Marine Microbiology, and Michael Hadfield from the University of Hawaii have now investigated the bacterial tenants of Trichoplax by sequencing their genomes and using high-resolution microscopy to see where they live. &quot;Despite being so simple, Trichoplax harbors two very different and highly unusual bacterial symbionts in its cells,&quot; says Gruber-Vodicka. &quot;Both symbionts are very picky -- or cell-specific, as we call it. Each symbiont lives in only one type of host cell.&quot;</p>
<p>Grellia -- the first known symbiont to live in the endoplasmic reticulum</p>
<p>One symbiont, named Grellia after the zoologist Karl Grell, lives inside the endoplasmic reticulum (ER) of Trichoplax, and is the first symbiont known to permanently live in an animal's ER. The ER plays a central role in protein and membrane production. Proving that Grellia is truly in the ER was challenging. &quot;We reconstructed a detailed three-dimensional model of the ER to show that Grellia lives inside of it, supported by the electron microscopy facility of the Max Planck Institute of Molecular Cell Biology and Genetics in Dresden,&quot; Niko Leisch explains. &quot;Other parasitic bacteria imitate the structure of the ER to trick the hosts into thinking they are not harmful. However, our imaging data clearly showed that Grellia lives inside its host's ER.&quot; Intriguingly, Grellia, although closely related to parasites, doesn't appear to be harmful for Trichoplax. &quot;Although it has genes that would allow it to steal energy from its host, it does not use them,&quot; Leisch continues.</p>
<p>Ruthmannia -- seeing microbial dark matter</p>
<p>The second symbiont of Trichoplax, Ruthmannia, belongs to a group of bacteria that were only recently discovered, the Margulisbacteria. &quot;Before our study, Margulisbacteria were part of the so-called microbial dark matter -- the vast majority of microbial organisms that biologists find through sequencing, but are unable to culture,&quot; explains Harald Gruber-Vodicka. &quot;We have never actually seen them, even though their genetic traces were found in aquatic samples all over the globe.&quot; Now Gruber-Vodicka and Leisch took the first images of a Margulisbacteria. &quot;It's the first time we could see a member of this group. For us, observing this microbial dark matter was just as exciting as imaging black holes.&quot; This symbiont lives in cells that Trichoplax uses to digest its algal food. &quot;Ruthmannia appears to only eat the fats and other lipids of the algae, and leaves the rest to its host. In return, we think Ruthmannia may provide Trichoplax with vitamins and amino acids.&quot; With Trichoplax thriving in the lab cellars of the Max Planck Institute for Marine Microbiology, the authors now have continuous access to this enigmatic group of bacteria.</p>
<p>What's next</p>
<p>&quot;In this study, we focused on the symbiotic partners of a single Trichoplax species,&quot; says Nicole Dubilier, Director at the Max Planck Institute for Marine Microbiology. &quot;However, at least 20 more species have been described, and our first results indicate that each host species has its own, very specific set of symbionts. We are excited about taking a closer look at this remarkable diversity and how it evolved. These tiny animals not only look like potato chips, they also pack a crunch when it comes to what's inside them.&quot;</p>
</li>
<li>
<p><em>abstract</em> Placozoa is an enigmatic phylum of simple, microscopic, marine metazoans1,2. Although intracellular bacteria have been found in all members of this phylum, almost nothing is known about their identity, location and interactions with their host3,4,5,6. We used metagenomic and metatranscriptomic sequencing of single host individuals, plus metaproteomic and imaging analyses, to show that the placozoan Trichoplax sp. H2 lives in symbiosis with two intracellular bacteria. One symbiont forms an undescribed genus in the Midichloriaceae (Rickettsiales)7,8 and has a genomic repertoire similar to that of rickettsial parasites9,10, but does not seem to express key genes for energy parasitism. Correlative image analyses and three-dimensional electron tomography revealed that this symbiont resides in the rough endoplasmic reticulum of its host’s internal fibre cells. The second symbiont belongs to the Margulisbacteria, a phylum without cultured representatives and not known to form intracellular associations11,12,13. This symbiont lives in the ventral epithelial cells of Trichoplax, probably metabolizes algal lipids digested by its host and has the capacity to supplement the placozoan’s nutrition. Our study shows that one of the simplest animals has evolved highly specific and intimate associations with symbiotic, intracellular bacteria and highlights that symbioses can provide access to otherwise elusive microbial dark matter.</p>
</li>
</ul>
</li>
</ul>
<h2>00494</h2>
<h2>00394</h2>
<ul>
<li>
<p><em>bacteriophage trigger antiviral immunity and prevent clearance of bacterial infection</em><br />
johanna m. sweere et al. 2019<br />
<a href="http://doi.org/10.1126/science.aat9691">doi.org/10.1126/science.aat9691</a></p>
<ul>
<li>
<p>P. aeruginosa weaponizes its resident virus to exploit the immune system's distinct responses to bacterial versus viral infections.</p>
<p>This marks the first time a bacteria-infecting virus, otherwise known as a bacteriophage or just phage, has been observed inducing the immune system to mount an antiviral response and, in doing so, causing it to ignore the bacterial infection. When the scientists generated a vaccine directed at the virus, they showed that it dramatically lowered the bacteria's ability to infect wounds in mice.</p>
<p>Detailed in a study to be published March 29 in Science, the findings could fuel new ways of preventing chronic, intractable infections by keeping antibiotic-resistant bacteria from getting a foothold in the first place. The discovery that phages foster bacterial infections also adds a previously unexpected layer of complexity to the relationship between us and the billions of bacteria inhabiting our gut and other organs.</p>
<p>Paul Bollyky, MD, PhD, assistant professor of infectious diseases and of microbiology and immunology, is the study's senior author. The lead author is former graduate student Johanna Sweere, PhD.</p>
<p>Quadrillions of phages in body</p>
<p>&quot;We've long known that you've got up to 10 quadrillion phages in your body, but we just figured whatever they were doing was strictly between them and your commensal bacteria,&quot; Bollyky said. &quot;Now we know that phages can get inside your cells, too, and make you sick.&quot;</p>
<p>There's currently no approved vaccine targeting P. aeruginosa, an increasingly drug-resistant pathogen that infects the lungs of most adults with cystic fibrosis and accounts for a sizable percentage of all infections of diabetic ulcers, bedsores and burn wounds.</p>
<p>In 2017, the World Health Association named P. aeruginosa one of the &quot;critical priority&quot; pathogens posing the greatest threat to human health.</p>
<p>&quot;I see this every day in my clinical practice,&quot; Bollyky said. &quot;What starts off as a little cut can't heal as a result of a persistent, drug-resistant bacterial infection. The toll in terms of sickness, death and dollars is enormous.&quot; Infected diabetic foot ulcers are the single biggest cause of amputation, he said.</p>
<p>P. aeruginosa is itself frequently infected with a phage called Pf. This phage lives inside the bacteria but can be shed from the bacterial surface into the surrounding environment (such as a wound), much like the virus herpes lives in our cells and is shed from cold sores. In the study, Bollyky's team showed Pf was common in wounds infected with P. aeruginosa. The researchers examined 111 patients with microbially infected, non-healing wounds and found that 37 of them were infected with P. aeruginosa. Two-thirds of those wounds infected with P. aeruginosa were carrying Pf -- a fraction that grew the longer a wound persisted.</p>
<p>To prove Pf actually promotes P. aeruginosa infections rather than merely co-exists with them, the scientists inoculated small wounds in the skin of mice with P. aeruginosa strains that either did or didn't contain Pf. They observed that the two strains differed greatly in their ability to establish wound infections. The inoculation dose necessary to result in a reliable P. aeruginosa infection was 50 times larger if it lacked Pf.</p>
<p>Next, the scientists looked to see what Pf might be doing to immune cells that could affect P. aeruginosa's ability to sustain an infection. In a lab dish, they found that the presence of the phage in P. aeruginosa reduced by 10-fold the number of invading bacteria that were engulfed by either mouse or human phagocytes -- immune cells that ingest, then digest, invading bacteria.</p>
<p>&quot;The phagocytes lost their appetite,&quot; Bollyky said.</p>
<p>Tripping molecular detectors</p>
<p>Bollyky's team determined that stretches of the phage's genomic material trigger molecular detectors in the phagocytes, steering the immune system's response from an antibacterial to an antiviral one.</p>
<p>When a phagocyte encounters bacteria, the appropriate response is to gobble them up, chew them up and call in more troops. But phagocytes' response to a virus is different, Bollyky said. &quot;If you're an immune cell, ingesting a virus is absolutely the worst thing you can do, because now you've let it get inside of you -- you're infected by it.&quot;</p>
<p>So it's only sensible for a phagocyte that comes in contact with a virus to shut down phagocytosis. The appropriate antiviral immune response involves the generation of antibodies to tag virally infected cells and to signal other types of immune cells to home in on and destroy any virus-carrying cell they come across.</p>
<p>What Pf does inside phagocytes, Bollyky said, is like somebody pulling the fire alarm when they should have called the police. &quot;If 20 fire engines pull up to the scene of the crime, it makes it easier for the thief to get away,&quot; he said.</p>
<p>The investigators generated a vaccine containing a component of a Pf protein and noted that it cut the incidence of wounds infected with Pf-positive P. aeruginosa by half. They also generated antibodies that specifically target the same protein component and showed that they worked at least as well as the vaccine.</p>
<p>Bollyky and his colleagues have filed for a patent on intellectual property associated with the vaccine, and they plan to test it in large animals as a step toward eventual clinical trials.</p>
<p>Bollyky's vision is to vaccinate people against Pf when they're first diagnosed with cystic fibrosis or diabetes, as well as people in nursing homes and hospitals, in order to protect them from P. aeruginosa infections. Since a vaccine takes time to arouse the immune system, he suggested that Pf-targeting antibodies (which can be produced in bulk and stored for long periods) could be useful in burn cases, when there's no advance warning.</p>
<p>The Pf vaccine might turn out to be effective against other pathogenic bacteria, such as E. coli and Klebsiella pneumoniae, which can also carry Pf and tend to co-infect wounds colonized by P. aeruginosa, Bollyky said.</p>
</li>
<li>
<p><em>abstract</em><br />
We have identified previously unsuspected, directly pathogenic roles for bacteriophage (phage) virions in bacterial infections. In particular, we report that internalization of phage by human and murine immune cells triggers maladaptive viral pattern recognition receptors and suppressed bacterial clearance from infected wounds.<br />
RATIONALE<br />
Bacteriophage are abundant at sites of bacterial infection, but their effect on mammalian immunity is unclear. To investigate this, we studied Pseudomonas aeruginosa (Pa), a major human pathogen associated with chronic wounds and other infections, and Pf, a filamentous phage produced by Pa. Notably, Pf is lysogenic and its production does not typically destroy its bacterial host, unlike the lytic phage used in phage therapy for bacterial infections. Previous work had suggested that Pf phage are important in the pathogenesis of Pa infections, although the underlying mechanisms were unclear. Here, we have examined the impact of Pf on Pa wound infections in humans and in animal models.<br />
RESULTS<br />
We report that Pf bacteriophage were present in 25 of 37 (68%) Pa-infected wounds in our cohort. Furthermore, wounds infected with Pf-positive strains were significantly older than wounds infected with Pf-negative strains, and Pf was more commonly found in chronic, nonhealing wounds. Consistent with this finding, in a murine wound infection model, Pf-positive strains of Pa required an average of 50 times fewer bacteria than Pf-negative strains to establish wound infections. Additionally, mice infected with Pf-positive strains of Pa exhibited greater morbidity and mortality than mice infected with Pf-negative strains.</p>
<p>Mechanistically, these effects were associated with endocytosis of Pf phage by mammalian immune cells, both in vivo and in vitro. We found that uptake of Pf phage resulted in the production of phage RNA, which, in turn, triggered Toll-like receptor 3 (TLR3)– and TIR domain–containing adapter-inducing interferon-β (TRIF)–dependent type I interferon production, the inhibition of tumor necrosis factor production, and the suppression of phagocytosis. These data suggest that a natural (unmodified) bacteriophage may be able to produce mRNA within human cells.</p>
<p>Consistent with a pathogenic role for Pf phage, we report that a vaccine against Pf phage protects against Pa wound infections. Passive immunization of mice with monoclonal antibodies against Pf was likewise effective in protecting against Pa infection by enhancing the opsonization of Pa bacteria.<br />
CONCLUSION<br />
These results reveal direct, pathogenic roles for phage virions in bacterial infections. Building upon these insights, we report that vaccination against phage virions represents a potential therapeutic strategy for the prevention of infections by antibiotic-resistant Pa. These findings may have broad utility and impact beyond the pathophysiology of chronic wound infections. Pa is a major pathogen in other clinical settings as well, including lung infections in cystic fibrosis. Moreover, many other Gram-negative bacteria, including Klebsiella pneumoniae, Salmonella enterica, Vibrio cholerae, and Escherichia coli, have the capacity to harbor similar filamentous phage (genus Inovirus). Indeed, several of these phage are known to contribute to the virulence potential of their host bacteria. We propose that filamentous phage may be relevant to human interactions with a broad range of pathogenic and commensal bacteria and that these viruses may have profound, direct effects on human health and disease.</p>
</li>
</ul>
</li>
<li>
<p><em>15-keto-prostaglandin e2 activates host peroxisome proliferator-activated receptor gamma (ppar-γ) to promote cryptococcus neoformans growth during infection</em><br />
robert j. evans et al. 2019<br />
<a href="http://doi.org/10.1371/journal.ppat.1007597">doi.org/10.1371/journal.ppat.1007597</a></p>
<ul>
<li>
<p>Life-threatening fungal infection is a major killer of people with immune system problems such as blood cancers, HIV infection or following organ transplant.</p>
<p>The new study focused on one of the most dangerous infections for people with HIV/AIDS -- Cryptococcus neoformans -- which causes hundreds of thousands of deaths worldwide every year.</p>
<p>Fungi are known to make molecules similar to those of our own immune system, but why fungi make these molecules and what their function is has been a longstanding mystery.</p>
<p>Now, scientists from the University of Sheffield have identified how specific immune signals called prostaglandins, made by fungi, are able to disarm immune cells.</p>
<p>The team, led by Dr Simon Johnston from the University's Department of Infection, Immunity and Cardiovascular Disease, found that fungi which are not able to make these signals were less able to grow during infection.</p>
<p>Dr Johnston, Senior Research Fellow in Infectious Disease, said: &quot;We've discovered that these immune signals -- fungal prostaglandins -- deactivate immune cells, preventing them from destroying the infection.</p>
<p>&quot;We found the fungus was activating a normal immune pathway that prevents overstimulation of the immune system, but is essential in stopping infections.</p>
<p>&quot;Opportunistic infections like Cryptococcus -- which normally pose no threat, but are potentially life-threatening in those with weakened immune systems -- are an increasing problem and are often very difficult to treat.</p>
<p>&quot;Understanding how opportunistic infections cause disease is vital in order to develop new and more effective treatments, especially with the increase in antibiotic resistant infections.&quot;</p>
<p>Dr Johnston added: &quot;We are now working to find other ways these fungal molecules are affecting immune cells and how the immune cells are deactivated.</p>
<p>&quot;The same deactivation of immune cells is seen in other diseases such as cancer. Our findings mean that we now have a new approach to solving this problem and will help the development of new treatments.&quot;</p>
<p>The study, published today (28 March 2019) in the journal PLoS Pathogens was funded by the Medical Research Council (MRC) and British Infection Association.</p>
<p>Dr Anna Kinsey, programme manager for viral and fungal infections at the MRC, said: &quot;The MRC's investment in fungal research, and in future leaders in this field, is important.</p>
<p>&quot;Current anti-fungal therapies are poorly tolerated and toxic and, significantly, resistance to these agents is increasing. As such, there is an urgent need for new treatments, which first requires a better understanding of the interaction of the fungal pathogen with the body's immune system.</p>
<p>This research provides a window into how C. neoformans manipulates the immune system to promote its own growth and increase infection.&quot;</p>
</li>
<li>
<p><em>abstract</em> Cryptococcus neoformans is one of the leading causes of invasive fungal infection in humans worldwide. C. neoformans uses macrophages as a proliferative niche to increase infective burden and avoid immune surveillance. However, the specific mechanisms by which C. neoformans manipulates host immunity to promote its growth during infection remain ill-defined. Here we demonstrate that eicosanoid lipid mediators manipulated and/or produced by C. neoformans play a key role in regulating pathogenesis. C. neoformans is known to secrete several eicosanoids that are highly similar to those found in vertebrate hosts. Using eicosanoid deficient cryptococcal mutants Δplb1 and Δlac1, we demonstrate that prostaglandin E2 is required by C. neoformans for proliferation within macrophages and in vivo during infection. Genetic and pharmacological disruption of host PGE2 synthesis is not required for promotion of cryptococcal growth by eicosanoid production. We find that PGE2 must be dehydrogenated into 15-keto-PGE2 to promote fungal growth, a finding that implicated the host nuclear receptor PPAR-γ. C. neoformans infection of macrophages activates host PPAR-γ and its inhibition is sufficient to abrogate the effect of 15-keto-PGE2 in promoting fungal growth during infection. Thus, we describe the first mechanism of reliance on pathogen-derived eicosanoids in fungal pathogenesis and the specific role of 15-keto-PGE2 and host PPAR-γ in cryptococcosis.</p>
</li>
</ul>
</li>
</ul>
<h2>00294</h2>
<h2>00194</h2>
<h2>00094</h2>
<ul>
<li>plutonium Pu (pluto)</li>
<li>le petit prince
<ul>
<li>talking to a snake</li>
<li>snake</li>
</ul>
</li>
<li>ID</li>
</ul>
<h1>…</h1>

</div>

</body>
</html>
